{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "3ad289eb",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "import matplotlib.pyplot as plt\n",
    "import cobra\n",
    "import cbmpy\n",
    "from cobra.io import validate_sbml_model\n",
    "from unipressed import UniprotkbClient\n",
    "import libsbml\n",
    "import importlib\n",
    "from xml.etree import ElementTree\n",
    "import Model_correction as mc\n",
    "\n",
    "\n",
    "#import cplex"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "d6bdd08d",
   "metadata": {},
   "outputs": [],
   "source": [
    "Hep_G2_dir = \"../Stage/liver_models/Hep-G2/Hep-G2.xml-5a91f1955e8a9c8a5c5d2ff4a1737c21/\"\n",
    "iHep_dir = \"../Stage/liver_models/iHepatocytes2322/\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b8f1cfca",
   "metadata": {},
   "source": [
    "## Hep_G2 model : Modèle de cellule cancéreuse du foie. \n",
    "\n",
    "[Article](./Hep-G2/Hep-G2.pdf)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "7fec21af",
   "metadata": {},
   "source": [
    "### Model loading"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "5e95bcd4",
   "metadata": {},
   "outputs": [],
   "source": [
    "Hep_G2_complete, errors = validate_sbml_model(Hep_G2_dir+\"Hep-G2_cbmpy_wgenes_unip_ab.xml\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "46301088",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'SBML_FATAL': [],\n",
       " 'SBML_ERROR': [\"E6705 (Error): SBML component consistency (fbc, L168323); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q92506' has already been declared.\\n\",\n",
       "  \"E6706 (Error): SBML component consistency (fbc, L168609); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P12532' has already been declared.\\n\",\n",
       "  \"E6707 (Error): SBML component consistency (fbc, L169653); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O96024' has already been declared.\\n\",\n",
       "  \"E6708 (Error): SBML component consistency (fbc, L169671); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q99943' has already been declared.\\n\",\n",
       "  \"E6709 (Error): SBML component consistency (fbc, L170273); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_D6RAY0' has already been declared.\\n\",\n",
       "  \"E6710 (Error): SBML component consistency (fbc, L170274); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P40394' has already been declared.\\n\",\n",
       "  \"E6711 (Error): SBML component consistency (fbc, L170275); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8N4Q0' has already been declared.\\n\",\n",
       "  \"E6712 (Error): SBML component consistency (fbc, L170276); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00326' has already been declared.\\n\",\n",
       "  \"E6713 (Error): SBML component consistency (fbc, L170277); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P28332' has already been declared.\\n\",\n",
       "  \"E6714 (Error): SBML component consistency (fbc, L170278); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B4DFI7' has already been declared.\\n\",\n",
       "  \"E6715 (Error): SBML component consistency (fbc, L170279); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00325' has already been declared.\\n\",\n",
       "  \"E6716 (Error): SBML component consistency (fbc, L170280); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P08319' has already been declared.\\n\",\n",
       "  \"E6717 (Error): SBML component consistency (fbc, L170281); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07327' has already been declared.\\n\",\n",
       "  \"E6718 (Error): SBML component consistency (fbc, L170282); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P14550' has already been declared.\\n\",\n",
       "  \"E6719 (Error): SBML component consistency (fbc, L170283); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NR19' has already been declared.\\n\",\n",
       "  \"E6720 (Error): SBML component consistency (fbc, L170284); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NUB1' has already been declared.\\n\",\n",
       "  \"E6721 (Error): SBML component consistency (fbc, L170285); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H6R3' has already been declared.\\n\",\n",
       "  \"E6722 (Error): SBML component consistency (fbc, L170286); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P11177' has already been declared.\\n\",\n",
       "  \"E6723 (Error): SBML component consistency (fbc, L170287); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O00330' has already been declared.\\n\",\n",
       "  \"E6724 (Error): SBML component consistency (fbc, L170288); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P10515' has already been declared.\\n\",\n",
       "  \"E6725 (Error): SBML component consistency (fbc, L170289); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P08559' has already been declared.\\n\",\n",
       "  \"E6726 (Error): SBML component consistency (fbc, L170290); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P09622' has already been declared.\\n\",\n",
       "  \"E6727 (Error): SBML component consistency (fbc, L170291); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P29803' has already been declared.\\n\",\n",
       "  \"E6728 (Error): SBML component consistency (fbc, L170292); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8IX04' has already been declared.\\n\",\n",
       "  \"E6729 (Error): SBML component consistency (fbc, L170293); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07195' has already been declared.\\n\",\n",
       "  \"E6730 (Error): SBML component consistency (fbc, L170294); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BYZ2' has already been declared.\\n\",\n",
       "  \"E6731 (Error): SBML component consistency (fbc, L170295); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6ZMR3' has already been declared.\\n\",\n",
       "  \"E6732 (Error): SBML component consistency (fbc, L170296); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07864' has already been declared.\\n\",\n",
       "  \"E6733 (Error): SBML component consistency (fbc, L170297); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P43353' has already been declared.\\n\",\n",
       "  \"E6734 (Error): SBML component consistency (fbc, L170298); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P47895' has already been declared.\\n\",\n",
       "  \"E6735 (Error): SBML component consistency (fbc, L170299); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30838' has already been declared.\\n\",\n",
       "  \"E6736 (Error): SBML component consistency (fbc, L170300); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P48448' has already been declared.\\n\",\n",
       "  \"E6737 (Error): SBML component consistency (fbc, L170301); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49419' has already been declared.\\n\",\n",
       "  \"E6738 (Error): SBML component consistency (fbc, L170302); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P05091' has already been declared.\\n\",\n",
       "  \"E6739 (Error): SBML component consistency (fbc, L170303); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51648' has already been declared.\\n\",\n",
       "  \"E6740 (Error): SBML component consistency (fbc, L170304); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30837' has already been declared.\\n\",\n",
       "  \"E6741 (Error): SBML component consistency (fbc, L170305); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q01813' has already been declared.\\n\",\n",
       "  \"E6742 (Error): SBML component consistency (fbc, L170306); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_F8WEU2' has already been declared.\\n\",\n",
       "  \"E6743 (Error): SBML component consistency (fbc, L170307); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43822' has already been declared.\\n\",\n",
       "  \"E6744 (Error): SBML component consistency (fbc, L170308); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P08237' has already been declared.\\n\",\n",
       "  \"E6745 (Error): SBML component consistency (fbc, L170309); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04075' has already been declared.\\n\",\n",
       "  \"E6746 (Error): SBML component consistency (fbc, L170310); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P09972' has already been declared.\\n\",\n",
       "  \"E6747 (Error): SBML component consistency (fbc, L170311); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P05062' has already been declared.\\n\",\n",
       "  \"E6748 (Error): SBML component consistency (fbc, L170312); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30613' has already been declared.\\n\",\n",
       "  \"E6749 (Error): SBML component consistency (fbc, L170313); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q969K7' has already been declared.\\n\",\n",
       "  \"E6750 (Error): SBML component consistency (fbc, L170314); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P14618' has already been declared.\\n\",\n",
       "  \"E6751 (Error): SBML component consistency (fbc, L170315); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9P1Z3' has already been declared.\\n\",\n",
       "  \"E6752 (Error): SBML component consistency (fbc, L170316); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P13929' has already been declared.\\n\",\n",
       "  \"E6753 (Error): SBML component consistency (fbc, L170317); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R4F1' has already been declared.\\n\",\n",
       "  \"E6754 (Error): SBML component consistency (fbc, L170318); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P09104' has already been declared.\\n\",\n",
       "  \"E6755 (Error): SBML component consistency (fbc, L170319); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P18669' has already been declared.\\n\",\n",
       "  \"E6756 (Error): SBML component consistency (fbc, L170320); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P15259' has already been declared.\\n\",\n",
       "  \"E6757 (Error): SBML component consistency (fbc, L170321); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8N0Y7' has already been declared.\\n\",\n",
       "  \"E6758 (Error): SBML component consistency (fbc, L170322); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R782' has already been declared.\\n\",\n",
       "  \"E6759 (Error): SBML component consistency (fbc, L170323); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00558' has already been declared.\\n\",\n",
       "  \"E6760 (Error): SBML component consistency (fbc, L170324); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5JRA6' has already been declared.\\n\",\n",
       "  \"E6761 (Error): SBML component consistency (fbc, L170325); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07205' has already been declared.\\n\",\n",
       "  \"E6762 (Error): SBML component consistency (fbc, L170326); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P14621' has already been declared.\\n\",\n",
       "  \"E6763 (Error): SBML component consistency (fbc, L170327); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07311' has already been declared.\\n\",\n",
       "  \"E6764 (Error): SBML component consistency (fbc, L170328); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O14556' has already been declared.\\n\",\n",
       "  \"E6765 (Error): SBML component consistency (fbc, L170329); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04406' has already been declared.\\n\",\n",
       "  \"E6766 (Error): SBML component consistency (fbc, L170330); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P06744' has already been declared.\\n\",\n",
       "  \"E6767 (Error): SBML component consistency (fbc, L170331); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P60174' has already been declared.\\n\",\n",
       "  \"E6768 (Error): SBML component consistency (fbc, L170332); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q2TB90' has already been declared.\\n\",\n",
       "  \"E6769 (Error): SBML component consistency (fbc, L170333); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P19367' has already been declared.\\n\",\n",
       "  \"E6770 (Error): SBML component consistency (fbc, L170334); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P52790' has already been declared.\\n\",\n",
       "  \"E6771 (Error): SBML component consistency (fbc, L170335); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P52789' has already been declared.\\n\",\n",
       "  \"E6772 (Error): SBML component consistency (fbc, L170336); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BRR6' has already been declared.\\n\",\n",
       "  \"E6773 (Error): SBML component consistency (fbc, L170337); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P35557' has already been declared.\\n\",\n",
       "  \"E6774 (Error): SBML component consistency (fbc, L170338); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96G03' has already been declared.\\n\",\n",
       "  \"E6775 (Error): SBML component consistency (fbc, L170339); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15124' has already been declared.\\n\",\n",
       "  \"E6776 (Error): SBML component consistency (fbc, L170340); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P36871' has already been declared.\\n\",\n",
       "  \"E6777 (Error): SBML component consistency (fbc, L170341); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NQR9' has already been declared.\\n\",\n",
       "  \"E6778 (Error): SBML component consistency (fbc, L170342); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P35575' has already been declared.\\n\",\n",
       "  \"E6779 (Error): SBML component consistency (fbc, L170343); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_K7ENK1' has already been declared.\\n\",\n",
       "  \"E6780 (Error): SBML component consistency (fbc, L170344); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96C23' has already been declared.\\n\",\n",
       "  \"E6781 (Error): SBML component consistency (fbc, L170345); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04040' has already been declared.\\n\",\n",
       "  \"E6782 (Error): SBML component consistency (fbc, L170346); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P20807' has already been declared.\\n\",\n",
       "  \"E6783 (Error): SBML component consistency (fbc, L170347); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P10253' has already been declared.\\n\",\n",
       "  \"E6784 (Error): SBML component consistency (fbc, L170348); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_H3BRQ6' has already been declared.\\n\",\n",
       "  \"E6785 (Error): SBML component consistency (fbc, L170349); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60701' has already been declared.\\n\",\n",
       "  \"E6786 (Error): SBML component consistency (fbc, L170350); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43280' has already been declared.\\n\",\n",
       "  \"E6787 (Error): SBML component consistency (fbc, L170351); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q32M88' has already been declared.\\n\",\n",
       "  \"E6788 (Error): SBML component consistency (fbc, L170352); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_G5E9W8' has already been declared.\\n\",\n",
       "  \"E6789 (Error): SBML component consistency (fbc, L170353); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O15488' has already been declared.\\n\",\n",
       "  \"E6790 (Error): SBML component consistency (fbc, L170354); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54840' has already been declared.\\n\",\n",
       "  \"E6791 (Error): SBML component consistency (fbc, L170355); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P13807' has already been declared.\\n\",\n",
       "  \"E6792 (Error): SBML component consistency (fbc, L170356); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UQK1' has already been declared.\\n\",\n",
       "  \"E6793 (Error): SBML component consistency (fbc, L170357); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q86XI6' has already been declared.\\n\",\n",
       "  \"E6794 (Error): SBML component consistency (fbc, L170358); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q04446' has already been declared.\\n\",\n",
       "  \"E6795 (Error): SBML component consistency (fbc, L170359); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P06737' has already been declared.\\n\",\n",
       "  \"E6796 (Error): SBML component consistency (fbc, L170360); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P11217' has already been declared.\\n\",\n",
       "  \"E6797 (Error): SBML component consistency (fbc, L170361); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P11216' has already been declared.\\n\",\n",
       "  \"E6798 (Error): SBML component consistency (fbc, L170362); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P35573' has already been declared.\\n\",\n",
       "  \"E6799 (Error): SBML component consistency (fbc, L170363); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P0DTE8' has already been declared.\\n\",\n",
       "  \"E6800 (Error): SBML component consistency (fbc, L170364); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04746' has already been declared.\\n\",\n",
       "  \"E6801 (Error): SBML component consistency (fbc, L170365); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P0DUB6' has already been declared.\\n\",\n",
       "  \"E6802 (Error): SBML component consistency (fbc, L170366); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P19961' has already been declared.\\n\",\n",
       "  \"E6803 (Error): SBML component consistency (fbc, L170367); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P0DTE7' has already been declared.\\n\",\n",
       "  \"E6804 (Error): SBML component consistency (fbc, L170368); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_C9J3M0' has already been declared.\\n\",\n",
       "  \"E6805 (Error): SBML component consistency (fbc, L170369); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5QPP1' has already been declared.\\n\",\n",
       "  \"E6806 (Error): SBML component consistency (fbc, L170370); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51570' has already been declared.\\n\",\n",
       "  \"E6807 (Error): SBML component consistency (fbc, L170371); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q01415' has already been declared.\\n\",\n",
       "  \"E6808 (Error): SBML component consistency (fbc, L170372); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07902' has already been declared.\\n\",\n",
       "  \"E6809 (Error): SBML component consistency (fbc, L170373); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P14410' has already been declared.\\n\",\n",
       "  \"E6810 (Error): SBML component consistency (fbc, L170374); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96M60' has already been declared.\\n\",\n",
       "  \"E6811 (Error): SBML component consistency (fbc, L170375); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6UWU2' has already been declared.\\n\",\n",
       "  \"E6812 (Error): SBML component consistency (fbc, L170376); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P09848' has already been declared.\\n\",\n",
       "  \"E6813 (Error): SBML component consistency (fbc, L170377); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_F8WEN1' has already been declared.\\n\",\n",
       "  \"E6814 (Error): SBML component consistency (fbc, L170378); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A3B3IRU3' has already been declared.\\n\",\n",
       "  \"E6815 (Error): SBML component consistency (fbc, L170379); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00709' has already been declared.\\n\",\n",
       "  \"E6816 (Error): SBML component consistency (fbc, L170380); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60909' has already been declared.\\n\",\n",
       "  \"E6817 (Error): SBML component consistency (fbc, L170381); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P15291' has already been declared.\\n\",\n",
       "  \"E6818 (Error): SBML component consistency (fbc, L170382); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P50053' has already been declared.\\n\",\n",
       "  \"E6819 (Error): SBML component consistency (fbc, L170383); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16875' has already been declared.\\n\",\n",
       "  \"E6820 (Error): SBML component consistency (fbc, L170384); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16877' has already been declared.\\n\",\n",
       "  \"E6821 (Error): SBML component consistency (fbc, L170385); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60825' has already been declared.\\n\",\n",
       "  \"E6822 (Error): SBML component consistency (fbc, L170386); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_I1Z9G3' has already been declared.\\n\",\n",
       "  \"E6823 (Error): SBML component consistency (fbc, L170387); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q00796' has already been declared.\\n\",\n",
       "  \"E6824 (Error): SBML component consistency (fbc, L170388); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60218' has already been declared.\\n\",\n",
       "  \"E6825 (Error): SBML component consistency (fbc, L170389); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P15121' has already been declared.\\n\",\n",
       "  \"E6826 (Error): SBML component consistency (fbc, L170390); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B4DYB8' has already been declared.\\n\",\n",
       "  \"E6827 (Error): SBML component consistency (fbc, L170391); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q92871' has already been declared.\\n\",\n",
       "  \"E6828 (Error): SBML component consistency (fbc, L170392); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_H3BM92' has already been declared.\\n\",\n",
       "  \"E6829 (Error): SBML component consistency (fbc, L170393); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R1P3' has already been declared.\\n\",\n",
       "  \"E6830 (Error): SBML component consistency (fbc, L170394); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5XPI4' has already been declared.\\n\",\n",
       "  \"E6831 (Error): SBML component consistency (fbc, L170395); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A804HJC3' has already been declared.\\n\",\n",
       "  \"E6832 (Error): SBML component consistency (fbc, L170396); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R329' has already been declared.\\n\",\n",
       "  \"E6833 (Error): SBML component consistency (fbc, L170397); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60547' has already been declared.\\n\",\n",
       "  \"E6834 (Error): SBML component consistency (fbc, L170398); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_E9PLH9' has already been declared.\\n\",\n",
       "  \"E6835 (Error): SBML component consistency (fbc, L170399); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q59H18' has already been declared.\\n\",\n",
       "  \"E6836 (Error): SBML component consistency (fbc, L170400); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8N0W3' has already been declared.\\n\",\n",
       "  \"E6837 (Error): SBML component consistency (fbc, L170401); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NQ88' has already been declared.\\n\",\n",
       "  \"E6838 (Error): SBML component consistency (fbc, L170402); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7Z4W1' has already been declared.\\n\",\n",
       "  \"E6839 (Error): SBML component consistency (fbc, L170403); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BY49' has already been declared.\\n\",\n",
       "  \"E6840 (Error): SBML component consistency (fbc, L170404); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BV79' has already been declared.\\n\",\n",
       "  \"E6841 (Error): SBML component consistency (fbc, L170405); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75191' has already been declared.\\n\",\n",
       "  \"E6842 (Error): SBML component consistency (fbc, L170406); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43488' has already been declared.\\n\",\n",
       "  \"E6843 (Error): SBML component consistency (fbc, L170407); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H477' has already been declared.\\n\",\n",
       "  \"E6844 (Error): SBML component consistency (fbc, L170408); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y673' has already been declared.\\n\",\n",
       "  \"E6845 (Error): SBML component consistency (fbc, L170409); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NBZ7' has already been declared.\\n\",\n",
       "  \"E6846 (Error): SBML component consistency (fbc, L170410); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_I3NI24' has already been declared.\\n\",\n",
       "  \"E6847 (Error): SBML component consistency (fbc, L170411); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q04760' has already been declared.\\n\",\n",
       "  \"E6848 (Error): SBML component consistency (fbc, L170412); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16775' has already been declared.\\n\",\n",
       "  \"E6849 (Error): SBML component consistency (fbc, L170413); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B4DED4' has already been declared.\\n\",\n",
       "  \"E6850 (Error): SBML component consistency (fbc, L170414); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6P9G0' has already been declared.\\n\",\n",
       "  \"E6851 (Error): SBML component consistency (fbc, L170415); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q86WU2' has already been declared.\\n\",\n",
       "  \"E6852 (Error): SBML component consistency (fbc, L170416); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_E9PND9' has already been declared.\\n\",\n",
       "  \"E6853 (Error): SBML component consistency (fbc, L170417); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23368' has already been declared.\\n\",\n",
       "  \"E6854 (Error): SBML component consistency (fbc, L170418); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P48163' has already been declared.\\n\",\n",
       "  \"E6855 (Error): SBML component consistency (fbc, L170419); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UIY3' has already been declared.\\n\",\n",
       "  \"E6856 (Error): SBML component consistency (fbc, L170420); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16822' has already been declared.\\n\",\n",
       "  \"E6857 (Error): SBML component consistency (fbc, L170421); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P11498' has already been declared.\\n\",\n",
       "  \"E6858 (Error): SBML component consistency (fbc, L170422); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UBQ7' has already been declared.\\n\",\n",
       "  \"E6859 (Error): SBML component consistency (fbc, L170423); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13825' has already been declared.\\n\",\n",
       "  \"E6860 (Error): SBML component consistency (fbc, L170424); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00338' has already been declared.\\n\",\n",
       "  \"E6861 (Error): SBML component consistency (fbc, L170425); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H845' has already been declared.\\n\",\n",
       "  \"E6862 (Error): SBML component consistency (fbc, L170426); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q709F0' has already been declared.\\n\",\n",
       "  \"E6863 (Error): SBML component consistency (fbc, L170427); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6JQN1' has already been declared.\\n\",\n",
       "  \"E6864 (Error): SBML component consistency (fbc, L170428); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UKU7' has already been declared.\\n\",\n",
       "  \"E6865 (Error): SBML component consistency (fbc, L170429); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P16219' has already been declared.\\n\",\n",
       "  \"E6866 (Error): SBML component consistency (fbc, L170430); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P11310' has already been declared.\\n\",\n",
       "  \"E6867 (Error): SBML component consistency (fbc, L170431); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P45954' has already been declared.\\n\",\n",
       "  \"E6868 (Error): SBML component consistency (fbc, L170432); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95822' has already been declared.\\n\",\n",
       "  \"E6869 (Error): SBML component consistency (fbc, L170433); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q02252' has already been declared.\\n\",\n",
       "  \"E6870 (Error): SBML component consistency (fbc, L170434); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15392' has already been declared.\\n\",\n",
       "  \"E6871 (Error): SBML component consistency (fbc, L170435); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6NVY1' has already been declared.\\n\",\n",
       "  \"E6872 (Error): SBML component consistency (fbc, L170436); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q92506' has already been declared.\\n\",\n",
       "  \"E6873 (Error): SBML component consistency (fbc, L170437); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8N4T8' has already been declared.\\n\",\n",
       "  \"E6874 (Error): SBML component consistency (fbc, L170438); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8N3Y7' has already been declared.\\n\",\n",
       "  \"E6875 (Error): SBML component consistency (fbc, L170439); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13363' has already been declared.\\n\",\n",
       "  \"E6876 (Error): SBML component consistency (fbc, L170440); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R7X6' has already been declared.\\n\",\n",
       "  \"E6877 (Error): SBML component consistency (fbc, L170441); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_D6RCD0' has already been declared.\\n\",\n",
       "  \"E6878 (Error): SBML component consistency (fbc, L170442); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H2F3' has already been declared.\\n\",\n",
       "  \"E6879 (Error): SBML component consistency (fbc, L170443); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NEX9' has already been declared.\\n\",\n",
       "  \"E6880 (Error): SBML component consistency (fbc, L170444); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NBN7' has already been declared.\\n\",\n",
       "  \"E6881 (Error): SBML component consistency (fbc, L170445); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q04828' has already been declared.\\n\",\n",
       "  \"E6882 (Error): SBML component consistency (fbc, L170446); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BUT1' has already been declared.\\n\",\n",
       "  \"E6883 (Error): SBML component consistency (fbc, L170447); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BPX1' has already been declared.\\n\",\n",
       "  \"E6884 (Error): SBML component consistency (fbc, L170448); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P80365' has already been declared.\\n\",\n",
       "  \"E6885 (Error): SBML component consistency (fbc, L170449); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9HBH5' has already been declared.\\n\",\n",
       "  \"E6886 (Error): SBML component consistency (fbc, L170450); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P17516' has already been declared.\\n\",\n",
       "  \"E6887 (Error): SBML component consistency (fbc, L170451); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BUP3' has already been declared.\\n\",\n",
       "  \"E6888 (Error): SBML component consistency (fbc, L170452); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5I0G3' has already been declared.\\n\",\n",
       "  \"E6889 (Error): SBML component consistency (fbc, L170453); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96NR8' has already been declared.\\n\",\n",
       "  \"E6890 (Error): SBML component consistency (fbc, L170454); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BWD1' has already been declared.\\n\",\n",
       "  \"E6891 (Error): SBML component consistency (fbc, L170455); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q86V21' has already been declared.\\n\",\n",
       "  \"E6892 (Error): SBML component consistency (fbc, L170456); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q02338' has already been declared.\\n\",\n",
       "  \"E6893 (Error): SBML component consistency (fbc, L170457); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54868' has already been declared.\\n\",\n",
       "  \"E6894 (Error): SBML component consistency (fbc, L170458); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q01581' has already been declared.\\n\",\n",
       "  \"E6895 (Error): SBML component consistency (fbc, L170459); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A6Q8PG31' has already been declared.\\n\",\n",
       "  \"E6896 (Error): SBML component consistency (fbc, L170460); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21108' has already been declared.\\n\",\n",
       "  \"E6897 (Error): SBML component consistency (fbc, L170461); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P11908' has already been declared.\\n\",\n",
       "  \"E6898 (Error): SBML component consistency (fbc, L170462); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P11413' has already been declared.\\n\",\n",
       "  \"E6899 (Error): SBML component consistency (fbc, L170463); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NXR7' has already been declared.\\n\",\n",
       "  \"E6900 (Error): SBML component consistency (fbc, L170464); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49247' has already been declared.\\n\",\n",
       "  \"E6901 (Error): SBML component consistency (fbc, L170465); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y315' has already been declared.\\n\",\n",
       "  \"E6902 (Error): SBML component consistency (fbc, L170466); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H0I9' has already been declared.\\n\",\n",
       "  \"E6903 (Error): SBML component consistency (fbc, L170467); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51854' has already been declared.\\n\",\n",
       "  \"E6904 (Error): SBML component consistency (fbc, L170468); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P29401' has already been declared.\\n\",\n",
       "  \"E6905 (Error): SBML component consistency (fbc, L170469); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P52209' has already been declared.\\n\",\n",
       "  \"E6906 (Error): SBML component consistency (fbc, L170470); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96AT9' has already been declared.\\n\",\n",
       "  \"E6907 (Error): SBML component consistency (fbc, L170471); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P37837' has already been declared.\\n\",\n",
       "  \"E6908 (Error): SBML component consistency (fbc, L170472); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95336' has already been declared.\\n\",\n",
       "  \"E6909 (Error): SBML component consistency (fbc, L170473); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95479' has already been declared.\\n\",\n",
       "  \"E6910 (Error): SBML component consistency (fbc, L170474); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_K7EIP9' has already been declared.\\n\",\n",
       "  \"E6911 (Error): SBML component consistency (fbc, L170475); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75355' has already been declared.\\n\",\n",
       "  \"E6912 (Error): SBML component consistency (fbc, L170476); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y227' has already been declared.\\n\",\n",
       "  \"E6913 (Error): SBML component consistency (fbc, L170477); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5MY95' has already been declared.\\n\",\n",
       "  \"E6914 (Error): SBML component consistency (fbc, L170478); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y5L3' has already been declared.\\n\",\n",
       "  \"E6915 (Error): SBML component consistency (fbc, L170479); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49961' has already been declared.\\n\",\n",
       "  \"E6916 (Error): SBML component consistency (fbc, L170480); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R6D3' has already been declared.\\n\",\n",
       "  \"E6917 (Error): SBML component consistency (fbc, L170481); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75354' has already been declared.\\n\",\n",
       "  \"E6918 (Error): SBML component consistency (fbc, L170482); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60266' has already been declared.\\n\",\n",
       "  \"E6919 (Error): SBML component consistency (fbc, L170483); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P25092' has already been declared.\\n\",\n",
       "  \"E6920 (Error): SBML component consistency (fbc, L170484); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P20594' has already been declared.\\n\",\n",
       "  \"E6921 (Error): SBML component consistency (fbc, L170485); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6PJR4' has already been declared.\\n\",\n",
       "  \"E6922 (Error): SBML component consistency (fbc, L170486); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60503' has already been declared.\\n\",\n",
       "  \"E6923 (Error): SBML component consistency (fbc, L170487); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43306' has already been declared.\\n\",\n",
       "  \"E6924 (Error): SBML component consistency (fbc, L170488); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NFM4' has already been declared.\\n\",\n",
       "  \"E6925 (Error): SBML component consistency (fbc, L170489); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95622' has already been declared.\\n\",\n",
       "  \"E6926 (Error): SBML component consistency (fbc, L170490); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51828' has already been declared.\\n\",\n",
       "  \"E6927 (Error): SBML component consistency (fbc, L170491); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P40145' has already been declared.\\n\",\n",
       "  \"E6928 (Error): SBML component consistency (fbc, L170492); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q02153' has already been declared.\\n\",\n",
       "  \"E6929 (Error): SBML component consistency (fbc, L170493); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P33402' has already been declared.\\n\",\n",
       "  \"E6930 (Error): SBML component consistency (fbc, L170494); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q08828' has already been declared.\\n\",\n",
       "  \"E6931 (Error): SBML component consistency (fbc, L170495); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P16066' has already been declared.\\n\",\n",
       "  \"E6932 (Error): SBML component consistency (fbc, L170496); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96PN6' has already been declared.\\n\",\n",
       "  \"E6933 (Error): SBML component consistency (fbc, L170497); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51841' has already been declared.\\n\",\n",
       "  \"E6934 (Error): SBML component consistency (fbc, L170498); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q08462' has already been declared.\\n\",\n",
       "  \"E6935 (Error): SBML component consistency (fbc, L170499); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q02846' has already been declared.\\n\",\n",
       "  \"E6936 (Error): SBML component consistency (fbc, L170500); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UIJ7' has already been declared.\\n\",\n",
       "  \"E6937 (Error): SBML component consistency (fbc, L170501); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54819' has already been declared.\\n\",\n",
       "  \"E6938 (Error): SBML component consistency (fbc, L170502); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y6K8' has already been declared.\\n\",\n",
       "  \"E6939 (Error): SBML component consistency (fbc, L170503); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96MA6' has already been declared.\\n\",\n",
       "  \"E6940 (Error): SBML component consistency (fbc, L170504); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96M32' has already been declared.\\n\",\n",
       "  \"E6941 (Error): SBML component consistency (fbc, L170505); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P27144' has already been declared.\\n\",\n",
       "  \"E6942 (Error): SBML component consistency (fbc, L170506); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00568' has already been declared.\\n\",\n",
       "  \"E6943 (Error): SBML component consistency (fbc, L170507); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8TCD5' has already been declared.\\n\",\n",
       "  \"E6944 (Error): SBML component consistency (fbc, L170508); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49902' has already been declared.\\n\",\n",
       "  \"E6945 (Error): SBML component consistency (fbc, L170509); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024RA81' has already been declared.\\n\",\n",
       "  \"E6946 (Error): SBML component consistency (fbc, L170510); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BXI3' has already been declared.\\n\",\n",
       "  \"E6947 (Error): SBML component consistency (fbc, L170511); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q969T7' has already been declared.\\n\",\n",
       "  \"E6948 (Error): SBML component consistency (fbc, L170512); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21589' has already been declared.\\n\",\n",
       "  \"E6949 (Error): SBML component consistency (fbc, L170513); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P15309' has already been declared.\\n\",\n",
       "  \"E6950 (Error): SBML component consistency (fbc, L170514); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NPB1' has already been declared.\\n\",\n",
       "  \"E6951 (Error): SBML component consistency (fbc, L170515); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96P26' has already been declared.\\n\",\n",
       "  \"E6952 (Error): SBML component consistency (fbc, L170516); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P55263' has already been declared.\\n\",\n",
       "  \"E6953 (Error): SBML component consistency (fbc, L170517); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P27707' has already been declared.\\n\",\n",
       "  \"E6954 (Error): SBML component consistency (fbc, L170518); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_G3V1C5' has already been declared.\\n\",\n",
       "  \"E6955 (Error): SBML component consistency (fbc, L170519); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P18545' has already been declared.\\n\",\n",
       "  \"E6956 (Error): SBML component consistency (fbc, L170520); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51160' has already been declared.\\n\",\n",
       "  \"E6957 (Error): SBML component consistency (fbc, L170521); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q01064' has already been declared.\\n\",\n",
       "  \"E6958 (Error): SBML component consistency (fbc, L170522); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P35913' has already been declared.\\n\",\n",
       "  \"E6959 (Error): SBML component consistency (fbc, L170523); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A1B1UZQ1' has already been declared.\\n\",\n",
       "  \"E6960 (Error): SBML component consistency (fbc, L170524); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O76074' has already been declared.\\n\",\n",
       "  \"E6961 (Error): SBML component consistency (fbc, L170525); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13946' has already been declared.\\n\",\n",
       "  \"E6962 (Error): SBML component consistency (fbc, L170526); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95263' has already been declared.\\n\",\n",
       "  \"E6963 (Error): SBML component consistency (fbc, L170527); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P27815' has already been declared.\\n\",\n",
       "  \"E6964 (Error): SBML component consistency (fbc, L170528); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43924' has already been declared.\\n\",\n",
       "  \"E6965 (Error): SBML component consistency (fbc, L170529); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O76083' has already been declared.\\n\",\n",
       "  \"E6966 (Error): SBML component consistency (fbc, L170530); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13956' has already been declared.\\n\",\n",
       "  \"E6967 (Error): SBML component consistency (fbc, L170531); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q08499' has already been declared.\\n\",\n",
       "  \"E6968 (Error): SBML component consistency (fbc, L170532); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q14123' has already been declared.\\n\",\n",
       "  \"E6969 (Error): SBML component consistency (fbc, L170533); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q08493' has already been declared.\\n\",\n",
       "  \"E6970 (Error): SBML component consistency (fbc, L170534); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_F1T0K3' has already been declared.\\n\",\n",
       "  \"E6971 (Error): SBML component consistency (fbc, L170535); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NP56' has already been declared.\\n\",\n",
       "  \"E6972 (Error): SBML component consistency (fbc, L170536); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60658' has already been declared.\\n\",\n",
       "  \"E6973 (Error): SBML component consistency (fbc, L170537); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13370' has already been declared.\\n\",\n",
       "  \"E6974 (Error): SBML component consistency (fbc, L170538); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O00408' has already been declared.\\n\",\n",
       "  \"E6975 (Error): SBML component consistency (fbc, L170539); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q14432' has already been declared.\\n\",\n",
       "  \"E6976 (Error): SBML component consistency (fbc, L170540); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9HCR9' has already been declared.\\n\",\n",
       "  \"E6977 (Error): SBML component consistency (fbc, L170541); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q07343' has already been declared.\\n\",\n",
       "  \"E6978 (Error): SBML component consistency (fbc, L170542); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54750' has already been declared.\\n\",\n",
       "  \"E6979 (Error): SBML component consistency (fbc, L170543); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16774' has already been declared.\\n\",\n",
       "  \"E6980 (Error): SBML component consistency (fbc, L170544); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P31939' has already been declared.\\n\",\n",
       "  \"E6981 (Error): SBML component consistency (fbc, L170545); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_C9J381' has already been declared.\\n\",\n",
       "  \"E6982 (Error): SBML component consistency (fbc, L170546); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P12268' has already been declared.\\n\",\n",
       "  \"E6983 (Error): SBML component consistency (fbc, L170547); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8N142' has already been declared.\\n\",\n",
       "  \"E6984 (Error): SBML component consistency (fbc, L170548); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30520' has already been declared.\\n\",\n",
       "  \"E6985 (Error): SBML component consistency (fbc, L170549); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49915' has already been declared.\\n\",\n",
       "  \"E6986 (Error): SBML component consistency (fbc, L170550); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A5F9UK94' has already been declared.\\n\",\n",
       "  \"E6987 (Error): SBML component consistency (fbc, L170551); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23109' has already been declared.\\n\",\n",
       "  \"E6988 (Error): SBML component consistency (fbc, L170552); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q01432' has already been declared.\\n\",\n",
       "  \"E6989 (Error): SBML component consistency (fbc, L170553); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07741' has already been declared.\\n\",\n",
       "  \"E6990 (Error): SBML component consistency (fbc, L170554); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BY32' has already been declared.\\n\",\n",
       "  \"E6991 (Error): SBML component consistency (fbc, L170555); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A8K4H7' has already been declared.\\n\",\n",
       "  \"E6992 (Error): SBML component consistency (fbc, L170556); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A096LP92' has already been declared.\\n\",\n",
       "  \"E6993 (Error): SBML component consistency (fbc, L170557); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UPQ9' has already been declared.\\n\",\n",
       "  \"E6994 (Error): SBML component consistency (fbc, L170558); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00492' has already been declared.\\n\",\n",
       "  \"E6995 (Error): SBML component consistency (fbc, L170559); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9P2T1' has already been declared.\\n\",\n",
       "  \"E6996 (Error): SBML component consistency (fbc, L170560); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P36959' has already been declared.\\n\",\n",
       "  \"E6997 (Error): SBML component consistency (fbc, L170561); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O15254' has already been declared.\\n\",\n",
       "  \"E6998 (Error): SBML component consistency (fbc, L170562); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49748' has already been declared.\\n\",\n",
       "  \"E6999 (Error): SBML component consistency (fbc, L170563); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00813' has already been declared.\\n\",\n",
       "  \"E7000 (Error): SBML component consistency (fbc, L170564); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NZK5' has already been declared.\\n\",\n",
       "  \"E7001 (Error): SBML component consistency (fbc, L170565); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00491' has already been declared.\\n\",\n",
       "  \"E7002 (Error): SBML component consistency (fbc, L170566); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16854' has already been declared.\\n\",\n",
       "  \"E7003 (Error): SBML component consistency (fbc, L170567); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_E5KRG5' has already been declared.\\n\",\n",
       "  \"E7004 (Error): SBML component consistency (fbc, L170568); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P31350' has already been declared.\\n\",\n",
       "  \"E7005 (Error): SBML component consistency (fbc, L170569); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P10599' has already been declared.\\n\",\n",
       "  \"E7006 (Error): SBML component consistency (fbc, L170570); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7LG56' has already been declared.\\n\",\n",
       "  \"E7007 (Error): SBML component consistency (fbc, L170571); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23921' has already been declared.\\n\",\n",
       "  \"E7008 (Error): SBML component consistency (fbc, L170572); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22102' has already been declared.\\n\",\n",
       "  \"E7009 (Error): SBML component consistency (fbc, L170573); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22234' has already been declared.\\n\",\n",
       "  \"E7010 (Error): SBML component consistency (fbc, L170574); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O15067' has already been declared.\\n\",\n",
       "  \"E7011 (Error): SBML component consistency (fbc, L170575); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9HA47' has already been declared.\\n\",\n",
       "  \"E7012 (Error): SBML component consistency (fbc, L170576); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BZX2' has already been declared.\\n\",\n",
       "  \"E7013 (Error): SBML component consistency (fbc, L170577); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NWZ5' has already been declared.\\n\",\n",
       "  \"E7014 (Error): SBML component consistency (fbc, L170578); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P11678' has already been declared.\\n\",\n",
       "  \"E7015 (Error): SBML component consistency (fbc, L170579); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A1KZ92' has already been declared.\\n\",\n",
       "  \"E7016 (Error): SBML component consistency (fbc, L170580); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30041' has already been declared.\\n\",\n",
       "  \"E7017 (Error): SBML component consistency (fbc, L170581); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22079' has already been declared.\\n\",\n",
       "  \"E7018 (Error): SBML component consistency (fbc, L170582); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UKK9' has already been declared.\\n\",\n",
       "  \"E7019 (Error): SBML component consistency (fbc, L170583); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22392' has already been declared.\\n\",\n",
       "  \"E7020 (Error): SBML component consistency (fbc, L170584); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A0C4DG91' has already been declared.\\n\",\n",
       "  \"E7021 (Error): SBML component consistency (fbc, L170585); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q32Q12' has already been declared.\\n\",\n",
       "  \"E7022 (Error): SBML component consistency (fbc, L170586); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P56597' has already been declared.\\n\",\n",
       "  \"E7023 (Error): SBML component consistency (fbc, L170587); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O00746' has already been declared.\\n\",\n",
       "  \"E7024 (Error): SBML component consistency (fbc, L170588); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P15531' has already been declared.\\n\",\n",
       "  \"E7025 (Error): SBML component consistency (fbc, L170589); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y5B8' has already been declared.\\n\",\n",
       "  \"E7026 (Error): SBML component consistency (fbc, L170590); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13232' has already been declared.\\n\",\n",
       "  \"E7027 (Error): SBML component consistency (fbc, L170591); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NNW7' has already been declared.\\n\",\n",
       "  \"E7028 (Error): SBML component consistency (fbc, L170592); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16881' has already been declared.\\n\",\n",
       "  \"E7029 (Error): SBML component consistency (fbc, L170593); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q86VQ6' has already been declared.\\n\",\n",
       "  \"E7030 (Error): SBML component consistency (fbc, L170594); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30085' has already been declared.\\n\",\n",
       "  \"E7031 (Error): SBML component consistency (fbc, L170595); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NRF8' has already been declared.\\n\",\n",
       "  \"E7032 (Error): SBML component consistency (fbc, L170596); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P17812' has already been declared.\\n\",\n",
       "  \"E7033 (Error): SBML component consistency (fbc, L170597); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P27708' has already been declared.\\n\",\n",
       "  \"E7034 (Error): SBML component consistency (fbc, L170598); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A8K5J1' has already been declared.\\n\",\n",
       "  \"E7035 (Error): SBML component consistency (fbc, L170599); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O14638' has already been declared.\\n\",\n",
       "  \"E7036 (Error): SBML component consistency (fbc, L170600); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95848' has already been declared.\\n\",\n",
       "  \"E7037 (Error): SBML component consistency (fbc, L170601); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22413' has already been declared.\\n\",\n",
       "  \"E7038 (Error): SBML component consistency (fbc, L170602); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75063' has already been declared.\\n\",\n",
       "  \"E7039 (Error): SBML component consistency (fbc, L170603); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q14195' has already been declared.\\n\",\n",
       "  \"E7040 (Error): SBML component consistency (fbc, L170604); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16555' has already been declared.\\n\",\n",
       "  \"E7041 (Error): SBML component consistency (fbc, L170605); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q14117' has already been declared.\\n\",\n",
       "  \"E7042 (Error): SBML component consistency (fbc, L170606); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9GZX7' has already been declared.\\n\",\n",
       "  \"E7043 (Error): SBML component consistency (fbc, L170607); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P32320' has already been declared.\\n\",\n",
       "  \"E7044 (Error): SBML component consistency (fbc, L170608); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95045' has already been declared.\\n\",\n",
       "  \"E7045 (Error): SBML component consistency (fbc, L170609); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16831' has already been declared.\\n\",\n",
       "  \"E7046 (Error): SBML component consistency (fbc, L170610); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8WW27' has already been declared.\\n\",\n",
       "  \"E7047 (Error): SBML component consistency (fbc, L170611); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q02127' has already been declared.\\n\",\n",
       "  \"E7048 (Error): SBML component consistency (fbc, L170612); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23919' has already been declared.\\n\",\n",
       "  \"E7049 (Error): SBML component consistency (fbc, L170613); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P33316' has already been declared.\\n\",\n",
       "  \"E7050 (Error): SBML component consistency (fbc, L170614); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04818' has already been declared.\\n\",\n",
       "  \"E7051 (Error): SBML component consistency (fbc, L170615); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04183' has already been declared.\\n\",\n",
       "  \"E7052 (Error): SBML component consistency (fbc, L170616); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O00142' has already been declared.\\n\",\n",
       "  \"E7053 (Error): SBML component consistency (fbc, L170617); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5EBM0' has already been declared.\\n\",\n",
       "  \"E7054 (Error): SBML component consistency (fbc, L170618); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A384ME17' has already been declared.\\n\",\n",
       "  \"E7055 (Error): SBML component consistency (fbc, L170619); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P68104' has already been declared.\\n\",\n",
       "  \"E7056 (Error): SBML component consistency (fbc, L170620); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q05193' has already been declared.\\n\",\n",
       "  \"E7057 (Error): SBML component consistency (fbc, L170621); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B4DDQ3' has already been declared.\\n\",\n",
       "  \"E7058 (Error): SBML component consistency (fbc, L170622); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96RP9' has already been declared.\\n\",\n",
       "  \"E7059 (Error): SBML component consistency (fbc, L170623); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UQ16' has already been declared.\\n\",\n",
       "  \"E7060 (Error): SBML component consistency (fbc, L170624); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7Z2Z2' has already been declared.\\n\",\n",
       "  \"E7061 (Error): SBML component consistency (fbc, L170625); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q969S9' has already been declared.\\n\",\n",
       "  \"E7062 (Error): SBML component consistency (fbc, L170626); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P13639' has already been declared.\\n\",\n",
       "  \"E7063 (Error): SBML component consistency (fbc, L170627); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q05639' has already been declared.\\n\",\n",
       "  \"E7064 (Error): SBML component consistency (fbc, L170628); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95340' has already been declared.\\n\",\n",
       "  \"E7065 (Error): SBML component consistency (fbc, L170629); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43252' has already been declared.\\n\",\n",
       "  \"E7066 (Error): SBML component consistency (fbc, L170630); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_D6R9P0' has already been declared.\\n\",\n",
       "  \"E7067 (Error): SBML component consistency (fbc, L170631); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15054' has already been declared.\\n\",\n",
       "  \"E7068 (Error): SBML component consistency (fbc, L170632); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75417' has already been declared.\\n\",\n",
       "  \"E7069 (Error): SBML component consistency (fbc, L170633); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P06746' has already been declared.\\n\",\n",
       "  \"E7070 (Error): SBML component consistency (fbc, L170634); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BQG0' has already been declared.\\n\",\n",
       "  \"E7071 (Error): SBML component consistency (fbc, L170635); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_D6RJD0' has already been declared.\\n\",\n",
       "  \"E7072 (Error): SBML component consistency (fbc, L170636); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y253' has already been declared.\\n\",\n",
       "  \"E7073 (Error): SBML component consistency (fbc, L170637); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P09884' has already been declared.\\n\",\n",
       "  \"E7074 (Error): SBML component consistency (fbc, L170638); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49005' has already been declared.\\n\",\n",
       "  \"E7075 (Error): SBML component consistency (fbc, L170639); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P56282' has already been declared.\\n\",\n",
       "  \"E7076 (Error): SBML component consistency (fbc, L170640); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q14181' has already been declared.\\n\",\n",
       "  \"E7077 (Error): SBML component consistency (fbc, L170641); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UNA4' has already been declared.\\n\",\n",
       "  \"E7078 (Error): SBML component consistency (fbc, L170642); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BWT3' has already been declared.\\n\",\n",
       "  \"E7079 (Error): SBML component consistency (fbc, L170643); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P62487' has already been declared.\\n\",\n",
       "  \"E7080 (Error): SBML component consistency (fbc, L170644); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6ZS17' has already been declared.\\n\",\n",
       "  \"E7081 (Error): SBML component consistency (fbc, L170645); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P0DPB6' has already been declared.\\n\",\n",
       "  \"E7082 (Error): SBML component consistency (fbc, L170646); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O15160' has already been declared.\\n\",\n",
       "  \"E7083 (Error): SBML component consistency (fbc, L170647); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NRJ5' has already been declared.\\n\",\n",
       "  \"E7084 (Error): SBML component consistency (fbc, L170648); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O14802' has already been declared.\\n\",\n",
       "  \"E7085 (Error): SBML component consistency (fbc, L170649); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6PIY7' has already been declared.\\n\",\n",
       "  \"E7086 (Error): SBML component consistency (fbc, L170650); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P36954' has already been declared.\\n\",\n",
       "  \"E7087 (Error): SBML component consistency (fbc, L170651); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95602' has already been declared.\\n\",\n",
       "  \"E7088 (Error): SBML component consistency (fbc, L170652); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P61218' has already been declared.\\n\",\n",
       "  \"E7089 (Error): SBML component consistency (fbc, L170653); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A6Q8PGB0' has already been declared.\\n\",\n",
       "  \"E7090 (Error): SBML component consistency (fbc, L170654); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H1D9' has already been declared.\\n\",\n",
       "  \"E7091 (Error): SBML component consistency (fbc, L170655); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NW08' has already been declared.\\n\",\n",
       "  \"E7092 (Error): SBML component consistency (fbc, L170656); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O00411' has already been declared.\\n\",\n",
       "  \"E7093 (Error): SBML component consistency (fbc, L170657); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9GZS1' has already been declared.\\n\",\n",
       "  \"E7094 (Error): SBML component consistency (fbc, L170658); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O15514' has already been declared.\\n\",\n",
       "  \"E7095 (Error): SBML component consistency (fbc, L170659); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P19388' has already been declared.\\n\",\n",
       "  \"E7096 (Error): SBML component consistency (fbc, L170660); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P52434' has already been declared.\\n\",\n",
       "  \"E7097 (Error): SBML component consistency (fbc, L170661); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P52435' has already been declared.\\n\",\n",
       "  \"E7098 (Error): SBML component consistency (fbc, L170662); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NVV4' has already been declared.\\n\",\n",
       "  \"E7099 (Error): SBML component consistency (fbc, L170663); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BT43' has already been declared.\\n\",\n",
       "  \"E7100 (Error): SBML component consistency (fbc, L170664); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30876' has already been declared.\\n\",\n",
       "  \"E7101 (Error): SBML component consistency (fbc, L170665); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8TCS8' has already been declared.\\n\",\n",
       "  \"E7102 (Error): SBML component consistency (fbc, L170666); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60671' has already been declared.\\n\",\n",
       "  \"E7103 (Error): SBML component consistency (fbc, L170667); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49448' has already been declared.\\n\",\n",
       "  \"E7104 (Error): SBML component consistency (fbc, L170668); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00367' has already been declared.\\n\",\n",
       "  \"E7105 (Error): SBML component consistency (fbc, L170669); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00966' has already been declared.\\n\",\n",
       "  \"E7106 (Error): SBML component consistency (fbc, L170670); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75575' has already been declared.\\n\",\n",
       "  \"E7107 (Error): SBML component consistency (fbc, L170671); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04424' has already been declared.\\n\",\n",
       "  \"E7108 (Error): SBML component consistency (fbc, L170672); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024RAC7' has already been declared.\\n\",\n",
       "  \"E7109 (Error): SBML component consistency (fbc, L170673); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00505' has already been declared.\\n\",\n",
       "  \"E7110 (Error): SBML component consistency (fbc, L170674); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P17174' has already been declared.\\n\",\n",
       "  \"E7111 (Error): SBML component consistency (fbc, L170675); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NHS2' has already been declared.\\n\",\n",
       "  \"E7112 (Error): SBML component consistency (fbc, L170676); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07954' has already been declared.\\n\",\n",
       "  \"E7113 (Error): SBML component consistency (fbc, L170677); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7L266' has already been declared.\\n\",\n",
       "  \"E7114 (Error): SBML component consistency (fbc, L170678); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q86U10' has already been declared.\\n\",\n",
       "  \"E7115 (Error): SBML component consistency (fbc, L170679); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00390' has already been declared.\\n\",\n",
       "  \"E7116 (Error): SBML component consistency (fbc, L170680); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P31327' has already been declared.\\n\",\n",
       "  \"E7117 (Error): SBML component consistency (fbc, L170681); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5TDP6' has already been declared.\\n\",\n",
       "  \"E7118 (Error): SBML component consistency (fbc, L170682); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A8YXX4' has already been declared.\\n\",\n",
       "  \"E7119 (Error): SBML component consistency (fbc, L170683); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B7Z509' has already been declared.\\n\",\n",
       "  \"E7120 (Error): SBML component consistency (fbc, L170684); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_G5E9X4' has already been declared.\\n\",\n",
       "  \"E7121 (Error): SBML component consistency (fbc, L170685); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P24298' has already been declared.\\n\",\n",
       "  \"E7122 (Error): SBML component consistency (fbc, L170686); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P08243' has already been declared.\\n\",\n",
       "  \"E7123 (Error): SBML component consistency (fbc, L170687); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8TD30' has already been declared.\\n\",\n",
       "  \"E7124 (Error): SBML component consistency (fbc, L170688); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NQR4' has already been declared.\\n\",\n",
       "  \"E7125 (Error): SBML component consistency (fbc, L170689); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16773' has already been declared.\\n\",\n",
       "  \"E7126 (Error): SBML component consistency (fbc, L170690); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8N5Z0' has already been declared.\\n\",\n",
       "  \"E7127 (Error): SBML component consistency (fbc, L170691); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6YP21' has already been declared.\\n\",\n",
       "  \"E7128 (Error): SBML component consistency (fbc, L170692); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P80404' has already been declared.\\n\",\n",
       "  \"E7129 (Error): SBML component consistency (fbc, L170693); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P17735' has already been declared.\\n\",\n",
       "  \"E7130 (Error): SBML component consistency (fbc, L170694); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q99489' has already been declared.\\n\",\n",
       "  \"E7131 (Error): SBML component consistency (fbc, L170695); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O94903' has already been declared.\\n\",\n",
       "  \"E7132 (Error): SBML component consistency (fbc, L170696); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P45381' has already been declared.\\n\",\n",
       "  \"E7133 (Error): SBML component consistency (fbc, L170697); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96HD9' has already been declared.\\n\",\n",
       "  \"E7134 (Error): SBML component consistency (fbc, L170698); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04181' has already been declared.\\n\",\n",
       "  \"E7135 (Error): SBML component consistency (fbc, L170699); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P05089' has already been declared.\\n\",\n",
       "  \"E7136 (Error): SBML component consistency (fbc, L170700); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P78540' has already been declared.\\n\",\n",
       "  \"E7137 (Error): SBML component consistency (fbc, L170701); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B5MD87' has already been declared.\\n\",\n",
       "  \"E7138 (Error): SBML component consistency (fbc, L170702); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96C36' has already been declared.\\n\",\n",
       "  \"E7139 (Error): SBML component consistency (fbc, L170703); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R8U9' has already been declared.\\n\",\n",
       "  \"E7140 (Error): SBML component consistency (fbc, L170704); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_E7EQL6' has already been declared.\\n\",\n",
       "  \"E7141 (Error): SBML component consistency (fbc, L170705); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A590UKC3' has already been declared.\\n\",\n",
       "  \"E7142 (Error): SBML component consistency (fbc, L170706); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54886' has already been declared.\\n\",\n",
       "  \"E7143 (Error): SBML component consistency (fbc, L170707); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P35218' has already been declared.\\n\",\n",
       "  \"E7144 (Error): SBML component consistency (fbc, L170708); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y2D0' has already been declared.\\n\",\n",
       "  \"E7145 (Error): SBML component consistency (fbc, L170709); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P35228' has already been declared.\\n\",\n",
       "  \"E7146 (Error): SBML component consistency (fbc, L170710); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P29475' has already been declared.\\n\",\n",
       "  \"E7147 (Error): SBML component consistency (fbc, L170711); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P29474' has already been declared.\\n\",\n",
       "  \"E7148 (Error): SBML component consistency (fbc, L170712); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5VXN5' has already been declared.\\n\",\n",
       "  \"E7149 (Error): SBML component consistency (fbc, L170713); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P19623' has already been declared.\\n\",\n",
       "  \"E7150 (Error): SBML component consistency (fbc, L170714); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P52788' has already been declared.\\n\",\n",
       "  \"E7151 (Error): SBML component consistency (fbc, L170715); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96A70' has already been declared.\\n\",\n",
       "  \"E7152 (Error): SBML component consistency (fbc, L170716); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P11926' has already been declared.\\n\",\n",
       "  \"E7153 (Error): SBML component consistency (fbc, L170717); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P19801' has already been declared.\\n\",\n",
       "  \"E7154 (Error): SBML component consistency (fbc, L170718); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75106' has already been declared.\\n\",\n",
       "  \"E7155 (Error): SBML component consistency (fbc, L170719); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16853' has already been declared.\\n\",\n",
       "  \"E7156 (Error): SBML component consistency (fbc, L170720); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BSE5' has already been declared.\\n\",\n",
       "  \"E7157 (Error): SBML component consistency (fbc, L170721); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_V9HWB2' has already been declared.\\n\",\n",
       "  \"E7158 (Error): SBML component consistency (fbc, L170722); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A5YM72' has already been declared.\\n\",\n",
       "  \"E7159 (Error): SBML component consistency (fbc, L170723); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96KP4' has already been declared.\\n\",\n",
       "  \"E7160 (Error): SBML component consistency (fbc, L170724); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96KN2' has already been declared.\\n\",\n",
       "  \"E7161 (Error): SBML component consistency (fbc, L170725); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7Z4N8' has already been declared.\\n\",\n",
       "  \"E7162 (Error): SBML component consistency (fbc, L170726); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P13674' has already been declared.\\n\",\n",
       "  \"E7163 (Error): SBML component consistency (fbc, L170727); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q86XE5' has already been declared.\\n\",\n",
       "  \"E7164 (Error): SBML component consistency (fbc, L170728); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13126' has already been declared.\\n\",\n",
       "  \"E7165 (Error): SBML component consistency (fbc, L170729); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96F10' has already been declared.\\n\",\n",
       "  \"E7166 (Error): SBML component consistency (fbc, L170730); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21673' has already been declared.\\n\",\n",
       "  \"E7167 (Error): SBML component consistency (fbc, L170731); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NWM0' has already been declared.\\n\",\n",
       "  \"E7168 (Error): SBML component consistency (fbc, L170732); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6QHF9' has already been declared.\\n\",\n",
       "  \"E7169 (Error): SBML component consistency (fbc, L170733); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P17540' has already been declared.\\n\",\n",
       "  \"E7170 (Error): SBML component consistency (fbc, L170734); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P12532' has already been declared.\\n\",\n",
       "  \"E7171 (Error): SBML component consistency (fbc, L170735); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P06732' has already been declared.\\n\",\n",
       "  \"E7172 (Error): SBML component consistency (fbc, L170736); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P12277' has already been declared.\\n\",\n",
       "  \"E7173 (Error): SBML component consistency (fbc, L170737); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UL12' has already been declared.\\n\",\n",
       "  \"E7174 (Error): SBML component consistency (fbc, L170738); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P27338' has already been declared.\\n\",\n",
       "  \"E7175 (Error): SBML component consistency (fbc, L170739); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q53YE7' has already been declared.\\n\",\n",
       "  \"E7176 (Error): SBML component consistency (fbc, L170740); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51649' has already been declared.\\n\",\n",
       "  \"E7177 (Error): SBML component consistency (fbc, L170741); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5VV43' has already been declared.\\n\",\n",
       "  \"E7178 (Error): SBML component consistency (fbc, L170742); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A1LQE8' has already been declared.\\n\",\n",
       "  \"E7179 (Error): SBML component consistency (fbc, L170743); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A286YER3' has already been declared.\\n\",\n",
       "  \"E7180 (Error): SBML component consistency (fbc, L170744); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NBM4' has already been declared.\\n\",\n",
       "  \"E7181 (Error): SBML component consistency (fbc, L170745); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y617' has already been declared.\\n\",\n",
       "  \"E7182 (Error): SBML component consistency (fbc, L170746); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P78330' has already been declared.\\n\",\n",
       "  \"E7183 (Error): SBML component consistency (fbc, L170747); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P34896' has already been declared.\\n\",\n",
       "  \"E7184 (Error): SBML component consistency (fbc, L170748); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75600' has already been declared.\\n\",\n",
       "  \"E7185 (Error): SBML component consistency (fbc, L170749); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9P0Z9' has already been declared.\\n\",\n",
       "  \"E7186 (Error): SBML component consistency (fbc, L170750); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96GA7' has already been declared.\\n\",\n",
       "  \"E7187 (Error): SBML component consistency (fbc, L170751); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9GZT4' has already been declared.\\n\",\n",
       "  \"E7188 (Error): SBML component consistency (fbc, L170752); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P20132' has already been declared.\\n\",\n",
       "  \"E7189 (Error): SBML component consistency (fbc, L170753); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_V9HW60' has already been declared.\\n\",\n",
       "  \"E7190 (Error): SBML component consistency (fbc, L170754); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22557' has already been declared.\\n\",\n",
       "  \"E7191 (Error): SBML component consistency (fbc, L170755); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UBL6' has already been declared.\\n\",\n",
       "  \"E7192 (Error): SBML component consistency (fbc, L170756); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P13196' has already been declared.\\n\",\n",
       "  \"E7193 (Error): SBML component consistency (fbc, L170757); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21549' has already been declared.\\n\",\n",
       "  \"E7194 (Error): SBML component consistency (fbc, L170758); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NE62' has already been declared.\\n\",\n",
       "  \"E7195 (Error): SBML component consistency (fbc, L170759); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q93088' has already been declared.\\n\",\n",
       "  \"E7196 (Error): SBML component consistency (fbc, L170760); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H2M3' has already been declared.\\n\",\n",
       "  \"E7197 (Error): SBML component consistency (fbc, L170761); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NCN5' has already been declared.\\n\",\n",
       "  \"E7198 (Error): SBML component consistency (fbc, L170762); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UI17' has already been declared.\\n\",\n",
       "  \"E7199 (Error): SBML component consistency (fbc, L170763); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BYV1' has already been declared.\\n\",\n",
       "  \"E7200 (Error): SBML component consistency (fbc, L170764); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_V9HW06' has already been declared.\\n\",\n",
       "  \"E7201 (Error): SBML component consistency (fbc, L170765); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5JU23' has already been declared.\\n\",\n",
       "  \"E7202 (Error): SBML component consistency (fbc, L170766); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q92820' has already been declared.\\n\",\n",
       "  \"E7203 (Error): SBML component consistency (fbc, L170767); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23378' has already been declared.\\n\",\n",
       "  \"E7204 (Error): SBML component consistency (fbc, L170768); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23434' has already been declared.\\n\",\n",
       "  \"E7205 (Error): SBML component consistency (fbc, L170769); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P48728' has already been declared.\\n\",\n",
       "  \"E7206 (Error): SBML component consistency (fbc, L170770); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P19113' has already been declared.\\n\",\n",
       "  \"E7207 (Error): SBML component consistency (fbc, L170771); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P20711' has already been declared.\\n\",\n",
       "  \"E7208 (Error): SBML component consistency (fbc, L170772); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P50135' has already been declared.\\n\",\n",
       "  \"E7209 (Error): SBML component consistency (fbc, L170773); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q4VB95' has already been declared.\\n\",\n",
       "  \"E7210 (Error): SBML component consistency (fbc, L170774); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A0G2JQF2' has already been declared.\\n\",\n",
       "  \"E7211 (Error): SBML component consistency (fbc, L170775); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96NU7' has already been declared.\\n\",\n",
       "  \"E7212 (Error): SBML component consistency (fbc, L170776); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96N76' has already been declared.\\n\",\n",
       "  \"E7213 (Error): SBML component consistency (fbc, L170777); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y530' has already been declared.\\n\",\n",
       "  \"E7214 (Error): SBML component consistency (fbc, L170778); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q02083' has already been declared.\\n\",\n",
       "  \"E7215 (Error): SBML component consistency (fbc, L170779); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96HA8' has already been declared.\\n\",\n",
       "  \"E7216 (Error): SBML component consistency (fbc, L170780); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NTG7' has already been declared.\\n\",\n",
       "  \"E7217 (Error): SBML component consistency (fbc, L170781); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8IXJ6' has already been declared.\\n\",\n",
       "  \"E7218 (Error): SBML component consistency (fbc, L170782); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NXA8' has already been declared.\\n\",\n",
       "  \"E7219 (Error): SBML component consistency (fbc, L170783); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BQ69' has already been declared.\\n\",\n",
       "  \"E7220 (Error): SBML component consistency (fbc, L170784); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96EB6' has already been declared.\\n\",\n",
       "  \"E7221 (Error): SBML component consistency (fbc, L170785); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q86SQ9' has already been declared.\\n\",\n",
       "  \"E7222 (Error): SBML component consistency (fbc, L170786); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q53H54' has already been declared.\\n\",\n",
       "  \"E7223 (Error): SBML component consistency (fbc, L170787); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q12887' has already been declared.\\n\",\n",
       "  \"E7224 (Error): SBML component consistency (fbc, L170788); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H5J4' has already been declared.\\n\",\n",
       "  \"E7225 (Error): SBML component consistency (fbc, L170789); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9GZR5' has already been declared.\\n\",\n",
       "  \"E7226 (Error): SBML component consistency (fbc, L170790); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NXB9' has already been declared.\\n\",\n",
       "  \"E7227 (Error): SBML component consistency (fbc, L170791); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9HB03' has already been declared.\\n\",\n",
       "  \"E7228 (Error): SBML component consistency (fbc, L170792); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_D6RBM2' has already been declared.\\n\",\n",
       "  \"E7229 (Error): SBML component consistency (fbc, L170793); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BW60' has already been declared.\\n\",\n",
       "  \"E7230 (Error): SBML component consistency (fbc, L170794); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024RD35' has already been declared.\\n\",\n",
       "  \"E7231 (Error): SBML component consistency (fbc, L170795); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P36957' has already been declared.\\n\",\n",
       "  \"E7232 (Error): SBML component consistency (fbc, L170796); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96HY7' has already been declared.\\n\",\n",
       "  \"E7233 (Error): SBML component consistency (fbc, L170797); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q02218' has already been declared.\\n\",\n",
       "  \"E7234 (Error): SBML component consistency (fbc, L170798); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9ULD0' has already been declared.\\n\",\n",
       "  \"E7235 (Error): SBML component consistency (fbc, L170799); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UDR5' has already been declared.\\n\",\n",
       "  \"E7236 (Error): SBML component consistency (fbc, L170800); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NBX0' has already been declared.\\n\",\n",
       "  \"E7237 (Error): SBML component consistency (fbc, L170801); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H5I1' has already been declared.\\n\",\n",
       "  \"E7238 (Error): SBML component consistency (fbc, L170802); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q4FZB7' has already been declared.\\n\",\n",
       "  \"E7239 (Error): SBML component consistency (fbc, L170803); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_H0YEU4' has already been declared.\\n\",\n",
       "  \"E7240 (Error): SBML component consistency (fbc, L170804); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O15047' has already been declared.\\n\",\n",
       "  \"E7241 (Error): SBML component consistency (fbc, L170805); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UPS6' has already been declared.\\n\",\n",
       "  \"E7242 (Error): SBML component consistency (fbc, L170806); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BZ95' has already been declared.\\n\",\n",
       "  \"E7243 (Error): SBML component consistency (fbc, L170807); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8IZD2' has already been declared.\\n\",\n",
       "  \"E7244 (Error): SBML component consistency (fbc, L170808); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O96028' has already been declared.\\n\",\n",
       "  \"E7245 (Error): SBML component consistency (fbc, L170809); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96T68' has already been declared.\\n\",\n",
       "  \"E7246 (Error): SBML component consistency (fbc, L170810); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8TEK3' has already been declared.\\n\",\n",
       "  \"E7247 (Error): SBML component consistency (fbc, L170811); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H7B4' has already been declared.\\n\",\n",
       "  \"E7248 (Error): SBML component consistency (fbc, L170812); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15005' has already been declared.\\n\",\n",
       "  \"E7249 (Error): SBML component consistency (fbc, L170813); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y6A9' has already been declared.\\n\",\n",
       "  \"E7250 (Error): SBML component consistency (fbc, L170814); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P61009' has already been declared.\\n\",\n",
       "  \"E7251 (Error): SBML component consistency (fbc, L170815); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NVH6' has already been declared.\\n\",\n",
       "  \"E7252 (Error): SBML component consistency (fbc, L170816); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75936' has already been declared.\\n\",\n",
       "  \"E7253 (Error): SBML component consistency (fbc, L170817); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5T4B2' has already been declared.\\n\",\n",
       "  \"E7254 (Error): SBML component consistency (fbc, L170818); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O00469' has already been declared.\\n\",\n",
       "  \"E7255 (Error): SBML component consistency (fbc, L170819); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60568' has already been declared.\\n\",\n",
       "  \"E7256 (Error): SBML component consistency (fbc, L170820); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8IYK4' has already been declared.\\n\",\n",
       "  \"E7257 (Error): SBML component consistency (fbc, L170821); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q92800' has already been declared.\\n\",\n",
       "  \"E7258 (Error): SBML component consistency (fbc, L170822); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P40939' has already been declared.\\n\",\n",
       "  \"E7259 (Error): SBML component consistency (fbc, L170823); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30084' has already been declared.\\n\",\n",
       "  \"E7260 (Error): SBML component consistency (fbc, L170824); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A0S2Z410' has already been declared.\\n\",\n",
       "  \"E7261 (Error): SBML component consistency (fbc, L170825); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16836' has already been declared.\\n\",\n",
       "  \"E7262 (Error): SBML component consistency (fbc, L170826); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P24752' has already been declared.\\n\",\n",
       "  \"E7263 (Error): SBML component consistency (fbc, L170827); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P33261' has already been declared.\\n\",\n",
       "  \"E7264 (Error): SBML component consistency (fbc, L170828); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16678' has already been declared.\\n\",\n",
       "  \"E7265 (Error): SBML component consistency (fbc, L170829); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P05177' has already been declared.\\n\",\n",
       "  \"E7266 (Error): SBML component consistency (fbc, L170830); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04798' has already been declared.\\n\",\n",
       "  \"E7267 (Error): SBML component consistency (fbc, L170831); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P35914' has already been declared.\\n\",\n",
       "  \"E7268 (Error): SBML component consistency (fbc, L170832); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P50225' has already been declared.\\n\",\n",
       "  \"E7269 (Error): SBML component consistency (fbc, L170833); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P50226' has already been declared.\\n\",\n",
       "  \"E7270 (Error): SBML component consistency (fbc, L170834); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P55084' has already been declared.\\n\",\n",
       "  \"E7271 (Error): SBML component consistency (fbc, L170835); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P42765' has already been declared.\\n\",\n",
       "  \"E7272 (Error): SBML component consistency (fbc, L170836); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P05165' has already been declared.\\n\",\n",
       "  \"E7273 (Error): SBML component consistency (fbc, L170837); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P05166' has already been declared.\\n\",\n",
       "  \"E7274 (Error): SBML component consistency (fbc, L170838); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96PE7' has already been declared.\\n\",\n",
       "  \"E7275 (Error): SBML component consistency (fbc, L170839); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22033' has already been declared.\\n\",\n",
       "  \"E7276 (Error): SBML component consistency (fbc, L170840); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54687' has already been declared.\\n\",\n",
       "  \"E7277 (Error): SBML component consistency (fbc, L170841); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O15382' has already been declared.\\n\",\n",
       "  \"E7278 (Error): SBML component consistency (fbc, L170842); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P31937' has already been declared.\\n\",\n",
       "  \"E7279 (Error): SBML component consistency (fbc, L170843); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5VVL7' has already been declared.\\n\",\n",
       "  \"E7280 (Error): SBML component consistency (fbc, L170844); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P12694' has already been declared.\\n\",\n",
       "  \"E7281 (Error): SBML component consistency (fbc, L170845); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21953' has already been declared.\\n\",\n",
       "  \"E7282 (Error): SBML component consistency (fbc, L170846); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00439' has already been declared.\\n\",\n",
       "  \"E7283 (Error): SBML component consistency (fbc, L170847); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P14902' has already been declared.\\n\",\n",
       "  \"E7284 (Error): SBML component consistency (fbc, L170848); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_D6RA50' has already been declared.\\n\",\n",
       "  \"E7285 (Error): SBML component consistency (fbc, L170849); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q63HM1' has already been declared.\\n\",\n",
       "  \"E7286 (Error): SBML component consistency (fbc, L170850); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16719' has already been declared.\\n\",\n",
       "  \"E7287 (Error): SBML component consistency (fbc, L170851); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q92947' has already been declared.\\n\",\n",
       "  \"E7288 (Error): SBML component consistency (fbc, L170852); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96E66' has already been declared.\\n\",\n",
       "  \"E7289 (Error): SBML component consistency (fbc, L170853); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21964' has already been declared.\\n\",\n",
       "  \"E7290 (Error): SBML component consistency (fbc, L170854); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A0B4J1R4' has already been declared.\\n\",\n",
       "  \"E7291 (Error): SBML component consistency (fbc, L170855); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q93099' has already been declared.\\n\",\n",
       "  \"E7292 (Error): SBML component consistency (fbc, L170856); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A0C4DFM0' has already been declared.\\n\",\n",
       "  \"E7293 (Error): SBML component consistency (fbc, L170857); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P16930' has already been declared.\\n\",\n",
       "  \"E7294 (Error): SBML component consistency (fbc, L170858); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95050' has already been declared.\\n\",\n",
       "  \"E7295 (Error): SBML component consistency (fbc, L170859); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5DT02' has already been declared.\\n\",\n",
       "  \"E7296 (Error): SBML component consistency (fbc, L170860); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6UWM9' has already been declared.\\n\",\n",
       "  \"E7297 (Error): SBML component consistency (fbc, L170861); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75795' has already been declared.\\n\",\n",
       "  \"E7298 (Error): SBML component consistency (fbc, L170862); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22310' has already been declared.\\n\",\n",
       "  \"E7299 (Error): SBML component consistency (fbc, L170863); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54855' has already been declared.\\n\",\n",
       "  \"E7300 (Error): SBML component consistency (fbc, L170864); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75310' has already been declared.\\n\",\n",
       "  \"E7301 (Error): SBML component consistency (fbc, L170865); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P19224' has already been declared.\\n\",\n",
       "  \"E7302 (Error): SBML component consistency (fbc, L170866); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9HAW9' has already been declared.\\n\",\n",
       "  \"E7303 (Error): SBML component consistency (fbc, L170867); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P16662' has already been declared.\\n\",\n",
       "  \"E7304 (Error): SBML component consistency (fbc, L170868); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BY64' has already been declared.\\n\",\n",
       "  \"E7305 (Error): SBML component consistency (fbc, L170869); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P06133' has already been declared.\\n\",\n",
       "  \"E7306 (Error): SBML component consistency (fbc, L170870); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22309' has already been declared.\\n\",\n",
       "  \"E7307 (Error): SBML component consistency (fbc, L170871); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P36537' has already been declared.\\n\",\n",
       "  \"E7308 (Error): SBML component consistency (fbc, L170872); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P0DTE4' has already been declared.\\n\",\n",
       "  \"E7309 (Error): SBML component consistency (fbc, L170873); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P05164' has already been declared.\\n\",\n",
       "  \"E7310 (Error): SBML component consistency (fbc, L170874); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07202' has already been declared.\\n\",\n",
       "  \"E7311 (Error): SBML component consistency (fbc, L170875); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P78417' has already been declared.\\n\",\n",
       "  \"E7312 (Error): SBML component consistency (fbc, L170876); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H4Y5' has already been declared.\\n\",\n",
       "  \"E7313 (Error): SBML component consistency (fbc, L170877); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7RTV2' has already been declared.\\n\",\n",
       "  \"E7314 (Error): SBML component consistency (fbc, L170878); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16772' has already been declared.\\n\",\n",
       "  \"E7315 (Error): SBML component consistency (fbc, L170879); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y2Q3' has already been declared.\\n\",\n",
       "  \"E7316 (Error): SBML component consistency (fbc, L170880); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P28161' has already been declared.\\n\",\n",
       "  \"E7317 (Error): SBML component consistency (fbc, L170881); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P09488' has already been declared.\\n\",\n",
       "  \"E7318 (Error): SBML component consistency (fbc, L170882); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P46439' has already been declared.\\n\",\n",
       "  \"E7319 (Error): SBML component consistency (fbc, L170883); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P09211' has already been declared.\\n\",\n",
       "  \"E7320 (Error): SBML component consistency (fbc, L170884); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P10620' has already been declared.\\n\",\n",
       "  \"E7321 (Error): SBML component consistency (fbc, L170885); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P08263' has already been declared.\\n\",\n",
       "  \"E7322 (Error): SBML component consistency (fbc, L170886); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60760' has already been declared.\\n\",\n",
       "  \"E7323 (Error): SBML component consistency (fbc, L170887); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P09210' has already been declared.\\n\",\n",
       "  \"E7324 (Error): SBML component consistency (fbc, L170888); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q03013' has already been declared.\\n\",\n",
       "  \"E7325 (Error): SBML component consistency (fbc, L170889); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q99735' has already been declared.\\n\",\n",
       "  \"E7326 (Error): SBML component consistency (fbc, L170890); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49895' has already been declared.\\n\",\n",
       "  \"E7327 (Error): SBML component consistency (fbc, L170891); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R6J8' has already been declared.\\n\",\n",
       "  \"E7328 (Error): SBML component consistency (fbc, L170892); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43529' has already been declared.\\n\",\n",
       "  \"E7329 (Error): SBML component consistency (fbc, L170893); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7LGA3' has already been declared.\\n\",\n",
       "  \"E7330 (Error): SBML component consistency (fbc, L170894); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7L1S5' has already been declared.\\n\",\n",
       "  \"E7331 (Error): SBML component consistency (fbc, L170895); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_C9IZ77' has already been declared.\\n\",\n",
       "  \"E7332 (Error): SBML component consistency (fbc, L170896); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P52849' has already been declared.\\n\",\n",
       "  \"E7333 (Error): SBML component consistency (fbc, L170897); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NS84' has already been declared.\\n\",\n",
       "  \"E7334 (Error): SBML component consistency (fbc, L170898); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NCG5' has already been declared.\\n\",\n",
       "  \"E7335 (Error): SBML component consistency (fbc, L170899); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y2C2' has already been declared.\\n\",\n",
       "  \"E7336 (Error): SBML component consistency (fbc, L170900); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P52848' has already been declared.\\n\",\n",
       "  \"E7337 (Error): SBML component consistency (fbc, L170901); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95803' has already been declared.\\n\",\n",
       "  \"E7338 (Error): SBML component consistency (fbc, L170902); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43704' has already been declared.\\n\",\n",
       "  \"E7339 (Error): SBML component consistency (fbc, L170903); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96A11' has already been declared.\\n\",\n",
       "  \"E7340 (Error): SBML component consistency (fbc, L170904); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y4C5' has already been declared.\\n\",\n",
       "  \"E7341 (Error): SBML component consistency (fbc, L170905); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8IZP7' has already been declared.\\n\",\n",
       "  \"E7342 (Error): SBML component consistency (fbc, L170906); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O00338' has already been declared.\\n\",\n",
       "  \"E7343 (Error): SBML component consistency (fbc, L170907); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75897' has already been declared.\\n\",\n",
       "  \"E7344 (Error): SBML component consistency (fbc, L170908); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A590UJL0' has already been declared.\\n\",\n",
       "  \"E7345 (Error): SBML component consistency (fbc, L170909); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B2C6G4' has already been declared.\\n\",\n",
       "  \"E7346 (Error): SBML component consistency (fbc, L170910); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P31153' has already been declared.\\n\",\n",
       "  \"E7347 (Error): SBML component consistency (fbc, L170911); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q00266' has already been declared.\\n\",\n",
       "  \"E7348 (Error): SBML component consistency (fbc, L170912); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NZL9' has already been declared.\\n\",\n",
       "  \"E7349 (Error): SBML component consistency (fbc, L170913); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23526' has already been declared.\\n\",\n",
       "  \"E7350 (Error): SBML component consistency (fbc, L170914); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43865' has already been declared.\\n\",\n",
       "  \"E7351 (Error): SBML component consistency (fbc, L170915); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96HN2' has already been declared.\\n\",\n",
       "  \"E7352 (Error): SBML component consistency (fbc, L170916); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P35520' has already been declared.\\n\",\n",
       "  \"E7353 (Error): SBML component consistency (fbc, L170917); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P32929' has already been declared.\\n\",\n",
       "  \"E7354 (Error): SBML component consistency (fbc, L170918); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q99707' has already been declared.\\n\",\n",
       "  \"E7355 (Error): SBML component consistency (fbc, L170919); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P25325' has already been declared.\\n\",\n",
       "  \"E7356 (Error): SBML component consistency (fbc, L170920); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16762' has already been declared.\\n\",\n",
       "  \"E7357 (Error): SBML component consistency (fbc, L170921); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q674R7' has already been declared.\\n\",\n",
       "  \"E7358 (Error): SBML component consistency (fbc, L170922); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51687' has already been declared.\\n\",\n",
       "  \"E7359 (Error): SBML component consistency (fbc, L170923); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BV20' has already been declared.\\n\",\n",
       "  \"E7360 (Error): SBML component consistency (fbc, L170924); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96GX9' has already been declared.\\n\",\n",
       "  \"E7361 (Error): SBML component consistency (fbc, L170925); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UHY7' has already been declared.\\n\",\n",
       "  \"E7362 (Error): SBML component consistency (fbc, L170926); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BV57' has already been declared.\\n\",\n",
       "  \"E7363 (Error): SBML component consistency (fbc, L170927); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P40925' has already been declared.\\n\",\n",
       "  \"E7364 (Error): SBML component consistency (fbc, L170928); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P40926' has already been declared.\\n\",\n",
       "  \"E7365 (Error): SBML component consistency (fbc, L170929); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7Z3E4' has already been declared.\\n\",\n",
       "  \"E7366 (Error): SBML component consistency (fbc, L170930); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y6K1' has already been declared.\\n\",\n",
       "  \"E7367 (Error): SBML component consistency (fbc, L170931); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O14841' has already been declared.\\n\",\n",
       "  \"E7368 (Error): SBML component consistency (fbc, L170932); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A140VJJ2' has already been declared.\\n\",\n",
       "  \"E7369 (Error): SBML component consistency (fbc, L170933); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q06830' has already been declared.\\n\",\n",
       "  \"E7370 (Error): SBML component consistency (fbc, L170934); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30048' has already been declared.\\n\",\n",
       "  \"E7371 (Error): SBML component consistency (fbc, L170935); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P18283' has already been declared.\\n\",\n",
       "  \"E7372 (Error): SBML component consistency (fbc, L170936); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P36969' has already been declared.\\n\",\n",
       "  \"E7373 (Error): SBML component consistency (fbc, L170937); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22352' has already been declared.\\n\",\n",
       "  \"E7374 (Error): SBML component consistency (fbc, L170938); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P59796' has already been declared.\\n\",\n",
       "  \"E7375 (Error): SBML component consistency (fbc, L170939); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07203' has already been declared.\\n\",\n",
       "  \"E7376 (Error): SBML component consistency (fbc, L170940); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75715' has already been declared.\\n\",\n",
       "  \"E7377 (Error): SBML component consistency (fbc, L170941); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A6NIW5' has already been declared.\\n\",\n",
       "  \"E7378 (Error): SBML component consistency (fbc, L170942); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96SL4' has already been declared.\\n\",\n",
       "  \"E7379 (Error): SBML component consistency (fbc, L170943); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P36269' has already been declared.\\n\",\n",
       "  \"E7380 (Error): SBML component consistency (fbc, L170944); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B4DPG6' has already been declared.\\n\",\n",
       "  \"E7381 (Error): SBML component consistency (fbc, L170945); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P19440' has already been declared.\\n\",\n",
       "  \"E7382 (Error): SBML component consistency (fbc, L170946); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UJ14' has already been declared.\\n\",\n",
       "  \"E7383 (Error): SBML component consistency (fbc, L170947); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q14390' has already been declared.\\n\",\n",
       "  \"E7384 (Error): SBML component consistency (fbc, L170948); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q14TF0' has already been declared.\\n\",\n",
       "  \"E7385 (Error): SBML component consistency (fbc, L170949); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P48507' has already been declared.\\n\",\n",
       "  \"E7386 (Error): SBML component consistency (fbc, L170950); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A0S2Z4J7' has already been declared.\\n\",\n",
       "  \"E7387 (Error): SBML component consistency (fbc, L170951); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P15144' has already been declared.\\n\",\n",
       "  \"E7388 (Error): SBML component consistency (fbc, L170952); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P28838' has already been declared.\\n\",\n",
       "  \"E7389 (Error): SBML component consistency (fbc, L170953); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P55786' has already been declared.\\n\",\n",
       "  \"E7390 (Error): SBML component consistency (fbc, L170954); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75223' has already been declared.\\n\",\n",
       "  \"E7391 (Error): SBML component consistency (fbc, L170955); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96I15' has already been declared.\\n\",\n",
       "  \"E7392 (Error): SBML component consistency (fbc, L170956); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49326' has already been declared.\\n\",\n",
       "  \"E7393 (Error): SBML component consistency (fbc, L170957); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P31513' has already been declared.\\n\",\n",
       "  \"E7394 (Error): SBML component consistency (fbc, L170958); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q99518' has already been declared.\\n\",\n",
       "  \"E7395 (Error): SBML component consistency (fbc, L170959); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P31512' has already been declared.\\n\",\n",
       "  \"E7396 (Error): SBML component consistency (fbc, L170960); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q01740' has already been declared.\\n\",\n",
       "  \"E7397 (Error): SBML component consistency (fbc, L170961); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q06278' has already been declared.\\n\",\n",
       "  \"E7398 (Error): SBML component consistency (fbc, L170962); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P26440' has already been declared.\\n\",\n",
       "  \"E7399 (Error): SBML component consistency (fbc, L170963); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9HCC0' has already been declared.\\n\",\n",
       "  \"E7400 (Error): SBML component consistency (fbc, L170964); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96RQ3' has already been declared.\\n\",\n",
       "  \"E7401 (Error): SBML component consistency (fbc, L170965); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5URX0' has already been declared.\\n\",\n",
       "  \"E7402 (Error): SBML component consistency (fbc, L170966); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_H3BTD4' has already been declared.\\n\",\n",
       "  \"E7403 (Error): SBML component consistency (fbc, L170967); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q3KQV9' has already been declared.\\n\",\n",
       "  \"E7404 (Error): SBML component consistency (fbc, L170968); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16222' has already been declared.\\n\",\n",
       "  \"E7405 (Error): SBML component consistency (fbc, L170969); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P46926' has already been declared.\\n\",\n",
       "  \"E7406 (Error): SBML component consistency (fbc, L170970); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8TDQ7' has already been declared.\\n\",\n",
       "  \"E7407 (Error): SBML component consistency (fbc, L170971); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q06210' has already been declared.\\n\",\n",
       "  \"E7408 (Error): SBML component consistency (fbc, L170972); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A0S2Z4X9' has already been declared.\\n\",\n",
       "  \"E7409 (Error): SBML component consistency (fbc, L170973); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UJ70' has already been declared.\\n\",\n",
       "  \"E7410 (Error): SBML component consistency (fbc, L170974); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y223' has already been declared.\\n\",\n",
       "  \"E7411 (Error): SBML component consistency (fbc, L170975); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51606' has already been declared.\\n\",\n",
       "  \"E7412 (Error): SBML component consistency (fbc, L170976); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NR45' has already been declared.\\n\",\n",
       "  \"E7413 (Error): SBML component consistency (fbc, L170977); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8TBE9' has already been declared.\\n\",\n",
       "  \"E7414 (Error): SBML component consistency (fbc, L170978); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NFW8' has already been declared.\\n\",\n",
       "  \"E7415 (Error): SBML component consistency (fbc, L170979); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96EK6' has already been declared.\\n\",\n",
       "  \"E7416 (Error): SBML component consistency (fbc, L170980); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y303' has already been declared.\\n\",\n",
       "  \"E7417 (Error): SBML component consistency (fbc, L170981); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95394' has already been declared.\\n\",\n",
       "  \"E7418 (Error): SBML component consistency (fbc, L170982); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95455' has already been declared.\\n\",\n",
       "  \"E7419 (Error): SBML component consistency (fbc, L170983); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H8J5' has already been declared.\\n\",\n",
       "  \"E7420 (Error): SBML component consistency (fbc, L170984); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13231' has already been declared.\\n\",\n",
       "  \"E7421 (Error): SBML component consistency (fbc, L170985); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BZP6' has already been declared.\\n\",\n",
       "  \"E7422 (Error): SBML component consistency (fbc, L170986); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y2Z4' has already been declared.\\n\",\n",
       "  \"E7423 (Error): SBML component consistency (fbc, L170987); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54577' has already been declared.\\n\",\n",
       "  \"E7424 (Error): SBML component consistency (fbc, L170988); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5JTZ9' has already been declared.\\n\",\n",
       "  \"E7425 (Error): SBML component consistency (fbc, L170989); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49588' has already been declared.\\n\",\n",
       "  \"E7426 (Error): SBML component consistency (fbc, L170990); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5T160' has already been declared.\\n\",\n",
       "  \"E7427 (Error): SBML component consistency (fbc, L170991); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96I59' has already been declared.\\n\",\n",
       "  \"E7428 (Error): SBML component consistency (fbc, L170992); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43776' has already been declared.\\n\",\n",
       "  \"E7429 (Error): SBML component consistency (fbc, L170993); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P14868' has already been declared.\\n\",\n",
       "  \"E7430 (Error): SBML component consistency (fbc, L170994); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A3B3IT89' has already been declared.\\n\",\n",
       "  \"E7431 (Error): SBML component consistency (fbc, L170995); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9HA77' has already been declared.\\n\",\n",
       "  \"E7432 (Error): SBML component consistency (fbc, L170996); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P47897' has already been declared.\\n\",\n",
       "  \"E7433 (Error): SBML component consistency (fbc, L170997); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5JPH6' has already been declared.\\n\",\n",
       "  \"E7434 (Error): SBML component consistency (fbc, L170998); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07814' has already been declared.\\n\",\n",
       "  \"E7435 (Error): SBML component consistency (fbc, L170999); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A6Q8PGW4' has already been declared.\\n\",\n",
       "  \"E7436 (Error): SBML component consistency (fbc, L171000); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B4E1C5' has already been declared.\\n\",\n",
       "  \"E7437 (Error): SBML component consistency (fbc, L171001); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B4DQ67' has already been declared.\\n\",\n",
       "  \"E7438 (Error): SBML component consistency (fbc, L171002); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NSE4' has already been declared.\\n\",\n",
       "  \"E7439 (Error): SBML component consistency (fbc, L171003); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P41252' has already been declared.\\n\",\n",
       "  \"E7440 (Error): SBML component consistency (fbc, L171004); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9P2J5' has already been declared.\\n\",\n",
       "  \"E7441 (Error): SBML component consistency (fbc, L171005); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15031' has already been declared.\\n\",\n",
       "  \"E7442 (Error): SBML component consistency (fbc, L171006); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15046' has already been declared.\\n\",\n",
       "  \"E7443 (Error): SBML component consistency (fbc, L171007); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P56192' has already been declared.\\n\",\n",
       "  \"E7444 (Error): SBML component consistency (fbc, L171008); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96GW9' has already been declared.\\n\",\n",
       "  \"E7445 (Error): SBML component consistency (fbc, L171009); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95363' has already been declared.\\n\",\n",
       "  \"E7446 (Error): SBML component consistency (fbc, L171010); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7L3T8' has already been declared.\\n\",\n",
       "  \"E7447 (Error): SBML component consistency (fbc, L171011); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NP81' has already been declared.\\n\",\n",
       "  \"E7448 (Error): SBML component consistency (fbc, L171012); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P26639' has already been declared.\\n\",\n",
       "  \"E7449 (Error): SBML component consistency (fbc, L171013); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BW92' has already been declared.\\n\",\n",
       "  \"E7450 (Error): SBML component consistency (fbc, L171014); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R6K8' has already been declared.\\n\",\n",
       "  \"E7451 (Error): SBML component consistency (fbc, L171015); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A2ABL6' has already been declared.\\n\",\n",
       "  \"E7452 (Error): SBML component consistency (fbc, L171016); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P26640' has already been declared.\\n\",\n",
       "  \"E7453 (Error): SBML component consistency (fbc, L171017); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q11201' has already been declared.\\n\",\n",
       "  \"E7454 (Error): SBML component consistency (fbc, L171018); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16842' has already been declared.\\n\",\n",
       "  \"E7455 (Error): SBML component consistency (fbc, L171019); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6ZMB0' has already been declared.\\n\",\n",
       "  \"E7456 (Error): SBML component consistency (fbc, L171020); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95395' has already been declared.\\n\",\n",
       "  \"E7457 (Error): SBML component consistency (fbc, L171021); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9P109' has already been declared.\\n\",\n",
       "  \"E7458 (Error): SBML component consistency (fbc, L171022); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q02742' has already been declared.\\n\",\n",
       "  \"E7459 (Error): SBML component consistency (fbc, L171023); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43286' has already been declared.\\n\",\n",
       "  \"E7460 (Error): SBML component consistency (fbc, L171024); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q58A54' has already been declared.\\n\",\n",
       "  \"E7461 (Error): SBML component consistency (fbc, L171025); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q10472' has already been declared.\\n\",\n",
       "  \"E7462 (Error): SBML component consistency (fbc, L171026); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7Z7M9' has already been declared.\\n\",\n",
       "  \"E7463 (Error): SBML component consistency (fbc, L171027); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NCW6' has already been declared.\\n\",\n",
       "  \"E7464 (Error): SBML component consistency (fbc, L171028); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q14435' has already been declared.\\n\",\n",
       "  \"E7465 (Error): SBML component consistency (fbc, L171029); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q10471' has already been declared.\\n\",\n",
       "  \"E7466 (Error): SBML component consistency (fbc, L171030); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8N3T1' has already been declared.\\n\",\n",
       "  \"E7467 (Error): SBML component consistency (fbc, L171031); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NY28' has already been declared.\\n\",\n",
       "  \"E7468 (Error): SBML component consistency (fbc, L171032); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96EU7' has already been declared.\\n\",\n",
       "  \"E7469 (Error): SBML component consistency (fbc, L171033); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NS00' has already been declared.\\n\",\n",
       "  \"E7470 (Error): SBML component consistency (fbc, L171034); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R3R2' has already been declared.\\n\",\n",
       "  \"E7471 (Error): SBML component consistency (fbc, L171035); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P14324' has already been declared.\\n\",\n",
       "  \"E7472 (Error): SBML component consistency (fbc, L171036); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q86YN1' has already been declared.\\n\",\n",
       "  \"E7473 (Error): SBML component consistency (fbc, L171037); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5BKT4' has already been declared.\\n\",\n",
       "  \"E7474 (Error): SBML component consistency (fbc, L171038); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UPQ8' has already been declared.\\n\",\n",
       "  \"E7475 (Error): SBML component consistency (fbc, L171039); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H3H5' has already been declared.\\n\",\n",
       "  \"E7476 (Error): SBML component consistency (fbc, L171040); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96F25' has already been declared.\\n\",\n",
       "  \"E7477 (Error): SBML component consistency (fbc, L171041); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BT22' has already been declared.\\n\",\n",
       "  \"E7478 (Error): SBML component consistency (fbc, L171042); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H553' has already been declared.\\n\",\n",
       "  \"E7479 (Error): SBML component consistency (fbc, L171043); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q2TAA5' has already been declared.\\n\",\n",
       "  \"E7480 (Error): SBML component consistency (fbc, L171044); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60762' has already been declared.\\n\",\n",
       "  \"E7481 (Error): SBML component consistency (fbc, L171045); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9P2X0' has already been declared.\\n\",\n",
       "  \"E7482 (Error): SBML component consistency (fbc, L171046); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q92685' has already been declared.\\n\",\n",
       "  \"E7483 (Error): SBML component consistency (fbc, L171047); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H6U8' has already been declared.\\n\",\n",
       "  \"E7484 (Error): SBML component consistency (fbc, L171048); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BV10' has already been declared.\\n\",\n",
       "  \"E7485 (Error): SBML component consistency (fbc, L171049); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y672' has already been declared.\\n\",\n",
       "  \"E7486 (Error): SBML component consistency (fbc, L171050); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BVK2' has already been declared.\\n\",\n",
       "  \"E7487 (Error): SBML component consistency (fbc, L171051); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q5I7T1' has already been declared.\\n\",\n",
       "  \"E7488 (Error): SBML component consistency (fbc, L171052); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P46977' has already been declared.\\n\",\n",
       "  \"E7489 (Error): SBML component consistency (fbc, L171053); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P39656' has already been declared.\\n\",\n",
       "  \"E7490 (Error): SBML component consistency (fbc, L171054); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P61803' has already been declared.\\n\",\n",
       "  \"E7491 (Error): SBML component consistency (fbc, L171055); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_D6RDV0' has already been declared.\\n\",\n",
       "  \"E7492 (Error): SBML component consistency (fbc, L171056); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04844' has already been declared.\\n\",\n",
       "  \"E7493 (Error): SBML component consistency (fbc, L171057); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04843' has already been declared.\\n\",\n",
       "  \"E7494 (Error): SBML component consistency (fbc, L171058); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8TCJ2' has already been declared.\\n\",\n",
       "  \"E7495 (Error): SBML component consistency (fbc, L171059); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13724' has already been declared.\\n\",\n",
       "  \"E7496 (Error): SBML component consistency (fbc, L171060); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q14697' has already been declared.\\n\",\n",
       "  \"E7497 (Error): SBML component consistency (fbc, L171061); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UKM7' has already been declared.\\n\",\n",
       "  \"E7498 (Error): SBML component consistency (fbc, L171062); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NR34' has already been declared.\\n\",\n",
       "  \"E7499 (Error): SBML component consistency (fbc, L171063); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P33908' has already been declared.\\n\",\n",
       "  \"E7500 (Error): SBML component consistency (fbc, L171064); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60476' has already been declared.\\n\",\n",
       "  \"E7501 (Error): SBML component consistency (fbc, L171065); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P26572' has already been declared.\\n\",\n",
       "  \"E7502 (Error): SBML component consistency (fbc, L171066); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A0C4DGL1' has already been declared.\\n\",\n",
       "  \"E7503 (Error): SBML component consistency (fbc, L171067); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16706' has already been declared.\\n\",\n",
       "  \"E7504 (Error): SBML component consistency (fbc, L171068); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q10469' has already been declared.\\n\",\n",
       "  \"E7505 (Error): SBML component consistency (fbc, L171069); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UBM8' has already been declared.\\n\",\n",
       "  \"E7506 (Error): SBML component consistency (fbc, L171070); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UM21' has already been declared.\\n\",\n",
       "  \"E7507 (Error): SBML component consistency (fbc, L171071); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UQ53' has already been declared.\\n\",\n",
       "  \"E7508 (Error): SBML component consistency (fbc, L171072); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q09328' has already been declared.\\n\",\n",
       "  \"E7509 (Error): SBML component consistency (fbc, L171073); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BYC5' has already been declared.\\n\",\n",
       "  \"E7510 (Error): SBML component consistency (fbc, L171074); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P15907' has already been declared.\\n\",\n",
       "  \"E7511 (Error): SBML component consistency (fbc, L171075); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96JF0' has already been declared.\\n\",\n",
       "  \"E7512 (Error): SBML component consistency (fbc, L171076); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O00754' has already been declared.\\n\",\n",
       "  \"E7513 (Error): SBML component consistency (fbc, L171077); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NTJ4' has already been declared.\\n\",\n",
       "  \"E7514 (Error): SBML component consistency (fbc, L171078); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O00462' has already been declared.\\n\",\n",
       "  \"E7515 (Error): SBML component consistency (fbc, L171079); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P20933' has already been declared.\\n\",\n",
       "  \"E7516 (Error): SBML component consistency (fbc, L171080); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NFI3' has already been declared.\\n\",\n",
       "  \"E7517 (Error): SBML component consistency (fbc, L171081); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q99519' has already been declared.\\n\",\n",
       "  \"E7518 (Error): SBML component consistency (fbc, L171082); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_E9PMZ3' has already been declared.\\n\",\n",
       "  \"E7519 (Error): SBML component consistency (fbc, L171083); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9Y3R4' has already been declared.\\n\",\n",
       "  \"E7520 (Error): SBML component consistency (fbc, L171084); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8WWR8' has already been declared.\\n\",\n",
       "  \"E7521 (Error): SBML component consistency (fbc, L171085); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_X6R8A1' has already been declared.\\n\",\n",
       "  \"E7522 (Error): SBML component consistency (fbc, L171086); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P34059' has already been declared.\\n\",\n",
       "  \"E7523 (Error): SBML component consistency (fbc, L171087); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BTY2' has already been declared.\\n\",\n",
       "  \"E7524 (Error): SBML component consistency (fbc, L171088); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04066' has already been declared.\\n\",\n",
       "  \"E7525 (Error): SBML component consistency (fbc, L171089); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A8I5QJL9' has already been declared.\\n\",\n",
       "  \"E7526 (Error): SBML component consistency (fbc, L171090); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A1MA61' has already been declared.\\n\",\n",
       "  \"E7527 (Error): SBML component consistency (fbc, L171091); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00519' has already been declared.\\n\",\n",
       "  \"E7528 (Error): SBML component consistency (fbc, L171092); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_E7ERN0' has already been declared.\\n\",\n",
       "  \"E7529 (Error): SBML component consistency (fbc, L171093); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P09619' has already been declared.\\n\",\n",
       "  \"E7530 (Error): SBML component consistency (fbc, L171094); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P29322' has already been declared.\\n\",\n",
       "  \"E7531 (Error): SBML component consistency (fbc, L171095); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6NVW1' has already been declared.\\n\",\n",
       "  \"E7532 (Error): SBML component consistency (fbc, L171096); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P29320' has already been declared.\\n\",\n",
       "  \"E7533 (Error): SBML component consistency (fbc, L171097); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_E5RH01' has already been declared.\\n\",\n",
       "  \"E7534 (Error): SBML component consistency (fbc, L171098); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q07912' has already been declared.\\n\",\n",
       "  \"E7535 (Error): SBML component consistency (fbc, L171099); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q01973' has already been declared.\\n\",\n",
       "  \"E7536 (Error): SBML component consistency (fbc, L171100); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13163' has already been declared.\\n\",\n",
       "  \"E7537 (Error): SBML component consistency (fbc, L171101); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P42680' has already been declared.\\n\",\n",
       "  \"E7538 (Error): SBML component consistency (fbc, L171102); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P33981' has already been declared.\\n\",\n",
       "  \"E7539 (Error): SBML component consistency (fbc, L171103); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9HAZ1' has already been declared.\\n\",\n",
       "  \"E7540 (Error): SBML component consistency (fbc, L171104); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13882' has already been declared.\\n\",\n",
       "  \"E7541 (Error): SBML component consistency (fbc, L171105); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P09769' has already been declared.\\n\",\n",
       "  \"E7542 (Error): SBML component consistency (fbc, L171106); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P16234' has already been declared.\\n\",\n",
       "  \"E7543 (Error): SBML component consistency (fbc, L171107); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NR20' has already been declared.\\n\",\n",
       "  \"E7544 (Error): SBML component consistency (fbc, L171108); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P06241' has already been declared.\\n\",\n",
       "  \"E7545 (Error): SBML component consistency (fbc, L171109); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96KB5' has already been declared.\\n\",\n",
       "  \"E7546 (Error): SBML component consistency (fbc, L171110); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P52564' has already been declared.\\n\",\n",
       "  \"E7547 (Error): SBML component consistency (fbc, L171111); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04629' has already been declared.\\n\",\n",
       "  \"E7548 (Error): SBML component consistency (fbc, L171112); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16620' has already been declared.\\n\",\n",
       "  \"E7549 (Error): SBML component consistency (fbc, L171113); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21860' has already been declared.\\n\",\n",
       "  \"E7550 (Error): SBML component consistency (fbc, L171114); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P41240' has already been declared.\\n\",\n",
       "  \"E7551 (Error): SBML component consistency (fbc, L171115); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P29597' has already been declared.\\n\",\n",
       "  \"E7552 (Error): SBML component consistency (fbc, L171116); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P10721' has already been declared.\\n\",\n",
       "  \"E7553 (Error): SBML component consistency (fbc, L171117); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15569' has already been declared.\\n\",\n",
       "  \"E7554 (Error): SBML component consistency (fbc, L171118); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P29317' has already been declared.\\n\",\n",
       "  \"E7555 (Error): SBML component consistency (fbc, L171119); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30530' has already been declared.\\n\",\n",
       "  \"E7556 (Error): SBML component consistency (fbc, L171120); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P42684' has already been declared.\\n\",\n",
       "  \"E7557 (Error): SBML component consistency (fbc, L171121); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00533' has already been declared.\\n\",\n",
       "  \"E7558 (Error): SBML component consistency (fbc, L171122); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P14616' has already been declared.\\n\",\n",
       "  \"E7559 (Error): SBML component consistency (fbc, L171123); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P49759' has already been declared.\\n\",\n",
       "  \"E7560 (Error): SBML component consistency (fbc, L171124); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q12866' has already been declared.\\n\",\n",
       "  \"E7561 (Error): SBML component consistency (fbc, L171125); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P16591' has already been declared.\\n\",\n",
       "  \"E7562 (Error): SBML component consistency (fbc, L171126); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P34925' has already been declared.\\n\",\n",
       "  \"E7563 (Error): SBML component consistency (fbc, L171127); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P36888' has already been declared.\\n\",\n",
       "  \"E7564 (Error): SBML component consistency (fbc, L171128); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60674' has already been declared.\\n\",\n",
       "  \"E7565 (Error): SBML component consistency (fbc, L171129); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q92630' has already been declared.\\n\",\n",
       "  \"E7566 (Error): SBML component consistency (fbc, L171130); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R7T2' has already been declared.\\n\",\n",
       "  \"E7567 (Error): SBML component consistency (fbc, L171131); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54753' has already been declared.\\n\",\n",
       "  \"E7568 (Error): SBML component consistency (fbc, L171132); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22455' has already been declared.\\n\",\n",
       "  \"E7569 (Error): SBML component consistency (fbc, L171133); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B4DXM2' has already been declared.\\n\",\n",
       "  \"E7570 (Error): SBML component consistency (fbc, L171134); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13470' has already been declared.\\n\",\n",
       "  \"E7571 (Error): SBML component consistency (fbc, L171135); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P08069' has already been declared.\\n\",\n",
       "  \"E7572 (Error): SBML component consistency (fbc, L171136); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q08881' has already been declared.\\n\",\n",
       "  \"E7573 (Error): SBML component consistency (fbc, L171137); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P04626' has already been declared.\\n\",\n",
       "  \"E7574 (Error): SBML component consistency (fbc, L171138); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P36507' has already been declared.\\n\",\n",
       "  \"E7575 (Error): SBML component consistency (fbc, L171139); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P43405' has already been declared.\\n\",\n",
       "  \"E7576 (Error): SBML component consistency (fbc, L171140); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_I6LM06' has already been declared.\\n\",\n",
       "  \"E7577 (Error): SBML component consistency (fbc, L171141); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07333' has already been declared.\\n\",\n",
       "  \"E7578 (Error): SBML component consistency (fbc, L171142); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07332' has already been declared.\\n\",\n",
       "  \"E7579 (Error): SBML component consistency (fbc, L171143); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51451' has already been declared.\\n\",\n",
       "  \"E7580 (Error): SBML component consistency (fbc, L171144); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8IWU2' has already been declared.\\n\",\n",
       "  \"E7581 (Error): SBML component consistency (fbc, L171145); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P42679' has already been declared.\\n\",\n",
       "  \"E7582 (Error): SBML component consistency (fbc, L171146); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_H3BRE4' has already been declared.\\n\",\n",
       "  \"E7583 (Error): SBML component consistency (fbc, L171147); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NCJ5' has already been declared.\\n\",\n",
       "  \"E7584 (Error): SBML component consistency (fbc, L171148); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21802' has already been declared.\\n\",\n",
       "  \"E7585 (Error): SBML component consistency (fbc, L171149); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P42685' has already been declared.\\n\",\n",
       "  \"E7586 (Error): SBML component consistency (fbc, L171150); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51813' has already been declared.\\n\",\n",
       "  \"E7587 (Error): SBML component consistency (fbc, L171151); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07947' has already been declared.\\n\",\n",
       "  \"E7588 (Error): SBML component consistency (fbc, L171152); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23458' has already been declared.\\n\",\n",
       "  \"E7589 (Error): SBML component consistency (fbc, L171153); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P06213' has already been declared.\\n\",\n",
       "  \"E7590 (Error): SBML component consistency (fbc, L171154); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21709' has already been declared.\\n\",\n",
       "  \"E7591 (Error): SBML component consistency (fbc, L171155); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O14733' has already been declared.\\n\",\n",
       "  \"E7592 (Error): SBML component consistency (fbc, L171156); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H792' has already been declared.\\n\",\n",
       "  \"E7593 (Error): SBML component consistency (fbc, L171157); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O15146' has already been declared.\\n\",\n",
       "  \"E7594 (Error): SBML component consistency (fbc, L171158); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q04912' has already been declared.\\n\",\n",
       "  \"E7595 (Error): SBML component consistency (fbc, L171159); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024R728' has already been declared.\\n\",\n",
       "  \"E7596 (Error): SBML component consistency (fbc, L171160); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_X5D7M5' has already been declared.\\n\",\n",
       "  \"E7597 (Error): SBML component consistency (fbc, L171161); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15303' has already been declared.\\n\",\n",
       "  \"E7598 (Error): SBML component consistency (fbc, L171162); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A7P0T9L5' has already been declared.\\n\",\n",
       "  \"E7599 (Error): SBML component consistency (fbc, L171163); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P11362' has already been declared.\\n\",\n",
       "  \"E7600 (Error): SBML component consistency (fbc, L171164); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54760' has already been declared.\\n\",\n",
       "  \"E7601 (Error): SBML component consistency (fbc, L171165); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q76I76' has already been declared.\\n\",\n",
       "  \"E7602 (Error): SBML component consistency (fbc, L171166); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8WYL5' has already been declared.\\n\",\n",
       "  \"E7603 (Error): SBML component consistency (fbc, L171167); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P43378' has already been declared.\\n\",\n",
       "  \"E7604 (Error): SBML component consistency (fbc, L171168); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BVJ7' has already been declared.\\n\",\n",
       "  \"E7605 (Error): SBML component consistency (fbc, L171169); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P54829' has already been declared.\\n\",\n",
       "  \"E7606 (Error): SBML component consistency (fbc, L171170); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8WUK0' has already been declared.\\n\",\n",
       "  \"E7607 (Error): SBML component consistency (fbc, L171171); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9HD43' has already been declared.\\n\",\n",
       "  \"E7608 (Error): SBML component consistency (fbc, L171172); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_S4R378' has already been declared.\\n\",\n",
       "  \"E7609 (Error): SBML component consistency (fbc, L171173); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8NEJ0' has already been declared.\\n\",\n",
       "  \"E7610 (Error): SBML component consistency (fbc, L171174); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16667' has already been declared.\\n\",\n",
       "  \"E7611 (Error): SBML component consistency (fbc, L171175); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q99956' has already been declared.\\n\",\n",
       "  \"E7612 (Error): SBML component consistency (fbc, L171176); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13615' has already been declared.\\n\",\n",
       "  \"E7613 (Error): SBML component consistency (fbc, L171177); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_E9PMT0' has already been declared.\\n\",\n",
       "  \"E7614 (Error): SBML component consistency (fbc, L171178); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16828' has already been declared.\\n\",\n",
       "  \"E7615 (Error): SBML component consistency (fbc, L171179); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H3S7' has already been declared.\\n\",\n",
       "  \"E7616 (Error): SBML component consistency (fbc, L171180); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q05209' has already been declared.\\n\",\n",
       "  \"E7617 (Error): SBML component consistency (fbc, L171181); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q12923' has already been declared.\\n\",\n",
       "  \"E7618 (Error): SBML component consistency (fbc, L171182); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P28827' has already been declared.\\n\",\n",
       "  \"E7619 (Error): SBML component consistency (fbc, L171183); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P17706' has already been declared.\\n\",\n",
       "  \"E7620 (Error): SBML component consistency (fbc, L171184); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_F5H5X4' has already been declared.\\n\",\n",
       "  \"E7621 (Error): SBML component consistency (fbc, L171185); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23467' has already been declared.\\n\",\n",
       "  \"E7622 (Error): SBML component consistency (fbc, L171186); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23469' has already been declared.\\n\",\n",
       "  \"E7623 (Error): SBML component consistency (fbc, L171187); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15678' has already been declared.\\n\",\n",
       "  \"E7624 (Error): SBML component consistency (fbc, L171188); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q99952' has already been declared.\\n\",\n",
       "  \"E7625 (Error): SBML component consistency (fbc, L171189); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UNH5' has already been declared.\\n\",\n",
       "  \"E7626 (Error): SBML component consistency (fbc, L171190); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16825' has already been declared.\\n\",\n",
       "  \"E7627 (Error): SBML component consistency (fbc, L171191); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P51452' has already been declared.\\n\",\n",
       "  \"E7628 (Error): SBML component consistency (fbc, L171192); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P28562' has already been declared.\\n\",\n",
       "  \"E7629 (Error): SBML component consistency (fbc, L171193); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15256' has already been declared.\\n\",\n",
       "  \"E7630 (Error): SBML component consistency (fbc, L171194); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P18031' has already been declared.\\n\",\n",
       "  \"E7631 (Error): SBML component consistency (fbc, L171195); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8WTR2' has already been declared.\\n\",\n",
       "  \"E7632 (Error): SBML component consistency (fbc, L171196); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q68DX3' has already been declared.\\n\",\n",
       "  \"E7633 (Error): SBML component consistency (fbc, L171197); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6B8I1' has already been declared.\\n\",\n",
       "  \"E7634 (Error): SBML component consistency (fbc, L171198); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q12974' has already been declared.\\n\",\n",
       "  \"E7635 (Error): SBML component consistency (fbc, L171199); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q12913' has already been declared.\\n\",\n",
       "  \"E7636 (Error): SBML component consistency (fbc, L171200); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P26045' has already been declared.\\n\",\n",
       "  \"E7637 (Error): SBML component consistency (fbc, L171201); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A2A3K4' has already been declared.\\n\",\n",
       "  \"E7638 (Error): SBML component consistency (fbc, L171202); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P18433' has already been declared.\\n\",\n",
       "  \"E7639 (Error): SBML component consistency (fbc, L171203); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A0B4J2F5' has already been declared.\\n\",\n",
       "  \"E7640 (Error): SBML component consistency (fbc, L171204); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P10586' has already been declared.\\n\",\n",
       "  \"E7641 (Error): SBML component consistency (fbc, L171205); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23468' has already been declared.\\n\",\n",
       "  \"E7642 (Error): SBML component consistency (fbc, L171206); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q68J44' has already been declared.\\n\",\n",
       "  \"E7643 (Error): SBML component consistency (fbc, L171207); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95677' has already been declared.\\n\",\n",
       "  \"E7644 (Error): SBML component consistency (fbc, L171208); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O00167' has already been declared.\\n\",\n",
       "  \"E7645 (Error): SBML component consistency (fbc, L171209); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P24666' has already been declared.\\n\",\n",
       "  \"E7646 (Error): SBML component consistency (fbc, L171210); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q92729' has already been declared.\\n\",\n",
       "  \"E7647 (Error): SBML component consistency (fbc, L171211); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P30304' has already been declared.\\n\",\n",
       "  \"E7648 (Error): SBML component consistency (fbc, L171212); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P53634' has already been declared.\\n\",\n",
       "  \"E7649 (Error): SBML component consistency (fbc, L171213); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P43234' has already been declared.\\n\",\n",
       "  \"E7650 (Error): SBML component consistency (fbc, L171214); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P14091' has already been declared.\\n\",\n",
       "  \"E7651 (Error): SBML component consistency (fbc, L171215); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_E9PSG5' has already been declared.\\n\",\n",
       "  \"E7652 (Error): SBML component consistency (fbc, L171216); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P56202' has already been declared.\\n\",\n",
       "  \"E7653 (Error): SBML component consistency (fbc, L171217); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P43235' has already been declared.\\n\",\n",
       "  \"E7654 (Error): SBML component consistency (fbc, L171218); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UBX1' has already been declared.\\n\",\n",
       "  \"E7655 (Error): SBML component consistency (fbc, L171219); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A7I2YQE6' has already been declared.\\n\",\n",
       "  \"E7656 (Error): SBML component consistency (fbc, L171220); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O14773' has already been declared.\\n\",\n",
       "  \"E7657 (Error): SBML component consistency (fbc, L171221); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9UBR2' has already been declared.\\n\",\n",
       "  \"E7658 (Error): SBML component consistency (fbc, L171222); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A7I2YQA2' has already been declared.\\n\",\n",
       "  \"E7659 (Error): SBML component consistency (fbc, L171223); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O60911' has already been declared.\\n\",\n",
       "  \"E7660 (Error): SBML component consistency (fbc, L171224); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07339' has already been declared.\\n\",\n",
       "  \"E7661 (Error): SBML component consistency (fbc, L171225); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_B0YJ81' has already been declared.\\n\",\n",
       "  \"E7662 (Error): SBML component consistency (fbc, L171226); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q6Y1H2' has already been declared.\\n\",\n",
       "  \"E7663 (Error): SBML component consistency (fbc, L171227); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P43166' has already been declared.\\n\",\n",
       "  \"E7664 (Error): SBML component consistency (fbc, L171228); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q13564' has already been declared.\\n\",\n",
       "  \"E7665 (Error): SBML component consistency (fbc, L171229); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A8K3J4' has already been declared.\\n\",\n",
       "  \"E7666 (Error): SBML component consistency (fbc, L171230); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q16790' has already been declared.\\n\",\n",
       "  \"E7667 (Error): SBML component consistency (fbc, L171231); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_V9HWE3' has already been declared.\\n\",\n",
       "  \"E7668 (Error): SBML component consistency (fbc, L171232); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P23280' has already been declared.\\n\",\n",
       "  \"E7669 (Error): SBML component consistency (fbc, L171233); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8N1Q1' has already been declared.\\n\",\n",
       "  \"E7670 (Error): SBML component consistency (fbc, L171234); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P22748' has already been declared.\\n\",\n",
       "  \"E7671 (Error): SBML component consistency (fbc, L171235); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P00918' has already been declared.\\n\",\n",
       "  \"E7672 (Error): SBML component consistency (fbc, L171236); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P07451' has already been declared.\\n\",\n",
       "  \"E7673 (Error): SBML component consistency (fbc, L171237); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P35219' has already been declared.\\n\",\n",
       "  \"E7674 (Error): SBML component consistency (fbc, L171238); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O43570' has already been declared.\\n\",\n",
       "  \"E7675 (Error): SBML component consistency (fbc, L171239); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O95861' has already been declared.\\n\",\n",
       "  \"E7676 (Error): SBML component consistency (fbc, L171240); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9NX62' has already been declared.\\n\",\n",
       "  \"E7677 (Error): SBML component consistency (fbc, L171241); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q96I99' has already been declared.\\n\",\n",
       "  \"E7678 (Error): SBML component consistency (fbc, L171242); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9P2R7' has already been declared.\\n\",\n",
       "  \"E7679 (Error): SBML component consistency (fbc, L171243); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P53597' has already been declared.\\n\",\n",
       "  \"E7680 (Error): SBML component consistency (fbc, L171244); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P48735' has already been declared.\\n\",\n",
       "  \"E7681 (Error): SBML component consistency (fbc, L171245); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_O75874' has already been declared.\\n\",\n",
       "  \"E7682 (Error): SBML component consistency (fbc, L171246); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P55809' has already been declared.\\n\",\n",
       "  \"E7683 (Error): SBML component consistency (fbc, L171247); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9BYC2' has already been declared.\\n\",\n",
       "  \"E7684 (Error): SBML component consistency (fbc, L171248); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A087WZN1' has already been declared.\\n\",\n",
       "  \"E7685 (Error): SBML component consistency (fbc, L171249); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q68D72' has already been declared.\\n\",\n",
       "  \"E7686 (Error): SBML component consistency (fbc, L171250); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_G5E9Q7' has already been declared.\\n\",\n",
       "  \"E7687 (Error): SBML component consistency (fbc, L171251); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A024RB75' has already been declared.\\n\",\n",
       "  \"E7688 (Error): SBML component consistency (fbc, L171252); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q14678' has already been declared.\\n\",\n",
       "  \"E7689 (Error): SBML component consistency (fbc, L171253); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P53396' has already been declared.\\n\",\n",
       "  \"E7690 (Error): SBML component consistency (fbc, L171254); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21399' has already been declared.\\n\",\n",
       "  \"E7691 (Error): SBML component consistency (fbc, L171255); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P48200' has already been declared.\\n\",\n",
       "  \"E7692 (Error): SBML component consistency (fbc, L171256); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q99798' has already been declared.\\n\",\n",
       "  \"E7693 (Error): SBML component consistency (fbc, L171257); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q99643' has already been declared.\\n\",\n",
       "  \"E7694 (Error): SBML component consistency (fbc, L171258); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P31040' has already been declared.\\n\",\n",
       "  \"E7695 (Error): SBML component consistency (fbc, L171259); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_G3V173' has already been declared.\\n\",\n",
       "  \"E7696 (Error): SBML component consistency (fbc, L171260); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P21912' has already been declared.\\n\",\n",
       "  \"E7697 (Error): SBML component consistency (fbc, L171261); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q8N0X4' has already been declared.\\n\",\n",
       "  \"E7698 (Error): SBML component consistency (fbc, L171262); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q15181' has already been declared.\\n\",\n",
       "  \"E7699 (Error): SBML component consistency (fbc, L171263); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H2U2' has already been declared.\\n\",\n",
       "  \"E7700 (Error): SBML component consistency (fbc, L171264); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q9H008' has already been declared.\\n\",\n",
       "  \"E7701 (Error): SBML component consistency (fbc, L171265); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_P38117' has already been declared.\\n\",\n",
       "  \"E7702 (Error): SBML component consistency (fbc, L171266); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A804HI48' has already been declared.\\n\",\n",
       "  \"E7703 (Error): SBML component consistency (fbc, L171267); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_A0A8I5KUF3' has already been declared.\\n\",\n",
       "  \"E7704 (Error): SBML component consistency (fbc, L171268); 'fbc:label' must be unique; The attribute 'fbc:label' on a <GeneProduct> must be unique among the set of all <GeneProduct> elements defined in the <Model>. \\nReference: L3V1 Fbc V3 Section 3.5\\n A GeneProduct with the label 'G_Q7GIM7' has already been declared.\\n\",\n",
       "  ...],\n",
       " 'SBML_SCHEMA_ERROR': [],\n",
       " 'SBML_WARNING': ['E0 (Warning): SBO term consistency (core, L25); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E1 (Warning): SBO term consistency (core, L32); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E2 (Warning): SBO term consistency (core, L39); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E3 (Warning): SBO term consistency (core, L46); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E4 (Warning): SBO term consistency (core, L53); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E5 (Warning): SBO term consistency (core, L60); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E6 (Warning): SBO term consistency (core, L67); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E7 (Warning): SBO term consistency (core, L74); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E8 (Warning): SBO term consistency (core, L81); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E9 (Warning): SBO term consistency (core, L88); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E10 (Warning): SBO term consistency (core, L95); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E11 (Warning): SBO term consistency (core, L102); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E12 (Warning): SBO term consistency (core, L109); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E13 (Warning): SBO term consistency (core, L116); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E14 (Warning): SBO term consistency (core, L123); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E15 (Warning): SBO term consistency (core, L130); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E16 (Warning): SBO term consistency (core, L137); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E17 (Warning): SBO term consistency (core, L144); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E18 (Warning): SBO term consistency (core, L151); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E19 (Warning): SBO term consistency (core, L158); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E20 (Warning): SBO term consistency (core, L165); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E21 (Warning): SBO term consistency (core, L172); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E22 (Warning): SBO term consistency (core, L179); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E23 (Warning): SBO term consistency (core, L186); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E24 (Warning): SBO term consistency (core, L193); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E25 (Warning): SBO term consistency (core, L200); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E26 (Warning): SBO term consistency (core, L207); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E27 (Warning): SBO term consistency (core, L214); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E28 (Warning): SBO term consistency (core, L221); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E29 (Warning): SBO term consistency (core, L228); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E30 (Warning): SBO term consistency (core, L235); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E31 (Warning): SBO term consistency (core, L242); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E32 (Warning): SBO term consistency (core, L249); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E33 (Warning): SBO term consistency (core, L256); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E34 (Warning): SBO term consistency (core, L263); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E35 (Warning): SBO term consistency (core, L270); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E36 (Warning): SBO term consistency (core, L277); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E37 (Warning): SBO term consistency (core, L284); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E38 (Warning): SBO term consistency (core, L291); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E39 (Warning): SBO term consistency (core, L298); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E40 (Warning): SBO term consistency (core, L305); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E41 (Warning): SBO term consistency (core, L312); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E42 (Warning): SBO term consistency (core, L319); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E43 (Warning): SBO term consistency (core, L326); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E44 (Warning): SBO term consistency (core, L333); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E45 (Warning): SBO term consistency (core, L340); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E46 (Warning): SBO term consistency (core, L347); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E47 (Warning): SBO term consistency (core, L354); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E48 (Warning): SBO term consistency (core, L361); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E49 (Warning): SBO term consistency (core, L368); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E50 (Warning): SBO term consistency (core, L375); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E51 (Warning): SBO term consistency (core, L382); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E52 (Warning): SBO term consistency (core, L389); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E53 (Warning): SBO term consistency (core, L396); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E54 (Warning): SBO term consistency (core, L403); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E55 (Warning): SBO term consistency (core, L410); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E56 (Warning): SBO term consistency (core, L417); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E57 (Warning): SBO term consistency (core, L424); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E58 (Warning): SBO term consistency (core, L431); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E59 (Warning): SBO term consistency (core, L438); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E60 (Warning): SBO term consistency (core, L445); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E61 (Warning): SBO term consistency (core, L452); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E62 (Warning): SBO term consistency (core, L459); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E63 (Warning): SBO term consistency (core, L466); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E64 (Warning): SBO term consistency (core, L473); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E65 (Warning): SBO term consistency (core, L480); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E66 (Warning): SBO term consistency (core, L487); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E67 (Warning): SBO term consistency (core, L494); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E68 (Warning): SBO term consistency (core, L501); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E69 (Warning): SBO term consistency (core, L508); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E70 (Warning): SBO term consistency (core, L515); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E71 (Warning): SBO term consistency (core, L522); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E72 (Warning): SBO term consistency (core, L529); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E73 (Warning): SBO term consistency (core, L536); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E74 (Warning): SBO term consistency (core, L543); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E75 (Warning): SBO term consistency (core, L550); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E76 (Warning): SBO term consistency (core, L557); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E77 (Warning): SBO term consistency (core, L564); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E78 (Warning): SBO term consistency (core, L571); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E79 (Warning): SBO term consistency (core, L578); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E80 (Warning): SBO term consistency (core, L585); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E81 (Warning): SBO term consistency (core, L592); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E82 (Warning): SBO term consistency (core, L599); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E83 (Warning): SBO term consistency (core, L606); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E84 (Warning): SBO term consistency (core, L613); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E85 (Warning): SBO term consistency (core, L620); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E86 (Warning): SBO term consistency (core, L627); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E87 (Warning): SBO term consistency (core, L634); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E88 (Warning): SBO term consistency (core, L641); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E89 (Warning): SBO term consistency (core, L648); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E90 (Warning): SBO term consistency (core, L655); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E91 (Warning): SBO term consistency (core, L662); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E92 (Warning): SBO term consistency (core, L669); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E93 (Warning): SBO term consistency (core, L676); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E94 (Warning): SBO term consistency (core, L683); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E95 (Warning): SBO term consistency (core, L690); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E96 (Warning): SBO term consistency (core, L697); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E97 (Warning): SBO term consistency (core, L704); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E98 (Warning): SBO term consistency (core, L711); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E99 (Warning): SBO term consistency (core, L718); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E100 (Warning): SBO term consistency (core, L725); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E101 (Warning): SBO term consistency (core, L732); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E102 (Warning): SBO term consistency (core, L739); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E103 (Warning): SBO term consistency (core, L746); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E104 (Warning): SBO term consistency (core, L753); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E105 (Warning): SBO term consistency (core, L760); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E106 (Warning): SBO term consistency (core, L767); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E107 (Warning): SBO term consistency (core, L774); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E108 (Warning): SBO term consistency (core, L781); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E109 (Warning): SBO term consistency (core, L788); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E110 (Warning): SBO term consistency (core, L795); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E111 (Warning): SBO term consistency (core, L802); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E112 (Warning): SBO term consistency (core, L809); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E113 (Warning): SBO term consistency (core, L816); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E114 (Warning): SBO term consistency (core, L823); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E115 (Warning): SBO term consistency (core, L830); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E116 (Warning): SBO term consistency (core, L837); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E117 (Warning): SBO term consistency (core, L844); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E118 (Warning): SBO term consistency (core, L851); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E119 (Warning): SBO term consistency (core, L858); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E120 (Warning): SBO term consistency (core, L865); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E121 (Warning): SBO term consistency (core, L872); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E122 (Warning): SBO term consistency (core, L879); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E123 (Warning): SBO term consistency (core, L886); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E124 (Warning): SBO term consistency (core, L893); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E125 (Warning): SBO term consistency (core, L900); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E126 (Warning): SBO term consistency (core, L907); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E127 (Warning): SBO term consistency (core, L914); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E128 (Warning): SBO term consistency (core, L921); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E129 (Warning): SBO term consistency (core, L928); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E130 (Warning): SBO term consistency (core, L935); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E131 (Warning): SBO term consistency (core, L942); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E132 (Warning): SBO term consistency (core, L949); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E133 (Warning): SBO term consistency (core, L956); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E134 (Warning): SBO term consistency (core, L963); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E135 (Warning): SBO term consistency (core, L970); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E136 (Warning): SBO term consistency (core, L977); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E137 (Warning): SBO term consistency (core, L984); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E138 (Warning): SBO term consistency (core, L991); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E139 (Warning): SBO term consistency (core, L998); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E140 (Warning): SBO term consistency (core, L1005); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E141 (Warning): SBO term consistency (core, L1012); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E142 (Warning): SBO term consistency (core, L1019); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E143 (Warning): SBO term consistency (core, L1026); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E144 (Warning): SBO term consistency (core, L1033); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E145 (Warning): SBO term consistency (core, L1040); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E146 (Warning): SBO term consistency (core, L1047); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E147 (Warning): SBO term consistency (core, L1054); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E148 (Warning): SBO term consistency (core, L1061); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E149 (Warning): SBO term consistency (core, L1068); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E150 (Warning): SBO term consistency (core, L1075); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E151 (Warning): SBO term consistency (core, L1082); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E152 (Warning): SBO term consistency (core, L1089); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E153 (Warning): SBO term consistency (core, L1096); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E154 (Warning): SBO term consistency (core, L1103); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E155 (Warning): SBO term consistency (core, L1110); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E156 (Warning): SBO term consistency (core, L1117); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E157 (Warning): SBO term consistency (core, L1124); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E158 (Warning): SBO term consistency (core, L1131); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E159 (Warning): SBO term consistency (core, L1138); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E160 (Warning): SBO term consistency (core, L1145); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E161 (Warning): SBO term consistency (core, L1152); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E162 (Warning): SBO term consistency (core, L1159); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E163 (Warning): SBO term consistency (core, L1166); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E164 (Warning): SBO term consistency (core, L1173); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E165 (Warning): SBO term consistency (core, L1180); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E166 (Warning): SBO term consistency (core, L1187); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E167 (Warning): SBO term consistency (core, L1194); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E168 (Warning): SBO term consistency (core, L1201); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E169 (Warning): SBO term consistency (core, L1208); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E170 (Warning): SBO term consistency (core, L1215); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E171 (Warning): SBO term consistency (core, L1222); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E172 (Warning): SBO term consistency (core, L1229); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E173 (Warning): SBO term consistency (core, L1236); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E174 (Warning): SBO term consistency (core, L1243); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E175 (Warning): SBO term consistency (core, L1250); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E176 (Warning): SBO term consistency (core, L1257); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E177 (Warning): SBO term consistency (core, L1264); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E178 (Warning): SBO term consistency (core, L1271); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E179 (Warning): SBO term consistency (core, L1278); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E180 (Warning): SBO term consistency (core, L1285); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E181 (Warning): SBO term consistency (core, L1292); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E182 (Warning): SBO term consistency (core, L1299); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E183 (Warning): SBO term consistency (core, L1306); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E184 (Warning): SBO term consistency (core, L1313); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E185 (Warning): SBO term consistency (core, L1320); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E186 (Warning): SBO term consistency (core, L1327); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E187 (Warning): SBO term consistency (core, L1334); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E188 (Warning): SBO term consistency (core, L1341); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E189 (Warning): SBO term consistency (core, L1348); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E190 (Warning): SBO term consistency (core, L1355); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E191 (Warning): SBO term consistency (core, L1362); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E192 (Warning): SBO term consistency (core, L1369); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E193 (Warning): SBO term consistency (core, L1376); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E194 (Warning): SBO term consistency (core, L1383); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E195 (Warning): SBO term consistency (core, L1390); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E196 (Warning): SBO term consistency (core, L1397); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E197 (Warning): SBO term consistency (core, L1404); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E198 (Warning): SBO term consistency (core, L1411); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E199 (Warning): SBO term consistency (core, L1418); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E200 (Warning): SBO term consistency (core, L1425); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E201 (Warning): SBO term consistency (core, L1432); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E202 (Warning): SBO term consistency (core, L1439); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E203 (Warning): SBO term consistency (core, L1446); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E204 (Warning): SBO term consistency (core, L1453); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E205 (Warning): SBO term consistency (core, L1460); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E206 (Warning): SBO term consistency (core, L1467); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E207 (Warning): SBO term consistency (core, L1474); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E208 (Warning): SBO term consistency (core, L1481); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E209 (Warning): SBO term consistency (core, L1488); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E210 (Warning): SBO term consistency (core, L1495); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E211 (Warning): SBO term consistency (core, L1502); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E212 (Warning): SBO term consistency (core, L1509); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E213 (Warning): SBO term consistency (core, L1516); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E214 (Warning): SBO term consistency (core, L1523); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E215 (Warning): SBO term consistency (core, L1530); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E216 (Warning): SBO term consistency (core, L1537); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E217 (Warning): SBO term consistency (core, L1544); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E218 (Warning): SBO term consistency (core, L1551); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E219 (Warning): SBO term consistency (core, L1558); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E220 (Warning): SBO term consistency (core, L1565); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E221 (Warning): SBO term consistency (core, L1572); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E222 (Warning): SBO term consistency (core, L1579); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E223 (Warning): SBO term consistency (core, L1586); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E224 (Warning): SBO term consistency (core, L1593); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E225 (Warning): SBO term consistency (core, L1600); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E226 (Warning): SBO term consistency (core, L1607); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E227 (Warning): SBO term consistency (core, L1614); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E228 (Warning): SBO term consistency (core, L1621); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E229 (Warning): SBO term consistency (core, L1628); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E230 (Warning): SBO term consistency (core, L1635); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E231 (Warning): SBO term consistency (core, L1642); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E232 (Warning): SBO term consistency (core, L1649); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E233 (Warning): SBO term consistency (core, L1656); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E234 (Warning): SBO term consistency (core, L1663); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E235 (Warning): SBO term consistency (core, L1670); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E236 (Warning): SBO term consistency (core, L1677); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E237 (Warning): SBO term consistency (core, L1684); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E238 (Warning): SBO term consistency (core, L1691); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E239 (Warning): SBO term consistency (core, L1698); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E240 (Warning): SBO term consistency (core, L1705); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E241 (Warning): SBO term consistency (core, L1712); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E242 (Warning): SBO term consistency (core, L1719); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E243 (Warning): SBO term consistency (core, L1726); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E244 (Warning): SBO term consistency (core, L1733); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E245 (Warning): SBO term consistency (core, L1740); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E246 (Warning): SBO term consistency (core, L1747); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E247 (Warning): SBO term consistency (core, L1754); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E248 (Warning): SBO term consistency (core, L1761); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E249 (Warning): SBO term consistency (core, L1768); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E250 (Warning): SBO term consistency (core, L1775); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E251 (Warning): SBO term consistency (core, L1782); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E252 (Warning): SBO term consistency (core, L1789); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E253 (Warning): SBO term consistency (core, L1796); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E254 (Warning): SBO term consistency (core, L1803); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E255 (Warning): SBO term consistency (core, L1810); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E256 (Warning): SBO term consistency (core, L1817); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E257 (Warning): SBO term consistency (core, L1824); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E258 (Warning): SBO term consistency (core, L1831); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E259 (Warning): SBO term consistency (core, L1838); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E260 (Warning): SBO term consistency (core, L1845); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E261 (Warning): SBO term consistency (core, L1852); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E262 (Warning): SBO term consistency (core, L1859); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E263 (Warning): SBO term consistency (core, L1866); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E264 (Warning): SBO term consistency (core, L1873); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E265 (Warning): SBO term consistency (core, L1880); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E266 (Warning): SBO term consistency (core, L1887); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E267 (Warning): SBO term consistency (core, L1894); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E268 (Warning): SBO term consistency (core, L1901); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E269 (Warning): SBO term consistency (core, L1908); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E270 (Warning): SBO term consistency (core, L1915); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E271 (Warning): SBO term consistency (core, L1922); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E272 (Warning): SBO term consistency (core, L1929); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E273 (Warning): SBO term consistency (core, L1936); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E274 (Warning): SBO term consistency (core, L1943); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E275 (Warning): SBO term consistency (core, L1950); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E276 (Warning): SBO term consistency (core, L1957); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E277 (Warning): SBO term consistency (core, L1964); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E278 (Warning): SBO term consistency (core, L1971); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E279 (Warning): SBO term consistency (core, L1978); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E280 (Warning): SBO term consistency (core, L1985); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E281 (Warning): SBO term consistency (core, L1992); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E282 (Warning): SBO term consistency (core, L1999); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E283 (Warning): SBO term consistency (core, L2006); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E284 (Warning): SBO term consistency (core, L2013); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E285 (Warning): SBO term consistency (core, L2020); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E286 (Warning): SBO term consistency (core, L2027); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E287 (Warning): SBO term consistency (core, L2034); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E288 (Warning): SBO term consistency (core, L2041); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E289 (Warning): SBO term consistency (core, L2048); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E290 (Warning): SBO term consistency (core, L2055); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E291 (Warning): SBO term consistency (core, L2062); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E292 (Warning): SBO term consistency (core, L2069); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E293 (Warning): SBO term consistency (core, L2076); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E294 (Warning): SBO term consistency (core, L2083); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E295 (Warning): SBO term consistency (core, L2090); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E296 (Warning): SBO term consistency (core, L2097); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E297 (Warning): SBO term consistency (core, L2104); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E298 (Warning): SBO term consistency (core, L2111); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E299 (Warning): SBO term consistency (core, L2118); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E300 (Warning): SBO term consistency (core, L2125); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E301 (Warning): SBO term consistency (core, L2132); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E302 (Warning): SBO term consistency (core, L2139); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E303 (Warning): SBO term consistency (core, L2146); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E304 (Warning): SBO term consistency (core, L2153); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E305 (Warning): SBO term consistency (core, L2160); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E306 (Warning): SBO term consistency (core, L2167); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E307 (Warning): SBO term consistency (core, L2174); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E308 (Warning): SBO term consistency (core, L2181); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E309 (Warning): SBO term consistency (core, L2188); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E310 (Warning): SBO term consistency (core, L2195); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E311 (Warning): SBO term consistency (core, L2202); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E312 (Warning): SBO term consistency (core, L2209); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E313 (Warning): SBO term consistency (core, L2216); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E314 (Warning): SBO term consistency (core, L2223); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E315 (Warning): SBO term consistency (core, L2230); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E316 (Warning): SBO term consistency (core, L2237); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E317 (Warning): SBO term consistency (core, L2244); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E318 (Warning): SBO term consistency (core, L2251); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E319 (Warning): SBO term consistency (core, L2258); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E320 (Warning): SBO term consistency (core, L2265); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E321 (Warning): SBO term consistency (core, L2272); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E322 (Warning): SBO term consistency (core, L2279); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E323 (Warning): SBO term consistency (core, L2286); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E324 (Warning): SBO term consistency (core, L2293); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E325 (Warning): SBO term consistency (core, L2300); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E326 (Warning): SBO term consistency (core, L2307); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E327 (Warning): SBO term consistency (core, L2314); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E328 (Warning): SBO term consistency (core, L2321); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E329 (Warning): SBO term consistency (core, L2328); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E330 (Warning): SBO term consistency (core, L2335); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E331 (Warning): SBO term consistency (core, L2342); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E332 (Warning): SBO term consistency (core, L2349); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E333 (Warning): SBO term consistency (core, L2356); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E334 (Warning): SBO term consistency (core, L2363); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E335 (Warning): SBO term consistency (core, L2370); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E336 (Warning): SBO term consistency (core, L2377); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E337 (Warning): SBO term consistency (core, L2384); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E338 (Warning): SBO term consistency (core, L2391); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E339 (Warning): SBO term consistency (core, L2398); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E340 (Warning): SBO term consistency (core, L2405); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E341 (Warning): SBO term consistency (core, L2406); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E342 (Warning): SBO term consistency (core, L2407); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E343 (Warning): SBO term consistency (core, L2414); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E344 (Warning): SBO term consistency (core, L2421); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E345 (Warning): SBO term consistency (core, L2428); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E346 (Warning): SBO term consistency (core, L2435); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E347 (Warning): SBO term consistency (core, L2442); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E348 (Warning): SBO term consistency (core, L2449); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E349 (Warning): SBO term consistency (core, L2456); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E350 (Warning): SBO term consistency (core, L2463); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E351 (Warning): SBO term consistency (core, L2470); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E352 (Warning): SBO term consistency (core, L2477); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E353 (Warning): SBO term consistency (core, L2484); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E354 (Warning): SBO term consistency (core, L2491); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E355 (Warning): SBO term consistency (core, L2498); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E356 (Warning): SBO term consistency (core, L2505); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E357 (Warning): SBO term consistency (core, L2512); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E358 (Warning): SBO term consistency (core, L2519); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E359 (Warning): SBO term consistency (core, L2526); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E360 (Warning): SBO term consistency (core, L2533); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E361 (Warning): SBO term consistency (core, L2540); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E362 (Warning): SBO term consistency (core, L2547); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E363 (Warning): SBO term consistency (core, L2554); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E364 (Warning): SBO term consistency (core, L2561); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E365 (Warning): SBO term consistency (core, L2568); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E366 (Warning): SBO term consistency (core, L2575); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E367 (Warning): SBO term consistency (core, L2582); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E368 (Warning): SBO term consistency (core, L2589); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E369 (Warning): SBO term consistency (core, L2596); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E370 (Warning): SBO term consistency (core, L2603); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E371 (Warning): SBO term consistency (core, L2610); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E372 (Warning): SBO term consistency (core, L2617); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E373 (Warning): SBO term consistency (core, L2624); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E374 (Warning): SBO term consistency (core, L2631); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E375 (Warning): SBO term consistency (core, L2638); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E376 (Warning): SBO term consistency (core, L2645); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E377 (Warning): SBO term consistency (core, L2652); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E378 (Warning): SBO term consistency (core, L2659); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E379 (Warning): SBO term consistency (core, L2666); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E380 (Warning): SBO term consistency (core, L2673); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E381 (Warning): SBO term consistency (core, L2680); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E382 (Warning): SBO term consistency (core, L2687); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E383 (Warning): SBO term consistency (core, L2694); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E384 (Warning): SBO term consistency (core, L2701); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E385 (Warning): SBO term consistency (core, L2708); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E386 (Warning): SBO term consistency (core, L2715); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E387 (Warning): SBO term consistency (core, L2722); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E388 (Warning): SBO term consistency (core, L2729); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E389 (Warning): SBO term consistency (core, L2736); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E390 (Warning): SBO term consistency (core, L2743); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E391 (Warning): SBO term consistency (core, L2750); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E392 (Warning): SBO term consistency (core, L2757); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E393 (Warning): SBO term consistency (core, L2764); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E394 (Warning): SBO term consistency (core, L2771); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E395 (Warning): SBO term consistency (core, L2778); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E396 (Warning): SBO term consistency (core, L2785); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E397 (Warning): SBO term consistency (core, L2792); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E398 (Warning): SBO term consistency (core, L2799); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E399 (Warning): SBO term consistency (core, L2806); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E400 (Warning): SBO term consistency (core, L2813); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E401 (Warning): SBO term consistency (core, L2820); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E402 (Warning): SBO term consistency (core, L2827); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E403 (Warning): SBO term consistency (core, L2834); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E404 (Warning): SBO term consistency (core, L2841); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E405 (Warning): SBO term consistency (core, L2848); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E406 (Warning): SBO term consistency (core, L2855); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E407 (Warning): SBO term consistency (core, L2862); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E408 (Warning): SBO term consistency (core, L2869); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E409 (Warning): SBO term consistency (core, L2876); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E410 (Warning): SBO term consistency (core, L2883); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E411 (Warning): SBO term consistency (core, L2890); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E412 (Warning): SBO term consistency (core, L2897); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E413 (Warning): SBO term consistency (core, L2904); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E414 (Warning): SBO term consistency (core, L2911); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E415 (Warning): SBO term consistency (core, L2918); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E416 (Warning): SBO term consistency (core, L2925); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E417 (Warning): SBO term consistency (core, L2932); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E418 (Warning): SBO term consistency (core, L2939); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E419 (Warning): SBO term consistency (core, L2946); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E420 (Warning): SBO term consistency (core, L2953); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E421 (Warning): SBO term consistency (core, L2960); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E422 (Warning): SBO term consistency (core, L2967); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E423 (Warning): SBO term consistency (core, L2974); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E424 (Warning): SBO term consistency (core, L2981); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E425 (Warning): SBO term consistency (core, L2988); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E426 (Warning): SBO term consistency (core, L2995); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E427 (Warning): SBO term consistency (core, L3002); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E428 (Warning): SBO term consistency (core, L3009); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E429 (Warning): SBO term consistency (core, L3016); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E430 (Warning): SBO term consistency (core, L3023); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E431 (Warning): SBO term consistency (core, L3030); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E432 (Warning): SBO term consistency (core, L3037); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E433 (Warning): SBO term consistency (core, L3044); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E434 (Warning): SBO term consistency (core, L3051); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E435 (Warning): SBO term consistency (core, L3058); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E436 (Warning): SBO term consistency (core, L3065); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E437 (Warning): SBO term consistency (core, L3072); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E438 (Warning): SBO term consistency (core, L3079); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E439 (Warning): SBO term consistency (core, L3086); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E440 (Warning): SBO term consistency (core, L3093); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E441 (Warning): SBO term consistency (core, L3100); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E442 (Warning): SBO term consistency (core, L3107); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E443 (Warning): SBO term consistency (core, L3114); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E444 (Warning): SBO term consistency (core, L3121); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E445 (Warning): SBO term consistency (core, L3128); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E446 (Warning): SBO term consistency (core, L3135); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E447 (Warning): SBO term consistency (core, L3142); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E448 (Warning): SBO term consistency (core, L3149); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E449 (Warning): SBO term consistency (core, L3156); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E450 (Warning): SBO term consistency (core, L3163); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E451 (Warning): SBO term consistency (core, L3170); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E452 (Warning): SBO term consistency (core, L3177); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E453 (Warning): SBO term consistency (core, L3184); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E454 (Warning): SBO term consistency (core, L3191); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E455 (Warning): SBO term consistency (core, L3198); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E456 (Warning): SBO term consistency (core, L3205); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E457 (Warning): SBO term consistency (core, L3212); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E458 (Warning): SBO term consistency (core, L3219); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E459 (Warning): SBO term consistency (core, L3226); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E460 (Warning): SBO term consistency (core, L3233); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E461 (Warning): SBO term consistency (core, L3240); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E462 (Warning): SBO term consistency (core, L3247); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E463 (Warning): SBO term consistency (core, L3254); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E464 (Warning): SBO term consistency (core, L3261); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E465 (Warning): SBO term consistency (core, L3268); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E466 (Warning): SBO term consistency (core, L3275); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E467 (Warning): SBO term consistency (core, L3282); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E468 (Warning): SBO term consistency (core, L3289); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E469 (Warning): SBO term consistency (core, L3296); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E470 (Warning): SBO term consistency (core, L3303); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E471 (Warning): SBO term consistency (core, L3310); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E472 (Warning): SBO term consistency (core, L3317); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E473 (Warning): SBO term consistency (core, L3324); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E474 (Warning): SBO term consistency (core, L3331); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E475 (Warning): SBO term consistency (core, L3338); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E476 (Warning): SBO term consistency (core, L3345); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E477 (Warning): SBO term consistency (core, L3352); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E478 (Warning): SBO term consistency (core, L3359); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E479 (Warning): SBO term consistency (core, L3366); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E480 (Warning): SBO term consistency (core, L3373); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E481 (Warning): SBO term consistency (core, L3380); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E482 (Warning): SBO term consistency (core, L3387); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E483 (Warning): SBO term consistency (core, L3394); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E484 (Warning): SBO term consistency (core, L3401); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E485 (Warning): SBO term consistency (core, L3408); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E486 (Warning): SBO term consistency (core, L3415); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E487 (Warning): SBO term consistency (core, L3422); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E488 (Warning): SBO term consistency (core, L3429); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E489 (Warning): SBO term consistency (core, L3436); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E490 (Warning): SBO term consistency (core, L3443); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E491 (Warning): SBO term consistency (core, L3450); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E492 (Warning): SBO term consistency (core, L3457); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E493 (Warning): SBO term consistency (core, L3464); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E494 (Warning): SBO term consistency (core, L3471); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E495 (Warning): SBO term consistency (core, L3478); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E496 (Warning): SBO term consistency (core, L3485); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E497 (Warning): SBO term consistency (core, L3492); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E498 (Warning): SBO term consistency (core, L3499); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E499 (Warning): SBO term consistency (core, L3506); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E500 (Warning): SBO term consistency (core, L3513); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E501 (Warning): SBO term consistency (core, L3520); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E502 (Warning): SBO term consistency (core, L3527); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E503 (Warning): SBO term consistency (core, L3534); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E504 (Warning): SBO term consistency (core, L3541); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E505 (Warning): SBO term consistency (core, L3548); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E506 (Warning): SBO term consistency (core, L3555); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E507 (Warning): SBO term consistency (core, L3562); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E508 (Warning): SBO term consistency (core, L3569); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E509 (Warning): SBO term consistency (core, L3576); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E510 (Warning): SBO term consistency (core, L3583); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E511 (Warning): SBO term consistency (core, L3590); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E512 (Warning): SBO term consistency (core, L3597); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E513 (Warning): SBO term consistency (core, L3604); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E514 (Warning): SBO term consistency (core, L3611); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E515 (Warning): SBO term consistency (core, L3618); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E516 (Warning): SBO term consistency (core, L3625); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E517 (Warning): SBO term consistency (core, L3632); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E518 (Warning): SBO term consistency (core, L3639); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E519 (Warning): SBO term consistency (core, L3646); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E520 (Warning): SBO term consistency (core, L3653); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E521 (Warning): SBO term consistency (core, L3660); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E522 (Warning): SBO term consistency (core, L3667); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E523 (Warning): SBO term consistency (core, L3674); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E524 (Warning): SBO term consistency (core, L3681); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E525 (Warning): SBO term consistency (core, L3688); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E526 (Warning): SBO term consistency (core, L3695); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E527 (Warning): SBO term consistency (core, L3702); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E528 (Warning): SBO term consistency (core, L3709); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E529 (Warning): SBO term consistency (core, L3716); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E530 (Warning): SBO term consistency (core, L3723); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E531 (Warning): SBO term consistency (core, L3730); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E532 (Warning): SBO term consistency (core, L3737); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E533 (Warning): SBO term consistency (core, L3744); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E534 (Warning): SBO term consistency (core, L3751); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E535 (Warning): SBO term consistency (core, L3758); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E536 (Warning): SBO term consistency (core, L3765); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E537 (Warning): SBO term consistency (core, L3772); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E538 (Warning): SBO term consistency (core, L3779); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E539 (Warning): SBO term consistency (core, L3786); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E540 (Warning): SBO term consistency (core, L3793); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E541 (Warning): SBO term consistency (core, L3800); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E542 (Warning): SBO term consistency (core, L3807); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E543 (Warning): SBO term consistency (core, L3814); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E544 (Warning): SBO term consistency (core, L3821); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E545 (Warning): SBO term consistency (core, L3828); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E546 (Warning): SBO term consistency (core, L3835); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E547 (Warning): SBO term consistency (core, L3842); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E548 (Warning): SBO term consistency (core, L3849); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E549 (Warning): SBO term consistency (core, L3856); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E550 (Warning): SBO term consistency (core, L3863); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E551 (Warning): SBO term consistency (core, L3870); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E552 (Warning): SBO term consistency (core, L3877); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E553 (Warning): SBO term consistency (core, L3884); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E554 (Warning): SBO term consistency (core, L3891); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E555 (Warning): SBO term consistency (core, L3898); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E556 (Warning): SBO term consistency (core, L3905); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E557 (Warning): SBO term consistency (core, L3912); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E558 (Warning): SBO term consistency (core, L3919); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E559 (Warning): SBO term consistency (core, L3926); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E560 (Warning): SBO term consistency (core, L3933); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E561 (Warning): SBO term consistency (core, L3940); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E562 (Warning): SBO term consistency (core, L3947); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E563 (Warning): SBO term consistency (core, L3954); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E564 (Warning): SBO term consistency (core, L3961); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E565 (Warning): SBO term consistency (core, L3968); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E566 (Warning): SBO term consistency (core, L3975); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E567 (Warning): SBO term consistency (core, L3982); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E568 (Warning): SBO term consistency (core, L3989); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E569 (Warning): SBO term consistency (core, L3996); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E570 (Warning): SBO term consistency (core, L4003); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E571 (Warning): SBO term consistency (core, L4010); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E572 (Warning): SBO term consistency (core, L4017); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E573 (Warning): SBO term consistency (core, L4024); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E574 (Warning): SBO term consistency (core, L4031); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E575 (Warning): SBO term consistency (core, L4038); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E576 (Warning): SBO term consistency (core, L4045); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E577 (Warning): SBO term consistency (core, L4052); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E578 (Warning): SBO term consistency (core, L4059); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E579 (Warning): SBO term consistency (core, L4066); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E580 (Warning): SBO term consistency (core, L4073); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E581 (Warning): SBO term consistency (core, L4080); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E582 (Warning): SBO term consistency (core, L4087); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E583 (Warning): SBO term consistency (core, L4094); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E584 (Warning): SBO term consistency (core, L4101); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E585 (Warning): SBO term consistency (core, L4108); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E586 (Warning): SBO term consistency (core, L4115); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E587 (Warning): SBO term consistency (core, L4122); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E588 (Warning): SBO term consistency (core, L4129); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E589 (Warning): SBO term consistency (core, L4136); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E590 (Warning): SBO term consistency (core, L4143); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E591 (Warning): SBO term consistency (core, L4150); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E592 (Warning): SBO term consistency (core, L4157); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E593 (Warning): SBO term consistency (core, L4164); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E594 (Warning): SBO term consistency (core, L4171); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E595 (Warning): SBO term consistency (core, L4178); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E596 (Warning): SBO term consistency (core, L4185); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E597 (Warning): SBO term consistency (core, L4192); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E598 (Warning): SBO term consistency (core, L4199); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E599 (Warning): SBO term consistency (core, L4206); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E600 (Warning): SBO term consistency (core, L4213); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E601 (Warning): SBO term consistency (core, L4220); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E602 (Warning): SBO term consistency (core, L4227); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E603 (Warning): SBO term consistency (core, L4234); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E604 (Warning): SBO term consistency (core, L4241); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E605 (Warning): SBO term consistency (core, L4248); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E606 (Warning): SBO term consistency (core, L4255); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E607 (Warning): SBO term consistency (core, L4262); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E608 (Warning): SBO term consistency (core, L4269); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E609 (Warning): SBO term consistency (core, L4276); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E610 (Warning): SBO term consistency (core, L4283); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E611 (Warning): SBO term consistency (core, L4290); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E612 (Warning): SBO term consistency (core, L4297); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E613 (Warning): SBO term consistency (core, L4304); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E614 (Warning): SBO term consistency (core, L4311); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E615 (Warning): SBO term consistency (core, L4318); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E616 (Warning): SBO term consistency (core, L4325); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E617 (Warning): SBO term consistency (core, L4332); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E618 (Warning): SBO term consistency (core, L4339); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E619 (Warning): SBO term consistency (core, L4346); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E620 (Warning): SBO term consistency (core, L4353); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E621 (Warning): SBO term consistency (core, L4360); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E622 (Warning): SBO term consistency (core, L4367); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E623 (Warning): SBO term consistency (core, L4374); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E624 (Warning): SBO term consistency (core, L4381); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E625 (Warning): SBO term consistency (core, L4388); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E626 (Warning): SBO term consistency (core, L4395); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E627 (Warning): SBO term consistency (core, L4402); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E628 (Warning): SBO term consistency (core, L4409); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E629 (Warning): SBO term consistency (core, L4416); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E630 (Warning): SBO term consistency (core, L4423); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E631 (Warning): SBO term consistency (core, L4430); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E632 (Warning): SBO term consistency (core, L4437); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E633 (Warning): SBO term consistency (core, L4444); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E634 (Warning): SBO term consistency (core, L4451); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E635 (Warning): SBO term consistency (core, L4458); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E636 (Warning): SBO term consistency (core, L4465); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E637 (Warning): SBO term consistency (core, L4472); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E638 (Warning): SBO term consistency (core, L4479); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E639 (Warning): SBO term consistency (core, L4486); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E640 (Warning): SBO term consistency (core, L4493); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E641 (Warning): SBO term consistency (core, L4500); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E642 (Warning): SBO term consistency (core, L4507); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E643 (Warning): SBO term consistency (core, L4514); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E644 (Warning): SBO term consistency (core, L4521); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E645 (Warning): SBO term consistency (core, L4528); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E646 (Warning): SBO term consistency (core, L4535); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E647 (Warning): SBO term consistency (core, L4542); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E648 (Warning): SBO term consistency (core, L4549); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E649 (Warning): SBO term consistency (core, L4556); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E650 (Warning): SBO term consistency (core, L4563); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E651 (Warning): SBO term consistency (core, L4570); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E652 (Warning): SBO term consistency (core, L4577); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E653 (Warning): SBO term consistency (core, L4584); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E654 (Warning): SBO term consistency (core, L4591); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E655 (Warning): SBO term consistency (core, L4598); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E656 (Warning): SBO term consistency (core, L4605); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E657 (Warning): SBO term consistency (core, L4612); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E658 (Warning): SBO term consistency (core, L4619); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E659 (Warning): SBO term consistency (core, L4626); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E660 (Warning): SBO term consistency (core, L4633); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E661 (Warning): SBO term consistency (core, L4640); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E662 (Warning): SBO term consistency (core, L4647); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E663 (Warning): SBO term consistency (core, L4654); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E664 (Warning): SBO term consistency (core, L4661); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E665 (Warning): SBO term consistency (core, L4668); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E666 (Warning): SBO term consistency (core, L4675); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E667 (Warning): SBO term consistency (core, L4682); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E668 (Warning): SBO term consistency (core, L4689); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E669 (Warning): SBO term consistency (core, L4696); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E670 (Warning): SBO term consistency (core, L4703); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E671 (Warning): SBO term consistency (core, L4710); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E672 (Warning): SBO term consistency (core, L4717); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E673 (Warning): SBO term consistency (core, L4724); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E674 (Warning): SBO term consistency (core, L4731); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E675 (Warning): SBO term consistency (core, L4738); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E676 (Warning): SBO term consistency (core, L4745); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E677 (Warning): SBO term consistency (core, L4752); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E678 (Warning): SBO term consistency (core, L4759); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E679 (Warning): SBO term consistency (core, L4766); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E680 (Warning): SBO term consistency (core, L4773); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E681 (Warning): SBO term consistency (core, L4780); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E682 (Warning): SBO term consistency (core, L4787); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E683 (Warning): SBO term consistency (core, L4794); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E684 (Warning): SBO term consistency (core, L4801); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E685 (Warning): SBO term consistency (core, L4808); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E686 (Warning): SBO term consistency (core, L4815); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E687 (Warning): SBO term consistency (core, L4822); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E688 (Warning): SBO term consistency (core, L4829); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E689 (Warning): SBO term consistency (core, L4836); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E690 (Warning): SBO term consistency (core, L4843); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E691 (Warning): SBO term consistency (core, L4850); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E692 (Warning): SBO term consistency (core, L4857); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E693 (Warning): SBO term consistency (core, L4864); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E694 (Warning): SBO term consistency (core, L4871); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E695 (Warning): SBO term consistency (core, L4878); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E696 (Warning): SBO term consistency (core, L4885); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E697 (Warning): SBO term consistency (core, L4892); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E698 (Warning): SBO term consistency (core, L4899); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E699 (Warning): SBO term consistency (core, L4906); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E700 (Warning): SBO term consistency (core, L4913); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E701 (Warning): SBO term consistency (core, L4920); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E702 (Warning): SBO term consistency (core, L4927); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E703 (Warning): SBO term consistency (core, L4934); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E704 (Warning): SBO term consistency (core, L4941); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E705 (Warning): SBO term consistency (core, L4948); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E706 (Warning): SBO term consistency (core, L4955); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E707 (Warning): SBO term consistency (core, L4962); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E708 (Warning): SBO term consistency (core, L4969); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E709 (Warning): SBO term consistency (core, L4976); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E710 (Warning): SBO term consistency (core, L4983); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E711 (Warning): SBO term consistency (core, L4990); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E712 (Warning): SBO term consistency (core, L4997); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E713 (Warning): SBO term consistency (core, L5004); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E714 (Warning): SBO term consistency (core, L5011); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E715 (Warning): SBO term consistency (core, L5018); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E716 (Warning): SBO term consistency (core, L5025); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E717 (Warning): SBO term consistency (core, L5032); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E718 (Warning): SBO term consistency (core, L5039); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E719 (Warning): SBO term consistency (core, L5046); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E720 (Warning): SBO term consistency (core, L5053); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E721 (Warning): SBO term consistency (core, L5060); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E722 (Warning): SBO term consistency (core, L5067); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E723 (Warning): SBO term consistency (core, L5074); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E724 (Warning): SBO term consistency (core, L5081); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E725 (Warning): SBO term consistency (core, L5088); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E726 (Warning): SBO term consistency (core, L5095); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E727 (Warning): SBO term consistency (core, L5102); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E728 (Warning): SBO term consistency (core, L5109); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E729 (Warning): SBO term consistency (core, L5116); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E730 (Warning): SBO term consistency (core, L5123); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E731 (Warning): SBO term consistency (core, L5130); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E732 (Warning): SBO term consistency (core, L5137); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E733 (Warning): SBO term consistency (core, L5144); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E734 (Warning): SBO term consistency (core, L5151); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E735 (Warning): SBO term consistency (core, L5158); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E736 (Warning): SBO term consistency (core, L5165); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E737 (Warning): SBO term consistency (core, L5172); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E738 (Warning): SBO term consistency (core, L5179); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E739 (Warning): SBO term consistency (core, L5186); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E740 (Warning): SBO term consistency (core, L5193); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E741 (Warning): SBO term consistency (core, L5200); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E742 (Warning): SBO term consistency (core, L5207); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E743 (Warning): SBO term consistency (core, L5214); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E744 (Warning): SBO term consistency (core, L5221); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E745 (Warning): SBO term consistency (core, L5228); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E746 (Warning): SBO term consistency (core, L5235); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E747 (Warning): SBO term consistency (core, L5242); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E748 (Warning): SBO term consistency (core, L5249); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E749 (Warning): SBO term consistency (core, L5256); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E750 (Warning): SBO term consistency (core, L5263); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E751 (Warning): SBO term consistency (core, L5270); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E752 (Warning): SBO term consistency (core, L5277); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E753 (Warning): SBO term consistency (core, L5284); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E754 (Warning): SBO term consistency (core, L5291); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E755 (Warning): SBO term consistency (core, L5298); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E756 (Warning): SBO term consistency (core, L5305); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E757 (Warning): SBO term consistency (core, L5312); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E758 (Warning): SBO term consistency (core, L5319); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E759 (Warning): SBO term consistency (core, L5326); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E760 (Warning): SBO term consistency (core, L5333); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E761 (Warning): SBO term consistency (core, L5340); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E762 (Warning): SBO term consistency (core, L5347); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E763 (Warning): SBO term consistency (core, L5354); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E764 (Warning): SBO term consistency (core, L5361); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E765 (Warning): SBO term consistency (core, L5368); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E766 (Warning): SBO term consistency (core, L5375); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E767 (Warning): SBO term consistency (core, L5382); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E768 (Warning): SBO term consistency (core, L5389); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E769 (Warning): SBO term consistency (core, L5396); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E770 (Warning): SBO term consistency (core, L5403); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E771 (Warning): SBO term consistency (core, L5410); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E772 (Warning): SBO term consistency (core, L5417); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E773 (Warning): SBO term consistency (core, L5424); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E774 (Warning): SBO term consistency (core, L5431); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E775 (Warning): SBO term consistency (core, L5438); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E776 (Warning): SBO term consistency (core, L5445); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E777 (Warning): SBO term consistency (core, L5452); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E778 (Warning): SBO term consistency (core, L5459); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E779 (Warning): SBO term consistency (core, L5466); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E780 (Warning): SBO term consistency (core, L5473); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E781 (Warning): SBO term consistency (core, L5480); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E782 (Warning): SBO term consistency (core, L5487); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E783 (Warning): SBO term consistency (core, L5494); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E784 (Warning): SBO term consistency (core, L5501); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E785 (Warning): SBO term consistency (core, L5508); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E786 (Warning): SBO term consistency (core, L5515); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E787 (Warning): SBO term consistency (core, L5522); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E788 (Warning): SBO term consistency (core, L5529); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E789 (Warning): SBO term consistency (core, L5536); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E790 (Warning): SBO term consistency (core, L5543); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E791 (Warning): SBO term consistency (core, L5550); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E792 (Warning): SBO term consistency (core, L5557); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E793 (Warning): SBO term consistency (core, L5564); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E794 (Warning): SBO term consistency (core, L5571); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E795 (Warning): SBO term consistency (core, L5578); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E796 (Warning): SBO term consistency (core, L5585); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E797 (Warning): SBO term consistency (core, L5592); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E798 (Warning): SBO term consistency (core, L5599); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E799 (Warning): SBO term consistency (core, L5606); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E800 (Warning): SBO term consistency (core, L5613); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E801 (Warning): SBO term consistency (core, L5620); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E802 (Warning): SBO term consistency (core, L5627); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E803 (Warning): SBO term consistency (core, L5634); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E804 (Warning): SBO term consistency (core, L5641); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E805 (Warning): SBO term consistency (core, L5648); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E806 (Warning): SBO term consistency (core, L5655); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E807 (Warning): SBO term consistency (core, L5662); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E808 (Warning): SBO term consistency (core, L5669); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E809 (Warning): SBO term consistency (core, L5676); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E810 (Warning): SBO term consistency (core, L5683); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E811 (Warning): SBO term consistency (core, L5690); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E812 (Warning): SBO term consistency (core, L5697); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E813 (Warning): SBO term consistency (core, L5704); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E814 (Warning): SBO term consistency (core, L5711); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E815 (Warning): SBO term consistency (core, L5718); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E816 (Warning): SBO term consistency (core, L5725); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E817 (Warning): SBO term consistency (core, L5732); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E818 (Warning): SBO term consistency (core, L5739); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E819 (Warning): SBO term consistency (core, L5746); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E820 (Warning): SBO term consistency (core, L5753); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E821 (Warning): SBO term consistency (core, L5760); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E822 (Warning): SBO term consistency (core, L5767); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E823 (Warning): SBO term consistency (core, L5774); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E824 (Warning): SBO term consistency (core, L5781); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E825 (Warning): SBO term consistency (core, L5788); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E826 (Warning): SBO term consistency (core, L5795); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E827 (Warning): SBO term consistency (core, L5802); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E828 (Warning): SBO term consistency (core, L5809); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E829 (Warning): SBO term consistency (core, L5816); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E830 (Warning): SBO term consistency (core, L5823); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E831 (Warning): SBO term consistency (core, L5830); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E832 (Warning): SBO term consistency (core, L5837); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E833 (Warning): SBO term consistency (core, L5844); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E834 (Warning): SBO term consistency (core, L5851); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E835 (Warning): SBO term consistency (core, L5858); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E836 (Warning): SBO term consistency (core, L5865); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E837 (Warning): SBO term consistency (core, L5872); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E838 (Warning): SBO term consistency (core, L5879); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E839 (Warning): SBO term consistency (core, L5886); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E840 (Warning): SBO term consistency (core, L5893); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E841 (Warning): SBO term consistency (core, L5900); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E842 (Warning): SBO term consistency (core, L5907); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E843 (Warning): SBO term consistency (core, L5914); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E844 (Warning): SBO term consistency (core, L5921); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E845 (Warning): SBO term consistency (core, L5928); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E846 (Warning): SBO term consistency (core, L5935); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E847 (Warning): SBO term consistency (core, L5942); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E848 (Warning): SBO term consistency (core, L5949); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E849 (Warning): SBO term consistency (core, L5956); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E850 (Warning): SBO term consistency (core, L5963); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E851 (Warning): SBO term consistency (core, L5970); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E852 (Warning): SBO term consistency (core, L5977); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E853 (Warning): SBO term consistency (core, L5984); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E854 (Warning): SBO term consistency (core, L5991); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E855 (Warning): SBO term consistency (core, L5998); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E856 (Warning): SBO term consistency (core, L6005); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E857 (Warning): SBO term consistency (core, L6012); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E858 (Warning): SBO term consistency (core, L6019); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E859 (Warning): SBO term consistency (core, L6026); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E860 (Warning): SBO term consistency (core, L6033); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E861 (Warning): SBO term consistency (core, L6040); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E862 (Warning): SBO term consistency (core, L6047); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E863 (Warning): SBO term consistency (core, L6054); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E864 (Warning): SBO term consistency (core, L6061); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E865 (Warning): SBO term consistency (core, L6068); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E866 (Warning): SBO term consistency (core, L6075); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E867 (Warning): SBO term consistency (core, L6082); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E868 (Warning): SBO term consistency (core, L6089); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E869 (Warning): SBO term consistency (core, L6096); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E870 (Warning): SBO term consistency (core, L6103); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E871 (Warning): SBO term consistency (core, L6110); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E872 (Warning): SBO term consistency (core, L6117); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E873 (Warning): SBO term consistency (core, L6124); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E874 (Warning): SBO term consistency (core, L6131); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E875 (Warning): SBO term consistency (core, L6138); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E876 (Warning): SBO term consistency (core, L6145); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E877 (Warning): SBO term consistency (core, L6152); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E878 (Warning): SBO term consistency (core, L6159); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E879 (Warning): SBO term consistency (core, L6166); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E880 (Warning): SBO term consistency (core, L6173); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E881 (Warning): SBO term consistency (core, L6180); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E882 (Warning): SBO term consistency (core, L6187); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E883 (Warning): SBO term consistency (core, L6194); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E884 (Warning): SBO term consistency (core, L6201); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E885 (Warning): SBO term consistency (core, L6208); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E886 (Warning): SBO term consistency (core, L6215); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E887 (Warning): SBO term consistency (core, L6222); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E888 (Warning): SBO term consistency (core, L6229); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E889 (Warning): SBO term consistency (core, L6236); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E890 (Warning): SBO term consistency (core, L6243); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E891 (Warning): SBO term consistency (core, L6250); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E892 (Warning): SBO term consistency (core, L6257); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E893 (Warning): SBO term consistency (core, L6264); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E894 (Warning): SBO term consistency (core, L6271); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E895 (Warning): SBO term consistency (core, L6278); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E896 (Warning): SBO term consistency (core, L6285); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E897 (Warning): SBO term consistency (core, L6292); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E898 (Warning): SBO term consistency (core, L6299); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E899 (Warning): SBO term consistency (core, L6306); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E900 (Warning): SBO term consistency (core, L6313); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E901 (Warning): SBO term consistency (core, L6320); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E902 (Warning): SBO term consistency (core, L6327); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E903 (Warning): SBO term consistency (core, L6334); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E904 (Warning): SBO term consistency (core, L6341); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E905 (Warning): SBO term consistency (core, L6348); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E906 (Warning): SBO term consistency (core, L6355); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E907 (Warning): SBO term consistency (core, L6362); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E908 (Warning): SBO term consistency (core, L6369); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E909 (Warning): SBO term consistency (core, L6376); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E910 (Warning): SBO term consistency (core, L6383); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E911 (Warning): SBO term consistency (core, L6390); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E912 (Warning): SBO term consistency (core, L6397); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E913 (Warning): SBO term consistency (core, L6404); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E914 (Warning): SBO term consistency (core, L6411); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E915 (Warning): SBO term consistency (core, L6418); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E916 (Warning): SBO term consistency (core, L6425); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E917 (Warning): SBO term consistency (core, L6432); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E918 (Warning): SBO term consistency (core, L6439); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E919 (Warning): SBO term consistency (core, L6446); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E920 (Warning): SBO term consistency (core, L6453); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E921 (Warning): SBO term consistency (core, L6460); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E922 (Warning): SBO term consistency (core, L6467); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E923 (Warning): SBO term consistency (core, L6474); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E924 (Warning): SBO term consistency (core, L6481); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E925 (Warning): SBO term consistency (core, L6488); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E926 (Warning): SBO term consistency (core, L6495); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E927 (Warning): SBO term consistency (core, L6502); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E928 (Warning): SBO term consistency (core, L6509); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E929 (Warning): SBO term consistency (core, L6516); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E930 (Warning): SBO term consistency (core, L6523); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E931 (Warning): SBO term consistency (core, L6530); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E932 (Warning): SBO term consistency (core, L6537); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E933 (Warning): SBO term consistency (core, L6544); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E934 (Warning): SBO term consistency (core, L6551); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E935 (Warning): SBO term consistency (core, L6558); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E936 (Warning): SBO term consistency (core, L6565); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E937 (Warning): SBO term consistency (core, L6572); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E938 (Warning): SBO term consistency (core, L6579); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E939 (Warning): SBO term consistency (core, L6586); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E940 (Warning): SBO term consistency (core, L6593); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E941 (Warning): SBO term consistency (core, L6600); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E942 (Warning): SBO term consistency (core, L6607); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E943 (Warning): SBO term consistency (core, L6614); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E944 (Warning): SBO term consistency (core, L6621); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E945 (Warning): SBO term consistency (core, L6628); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E946 (Warning): SBO term consistency (core, L6635); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E947 (Warning): SBO term consistency (core, L6642); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E948 (Warning): SBO term consistency (core, L6649); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E949 (Warning): SBO term consistency (core, L6656); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E950 (Warning): SBO term consistency (core, L6663); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E951 (Warning): SBO term consistency (core, L6670); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E952 (Warning): SBO term consistency (core, L6677); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E953 (Warning): SBO term consistency (core, L6684); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E954 (Warning): SBO term consistency (core, L6691); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E955 (Warning): SBO term consistency (core, L6698); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E956 (Warning): SBO term consistency (core, L6705); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E957 (Warning): SBO term consistency (core, L6712); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E958 (Warning): SBO term consistency (core, L6719); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E959 (Warning): SBO term consistency (core, L6726); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E960 (Warning): SBO term consistency (core, L6733); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E961 (Warning): SBO term consistency (core, L6740); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E962 (Warning): SBO term consistency (core, L6747); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E963 (Warning): SBO term consistency (core, L6754); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E964 (Warning): SBO term consistency (core, L6761); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E965 (Warning): SBO term consistency (core, L6768); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E966 (Warning): SBO term consistency (core, L6775); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E967 (Warning): SBO term consistency (core, L6782); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E968 (Warning): SBO term consistency (core, L6789); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E969 (Warning): SBO term consistency (core, L6796); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E970 (Warning): SBO term consistency (core, L6803); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E971 (Warning): SBO term consistency (core, L6810); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E972 (Warning): SBO term consistency (core, L6817); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E973 (Warning): SBO term consistency (core, L6824); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E974 (Warning): SBO term consistency (core, L6831); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E975 (Warning): SBO term consistency (core, L6838); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E976 (Warning): SBO term consistency (core, L6845); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E977 (Warning): SBO term consistency (core, L6852); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E978 (Warning): SBO term consistency (core, L6859); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E979 (Warning): SBO term consistency (core, L6866); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E980 (Warning): SBO term consistency (core, L6873); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E981 (Warning): SBO term consistency (core, L6880); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E982 (Warning): SBO term consistency (core, L6887); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E983 (Warning): SBO term consistency (core, L6894); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E984 (Warning): SBO term consistency (core, L6901); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E985 (Warning): SBO term consistency (core, L6908); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E986 (Warning): SBO term consistency (core, L6915); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E987 (Warning): SBO term consistency (core, L6922); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E988 (Warning): SBO term consistency (core, L6929); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E989 (Warning): SBO term consistency (core, L6936); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E990 (Warning): SBO term consistency (core, L6943); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E991 (Warning): SBO term consistency (core, L6950); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E992 (Warning): SBO term consistency (core, L6957); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E993 (Warning): SBO term consistency (core, L6964); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E994 (Warning): SBO term consistency (core, L6971); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E995 (Warning): SBO term consistency (core, L6978); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E996 (Warning): SBO term consistency (core, L6985); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E997 (Warning): SBO term consistency (core, L6992); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E998 (Warning): SBO term consistency (core, L6999); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  'E999 (Warning): SBO term consistency (core, L7006); Invalid \\'sboTerm\\' attribute value for a Species object; The value of the \\'sboTerm\\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, \"participant physical type\"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, \"material entity\").\\nReference: L3V1 Section 5\\n SBO term \\'SBO:0000299\\' on the <species> is not in the appropriate branch.\\n',\n",
       "  ...],\n",
       " 'COBRA_FATAL': ['id Q92506 is already present in list'],\n",
       " 'COBRA_ERROR': [],\n",
       " 'COBRA_WARNING': [],\n",
       " 'COBRA_CHECK': []}"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "errors"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "7fcdcf2a",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "COBRA errors in validation, check error log for details.\n",
      "COBRA warnings in validation, check error log for details.\n"
     ]
    }
   ],
   "source": [
    "# Hep_G2 : Cancer liver from 2015\n",
    "# Built from HMR 2.0 using tINIT.\n",
    "\n",
    "#Hep_G2_temp = cbmpy.CBRead.readSBML2FBA('./Hep-G2/Hep-G2.xml-5a91f1955e8a9c8a5c5d2ff4a1737c21/Hep-G2.xml')\n",
    "\n",
    "\n",
    "#path= './Hep-G2/Hep-G2.xml-5a91f1955e8a9c8a5c5d2ff4a1737c21/Hep-G2_cbmpy_wgenes_unip_ab.xml'\n",
    "Hep_G2, errors = validate_sbml_model(Hep_G2_dir+\"Hep-G2_cbmpy.xml\")\n",
    "#Hep_G2 = cobra.io.read_sbml_model('./Hep-G2/Hep-G2.xml-5a91f1955e8a9c8a5c5d2ff4a1737c21/Hep-G2.xml')\n"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "aab764e5",
   "metadata": {},
   "source": [
    "### Correction du modèle"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "85963d1d",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "SBML warnings in validation, check error log for details.\n",
      "COBRA warnings in validation, check error log for details.\n"
     ]
    }
   ],
   "source": [
    "# Importation du modèle de cellule saine hépatique pour aider à la correction du modèle.\n",
    "iHep, errors = validate_sbml_model(iHep_dir+\"iHepatocytes2322_Cobra.xml\")"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "3f1d4429",
   "metadata": {},
   "source": [
    "All the functions used to correct the model are gathered in a [separate python file](./Model_correction.py)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "717c9d46",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<module 'Model_correction' from '/home/jmuller/Documents/Stage_metabo/Model_correction.py'>"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "importlib.reload(mc)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "b02f7ff1",
   "metadata": {},
   "source": [
    "---\n",
    "#### Ajout des gènes associés aux réactions, en les copiant du modèle de cellule saine."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "aa582a8c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00097x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00157x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00228x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00242x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00266x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00267x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00268x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00269x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00402x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00545x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00648x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00932x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m00970x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01020x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01070x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01100x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01107x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01111x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01115x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01159x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01174x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01252x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01253x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01256x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01260x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01280x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01285x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01286x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01287x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01288x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01290x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01296x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01306x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01307x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01308x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01327x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01330x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01334x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01339x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01361x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01365x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01368x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01369x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01370x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01374x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01383x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01393x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01396x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01397x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01398x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01400x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01401x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01403x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01410x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01419x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01433x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01438x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01442x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01445x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01450x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01513x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01517x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01525x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01533x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01543x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01554x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01569x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01570x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01587x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01588x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01590x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01596x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01615x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01619x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01621x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01626x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01628x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01629x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01630x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01638x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01641x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01648x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01652x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01655x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01659x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01666x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01668x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01669x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01671x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01673x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01682x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01700x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01714x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01716x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01736x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01742x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01743x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01744x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01745x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01758x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01765x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01786x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01788x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01789x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01795x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01796x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01797x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01799x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01800x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01821x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01822x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01830x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01833x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01840x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01874x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01910x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01913x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01935x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01938x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01948x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01959x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01962x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01965x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01974x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01975x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01983x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01986x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01987x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m01988x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02001x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02016x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02023x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02026x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02027x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02034x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02038x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02039x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02040x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02041x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02046x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02047x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02049x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02054x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02104x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02105x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02108x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02118x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02119x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02120x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02125x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02136x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02141x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02145x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02147x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02161x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02167x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02170x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02171x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02184x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02193x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02200x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02237x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02278x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02288x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02303x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02332x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02337x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02348x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02352x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02353x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02354x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02357x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02360x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02362x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02366x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02369x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02387x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02389x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02394x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02403x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02418x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02426x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02445x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02450x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02453x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02470x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02471x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02510x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02519x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02556x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02560x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02578x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02614x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02617x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02630x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02631x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02642x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02658x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02661x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02672x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02680x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02691x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02692x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02724x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02746x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02751x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02754x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02770x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02772x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02785x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02786x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02789x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02815x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02819x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02833x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02842x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02885x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02896x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02897x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02924x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02926x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02930x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02943x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02945x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02946x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02949x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02961x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02962x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02963x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02967x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02980x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02982x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02983x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02985x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02993x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02996x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m02998x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03039x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03044x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03052x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03089x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03101x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03106x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03114x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03117x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03120x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03121x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03123x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03130x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03134x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03135x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03141x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03146x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03147x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03151x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03154x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_m03157x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_temp001x might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model EX_temp006x might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3905\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3907\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4097\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4099\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4108\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4133\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4137\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4281\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4388\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4283\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8357\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4379\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4301\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4355\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4358\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4360\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4363\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4365\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4368\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4370\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4371\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4372\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4373\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4375\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4381\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4391\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4394\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4396\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4521\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6410\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6412\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7745\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7746\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7747\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7748\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7749\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8360\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3989\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4122\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4837\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5395\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5396\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9727\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5397\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5398\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5399\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5400\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5401\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8568\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8570\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8572\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8574\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8576\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8578\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8591\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8592\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8587\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8589\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3944\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4128\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4130\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4131\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4132\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4303\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4414\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4415\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4416\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4774\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4775\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7674\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8761\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8762\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8767\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4297\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4315\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4316\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4319\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4383\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4385\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4386\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4387\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4399\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4400\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4401\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4402\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4490\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4706\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8768\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4590\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4591\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4592\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4595\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8341\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8726\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8727\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8729\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6537\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1568\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3853\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3854\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3855\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3857\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3859\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4087\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4089\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4091\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4103\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4143\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4193\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8497\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8500\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8501\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8502\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8503\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8504\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8506\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8507\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8508\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8509\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8511\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8512\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8514\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8516\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4280\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0153\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3212\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3797\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4105\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4106\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4107\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4282\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4331\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4459\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4460\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4497\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4741\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9804\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1434\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1436\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3163\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4461\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4604\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5351\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4052\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4304\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4306\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4350\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4351\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4352\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4354\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4398\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4404\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4473\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4474\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4477\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4501\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4565\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4567\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4568\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4623\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4625\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4710\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4841\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8074\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8653\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3998\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4000\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4002\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4004\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4010\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4081\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4082\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4012\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4014\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4020\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4022\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4038\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4040\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4042\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4046\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4048\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4080\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4085\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4134\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4168\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4171\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4290\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4406\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4412\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4417\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4418\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4419\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4421\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4449\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4450\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4452\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4453\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4480\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4481\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4482\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4486\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4487\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4488\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4489\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4492\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4493\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4495\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4496\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4573\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4574\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4600\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4602\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4603\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4611\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4612\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4614\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4615\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4617\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4618\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4619\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4621\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4632\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4651\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4663\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4694\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4695\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4705\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4709\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4799\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4802\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4804\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4808\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4810\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4812\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4814\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5301\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5353\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6602\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6603\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7144\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4016\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4018\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4044\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4291\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4420\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4570\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4572\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4006\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4028\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4030\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4670\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4673\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4675\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6614\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6615\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6616\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4635\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3931\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3969\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3970\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4008\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4032\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4034\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4036\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4050\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4056\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4059\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4060\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4127\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4177\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4179\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4182\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4183\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4192\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4194\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4210\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4343\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4346\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4484\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4485\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4510\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4512\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4513\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4514\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4577\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4579\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4580\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4608\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4575\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4637\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4642\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4643\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4644\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4676\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5299\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5352\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6612\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6621\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6622\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6623\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6624\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6627\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8072\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3965\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3966\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3967\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3968\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3999\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4024\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4026\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4054\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4061\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4136\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4180\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4185\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4186\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4342\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4344\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4483\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4507\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4516\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4601\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4633\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4640\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4641\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4680\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4806\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6625\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6626\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7160\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7161\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7162\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7163\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7164\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7713\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7716\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7721\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7725\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7728\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7801\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7863\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7864\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7865\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7866\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7867\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7868\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7869\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7870\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7871\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7872\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7873\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7874\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7876\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7878\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7879\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7880\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7881\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7882\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7883\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7884\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7885\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7886\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7887\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7888\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7889\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7890\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7891\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7892\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7893\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8083\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8085\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8087\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8443\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8444\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8448\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8449\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8450\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8451\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8452\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8453\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8454\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8455\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8456\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8457\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8458\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8459\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8460\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8462\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8463\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8464\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8465\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8466\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8467\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8468\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8469\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8470\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8471\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8472\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8473\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8474\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8478\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8479\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8482\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8483\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8485\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8486\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8487\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8488\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8489\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3802\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3804\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3811\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3813\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3822\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3827\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3829\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3831\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3862\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3865\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3870\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3873\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8654\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3890\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3892\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9802\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3899\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3903\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4109\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4114\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4115\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4118\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4172\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4196\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4197\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4287\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4693\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6780\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7641\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7642\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8626\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8628\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3806\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3807\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3816\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3819\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3820\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3833\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3835\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3837\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3838\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3895\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3897\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3956\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3993\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4073\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4075\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4077\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4190\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4191\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4212\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4422\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4423\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4424\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4583\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4606\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4607\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4776\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4777\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4778\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4779\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4780\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4781\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4784\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4785\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4786\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4787\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5384\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6929\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6930\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6931\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6932\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6933\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6934\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6935\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6936\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6938\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6939\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6941\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6942\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6943\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6944\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6945\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6946\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6947\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6948\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6949\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6950\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6951\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6952\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6956\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6957\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6958\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6959\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6961\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6963\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6964\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8096\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8097\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8098\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8426\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8427\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8432\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8603\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8604\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8605\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8609\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8610\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8611\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4285\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0457\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0460\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3752\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3770\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3784\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3839\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3841\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3843\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3845\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3847\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3849\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3852\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3856\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3860\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3883\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3901\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3939\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3974\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4198\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4200\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4348\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4467\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4584\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4696\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4697\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4698\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4699\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4700\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4742\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4788\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4791\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4792\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5392\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5393\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6409\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7702\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8433\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8434\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8435\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8436\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8437\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8439\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8440\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8441\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8442\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9718\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9486\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4428\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4429\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4430\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4431\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4437\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4658\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4660\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4712\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5336\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5337\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5338\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5339\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5340\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8783\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8784\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8786\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4239\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4288\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4596\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4597\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4599\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4667\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4668\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4737\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4739\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6415\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6974\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6975\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6976\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6977\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6978\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6979\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6980\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6981\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6982\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6983\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6984\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6985\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8017\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8018\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8019\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8021\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8025\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8026\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8027\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8029\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3164\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3166\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3885\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4250\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8091\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8092\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8563\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8564\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8566\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3935\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4681\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4682\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4683\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4685\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4686\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4687\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7689\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8529\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8530\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8534\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8535\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3206\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3208\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3211\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3213\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3215\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3747\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3744\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3757\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3761\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3777\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3778\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3795\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4797\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6417\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6419\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6420\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6421\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6422\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6923\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3765\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8088\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8787\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3940\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4214\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4216\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4222\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4225\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4242\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4243\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4244\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4246\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6707\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6713\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6714\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6715\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6716\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6717\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6718\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6719\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6730\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6731\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6734\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6736\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6738\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6740\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6745\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6746\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6747\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6748\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6749\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6750\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6751\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6752\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6753\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6754\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6755\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6756\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6758\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6760\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6761\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6762\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6763\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6764\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6766\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6767\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6768\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6772\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6774\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6776\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6778\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6782\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6783\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6784\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6785\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6786\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6787\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6788\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6789\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6790\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6791\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6800\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6801\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6808\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6813\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6815\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6826\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6827\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6829\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6831\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6834\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6835\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6836\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6837\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6838\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6839\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6841\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6844\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6845\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6849\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6850\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6851\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6854\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6855\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7628\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7756\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8539\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3875\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3877\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3879\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3881\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3912\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3917\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3928\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3995\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4189\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4302\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4323\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5385\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5386\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5387\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5388\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5389\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5391\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6515\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6518\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6519\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6520\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8062\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8066\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8067\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8068\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8069\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8641\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3868\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3869\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3889\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3933\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3937\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4116\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4120\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4121\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4175\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4176\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4324\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4326\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4328\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4329\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4708\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8769\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8770\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4079\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4330\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4425\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4427\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4689\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4735\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7991\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7992\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7993\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7128\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7129\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7130\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7131\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7133\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7134\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7135\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7136\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7137\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7138\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7139\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7140\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7141\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8640\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3748\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3751\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3753\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3755\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3759\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3763\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3767\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3769\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3773\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3775\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3780\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3783\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3785\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3790\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3792\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3796\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3823\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6416\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3986\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3988\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4124\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4158\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4159\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4299\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4300\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4494\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4524\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4525\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4526\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4527\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4528\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4529\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4530\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4532\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4536\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4628\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4629\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4631\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7696\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7697\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8371\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8375\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8377\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8672\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5130\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5131\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5132\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5133\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5134\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5135\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5136\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5137\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5138\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5139\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5140\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5141\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5142\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5143\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5145\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5146\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5147\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5148\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5149\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5150\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7171\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7174\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7175\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7180\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7181\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7436\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7438\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7440\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8257\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1532\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7254\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7255\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7256\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7261\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7263\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7264\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7265\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7266\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7267\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7268\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7269\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7270\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7271\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7274\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7275\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7276\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7277\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7278\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7279\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7280\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7281\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7285\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7286\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7287\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7288\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7289\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7290\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7291\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7293\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7300\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7308\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7309\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7310\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7311\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7312\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7313\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7314\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7315\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7316\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7317\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7318\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7319\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7320\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7321\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7322\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7323\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7324\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7325\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7327\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7328\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7332\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7333\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7334\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7428\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7429\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7574\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7575\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7576\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7577\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7578\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7580\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7582\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7585\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7586\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7587\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8691\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8693\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8694\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5151\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5166\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5169\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5170\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5171\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5172\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7617\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7618\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7621\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5258\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5259\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5282\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5284\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5285\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5286\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5287\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5288\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5407\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3861\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3866\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3915\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4084\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4187\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4701\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4840\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8662\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8663\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8667\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8668\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0710\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3787\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3957\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3958\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4111\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4112\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4113\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4139\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4141\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4145\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4147\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4149\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4152\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4209\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4408\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4410\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4454\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4456\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4458\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4465\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4585\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4586\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4587\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4588\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4589\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4652\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5294\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5297\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6411\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6413\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6414\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8743\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8772\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8773\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8775\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8777\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8778\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8780\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8782\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3975\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3977\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3979\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6911\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6912\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6914\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6916\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6918\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6921\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3960\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3980\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3982\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4767\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8410\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8413\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8415\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0156\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0165\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0168\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0171\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0174\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0177\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0180\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0184\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0188\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0192\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0196\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0200\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0204\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0209\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0213\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0217\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0226\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0233\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0237\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0241\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0245\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0249\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0255\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0259\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0263\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0267\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0271\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0275\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0279\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0283\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0289\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0293\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0297\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0301\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0305\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0309\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0313\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0319\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0323\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0327\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0331\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0337\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0341\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0345\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0349\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0353\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0357\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0361\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0365\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0369\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0373\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0377\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0381\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0385\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0389\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0393\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0397\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0401\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0405\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0409\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0413\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0417\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0421\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0425\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0429\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0433\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0437\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2942\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2943\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2944\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2945\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2946\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2947\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2948\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2949\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2951\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2952\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2954\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2955\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2956\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2957\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2959\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2961\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2962\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2963\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2964\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2965\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2966\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2967\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2968\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2969\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2970\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2971\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2972\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2973\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2974\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2975\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2976\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2977\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2978\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2979\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2980\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2981\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2982\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2983\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2984\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2985\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2986\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2987\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2988\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2989\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2990\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2991\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2992\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2994\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2996\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2998\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2999\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3000\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3001\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3002\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3003\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2152\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2153\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2154\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2155\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2156\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2157\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2158\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2159\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2160\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2161\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2162\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2163\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2164\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2165\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2166\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2167\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2168\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2169\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2170\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2171\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2173\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2174\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2175\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2176\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2178\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2179\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2180\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2181\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4156\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4295\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2227\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2228\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2229\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2230\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2231\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2232\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2233\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2234\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2235\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2236\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2237\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2238\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2239\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2240\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2241\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2242\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2243\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2244\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2245\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2246\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2247\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2249\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2250\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2251\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2252\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2254\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2255\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2256\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2257\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2305\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2307\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2309\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2311\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2315\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2317\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2319\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2321\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2324\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2326\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2328\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2330\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2332\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2334\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2336\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2338\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2342\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2343\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2344\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2345\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2347\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2348\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2349\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2350\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2353\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2354\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2355\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2356\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2361\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2362\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2363\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2364\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2190\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2191\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2193\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2194\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2201\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2202\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2203\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2204\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2205\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2208\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2209\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2210\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2211\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2212\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2213\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2214\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2215\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2217\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2218\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2219\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2259\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2260\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2261\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2262\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2263\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2264\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2265\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2266\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2267\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2268\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2269\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2270\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2281\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2282\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2284\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2286\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2287\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2292\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2293\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2294\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2295\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2296\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2359\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2365\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2288\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2289\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0709\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2150\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2151\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2172\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2177\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2182\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2248\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2253\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2258\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2440\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2441\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2455\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2456\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2457\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2458\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2459\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2460\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2463\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2464\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2465\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2466\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2467\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2468\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2469\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2470\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6398\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6400\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6401\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6402\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6403\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2472\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2475\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2478\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2480\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2482\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2484\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2486\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2489\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2491\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2493\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2497\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2499\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2501\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2503\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2505\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2510\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2511\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2512\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2513\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2515\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2516\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2518\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2520\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2522\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2524\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2526\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2528\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2530\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2533\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2535\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2537\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2540\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2541\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2542\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2543\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2544\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2546\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2548\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2549\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2551\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2553\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2555\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2556\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2557\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2560\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2561\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2562\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2563\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2570\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2571\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2572\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2573\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2574\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2576\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2577\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2578\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2579\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2580\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2582\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2584\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2368\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2371\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2374\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2376\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2378\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2380\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2383\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2387\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2389\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2395\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2397\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2399\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2401\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2403\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2408\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2409\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2410\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2411\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2415\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2417\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2419\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2421\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2423\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2426\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2428\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2430\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2433\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2434\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2435\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2436\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3444\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3446\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3447\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3448\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3449\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3450\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3451\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3452\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3453\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3454\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3455\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3456\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3457\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3458\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3459\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3461\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3462\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3463\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3464\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3465\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3466\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3467\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3468\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3469\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3470\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3471\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3472\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0931\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0932\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0933\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0934\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0935\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0936\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0937\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0938\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0939\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0940\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0942\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0943\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0944\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0945\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0946\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0947\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0948\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0949\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0950\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0951\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0953\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0954\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0955\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0956\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0957\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0958\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0960\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0963\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0964\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0969\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0973\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0976\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0979\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0980\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0981\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0991\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0992\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0993\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0994\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1002\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1003\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1004\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1006\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1007\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1008\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1020\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1021\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1022\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1023\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1025\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1026\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1027\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1028\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1029\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1033\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1037\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1039\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1043\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1048\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1049\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1050\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1054\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1055\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1057\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1058\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1059\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1061\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1065\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1066\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1067\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1068\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1069\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1070\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1071\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1072\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1074\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3015\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1080\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1081\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1084\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1085\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1088\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1091\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1098\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1100\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1101\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1102\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1103\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1105\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1106\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1107\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1108\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1109\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1110\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1111\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1112\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1116\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1117\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1119\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1120\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1121\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1122\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1123\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1124\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1125\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1126\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1127\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1128\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1129\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1130\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1131\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1132\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1133\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1134\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1137\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1138\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1139\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1140\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1146\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1147\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1148\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1149\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1150\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1151\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1152\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1153\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1154\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1155\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1156\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1157\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1158\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1159\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1160\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1161\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1162\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1163\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1164\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1165\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1166\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1167\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1168\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1170\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1171\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1186\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1190\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1191\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1192\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1193\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1194\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1195\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1196\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1200\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1201\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1202\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1203\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1204\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1205\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1206\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1207\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1208\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1209\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1210\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1211\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1212\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1213\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1228\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1229\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1230\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1231\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1232\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1233\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1234\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1235\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1236\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1237\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1238\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1239\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1241\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1244\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1245\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1246\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1248\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1249\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1250\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1251\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1253\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1254\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1255\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1256\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1257\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1258\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1259\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1260\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1261\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1262\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1263\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1264\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1265\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1266\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1270\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1271\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1272\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1273\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1274\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1275\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1276\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1277\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1278\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1279\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1280\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1281\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1282\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1287\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1288\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1289\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1290\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1291\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1292\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1293\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1294\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1295\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1296\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1297\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1298\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1299\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1300\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1301\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1303\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1304\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4796\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8546\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8548\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8549\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8556\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8560\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0154\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0189\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0193\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0197\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0201\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0206\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0210\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0214\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0223\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0230\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0234\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0238\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0242\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0246\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0250\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0256\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0260\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0264\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0268\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0272\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0276\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0280\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0284\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0290\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0294\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0298\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0302\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0306\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0310\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0314\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0320\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0324\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0328\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0332\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0338\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0342\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0346\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0350\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0354\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0358\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0362\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0366\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0370\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0374\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0378\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0382\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0386\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0390\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0394\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0398\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0402\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0406\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0410\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0414\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0418\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0422\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0426\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0430\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0434\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0438\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3475\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0159\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2591\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2594\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2599\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2602\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2605\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2608\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2611\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2614\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2620\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2623\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2626\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2633\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2636\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2642\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2648\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2651\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2654\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2657\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2660\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2663\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2666\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2669\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2672\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2675\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2678\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2681\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2684\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2687\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2690\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2693\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2699\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2702\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2705\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2708\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2711\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2714\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2717\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2720\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2726\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2730\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2733\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2736\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2739\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2742\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2746\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2752\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2755\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2759\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2762\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2768\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2771\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2774\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2877\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3032\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3521\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0160\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0161\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0162\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0163\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2592\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2593\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2596\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2598\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2600\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2601\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2603\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2604\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2606\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2607\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2609\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2610\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2612\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2613\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2616\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2618\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2621\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2622\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2624\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2625\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2629\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2630\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2634\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2635\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2638\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2640\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2644\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2646\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2649\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2650\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2652\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2653\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2655\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2656\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2658\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2659\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2661\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2662\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2664\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2665\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2667\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2668\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2670\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2671\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2673\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2674\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2676\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2677\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2679\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2680\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2682\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2683\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2685\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2686\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2688\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2689\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2691\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2692\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2695\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2697\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2700\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2701\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2703\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2704\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2706\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2707\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2709\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2710\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2712\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2713\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2715\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2716\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2718\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2719\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2722\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2724\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2727\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2729\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2731\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2732\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2734\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2735\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2737\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2738\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2740\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2741\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2744\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2745\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2748\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2750\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2753\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2754\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2757\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2758\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2760\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2761\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2764\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2766\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2769\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2770\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2772\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2773\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2775\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2776\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3519\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3520\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3027\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3028\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3030\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3033\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3034\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3035\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3037\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2787\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2788\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2789\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2790\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2791\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2792\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2793\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2794\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2795\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2796\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2797\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2798\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2799\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2800\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2801\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2803\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2805\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2806\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2807\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2809\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2811\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2812\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2813\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2814\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2815\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2816\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2817\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2819\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2821\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2822\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2823\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2824\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2825\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2827\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2829\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2830\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2831\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2832\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2833\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2834\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2835\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2836\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2837\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2838\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2839\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2840\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2841\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2842\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2843\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2844\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2845\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2846\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2847\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2848\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2849\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2850\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2851\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2852\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2853\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2854\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2855\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2856\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2857\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2859\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2861\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2862\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2863\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2864\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2865\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2866\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2867\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2868\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2869\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2870\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2871\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2872\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2873\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2874\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2875\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2876\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2878\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2879\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2880\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2882\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2884\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2886\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2888\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2890\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2892\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2894\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2896\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2897\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2898\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2899\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2900\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2901\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2902\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2903\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2904\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2905\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2906\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2907\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2908\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2909\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2910\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2911\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2912\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2913\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2914\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2915\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2916\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2917\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2918\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2919\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2920\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2921\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2922\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2923\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2924\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2925\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2926\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2927\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2928\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2929\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2930\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2931\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2932\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0001 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0002 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0003 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0005 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0604\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0605\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0665\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0667\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0668\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0669\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0670\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0671\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0672\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0679\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0680\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0681\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0682\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0683\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0684\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7588\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3053\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3057\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3058\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3059\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3060\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3062\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3063\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3064\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3065\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3066\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3067\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3068\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3069\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3070\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3071\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3072\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3073\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3074\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3075\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3076\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3077\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3078\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3079\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3080\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3081\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3082\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3083\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3084\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3085\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3086\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3087\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3088\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3089\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3090\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3091\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3092\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3093\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3094\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3095\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3096\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3097\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3098\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3099\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3100\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3101\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3102\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3103\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3104\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3326\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3327\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3328\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3329\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3330\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3331\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3332\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3333\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3334\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3335\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3336\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3337\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3338\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3339\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3340\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3341\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3342\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3343\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3344\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3345\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3346\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3347\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3348\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3349\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3350\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3351\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3352\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3353\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3355\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3364\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3365\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3367\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3368\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3369\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3370\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3372\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3373\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3375\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3480\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3481\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3482\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3484\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3486\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3488\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3489\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3498\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3501\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3503\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3505\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3506\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3508\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3509\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3510\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3511\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3512\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3513\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3514\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3515\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3517\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3301\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3302\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3304\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3305\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3306\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3307\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3309\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3310\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3311\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3312\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3314\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3315\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3316\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3317\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3319\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3320\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3321\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3322\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3323\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3107\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3108\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3109\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3110\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3111\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3112\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3113\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3114\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3115\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3116\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3117\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3118\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3121\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3122\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3123\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3125\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3128\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3129\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3130\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3132\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3135\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3136\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3137\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3139\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3142\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3143\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3144\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3146\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3149\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3150\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3151\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3153\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3156\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3157\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3158\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3160\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3398\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3406\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3407\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3408\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3409\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3411\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3413\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3414\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3416\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3421\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3422\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3423\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3424\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3425\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3426\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9719\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3427\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3428\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3429\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3430\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3431\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3432\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3433\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3170\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3171\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3172\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3173\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3174\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3175\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3176\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3177\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3178\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3179\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3180\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3181\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3182\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3183\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3184\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3185\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3186\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3187\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3188\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3189\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3190\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3191\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3192\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3193\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3194\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3195\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3196\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3197\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3198\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3199\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3200\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3201\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3202\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3203\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3204\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3205\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3396\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3240\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3241\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3242\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3243\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3244\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3245\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3246\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3247\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3250\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3252\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3254\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3256\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3258\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3260\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3262\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3264\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3272\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3356\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3357\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3358\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3359\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3360\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3361\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3362\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3363\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3218\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3219\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3220\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3221\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3222\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3223\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3224\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3225\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3226\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3227\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3228\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3229\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3230\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3231\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3232\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3233\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3234\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3235\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3236\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3237\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3239\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3275\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3277\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3278\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3279\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3280\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3281\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3282\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3283\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3284\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3285\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3286\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3287\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3288\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3290\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3292\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3293\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3294\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3296\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3298\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1174\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1175\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1176\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1177\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1178\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1179\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1180\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1181\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1182\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1183\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1184\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1216\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1217\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1218\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1219\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1220\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1221\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1222\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1223\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1224\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1225\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1226\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3522\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3523\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3524\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3525\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3526\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3527\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3528\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3529\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3530\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3531\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3532\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3533\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3534\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1573\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6901\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6903\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6904\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6905\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6906\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6907\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6908\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6909\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6910\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7165\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7166\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7167\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7168\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7169\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7170\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1644\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1651\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1920\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1924\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1926\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1927\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1940\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1948\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1990\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1991\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1992\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2002\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2003\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2004\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2005\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2007\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4766\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7648\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7930\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7935\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7940\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7941\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7943\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7945\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7948\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7950\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7952\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7955\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7958\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7959\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7962\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7965\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7968\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2041\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7970\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7971\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7974\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7976\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7978\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7980\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7984\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7987\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1944\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1952\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1953\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1962\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1963\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1973\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1974\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1976\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1977\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2014\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2015\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2016\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2017\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2018\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2019\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2025\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1440\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1445\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1448\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1454\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1465\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1467\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1470\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1473\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1477\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1478\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1479\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1484\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1490\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1493\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1494\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1495\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1496\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1500\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1502\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1503\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1504\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1505\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1509\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1512\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1516\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1519\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1526\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1533\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1557\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1558\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1565\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1535\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1536\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1538\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1539\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1540\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1543\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1544\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1545\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1546\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1547\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1548\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1549\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1550\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1551\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1552\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1553\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1437\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1451\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1457\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1460\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1531\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1570\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1576\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1577\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3105\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4630\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2029\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2030\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2031\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2032\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2033\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2034\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2036\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2038\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2042\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2043\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2044\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2045\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2048\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2052\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2053\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2054\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2058\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2059\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2060\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2061\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2063\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2068\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2069\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2070\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2071\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2072\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2073\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2074\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2075\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2076\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2078\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2083\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2084\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2085\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2086\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2088\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2089\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2095\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2096\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2097\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2098\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2099\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2100\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2106\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2107\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2108\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2109\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2110\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2111\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2112\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2113\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3537\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3540\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3545\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3546\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3551\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3552\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3574\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3577\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3581\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3586\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3588\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3589\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3625\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3626\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3627\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3628\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3629\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3630\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3631\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3632\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3633\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3634\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3635\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3636\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3637\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3638\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3639\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3640\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3641\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3642\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3643\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3644\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3645\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3646\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3647\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3648\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3649\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3650\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3651\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3652\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3653\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3654\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3655\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3656\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3657\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3658\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3659\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3660\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3661\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3662\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3663\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3664\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3665\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3666\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3667\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3668\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3669\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3670\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3671\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3672\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3673\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3674\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3675\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3676\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3677\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3678\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3679\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3680\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3681\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3682\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3683\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3684\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3685\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3686\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3687\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3688\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3689\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3690\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3691\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3692\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3693\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3694\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3695\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3696\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3697\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3698\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3699\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3700\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3701\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3703\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3704\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3705\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3706\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3707\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3708\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3709\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3710\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3711\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3712\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3713\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3714\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3715\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3716\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3717\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3718\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3719\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3720\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3721\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3722\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3723\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3724\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3725\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3726\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3727\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3728\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3729\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3732\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3733\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3734\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3735\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3736\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3737\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3738\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3739\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3740\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3741\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0715\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0716\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0735\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0736\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0738\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0741\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0744\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0746\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0748\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0750\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0753\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0754\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0758\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0760\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0761\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0762\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0765\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0766\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0767\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0773\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0775\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0779\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0781\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0783\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0795\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8147\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8148\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8149\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8150\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8151\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8152\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8155\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8165\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8166\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8167\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8168\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8172\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8173\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8175\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8178\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8179\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8180\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8184\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8185\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8186\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8188\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8189\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8190\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8191\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8192\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8194\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8198\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8201\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0786\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8206\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8209\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8211\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8212\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8217\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8219\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8220\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8224\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8226\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8227\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8228\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8233\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8235\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8238\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8245\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8246\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8248\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8249\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8250\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8251\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0453\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0456\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0458\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0459\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0486\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0487\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0488\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0489\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0490\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0491\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0492\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0493\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0494\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0495\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0496\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0497\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0498\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0499\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0500\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0501\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0502\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0503\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0504\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0505\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0506\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0507\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0508\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0509\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0510\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0511\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0512\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0513\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0514\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0515\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0516\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0517\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0518\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0519\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0520\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0521\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0522\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0523\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0524\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0525\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0526\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0527\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0528\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0529\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0530\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0531\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0532\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0533\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0534\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0535\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0536\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0537\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0538\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0539\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0540\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0541\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0542\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0543\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0544\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0588\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0596\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0673\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0674\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0675\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0676\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0677\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0678\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4296\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4390\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5254\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7599\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0448\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0450\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0468\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0478\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0479\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0481\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0579\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0580\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0581\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0582\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0584\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0586\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0589\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0590\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0591\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0592\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0593\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0594\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0597\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0598\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0599\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0600\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0601\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0602\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0607\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0610\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0613\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0614\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0615\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0616\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0622\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0623\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0625\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0627\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0629\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0630\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0632\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0633\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0634\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0635\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0636\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0638\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0642\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0643\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0644\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0645\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0646\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0648\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0651\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0653\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0654\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0657\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0660\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4627\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4838\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4839\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7591\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7594\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7597\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7601\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7611\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7613\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7755\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8518\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8523\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8525\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0815\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0816\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0817\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0819\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0820\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0821\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0823\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0824\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0825\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0827\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0828\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0830\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0837\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0838\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0841\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0842\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0844\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0845\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0847\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0848\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0849\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0850\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0855\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0856\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0803\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0807\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0809\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0810\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0811\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0812\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0859\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0873\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0875\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0876\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0889\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0890\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0891\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0892\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0893\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0894\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0895\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0898\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0899\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0900\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0906\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0910\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0911\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0787\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0790\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0792\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0794\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0797\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0804\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0808\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0826\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0832\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0835\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0836\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0920\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0924\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0925\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0926\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8379\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7185\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8380\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7186\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8381\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7187\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8383\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8384\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8385\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8387\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8388\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8389\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8390\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8391\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8392\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8393\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8394\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8395\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8396\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8397\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8398\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8399\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8401\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8402\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8403\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8404\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8405\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8406\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8407\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1981\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1985\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1988\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1312\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_1313 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1321\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1325\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1326\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1327\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1329\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_1339 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1347\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1367\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1370\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1379\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1382\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1384\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1387\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1394\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1395\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1398\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0703\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7598\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7600\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7602\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7603\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7604\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7614\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7615\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0025 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0030 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7200\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7202\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7203\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7205\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7206\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7207\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7208\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7209\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7210\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7211\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7212\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7213\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7214\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7215\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7216\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7217\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7218\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7219\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7491\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7492\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7493\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7494\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7495\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7496\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7497\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7498\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7509\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7519\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7520\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7521\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7522\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7534\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7535\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7536\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7537\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7551\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7552\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7553\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7501\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7502\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7503\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7504\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7505\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7506\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7507\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7513\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7514\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7515\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7516\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7517\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7525\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7526\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7527\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7528\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7529\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7530\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7531\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7532\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7541\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7542\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7543\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7544\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7545\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7546\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7547\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7548\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7549\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7557\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7558\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7559\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7560\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7561\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7562\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7563\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7564\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7565\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7566\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7225\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7226\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7227\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7228\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7229\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7230\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7231\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7232\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7233\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7234\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7235\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7236\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7237\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7238\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7239\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7240\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7241\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7242\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7243\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7244\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7245\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7246\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7247\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7248\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7249\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7250\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7571\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7572\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7573\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7335\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7336\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7337\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7338\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7339\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7340\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7341\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7342\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7343\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7344\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7345\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7346\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7347\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7348\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7349\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7350\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7351\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7352\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7353\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7354\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7355\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7356\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7357\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7358\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7359\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7360\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7361\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7362\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7363\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7364\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7365\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7366\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7367\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7368\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7369\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7370\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7371\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7442\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7443\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7444\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7445\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7446\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7447\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7448\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7449\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7450\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7451\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7452\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7471\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7472\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7473\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7474\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7475\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7476\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7477\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7478\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7479\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7480\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7481\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7373\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7375\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7376\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7377\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7378\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7379\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7380\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7381\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7382\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7383\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7384\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7385\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7386\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7387\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7388\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7389\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7390\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7391\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7392\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7393\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7394\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7395\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7396\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7397\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7398\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7399\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7400\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7401\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7402\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7403\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7404\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7405\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7406\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7407\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7408\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7409\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7410\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7411\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7412\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7413\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7414\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7415\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7416\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7417\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7418\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7419\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7420\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7421\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7422\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7423\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7424\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7425\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7426\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7427\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7455\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7456\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7457\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7458\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7459\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7460\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7461\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7462\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7463\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7464\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7465\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7466\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7467\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7468\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7469\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7484\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7485\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7486\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7487\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7488\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7489\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1581\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1584\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1585\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1590\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1593\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1595\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1598\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1599\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1604\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1605\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1608\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1609\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1610\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1611\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1613\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1614\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1619\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1620\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1623\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1624\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1625\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1627\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1630\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1632\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1635\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1637\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1638\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1639\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1642\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1646\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1652\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1653\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1659\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1660\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1662\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1663\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1665\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1666\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1673\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1676\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1678\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1679\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1681\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1682\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1684\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1685\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1687\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1689\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1691\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1692\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1693\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1694\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1695\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1696\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1697\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1699\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1700\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1703\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1704\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1706\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1708\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1710\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1718\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1726\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1727\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1729\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1730\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1735\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1737\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1738\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1739\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1740\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1741\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1742\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1743\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1744\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1745\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1746\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1747\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1748\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1749\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1750\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1751\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1752\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1753\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1754\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1756\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1758\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1759\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1760\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1761\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1762\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1766\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1767\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1768\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1769\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1770\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1771\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1784\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1786\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1790\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1792\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1794\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1796\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1797\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1798\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1800\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1805\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1806\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1807\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1810\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1811\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1813\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1832\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1833\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1846\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3799\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3908\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3910\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4679\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8065\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1847\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1849\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1852\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1853\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1859\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1868\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1870\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1872\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1874\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1875\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1876\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1877\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1878\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1879\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1880\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1881\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1882\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1883\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1884\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1885\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1886\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1887\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1888\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1889\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1890\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1891\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1892\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1893\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1894\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1895\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1896\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1897\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8262\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8263\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8264\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8267\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8268\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8271\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8274\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8275\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8276\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8277\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8278\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8279\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8280\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8284\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8285\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8318\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8325\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0712\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4228\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4251\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4252\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4254\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4255\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4257\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4260\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4261\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4262\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4263\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4264\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4265\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4269\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4270\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4271\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4276\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4278\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4662\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7623\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7625\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8790\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8791\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3871\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4058\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4498\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4500\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4714\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4715\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4716\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4717\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4718\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4723\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4725\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4727\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4730\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4731\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4732\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4733\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4734\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7731\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0663\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4308\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6539\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6540\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6542\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6543\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6544\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6545\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6546\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6547\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6548\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6549\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6550\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6551\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6552\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6553\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6554\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6555\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6556\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6557\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6558\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6559\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6560\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6561\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6562\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6563\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6564\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6565\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6566\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6567\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6568\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6569\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6570\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6571\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6572\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6573\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6574\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6575\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6576\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6579\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6581\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6583\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6585\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6587\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6591\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6592\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6595\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7652\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7654\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7655\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7656\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8799\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8800\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8803\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8804\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8805\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8807\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8813\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8815\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8816\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8818\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8821\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8822\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8823\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8824\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8830\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8832\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8833\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8835\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8836\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3923\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3925\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3929\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4332\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4333\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4335\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4336\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4338\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4340\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4440\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4442\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4444\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4503\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4505\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4654\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4655\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4656\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4665\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9726 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4666\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7145\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7146\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7147\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7908\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7909\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7910\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7911\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7912\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7913\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7916\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7919\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7920\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7921\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7922\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7925\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8105\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8106\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8107\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8108\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8109\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8110\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8112\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8113\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8114\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8115\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8116\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8117\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8129\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8130\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8132\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8133\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8135\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8136\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8137\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8138\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8139\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8140\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8141\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8142\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8143\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8144\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8758\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7661\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7662\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7663\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7668\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7669\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7670\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7671\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7672\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7673\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4160\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4162\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4163\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4165\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4166\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4167\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4169\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4170\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4523\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4539\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4540\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4541\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4542\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4543\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4544\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4816\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4817\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4818\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4833\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4834\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4835\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4836\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8538\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8738\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8739\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6396\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3991\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4657\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4744\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4746\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4748\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4750\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4752\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4755\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4757\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4758\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4762\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4771\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4772\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6630\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6631\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6632\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6633\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6634\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6635\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6636\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6638\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6639\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6641\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6642\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6643\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6644\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6645\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6646\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6647\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6648\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6649\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6650\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6651\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6652\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6653\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6654\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6655\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6656\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6657\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6658\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6659\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6664\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6665\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6668\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6669\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6671\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6674\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6675\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6676\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6677\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6678\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6679\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6680\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6682\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6687\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6690\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6691\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6692\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6693\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6700\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6701\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6702\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6703\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8698\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8709\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8710\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8711\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6506\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6507\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6508\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6509\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6511\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4537\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4545\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4548\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4558\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4559\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4560\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4204\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4206\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4207\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4208\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8613\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8615\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8616\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4064\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4065\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4066\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4067\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4068\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4069\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4070\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4071\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8102\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8724\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8725\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2114\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2117\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2128\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2129\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2130\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2131\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2132\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2133\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2134\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2135\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2136\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2137\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2138\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2139\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2140\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2142\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2143\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2144\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2145\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7996\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7999\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_8008 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6426\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6427\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6428\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6429\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6432\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6433\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6434\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6435\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6436\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6441\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6442\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6443\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6444\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6447\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6448\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6450\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6451\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6453\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6464\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6465\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6466\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6467\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6470\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6471\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6472\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6473\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6476\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6477\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6478\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6479\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6482\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6484\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6486\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6490\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6492\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6495\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6496\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6992\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6993\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6994\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6995\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6996\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6997\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6998\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6999\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7000\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7001\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7002\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7003\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7004\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7005\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7013\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7014\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7017\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7019\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7021\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7022\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7023\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7024\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7025\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7026\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7027\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7028\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7029\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7030\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7033\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7034\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7035\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7036\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7038\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7039\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7040\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7041\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7042\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7048\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7050\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7052\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7053\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7054\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7055\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7056\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7057\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7058\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7059\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7060\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7061\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7065\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7066\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7078\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7079\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7080\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7081\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7082\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7083\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7084\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7088\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7089\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7099\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7100\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7688\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8059\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6535\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6536\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9569\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3955\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3984\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4201\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4202\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5406\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8751\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8752\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9488 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9489 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9493 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9494 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9516 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9517 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9577 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9578 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9581 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9582 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9583 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9584 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0010 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0014 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0031\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0032\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0545\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0546\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0547\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0548\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0549\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0550\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0551\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0553\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0556\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0559\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0560\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0561\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0689\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5233\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5238\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5247\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0004 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0006 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0007 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0008 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0009 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0015\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0018 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0019 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0155\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0164\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0167\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0170\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0173\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0176\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0179\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0183\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0187\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0190\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0191\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0194\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0195\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0198\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0199\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0202\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0203\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0207\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0208\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0211\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0212\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0215\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0216\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0224\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0225\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0231\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0232\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0235\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0236\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0239\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0240\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0243\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0244\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0247\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0248\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0253\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0254\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0257\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0258\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0261\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0262\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0265\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0266\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0269\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0270\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0273\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0274\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0277\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0278\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0281\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0282\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0287\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0288\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0291\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0292\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0295\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0296\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0299\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0300\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0303\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0304\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0307\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0308\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0311\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0312\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0317\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0318\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0321\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0322\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0325\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0326\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0329\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0330\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0335\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0336\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0339\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0340\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0343\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0344\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0347\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0348\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0351\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0352\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0355\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0356\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0359\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0360\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0363\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0364\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0367\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0368\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0371\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0372\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0375\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0376\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0379\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0380\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0383\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0384\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0387\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0388\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0391\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0392\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0395\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0396\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0399\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0400\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0403\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0404\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0407\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0408\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0411\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0412\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0415\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0416\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0419\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0420\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0423\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0424\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0427\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0428\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0431\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0432\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0435\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0436\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0439\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0440\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0444\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0470\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0476\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0770\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1082\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1086\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1898\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1899\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1900\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1901\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1902\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1903\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1904\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1905\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1906\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1907\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1908\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1909\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1911\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1914\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1919\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3858\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3959\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4249\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4321\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4691\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4721\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4844\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4849\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4858\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4861\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4873\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4885\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4896\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4910\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4911\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4912\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4919\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4928\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4931\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4932\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4933\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4935\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4938\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4946\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4947\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4954\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4956\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4973\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4975\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4976\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4994\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4996\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4999\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5000\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5003\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5008\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5010\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5018\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5021\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5023\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5029\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5032\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5034\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5035\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5037\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5042\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5068\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5070\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5071\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5073\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5074\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5076\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5077\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5078\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5079\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5080\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5082\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5084\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5085\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5087\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5088\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5089\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5091\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5092\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5094\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5198\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5232\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5237\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5241\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5246\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5295\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5296\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5303\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5305\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5307\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5308\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5310\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5311\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5313\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5314\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5315\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5316\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5317\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5318\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5319\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5320\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5322\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5324\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5326\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5328\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5330\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5333\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5413\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5429\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5430\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5432\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5440\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5442\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5443\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5444\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5445\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5446\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5447\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5450\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5451\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5454\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5455\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5458\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5459\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5460\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5461\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5462\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5463\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5464\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5465\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5466\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5467\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5468\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5469\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5470\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5471\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5472\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5473\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5474\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5475\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5476\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5477\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5478\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5479\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5480\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5481\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5482\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5483\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5484\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5485\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5486\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5487\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5488\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5489\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5490\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5491\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5492\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5493\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5494\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5495\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5496\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5497\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5498\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5499\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5500\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5501\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5502\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5503\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5504\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5505\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5506\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5507\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5508\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5509\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5510\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5511\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5512\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5513\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5514\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5515\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5516\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5517\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5518\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5519\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5520\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5521\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5522\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5523\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5524\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5525\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5526\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5527\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5528\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5529\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5530\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5531\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5532\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5533\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5534\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5535\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5536\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5537\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5538\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5539\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5540\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5541\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5542\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5543\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5544\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5545\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5546\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5547\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5548\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5549\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5550\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5551\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5552\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5553\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5554\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5555\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5556\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5557\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5558\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5559\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5560\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5561\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5562\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5563\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5564\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5565\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5566\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5567\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5568\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5569\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5570\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5571\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5572\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5573\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5574\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5575\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5576\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5577\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5578\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5579\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5580\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5581\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5582\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5583\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5584\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5585\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5586\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5587\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5588\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5589\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5590\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5591\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5592\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5593\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5594\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5595\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5596\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5597\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5598\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5599\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5600\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5601\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5602\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5603\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5604\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5605\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5606\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5607\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5608\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5609\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5610\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5611\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5612\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5613\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5614\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5615\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5616\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5617\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5618\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5619\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5620\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5621\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5622\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5623\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5624\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5625\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5626\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5627\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5628\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5629\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5630\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5631\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5632\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5633\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5634\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5635\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5636\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5637\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5638\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5639\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5640\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5641\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5642\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5643\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5644\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5645\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5646\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5647\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5648\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5649\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5650\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5651\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5652\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5653\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5654\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5655\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5656\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5657\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5658\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5659\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5660\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5661\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5662\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5663\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5664\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5665\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5666\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5667\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5668\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5669\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5670\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5671\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5672\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5673\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5674\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5675\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5676\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5677\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5678\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5679\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5680\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5681\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5682\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5683\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5684\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5685\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5686\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5687\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5688\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5689\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5690\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5691\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5692\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5693\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5694\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5695\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5696\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5697\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5698\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5699\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5700\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5701\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5702\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5703\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5704\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5705\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5706\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5707\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5708\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5709\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5710\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5711\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5712\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5713\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5714\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5715\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5716\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5717\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5718\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5719\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5720\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5721\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5722\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5723\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5724\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5725\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5726\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5727\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5728\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5729\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5730\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5731\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5732\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5733\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5734\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5735\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5736\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5737\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5738\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5739\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5740\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5741\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5742\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5743\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5744\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5745\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5746\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5747\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5748\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5749\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5750\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5751\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5752\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5753\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5754\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5755\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5756\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5757\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5758\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5759\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5760\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5761\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5762\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5763\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5764\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5765\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5766\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5767\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5768\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5769\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5770\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5771\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5772\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5773\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5774\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5775\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5776\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5777\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5778\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5779\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5780\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5781\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5782\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5783\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5784\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5785\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5786\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5787\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5788\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5789\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5790\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5791\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5792\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5793\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5794\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5795\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5796\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5797\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5798\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5799\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5800\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5801\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5802\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5803\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5804\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5805\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5806\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5807\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5808\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5809\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5810\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5811\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5812\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5813\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5814\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5815\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5816\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5817\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5818\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5819\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5820\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5821\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5822\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5823\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5824\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5825\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5826\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5827\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5828\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5829\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5830\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5831\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5832\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5833\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5834\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5835\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5836\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5837\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5838\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5839\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5840\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5841\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5842\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5843\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5844\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5845\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5846\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5847\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5848\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5849\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5850\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5851\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5852\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5853\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5854\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5855\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5856\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5857\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5858\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5859\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5860\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5861\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5862\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5863\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5864\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5865\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5866\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5867\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5868\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5869\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5870\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5871\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5872\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5873\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5874\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5875\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5876\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5877\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5878\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5879\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5880\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5881\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5882\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5883\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5884\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5885\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5886\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5887\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5888\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5889\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5890\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5891\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5892\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5893\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5894\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5895\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5896\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5897\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5898\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5899\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5900\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5901\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5902\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5903\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5904\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5905\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5906\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5907\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5908\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5909\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5910\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5911\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5912\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5913\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5914\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5915\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5916\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5917\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5918\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5919\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5920\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5921\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5922\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5923\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5924\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5925\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5926\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5927\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5928\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5929\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5930\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5931\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5932\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5933\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5934\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5935\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5936\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5937\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5938\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5939\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5940\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5941\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5942\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5943\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5944\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5945\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5946\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5947\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5948\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5949\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5950\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5951\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5952\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5953\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5954\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5955\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5956\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5957\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5958\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5959\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5960\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5961\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5962\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5963\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5964\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5965\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5966\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5967\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5968\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5969\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5970\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5971\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5972\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5973\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5974\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5975\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5976\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5977\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5978\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5979\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5980\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5981\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5982\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5983\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5984\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5985\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5986\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5987\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5989\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5990\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5992\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5994\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5996\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5997\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5998\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6000\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6001\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6002\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6003\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6004\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6005\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6006\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6007\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6008\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6009\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6010\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6011\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6012\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6013\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6014\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6015\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6016\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6017\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6018\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6019\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6020\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6021\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6022\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6023\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6024\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6025\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6026\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6027\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6028\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6029\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6030\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6031\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6032\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6033\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6034\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6035\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6036\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6037\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6038\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6039\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6040\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6041\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6042\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6043\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6044\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6045\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6046\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6047\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6048\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6049\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6050\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6051\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6052\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6053\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6054\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6055\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_6058 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6059\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6091\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6096\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6098\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6101\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6129\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6190\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6241\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6243\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6244\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6245\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6246\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6247\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6248\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6249\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6250\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6254\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6257\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6258\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6275\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6366\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6367\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6368\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6369\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6370\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6371\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6372\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6373\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6374\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6375\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6377\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6379\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6380\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6381\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6382\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6383\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6384\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6392\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6406\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6424\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6439\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6462\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6474\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6505\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6516\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6525\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6526\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6527\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6530\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6531\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6532\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6533\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6534\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6732\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7109\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7111\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7113\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7115\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7117\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7119\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7121\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7123\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7125\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7127\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7633\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7634\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7635\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7636\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7637\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7639\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7643\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7644\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7645\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7650\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7651\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7666\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7667\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7682\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7683\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7684\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7685\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7686\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7687\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7690\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7691\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7692\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7735\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7787\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7896\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7898\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7900\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7902\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7904\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7905\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7951\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7964\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7967\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7982\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7986\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7990\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7994\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8057\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8075\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8076\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8077\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8081\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8225\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8340\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8354\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8364\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8374\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8408\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8429\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8567\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8586\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8612\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8629\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8633\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8658\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8670\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8676\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8677\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8678\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8687\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8733\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8734\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8735\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8736\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8760\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8846\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8847\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8850\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8867\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8872\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8876\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8878\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8889\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8901\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8911\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8912\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8917\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8925\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8929\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8930\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8934\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8935\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8939\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8940\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8941\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8942\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8943\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8944\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8945\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8946\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8947\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9173\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9176\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9178\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9183\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9184\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9185\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9190\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9191\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9192\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9197\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9198\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9590\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9591\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9592\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9593\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9594\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9595\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9596\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9597\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9598\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9599\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9600\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9601\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9602\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9603\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9604\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9605\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9606\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9607\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9608\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9609\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9611\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9613\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9616\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9619\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9623\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9624\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9643\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9731\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4312\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8022\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0158\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0446\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0578\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0587\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0962\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1145\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1187\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1243\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1572\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1602\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1925\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1936\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2116\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2141\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2590\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3762\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3825\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3863\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3864\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3954\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3971\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4237\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4692\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4720\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4729\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4738\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4743\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4754\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4756\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4760\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4843\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4851\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4852\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4854\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4855\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4862\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4864\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4865\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4867\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4868\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4870\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4871\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4872\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4898\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4905\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4914\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4922\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4926\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4940\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4944\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4964\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4971\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4972\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4977\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4997\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5006\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5015\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5016\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5031\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5033\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5043\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5046\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5096\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5099\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5101\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5105\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5107\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5109\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5118\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5122\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5225\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5227\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5292\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5304\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5348\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5349\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5411\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5420\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6276\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6277\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6286\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6287\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6288\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6289\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6290\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6291\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6292\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6293\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6294\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6295\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6296\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6297\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6298\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6301\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6302\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6307\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6308\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6313\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6314\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6317\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6318\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6321\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6323\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6324\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6327\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6328\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6330\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6331\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6332\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6333\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6334\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6335\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6336\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6337\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6338\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6339\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6340\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6341\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6342\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6343\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6391\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6431\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6438\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6446\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6454\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6469\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6481\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6521\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6522\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6618\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6620\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6890\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6902\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7638\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7719\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7758\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7769\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7804\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7806\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7808\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7810\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7812\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7814\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7815\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7816\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7818\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7820\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7822\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7824\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7825\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7826\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7827\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7828\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7829\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7830\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7831\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7832\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7833\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7834\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7835\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7836\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7837\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7838\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7839\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7840\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7841\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7842\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7843\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7844\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7845\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7846\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7847\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7848\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7849\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7850\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7851\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7852\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7853\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7854\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7995\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8009\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8356\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8365\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8411\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8438\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8505\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8510\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8513\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8562\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8614\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8627\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8657\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8741\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8742\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8776\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8785\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0713\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1104\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1144\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1188\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1242\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1286\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2407\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2509\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2575\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3007\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3008\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3011\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3013\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3014\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3017\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3018\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3019\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3020\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3021\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3022\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3023\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3024\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3025\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3473\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3850\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4393\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4468\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4930\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5452\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6928\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6986\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7590\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7596\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7640\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7646\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7707\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7788\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8016\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8414\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8756\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9678\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2564\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4274\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4277\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4535\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6617\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6619\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7653\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7664\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7765\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7767\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7792\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7796\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8024\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8028\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8082\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8412\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8476\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8639\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8789\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8828\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8840\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8844\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8638\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0791\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0831\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0929\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1917\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3987\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4907\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5047\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5048\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5049\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5050\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5051\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5052\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5053\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5054\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5055\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5056\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5057\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5058\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5059\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5060\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5061\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5062\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5063\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5064\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5065\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5066\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5067\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5197\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5224\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5226\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5228\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7196\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7224\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7374\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7430\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7454\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7483\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7500\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7512\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7524\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7540\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7556\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7570\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7581\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7584\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7711\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7714\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7715\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7718\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7720\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7722\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7724\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7726\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7727\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7729\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7730\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7762\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7799\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7924\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8125\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8127\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8200\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8208\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8216\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8590\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8753\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8858\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8887\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8908\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0612\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3597\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5251\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5252\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7198\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7284\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0731\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0734\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0918\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1916\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4125\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7201\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7204\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7433\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7434\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7435\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7675\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7693\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7733\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7740\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7743\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8084\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8086\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8146\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8193\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8199\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8205\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8207\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8496\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8661\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7432\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7307\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1809\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0461\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0467\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0664\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0730\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1035\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1269\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1311\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1672\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2554\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4765\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4856\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4860\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4878\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4880\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4881\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4894\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4906\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4942\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_5027\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6385\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6386\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6388\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6425\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6440\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6463\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6475\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6629\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_6797\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7262\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7272\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7273\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7649\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7789\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7944\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7947\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7949\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7963\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7966\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7981\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7983\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7985\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7989\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8058\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8234\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8239\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8594\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8851\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_8915\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9177\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9732\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9725\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction cofactors_vitamins\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction vitaminA\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction vitaminD\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction vitaminE\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4953\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3106\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_3947\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0685\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_0686\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_2393\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_4783\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7777\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7779\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_1095\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_0020 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7108\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7110\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7112\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7114\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7116\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7118\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7120\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7122\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7124\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_7126\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9024 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9025 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9027 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9028\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9029\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9030\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9031\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9808 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9809 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9810 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9811 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9812 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9813 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9814 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9815 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9816 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9033\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9034\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9035\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9036\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9037\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9038\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9039\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9040\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9041\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9042\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9043\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9044\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9045\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9046\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9047\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9048\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9049\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9051\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9052 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9053 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9054 might not have a counterpart in origin_model.\n",
      "\n",
      "ERROR -- KeyError, The reaction from target model HMR_9055 might not have a counterpart in origin_model.\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9058\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9061\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9062\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9063\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9064\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9065\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9066\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9067\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9068\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9069\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9070\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9071\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9072\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9073\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9074\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9075\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9076\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9077\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9078\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9079\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9080\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9081\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9083\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9084\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9085\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9086\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9087\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9088\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9091\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9092\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9093\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9094\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9095\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9096\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9097\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9098\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9099\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9100\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9101\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9102\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9105\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9106\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9107\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9109\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9110\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9111\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9113\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9114\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9115\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9116\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9117\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9118\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9119\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9120\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9121\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9122\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9124\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9125\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9126\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9129\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9132\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9133\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9134\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9135\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9136\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9137\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9139\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9140\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9141\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9143\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9145\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9146\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9150\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9151\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9152\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9153\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9154\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9155\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9156\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9158\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9159\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9160\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9161\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9162\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9165\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9166\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9167\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9168\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9169\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9201\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9203\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9204\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9205\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9206\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9207\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9208\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9216\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9219\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9220\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9230\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9232\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9234\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9235\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9236\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9238\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9239\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9240\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9243\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9247\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9254\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9255\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9257\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9259\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9260\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9262\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9264\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9269\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9270\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9271\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9272\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9273\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9275\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9279\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9280\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9281\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9282\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9283\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9284\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9285\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9286\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9287\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9290\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9292\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9293\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9295\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9296\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9297\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9300\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9301\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9302\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9303\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9305\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9310\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9312\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9315\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9316\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9317\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9318\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9330\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9336\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9338\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9340\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9341\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9343\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9346\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9348\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9350\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9351\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9352\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9354\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9359\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9360\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9361\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9362\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9363\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9364\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9366\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9367\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9368\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9370\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9372\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9383\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9386\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9394\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9396\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9397\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9398\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9399\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9404\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9407\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9411\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9412\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9415\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9416\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9417\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9418\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9421\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9423\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9424\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9426\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9427\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9430\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9435\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9436\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9438\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9439\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9440\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9441\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9447\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9448\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9450\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9452\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9453\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9456\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9457\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9458\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9460\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9682\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9683\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9685\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9692\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9693\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9695\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9715\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9730\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction HMR_9736\n",
      "\n",
      "DEBUG -- Added gene_reaction_rule to reaction biomass_components\n"
     ]
    }
   ],
   "source": [
    "Hep_G2 = mc.copy_genes(iHep, Hep_G2)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "43e22fc7",
   "metadata": {},
   "source": [
    "--- \n",
    "#### Conversion des IDs ENSEMBL en IDs uniprot pour chaque gène."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "57f8033c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Creating working copy of the model...\n",
      "\n",
      "Model copied.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172955\n",
      "\n",
      "Gene ensembl id ENSG00000172955 changed to uniprot id : P28332.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000180011\n",
      "\n",
      "Gene ensembl id ENSG00000180011 changed to uniprot id : Q8N4Q0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196344\n",
      "\n",
      "Gene ensembl id ENSG00000196344 changed to uniprot id : P40394.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147576\n",
      "\n",
      "Gene ensembl id ENSG00000147576 changed to uniprot id : B4DFI7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000187758\n",
      "\n",
      "Gene ensembl id ENSG00000187758 changed to uniprot id : P07327.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196616\n",
      "\n",
      "Gene ensembl id ENSG00000196616 changed to uniprot id : P00325.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197894\n",
      "\n",
      "Gene ensembl id ENSG00000197894 changed to uniprot id : D6RAY0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000248144\n",
      "\n",
      "Gene ensembl id ENSG00000248144 changed to uniprot id : P00326.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198099\n",
      "\n",
      "Gene ensembl id ENSG00000198099 changed to uniprot id : P08319.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117448\n",
      "\n",
      "Gene ensembl id ENSG00000117448 changed to uniprot id : P14550.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131069\n",
      "\n",
      "Gene ensembl id ENSG00000131069 changed to uniprot id : Q9NR19.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111058\n",
      "\n",
      "Gene ensembl id ENSG00000111058 changed to uniprot id : Q9H6R3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154930\n",
      "\n",
      "Gene ensembl id ENSG00000154930 changed to uniprot id : Q9NUB1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168291\n",
      "\n",
      "Gene ensembl id ENSG00000168291 changed to uniprot id : P11177.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110435\n",
      "\n",
      "Gene ensembl id ENSG00000110435 changed to uniprot id : O00330.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131828\n",
      "\n",
      "Gene ensembl id ENSG00000131828 changed to uniprot id : P08559.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000091140\n",
      "\n",
      "Gene ensembl id ENSG00000091140 changed to uniprot id : P09622.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163114\n",
      "\n",
      "Gene ensembl id ENSG00000163114 changed to uniprot id : P29803.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000150768\n",
      "\n",
      "Gene ensembl id ENSG00000150768 changed to uniprot id : P10515.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166796\n",
      "\n",
      "Gene ensembl id ENSG00000166796 changed to uniprot id : P07864.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111716\n",
      "\n",
      "Gene ensembl id ENSG00000111716 changed to uniprot id : P07195.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151116\n",
      "\n",
      "Gene ensembl id ENSG00000151116 changed to uniprot id : Q8IX04.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166800\n",
      "\n",
      "Gene ensembl id ENSG00000166800 changed to uniprot id : Q6ZMR3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171989\n",
      "\n",
      "Gene ensembl id ENSG00000171989 changed to uniprot id : Q9BYZ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132746\n",
      "\n",
      "Gene ensembl id ENSG00000132746 changed to uniprot id : P48448.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184254\n",
      "\n",
      "Gene ensembl id ENSG00000184254 changed to uniprot id : P47895.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000006534\n",
      "\n",
      "Gene ensembl id ENSG00000006534 changed to uniprot id : P43353.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108602\n",
      "\n",
      "Gene ensembl id ENSG00000108602 changed to uniprot id : P30838.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143149\n",
      "\n",
      "No uniprot id found for gene ENSG00000143149.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000072210\n",
      "\n",
      "Gene ensembl id ENSG00000072210 changed to uniprot id : P51648.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137124\n",
      "\n",
      "Gene ensembl id ENSG00000137124 changed to uniprot id : P30837.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111275\n",
      "\n",
      "Gene ensembl id ENSG00000111275 changed to uniprot id : P05091.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164904\n",
      "\n",
      "Gene ensembl id ENSG00000164904 changed to uniprot id : P49419.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000067057\n",
      "\n",
      "Gene ensembl id ENSG00000067057 changed to uniprot id : Q01813.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141959\n",
      "\n",
      "Gene ensembl id ENSG00000141959 changed to uniprot id : F8WEU2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152556\n",
      "\n",
      "Gene ensembl id ENSG00000152556 changed to uniprot id : P08237.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160226\n",
      "\n",
      "Gene ensembl id ENSG00000160226 changed to uniprot id : O43822.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109107\n",
      "\n",
      "Gene ensembl id ENSG00000109107 changed to uniprot id : P09972.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149925\n",
      "\n",
      "Gene ensembl id ENSG00000149925 changed to uniprot id : P04075.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136872\n",
      "\n",
      "Gene ensembl id ENSG00000136872 changed to uniprot id : P05062.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121900\n",
      "\n",
      "Gene ensembl id ENSG00000121900 changed to uniprot id : Q969K7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000067225\n",
      "\n",
      "Gene ensembl id ENSG00000067225 changed to uniprot id : P14618.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143630\n",
      "\n",
      "Gene ensembl id ENSG00000143630 changed to uniprot id : Q9P1Z3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143627\n",
      "\n",
      "Gene ensembl id ENSG00000143627 changed to uniprot id : P30613.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000074800\n",
      "\n",
      "Gene ensembl id ENSG00000074800 changed to uniprot id : A0A024R4F1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108515\n",
      "\n",
      "Gene ensembl id ENSG00000108515 changed to uniprot id : P13929.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111674\n",
      "\n",
      "Gene ensembl id ENSG00000111674 changed to uniprot id : P09104.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172331\n",
      "\n",
      "Gene ensembl id ENSG00000172331 changed to uniprot id : A0A024R782.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171314\n",
      "\n",
      "Gene ensembl id ENSG00000171314 changed to uniprot id : P18669.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164708\n",
      "\n",
      "Gene ensembl id ENSG00000164708 changed to uniprot id : P15259.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000226784\n",
      "\n",
      "Gene ensembl id ENSG00000226784 changed to uniprot id : Q8N0Y7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154305\n",
      "\n",
      "Gene ensembl id ENSG00000154305 changed to uniprot id : Q5JRA6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102144\n",
      "\n",
      "Gene ensembl id ENSG00000102144 changed to uniprot id : P00558.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170950\n",
      "\n",
      "Gene ensembl id ENSG00000170950 changed to uniprot id : P07205.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119640\n",
      "\n",
      "Gene ensembl id ENSG00000119640 changed to uniprot id : P07311.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170634\n",
      "\n",
      "Gene ensembl id ENSG00000170634 changed to uniprot id : P14621.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105679\n",
      "\n",
      "Gene ensembl id ENSG00000105679 changed to uniprot id : O14556.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111640\n",
      "\n",
      "Gene ensembl id ENSG00000111640 changed to uniprot id : P04406.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105220\n",
      "\n",
      "Gene ensembl id ENSG00000105220 changed to uniprot id : P06744.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111669\n",
      "\n",
      "Gene ensembl id ENSG00000111669 changed to uniprot id : P60174.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000230359\n",
      "\n",
      "No uniprot id found for gene ENSG00000230359.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106633\n",
      "\n",
      "Gene ensembl id ENSG00000106633 changed to uniprot id : P35557.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156510\n",
      "\n",
      "Gene ensembl id ENSG00000156510 changed to uniprot id : Q2TB90.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156515\n",
      "\n",
      "Gene ensembl id ENSG00000156515 changed to uniprot id : P19367.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160883\n",
      "\n",
      "Gene ensembl id ENSG00000160883 changed to uniprot id : P52790.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159399\n",
      "\n",
      "Gene ensembl id ENSG00000159399 changed to uniprot id : P52789.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159322\n",
      "\n",
      "Gene ensembl id ENSG00000159322 changed to uniprot id : Q9BRR6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154330\n",
      "\n",
      "Gene ensembl id ENSG00000154330 changed to uniprot id : Q15124.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169299\n",
      "\n",
      "Gene ensembl id ENSG00000169299 changed to uniprot id : Q96G03.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000079739\n",
      "\n",
      "Gene ensembl id ENSG00000079739 changed to uniprot id : P36871.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131482\n",
      "\n",
      "Gene ensembl id ENSG00000131482 changed to uniprot id : P35575.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152254\n",
      "\n",
      "Gene ensembl id ENSG00000152254 changed to uniprot id : Q9NQR9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141349\n",
      "\n",
      "Gene ensembl id ENSG00000141349 changed to uniprot id : K7ENK1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143891\n",
      "\n",
      "Gene ensembl id ENSG00000143891 changed to uniprot id : Q96C23.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121691\n",
      "\n",
      "Gene ensembl id ENSG00000121691 changed to uniprot id : P04040.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000214013\n",
      "\n",
      "Gene ensembl id ENSG00000214013 changed to uniprot id : H3BRQ6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171298\n",
      "\n",
      "Gene ensembl id ENSG00000171298 changed to uniprot id : P10253.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000092529\n",
      "\n",
      "Gene ensembl id ENSG00000092529 changed to uniprot id : P20807.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000179087\n",
      "\n",
      "No uniprot id found for gene ENSG00000179087.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109814\n",
      "\n",
      "Gene ensembl id ENSG00000109814 changed to uniprot id : O60701.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000118094\n",
      "\n",
      "Gene ensembl id ENSG00000118094 changed to uniprot id : O43280.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142102\n",
      "\n",
      "Gene ensembl id ENSG00000142102 changed to uniprot id : Q32M88.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163754\n",
      "\n",
      "Gene ensembl id ENSG00000163754 changed to uniprot id : G5E9W8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000056998\n",
      "\n",
      "Gene ensembl id ENSG00000056998 changed to uniprot id : O15488.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104812\n",
      "\n",
      "Gene ensembl id ENSG00000104812 changed to uniprot id : P13807.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111713\n",
      "\n",
      "Gene ensembl id ENSG00000111713 changed to uniprot id : P54840.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173281\n",
      "\n",
      "Gene ensembl id ENSG00000173281 changed to uniprot id : Q86XI6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119938\n",
      "\n",
      "Gene ensembl id ENSG00000119938 changed to uniprot id : Q9UQK1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114480\n",
      "\n",
      "Gene ensembl id ENSG00000114480 changed to uniprot id : Q04446.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100504\n",
      "\n",
      "Gene ensembl id ENSG00000100504 changed to uniprot id : P06737.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100994\n",
      "\n",
      "Gene ensembl id ENSG00000100994 changed to uniprot id : P11216.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000068976\n",
      "\n",
      "Gene ensembl id ENSG00000068976 changed to uniprot id : P11217.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162688\n",
      "\n",
      "Gene ensembl id ENSG00000162688 changed to uniprot id : P35573.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000243480\n",
      "\n",
      "Gene ensembl id ENSG00000243480 changed to uniprot id : P04746.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174876\n",
      "\n",
      "Gene ensembl id ENSG00000174876 changed to uniprot id : P0DTE7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000240038\n",
      "\n",
      "Gene ensembl id ENSG00000240038 changed to uniprot id : P19961.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000187733\n",
      "\n",
      "Gene ensembl id ENSG00000187733 changed to uniprot id : P0DTE8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000237763\n",
      "\n",
      "Gene ensembl id ENSG00000237763 changed to uniprot id : P0DUB6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169764\n",
      "\n",
      "Gene ensembl id ENSG00000169764 changed to uniprot id : C9J3M0.\n",
      "\n",
      "Gene already has uniprot id : UGP1\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117308\n",
      "\n",
      "Gene ensembl id ENSG00000117308 changed to uniprot id : Q5QPP1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156958\n",
      "\n",
      "Gene ensembl id ENSG00000156958 changed to uniprot id : Q01415.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108479\n",
      "\n",
      "Gene ensembl id ENSG00000108479 changed to uniprot id : P51570.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213930\n",
      "\n",
      "Gene ensembl id ENSG00000213930 changed to uniprot id : P07902.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000090402\n",
      "\n",
      "Gene ensembl id ENSG00000090402 changed to uniprot id : P14410.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166262\n",
      "\n",
      "Gene ensembl id ENSG00000166262 changed to uniprot id : Q96M60.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163521\n",
      "\n",
      "Gene ensembl id ENSG00000163521 changed to uniprot id : Q6UWU2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170266\n",
      "\n",
      "Gene ensembl id ENSG00000170266 changed to uniprot id : F8WEN1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115850\n",
      "\n",
      "Gene ensembl id ENSG00000115850 changed to uniprot id : P09848.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102393\n",
      "\n",
      "Gene ensembl id ENSG00000102393 changed to uniprot id : A0A3B3IRU3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117411\n",
      "\n",
      "Gene ensembl id ENSG00000117411 changed to uniprot id : O60909.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000086062\n",
      "\n",
      "Gene ensembl id ENSG00000086062 changed to uniprot id : P15291.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167531\n",
      "\n",
      "Gene ensembl id ENSG00000167531 changed to uniprot id : P00709.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138030\n",
      "\n",
      "Gene ensembl id ENSG00000138030 changed to uniprot id : P50053.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170525\n",
      "\n",
      "Gene ensembl id ENSG00000170525 changed to uniprot id : Q16875.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123836\n",
      "\n",
      "Gene ensembl id ENSG00000123836 changed to uniprot id : O60825.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114268\n",
      "\n",
      "Gene ensembl id ENSG00000114268 changed to uniprot id : Q16877.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158571\n",
      "\n",
      "Gene ensembl id ENSG00000158571 changed to uniprot id : I1Z9G3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140263\n",
      "\n",
      "Gene ensembl id ENSG00000140263 changed to uniprot id : Q00796.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198074\n",
      "\n",
      "Gene ensembl id ENSG00000198074 changed to uniprot id : O60218.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000085662\n",
      "\n",
      "Gene ensembl id ENSG00000085662 changed to uniprot id : P15121.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178802\n",
      "\n",
      "Gene ensembl id ENSG00000178802 changed to uniprot id : B4DYB8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100417\n",
      "\n",
      "Gene ensembl id ENSG00000100417 changed to uniprot id : Q92871.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100413\n",
      "\n",
      "Gene ensembl id ENSG00000100413 changed to uniprot id : A0A024R1P3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140650\n",
      "\n",
      "Gene ensembl id ENSG00000140650 changed to uniprot id : H3BM92.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173540\n",
      "\n",
      "Gene ensembl id ENSG00000173540 changed to uniprot id : A0A024R329.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000144591\n",
      "\n",
      "Gene ensembl id ENSG00000144591 changed to uniprot id : A0A804HJC3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164068\n",
      "\n",
      "Gene ensembl id ENSG00000164068 changed to uniprot id : Q5XPI4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112699\n",
      "\n",
      "Gene ensembl id ENSG00000112699 changed to uniprot id : O60547.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104522\n",
      "\n",
      "Gene ensembl id ENSG00000104522 changed to uniprot id : E9PLH9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116783\n",
      "\n",
      "Gene ensembl id ENSG00000116783 changed to uniprot id : Q59H18.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157353\n",
      "\n",
      "Gene ensembl id ENSG00000157353 changed to uniprot id : Q8N0W3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000078237\n",
      "\n",
      "Gene ensembl id ENSG00000078237 changed to uniprot id : Q9NQ88.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169738\n",
      "\n",
      "Gene ensembl id ENSG00000169738 changed to uniprot id : Q7Z4W1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116353\n",
      "\n",
      "Gene ensembl id ENSG00000116353 changed to uniprot id : Q9BV79.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115425\n",
      "\n",
      "Gene ensembl id ENSG00000115425 changed to uniprot id : Q9BY49.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000093217\n",
      "\n",
      "Gene ensembl id ENSG00000093217 changed to uniprot id : O75191.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000053371\n",
      "\n",
      "Gene ensembl id ENSG00000053371 changed to uniprot id : O43488.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171174\n",
      "\n",
      "Gene ensembl id ENSG00000171174 changed to uniprot id : Q9H477.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120697\n",
      "\n",
      "Gene ensembl id ENSG00000120697 changed to uniprot id : Q9Y673.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115652\n",
      "\n",
      "Gene ensembl id ENSG00000115652 changed to uniprot id : Q8NBZ7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124767\n",
      "\n",
      "Gene ensembl id ENSG00000124767 changed to uniprot id : Q04760.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167699\n",
      "\n",
      "Gene ensembl id ENSG00000167699 changed to uniprot id : I3NI24.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000063854\n",
      "\n",
      "Gene ensembl id ENSG00000063854 changed to uniprot id : Q16775.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103253\n",
      "\n",
      "Gene ensembl id ENSG00000103253 changed to uniprot id : B4DED4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182224\n",
      "\n",
      "Gene ensembl id ENSG00000182224 changed to uniprot id : Q6P9G0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166816\n",
      "\n",
      "Gene ensembl id ENSG00000166816 changed to uniprot id : Q86WU2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151376\n",
      "\n",
      "Gene ensembl id ENSG00000151376 changed to uniprot id : E9PND9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000082212\n",
      "\n",
      "Gene ensembl id ENSG00000082212 changed to uniprot id : P23368.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000013392\n",
      "\n",
      "Gene ensembl id ENSG00000013392 changed to uniprot id : Q9UIY3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000065833\n",
      "\n",
      "Gene ensembl id ENSG00000065833 changed to uniprot id : P48163.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100889\n",
      "\n",
      "Gene ensembl id ENSG00000100889 changed to uniprot id : Q16822.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173599\n",
      "\n",
      "Gene ensembl id ENSG00000173599 changed to uniprot id : P11498.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137106\n",
      "\n",
      "Gene ensembl id ENSG00000137106 changed to uniprot id : Q9UBQ7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148090\n",
      "\n",
      "Gene ensembl id ENSG00000148090 changed to uniprot id : Q13825.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134333\n",
      "\n",
      "Gene ensembl id ENSG00000134333 changed to uniprot id : P00338.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196177\n",
      "\n",
      "Gene ensembl id ENSG00000196177 changed to uniprot id : P45954.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122971\n",
      "\n",
      "Gene ensembl id ENSG00000122971 changed to uniprot id : P16219.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151498\n",
      "\n",
      "Gene ensembl id ENSG00000151498 changed to uniprot id : Q9UKU7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117054\n",
      "\n",
      "Gene ensembl id ENSG00000117054 changed to uniprot id : P11310.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111271\n",
      "\n",
      "Gene ensembl id ENSG00000111271 changed to uniprot id : Q6JQN1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000240303\n",
      "\n",
      "Gene ensembl id ENSG00000240303 changed to uniprot id : Q709F0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000177646\n",
      "\n",
      "Gene ensembl id ENSG00000177646 changed to uniprot id : Q9H845.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103150\n",
      "\n",
      "Gene ensembl id ENSG00000103150 changed to uniprot id : O95822.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119711\n",
      "\n",
      "Gene ensembl id ENSG00000119711 changed to uniprot id : Q02252.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116133\n",
      "\n",
      "Gene ensembl id ENSG00000116133 changed to uniprot id : Q15392.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198130\n",
      "\n",
      "Gene ensembl id ENSG00000198130 changed to uniprot id : Q6NVY1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138400\n",
      "\n",
      "Gene ensembl id ENSG00000138400 changed to uniprot id : Q5I0G3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139988\n",
      "\n",
      "Gene ensembl id ENSG00000139988 changed to uniprot id : Q96NR8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170786\n",
      "\n",
      "Gene ensembl id ENSG00000170786 changed to uniprot id : Q8N3Y7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170426\n",
      "\n",
      "Gene ensembl id ENSG00000170426 changed to uniprot id : Q8NEX9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000240857\n",
      "\n",
      "Gene ensembl id ENSG00000240857 changed to uniprot id : Q9HBH5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109854\n",
      "\n",
      "Gene ensembl id ENSG00000109854 changed to uniprot id : Q9BUP3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159692\n",
      "\n",
      "Gene ensembl id ENSG00000159692 changed to uniprot id : Q13363.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112474\n",
      "\n",
      "Gene ensembl id ENSG00000112474 changed to uniprot id : Q92506.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204228\n",
      "\n",
      "Gene ensembl id ENSG00000204228 changed to uniprot id : Q92506.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121039\n",
      "\n",
      "Gene ensembl id ENSG00000121039 changed to uniprot id : A0A024R7X6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000187134\n",
      "\n",
      "Gene ensembl id ENSG00000187134 changed to uniprot id : Q04828.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198610\n",
      "\n",
      "Gene ensembl id ENSG00000198610 changed to uniprot id : P17516.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099377\n",
      "\n",
      "Gene ensembl id ENSG00000099377 changed to uniprot id : Q9H2F3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000087076\n",
      "\n",
      "Gene ensembl id ENSG00000087076 changed to uniprot id : Q9BPX1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164039\n",
      "\n",
      "Gene ensembl id ENSG00000164039 changed to uniprot id : Q9BUT1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198189\n",
      "\n",
      "Gene ensembl id ENSG00000198189 changed to uniprot id : D6RCD0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176387\n",
      "\n",
      "Gene ensembl id ENSG00000176387 changed to uniprot id : P80365.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145439\n",
      "\n",
      "Gene ensembl id ENSG00000145439 changed to uniprot id : Q8N4T8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160439\n",
      "\n",
      "Gene ensembl id ENSG00000160439 changed to uniprot id : Q8NBN7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120437\n",
      "\n",
      "Gene ensembl id ENSG00000120437 changed to uniprot id : Q9BWD1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000081760\n",
      "\n",
      "Gene ensembl id ENSG00000081760 changed to uniprot id : Q86V21.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000161267\n",
      "\n",
      "Gene ensembl id ENSG00000161267 changed to uniprot id : Q02338.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134240\n",
      "\n",
      "Gene ensembl id ENSG00000134240 changed to uniprot id : P54868.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112972\n",
      "\n",
      "Gene ensembl id ENSG00000112972 changed to uniprot id : Q01581.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000229937\n",
      "\n",
      "Gene ensembl id ENSG00000229937 changed to uniprot id : P21108.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101911\n",
      "\n",
      "Gene ensembl id ENSG00000101911 changed to uniprot id : P11908.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147224\n",
      "\n",
      "Gene ensembl id ENSG00000147224 changed to uniprot id : A0A6Q8PG31.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160211\n",
      "\n",
      "Gene ensembl id ENSG00000160211 changed to uniprot id : P11413.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158019\n",
      "\n",
      "Gene ensembl id ENSG00000158019 changed to uniprot id : Q9NXR7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000153574\n",
      "\n",
      "Gene ensembl id ENSG00000153574 changed to uniprot id : P49247.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000023697\n",
      "\n",
      "Gene ensembl id ENSG00000023697 changed to uniprot id : Q9Y315.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000007350\n",
      "\n",
      "Gene ensembl id ENSG00000007350 changed to uniprot id : P51854.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163931\n",
      "\n",
      "Gene ensembl id ENSG00000163931 changed to uniprot id : P29401.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151005\n",
      "\n",
      "Gene ensembl id ENSG00000151005 changed to uniprot id : Q9H0I9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142657\n",
      "\n",
      "Gene ensembl id ENSG00000142657 changed to uniprot id : P52209.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197713\n",
      "\n",
      "Gene ensembl id ENSG00000197713 changed to uniprot id : Q96AT9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000177156\n",
      "\n",
      "Gene ensembl id ENSG00000177156 changed to uniprot id : P37837.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000049239\n",
      "\n",
      "Gene ensembl id ENSG00000049239 changed to uniprot id : O95479.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130313\n",
      "\n",
      "Gene ensembl id ENSG00000130313 changed to uniprot id : O95336.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138185\n",
      "\n",
      "Gene ensembl id ENSG00000138185 changed to uniprot id : P49961.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168032\n",
      "\n",
      "Gene ensembl id ENSG00000168032 changed to uniprot id : O75355.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000187097\n",
      "\n",
      "Gene ensembl id ENSG00000187097 changed to uniprot id : A0A024R6D3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171302\n",
      "\n",
      "Gene ensembl id ENSG00000171302 changed to uniprot id : K7EIP9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197586\n",
      "\n",
      "Gene ensembl id ENSG00000197586 changed to uniprot id : O75354.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000188833\n",
      "\n",
      "Gene ensembl id ENSG00000188833 changed to uniprot id : Q5MY95.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197217\n",
      "\n",
      "Gene ensembl id ENSG00000197217 changed to uniprot id : Q9Y227.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000054179\n",
      "\n",
      "Gene ensembl id ENSG00000054179 changed to uniprot id : Q9Y5L3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123201\n",
      "\n",
      "No uniprot id found for gene ENSG00000123201.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000061918\n",
      "\n",
      "Gene ensembl id ENSG00000061918 changed to uniprot id : Q02153.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101890\n",
      "\n",
      "Gene ensembl id ENSG00000101890 changed to uniprot id : P51841.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152402\n",
      "\n",
      "Gene ensembl id ENSG00000152402 changed to uniprot id : P33402.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164116\n",
      "\n",
      "Gene ensembl id ENSG00000164116 changed to uniprot id : Q6PJR4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143199\n",
      "\n",
      "Gene ensembl id ENSG00000143199 changed to uniprot id : Q96PN6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164742\n",
      "\n",
      "Gene ensembl id ENSG00000164742 changed to uniprot id : Q08828.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159899\n",
      "\n",
      "Gene ensembl id ENSG00000159899 changed to uniprot id : P20594.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155897\n",
      "\n",
      "Gene ensembl id ENSG00000155897 changed to uniprot id : P40145.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138031\n",
      "\n",
      "Gene ensembl id ENSG00000138031 changed to uniprot id : O60266.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129467\n",
      "\n",
      "Gene ensembl id ENSG00000129467 changed to uniprot id : Q8NFM4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174233\n",
      "\n",
      "Gene ensembl id ENSG00000174233 changed to uniprot id : O43306.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000078295\n",
      "\n",
      "Gene ensembl id ENSG00000078295 changed to uniprot id : Q08462.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173175\n",
      "\n",
      "Gene ensembl id ENSG00000173175 changed to uniprot id : O95622.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132518\n",
      "\n",
      "Gene ensembl id ENSG00000132518 changed to uniprot id : Q02846.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070019\n",
      "\n",
      "Gene ensembl id ENSG00000070019 changed to uniprot id : P25092.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162104\n",
      "\n",
      "Gene ensembl id ENSG00000162104 changed to uniprot id : O60503.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169418\n",
      "\n",
      "Gene ensembl id ENSG00000169418 changed to uniprot id : P16066.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121281\n",
      "\n",
      "Gene ensembl id ENSG00000121281 changed to uniprot id : P51828.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162433\n",
      "\n",
      "Gene ensembl id ENSG00000162433 changed to uniprot id : P27144.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106992\n",
      "\n",
      "Gene ensembl id ENSG00000106992 changed to uniprot id : P00568.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140057\n",
      "\n",
      "Gene ensembl id ENSG00000140057 changed to uniprot id : Q96M32.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165695\n",
      "\n",
      "Gene ensembl id ENSG00000165695 changed to uniprot id : Q96MA6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154027\n",
      "\n",
      "Gene ensembl id ENSG00000154027 changed to uniprot id : Q9Y6K8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147853\n",
      "\n",
      "Gene ensembl id ENSG00000147853 changed to uniprot id : Q9UIJ7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000004455\n",
      "\n",
      "Gene ensembl id ENSG00000004455 changed to uniprot id : P54819.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122643\n",
      "\n",
      "Gene ensembl id ENSG00000122643 changed to uniprot id : A0A024RA81.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135318\n",
      "\n",
      "Gene ensembl id ENSG00000135318 changed to uniprot id : P21589.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000205309\n",
      "\n",
      "Gene ensembl id ENSG00000205309 changed to uniprot id : Q9NPB1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125458\n",
      "\n",
      "Gene ensembl id ENSG00000125458 changed to uniprot id : Q8TCD5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000076685\n",
      "\n",
      "Gene ensembl id ENSG00000076685 changed to uniprot id : P49902.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000014257\n",
      "\n",
      "Gene ensembl id ENSG00000014257 changed to uniprot id : P15309.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141698\n",
      "\n",
      "Gene ensembl id ENSG00000141698 changed to uniprot id : Q969T7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185013\n",
      "\n",
      "Gene ensembl id ENSG00000185013 changed to uniprot id : Q96P26.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116981\n",
      "\n",
      "Gene ensembl id ENSG00000116981 changed to uniprot id : Q9BXI3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156136\n",
      "\n",
      "Gene ensembl id ENSG00000156136 changed to uniprot id : P27707.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156110\n",
      "\n",
      "Gene ensembl id ENSG00000156110 changed to uniprot id : P55263.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133256\n",
      "\n",
      "Gene ensembl id ENSG00000133256 changed to uniprot id : P35913.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113448\n",
      "\n",
      "Gene ensembl id ENSG00000113448 changed to uniprot id : Q08499.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000095464\n",
      "\n",
      "Gene ensembl id ENSG00000095464 changed to uniprot id : P51160.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000073417\n",
      "\n",
      "Gene ensembl id ENSG00000073417 changed to uniprot id : O60658.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137496\n",
      "\n",
      "Gene ensembl id ENSG00000137496 changed to uniprot id : G3V1C5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160191\n",
      "\n",
      "Gene ensembl id ENSG00000160191 changed to uniprot id : O76083.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105650\n",
      "\n",
      "Gene ensembl id ENSG00000105650 changed to uniprot id : Q08493.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113231\n",
      "\n",
      "Gene ensembl id ENSG00000113231 changed to uniprot id : O95263.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115252\n",
      "\n",
      "Gene ensembl id ENSG00000115252 changed to uniprot id : P54750.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132915\n",
      "\n",
      "Gene ensembl id ENSG00000132915 changed to uniprot id : F1T0K3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154678\n",
      "\n",
      "Gene ensembl id ENSG00000154678 changed to uniprot id : Q14123.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152270\n",
      "\n",
      "Gene ensembl id ENSG00000152270 changed to uniprot id : Q13370.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139053\n",
      "\n",
      "Gene ensembl id ENSG00000139053 changed to uniprot id : Q13956.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138735\n",
      "\n",
      "Gene ensembl id ENSG00000138735 changed to uniprot id : O76074.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185527\n",
      "\n",
      "Gene ensembl id ENSG00000185527 changed to uniprot id : P18545.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171408\n",
      "\n",
      "Gene ensembl id ENSG00000171408 changed to uniprot id : Q9NP56.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000205268\n",
      "\n",
      "Gene ensembl id ENSG00000205268 changed to uniprot id : Q13946.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000065989\n",
      "\n",
      "Gene ensembl id ENSG00000065989 changed to uniprot id : P27815.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186642\n",
      "\n",
      "Gene ensembl id ENSG00000186642 changed to uniprot id : O00408.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156973\n",
      "\n",
      "Gene ensembl id ENSG00000156973 changed to uniprot id : O43924.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128655\n",
      "\n",
      "Gene ensembl id ENSG00000128655 changed to uniprot id : Q9HCR9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184588\n",
      "\n",
      "Gene ensembl id ENSG00000184588 changed to uniprot id : Q07343.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123360\n",
      "\n",
      "Gene ensembl id ENSG00000123360 changed to uniprot id : Q01064.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112541\n",
      "\n",
      "Gene ensembl id ENSG00000112541 changed to uniprot id : A0A1B1UZQ1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172572\n",
      "\n",
      "Gene ensembl id ENSG00000172572 changed to uniprot id : Q14432.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143774\n",
      "\n",
      "Gene ensembl id ENSG00000143774 changed to uniprot id : Q16774.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138363\n",
      "\n",
      "Gene ensembl id ENSG00000138363 changed to uniprot id : P31939.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178035\n",
      "\n",
      "Gene ensembl id ENSG00000178035 changed to uniprot id : P12268.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106348\n",
      "\n",
      "Gene ensembl id ENSG00000106348 changed to uniprot id : C9J381.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000035687\n",
      "\n",
      "Gene ensembl id ENSG00000035687 changed to uniprot id : P30520.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185100\n",
      "\n",
      "Gene ensembl id ENSG00000185100 changed to uniprot id : Q8N142.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163655\n",
      "\n",
      "Gene ensembl id ENSG00000163655 changed to uniprot id : P49915.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133805\n",
      "\n",
      "Gene ensembl id ENSG00000133805 changed to uniprot id : Q01432.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116748\n",
      "\n",
      "Gene ensembl id ENSG00000116748 changed to uniprot id : P23109.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116337\n",
      "\n",
      "Gene ensembl id ENSG00000116337 changed to uniprot id : A0A5F9UK94.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198931\n",
      "\n",
      "Gene ensembl id ENSG00000198931 changed to uniprot id : P07741.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125877\n",
      "\n",
      "Gene ensembl id ENSG00000125877 changed to uniprot id : Q9BY32.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128059\n",
      "\n",
      "Gene ensembl id ENSG00000128059 changed to uniprot id : A8K4H7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100354\n",
      "\n",
      "Gene ensembl id ENSG00000100354 changed to uniprot id : Q9UPQ9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000239900\n",
      "\n",
      "Gene ensembl id ENSG00000239900 changed to uniprot id : A0A096LP92.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165704\n",
      "\n",
      "Gene ensembl id ENSG00000165704 changed to uniprot id : P00492.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137198\n",
      "\n",
      "Gene ensembl id ENSG00000137198 changed to uniprot id : P36959.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100938\n",
      "\n",
      "Gene ensembl id ENSG00000100938 changed to uniprot id : Q9P2T1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000087008\n",
      "\n",
      "Gene ensembl id ENSG00000087008 changed to uniprot id : O15254.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000072778\n",
      "\n",
      "Gene ensembl id ENSG00000072778 changed to uniprot id : P49748.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196839\n",
      "\n",
      "Gene ensembl id ENSG00000196839 changed to uniprot id : P00813.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000093072\n",
      "\n",
      "Gene ensembl id ENSG00000093072 changed to uniprot id : Q9NZK5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198805\n",
      "\n",
      "Gene ensembl id ENSG00000198805 changed to uniprot id : P00491.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114956\n",
      "\n",
      "Gene ensembl id ENSG00000114956 changed to uniprot id : Q16854.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000025708\n",
      "\n",
      "Gene ensembl id ENSG00000025708 changed to uniprot id : E5KRG5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167325\n",
      "\n",
      "Gene ensembl id ENSG00000167325 changed to uniprot id : P23921.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171848\n",
      "\n",
      "Gene ensembl id ENSG00000171848 changed to uniprot id : P31350.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000048392\n",
      "\n",
      "Gene ensembl id ENSG00000048392 changed to uniprot id : Q7LG56.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136810\n",
      "\n",
      "Gene ensembl id ENSG00000136810 changed to uniprot id : P10599.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159131\n",
      "\n",
      "Gene ensembl id ENSG00000159131 changed to uniprot id : P22102.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128050\n",
      "\n",
      "Gene ensembl id ENSG00000128050 changed to uniprot id : P22234.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178921\n",
      "\n",
      "Gene ensembl id ENSG00000178921 changed to uniprot id : O15067.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198276\n",
      "\n",
      "Gene ensembl id ENSG00000198276 changed to uniprot id : Q9NWZ5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143179\n",
      "\n",
      "Gene ensembl id ENSG00000143179 changed to uniprot id : Q9BZX2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130717\n",
      "\n",
      "Gene ensembl id ENSG00000130717 changed to uniprot id : Q9HA47.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121053\n",
      "\n",
      "Gene ensembl id ENSG00000121053 changed to uniprot id : P11678.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117592\n",
      "\n",
      "Gene ensembl id ENSG00000117592 changed to uniprot id : P30041.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167419\n",
      "\n",
      "Gene ensembl id ENSG00000167419 changed to uniprot id : P22079.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147485\n",
      "\n",
      "Gene ensembl id ENSG00000147485 changed to uniprot id : A1KZ92.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165609\n",
      "\n",
      "Gene ensembl id ENSG00000165609 changed to uniprot id : Q9UKK9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000011052\n",
      "\n",
      "Gene ensembl id ENSG00000011052 changed to uniprot id : Q32Q12.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000243678\n",
      "\n",
      "Gene ensembl id ENSG00000243678 changed to uniprot id : P22392.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103202\n",
      "\n",
      "Gene ensembl id ENSG00000103202 changed to uniprot id : O00746.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172113\n",
      "\n",
      "Gene ensembl id ENSG00000172113 changed to uniprot id : A0A0C4DG91.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103024\n",
      "\n",
      "Gene ensembl id ENSG00000103024 changed to uniprot id : Q13232.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000239672\n",
      "\n",
      "Gene ensembl id ENSG00000239672 changed to uniprot id : P15531.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143156\n",
      "\n",
      "Gene ensembl id ENSG00000143156 changed to uniprot id : Q9Y5B8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112981\n",
      "\n",
      "Gene ensembl id ENSG00000112981 changed to uniprot id : P56597.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184470\n",
      "\n",
      "Gene ensembl id ENSG00000184470 changed to uniprot id : Q9NNW7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197763\n",
      "\n",
      "Gene ensembl id ENSG00000197763 changed to uniprot id : Q86VQ6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198431\n",
      "\n",
      "Gene ensembl id ENSG00000198431 changed to uniprot id : Q16881.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162368\n",
      "\n",
      "Gene ensembl id ENSG00000162368 changed to uniprot id : P30085.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000047230\n",
      "\n",
      "Gene ensembl id ENSG00000047230 changed to uniprot id : Q9NRF8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171793\n",
      "\n",
      "Gene ensembl id ENSG00000171793 changed to uniprot id : P17812.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000084774\n",
      "\n",
      "Gene ensembl id ENSG00000084774 changed to uniprot id : P27708.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114491\n",
      "\n",
      "Gene ensembl id ENSG00000114491 changed to uniprot id : A8K5J1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197594\n",
      "\n",
      "Gene ensembl id ENSG00000197594 changed to uniprot id : P22413.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183828\n",
      "\n",
      "Gene ensembl id ENSG00000183828 changed to uniprot id : O95848.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154269\n",
      "\n",
      "Gene ensembl id ENSG00000154269 changed to uniprot id : O14638.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116199\n",
      "\n",
      "Gene ensembl id ENSG00000116199 changed to uniprot id : O75063.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113657\n",
      "\n",
      "Gene ensembl id ENSG00000113657 changed to uniprot id : Q14195.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147647\n",
      "\n",
      "Gene ensembl id ENSG00000147647 changed to uniprot id : Q14117.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000092964\n",
      "\n",
      "Gene ensembl id ENSG00000092964 changed to uniprot id : Q16555.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111732\n",
      "\n",
      "Gene ensembl id ENSG00000111732 changed to uniprot id : Q9GZX7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158825\n",
      "\n",
      "Gene ensembl id ENSG00000158825 changed to uniprot id : P32320.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183696\n",
      "\n",
      "Gene ensembl id ENSG00000183696 changed to uniprot id : Q16831.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000007001\n",
      "\n",
      "Gene ensembl id ENSG00000007001 changed to uniprot id : O95045.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173627\n",
      "\n",
      "Gene ensembl id ENSG00000173627 changed to uniprot id : Q8WW27.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102967\n",
      "\n",
      "Gene ensembl id ENSG00000102967 changed to uniprot id : Q02127.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168393\n",
      "\n",
      "Gene ensembl id ENSG00000168393 changed to uniprot id : P23919.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128951\n",
      "\n",
      "Gene ensembl id ENSG00000128951 changed to uniprot id : P33316.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176890\n",
      "\n",
      "Gene ensembl id ENSG00000176890 changed to uniprot id : P04818.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167900\n",
      "\n",
      "Gene ensembl id ENSG00000167900 changed to uniprot id : P04183.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166548\n",
      "\n",
      "Gene ensembl id ENSG00000166548 changed to uniprot id : O00142.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134326\n",
      "\n",
      "Gene ensembl id ENSG00000134326 changed to uniprot id : Q5EBM0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000087470\n",
      "\n",
      "Gene ensembl id ENSG00000087470 changed to uniprot id : B4DDQ3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178952\n",
      "\n",
      "Gene ensembl id ENSG00000178952 changed to uniprot id : A0A384ME17.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197959\n",
      "\n",
      "Gene ensembl id ENSG00000197959 changed to uniprot id : Q9UQ16.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167658\n",
      "\n",
      "Gene ensembl id ENSG00000167658 changed to uniprot id : P13639.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168827\n",
      "\n",
      "Gene ensembl id ENSG00000168827 changed to uniprot id : Q96RP9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164347\n",
      "\n",
      "Gene ensembl id ENSG00000164347 changed to uniprot id : Q969S9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140598\n",
      "\n",
      "Gene ensembl id ENSG00000140598 changed to uniprot id : Q7Z2Z2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156508\n",
      "\n",
      "Gene ensembl id ENSG00000156508 changed to uniprot id : P68104.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106976\n",
      "\n",
      "Gene ensembl id ENSG00000106976 changed to uniprot id : Q05193.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101210\n",
      "\n",
      "Gene ensembl id ENSG00000101210 changed to uniprot id : Q05639.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198682\n",
      "\n",
      "Gene ensembl id ENSG00000198682 changed to uniprot id : O95340.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138801\n",
      "\n",
      "Gene ensembl id ENSG00000138801 changed to uniprot id : O43252.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129187\n",
      "\n",
      "Gene ensembl id ENSG00000129187 changed to uniprot id : D6R9P0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100479\n",
      "\n",
      "Gene ensembl id ENSG00000100479 changed to uniprot id : P56282.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112941\n",
      "\n",
      "Gene ensembl id ENSG00000112941 changed to uniprot id : D6RJD0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170734\n",
      "\n",
      "Gene ensembl id ENSG00000170734 changed to uniprot id : Q9Y253.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101751\n",
      "\n",
      "Gene ensembl id ENSG00000101751 changed to uniprot id : Q9UNA4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106628\n",
      "\n",
      "Gene ensembl id ENSG00000106628 changed to uniprot id : P49005.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070501\n",
      "\n",
      "Gene ensembl id ENSG00000070501 changed to uniprot id : P06746.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000077514\n",
      "\n",
      "Gene ensembl id ENSG00000077514 changed to uniprot id : Q15054.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132382\n",
      "\n",
      "Gene ensembl id ENSG00000132382 changed to uniprot id : Q9BQG0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000051341\n",
      "\n",
      "Gene ensembl id ENSG00000051341 changed to uniprot id : O75417.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101868\n",
      "\n",
      "Gene ensembl id ENSG00000101868 changed to uniprot id : P09884.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000014138\n",
      "\n",
      "Gene ensembl id ENSG00000014138 changed to uniprot id : Q14181.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000068654\n",
      "\n",
      "Gene ensembl id ENSG00000068654 changed to uniprot id : O95602.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000005075\n",
      "\n",
      "Gene ensembl id ENSG00000005075 changed to uniprot id : P52435.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000107951\n",
      "\n",
      "Gene ensembl id ENSG00000107951 changed to uniprot id : Q9NVV4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115421\n",
      "\n",
      "Gene ensembl id ENSG00000115421 changed to uniprot id : Q9BWT3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099817\n",
      "\n",
      "Gene ensembl id ENSG00000099817 changed to uniprot id : P19388.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100142\n",
      "\n",
      "Gene ensembl id ENSG00000100142 changed to uniprot id : P61218.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099821\n",
      "\n",
      "Gene ensembl id ENSG00000099821 changed to uniprot id : O00411.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000047315\n",
      "\n",
      "Gene ensembl id ENSG00000047315 changed to uniprot id : P30876.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000144231\n",
      "\n",
      "Gene ensembl id ENSG00000144231 changed to uniprot id : O15514.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000039523\n",
      "\n",
      "Gene ensembl id ENSG00000039523 changed to uniprot id : Q6ZS17.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000218823\n",
      "\n",
      "Gene ensembl id ENSG00000218823 changed to uniprot id : Q9NRJ5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164329\n",
      "\n",
      "Gene ensembl id ENSG00000164329 changed to uniprot id : Q6PIY7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000181222\n",
      "\n",
      "Gene ensembl id ENSG00000181222 changed to uniprot id : A0A6Q8PGB0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000013503\n",
      "\n",
      "Gene ensembl id ENSG00000013503 changed to uniprot id : Q9NW08.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148606\n",
      "\n",
      "Gene ensembl id ENSG00000148606 changed to uniprot id : O14802.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163882\n",
      "\n",
      "Gene ensembl id ENSG00000163882 changed to uniprot id : P52434.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171453\n",
      "\n",
      "Gene ensembl id ENSG00000171453 changed to uniprot id : O15160.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186184\n",
      "\n",
      "Gene ensembl id ENSG00000186184 changed to uniprot id : P0DPB6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137054\n",
      "\n",
      "Gene ensembl id ENSG00000137054 changed to uniprot id : Q9GZS1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121851\n",
      "\n",
      "Gene ensembl id ENSG00000121851 changed to uniprot id : Q9BT43.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105258\n",
      "\n",
      "Gene ensembl id ENSG00000105258 changed to uniprot id : P36954.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168002\n",
      "\n",
      "Gene ensembl id ENSG00000168002 changed to uniprot id : P62487.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132664\n",
      "\n",
      "Gene ensembl id ENSG00000132664 changed to uniprot id : Q9H1D9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138035\n",
      "\n",
      "Gene ensembl id ENSG00000138035 changed to uniprot id : Q8TCS8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113456\n",
      "\n",
      "Gene ensembl id ENSG00000113456 changed to uniprot id : O60671.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148672\n",
      "\n",
      "Gene ensembl id ENSG00000148672 changed to uniprot id : P00367.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182890\n",
      "\n",
      "Gene ensembl id ENSG00000182890 changed to uniprot id : P49448.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130707\n",
      "\n",
      "Gene ensembl id ENSG00000130707 changed to uniprot id : P00966.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000126522\n",
      "\n",
      "Gene ensembl id ENSG00000126522 changed to uniprot id : P04424.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000241258\n",
      "\n",
      "Gene ensembl id ENSG00000241258 changed to uniprot id : O75575.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159423\n",
      "\n",
      "Gene ensembl id ENSG00000159423 changed to uniprot id : A0A024RAC7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125166\n",
      "\n",
      "Gene ensembl id ENSG00000125166 changed to uniprot id : P00505.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169154\n",
      "\n",
      "Gene ensembl id ENSG00000169154 changed to uniprot id : Q8NHS2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120053\n",
      "\n",
      "Gene ensembl id ENSG00000120053 changed to uniprot id : P17174.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000091483\n",
      "\n",
      "Gene ensembl id ENSG00000091483 changed to uniprot id : P07954.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166183\n",
      "\n",
      "Gene ensembl id ENSG00000166183 changed to uniprot id : Q86U10.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162174\n",
      "\n",
      "Gene ensembl id ENSG00000162174 changed to uniprot id : Q7L266.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104687\n",
      "\n",
      "Gene ensembl id ENSG00000104687 changed to uniprot id : P00390.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000021826\n",
      "\n",
      "Gene ensembl id ENSG00000021826 changed to uniprot id : P31327.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135821\n",
      "\n",
      "Gene ensembl id ENSG00000135821 changed to uniprot id : A8YXX4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146166\n",
      "\n",
      "Gene ensembl id ENSG00000146166 changed to uniprot id : Q5TDP6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115419\n",
      "\n",
      "Gene ensembl id ENSG00000115419 changed to uniprot id : B7Z509.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135423\n",
      "\n",
      "Gene ensembl id ENSG00000135423 changed to uniprot id : G5E9X4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167701\n",
      "\n",
      "Gene ensembl id ENSG00000167701 changed to uniprot id : P24298.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070669\n",
      "\n",
      "Gene ensembl id ENSG00000070669 changed to uniprot id : P08243.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166123\n",
      "\n",
      "Gene ensembl id ENSG00000166123 changed to uniprot id : Q8TD30.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114021\n",
      "\n",
      "Gene ensembl id ENSG00000114021 changed to uniprot id : Q9NQR4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171097\n",
      "\n",
      "Gene ensembl id ENSG00000171097 changed to uniprot id : Q16773.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109576\n",
      "\n",
      "Gene ensembl id ENSG00000109576 changed to uniprot id : Q8N5Z0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137944\n",
      "\n",
      "Gene ensembl id ENSG00000137944 changed to uniprot id : Q6YP21.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183044\n",
      "\n",
      "Gene ensembl id ENSG00000183044 changed to uniprot id : P80404.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198650\n",
      "\n",
      "Gene ensembl id ENSG00000198650 changed to uniprot id : P17735.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000203797\n",
      "\n",
      "Gene ensembl id ENSG00000203797 changed to uniprot id : Q99489.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147471\n",
      "\n",
      "Gene ensembl id ENSG00000147471 changed to uniprot id : O94903.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108381\n",
      "\n",
      "Gene ensembl id ENSG00000108381 changed to uniprot id : P45381.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132744\n",
      "\n",
      "Gene ensembl id ENSG00000132744 changed to uniprot id : Q96HD9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000065154\n",
      "\n",
      "Gene ensembl id ENSG00000065154 changed to uniprot id : P04181.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000081181\n",
      "\n",
      "Gene ensembl id ENSG00000081181 changed to uniprot id : P78540.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000118520\n",
      "\n",
      "Gene ensembl id ENSG00000118520 changed to uniprot id : P05089.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104524\n",
      "\n",
      "Gene ensembl id ENSG00000104524 changed to uniprot id : B5MD87.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183010\n",
      "\n",
      "Gene ensembl id ENSG00000183010 changed to uniprot id : A0A024R8U9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143811\n",
      "\n",
      "Gene ensembl id ENSG00000143811 changed to uniprot id : Q96C36.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000250799\n",
      "\n",
      "Gene ensembl id ENSG00000250799 changed to uniprot id : A0A590UKC3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100033\n",
      "\n",
      "Gene ensembl id ENSG00000100033 changed to uniprot id : E7EQL6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000059573\n",
      "\n",
      "Gene ensembl id ENSG00000059573 changed to uniprot id : P54886.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169239\n",
      "\n",
      "Gene ensembl id ENSG00000169239 changed to uniprot id : Q9Y2D0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174990\n",
      "\n",
      "Gene ensembl id ENSG00000174990 changed to uniprot id : P35218.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167494\n",
      "\n",
      "No uniprot id found for gene ENSG00000167494.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164867\n",
      "\n",
      "Gene ensembl id ENSG00000164867 changed to uniprot id : P29474.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000007171\n",
      "\n",
      "Gene ensembl id ENSG00000007171 changed to uniprot id : P35228.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000264216\n",
      "\n",
      "No uniprot id found for gene ENSG00000264216.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000089250\n",
      "\n",
      "Gene ensembl id ENSG00000089250 changed to uniprot id : P29475.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123505\n",
      "\n",
      "Gene ensembl id ENSG00000123505 changed to uniprot id : Q5VXN5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116649\n",
      "\n",
      "Gene ensembl id ENSG00000116649 changed to uniprot id : P19623.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102172\n",
      "\n",
      "Gene ensembl id ENSG00000102172 changed to uniprot id : P52788.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142920\n",
      "\n",
      "Gene ensembl id ENSG00000142920 changed to uniprot id : Q96A70.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115758\n",
      "\n",
      "Gene ensembl id ENSG00000115758 changed to uniprot id : P11926.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131480\n",
      "\n",
      "Gene ensembl id ENSG00000131480 changed to uniprot id : O75106.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131471\n",
      "\n",
      "Gene ensembl id ENSG00000131471 changed to uniprot id : Q16853.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000002726\n",
      "\n",
      "Gene ensembl id ENSG00000002726 changed to uniprot id : P19801.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116771\n",
      "\n",
      "Gene ensembl id ENSG00000116771 changed to uniprot id : Q9BSE5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130005\n",
      "\n",
      "Gene ensembl id ENSG00000130005 changed to uniprot id : V9HWB2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172508\n",
      "\n",
      "Gene ensembl id ENSG00000172508 changed to uniprot id : A5YM72.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133313\n",
      "\n",
      "Gene ensembl id ENSG00000133313 changed to uniprot id : Q96KP4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000150656\n",
      "\n",
      "Gene ensembl id ENSG00000150656 changed to uniprot id : Q96KN2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149380\n",
      "\n",
      "Gene ensembl id ENSG00000149380 changed to uniprot id : Q7Z4N8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122884\n",
      "\n",
      "Gene ensembl id ENSG00000122884 changed to uniprot id : P13674.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000241935\n",
      "\n",
      "Gene ensembl id ENSG00000241935 changed to uniprot id : Q86XE5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099810\n",
      "\n",
      "Gene ensembl id ENSG00000099810 changed to uniprot id : Q13126.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130066\n",
      "\n",
      "Gene ensembl id ENSG00000130066 changed to uniprot id : P21673.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141504\n",
      "\n",
      "Gene ensembl id ENSG00000141504 changed to uniprot id : Q96F10.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148832\n",
      "\n",
      "Gene ensembl id ENSG00000148832 changed to uniprot id : Q6QHF9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000088826\n",
      "\n",
      "Gene ensembl id ENSG00000088826 changed to uniprot id : Q9NWM0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000223572\n",
      "\n",
      "Gene ensembl id ENSG00000223572 changed to uniprot id : P12532.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166165\n",
      "\n",
      "Gene ensembl id ENSG00000166165 changed to uniprot id : P12277.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104879\n",
      "\n",
      "Gene ensembl id ENSG00000104879 changed to uniprot id : P06732.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000237289\n",
      "\n",
      "Gene ensembl id ENSG00000237289 changed to uniprot id : P12532.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131730\n",
      "\n",
      "Gene ensembl id ENSG00000131730 changed to uniprot id : P17540.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123453\n",
      "\n",
      "Gene ensembl id ENSG00000123453 changed to uniprot id : Q9UL12.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000069535\n",
      "\n",
      "Gene ensembl id ENSG00000069535 changed to uniprot id : P27338.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000189221\n",
      "\n",
      "Gene ensembl id ENSG00000189221 changed to uniprot id : Q53YE7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112294\n",
      "\n",
      "Gene ensembl id ENSG00000112294 changed to uniprot id : P51649.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137261\n",
      "\n",
      "Gene ensembl id ENSG00000137261 changed to uniprot id : Q5VV43.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168237\n",
      "\n",
      "Gene ensembl id ENSG00000168237 changed to uniprot id : A1LQE8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000092621\n",
      "\n",
      "Gene ensembl id ENSG00000092621 changed to uniprot id : A0A286YER3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134882\n",
      "\n",
      "Gene ensembl id ENSG00000134882 changed to uniprot id : Q8NBM4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135069\n",
      "\n",
      "Gene ensembl id ENSG00000135069 changed to uniprot id : Q9Y617.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146733\n",
      "\n",
      "Gene ensembl id ENSG00000146733 changed to uniprot id : P78330.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000226278\n",
      "\n",
      "No uniprot id found for gene ENSG00000226278.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176974\n",
      "\n",
      "Gene ensembl id ENSG00000176974 changed to uniprot id : P34896.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100116\n",
      "\n",
      "Gene ensembl id ENSG00000100116 changed to uniprot id : O75600.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000179761\n",
      "\n",
      "Gene ensembl id ENSG00000179761 changed to uniprot id : Q9P0Z9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135094\n",
      "\n",
      "Gene ensembl id ENSG00000135094 changed to uniprot id : P20132.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167720\n",
      "\n",
      "Gene ensembl id ENSG00000167720 changed to uniprot id : Q9GZT4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139410\n",
      "\n",
      "Gene ensembl id ENSG00000139410 changed to uniprot id : Q96GA7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124713\n",
      "\n",
      "Gene ensembl id ENSG00000124713 changed to uniprot id : V9HW60.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000023330\n",
      "\n",
      "Gene ensembl id ENSG00000023330 changed to uniprot id : P13196.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178773\n",
      "\n",
      "Gene ensembl id ENSG00000178773 changed to uniprot id : Q9UBL6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158578\n",
      "\n",
      "Gene ensembl id ENSG00000158578 changed to uniprot id : P22557.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172482\n",
      "\n",
      "Gene ensembl id ENSG00000172482 changed to uniprot id : P21549.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000016391\n",
      "\n",
      "Gene ensembl id ENSG00000016391 changed to uniprot id : Q8NE62.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132840\n",
      "\n",
      "Gene ensembl id ENSG00000132840 changed to uniprot id : Q9H2M3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145692\n",
      "\n",
      "Gene ensembl id ENSG00000145692 changed to uniprot id : Q93088.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132837\n",
      "\n",
      "Gene ensembl id ENSG00000132837 changed to uniprot id : Q9UI17.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000090857\n",
      "\n",
      "Gene ensembl id ENSG00000090857 changed to uniprot id : Q8NCN5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113492\n",
      "\n",
      "Gene ensembl id ENSG00000113492 changed to uniprot id : Q9BYV1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182199\n",
      "\n",
      "Gene ensembl id ENSG00000182199 changed to uniprot id : V9HW06.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136877\n",
      "\n",
      "Gene ensembl id ENSG00000136877 changed to uniprot id : Q5JU23.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137563\n",
      "\n",
      "Gene ensembl id ENSG00000137563 changed to uniprot id : Q92820.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140905\n",
      "\n",
      "Gene ensembl id ENSG00000140905 changed to uniprot id : P23434.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145020\n",
      "\n",
      "Gene ensembl id ENSG00000145020 changed to uniprot id : P48728.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178445\n",
      "\n",
      "Gene ensembl id ENSG00000178445 changed to uniprot id : P23378.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140287\n",
      "\n",
      "Gene ensembl id ENSG00000140287 changed to uniprot id : P19113.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132437\n",
      "\n",
      "Gene ensembl id ENSG00000132437 changed to uniprot id : P20711.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000150540\n",
      "\n",
      "Gene ensembl id ENSG00000150540 changed to uniprot id : P50135.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000084110\n",
      "\n",
      "Gene ensembl id ENSG00000084110 changed to uniprot id : Q4VB95.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160282\n",
      "\n",
      "Gene ensembl id ENSG00000160282 changed to uniprot id : A0A0G2JQF2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139344\n",
      "\n",
      "Gene ensembl id ENSG00000139344 changed to uniprot id : Q96NU7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159650\n",
      "\n",
      "Gene ensembl id ENSG00000159650 changed to uniprot id : Q96N76.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124523\n",
      "\n",
      "Gene ensembl id ENSG00000124523 changed to uniprot id : Q9NXA8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138744\n",
      "\n",
      "Gene ensembl id ENSG00000138744 changed to uniprot id : Q02083.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124596\n",
      "\n",
      "Gene ensembl id ENSG00000124596 changed to uniprot id : Q9Y530.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000096717\n",
      "\n",
      "Gene ensembl id ENSG00000096717 changed to uniprot id : Q96EB6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133315\n",
      "\n",
      "Gene ensembl id ENSG00000133315 changed to uniprot id : Q9BQ69.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000068903\n",
      "\n",
      "Gene ensembl id ENSG00000068903 changed to uniprot id : Q8IXJ6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156795\n",
      "\n",
      "Gene ensembl id ENSG00000156795 changed to uniprot id : Q96HA8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142082\n",
      "\n",
      "Gene ensembl id ENSG00000142082 changed to uniprot id : Q9NTG7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117682\n",
      "\n",
      "Gene ensembl id ENSG00000117682 changed to uniprot id : Q86SQ9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000006695\n",
      "\n",
      "Gene ensembl id ENSG00000006695 changed to uniprot id : Q12887.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183665\n",
      "\n",
      "Gene ensembl id ENSG00000183665 changed to uniprot id : Q53H54.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000012660\n",
      "\n",
      "Gene ensembl id ENSG00000012660 changed to uniprot id : A0A024RD35.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164181\n",
      "\n",
      "Gene ensembl id ENSG00000164181 changed to uniprot id : D6RBM2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119915\n",
      "\n",
      "Gene ensembl id ENSG00000119915 changed to uniprot id : Q9HB03.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000066322\n",
      "\n",
      "Gene ensembl id ENSG00000066322 changed to uniprot id : Q9BW60.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197977\n",
      "\n",
      "Gene ensembl id ENSG00000197977 changed to uniprot id : Q9NXB9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170522\n",
      "\n",
      "Gene ensembl id ENSG00000170522 changed to uniprot id : Q9H5J4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000118402\n",
      "\n",
      "Gene ensembl id ENSG00000118402 changed to uniprot id : Q9GZR5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105953\n",
      "\n",
      "Gene ensembl id ENSG00000105953 changed to uniprot id : Q02218.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197444\n",
      "\n",
      "Gene ensembl id ENSG00000197444 changed to uniprot id : Q9ULD0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119689\n",
      "\n",
      "Gene ensembl id ENSG00000119689 changed to uniprot id : P36957.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000181192\n",
      "\n",
      "Gene ensembl id ENSG00000181192 changed to uniprot id : Q96HY7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000008311\n",
      "\n",
      "Gene ensembl id ENSG00000008311 changed to uniprot id : Q9UDR5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143653\n",
      "\n",
      "Gene ensembl id ENSG00000143653 changed to uniprot id : Q8NBX0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152455\n",
      "\n",
      "Gene ensembl id ENSG00000152455 changed to uniprot id : Q9H5I1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109685\n",
      "\n",
      "Gene ensembl id ENSG00000109685 changed to uniprot id : O96028.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000005483\n",
      "\n",
      "Gene ensembl id ENSG00000005483 changed to uniprot id : Q8IZD2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110066\n",
      "\n",
      "Gene ensembl id ENSG00000110066 changed to uniprot id : Q4FZB7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099381\n",
      "\n",
      "Gene ensembl id ENSG00000099381 changed to uniprot id : O15047.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105663\n",
      "\n",
      "No uniprot id found for gene ENSG00000105663.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139718\n",
      "\n",
      "Gene ensembl id ENSG00000139718 changed to uniprot id : Q9UPS6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185420\n",
      "\n",
      "Gene ensembl id ENSG00000185420 changed to uniprot id : Q9H7B4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000118058\n",
      "\n",
      "Gene ensembl id ENSG00000118058 changed to uniprot id : H0YEU4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136169\n",
      "\n",
      "Gene ensembl id ENSG00000136169 changed to uniprot id : Q96T68.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104885\n",
      "\n",
      "Gene ensembl id ENSG00000104885 changed to uniprot id : Q8TEK3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147548\n",
      "\n",
      "Gene ensembl id ENSG00000147548 changed to uniprot id : Q9BZ95.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114902\n",
      "\n",
      "Gene ensembl id ENSG00000114902 changed to uniprot id : Q9Y6A9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129128\n",
      "\n",
      "Gene ensembl id ENSG00000129128 changed to uniprot id : P61009.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000118363\n",
      "\n",
      "Gene ensembl id ENSG00000118363 changed to uniprot id : Q15005.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185973\n",
      "\n",
      "Gene ensembl id ENSG00000185973 changed to uniprot id : Q9NVH6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129151\n",
      "\n",
      "Gene ensembl id ENSG00000129151 changed to uniprot id : O75936.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167123\n",
      "\n",
      "Gene ensembl id ENSG00000167123 changed to uniprot id : Q5T4B2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106397\n",
      "\n",
      "Gene ensembl id ENSG00000106397 changed to uniprot id : O60568.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152952\n",
      "\n",
      "Gene ensembl id ENSG00000152952 changed to uniprot id : O00469.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198756\n",
      "\n",
      "Gene ensembl id ENSG00000198756 changed to uniprot id : Q8IYK4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108799\n",
      "\n",
      "Gene ensembl id ENSG00000108799 changed to uniprot id : Q92800.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000127884\n",
      "\n",
      "Gene ensembl id ENSG00000127884 changed to uniprot id : P30084.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000084754\n",
      "\n",
      "Gene ensembl id ENSG00000084754 changed to uniprot id : P40939.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000072506\n",
      "\n",
      "Gene ensembl id ENSG00000072506 changed to uniprot id : A0A0S2Z410.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138796\n",
      "\n",
      "Gene ensembl id ENSG00000138796 changed to uniprot id : Q16836.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000075239\n",
      "\n",
      "Gene ensembl id ENSG00000075239 changed to uniprot id : P24752.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140505\n",
      "\n",
      "Gene ensembl id ENSG00000140505 changed to uniprot id : P05177.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140465\n",
      "\n",
      "Gene ensembl id ENSG00000140465 changed to uniprot id : P04798.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138061\n",
      "\n",
      "Gene ensembl id ENSG00000138061 changed to uniprot id : Q16678.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165841\n",
      "\n",
      "Gene ensembl id ENSG00000165841 changed to uniprot id : P33261.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117305\n",
      "\n",
      "Gene ensembl id ENSG00000117305 changed to uniprot id : P35914.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197165\n",
      "\n",
      "Gene ensembl id ENSG00000197165 changed to uniprot id : P50226.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196502\n",
      "\n",
      "Gene ensembl id ENSG00000196502 changed to uniprot id : P50225.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167315\n",
      "\n",
      "Gene ensembl id ENSG00000167315 changed to uniprot id : P42765.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138029\n",
      "\n",
      "Gene ensembl id ENSG00000138029 changed to uniprot id : P55084.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175198\n",
      "\n",
      "Gene ensembl id ENSG00000175198 changed to uniprot id : P05165.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114054\n",
      "\n",
      "Gene ensembl id ENSG00000114054 changed to uniprot id : P05166.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124370\n",
      "\n",
      "Gene ensembl id ENSG00000124370 changed to uniprot id : Q96PE7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146085\n",
      "\n",
      "Gene ensembl id ENSG00000146085 changed to uniprot id : P22033.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000060982\n",
      "\n",
      "Gene ensembl id ENSG00000060982 changed to uniprot id : P54687.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105552\n",
      "\n",
      "Gene ensembl id ENSG00000105552 changed to uniprot id : O15382.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106049\n",
      "\n",
      "Gene ensembl id ENSG00000106049 changed to uniprot id : P31937.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137992\n",
      "\n",
      "Gene ensembl id ENSG00000137992 changed to uniprot id : Q5VVL7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000248098\n",
      "\n",
      "Gene ensembl id ENSG00000248098 changed to uniprot id : P12694.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000083123\n",
      "\n",
      "Gene ensembl id ENSG00000083123 changed to uniprot id : P21953.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171759\n",
      "\n",
      "Gene ensembl id ENSG00000171759 changed to uniprot id : P00439.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131203\n",
      "\n",
      "Gene ensembl id ENSG00000131203 changed to uniprot id : P14902.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151790\n",
      "\n",
      "Gene ensembl id ENSG00000151790 changed to uniprot id : D6RA50.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183077\n",
      "\n",
      "Gene ensembl id ENSG00000183077 changed to uniprot id : Q63HM1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115919\n",
      "\n",
      "Gene ensembl id ENSG00000115919 changed to uniprot id : Q16719.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105607\n",
      "\n",
      "Gene ensembl id ENSG00000105607 changed to uniprot id : Q92947.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184154\n",
      "\n",
      "Gene ensembl id ENSG00000184154 changed to uniprot id : Q96E66.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000093010\n",
      "\n",
      "Gene ensembl id ENSG00000093010 changed to uniprot id : P21964.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158104\n",
      "\n",
      "Gene ensembl id ENSG00000158104 changed to uniprot id : A0A0B4J1R4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113924\n",
      "\n",
      "Gene ensembl id ENSG00000113924 changed to uniprot id : Q93099.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100577\n",
      "\n",
      "Gene ensembl id ENSG00000100577 changed to uniprot id : A0A0C4DFM0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103876\n",
      "\n",
      "Gene ensembl id ENSG00000103876 changed to uniprot id : P16930.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000241644\n",
      "\n",
      "Gene ensembl id ENSG00000241644 changed to uniprot id : O95050.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109181\n",
      "\n",
      "Gene ensembl id ENSG00000109181 changed to uniprot id : P36537.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135220\n",
      "\n",
      "Gene ensembl id ENSG00000135220 changed to uniprot id : Q6UWM9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171234\n",
      "\n",
      "Gene ensembl id ENSG00000171234 changed to uniprot id : P16662.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000242366\n",
      "\n",
      "Gene ensembl id ENSG00000242366 changed to uniprot id : Q9HAW9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213759\n",
      "\n",
      "Gene ensembl id ENSG00000213759 changed to uniprot id : O75310.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000241635\n",
      "\n",
      "Gene ensembl id ENSG00000241635 changed to uniprot id : P22309.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000244474\n",
      "\n",
      "Gene ensembl id ENSG00000244474 changed to uniprot id : P22310.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197888\n",
      "\n",
      "Gene ensembl id ENSG00000197888 changed to uniprot id : O75795.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000242515\n",
      "\n",
      "Gene ensembl id ENSG00000242515 changed to uniprot id : Q5DT02.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167165\n",
      "\n",
      "Gene ensembl id ENSG00000167165 changed to uniprot id : P19224.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000240224\n",
      "\n",
      "No uniprot id found for gene ENSG00000240224.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135226\n",
      "\n",
      "Gene ensembl id ENSG00000135226 changed to uniprot id : Q9BY64.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156096\n",
      "\n",
      "Gene ensembl id ENSG00000156096 changed to uniprot id : P06133.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000243135\n",
      "\n",
      "No uniprot id found for gene ENSG00000243135.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173610\n",
      "\n",
      "Gene ensembl id ENSG00000173610 changed to uniprot id : P0DTE4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196620\n",
      "\n",
      "Gene ensembl id ENSG00000196620 changed to uniprot id : P54855.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115705\n",
      "\n",
      "Gene ensembl id ENSG00000115705 changed to uniprot id : P07202.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000005381\n",
      "\n",
      "Gene ensembl id ENSG00000005381 changed to uniprot id : P05164.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000008394\n",
      "\n",
      "Gene ensembl id ENSG00000008394 changed to uniprot id : P10620.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197448\n",
      "\n",
      "Gene ensembl id ENSG00000197448 changed to uniprot id : Q9Y2Q3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182793\n",
      "\n",
      "Gene ensembl id ENSG00000182793 changed to uniprot id : Q7RTV2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134201\n",
      "\n",
      "Gene ensembl id ENSG00000134201 changed to uniprot id : P46439.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000244067\n",
      "\n",
      "Gene ensembl id ENSG00000244067 changed to uniprot id : P09210.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000243955\n",
      "\n",
      "Gene ensembl id ENSG00000243955 changed to uniprot id : P08263.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148834\n",
      "\n",
      "Gene ensembl id ENSG00000148834 changed to uniprot id : P78417.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000084207\n",
      "\n",
      "Gene ensembl id ENSG00000084207 changed to uniprot id : P09211.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000085871\n",
      "\n",
      "Gene ensembl id ENSG00000085871 changed to uniprot id : Q99735.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168765\n",
      "\n",
      "Gene ensembl id ENSG00000168765 changed to uniprot id : Q03013.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000065621\n",
      "\n",
      "Gene ensembl id ENSG00000065621 changed to uniprot id : Q9H4Y5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213366\n",
      "\n",
      "Gene ensembl id ENSG00000213366 changed to uniprot id : P28161.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163106\n",
      "\n",
      "Gene ensembl id ENSG00000163106 changed to uniprot id : O60760.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174156\n",
      "\n",
      "Gene ensembl id ENSG00000174156 changed to uniprot id : Q16772.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134184\n",
      "\n",
      "Gene ensembl id ENSG00000134184 changed to uniprot id : P09488.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000211448\n",
      "\n",
      "Gene ensembl id ENSG00000211448 changed to uniprot id : A0A024R6J8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000211452\n",
      "\n",
      "Gene ensembl id ENSG00000211452 changed to uniprot id : P49895.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185352\n",
      "\n",
      "Gene ensembl id ENSG00000185352 changed to uniprot id : Q8IZP7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198203\n",
      "\n",
      "Gene ensembl id ENSG00000198203 changed to uniprot id : O00338.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175040\n",
      "\n",
      "Gene ensembl id ENSG00000175040 changed to uniprot id : Q9Y4C5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000153936\n",
      "\n",
      "Gene ensembl id ENSG00000153936 changed to uniprot id : Q7LGA3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164100\n",
      "\n",
      "Gene ensembl id ENSG00000164100 changed to uniprot id : O95803.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197093\n",
      "\n",
      "Gene ensembl id ENSG00000197093 changed to uniprot id : C9IZ77.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166507\n",
      "\n",
      "Gene ensembl id ENSG00000166507 changed to uniprot id : P52849.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198075\n",
      "\n",
      "Gene ensembl id ENSG00000198075 changed to uniprot id : O75897.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140835\n",
      "\n",
      "Gene ensembl id ENSG00000140835 changed to uniprot id : Q8NCG5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173597\n",
      "\n",
      "Gene ensembl id ENSG00000173597 changed to uniprot id : O43704.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070614\n",
      "\n",
      "Gene ensembl id ENSG00000070614 changed to uniprot id : P52848.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147119\n",
      "\n",
      "Gene ensembl id ENSG00000147119 changed to uniprot id : Q9NS84.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213599\n",
      "\n",
      "No uniprot id found for gene ENSG00000213599.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154080\n",
      "\n",
      "Gene ensembl id ENSG00000154080 changed to uniprot id : Q7L1S5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115526\n",
      "\n",
      "Gene ensembl id ENSG00000115526 changed to uniprot id : O43529.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111962\n",
      "\n",
      "Gene ensembl id ENSG00000111962 changed to uniprot id : Q9Y2C2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175229\n",
      "\n",
      "Gene ensembl id ENSG00000175229 changed to uniprot id : Q96A11.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101846\n",
      "\n",
      "Gene ensembl id ENSG00000101846 changed to uniprot id : A0A590UJL0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165591\n",
      "\n",
      "Gene ensembl id ENSG00000165591 changed to uniprot id : B2C6G4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000038274\n",
      "\n",
      "Gene ensembl id ENSG00000038274 changed to uniprot id : Q9NZL9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151224\n",
      "\n",
      "Gene ensembl id ENSG00000151224 changed to uniprot id : Q00266.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168906\n",
      "\n",
      "Gene ensembl id ENSG00000168906 changed to uniprot id : P31153.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158467\n",
      "\n",
      "Gene ensembl id ENSG00000158467 changed to uniprot id : Q96HN2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101444\n",
      "\n",
      "Gene ensembl id ENSG00000101444 changed to uniprot id : P23526.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168710\n",
      "\n",
      "Gene ensembl id ENSG00000168710 changed to uniprot id : O43865.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160200\n",
      "\n",
      "Gene ensembl id ENSG00000160200 changed to uniprot id : P35520.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116761\n",
      "\n",
      "Gene ensembl id ENSG00000116761 changed to uniprot id : P32929.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116984\n",
      "\n",
      "Gene ensembl id ENSG00000116984 changed to uniprot id : Q99707.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128309\n",
      "\n",
      "Gene ensembl id ENSG00000128309 changed to uniprot id : P25325.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128311\n",
      "\n",
      "Gene ensembl id ENSG00000128311 changed to uniprot id : Q16762.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000181652\n",
      "\n",
      "Gene ensembl id ENSG00000181652 changed to uniprot id : Q674R7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139531\n",
      "\n",
      "Gene ensembl id ENSG00000139531 changed to uniprot id : P51687.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000037757\n",
      "\n",
      "Gene ensembl id ENSG00000037757 changed to uniprot id : Q9BV20.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149089\n",
      "\n",
      "Gene ensembl id ENSG00000149089 changed to uniprot id : Q96GX9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145293\n",
      "\n",
      "Gene ensembl id ENSG00000145293 changed to uniprot id : Q9UHY7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182551\n",
      "\n",
      "Gene ensembl id ENSG00000182551 changed to uniprot id : Q9BV57.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000014641\n",
      "\n",
      "Gene ensembl id ENSG00000014641 changed to uniprot id : P40925.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146701\n",
      "\n",
      "Gene ensembl id ENSG00000146701 changed to uniprot id : P40926.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119772\n",
      "\n",
      "Gene ensembl id ENSG00000119772 changed to uniprot id : Q9Y6K1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000107614\n",
      "\n",
      "Gene ensembl id ENSG00000107614 changed to uniprot id : Q7Z3E4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178814\n",
      "\n",
      "Gene ensembl id ENSG00000178814 changed to uniprot id : O14841.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139684\n",
      "\n",
      "Gene ensembl id ENSG00000139684 changed to uniprot id : A0A140VJJ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198704\n",
      "\n",
      "Gene ensembl id ENSG00000198704 changed to uniprot id : P59796.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167468\n",
      "\n",
      "Gene ensembl id ENSG00000167468 changed to uniprot id : P36969.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167815\n",
      "\n",
      "Gene ensembl id ENSG00000167815 changed to uniprot id : A6NIW5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000233276\n",
      "\n",
      "Gene ensembl id ENSG00000233276 changed to uniprot id : P07203.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165672\n",
      "\n",
      "Gene ensembl id ENSG00000165672 changed to uniprot id : P30048.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000211445\n",
      "\n",
      "Gene ensembl id ENSG00000211445 changed to uniprot id : P22352.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000224586\n",
      "\n",
      "Gene ensembl id ENSG00000224586 changed to uniprot id : O75715.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176153\n",
      "\n",
      "Gene ensembl id ENSG00000176153 changed to uniprot id : P18283.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116157\n",
      "\n",
      "Gene ensembl id ENSG00000116157 changed to uniprot id : Q96SL4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117450\n",
      "\n",
      "Gene ensembl id ENSG00000117450 changed to uniprot id : Q06830.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167741\n",
      "\n",
      "Gene ensembl id ENSG00000167741 changed to uniprot id : B4DPG6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100031\n",
      "\n",
      "Gene ensembl id ENSG00000100031 changed to uniprot id : P19440.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183038\n",
      "\n",
      "No uniprot id found for gene ENSG00000183038.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131067\n",
      "\n",
      "Gene ensembl id ENSG00000131067 changed to uniprot id : Q9UJ14.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099998\n",
      "\n",
      "Gene ensembl id ENSG00000099998 changed to uniprot id : P36269.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100121\n",
      "\n",
      "Gene ensembl id ENSG00000100121 changed to uniprot id : Q14390.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133475\n",
      "\n",
      "No uniprot id found for gene ENSG00000133475.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000023909\n",
      "\n",
      "Gene ensembl id ENSG00000023909 changed to uniprot id : P48507.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000001084\n",
      "\n",
      "Gene ensembl id ENSG00000001084 changed to uniprot id : Q14TF0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100983\n",
      "\n",
      "Gene ensembl id ENSG00000100983 changed to uniprot id : A0A0S2Z4J7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166825\n",
      "\n",
      "Gene ensembl id ENSG00000166825 changed to uniprot id : P15144.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141279\n",
      "\n",
      "Gene ensembl id ENSG00000141279 changed to uniprot id : P55786.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000002549\n",
      "\n",
      "Gene ensembl id ENSG00000002549 changed to uniprot id : P28838.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000006625\n",
      "\n",
      "Gene ensembl id ENSG00000006625 changed to uniprot id : O75223.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197421\n",
      "\n",
      "No uniprot id found for gene ENSG00000197421.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132330\n",
      "\n",
      "Gene ensembl id ENSG00000132330 changed to uniprot id : Q96I15.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000094963\n",
      "\n",
      "Gene ensembl id ENSG00000094963 changed to uniprot id : Q99518.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000076258\n",
      "\n",
      "Gene ensembl id ENSG00000076258 changed to uniprot id : P31512.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000010932\n",
      "\n",
      "Gene ensembl id ENSG00000010932 changed to uniprot id : Q01740.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000007933\n",
      "\n",
      "Gene ensembl id ENSG00000007933 changed to uniprot id : P31513.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131781\n",
      "\n",
      "Gene ensembl id ENSG00000131781 changed to uniprot id : P49326.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138356\n",
      "\n",
      "Gene ensembl id ENSG00000138356 changed to uniprot id : Q06278.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128928\n",
      "\n",
      "Gene ensembl id ENSG00000128928 changed to uniprot id : P26440.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131844\n",
      "\n",
      "Gene ensembl id ENSG00000131844 changed to uniprot id : Q9HCC0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000078070\n",
      "\n",
      "Gene ensembl id ENSG00000078070 changed to uniprot id : Q96RQ3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000049860\n",
      "\n",
      "Gene ensembl id ENSG00000049860 changed to uniprot id : Q5URX0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213614\n",
      "\n",
      "Gene ensembl id ENSG00000213614 changed to uniprot id : H3BTD4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197355\n",
      "\n",
      "Gene ensembl id ENSG00000197355 changed to uniprot id : Q3KQV9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117143\n",
      "\n",
      "Gene ensembl id ENSG00000117143 changed to uniprot id : Q16222.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113552\n",
      "\n",
      "Gene ensembl id ENSG00000113552 changed to uniprot id : P46926.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163281\n",
      "\n",
      "Gene ensembl id ENSG00000163281 changed to uniprot id : Q8TDQ7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131459\n",
      "\n",
      "Gene ensembl id ENSG00000131459 changed to uniprot id : A0A0S2Z4X9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198380\n",
      "\n",
      "Gene ensembl id ENSG00000198380 changed to uniprot id : Q06210.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124357\n",
      "\n",
      "Gene ensembl id ENSG00000124357 changed to uniprot id : Q9UJ70.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159921\n",
      "\n",
      "Gene ensembl id ENSG00000159921 changed to uniprot id : Q9Y223.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102032\n",
      "\n",
      "Gene ensembl id ENSG00000102032 changed to uniprot id : P51606.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000095380\n",
      "\n",
      "Gene ensembl id ENSG00000095380 changed to uniprot id : Q9NR45.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170191\n",
      "\n",
      "Gene ensembl id ENSG00000170191 changed to uniprot id : Q8TBE9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111726\n",
      "\n",
      "Gene ensembl id ENSG00000111726 changed to uniprot id : Q8NFW8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100522\n",
      "\n",
      "Gene ensembl id ENSG00000100522 changed to uniprot id : Q96EK6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162066\n",
      "\n",
      "Gene ensembl id ENSG00000162066 changed to uniprot id : Q9Y303.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000013375\n",
      "\n",
      "Gene ensembl id ENSG00000013375 changed to uniprot id : O95394.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000088451\n",
      "\n",
      "Gene ensembl id ENSG00000088451 changed to uniprot id : O95455.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111261\n",
      "\n",
      "Gene ensembl id ENSG00000111261 changed to uniprot id : Q9H8J5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134216\n",
      "\n",
      "Gene ensembl id ENSG00000134216 changed to uniprot id : Q9BZP6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133063\n",
      "\n",
      "Gene ensembl id ENSG00000133063 changed to uniprot id : Q13231.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139131\n",
      "\n",
      "Gene ensembl id ENSG00000139131 changed to uniprot id : Q9Y2Z4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134684\n",
      "\n",
      "Gene ensembl id ENSG00000134684 changed to uniprot id : P54577.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124608\n",
      "\n",
      "Gene ensembl id ENSG00000124608 changed to uniprot id : Q5JTZ9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000090861\n",
      "\n",
      "Gene ensembl id ENSG00000090861 changed to uniprot id : P49588.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146282\n",
      "\n",
      "Gene ensembl id ENSG00000146282 changed to uniprot id : Q5T160.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137513\n",
      "\n",
      "Gene ensembl id ENSG00000137513 changed to uniprot id : Q96I59.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134440\n",
      "\n",
      "Gene ensembl id ENSG00000134440 changed to uniprot id : O43776.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115866\n",
      "\n",
      "Gene ensembl id ENSG00000115866 changed to uniprot id : P14868.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117593\n",
      "\n",
      "Gene ensembl id ENSG00000117593 changed to uniprot id : A0A3B3IT89.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134905\n",
      "\n",
      "Gene ensembl id ENSG00000134905 changed to uniprot id : Q9HA77.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172053\n",
      "\n",
      "Gene ensembl id ENSG00000172053 changed to uniprot id : P47897.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103356\n",
      "\n",
      "Gene ensembl id ENSG00000103356 changed to uniprot id : Q5JPH6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136628\n",
      "\n",
      "Gene ensembl id ENSG00000136628 changed to uniprot id : P07814.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106105\n",
      "\n",
      "Gene ensembl id ENSG00000106105 changed to uniprot id : A0A6Q8PGW4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170445\n",
      "\n",
      "Gene ensembl id ENSG00000170445 changed to uniprot id : B4E1C5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112855\n",
      "\n",
      "Gene ensembl id ENSG00000112855 changed to uniprot id : B4DQ67.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196305\n",
      "\n",
      "Gene ensembl id ENSG00000196305 changed to uniprot id : P41252.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000067704\n",
      "\n",
      "Gene ensembl id ENSG00000067704 changed to uniprot id : Q9NSE4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133706\n",
      "\n",
      "Gene ensembl id ENSG00000133706 changed to uniprot id : Q9P2J5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000011376\n",
      "\n",
      "Gene ensembl id ENSG00000011376 changed to uniprot id : Q15031.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000065427\n",
      "\n",
      "Gene ensembl id ENSG00000065427 changed to uniprot id : Q15046.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166986\n",
      "\n",
      "Gene ensembl id ENSG00000166986 changed to uniprot id : P56192.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000247626\n",
      "\n",
      "Gene ensembl id ENSG00000247626 changed to uniprot id : Q96GW9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145982\n",
      "\n",
      "Gene ensembl id ENSG00000145982 changed to uniprot id : O95363.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162396\n",
      "\n",
      "Gene ensembl id ENSG00000162396 changed to uniprot id : Q7L3T8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104835\n",
      "\n",
      "Gene ensembl id ENSG00000104835 changed to uniprot id : Q9NP81.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143374\n",
      "\n",
      "Gene ensembl id ENSG00000143374 changed to uniprot id : Q9BW92.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113407\n",
      "\n",
      "Gene ensembl id ENSG00000113407 changed to uniprot id : P26639.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140105\n",
      "\n",
      "Gene ensembl id ENSG00000140105 changed to uniprot id : A0A024R6K8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204394\n",
      "\n",
      "Gene ensembl id ENSG00000204394 changed to uniprot id : P26640.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137411\n",
      "\n",
      "Gene ensembl id ENSG00000137411 changed to uniprot id : A2ABL6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000008513\n",
      "\n",
      "Gene ensembl id ENSG00000008513 changed to uniprot id : Q11201.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157350\n",
      "\n",
      "Gene ensembl id ENSG00000157350 changed to uniprot id : Q16842.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198488\n",
      "\n",
      "Gene ensembl id ENSG00000198488 changed to uniprot id : Q6ZMB0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140297\n",
      "\n",
      "Gene ensembl id ENSG00000140297 changed to uniprot id : O95395.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000187210\n",
      "\n",
      "Gene ensembl id ENSG00000187210 changed to uniprot id : Q02742.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176928\n",
      "\n",
      "Gene ensembl id ENSG00000176928 changed to uniprot id : Q9P109.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158470\n",
      "\n",
      "Gene ensembl id ENSG00000158470 changed to uniprot id : O43286.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178234\n",
      "\n",
      "Gene ensembl id ENSG00000178234 changed to uniprot id : Q8NCW6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131386\n",
      "\n",
      "Gene ensembl id ENSG00000131386 changed to uniprot id : Q8N3T1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110328\n",
      "\n",
      "Gene ensembl id ENSG00000110328 changed to uniprot id : Q58A54.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130035\n",
      "\n",
      "Gene ensembl id ENSG00000130035 changed to uniprot id : Q9NY28.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141429\n",
      "\n",
      "Gene ensembl id ENSG00000141429 changed to uniprot id : Q10472.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115339\n",
      "\n",
      "Gene ensembl id ENSG00000115339 changed to uniprot id : Q14435.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143641\n",
      "\n",
      "Gene ensembl id ENSG00000143641 changed to uniprot id : Q10471.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136542\n",
      "\n",
      "Gene ensembl id ENSG00000136542 changed to uniprot id : Q7Z7M9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171155\n",
      "\n",
      "Gene ensembl id ENSG00000171155 changed to uniprot id : Q96EU7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106392\n",
      "\n",
      "Gene ensembl id ENSG00000106392 changed to uniprot id : Q9NS00.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160752\n",
      "\n",
      "Gene ensembl id ENSG00000160752 changed to uniprot id : P14324.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152904\n",
      "\n",
      "Gene ensembl id ENSG00000152904 changed to uniprot id : A0A024R3R2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167130\n",
      "\n",
      "Gene ensembl id ENSG00000167130 changed to uniprot id : Q86YN1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139133\n",
      "\n",
      "Gene ensembl id ENSG00000139133 changed to uniprot id : Q5BKT4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175283\n",
      "\n",
      "Gene ensembl id ENSG00000175283 changed to uniprot id : Q9UPQ8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172269\n",
      "\n",
      "Gene ensembl id ENSG00000172269 changed to uniprot id : Q9H3H5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172339\n",
      "\n",
      "Gene ensembl id ENSG00000172339 changed to uniprot id : Q96F25.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000033011\n",
      "\n",
      "Gene ensembl id ENSG00000033011 changed to uniprot id : Q9BT22.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119523\n",
      "\n",
      "Gene ensembl id ENSG00000119523 changed to uniprot id : Q9H553.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000253710\n",
      "\n",
      "Gene ensembl id ENSG00000253710 changed to uniprot id : Q2TAA5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000179085\n",
      "\n",
      "Gene ensembl id ENSG00000179085 changed to uniprot id : Q9P2X0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000000419\n",
      "\n",
      "Gene ensembl id ENSG00000000419 changed to uniprot id : O60762.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000214160\n",
      "\n",
      "Gene ensembl id ENSG00000214160 changed to uniprot id : Q92685.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000086848\n",
      "\n",
      "Gene ensembl id ENSG00000086848 changed to uniprot id : Q9H6U8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182858\n",
      "\n",
      "Gene ensembl id ENSG00000182858 changed to uniprot id : Q9BV10.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000088035\n",
      "\n",
      "Gene ensembl id ENSG00000088035 changed to uniprot id : Q9Y672.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159063\n",
      "\n",
      "Gene ensembl id ENSG00000159063 changed to uniprot id : Q9BVK2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175548\n",
      "\n",
      "Gene ensembl id ENSG00000175548 changed to uniprot id : Q5I7T1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000244038\n",
      "\n",
      "Gene ensembl id ENSG00000244038 changed to uniprot id : P39656.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000118705\n",
      "\n",
      "Gene ensembl id ENSG00000118705 changed to uniprot id : P04844.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163902\n",
      "\n",
      "Gene ensembl id ENSG00000163902 changed to uniprot id : P04843.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134910\n",
      "\n",
      "Gene ensembl id ENSG00000134910 changed to uniprot id : P46977.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129562\n",
      "\n",
      "Gene ensembl id ENSG00000129562 changed to uniprot id : P61803.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163527\n",
      "\n",
      "Gene ensembl id ENSG00000163527 changed to uniprot id : Q8TCJ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104723\n",
      "\n",
      "Gene ensembl id ENSG00000104723 changed to uniprot id : D6RDV0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115275\n",
      "\n",
      "Gene ensembl id ENSG00000115275 changed to uniprot id : Q13724.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000089597\n",
      "\n",
      "Gene ensembl id ENSG00000089597 changed to uniprot id : Q14697.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117643\n",
      "\n",
      "Gene ensembl id ENSG00000117643 changed to uniprot id : Q9NR34.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111885\n",
      "\n",
      "Gene ensembl id ENSG00000111885 changed to uniprot id : P33908.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198162\n",
      "\n",
      "Gene ensembl id ENSG00000198162 changed to uniprot id : O60476.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000177239\n",
      "\n",
      "Gene ensembl id ENSG00000177239 changed to uniprot id : Q9UKM7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131446\n",
      "\n",
      "Gene ensembl id ENSG00000131446 changed to uniprot id : P26572.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112893\n",
      "\n",
      "Gene ensembl id ENSG00000112893 changed to uniprot id : Q16706.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196547\n",
      "\n",
      "Gene ensembl id ENSG00000196547 changed to uniprot id : A0A0C4DGL1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168282\n",
      "\n",
      "Gene ensembl id ENSG00000168282 changed to uniprot id : Q10469.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000161013\n",
      "\n",
      "Gene ensembl id ENSG00000161013 changed to uniprot id : Q9UQ53.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182050\n",
      "\n",
      "Gene ensembl id ENSG00000182050 changed to uniprot id : Q9UBM8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000071073\n",
      "\n",
      "Gene ensembl id ENSG00000071073 changed to uniprot id : Q9UM21.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152127\n",
      "\n",
      "Gene ensembl id ENSG00000152127 changed to uniprot id : Q09328.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000033170\n",
      "\n",
      "Gene ensembl id ENSG00000033170 changed to uniprot id : Q9BYC5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000144057\n",
      "\n",
      "Gene ensembl id ENSG00000144057 changed to uniprot id : Q96JF0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000073849\n",
      "\n",
      "Gene ensembl id ENSG00000073849 changed to uniprot id : P15907.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140400\n",
      "\n",
      "Gene ensembl id ENSG00000140400 changed to uniprot id : Q9NTJ4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104774\n",
      "\n",
      "Gene ensembl id ENSG00000104774 changed to uniprot id : O00754.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109323\n",
      "\n",
      "Gene ensembl id ENSG00000109323 changed to uniprot id : O00462.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000038002\n",
      "\n",
      "Gene ensembl id ENSG00000038002 changed to uniprot id : P20933.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167280\n",
      "\n",
      "Gene ensembl id ENSG00000167280 changed to uniprot id : Q8NFI3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000064601\n",
      "\n",
      "Gene ensembl id ENSG00000064601 changed to uniprot id : X6R8A1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204386\n",
      "\n",
      "Gene ensembl id ENSG00000204386 changed to uniprot id : Q99519.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115488\n",
      "\n",
      "Gene ensembl id ENSG00000115488 changed to uniprot id : Q9Y3R4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162139\n",
      "\n",
      "Gene ensembl id ENSG00000162139 changed to uniprot id : E9PMZ3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204099\n",
      "\n",
      "Gene ensembl id ENSG00000204099 changed to uniprot id : Q8WWR8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141012\n",
      "\n",
      "Gene ensembl id ENSG00000141012 changed to uniprot id : P34059.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000001036\n",
      "\n",
      "Gene ensembl id ENSG00000001036 changed to uniprot id : Q9BTY2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000179163\n",
      "\n",
      "Gene ensembl id ENSG00000179163 changed to uniprot id : P04066.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113240\n",
      "\n",
      "Gene ensembl id ENSG00000113240 changed to uniprot id : Q9HAZ1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148053\n",
      "\n",
      "Gene ensembl id ENSG00000148053 changed to uniprot id : Q16620.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133216\n",
      "\n",
      "Gene ensembl id ENSG00000133216 changed to uniprot id : Q6NVW1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163785\n",
      "\n",
      "Gene ensembl id ENSG00000163785 changed to uniprot id : P34925.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000013441\n",
      "\n",
      "Gene ensembl id ENSG00000013441 changed to uniprot id : P49759.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103653\n",
      "\n",
      "Gene ensembl id ENSG00000103653 changed to uniprot id : P41240.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000076984\n",
      "\n",
      "Gene ensembl id ENSG00000076984 changed to uniprot id : O14733.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102010\n",
      "\n",
      "Gene ensembl id ENSG00000102010 changed to uniprot id : P51813.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105397\n",
      "\n",
      "Gene ensembl id ENSG00000105397 changed to uniprot id : P29597.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000010219\n",
      "\n",
      "Gene ensembl id ENSG00000010219 changed to uniprot id : Q9NR20.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196411\n",
      "\n",
      "Gene ensembl id ENSG00000196411 changed to uniprot id : P54760.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204580\n",
      "\n",
      "Gene ensembl id ENSG00000204580 changed to uniprot id : E7ERN0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198400\n",
      "\n",
      "Gene ensembl id ENSG00000198400 changed to uniprot id : P04629.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000179335\n",
      "\n",
      "Gene ensembl id ENSG00000179335 changed to uniprot id : H3BRE4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171105\n",
      "\n",
      "Gene ensembl id ENSG00000171105 changed to uniprot id : P06213.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173517\n",
      "\n",
      "Gene ensembl id ENSG00000173517 changed to uniprot id : Q9H792.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070886\n",
      "\n",
      "Gene ensembl id ENSG00000070886 changed to uniprot id : P29322.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122025\n",
      "\n",
      "Gene ensembl id ENSG00000122025 changed to uniprot id : P36888.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000097007\n",
      "\n",
      "Gene ensembl id ENSG00000097007 changed to uniprot id : P00519.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000068078\n",
      "\n",
      "Gene ensembl id ENSG00000068078 changed to uniprot id : I6LM06.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151422\n",
      "\n",
      "Gene ensembl id ENSG00000151422 changed to uniprot id : P16591.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174292\n",
      "\n",
      "Gene ensembl id ENSG00000174292 changed to uniprot id : Q13470.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176105\n",
      "\n",
      "Gene ensembl id ENSG00000176105 changed to uniprot id : P07947.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165731\n",
      "\n",
      "Gene ensembl id ENSG00000165731 changed to uniprot id : A0A024R7T2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111816\n",
      "\n",
      "Gene ensembl id ENSG00000111816 changed to uniprot id : P42685.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178568\n",
      "\n",
      "Gene ensembl id ENSG00000178568 changed to uniprot id : Q15303.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000027644\n",
      "\n",
      "Gene ensembl id ENSG00000027644 changed to uniprot id : P14616.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143322\n",
      "\n",
      "Gene ensembl id ENSG00000143322 changed to uniprot id : P42684.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000065361\n",
      "\n",
      "Gene ensembl id ENSG00000065361 changed to uniprot id : P21860.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101213\n",
      "\n",
      "Gene ensembl id ENSG00000101213 changed to uniprot id : Q13882.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182578\n",
      "\n",
      "Gene ensembl id ENSG00000182578 changed to uniprot id : P07333.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146648\n",
      "\n",
      "Gene ensembl id ENSG00000146648 changed to uniprot id : P00533.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157404\n",
      "\n",
      "Gene ensembl id ENSG00000157404 changed to uniprot id : P10721.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167601\n",
      "\n",
      "Gene ensembl id ENSG00000167601 changed to uniprot id : P30530.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000126934\n",
      "\n",
      "Gene ensembl id ENSG00000126934 changed to uniprot id : P36507.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146904\n",
      "\n",
      "Gene ensembl id ENSG00000146904 changed to uniprot id : P21709.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135333\n",
      "\n",
      "Gene ensembl id ENSG00000135333 changed to uniprot id : A0A7P0T9L5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000080224\n",
      "\n",
      "Gene ensembl id ENSG00000080224 changed to uniprot id : B4DXM2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000127334\n",
      "\n",
      "Gene ensembl id ENSG00000127334 changed to uniprot id : Q92630.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112742\n",
      "\n",
      "Gene ensembl id ENSG00000112742 changed to uniprot id : P33981.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169032\n",
      "\n",
      "Gene ensembl id ENSG00000169032 changed to uniprot id : A0A8I5QJL9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182580\n",
      "\n",
      "Gene ensembl id ENSG00000182580 changed to uniprot id : P54753.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000044524\n",
      "\n",
      "Gene ensembl id ENSG00000044524 changed to uniprot id : P29320.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113721\n",
      "\n",
      "Gene ensembl id ENSG00000113721 changed to uniprot id : P09619.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108984\n",
      "\n",
      "Gene ensembl id ENSG00000108984 changed to uniprot id : P52564.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000007264\n",
      "\n",
      "Gene ensembl id ENSG00000007264 changed to uniprot id : P42679.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116106\n",
      "\n",
      "Gene ensembl id ENSG00000116106 changed to uniprot id : A1MA61.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167778\n",
      "\n",
      "Gene ensembl id ENSG00000167778 changed to uniprot id : Q8NCJ5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137764\n",
      "\n",
      "Gene ensembl id ENSG00000137764 changed to uniprot id : Q13163.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000153208\n",
      "\n",
      "Gene ensembl id ENSG00000153208 changed to uniprot id : Q12866.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162434\n",
      "\n",
      "Gene ensembl id ENSG00000162434 changed to uniprot id : P23458.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164078\n",
      "\n",
      "Gene ensembl id ENSG00000164078 changed to uniprot id : Q04912.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185483\n",
      "\n",
      "Gene ensembl id ENSG00000185483 changed to uniprot id : Q01973.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105976\n",
      "\n",
      "Gene ensembl id ENSG00000105976 changed to uniprot id : A0A024R728.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000000938\n",
      "\n",
      "Gene ensembl id ENSG00000000938 changed to uniprot id : P09769.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141736\n",
      "\n",
      "Gene ensembl id ENSG00000141736 changed to uniprot id : P04626.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182319\n",
      "\n",
      "No uniprot id found for gene ENSG00000182319.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000066468\n",
      "\n",
      "Gene ensembl id ENSG00000066468 changed to uniprot id : P21802.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135605\n",
      "\n",
      "Gene ensembl id ENSG00000135605 changed to uniprot id : P42680.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000061938\n",
      "\n",
      "Gene ensembl id ENSG00000061938 changed to uniprot id : Q07912.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164715\n",
      "\n",
      "Gene ensembl id ENSG00000164715 changed to uniprot id : Q8IWU2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000030304\n",
      "\n",
      "Gene ensembl id ENSG00000030304 changed to uniprot id : O15146.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000077782\n",
      "\n",
      "Gene ensembl id ENSG00000077782 changed to uniprot id : P11362.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136573\n",
      "\n",
      "Gene ensembl id ENSG00000136573 changed to uniprot id : P51451.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142627\n",
      "\n",
      "Gene ensembl id ENSG00000142627 changed to uniprot id : P29317.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182511\n",
      "\n",
      "Gene ensembl id ENSG00000182511 changed to uniprot id : P07332.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000107140\n",
      "\n",
      "Gene ensembl id ENSG00000107140 changed to uniprot id : Q15569.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168078\n",
      "\n",
      "Gene ensembl id ENSG00000168078 changed to uniprot id : Q96KB5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140538\n",
      "\n",
      "Gene ensembl id ENSG00000140538 changed to uniprot id : X5D7M5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113263\n",
      "\n",
      "Gene ensembl id ENSG00000113263 changed to uniprot id : Q08881.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000010810\n",
      "\n",
      "Gene ensembl id ENSG00000010810 changed to uniprot id : P06241.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134853\n",
      "\n",
      "Gene ensembl id ENSG00000134853 changed to uniprot id : P16234.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160867\n",
      "\n",
      "Gene ensembl id ENSG00000160867 changed to uniprot id : P22455.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169398\n",
      "\n",
      "Gene ensembl id ENSG00000169398 changed to uniprot id : E5RH01.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140443\n",
      "\n",
      "Gene ensembl id ENSG00000140443 changed to uniprot id : P08069.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165025\n",
      "\n",
      "Gene ensembl id ENSG00000165025 changed to uniprot id : P43405.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000096968\n",
      "\n",
      "Gene ensembl id ENSG00000096968 changed to uniprot id : O60674.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000188716\n",
      "\n",
      "Gene ensembl id ENSG00000188716 changed to uniprot id : Q68J44.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070778\n",
      "\n",
      "Gene ensembl id ENSG00000070778 changed to uniprot id : Q16825.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158079\n",
      "\n",
      "Gene ensembl id ENSG00000158079 changed to uniprot id : A2A3K4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120129\n",
      "\n",
      "Gene ensembl id ENSG00000120129 changed to uniprot id : P28562.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143727\n",
      "\n",
      "Gene ensembl id ENSG00000143727 changed to uniprot id : P24666.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167065\n",
      "\n",
      "Gene ensembl id ENSG00000167065 changed to uniprot id : Q8NEJ0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110536\n",
      "\n",
      "Gene ensembl id ENSG00000110536 changed to uniprot id : Q8WUK0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152104\n",
      "\n",
      "Gene ensembl id ENSG00000152104 changed to uniprot id : Q15678.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000076201\n",
      "\n",
      "Gene ensembl id ENSG00000076201 changed to uniprot id : Q9H3S7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000161326\n",
      "\n",
      "No uniprot id found for gene ENSG00000161326.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111266\n",
      "\n",
      "Gene ensembl id ENSG00000111266 changed to uniprot id : F5H5X4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139318\n",
      "\n",
      "Gene ensembl id ENSG00000139318 changed to uniprot id : Q16828.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163629\n",
      "\n",
      "Gene ensembl id ENSG00000163629 changed to uniprot id : Q12923.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142949\n",
      "\n",
      "Gene ensembl id ENSG00000142949 changed to uniprot id : P10586.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196396\n",
      "\n",
      "Gene ensembl id ENSG00000196396 changed to uniprot id : P18031.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134242\n",
      "\n",
      "Gene ensembl id ENSG00000134242 changed to uniprot id : E9PMT0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070159\n",
      "\n",
      "Gene ensembl id ENSG00000070159 changed to uniprot id : P26045.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143507\n",
      "\n",
      "Gene ensembl id ENSG00000143507 changed to uniprot id : A0A0B4J2F5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170324\n",
      "\n",
      "Gene ensembl id ENSG00000170324 changed to uniprot id : Q68DX3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112319\n",
      "\n",
      "Gene ensembl id ENSG00000112319 changed to uniprot id : O95677.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112679\n",
      "\n",
      "Gene ensembl id ENSG00000112679 changed to uniprot id : S4R378.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000084112\n",
      "\n",
      "Gene ensembl id ENSG00000084112 changed to uniprot id : Q8WYL5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100330\n",
      "\n",
      "Gene ensembl id ENSG00000100330 changed to uniprot id : Q13615.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000060656\n",
      "\n",
      "Gene ensembl id ENSG00000060656 changed to uniprot id : Q92729.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000153233\n",
      "\n",
      "Gene ensembl id ENSG00000153233 changed to uniprot id : Q15256.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100526\n",
      "\n",
      "Gene ensembl id ENSG00000100526 changed to uniprot id : Q16667.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000079393\n",
      "\n",
      "Gene ensembl id ENSG00000079393 changed to uniprot id : Q6B8I1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000080031\n",
      "\n",
      "Gene ensembl id ENSG00000080031 changed to uniprot id : Q9HD43.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000127329\n",
      "\n",
      "Gene ensembl id ENSG00000127329 changed to uniprot id : P23467.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000153707\n",
      "\n",
      "Gene ensembl id ENSG00000153707 changed to uniprot id : P23468.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132670\n",
      "\n",
      "Gene ensembl id ENSG00000132670 changed to uniprot id : P18433.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130829\n",
      "\n",
      "Gene ensembl id ENSG00000130829 changed to uniprot id : Q99956.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149177\n",
      "\n",
      "Gene ensembl id ENSG00000149177 changed to uniprot id : Q12913.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158716\n",
      "\n",
      "Gene ensembl id ENSG00000158716 changed to uniprot id : Q9BVJ7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164045\n",
      "\n",
      "Gene ensembl id ENSG00000164045 changed to uniprot id : P30304.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175354\n",
      "\n",
      "Gene ensembl id ENSG00000175354 changed to uniprot id : P17706.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000127947\n",
      "\n",
      "Gene ensembl id ENSG00000127947 changed to uniprot id : Q05209.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110786\n",
      "\n",
      "Gene ensembl id ENSG00000110786 changed to uniprot id : P54829.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000064655\n",
      "\n",
      "Gene ensembl id ENSG00000064655 changed to uniprot id : O00167.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173482\n",
      "\n",
      "Gene ensembl id ENSG00000173482 changed to uniprot id : P28827.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000072135\n",
      "\n",
      "Gene ensembl id ENSG00000072135 changed to uniprot id : Q99952.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162999\n",
      "\n",
      "Gene ensembl id ENSG00000162999 changed to uniprot id : Q8WTR2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141298\n",
      "\n",
      "Gene ensembl id ENSG00000141298 changed to uniprot id : Q76I76.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108861\n",
      "\n",
      "Gene ensembl id ENSG00000108861 changed to uniprot id : P51452.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000079335\n",
      "\n",
      "Gene ensembl id ENSG00000079335 changed to uniprot id : Q9UNH5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132334\n",
      "\n",
      "Gene ensembl id ENSG00000132334 changed to uniprot id : P23469.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169410\n",
      "\n",
      "Gene ensembl id ENSG00000169410 changed to uniprot id : P43378.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184007\n",
      "\n",
      "Gene ensembl id ENSG00000184007 changed to uniprot id : Q12974.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143387\n",
      "\n",
      "Gene ensembl id ENSG00000143387 changed to uniprot id : P43235.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109861\n",
      "\n",
      "Gene ensembl id ENSG00000109861 changed to uniprot id : P53634.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196188\n",
      "\n",
      "Gene ensembl id ENSG00000196188 changed to uniprot id : P14091.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000256043\n",
      "\n",
      "Gene ensembl id ENSG00000256043 changed to uniprot id : P43234.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164733\n",
      "\n",
      "Gene ensembl id ENSG00000164733 changed to uniprot id : E9PSG5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101160\n",
      "\n",
      "Gene ensembl id ENSG00000101160 changed to uniprot id : Q9UBR2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117984\n",
      "\n",
      "Gene ensembl id ENSG00000117984 changed to uniprot id : P07339.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103811\n",
      "\n",
      "Gene ensembl id ENSG00000103811 changed to uniprot id : A0A7I2YQE6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136943\n",
      "\n",
      "Gene ensembl id ENSG00000136943 changed to uniprot id : O60911.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166340\n",
      "\n",
      "Gene ensembl id ENSG00000166340 changed to uniprot id : O14773.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174080\n",
      "\n",
      "Gene ensembl id ENSG00000174080 changed to uniprot id : Q9UBX1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172543\n",
      "\n",
      "Gene ensembl id ENSG00000172543 changed to uniprot id : P56202.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135047\n",
      "\n",
      "Gene ensembl id ENSG00000135047 changed to uniprot id : A0A7I2YQA2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000206527\n",
      "\n",
      "Gene ensembl id ENSG00000206527 changed to uniprot id : Q6Y1H2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165996\n",
      "\n",
      "Gene ensembl id ENSG00000165996 changed to uniprot id : B0YJ81.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167434\n",
      "\n",
      "Gene ensembl id ENSG00000167434 changed to uniprot id : P22748.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000074410\n",
      "\n",
      "Gene ensembl id ENSG00000074410 changed to uniprot id : O43570.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159593\n",
      "\n",
      "Gene ensembl id ENSG00000159593 changed to uniprot id : Q13564.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000107159\n",
      "\n",
      "Gene ensembl id ENSG00000107159 changed to uniprot id : Q16790.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131686\n",
      "\n",
      "Gene ensembl id ENSG00000131686 changed to uniprot id : P23280.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164879\n",
      "\n",
      "Gene ensembl id ENSG00000164879 changed to uniprot id : P07451.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168748\n",
      "\n",
      "Gene ensembl id ENSG00000168748 changed to uniprot id : P43166.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000118298\n",
      "\n",
      "Gene ensembl id ENSG00000118298 changed to uniprot id : A8K3J4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185015\n",
      "\n",
      "Gene ensembl id ENSG00000185015 changed to uniprot id : Q8N1Q1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133742\n",
      "\n",
      "Gene ensembl id ENSG00000133742 changed to uniprot id : V9HWE3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178538\n",
      "\n",
      "Gene ensembl id ENSG00000178538 changed to uniprot id : P35219.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104267\n",
      "\n",
      "Gene ensembl id ENSG00000104267 changed to uniprot id : P00918.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162813\n",
      "\n",
      "Gene ensembl id ENSG00000162813 changed to uniprot id : O95861.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104331\n",
      "\n",
      "Gene ensembl id ENSG00000104331 changed to uniprot id : Q9NX62.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163541\n",
      "\n",
      "Gene ensembl id ENSG00000163541 changed to uniprot id : P53597.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136143\n",
      "\n",
      "Gene ensembl id ENSG00000136143 changed to uniprot id : Q9P2R7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172340\n",
      "\n",
      "Gene ensembl id ENSG00000172340 changed to uniprot id : Q96I99.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182054\n",
      "\n",
      "Gene ensembl id ENSG00000182054 changed to uniprot id : P48735.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138413\n",
      "\n",
      "Gene ensembl id ENSG00000138413 changed to uniprot id : O75874.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000083720\n",
      "\n",
      "Gene ensembl id ENSG00000083720 changed to uniprot id : P55809.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198754\n",
      "\n",
      "Gene ensembl id ENSG00000198754 changed to uniprot id : Q9BYC2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000067829\n",
      "\n",
      "Gene ensembl id ENSG00000067829 changed to uniprot id : G5E9Q7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101365\n",
      "\n",
      "Gene ensembl id ENSG00000101365 changed to uniprot id : A0A087WZN1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166411\n",
      "\n",
      "Gene ensembl id ENSG00000166411 changed to uniprot id : Q68D72.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000062485\n",
      "\n",
      "Gene ensembl id ENSG00000062485 changed to uniprot id : A0A024RB75.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000107104\n",
      "\n",
      "Gene ensembl id ENSG00000107104 changed to uniprot id : Q14678.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131473\n",
      "\n",
      "Gene ensembl id ENSG00000131473 changed to uniprot id : P53396.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136381\n",
      "\n",
      "Gene ensembl id ENSG00000136381 changed to uniprot id : P48200.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122729\n",
      "\n",
      "Gene ensembl id ENSG00000122729 changed to uniprot id : P21399.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100412\n",
      "\n",
      "Gene ensembl id ENSG00000100412 changed to uniprot id : Q99798.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117118\n",
      "\n",
      "Gene ensembl id ENSG00000117118 changed to uniprot id : P21912.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143252\n",
      "\n",
      "Gene ensembl id ENSG00000143252 changed to uniprot id : Q99643.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204370\n",
      "\n",
      "Gene ensembl id ENSG00000204370 changed to uniprot id : G3V173.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000073578\n",
      "\n",
      "Gene ensembl id ENSG00000073578 changed to uniprot id : P31040.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125246\n",
      "\n",
      "Gene ensembl id ENSG00000125246 changed to uniprot id : Q8N0X4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138777\n",
      "\n",
      "Gene ensembl id ENSG00000138777 changed to uniprot id : Q9H2U2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000107902\n",
      "\n",
      "Gene ensembl id ENSG00000107902 changed to uniprot id : Q9H008.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000180817\n",
      "\n",
      "Gene ensembl id ENSG00000180817 changed to uniprot id : Q15181.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140374\n",
      "\n",
      "Gene ensembl id ENSG00000140374 changed to uniprot id : A0A8I5KUF3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171503\n",
      "\n",
      "Gene ensembl id ENSG00000171503 changed to uniprot id : A0A804HI48.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105379\n",
      "\n",
      "Gene ensembl id ENSG00000105379 changed to uniprot id : P38117.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170516\n",
      "\n",
      "Gene ensembl id ENSG00000170516 changed to uniprot id : Q8TF08.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198938\n",
      "\n",
      "Gene ensembl id ENSG00000198938 changed to uniprot id : Q7GIM7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000187581\n",
      "\n",
      "Gene ensembl id ENSG00000187581 changed to uniprot id : Q7Z4L0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156885\n",
      "\n",
      "Gene ensembl id ENSG00000156885 changed to uniprot id : Q02221.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178741\n",
      "\n",
      "Gene ensembl id ENSG00000178741 changed to uniprot id : P20674.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112695\n",
      "\n",
      "Gene ensembl id ENSG00000112695 changed to uniprot id : P14406.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131143\n",
      "\n",
      "Gene ensembl id ENSG00000131143 changed to uniprot id : P13073.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160471\n",
      "\n",
      "Gene ensembl id ENSG00000160471 changed to uniprot id : Q6YFQ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000127184\n",
      "\n",
      "Gene ensembl id ENSG00000127184 changed to uniprot id : P15954.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131174\n",
      "\n",
      "Gene ensembl id ENSG00000131174 changed to uniprot id : P24311.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135940\n",
      "\n",
      "Gene ensembl id ENSG00000135940 changed to uniprot id : P10606.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198712\n",
      "\n",
      "Gene ensembl id ENSG00000198712 changed to uniprot id : A0A1X7RBF7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111775\n",
      "\n",
      "Gene ensembl id ENSG00000111775 changed to uniprot id : P12074.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198804\n",
      "\n",
      "Gene ensembl id ENSG00000198804 changed to uniprot id : P00395.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176340\n",
      "\n",
      "Gene ensembl id ENSG00000176340 changed to uniprot id : Q53XN1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115944\n",
      "\n",
      "Gene ensembl id ENSG00000115944 changed to uniprot id : O14548.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131055\n",
      "\n",
      "Gene ensembl id ENSG00000131055 changed to uniprot id : Q96KJ9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000014919\n",
      "\n",
      "Gene ensembl id ENSG00000014919 changed to uniprot id : Q7KZN9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138495\n",
      "\n",
      "Gene ensembl id ENSG00000138495 changed to uniprot id : Q14061.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000161281\n",
      "\n",
      "Gene ensembl id ENSG00000161281 changed to uniprot id : P24310.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164919\n",
      "\n",
      "Gene ensembl id ENSG00000164919 changed to uniprot id : P09669.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000126267\n",
      "\n",
      "Gene ensembl id ENSG00000126267 changed to uniprot id : P14854.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140829\n",
      "\n",
      "Gene ensembl id ENSG00000140829 changed to uniprot id : Q92620.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000187240\n",
      "\n",
      "Gene ensembl id ENSG00000187240 changed to uniprot id : Q8NCM8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000069849\n",
      "\n",
      "Gene ensembl id ENSG00000069849 changed to uniprot id : P54709.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125375\n",
      "\n",
      "Gene ensembl id ENSG00000125375 changed to uniprot id : A0A5F9ZGZ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111642\n",
      "\n",
      "Gene ensembl id ENSG00000111642 changed to uniprot id : Q14839.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134452\n",
      "\n",
      "Gene ensembl id ENSG00000134452 changed to uniprot id : Q8NFZ0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000039123\n",
      "\n",
      "Gene ensembl id ENSG00000039123 changed to uniprot id : P42285.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000144045\n",
      "\n",
      "Gene ensembl id ENSG00000144045 changed to uniprot id : Q8TE96.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136271\n",
      "\n",
      "Gene ensembl id ENSG00000136271 changed to uniprot id : Q9NY93.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000249222\n",
      "\n",
      "Gene ensembl id ENSG00000249222 changed to uniprot id : Q7Z4Y8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116459\n",
      "\n",
      "Gene ensembl id ENSG00000116459 changed to uniprot id : P24539.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135390\n",
      "\n",
      "Gene ensembl id ENSG00000135390 changed to uniprot id : Q06055.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117410\n",
      "\n",
      "Gene ensembl id ENSG00000117410 changed to uniprot id : Q99437.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156976\n",
      "\n",
      "Gene ensembl id ENSG00000156976 changed to uniprot id : E9PBH4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140451\n",
      "\n",
      "Gene ensembl id ENSG00000140451 changed to uniprot id : Q9H611.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000058668\n",
      "\n",
      "Gene ensembl id ENSG00000058668 changed to uniprot id : A0A024R968.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152234\n",
      "\n",
      "Gene ensembl id ENSG00000152234 changed to uniprot id : P25705.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145833\n",
      "\n",
      "Gene ensembl id ENSG00000145833 changed to uniprot id : Q7L014.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000073969\n",
      "\n",
      "Gene ensembl id ENSG00000073969 changed to uniprot id : I3L2G1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123136\n",
      "\n",
      "Gene ensembl id ENSG00000123136 changed to uniprot id : O00148.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000033627\n",
      "\n",
      "Gene ensembl id ENSG00000033627 changed to uniprot id : B7Z641.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186009\n",
      "\n",
      "Gene ensembl id ENSG00000186009 changed to uniprot id : P51164.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167216\n",
      "\n",
      "Gene ensembl id ENSG00000167216 changed to uniprot id : Q8IYT4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171953\n",
      "\n",
      "Gene ensembl id ENSG00000171953 changed to uniprot id : Q8N5M1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169020\n",
      "\n",
      "Gene ensembl id ENSG00000169020 changed to uniprot id : P56385.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114573\n",
      "\n",
      "Gene ensembl id ENSG00000114573 changed to uniprot id : P38606.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154518\n",
      "\n",
      "Gene ensembl id ENSG00000154518 changed to uniprot id : P48201.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000018625\n",
      "\n",
      "Gene ensembl id ENSG00000018625 changed to uniprot id : P50993.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000081923\n",
      "\n",
      "Gene ensembl id ENSG00000081923 changed to uniprot id : O43520.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000073146\n",
      "\n",
      "Gene ensembl id ENSG00000073146 changed to uniprot id : F2Z2H1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110719\n",
      "\n",
      "Gene ensembl id ENSG00000110719 changed to uniprot id : Q13488.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124406\n",
      "\n",
      "Gene ensembl id ENSG00000124406 changed to uniprot id : Q9Y2Q0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105671\n",
      "\n",
      "Gene ensembl id ENSG00000105671 changed to uniprot id : Q9Y6V7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000241468\n",
      "\n",
      "Gene ensembl id ENSG00000241468 changed to uniprot id : P56134.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100554\n",
      "\n",
      "Gene ensembl id ENSG00000100554 changed to uniprot id : Q9Y5K8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198899\n",
      "\n",
      "Gene ensembl id ENSG00000198899 changed to uniprot id : P00846.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136492\n",
      "\n",
      "Gene ensembl id ENSG00000136492 changed to uniprot id : Q9BX63.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159720\n",
      "\n",
      "Gene ensembl id ENSG00000159720 changed to uniprot id : P61421.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164080\n",
      "\n",
      "Gene ensembl id ENSG00000164080 changed to uniprot id : Q9Y4B4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000001626\n",
      "\n",
      "Gene ensembl id ENSG00000001626 changed to uniprot id : P13569.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070961\n",
      "\n",
      "Gene ensembl id ENSG00000070961 changed to uniprot id : A0A024RBC7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204560\n",
      "\n",
      "Gene ensembl id ENSG00000204560 changed to uniprot id : O60231.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101892\n",
      "\n",
      "Gene ensembl id ENSG00000101892 changed to uniprot id : B7ZKW0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147614\n",
      "\n",
      "Gene ensembl id ENSG00000147614 changed to uniprot id : A0A024R991.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110367\n",
      "\n",
      "Gene ensembl id ENSG00000110367 changed to uniprot id : P26196.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110955\n",
      "\n",
      "Gene ensembl id ENSG00000110955 changed to uniprot id : P06576.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132681\n",
      "\n",
      "Gene ensembl id ENSG00000132681 changed to uniprot id : Q13733.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174243\n",
      "\n",
      "Gene ensembl id ENSG00000174243 changed to uniprot id : Q9BUQ8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000234920\n",
      "\n",
      "Gene ensembl id ENSG00000234920 changed to uniprot id : O95670.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101974\n",
      "\n",
      "Gene ensembl id ENSG00000101974 changed to uniprot id : Q8NB49.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000153922\n",
      "\n",
      "Gene ensembl id ENSG00000153922 changed to uniprot id : O14646.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111877\n",
      "\n",
      "Gene ensembl id ENSG00000111877 changed to uniprot id : Q9NXL9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000074370\n",
      "\n",
      "Gene ensembl id ENSG00000074370 changed to uniprot id : Q93084.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115267\n",
      "\n",
      "Gene ensembl id ENSG00000115267 changed to uniprot id : Q9BYX4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108654\n",
      "\n",
      "Gene ensembl id ENSG00000108654 changed to uniprot id : P17844.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000017260\n",
      "\n",
      "Gene ensembl id ENSG00000017260 changed to uniprot id : P98194.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147416\n",
      "\n",
      "Gene ensembl id ENSG00000147416 changed to uniprot id : P21281.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116039\n",
      "\n",
      "Gene ensembl id ENSG00000116039 changed to uniprot id : P15313.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000088205\n",
      "\n",
      "Gene ensembl id ENSG00000088205 changed to uniprot id : Q9NVP1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167863\n",
      "\n",
      "Gene ensembl id ENSG00000167863 changed to uniprot id : O75947.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104738\n",
      "\n",
      "Gene ensembl id ENSG00000104738 changed to uniprot id : P33991.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145246\n",
      "\n",
      "Gene ensembl id ENSG00000145246 changed to uniprot id : Q9P241.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165629\n",
      "\n",
      "Gene ensembl id ENSG00000165629 changed to uniprot id : P36542.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185883\n",
      "\n",
      "Gene ensembl id ENSG00000185883 changed to uniprot id : P27449.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154723\n",
      "\n",
      "Gene ensembl id ENSG00000154723 changed to uniprot id : P18859.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116254\n",
      "\n",
      "Gene ensembl id ENSG00000116254 changed to uniprot id : Q8TDI0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102781\n",
      "\n",
      "Gene ensembl id ENSG00000102781 changed to uniprot id : Q9BW62.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000150990\n",
      "\n",
      "Gene ensembl id ENSG00000150990 changed to uniprot id : Q8IY37.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000085224\n",
      "\n",
      "Gene ensembl id ENSG00000085224 changed to uniprot id : P46100.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108469\n",
      "\n",
      "Gene ensembl id ENSG00000108469 changed to uniprot id : A0A024R8M9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000188342\n",
      "\n",
      "Gene ensembl id ENSG00000188342 changed to uniprot id : P13984.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138346\n",
      "\n",
      "Gene ensembl id ENSG00000138346 changed to uniprot id : P51530.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113732\n",
      "\n",
      "Gene ensembl id ENSG00000113732 changed to uniprot id : O15342.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132153\n",
      "\n",
      "Gene ensembl id ENSG00000132153 changed to uniprot id : Q7L2E3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131100\n",
      "\n",
      "Gene ensembl id ENSG00000131100 changed to uniprot id : P36543.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185344\n",
      "\n",
      "Gene ensembl id ENSG00000185344 changed to uniprot id : Q8TBM3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136888\n",
      "\n",
      "Gene ensembl id ENSG00000136888 changed to uniprot id : O75348.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151418\n",
      "\n",
      "Gene ensembl id ENSG00000151418 changed to uniprot id : Q96LB4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174953\n",
      "\n",
      "Gene ensembl id ENSG00000174953 changed to uniprot id : A0A0G2JQU7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105675\n",
      "\n",
      "Gene ensembl id ENSG00000105675 changed to uniprot id : P20648.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000089737\n",
      "\n",
      "Gene ensembl id ENSG00000089737 changed to uniprot id : Q9GZR7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123472\n",
      "\n",
      "Gene ensembl id ENSG00000123472 changed to uniprot id : Q5TC12.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166833\n",
      "\n",
      "Gene ensembl id ENSG00000166833 changed to uniprot id : Q8IVL1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157087\n",
      "\n",
      "Gene ensembl id ENSG00000157087 changed to uniprot id : A0A024R2E4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155097\n",
      "\n",
      "Gene ensembl id ENSG00000155097 changed to uniprot id : P21283.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163161\n",
      "\n",
      "Gene ensembl id ENSG00000163161 changed to uniprot id : P19447.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105929\n",
      "\n",
      "Gene ensembl id ENSG00000105929 changed to uniprot id : A0A024R791.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099624\n",
      "\n",
      "Gene ensembl id ENSG00000099624 changed to uniprot id : P30049.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143153\n",
      "\n",
      "Gene ensembl id ENSG00000143153 changed to uniprot id : P05026.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000241837\n",
      "\n",
      "Gene ensembl id ENSG00000241837 changed to uniprot id : P48047.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159199\n",
      "\n",
      "Gene ensembl id ENSG00000159199 changed to uniprot id : P05496.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135829\n",
      "\n",
      "Gene ensembl id ENSG00000135829 changed to uniprot id : Q08211.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198563\n",
      "\n",
      "Gene ensembl id ENSG00000198563 changed to uniprot id : F6S2B7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163399\n",
      "\n",
      "Gene ensembl id ENSG00000163399 changed to uniprot id : P05023.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105409\n",
      "\n",
      "Gene ensembl id ENSG00000105409 changed to uniprot id : P13637.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129244\n",
      "\n",
      "Gene ensembl id ENSG00000129244 changed to uniprot id : P14415.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138375\n",
      "\n",
      "Gene ensembl id ENSG00000138375 changed to uniprot id : Q9NZC9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000229604\n",
      "\n",
      "No uniprot id found for gene ENSG00000229604.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141543\n",
      "\n",
      "Gene ensembl id ENSG00000141543 changed to uniprot id : P38919.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124228\n",
      "\n",
      "Gene ensembl id ENSG00000124228 changed to uniprot id : B7Z6D5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143882\n",
      "\n",
      "Gene ensembl id ENSG00000143882 changed to uniprot id : Q8NEY4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124172\n",
      "\n",
      "Gene ensembl id ENSG00000124172 changed to uniprot id : P56381.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167283\n",
      "\n",
      "Gene ensembl id ENSG00000167283 changed to uniprot id : O75964.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000064703\n",
      "\n",
      "Gene ensembl id ENSG00000064703 changed to uniprot id : Q9UHI6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000079785\n",
      "\n",
      "Gene ensembl id ENSG00000079785 changed to uniprot id : Q92499.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186625\n",
      "\n",
      "Gene ensembl id ENSG00000186625 changed to uniprot id : O75449.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000047249\n",
      "\n",
      "Gene ensembl id ENSG00000047249 changed to uniprot id : Q9UI12.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000080007\n",
      "\n",
      "Gene ensembl id ENSG00000080007 changed to uniprot id : Q9NXZ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198231\n",
      "\n",
      "Gene ensembl id ENSG00000198231 changed to uniprot id : Q86XP3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183207\n",
      "\n",
      "Gene ensembl id ENSG00000183207 changed to uniprot id : Q9Y230.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000004700\n",
      "\n",
      "Gene ensembl id ENSG00000004700 changed to uniprot id : P46063.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000177200\n",
      "\n",
      "Gene ensembl id ENSG00000177200 changed to uniprot id : Q3L8U1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000206190\n",
      "\n",
      "Gene ensembl id ENSG00000206190 changed to uniprot id : O60312.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104043\n",
      "\n",
      "Gene ensembl id ENSG00000104043 changed to uniprot id : Q8TF62.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000250565\n",
      "\n",
      "Gene ensembl id ENSG00000250565 changed to uniprot id : Q96A05.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196296\n",
      "\n",
      "Gene ensembl id ENSG00000196296 changed to uniprot id : O14983.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000071553\n",
      "\n",
      "Gene ensembl id ENSG00000071553 changed to uniprot id : Q15904.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000021574\n",
      "\n",
      "Gene ensembl id ENSG00000021574 changed to uniprot id : E5KRP5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213782\n",
      "\n",
      "Gene ensembl id ENSG00000213782 changed to uniprot id : Q9H0S4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128524\n",
      "\n",
      "Gene ensembl id ENSG00000128524 changed to uniprot id : A4D1K0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000010256\n",
      "\n",
      "Gene ensembl id ENSG00000010256 changed to uniprot id : P31930.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140740\n",
      "\n",
      "Gene ensembl id ENSG00000140740 changed to uniprot id : P22695.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184076\n",
      "\n",
      "Gene ensembl id ENSG00000184076 changed to uniprot id : Q9UDW1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000127540\n",
      "\n",
      "Gene ensembl id ENSG00000127540 changed to uniprot id : O14957.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156467\n",
      "\n",
      "Gene ensembl id ENSG00000156467 changed to uniprot id : P14927.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164405\n",
      "\n",
      "Gene ensembl id ENSG00000164405 changed to uniprot id : O14949.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169021\n",
      "\n",
      "Gene ensembl id ENSG00000169021 changed to uniprot id : P47985.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198727\n",
      "\n",
      "Gene ensembl id ENSG00000198727 changed to uniprot id : P00156.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173660\n",
      "\n",
      "Gene ensembl id ENSG00000173660 changed to uniprot id : P07919.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000179091\n",
      "\n",
      "Gene ensembl id ENSG00000179091 changed to uniprot id : P08574.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140990\n",
      "\n",
      "Gene ensembl id ENSG00000140990 changed to uniprot id : A8K761.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000004779\n",
      "\n",
      "Gene ensembl id ENSG00000004779 changed to uniprot id : O14561.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198786\n",
      "\n",
      "Gene ensembl id ENSG00000198786 changed to uniprot id : P03915.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125356\n",
      "\n",
      "Gene ensembl id ENSG00000125356 changed to uniprot id : O15239.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184983\n",
      "\n",
      "Gene ensembl id ENSG00000184983 changed to uniprot id : R4GN43.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198695\n",
      "\n",
      "Gene ensembl id ENSG00000198695 changed to uniprot id : P03923.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178127\n",
      "\n",
      "Gene ensembl id ENSG00000178127 changed to uniprot id : P19404.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130414\n",
      "\n",
      "Gene ensembl id ENSG00000130414 changed to uniprot id : O95299.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151366\n",
      "\n",
      "Gene ensembl id ENSG00000151366 changed to uniprot id : O95298.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109390\n",
      "\n",
      "Gene ensembl id ENSG00000109390 changed to uniprot id : O43677.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119013\n",
      "\n",
      "Gene ensembl id ENSG00000119013 changed to uniprot id : O43676.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115286\n",
      "\n",
      "Gene ensembl id ENSG00000115286 changed to uniprot id : O75251.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198888\n",
      "\n",
      "Gene ensembl id ENSG00000198888 changed to uniprot id : P03886.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185633\n",
      "\n",
      "Gene ensembl id ENSG00000185633 changed to uniprot id : Q9NRX3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000212907\n",
      "\n",
      "Gene ensembl id ENSG00000212907 changed to uniprot id : P03901.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213619\n",
      "\n",
      "Gene ensembl id ENSG00000213619 changed to uniprot id : O75489.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139180\n",
      "\n",
      "Gene ensembl id ENSG00000139180 changed to uniprot id : Q16795.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147123\n",
      "\n",
      "Gene ensembl id ENSG00000147123 changed to uniprot id : Q9NX14.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186010\n",
      "\n",
      "Gene ensembl id ENSG00000186010 changed to uniprot id : Q9P0J0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166136\n",
      "\n",
      "Gene ensembl id ENSG00000166136 changed to uniprot id : E9PIZ8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000189043\n",
      "\n",
      "Gene ensembl id ENSG00000189043 changed to uniprot id : O00483.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147684\n",
      "\n",
      "Gene ensembl id ENSG00000147684 changed to uniprot id : Q9Y6M9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160194\n",
      "\n",
      "Gene ensembl id ENSG00000160194 changed to uniprot id : P56181.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174886\n",
      "\n",
      "Gene ensembl id ENSG00000174886 changed to uniprot id : Q86Y39.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198886\n",
      "\n",
      "Gene ensembl id ENSG00000198886 changed to uniprot id : P03905.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110717\n",
      "\n",
      "Gene ensembl id ENSG00000110717 changed to uniprot id : O00217.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145494\n",
      "\n",
      "Gene ensembl id ENSG00000145494 changed to uniprot id : O75380.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000023228\n",
      "\n",
      "Gene ensembl id ENSG00000023228 changed to uniprot id : B4DJ81.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198763\n",
      "\n",
      "Gene ensembl id ENSG00000198763 changed to uniprot id : P03891.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167792\n",
      "\n",
      "Gene ensembl id ENSG00000167792 changed to uniprot id : P49821.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170906\n",
      "\n",
      "Gene ensembl id ENSG00000170906 changed to uniprot id : O95167.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168653\n",
      "\n",
      "Gene ensembl id ENSG00000168653 changed to uniprot id : O43920.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164258\n",
      "\n",
      "Gene ensembl id ENSG00000164258 changed to uniprot id : A0A0S2Z433.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099795\n",
      "\n",
      "Gene ensembl id ENSG00000099795 changed to uniprot id : P17568.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158864\n",
      "\n",
      "Gene ensembl id ENSG00000158864 changed to uniprot id : O75306.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165264\n",
      "\n",
      "Gene ensembl id ENSG00000165264 changed to uniprot id : O95139.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128609\n",
      "\n",
      "Gene ensembl id ENSG00000128609 changed to uniprot id : Q16718.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183648\n",
      "\n",
      "Gene ensembl id ENSG00000183648 changed to uniprot id : O75438.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000090266\n",
      "\n",
      "Gene ensembl id ENSG00000090266 changed to uniprot id : O95178.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000065518\n",
      "\n",
      "Gene ensembl id ENSG00000065518 changed to uniprot id : O95168.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131495\n",
      "\n",
      "Gene ensembl id ENSG00000131495 changed to uniprot id : O43678.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136521\n",
      "\n",
      "Gene ensembl id ENSG00000136521 changed to uniprot id : A0A087WYD0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184752\n",
      "\n",
      "Gene ensembl id ENSG00000184752 changed to uniprot id : Q9UI09.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198840\n",
      "\n",
      "Gene ensembl id ENSG00000198840 changed to uniprot id : P03897.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167774\n",
      "\n",
      "No uniprot id found for gene ENSG00000167774.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119421\n",
      "\n",
      "Gene ensembl id ENSG00000119421 changed to uniprot id : P51970.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109610\n",
      "\n",
      "Gene ensembl id ENSG00000109610 changed to uniprot id : P08294.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142168\n",
      "\n",
      "Gene ensembl id ENSG00000142168 changed to uniprot id : P00441.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112096\n",
      "\n",
      "No uniprot id found for gene ENSG00000112096.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183747\n",
      "\n",
      "Gene ensembl id ENSG00000183747 changed to uniprot id : Q08AH3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000215009\n",
      "\n",
      "Gene ensembl id ENSG00000215009 changed to uniprot id : P0C7M7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000005187\n",
      "\n",
      "Gene ensembl id ENSG00000005187 changed to uniprot id : Q53FZ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183549\n",
      "\n",
      "Gene ensembl id ENSG00000183549 changed to uniprot id : Q6NUN0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166743\n",
      "\n",
      "Gene ensembl id ENSG00000166743 changed to uniprot id : Q08AH1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151726\n",
      "\n",
      "Gene ensembl id ENSG00000151726 changed to uniprot id : P33121.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000068366\n",
      "\n",
      "Gene ensembl id ENSG00000068366 changed to uniprot id : O60488.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197142\n",
      "\n",
      "Gene ensembl id ENSG00000197142 changed to uniprot id : Q9ULC5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123983\n",
      "\n",
      "Gene ensembl id ENSG00000123983 changed to uniprot id : O95573.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140284\n",
      "\n",
      "Gene ensembl id ENSG00000140284 changed to uniprot id : O14975.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130377\n",
      "\n",
      "Gene ensembl id ENSG00000130377 changed to uniprot id : Q5FVE4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164398\n",
      "\n",
      "Gene ensembl id ENSG00000164398 changed to uniprot id : Q9UKU0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119673\n",
      "\n",
      "Gene ensembl id ENSG00000119673 changed to uniprot id : P49753.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000177465\n",
      "\n",
      "Gene ensembl id ENSG00000177465 changed to uniprot id : Q8N9L9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169710\n",
      "\n",
      "Gene ensembl id ENSG00000169710 changed to uniprot id : P49327.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151093\n",
      "\n",
      "Gene ensembl id ENSG00000151093 changed to uniprot id : Q9NWU1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132142\n",
      "\n",
      "No uniprot id found for gene ENSG00000132142.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000076555\n",
      "\n",
      "Gene ensembl id ENSG00000076555 changed to uniprot id : O00763.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149084\n",
      "\n",
      "Gene ensembl id ENSG00000149084 changed to uniprot id : Q53GQ0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146066\n",
      "\n",
      "Gene ensembl id ENSG00000146066 changed to uniprot id : Q9BW72.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000074696\n",
      "\n",
      "Gene ensembl id ENSG00000074696 changed to uniprot id : Q9P035.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099797\n",
      "\n",
      "Gene ensembl id ENSG00000099797 changed to uniprot id : Q9NZ01.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099194\n",
      "\n",
      "Gene ensembl id ENSG00000099194 changed to uniprot id : O00767.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145284\n",
      "\n",
      "Gene ensembl id ENSG00000145284 changed to uniprot id : Q86SK9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172782\n",
      "\n",
      "Gene ensembl id ENSG00000172782 changed to uniprot id : A0A087WWN9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134824\n",
      "\n",
      "Gene ensembl id ENSG00000134824 changed to uniprot id : O95864.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000221968\n",
      "\n",
      "Gene ensembl id ENSG00000221968 changed to uniprot id : Q9Y5Q0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149485\n",
      "\n",
      "Gene ensembl id ENSG00000149485 changed to uniprot id : A0A0A0MR51.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000221988\n",
      "\n",
      "Gene ensembl id ENSG00000221988 changed to uniprot id : A0A0G2JHB6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184227\n",
      "\n",
      "Gene ensembl id ENSG00000184227 changed to uniprot id : Q86TX2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000097021\n",
      "\n",
      "Gene ensembl id ENSG00000097021 changed to uniprot id : O00154.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131238\n",
      "\n",
      "Gene ensembl id ENSG00000131238 changed to uniprot id : P50897.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136881\n",
      "\n",
      "Gene ensembl id ENSG00000136881 changed to uniprot id : Q14032.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100294\n",
      "\n",
      "Gene ensembl id ENSG00000100294 changed to uniprot id : Q8IVS2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152463\n",
      "\n",
      "Gene ensembl id ENSG00000152463 changed to uniprot id : Q9NV23.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000161905\n",
      "\n",
      "Gene ensembl id ENSG00000161905 changed to uniprot id : P16050.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000021461\n",
      "\n",
      "Gene ensembl id ENSG00000021461 changed to uniprot id : Q9HB55.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197446\n",
      "\n",
      "Gene ensembl id ENSG00000197446 changed to uniprot id : P24903.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137869\n",
      "\n",
      "Gene ensembl id ENSG00000137869 changed to uniprot id : P11511.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186204\n",
      "\n",
      "Gene ensembl id ENSG00000186204 changed to uniprot id : Q9HCS2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197408\n",
      "\n",
      "Gene ensembl id ENSG00000197408 changed to uniprot id : P20813.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160868\n",
      "\n",
      "Gene ensembl id ENSG00000160868 changed to uniprot id : P08684.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138109\n",
      "\n",
      "Gene ensembl id ENSG00000138109 changed to uniprot id : S5RV20.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108242\n",
      "\n",
      "Gene ensembl id ENSG00000108242 changed to uniprot id : P33260.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100197\n",
      "\n",
      "Gene ensembl id ENSG00000100197 changed to uniprot id : Q6NWU0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138115\n",
      "\n",
      "Gene ensembl id ENSG00000138115 changed to uniprot id : P10632.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000255974\n",
      "\n",
      "Gene ensembl id ENSG00000255974 changed to uniprot id : P11509.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186160\n",
      "\n",
      "Gene ensembl id ENSG00000186160 changed to uniprot id : Q86W10.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198077\n",
      "\n",
      "Gene ensembl id ENSG00000198077 changed to uniprot id : P20853.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171903\n",
      "\n",
      "Gene ensembl id ENSG00000171903 changed to uniprot id : Q9HBI6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130649\n",
      "\n",
      "Gene ensembl id ENSG00000130649 changed to uniprot id : P05181.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197838\n",
      "\n",
      "Gene ensembl id ENSG00000197838 changed to uniprot id : Q16696.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160870\n",
      "\n",
      "Gene ensembl id ENSG00000160870 changed to uniprot id : P24462.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106258\n",
      "\n",
      "Gene ensembl id ENSG00000106258 changed to uniprot id : P20815.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120915\n",
      "\n",
      "Gene ensembl id ENSG00000120915 changed to uniprot id : P34913.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143819\n",
      "\n",
      "Gene ensembl id ENSG00000143819 changed to uniprot id : P07099.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000144182\n",
      "\n",
      "Gene ensembl id ENSG00000144182 changed to uniprot id : Q9Y234.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121897\n",
      "\n",
      "Gene ensembl id ENSG00000121897 changed to uniprot id : O43766.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175536\n",
      "\n",
      "Gene ensembl id ENSG00000175536 changed to uniprot id : A6NK58.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000161533\n",
      "\n",
      "Gene ensembl id ENSG00000161533 changed to uniprot id : Q15067.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113790\n",
      "\n",
      "Gene ensembl id ENSG00000113790 changed to uniprot id : Q08426.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133835\n",
      "\n",
      "Gene ensembl id ENSG00000133835 changed to uniprot id : A0A2R8Y674.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167306\n",
      "\n",
      "Gene ensembl id ENSG00000167306 changed to uniprot id : Q9ULV0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000060971\n",
      "\n",
      "Gene ensembl id ENSG00000060971 changed to uniprot id : P09110.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000012779\n",
      "\n",
      "Gene ensembl id ENSG00000012779 changed to uniprot id : P09917.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159231\n",
      "\n",
      "Gene ensembl id ENSG00000159231 changed to uniprot id : O75828.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159228\n",
      "\n",
      "Gene ensembl id ENSG00000159228 changed to uniprot id : P16152.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157326\n",
      "\n",
      "Gene ensembl id ENSG00000157326 changed to uniprot id : Q9BTZ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111144\n",
      "\n",
      "Gene ensembl id ENSG00000111144 changed to uniprot id : P09960.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186115\n",
      "\n",
      "Gene ensembl id ENSG00000186115 changed to uniprot id : P78329.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000073756\n",
      "\n",
      "Gene ensembl id ENSG00000073756 changed to uniprot id : P35354.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000095303\n",
      "\n",
      "Gene ensembl id ENSG00000095303 changed to uniprot id : P23219.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105254\n",
      "\n",
      "Gene ensembl id ENSG00000105254 changed to uniprot id : Q99426.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148344\n",
      "\n",
      "Gene ensembl id ENSG00000148344 changed to uniprot id : O14684.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148334\n",
      "\n",
      "Gene ensembl id ENSG00000148334 changed to uniprot id : Q9H7Z7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115361\n",
      "\n",
      "Gene ensembl id ENSG00000115361 changed to uniprot id : P28330.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000242612\n",
      "\n",
      "Gene ensembl id ENSG00000242612 changed to uniprot id : A0A0J9YY83.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104325\n",
      "\n",
      "Gene ensembl id ENSG00000104325 changed to uniprot id : Q16698.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167969\n",
      "\n",
      "Gene ensembl id ENSG00000167969 changed to uniprot id : P42126.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198721\n",
      "\n",
      "Gene ensembl id ENSG00000198721 changed to uniprot id : O75521.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104823\n",
      "\n",
      "Gene ensembl id ENSG00000104823 changed to uniprot id : Q13011.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108839\n",
      "\n",
      "Gene ensembl id ENSG00000108839 changed to uniprot id : P18054.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167107\n",
      "\n",
      "Gene ensembl id ENSG00000167107 changed to uniprot id : Q96CM8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000242110\n",
      "\n",
      "Gene ensembl id ENSG00000242110 changed to uniprot id : Q9UHK6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101473\n",
      "\n",
      "Gene ensembl id ENSG00000101473 changed to uniprot id : O14734.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130304\n",
      "\n",
      "Gene ensembl id ENSG00000130304 changed to uniprot id : B7Z225.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157426\n",
      "\n",
      "Gene ensembl id ENSG00000157426 changed to uniprot id : Q4L235.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167114\n",
      "\n",
      "Gene ensembl id ENSG00000167114 changed to uniprot id : Q6P1M0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143554\n",
      "\n",
      "Gene ensembl id ENSG00000143554 changed to uniprot id : Q5K4L6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176715\n",
      "\n",
      "Gene ensembl id ENSG00000176715 changed to uniprot id : Q4G176.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106853\n",
      "\n",
      "Gene ensembl id ENSG00000106853 changed to uniprot id : Q14914.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213316\n",
      "\n",
      "Gene ensembl id ENSG00000213316 changed to uniprot id : Q16873.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167261\n",
      "\n",
      "Gene ensembl id ENSG00000167261 changed to uniprot id : Q9H4A9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000180776\n",
      "\n",
      "Gene ensembl id ENSG00000180776 changed to uniprot id : Q5W0Z9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141446\n",
      "\n",
      "Gene ensembl id ENSG00000141446 changed to uniprot id : Q5FWF5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167011\n",
      "\n",
      "Gene ensembl id ENSG00000167011 changed to uniprot id : Q8N8M0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000054392\n",
      "\n",
      "Gene ensembl id ENSG00000054392 changed to uniprot id : A0A075B6R5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102383\n",
      "\n",
      "Gene ensembl id ENSG00000102383 changed to uniprot id : Q96MV8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163812\n",
      "\n",
      "Gene ensembl id ENSG00000163812 changed to uniprot id : Q9NYG2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171307\n",
      "\n",
      "Gene ensembl id ENSG00000171307 changed to uniprot id : Q969W1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136247\n",
      "\n",
      "Gene ensembl id ENSG00000136247 changed to uniprot id : Q9NPG8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100344\n",
      "\n",
      "Gene ensembl id ENSG00000100344 changed to uniprot id : Q9NST1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172197\n",
      "\n",
      "Gene ensembl id ENSG00000172197 changed to uniprot id : Q6ZNC8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111684\n",
      "\n",
      "Gene ensembl id ENSG00000111684 changed to uniprot id : Q6P1A2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172954\n",
      "\n",
      "Gene ensembl id ENSG00000172954 changed to uniprot id : Q6UWP7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174165\n",
      "\n",
      "Gene ensembl id ENSG00000174165 changed to uniprot id : Q6UX98.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110583\n",
      "\n",
      "Gene ensembl id ENSG00000110583 changed to uniprot id : Q86UY6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125505\n",
      "\n",
      "Gene ensembl id ENSG00000125505 changed to uniprot id : C9J4E9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106636\n",
      "\n",
      "Gene ensembl id ENSG00000106636 changed to uniprot id : O15498.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156599\n",
      "\n",
      "Gene ensembl id ENSG00000156599 changed to uniprot id : Q9C0B5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103066\n",
      "\n",
      "Gene ensembl id ENSG00000103066 changed to uniprot id : Q8NCC3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000177054\n",
      "\n",
      "Gene ensembl id ENSG00000177054 changed to uniprot id : Q8IUH4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171320\n",
      "\n",
      "Gene ensembl id ENSG00000171320 changed to uniprot id : Q56NI9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122390\n",
      "\n",
      "Gene ensembl id ENSG00000122390 changed to uniprot id : Q9H7X0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175048\n",
      "\n",
      "Gene ensembl id ENSG00000175048 changed to uniprot id : Q8IZN3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159714\n",
      "\n",
      "Gene ensembl id ENSG00000159714 changed to uniprot id : Q8WTX9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000188706\n",
      "\n",
      "Gene ensembl id ENSG00000188706 changed to uniprot id : Q9Y397.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184210\n",
      "\n",
      "Gene ensembl id ENSG00000184210 changed to uniprot id : Q6ZPD8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186908\n",
      "\n",
      "Gene ensembl id ENSG00000186908 changed to uniprot id : Q8IUH5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000097033\n",
      "\n",
      "Gene ensembl id ENSG00000097033 changed to uniprot id : Q9Y371.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099904\n",
      "\n",
      "Gene ensembl id ENSG00000099904 changed to uniprot id : Q9ULC8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000243477\n",
      "\n",
      "Gene ensembl id ENSG00000243477 changed to uniprot id : Q93015.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000144035\n",
      "\n",
      "Gene ensembl id ENSG00000144035 changed to uniprot id : Q9UHE5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000153395\n",
      "\n",
      "Gene ensembl id ENSG00000153395 changed to uniprot id : Q8NF37.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143797\n",
      "\n",
      "Gene ensembl id ENSG00000143797 changed to uniprot id : Q6ZWT7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184307\n",
      "\n",
      "Gene ensembl id ENSG00000184307 changed to uniprot id : B3KXV3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123684\n",
      "\n",
      "Gene ensembl id ENSG00000123684 changed to uniprot id : Q92604.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176454\n",
      "\n",
      "Gene ensembl id ENSG00000176454 changed to uniprot id : Q643R3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102030\n",
      "\n",
      "Gene ensembl id ENSG00000102030 changed to uniprot id : P41227.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000087253\n",
      "\n",
      "Gene ensembl id ENSG00000087253 changed to uniprot id : Q7L5N7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169169\n",
      "\n",
      "Gene ensembl id ENSG00000169169 changed to uniprot id : Q8TCG5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000205560\n",
      "\n",
      "Gene ensembl id ENSG00000205560 changed to uniprot id : Q92523.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110090\n",
      "\n",
      "Gene ensembl id ENSG00000110090 changed to uniprot id : P50416.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000178537\n",
      "\n",
      "Gene ensembl id ENSG00000178537 changed to uniprot id : O43772.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000095321\n",
      "\n",
      "Gene ensembl id ENSG00000095321 changed to uniprot id : P43155.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157184\n",
      "\n",
      "Gene ensembl id ENSG00000157184 changed to uniprot id : P23786.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000005469\n",
      "\n",
      "Gene ensembl id ENSG00000005469 changed to uniprot id : Q9UKG9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197119\n",
      "\n",
      "Gene ensembl id ENSG00000197119 changed to uniprot id : Q8N8R3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149742\n",
      "\n",
      "Gene ensembl id ENSG00000149742 changed to uniprot id : Q8IVM8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197208\n",
      "\n",
      "Gene ensembl id ENSG00000197208 changed to uniprot id : Q9H015.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175003\n",
      "\n",
      "Gene ensembl id ENSG00000175003 changed to uniprot id : O15245.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197375\n",
      "\n",
      "Gene ensembl id ENSG00000197375 changed to uniprot id : O76082.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145214\n",
      "\n",
      "Gene ensembl id ENSG00000145214 changed to uniprot id : P52824.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000058866\n",
      "\n",
      "Gene ensembl id ENSG00000058866 changed to uniprot id : P49619.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102780\n",
      "\n",
      "Gene ensembl id ENSG00000102780 changed to uniprot id : Q86XP1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000153933\n",
      "\n",
      "Gene ensembl id ENSG00000153933 changed to uniprot id : P52429.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000065357\n",
      "\n",
      "Gene ensembl id ENSG00000065357 changed to uniprot id : F8W1M3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157680\n",
      "\n",
      "Gene ensembl id ENSG00000157680 changed to uniprot id : O75912.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149091\n",
      "\n",
      "Gene ensembl id ENSG00000149091 changed to uniprot id : Q13574.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204466\n",
      "\n",
      "No uniprot id found for gene ENSG00000204466.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000077044\n",
      "\n",
      "Gene ensembl id ENSG00000077044 changed to uniprot id : H7C0L1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136267\n",
      "\n",
      "Gene ensembl id ENSG00000136267 changed to uniprot id : Q9Y6T7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124003\n",
      "\n",
      "Gene ensembl id ENSG00000124003 changed to uniprot id : Q96PD6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000062282\n",
      "\n",
      "Gene ensembl id ENSG00000062282 changed to uniprot id : Q96PD7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166391\n",
      "\n",
      "Gene ensembl id ENSG00000166391 changed to uniprot id : Q3SYC2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185000\n",
      "\n",
      "Gene ensembl id ENSG00000185000 changed to uniprot id : O75907.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106384\n",
      "\n",
      "Gene ensembl id ENSG00000106384 changed to uniprot id : Q86VF5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000203837\n",
      "\n",
      "Gene ensembl id ENSG00000203837 changed to uniprot id : Q17RR3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175535\n",
      "\n",
      "Gene ensembl id ENSG00000175535 changed to uniprot id : P16233.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170835\n",
      "\n",
      "Gene ensembl id ENSG00000170835 changed to uniprot id : P19835.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165862\n",
      "\n",
      "No uniprot id found for gene ENSG00000165862.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114771\n",
      "\n",
      "Gene ensembl id ENSG00000114771 changed to uniprot id : P22760.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000079435\n",
      "\n",
      "Gene ensembl id ENSG00000079435 changed to uniprot id : Q05469.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101670\n",
      "\n",
      "Gene ensembl id ENSG00000101670 changed to uniprot id : Q9Y5X9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000006757\n",
      "\n",
      "Gene ensembl id ENSG00000006757 changed to uniprot id : P41247.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000187021\n",
      "\n",
      "Gene ensembl id ENSG00000187021 changed to uniprot id : P54315.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182333\n",
      "\n",
      "Gene ensembl id ENSG00000182333 changed to uniprot id : P07098.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166035\n",
      "\n",
      "Gene ensembl id ENSG00000166035 changed to uniprot id : P11150.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000074416\n",
      "\n",
      "Gene ensembl id ENSG00000074416 changed to uniprot id : Q99685.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163686\n",
      "\n",
      "Gene ensembl id ENSG00000163686 changed to uniprot id : Q9BV23.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116906\n",
      "\n",
      "Gene ensembl id ENSG00000116906 changed to uniprot id : O15228.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000107537\n",
      "\n",
      "Gene ensembl id ENSG00000107537 changed to uniprot id : O14832.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131373\n",
      "\n",
      "Gene ensembl id ENSG00000131373 changed to uniprot id : B3KPX4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148459\n",
      "\n",
      "Gene ensembl id ENSG00000148459 changed to uniprot id : Q5T2R2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164494\n",
      "\n",
      "Gene ensembl id ENSG00000164494 changed to uniprot id : Q86YH6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173085\n",
      "\n",
      "Gene ensembl id ENSG00000173085 changed to uniprot id : A0A1D8H0A6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119723\n",
      "\n",
      "Gene ensembl id ENSG00000119723 changed to uniprot id : Q9Y2Z9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132423\n",
      "\n",
      "Gene ensembl id ENSG00000132423 changed to uniprot id : Q9NZJ6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110871\n",
      "\n",
      "Gene ensembl id ENSG00000110871 changed to uniprot id : Q5HYK3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167186\n",
      "\n",
      "Gene ensembl id ENSG00000167186 changed to uniprot id : Q99807.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125954\n",
      "\n",
      "Gene ensembl id ENSG00000125954 changed to uniprot id : B4DL54.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168522\n",
      "\n",
      "Gene ensembl id ENSG00000168522 changed to uniprot id : P49354.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116237\n",
      "\n",
      "Gene ensembl id ENSG00000116237 changed to uniprot id : O60725.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116005\n",
      "\n",
      "Gene ensembl id ENSG00000116005 changed to uniprot id : Q9UHG3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174448\n",
      "\n",
      "Gene ensembl id ENSG00000174448 changed to uniprot id : P59095.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172345\n",
      "\n",
      "Gene ensembl id ENSG00000172345 changed to uniprot id : Q9NSY2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133121\n",
      "\n",
      "Gene ensembl id ENSG00000133121 changed to uniprot id : Q9Y3M8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000084090\n",
      "\n",
      "Gene ensembl id ENSG00000084090 changed to uniprot id : Q9NQZ5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159433\n",
      "\n",
      "Gene ensembl id ENSG00000159433 changed to uniprot id : Q9P2P6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164211\n",
      "\n",
      "Gene ensembl id ENSG00000164211 changed to uniprot id : Q96DR4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145476\n",
      "\n",
      "Gene ensembl id ENSG00000145476 changed to uniprot id : Q6ZWL3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000231852\n",
      "\n",
      "Gene ensembl id ENSG00000231852 changed to uniprot id : P08686.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148795\n",
      "\n",
      "Gene ensembl id ENSG00000148795 changed to uniprot id : P05093.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151632\n",
      "\n",
      "Gene ensembl id ENSG00000151632 changed to uniprot id : P52895.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196139\n",
      "\n",
      "Gene ensembl id ENSG00000196139 changed to uniprot id : P42330.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000203857\n",
      "\n",
      "Gene ensembl id ENSG00000203857 changed to uniprot id : P14060.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000203859\n",
      "\n",
      "Gene ensembl id ENSG00000203859 changed to uniprot id : P26439.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140459\n",
      "\n",
      "Gene ensembl id ENSG00000140459 changed to uniprot id : P05108.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000086696\n",
      "\n",
      "Gene ensembl id ENSG00000086696 changed to uniprot id : P37059.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000025423\n",
      "\n",
      "Gene ensembl id ENSG00000025423 changed to uniprot id : O14756.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000099251\n",
      "\n",
      "No uniprot id found for gene ENSG00000099251.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108786\n",
      "\n",
      "Gene ensembl id ENSG00000108786 changed to uniprot id : P14061.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132196\n",
      "\n",
      "Gene ensembl id ENSG00000132196 changed to uniprot id : P56937.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105398\n",
      "\n",
      "Gene ensembl id ENSG00000105398 changed to uniprot id : Q06520.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000088002\n",
      "\n",
      "Gene ensembl id ENSG00000088002 changed to uniprot id : O00204.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128039\n",
      "\n",
      "Gene ensembl id ENSG00000128039 changed to uniprot id : Q9H8P0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122787\n",
      "\n",
      "Gene ensembl id ENSG00000122787 changed to uniprot id : P51857.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113161\n",
      "\n",
      "Gene ensembl id ENSG00000113161 changed to uniprot id : P04035.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110921\n",
      "\n",
      "Gene ensembl id ENSG00000110921 changed to uniprot id : F5H163.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163344\n",
      "\n",
      "Gene ensembl id ENSG00000163344 changed to uniprot id : Q15126.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148377\n",
      "\n",
      "Gene ensembl id ENSG00000148377 changed to uniprot id : Q9BXS1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000067064\n",
      "\n",
      "Gene ensembl id ENSG00000067064 changed to uniprot id : Q13907.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000079459\n",
      "\n",
      "Gene ensembl id ENSG00000079459 changed to uniprot id : Q6IAX1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104549\n",
      "\n",
      "Gene ensembl id ENSG00000104549 changed to uniprot id : Q14534.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160285\n",
      "\n",
      "Gene ensembl id ENSG00000160285 changed to uniprot id : P48449.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000001630\n",
      "\n",
      "Gene ensembl id ENSG00000001630 changed to uniprot id : Q16850.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149809\n",
      "\n",
      "Gene ensembl id ENSG00000149809 changed to uniprot id : A0A024R577.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170271\n",
      "\n",
      "Gene ensembl id ENSG00000170271 changed to uniprot id : Q96IV6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000052802\n",
      "\n",
      "Gene ensembl id ENSG00000052802 changed to uniprot id : Q15800.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147383\n",
      "\n",
      "Gene ensembl id ENSG00000147383 changed to uniprot id : Q15738.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147155\n",
      "\n",
      "Gene ensembl id ENSG00000147155 changed to uniprot id : A0A024QYX0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109929\n",
      "\n",
      "Gene ensembl id ENSG00000109929 changed to uniprot id : O75845.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172893\n",
      "\n",
      "Gene ensembl id ENSG00000172893 changed to uniprot id : B4E1K5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167508\n",
      "\n",
      "Gene ensembl id ENSG00000167508 changed to uniprot id : P53602.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135929\n",
      "\n",
      "Gene ensembl id ENSG00000135929 changed to uniprot id : Q02318.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146151\n",
      "\n",
      "Gene ensembl id ENSG00000146151 changed to uniprot id : B7Z212.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000006756\n",
      "\n",
      "Gene ensembl id ENSG00000006756 changed to uniprot id : P51689.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000107798\n",
      "\n",
      "Gene ensembl id ENSG00000107798 changed to uniprot id : P38571.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000057252\n",
      "\n",
      "Gene ensembl id ENSG00000057252 changed to uniprot id : P35610.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167780\n",
      "\n",
      "Gene ensembl id ENSG00000167780 changed to uniprot id : O75908.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168350\n",
      "\n",
      "Gene ensembl id ENSG00000168350 changed to uniprot id : Q6QHC5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000177076\n",
      "\n",
      "Gene ensembl id ENSG00000177076 changed to uniprot id : Q5QJU3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167769\n",
      "\n",
      "Gene ensembl id ENSG00000167769 changed to uniprot id : Q8TDN7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104763\n",
      "\n",
      "Gene ensembl id ENSG00000104763 changed to uniprot id : Q13510.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198964\n",
      "\n",
      "Gene ensembl id ENSG00000198964 changed to uniprot id : Q86VZ5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164023\n",
      "\n",
      "Gene ensembl id ENSG00000164023 changed to uniprot id : Q8NHU3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100596\n",
      "\n",
      "Gene ensembl id ENSG00000100596 changed to uniprot id : O15270.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000090054\n",
      "\n",
      "Gene ensembl id ENSG00000090054 changed to uniprot id : A0A8I5QJP4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119537\n",
      "\n",
      "Gene ensembl id ENSG00000119537 changed to uniprot id : Q06136.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176170\n",
      "\n",
      "Gene ensembl id ENSG00000176170 changed to uniprot id : Q9NYA1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000063176\n",
      "\n",
      "Gene ensembl id ENSG00000063176 changed to uniprot id : Q9NRA0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101577\n",
      "\n",
      "Gene ensembl id ENSG00000101577 changed to uniprot id : Q92539.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162407\n",
      "\n",
      "Gene ensembl id ENSG00000162407 changed to uniprot id : O14495.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141934\n",
      "\n",
      "Gene ensembl id ENSG00000141934 changed to uniprot id : O43688.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134324\n",
      "\n",
      "Gene ensembl id ENSG00000134324 changed to uniprot id : Q14693.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000067113\n",
      "\n",
      "Gene ensembl id ENSG00000067113 changed to uniprot id : O14494.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166224\n",
      "\n",
      "Gene ensembl id ENSG00000166224 changed to uniprot id : O95470.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000090661\n",
      "\n",
      "Gene ensembl id ENSG00000090661 changed to uniprot id : Q9HA82.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154227\n",
      "\n",
      "Gene ensembl id ENSG00000154227 changed to uniprot id : Q8IU89.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000223802\n",
      "\n",
      "Gene ensembl id ENSG00000223802 changed to uniprot id : P27544.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172292\n",
      "\n",
      "Gene ensembl id ENSG00000172292 changed to uniprot id : Q6ZMG9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139624\n",
      "\n",
      "Gene ensembl id ENSG00000139624 changed to uniprot id : Q8N5B7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143418\n",
      "\n",
      "Gene ensembl id ENSG00000143418 changed to uniprot id : Q96G23.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143753\n",
      "\n",
      "Gene ensembl id ENSG00000143753 changed to uniprot id : O15121.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100422\n",
      "\n",
      "Gene ensembl id ENSG00000100422 changed to uniprot id : Q8TCT0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148154\n",
      "\n",
      "Gene ensembl id ENSG00000148154 changed to uniprot id : Q16739.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070610\n",
      "\n",
      "Gene ensembl id ENSG00000070610 changed to uniprot id : Q9HCG7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000177628\n",
      "\n",
      "Gene ensembl id ENSG00000177628 changed to uniprot id : P04062.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000085998\n",
      "\n",
      "Gene ensembl id ENSG00000085998 changed to uniprot id : A0A8I5KR25.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000179913\n",
      "\n",
      "Gene ensembl id ENSG00000179913 changed to uniprot id : Q9Y2A9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183778\n",
      "\n",
      "Gene ensembl id ENSG00000183778 changed to uniprot id : Q9Y2C3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148288\n",
      "\n",
      "Gene ensembl id ENSG00000148288 changed to uniprot id : Q8N5D6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000118017\n",
      "\n",
      "Gene ensembl id ENSG00000118017 changed to uniprot id : Q9UNA3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124091\n",
      "\n",
      "No uniprot id found for gene ENSG00000124091.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162885\n",
      "\n",
      "Gene ensembl id ENSG00000162885 changed to uniprot id : A0A087WY64.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000060642\n",
      "\n",
      "Gene ensembl id ENSG00000060642 changed to uniprot id : A0A6I8PU65.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139044\n",
      "\n",
      "Gene ensembl id ENSG00000139044 changed to uniprot id : Q6L9W6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172461\n",
      "\n",
      "Gene ensembl id ENSG00000172461 changed to uniprot id : Q9Y231.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121578\n",
      "\n",
      "Gene ensembl id ENSG00000121578 changed to uniprot id : O60513.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136731\n",
      "\n",
      "Gene ensembl id ENSG00000136731 changed to uniprot id : Q9NYU2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196968\n",
      "\n",
      "Gene ensembl id ENSG00000196968 changed to uniprot id : Q495W5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120820\n",
      "\n",
      "Gene ensembl id ENSG00000120820 changed to uniprot id : Q9H1C3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172318\n",
      "\n",
      "Gene ensembl id ENSG00000172318 changed to uniprot id : Q9Y5Z6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119227\n",
      "\n",
      "Gene ensembl id ENSG00000119227 changed to uniprot id : Q86VD9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000016864\n",
      "\n",
      "Gene ensembl id ENSG00000016864 changed to uniprot id : Q68CQ7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147162\n",
      "\n",
      "Gene ensembl id ENSG00000147162 changed to uniprot id : O15294.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182272\n",
      "\n",
      "Gene ensembl id ENSG00000182272 changed to uniprot id : Q76KP1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000069943\n",
      "\n",
      "Gene ensembl id ENSG00000069943 changed to uniprot id : Q92521.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000118276\n",
      "\n",
      "Gene ensembl id ENSG00000118276 changed to uniprot id : Q9UBX8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000205301\n",
      "\n",
      "Gene ensembl id ENSG00000205301 changed to uniprot id : A6NG13.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000237172\n",
      "\n",
      "Gene ensembl id ENSG00000237172 changed to uniprot id : Q6UX72.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204007\n",
      "\n",
      "Gene ensembl id ENSG00000204007 changed to uniprot id : Q7Z4J2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176597\n",
      "\n",
      "Gene ensembl id ENSG00000176597 changed to uniprot id : Q9BYG0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175711\n",
      "\n",
      "Gene ensembl id ENSG00000175711 changed to uniprot id : I3L232.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000027847\n",
      "\n",
      "Gene ensembl id ENSG00000027847 changed to uniprot id : Q9UBV7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167080\n",
      "\n",
      "Gene ensembl id ENSG00000167080 changed to uniprot id : Q8NHY0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170340\n",
      "\n",
      "Gene ensembl id ENSG00000170340 changed to uniprot id : Q9NY97.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105520\n",
      "\n",
      "Gene ensembl id ENSG00000105520 changed to uniprot id : A0A024R7E2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117600\n",
      "\n",
      "No uniprot id found for gene ENSG00000117600.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000203805\n",
      "\n",
      "Gene ensembl id ENSG00000203805 changed to uniprot id : Q5VZY2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147535\n",
      "\n",
      "Gene ensembl id ENSG00000147535 changed to uniprot id : Q8NEB5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166311\n",
      "\n",
      "Gene ensembl id ENSG00000166311 changed to uniprot id : P17405.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103056\n",
      "\n",
      "Gene ensembl id ENSG00000103056 changed to uniprot id : Q9NY59.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135587\n",
      "\n",
      "Gene ensembl id ENSG00000135587 changed to uniprot id : O60906.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128274\n",
      "\n",
      "Gene ensembl id ENSG00000128274 changed to uniprot id : Q9NPC4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169255\n",
      "\n",
      "Gene ensembl id ENSG00000169255 changed to uniprot id : O75752.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135454\n",
      "\n",
      "Gene ensembl id ENSG00000135454 changed to uniprot id : Q00973.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000206285\n",
      "\n",
      "Gene ensembl id ENSG00000206285 changed to uniprot id : O96024.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000235863\n",
      "\n",
      "Gene ensembl id ENSG00000235863 changed to uniprot id : O96024.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160408\n",
      "\n",
      "Gene ensembl id ENSG00000160408 changed to uniprot id : Q969X2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070731\n",
      "\n",
      "Gene ensembl id ENSG00000070731 changed to uniprot id : Q9UJ37.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111728\n",
      "\n",
      "Gene ensembl id ENSG00000111728 changed to uniprot id : Q92185.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101638\n",
      "\n",
      "Gene ensembl id ENSG00000101638 changed to uniprot id : O15466.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115525\n",
      "\n",
      "Gene ensembl id ENSG00000115525 changed to uniprot id : Q9UNP4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198951\n",
      "\n",
      "Gene ensembl id ENSG00000198951 changed to uniprot id : P17050.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128242\n",
      "\n",
      "Gene ensembl id ENSG00000128242 changed to uniprot id : Q99999.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100299\n",
      "\n",
      "Gene ensembl id ENSG00000100299 changed to uniprot id : A0A0C4DFZ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169359\n",
      "\n",
      "Gene ensembl id ENSG00000169359 changed to uniprot id : O00400.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000126821\n",
      "\n",
      "Gene ensembl id ENSG00000126821 changed to uniprot id : Q9BX95.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158669\n",
      "\n",
      "Gene ensembl id ENSG00000158669 changed to uniprot id : Q86UL3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119927\n",
      "\n",
      "Gene ensembl id ENSG00000119927 changed to uniprot id : Q5VW52.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138678\n",
      "\n",
      "Gene ensembl id ENSG00000138678 changed to uniprot id : A0A024RDG5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169692\n",
      "\n",
      "Gene ensembl id ENSG00000169692 changed to uniprot id : O15120.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000026652\n",
      "\n",
      "Gene ensembl id ENSG00000026652 changed to uniprot id : Q9NRZ5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155189\n",
      "\n",
      "Gene ensembl id ENSG00000155189 changed to uniprot id : Q9NUQ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160216\n",
      "\n",
      "Gene ensembl id ENSG00000160216 changed to uniprot id : Q9NRZ7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204310\n",
      "\n",
      "Gene ensembl id ENSG00000204310 changed to uniprot id : Q99943.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000206324\n",
      "\n",
      "Gene ensembl id ENSG00000206324 changed to uniprot id : Q99943.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000006530\n",
      "\n",
      "Gene ensembl id ENSG00000006530 changed to uniprot id : Q53H12.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164089\n",
      "\n",
      "Gene ensembl id ENSG00000164089 changed to uniprot id : Q8TBG4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163283\n",
      "\n",
      "Gene ensembl id ENSG00000163283 changed to uniprot id : P05187.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162551\n",
      "\n",
      "Gene ensembl id ENSG00000162551 changed to uniprot id : A0A024RAB4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163286\n",
      "\n",
      "Gene ensembl id ENSG00000163286 changed to uniprot id : P10696.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163295\n",
      "\n",
      "Gene ensembl id ENSG00000163295 changed to uniprot id : P09923.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170890\n",
      "\n",
      "Gene ensembl id ENSG00000170890 changed to uniprot id : Q9BS22.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000188784\n",
      "\n",
      "Gene ensembl id ENSG00000188784 changed to uniprot id : Q9NZK7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158786\n",
      "\n",
      "No uniprot id found for gene ENSG00000158786.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117215\n",
      "\n",
      "Gene ensembl id ENSG00000117215 changed to uniprot id : Q9UNK4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163803\n",
      "\n",
      "Gene ensembl id ENSG00000163803 changed to uniprot id : Q6P1J6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000188089\n",
      "\n",
      "Gene ensembl id ENSG00000188089 changed to uniprot id : Q3MJ16.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184381\n",
      "\n",
      "Gene ensembl id ENSG00000184381 changed to uniprot id : A0A590UJZ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123739\n",
      "\n",
      "Gene ensembl id ENSG00000123739 changed to uniprot id : Q9BZM1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000069764\n",
      "\n",
      "Gene ensembl id ENSG00000069764 changed to uniprot id : O15496.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105499\n",
      "\n",
      "Gene ensembl id ENSG00000105499 changed to uniprot id : Q9UP65.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116711\n",
      "\n",
      "Gene ensembl id ENSG00000116711 changed to uniprot id : P47712.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100078\n",
      "\n",
      "Gene ensembl id ENSG00000100078 changed to uniprot id : Q9NZ20.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159337\n",
      "\n",
      "Gene ensembl id ENSG00000159337 changed to uniprot id : Q86XP0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000127472\n",
      "\n",
      "Gene ensembl id ENSG00000127472 changed to uniprot id : P39877.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168970\n",
      "\n",
      "No uniprot id found for gene ENSG00000168970.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168907\n",
      "\n",
      "Gene ensembl id ENSG00000168907 changed to uniprot id : Q68DD2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176485\n",
      "\n",
      "Gene ensembl id ENSG00000176485 changed to uniprot id : P53816.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000188257\n",
      "\n",
      "Gene ensembl id ENSG00000188257 changed to uniprot id : P14555.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134255\n",
      "\n",
      "Gene ensembl id ENSG00000134255 changed to uniprot id : Q9Y6K0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175066\n",
      "\n",
      "Gene ensembl id ENSG00000175066 changed to uniprot id : Q6ZS86.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196475\n",
      "\n",
      "Gene ensembl id ENSG00000196475 changed to uniprot id : Q14410.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198814\n",
      "\n",
      "Gene ensembl id ENSG00000198814 changed to uniprot id : P32189.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000229894\n",
      "\n",
      "Gene ensembl id ENSG00000229894 changed to uniprot id : Q14409.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111666\n",
      "\n",
      "Gene ensembl id ENSG00000111666 changed to uniprot id : Q8WUD6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167588\n",
      "\n",
      "Gene ensembl id ENSG00000167588 changed to uniprot id : A0A024R138.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152642\n",
      "\n",
      "Gene ensembl id ENSG00000152642 changed to uniprot id : Q8N335.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163624\n",
      "\n",
      "Gene ensembl id ENSG00000163624 changed to uniprot id : Q92903.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101290\n",
      "\n",
      "Gene ensembl id ENSG00000101290 changed to uniprot id : O95674.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000087157\n",
      "\n",
      "Gene ensembl id ENSG00000087157 changed to uniprot id : Q32NB8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103502\n",
      "\n",
      "Gene ensembl id ENSG00000103502 changed to uniprot id : B3KY94.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000241878\n",
      "\n",
      "Gene ensembl id ENSG00000241878 changed to uniprot id : A0A024R1K5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174915\n",
      "\n",
      "Gene ensembl id ENSG00000174915 changed to uniprot id : Q9BVG9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156471\n",
      "\n",
      "Gene ensembl id ENSG00000156471 changed to uniprot id : P48651.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105223\n",
      "\n",
      "Gene ensembl id ENSG00000105223 changed to uniprot id : A0A024R0Q4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129219\n",
      "\n",
      "Gene ensembl id ENSG00000129219 changed to uniprot id : O14939.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000075651\n",
      "\n",
      "Gene ensembl id ENSG00000075651 changed to uniprot id : Q13393.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000179598\n",
      "\n",
      "Gene ensembl id ENSG00000179598 changed to uniprot id : Q8N2A8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120992\n",
      "\n",
      "Gene ensembl id ENSG00000120992 changed to uniprot id : Q6IAQ1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175505\n",
      "\n",
      "Gene ensembl id ENSG00000175505 changed to uniprot id : Q9UBD9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135241\n",
      "\n",
      "Gene ensembl id ENSG00000135241 changed to uniprot id : A0A024R746.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105198\n",
      "\n",
      "Gene ensembl id ENSG00000105198 changed to uniprot id : Q9UHV8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130653\n",
      "\n",
      "Gene ensembl id ENSG00000130653 changed to uniprot id : Q6ZV29.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105205\n",
      "\n",
      "Gene ensembl id ENSG00000105205 changed to uniprot id : Q05315.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000032444\n",
      "\n",
      "Gene ensembl id ENSG00000032444 changed to uniprot id : Q8IY17.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213398\n",
      "\n",
      "Gene ensembl id ENSG00000213398 changed to uniprot id : P04180.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125772\n",
      "\n",
      "Gene ensembl id ENSG00000125772 changed to uniprot id : Q9NPB8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110721\n",
      "\n",
      "Gene ensembl id ENSG00000110721 changed to uniprot id : P35790.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100288\n",
      "\n",
      "Gene ensembl id ENSG00000100288 changed to uniprot id : Q9Y259.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102230\n",
      "\n",
      "Gene ensembl id ENSG00000102230 changed to uniprot id : Q9Y5K3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000161217\n",
      "\n",
      "Gene ensembl id ENSG00000161217 changed to uniprot id : P49585.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143845\n",
      "\n",
      "Gene ensembl id ENSG00000143845 changed to uniprot id : Q9NVF9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139163\n",
      "\n",
      "Gene ensembl id ENSG00000139163 changed to uniprot id : Q9HBU6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185813\n",
      "\n",
      "Gene ensembl id ENSG00000185813 changed to uniprot id : Q99447.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133027\n",
      "\n",
      "Gene ensembl id ENSG00000133027 changed to uniprot id : Q9UBM1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000018510\n",
      "\n",
      "Gene ensembl id ENSG00000018510 changed to uniprot id : O00116.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158006\n",
      "\n",
      "Gene ensembl id ENSG00000158006 changed to uniprot id : Q99487.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000007168\n",
      "\n",
      "Gene ensembl id ENSG00000007168 changed to uniprot id : P43034.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000079462\n",
      "\n",
      "Gene ensembl id ENSG00000079462 changed to uniprot id : A0A024R0L6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168092\n",
      "\n",
      "Gene ensembl id ENSG00000168092 changed to uniprot id : P68402.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149476\n",
      "\n",
      "Gene ensembl id ENSG00000149476 changed to uniprot id : Q3LXA3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000156413\n",
      "\n",
      "Gene ensembl id ENSG00000156413 changed to uniprot id : P51993.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171124\n",
      "\n",
      "Gene ensembl id ENSG00000171124 changed to uniprot id : K7EQ20.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130383\n",
      "\n",
      "Gene ensembl id ENSG00000130383 changed to uniprot id : K7ENC0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000064225\n",
      "\n",
      "Gene ensembl id ENSG00000064225 changed to uniprot id : Q9Y274.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174684\n",
      "\n",
      "Gene ensembl id ENSG00000174684 changed to uniprot id : O43505.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174951\n",
      "\n",
      "Gene ensembl id ENSG00000174951 changed to uniprot id : Q6IZA2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176920\n",
      "\n",
      "Gene ensembl id ENSG00000176920 changed to uniprot id : Q10981.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111846\n",
      "\n",
      "Gene ensembl id ENSG00000111846 changed to uniprot id : Q8N0V5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196743\n",
      "\n",
      "Gene ensembl id ENSG00000196743 changed to uniprot id : P17900.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174607\n",
      "\n",
      "Gene ensembl id ENSG00000174607 changed to uniprot id : Q16880.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000007541\n",
      "\n",
      "Gene ensembl id ENSG00000007541 changed to uniprot id : Q9BRB3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145337\n",
      "\n",
      "Gene ensembl id ENSG00000145337 changed to uniprot id : Q96I23.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185808\n",
      "\n",
      "Gene ensembl id ENSG00000185808 changed to uniprot id : P57054.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165195\n",
      "\n",
      "Gene ensembl id ENSG00000165195 changed to uniprot id : A0A2K4ZA02.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135845\n",
      "\n",
      "Gene ensembl id ENSG00000135845 changed to uniprot id : Q92535.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108474\n",
      "\n",
      "Gene ensembl id ENSG00000108474 changed to uniprot id : Q9Y2B2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184886\n",
      "\n",
      "No uniprot id found for gene ENSG00000184886.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163964\n",
      "\n",
      "Gene ensembl id ENSG00000163964 changed to uniprot id : Q8TBF5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197563\n",
      "\n",
      "Gene ensembl id ENSG00000197563 changed to uniprot id : A0A1W2PNH8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151665\n",
      "\n",
      "Gene ensembl id ENSG00000151665 changed to uniprot id : Q07326.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174227\n",
      "\n",
      "Gene ensembl id ENSG00000174227 changed to uniprot id : Q5H8A4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000087111\n",
      "\n",
      "Gene ensembl id ENSG00000087111 changed to uniprot id : Q96S52.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101464\n",
      "\n",
      "Gene ensembl id ENSG00000101464 changed to uniprot id : Q9H490.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197858\n",
      "\n",
      "Gene ensembl id ENSG00000197858 changed to uniprot id : O43292.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197580\n",
      "\n",
      "Gene ensembl id ENSG00000197580 changed to uniprot id : Q9BYV7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163684\n",
      "\n",
      "Gene ensembl id ENSG00000163684 changed to uniprot id : O95059.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000244486\n",
      "\n",
      "Gene ensembl id ENSG00000244486 changed to uniprot id : Q96GP6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117480\n",
      "\n",
      "Gene ensembl id ENSG00000117480 changed to uniprot id : O00519.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000064763\n",
      "\n",
      "Gene ensembl id ENSG00000064763 changed to uniprot id : Q96K12.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197601\n",
      "\n",
      "Gene ensembl id ENSG00000197601 changed to uniprot id : Q8WVX9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000015532\n",
      "\n",
      "Gene ensembl id ENSG00000015532 changed to uniprot id : B4DT06.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103489\n",
      "\n",
      "Gene ensembl id ENSG00000103489 changed to uniprot id : Q86Y38.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176022\n",
      "\n",
      "Gene ensembl id ENSG00000176022 changed to uniprot id : Q96L58.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109956\n",
      "\n",
      "Gene ensembl id ENSG00000109956 changed to uniprot id : Q9P2W7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112309\n",
      "\n",
      "Gene ensembl id ENSG00000112309 changed to uniprot id : Q9NPZ5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147408\n",
      "\n",
      "Gene ensembl id ENSG00000147408 changed to uniprot id : Q8TDX6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169826\n",
      "\n",
      "Gene ensembl id ENSG00000169826 changed to uniprot id : Q8N6G5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162694\n",
      "\n",
      "Gene ensembl id ENSG00000162694 changed to uniprot id : Q9UBQ6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000012232\n",
      "\n",
      "Gene ensembl id ENSG00000012232 changed to uniprot id : O43909.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182197\n",
      "\n",
      "Gene ensembl id ENSG00000182197 changed to uniprot id : Q16394.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158008\n",
      "\n",
      "Gene ensembl id ENSG00000158008 changed to uniprot id : Q92935.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138604\n",
      "\n",
      "Gene ensembl id ENSG00000138604 changed to uniprot id : O94923.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136213\n",
      "\n",
      "Gene ensembl id ENSG00000136213 changed to uniprot id : A0A024R860.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171310\n",
      "\n",
      "Gene ensembl id ENSG00000171310 changed to uniprot id : Q9NPF2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123989\n",
      "\n",
      "Gene ensembl id ENSG00000123989 changed to uniprot id : Q8IZ52.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000033100\n",
      "\n",
      "Gene ensembl id ENSG00000033100 changed to uniprot id : Q9P2E5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182022\n",
      "\n",
      "Gene ensembl id ENSG00000182022 changed to uniprot id : Q7LFX5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122863\n",
      "\n",
      "Gene ensembl id ENSG00000122863 changed to uniprot id : Q7LGC8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113273\n",
      "\n",
      "Gene ensembl id ENSG00000113273 changed to uniprot id : P15848.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114378\n",
      "\n",
      "Gene ensembl id ENSG00000114378 changed to uniprot id : Q12794.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106302\n",
      "\n",
      "Gene ensembl id ENSG00000106302 changed to uniprot id : Q2M3T9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186792\n",
      "\n",
      "Gene ensembl id ENSG00000186792 changed to uniprot id : O43820.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169919\n",
      "\n",
      "Gene ensembl id ENSG00000169919 changed to uniprot id : P08236.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133116\n",
      "\n",
      "Gene ensembl id ENSG00000133116 changed to uniprot id : Q9UEF7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000010404\n",
      "\n",
      "Gene ensembl id ENSG00000010404 changed to uniprot id : O60597.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172987\n",
      "\n",
      "Gene ensembl id ENSG00000172987 changed to uniprot id : Q8WWQ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173083\n",
      "\n",
      "Gene ensembl id ENSG00000173083 changed to uniprot id : Q9Y251.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135677\n",
      "\n",
      "Gene ensembl id ENSG00000135677 changed to uniprot id : P15586.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000181523\n",
      "\n",
      "Gene ensembl id ENSG00000181523 changed to uniprot id : P51688.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165102\n",
      "\n",
      "Gene ensembl id ENSG00000165102 changed to uniprot id : E5RJC4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108784\n",
      "\n",
      "Gene ensembl id ENSG00000108784 changed to uniprot id : P54802.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000127415\n",
      "\n",
      "Gene ensembl id ENSG00000127415 changed to uniprot id : P35475.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000126091\n",
      "\n",
      "Gene ensembl id ENSG00000126091 changed to uniprot id : Q11203.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175264\n",
      "\n",
      "Gene ensembl id ENSG00000175264 changed to uniprot id : O43916.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167910\n",
      "\n",
      "Gene ensembl id ENSG00000167910 changed to uniprot id : P22680.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000180432\n",
      "\n",
      "Gene ensembl id ENSG00000180432 changed to uniprot id : Q9UNU6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000083807\n",
      "\n",
      "Gene ensembl id ENSG00000083807 changed to uniprot id : Q9Y2P5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116171\n",
      "\n",
      "Gene ensembl id ENSG00000116171 changed to uniprot id : P22307.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000180879\n",
      "\n",
      "Gene ensembl id ENSG00000180879 changed to uniprot id : P51571.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130822\n",
      "\n",
      "Gene ensembl id ENSG00000130822 changed to uniprot id : Q6P2M8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130821\n",
      "\n",
      "Gene ensembl id ENSG00000130821 changed to uniprot id : P48029.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000067840\n",
      "\n",
      "Gene ensembl id ENSG00000067840 changed to uniprot id : Q76G19.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101986\n",
      "\n",
      "Gene ensembl id ENSG00000101986 changed to uniprot id : P33897.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000215151\n",
      "\n",
      "No uniprot id found for gene ENSG00000215151.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185825\n",
      "\n",
      "Gene ensembl id ENSG00000185825 changed to uniprot id : P51572.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168306\n",
      "\n",
      "Gene ensembl id ENSG00000168306 changed to uniprot id : Q99424.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000036530\n",
      "\n",
      "Gene ensembl id ENSG00000036530 changed to uniprot id : Q9Y6A2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146233\n",
      "\n",
      "Gene ensembl id ENSG00000146233 changed to uniprot id : Q9NYL5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138135\n",
      "\n",
      "Gene ensembl id ENSG00000138135 changed to uniprot id : O95992.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129596\n",
      "\n",
      "Gene ensembl id ENSG00000129596 changed to uniprot id : Q16878.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136750\n",
      "\n",
      "Gene ensembl id ENSG00000136750 changed to uniprot id : Q05329.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139631\n",
      "\n",
      "Gene ensembl id ENSG00000139631 changed to uniprot id : A0A024RAX7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128683\n",
      "\n",
      "Gene ensembl id ENSG00000128683 changed to uniprot id : Q99259.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000181915\n",
      "\n",
      "Gene ensembl id ENSG00000181915 changed to uniprot id : Q96SZ5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000073734\n",
      "\n",
      "Gene ensembl id ENSG00000073734 changed to uniprot id : O95342.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108846\n",
      "\n",
      "Gene ensembl id ENSG00000108846 changed to uniprot id : O15438.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000023839\n",
      "\n",
      "Gene ensembl id ENSG00000023839 changed to uniprot id : Q92887.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103222\n",
      "\n",
      "Gene ensembl id ENSG00000103222 changed to uniprot id : A0A0A0MS99.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163959\n",
      "\n",
      "Gene ensembl id ENSG00000163959 changed to uniprot id : Q86UW1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125257\n",
      "\n",
      "Gene ensembl id ENSG00000125257 changed to uniprot id : O15439.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186198\n",
      "\n",
      "Gene ensembl id ENSG00000186198 changed to uniprot id : Q86UW2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111700\n",
      "\n",
      "Gene ensembl id ENSG00000111700 changed to uniprot id : Q9NPD5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000021488\n",
      "\n",
      "Gene ensembl id ENSG00000021488 changed to uniprot id : P82251.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000084453\n",
      "\n",
      "Gene ensembl id ENSG00000084453 changed to uniprot id : B4DJE6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134538\n",
      "\n",
      "Gene ensembl id ENSG00000134538 changed to uniprot id : Q9Y6L6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101187\n",
      "\n",
      "Gene ensembl id ENSG00000101187 changed to uniprot id : Q96BD0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175164\n",
      "\n",
      "Gene ensembl id ENSG00000175164 changed to uniprot id : W6JLF7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000256062\n",
      "\n",
      "No uniprot id found for gene ENSG00000256062.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162882\n",
      "\n",
      "Gene ensembl id ENSG00000162882 changed to uniprot id : P46952.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103485\n",
      "\n",
      "Gene ensembl id ENSG00000103485 changed to uniprot id : Q15274.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147813\n",
      "\n",
      "Gene ensembl id ENSG00000147813 changed to uniprot id : Q6XQN6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112874\n",
      "\n",
      "Gene ensembl id ENSG00000112874 changed to uniprot id : Q9BQG2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157064\n",
      "\n",
      "Gene ensembl id ENSG00000157064 changed to uniprot id : Q9BZQ4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163864\n",
      "\n",
      "Gene ensembl id ENSG00000163864 changed to uniprot id : A0A2R8YFG2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173614\n",
      "\n",
      "Gene ensembl id ENSG00000173614 changed to uniprot id : Q9HAN9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172890\n",
      "\n",
      "Gene ensembl id ENSG00000172890 changed to uniprot id : Q6IA69.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105835\n",
      "\n",
      "Gene ensembl id ENSG00000105835 changed to uniprot id : P43490.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106733\n",
      "\n",
      "Gene ensembl id ENSG00000106733 changed to uniprot id : Q9NWW6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000008130\n",
      "\n",
      "Gene ensembl id ENSG00000008130 changed to uniprot id : O95544.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152620\n",
      "\n",
      "Gene ensembl id ENSG00000152620 changed to uniprot id : Q4G0N4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162836\n",
      "\n",
      "Gene ensembl id ENSG00000162836 changed to uniprot id : Q9NPH0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142513\n",
      "\n",
      "Gene ensembl id ENSG00000142513 changed to uniprot id : Q9BZG2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102575\n",
      "\n",
      "Gene ensembl id ENSG00000102575 changed to uniprot id : P13686.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155893\n",
      "\n",
      "Gene ensembl id ENSG00000155893 changed to uniprot id : Q8TE99.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112992\n",
      "\n",
      "Gene ensembl id ENSG00000112992 changed to uniprot id : Q13423.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129744\n",
      "\n",
      "Gene ensembl id ENSG00000129744 changed to uniprot id : P52961.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167311\n",
      "\n",
      "Gene ensembl id ENSG00000167311 changed to uniprot id : Q96L15.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000068120\n",
      "\n",
      "Gene ensembl id ENSG00000068120 changed to uniprot id : Q13057.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125779\n",
      "\n",
      "Gene ensembl id ENSG00000125779 changed to uniprot id : Q9BZ23.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000152782\n",
      "\n",
      "Gene ensembl id ENSG00000152782 changed to uniprot id : A0A8C8KBT8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120137\n",
      "\n",
      "Gene ensembl id ENSG00000120137 changed to uniprot id : Q9H999.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157881\n",
      "\n",
      "Gene ensembl id ENSG00000157881 changed to uniprot id : H0Y9E4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112299\n",
      "\n",
      "Gene ensembl id ENSG00000112299 changed to uniprot id : O95497.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112303\n",
      "\n",
      "Gene ensembl id ENSG00000112303 changed to uniprot id : O95498.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000127125\n",
      "\n",
      "Gene ensembl id ENSG00000127125 changed to uniprot id : Q9HAB8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138621\n",
      "\n",
      "Gene ensembl id ENSG00000138621 changed to uniprot id : H3BU63.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134575\n",
      "\n",
      "Gene ensembl id ENSG00000134575 changed to uniprot id : B7Z7D2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115556\n",
      "\n",
      "Gene ensembl id ENSG00000115556 changed to uniprot id : Q9BRC7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139151\n",
      "\n",
      "Gene ensembl id ENSG00000139151 changed to uniprot id : Q8N7S5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149527\n",
      "\n",
      "Gene ensembl id ENSG00000149527 changed to uniprot id : O75038.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197943\n",
      "\n",
      "Gene ensembl id ENSG00000197943 changed to uniprot id : P16885.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000240891\n",
      "\n",
      "Gene ensembl id ENSG00000240891 changed to uniprot id : Q0VAA5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000187091\n",
      "\n",
      "Gene ensembl id ENSG00000187091 changed to uniprot id : P51178.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115896\n",
      "\n",
      "Gene ensembl id ENSG00000115896 changed to uniprot id : Q15111.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137841\n",
      "\n",
      "Gene ensembl id ENSG00000137841 changed to uniprot id : Q9BVT6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124181\n",
      "\n",
      "Gene ensembl id ENSG00000124181 changed to uniprot id : P19174.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101333\n",
      "\n",
      "Gene ensembl id ENSG00000101333 changed to uniprot id : A0A8I5KXD7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182621\n",
      "\n",
      "Gene ensembl id ENSG00000182621 changed to uniprot id : Q9NQ66.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114805\n",
      "\n",
      "Gene ensembl id ENSG00000114805 changed to uniprot id : Q4KWH8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149782\n",
      "\n",
      "Gene ensembl id ENSG00000149782 changed to uniprot id : Q01970.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000161714\n",
      "\n",
      "Gene ensembl id ENSG00000161714 changed to uniprot id : Q8N3E9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138193\n",
      "\n",
      "Gene ensembl id ENSG00000138193 changed to uniprot id : Q9P212.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000154822\n",
      "\n",
      "Gene ensembl id ENSG00000154822 changed to uniprot id : Q9UPR0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105655\n",
      "\n",
      "Gene ensembl id ENSG00000105655 changed to uniprot id : Q9NPH2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100253\n",
      "\n",
      "Gene ensembl id ENSG00000100253 changed to uniprot id : Q9UGB7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141401\n",
      "\n",
      "Gene ensembl id ENSG00000141401 changed to uniprot id : O14732.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133731\n",
      "\n",
      "Gene ensembl id ENSG00000133731 changed to uniprot id : P29218.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000078142\n",
      "\n",
      "Gene ensembl id ENSG00000078142 changed to uniprot id : Q8NEB9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000150712\n",
      "\n",
      "Gene ensembl id ENSG00000150712 changed to uniprot id : Q9C0I1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171100\n",
      "\n",
      "Gene ensembl id ENSG00000171100 changed to uniprot id : Q13496.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115020\n",
      "\n",
      "Gene ensembl id ENSG00000115020 changed to uniprot id : Q9Y2I7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112367\n",
      "\n",
      "Gene ensembl id ENSG00000112367 changed to uniprot id : A0A6Q8PG49.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139304\n",
      "\n",
      "Gene ensembl id ENSG00000139304 changed to uniprot id : F8VW52.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000054148\n",
      "\n",
      "Gene ensembl id ENSG00000054148 changed to uniprot id : Q9NRX4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137770\n",
      "\n",
      "Gene ensembl id ENSG00000137770 changed to uniprot id : Q05D32.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000063601\n",
      "\n",
      "Gene ensembl id ENSG00000063601 changed to uniprot id : Q13613.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198825\n",
      "\n",
      "Gene ensembl id ENSG00000198825 changed to uniprot id : Q9Y2H2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000211456\n",
      "\n",
      "Gene ensembl id ENSG00000211456 changed to uniprot id : Q9NTJ5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111696\n",
      "\n",
      "Gene ensembl id ENSG00000111696 changed to uniprot id : Q86UY8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196455\n",
      "\n",
      "Gene ensembl id ENSG00000196455 changed to uniprot id : Q99570.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000003987\n",
      "\n",
      "Gene ensembl id ENSG00000003987 changed to uniprot id : Q9Y216.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117598\n",
      "\n",
      "Gene ensembl id ENSG00000117598 changed to uniprot id : Q32ZL2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163082\n",
      "\n",
      "Gene ensembl id ENSG00000163082 changed to uniprot id : Q8IWX5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139505\n",
      "\n",
      "Gene ensembl id ENSG00000139505 changed to uniprot id : Q9Y217.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213920\n",
      "\n",
      "Gene ensembl id ENSG00000213920 changed to uniprot id : Q86V88.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111077\n",
      "\n",
      "Gene ensembl id ENSG00000111077 changed to uniprot id : A0A024RB48.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000087053\n",
      "\n",
      "Gene ensembl id ENSG00000087053 changed to uniprot id : A0A6Q8PGT1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165458\n",
      "\n",
      "Gene ensembl id ENSG00000165458 changed to uniprot id : O15357.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000205808\n",
      "\n",
      "Gene ensembl id ENSG00000205808 changed to uniprot id : Q8IY26.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122484\n",
      "\n",
      "Gene ensembl id ENSG00000122484 changed to uniprot id : Q8IXW5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165688\n",
      "\n",
      "Gene ensembl id ENSG00000165688 changed to uniprot id : Q10713.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000241973\n",
      "\n",
      "Gene ensembl id ENSG00000241973 changed to uniprot id : P42356.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143393\n",
      "\n",
      "Gene ensembl id ENSG00000143393 changed to uniprot id : Q9UBF8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155252\n",
      "\n",
      "Gene ensembl id ENSG00000155252 changed to uniprot id : Q9BTU6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000038210\n",
      "\n",
      "Gene ensembl id ENSG00000038210 changed to uniprot id : Q8TCG2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000011405\n",
      "\n",
      "Gene ensembl id ENSG00000011405 changed to uniprot id : O00443.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133056\n",
      "\n",
      "Gene ensembl id ENSG00000133056 changed to uniprot id : O00750.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141506\n",
      "\n",
      "Gene ensembl id ENSG00000141506 changed to uniprot id : Q8WYR1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139144\n",
      "\n",
      "Gene ensembl id ENSG00000139144 changed to uniprot id : F5H369.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109452\n",
      "\n",
      "Gene ensembl id ENSG00000109452 changed to uniprot id : O15327.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000040933\n",
      "\n",
      "Gene ensembl id ENSG00000040933 changed to uniprot id : Q96PE3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171862\n",
      "\n",
      "Gene ensembl id ENSG00000171862 changed to uniprot id : P60484.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186111\n",
      "\n",
      "Gene ensembl id ENSG00000186111 changed to uniprot id : O60331.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143398\n",
      "\n",
      "Gene ensembl id ENSG00000143398 changed to uniprot id : Q99755.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000107242\n",
      "\n",
      "Gene ensembl id ENSG00000107242 changed to uniprot id : O14986.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122126\n",
      "\n",
      "Gene ensembl id ENSG00000122126 changed to uniprot id : Q01968.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204084\n",
      "\n",
      "Gene ensembl id ENSG00000204084 changed to uniprot id : P32019.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159082\n",
      "\n",
      "Gene ensembl id ENSG00000159082 changed to uniprot id : O43426.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148384\n",
      "\n",
      "Gene ensembl id ENSG00000148384 changed to uniprot id : Q9NRR6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000078269\n",
      "\n",
      "Gene ensembl id ENSG00000078269 changed to uniprot id : O15056.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165782\n",
      "\n",
      "Gene ensembl id ENSG00000165782 changed to uniprot id : Q86T03.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155099\n",
      "\n",
      "Gene ensembl id ENSG00000155099 changed to uniprot id : Q8N4L2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166908\n",
      "\n",
      "Gene ensembl id ENSG00000166908 changed to uniprot id : Q8TBX8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141720\n",
      "\n",
      "No uniprot id found for gene ENSG00000141720.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000150867\n",
      "\n",
      "Gene ensembl id ENSG00000150867 changed to uniprot id : P48426.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121879\n",
      "\n",
      "Gene ensembl id ENSG00000121879 changed to uniprot id : P42336.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000126264\n",
      "\n",
      "Gene ensembl id ENSG00000126264 changed to uniprot id : Q9UBK5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171608\n",
      "\n",
      "Gene ensembl id ENSG00000171608 changed to uniprot id : O00329.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105851\n",
      "\n",
      "Gene ensembl id ENSG00000105851 changed to uniprot id : A0A024R720.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117461\n",
      "\n",
      "Gene ensembl id ENSG00000117461 changed to uniprot id : Q92569.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145675\n",
      "\n",
      "Gene ensembl id ENSG00000145675 changed to uniprot id : P27986.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105647\n",
      "\n",
      "Gene ensembl id ENSG00000105647 changed to uniprot id : O00459.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000051382\n",
      "\n",
      "Gene ensembl id ENSG00000051382 changed to uniprot id : P42338.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000068383\n",
      "\n",
      "Gene ensembl id ENSG00000068383 changed to uniprot id : Q14642.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000185133\n",
      "\n",
      "Gene ensembl id ENSG00000185133 changed to uniprot id : B5MBZ3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151689\n",
      "\n",
      "Gene ensembl id ENSG00000151689 changed to uniprot id : P49441.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137825\n",
      "\n",
      "Gene ensembl id ENSG00000137825 changed to uniprot id : P23677.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143772\n",
      "\n",
      "Gene ensembl id ENSG00000143772 changed to uniprot id : P27987.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000086544\n",
      "\n",
      "Gene ensembl id ENSG00000086544 changed to uniprot id : Q96DU7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000107789\n",
      "\n",
      "Gene ensembl id ENSG00000107789 changed to uniprot id : Q9UNW1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168918\n",
      "\n",
      "Gene ensembl id ENSG00000168918 changed to uniprot id : Q92835.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100605\n",
      "\n",
      "Gene ensembl id ENSG00000100605 changed to uniprot id : Q13572.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151151\n",
      "\n",
      "Gene ensembl id ENSG00000151151 changed to uniprot id : Q8NFU5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000127080\n",
      "\n",
      "Gene ensembl id ENSG00000127080 changed to uniprot id : Q9H8X2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168781\n",
      "\n",
      "Gene ensembl id ENSG00000168781 changed to uniprot id : Q6PFW1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145725\n",
      "\n",
      "Gene ensembl id ENSG00000145725 changed to uniprot id : A0A087WZV0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000068745\n",
      "\n",
      "Gene ensembl id ENSG00000068745 changed to uniprot id : B2RCP4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151552\n",
      "\n",
      "Gene ensembl id ENSG00000151552 changed to uniprot id : P09417.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000228716\n",
      "\n",
      "Gene ensembl id ENSG00000228716 changed to uniprot id : B4DM58.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136010\n",
      "\n",
      "Gene ensembl id ENSG00000136010 changed to uniprot id : Q3SY69.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000144908\n",
      "\n",
      "Gene ensembl id ENSG00000144908 changed to uniprot id : O75891.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182021\n",
      "\n",
      "No uniprot id found for gene ENSG00000182021.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100714\n",
      "\n",
      "Gene ensembl id ENSG00000100714 changed to uniprot id : P11586.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000065911\n",
      "\n",
      "Gene ensembl id ENSG00000065911 changed to uniprot id : P13995.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163738\n",
      "\n",
      "Gene ensembl id ENSG00000163738 changed to uniprot id : Q9H903.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120254\n",
      "\n",
      "Gene ensembl id ENSG00000120254 changed to uniprot id : Q6UB35.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105618\n",
      "\n",
      "Gene ensembl id ENSG00000105618 changed to uniprot id : E7EU94.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136371\n",
      "\n",
      "Gene ensembl id ENSG00000136371 changed to uniprot id : P49914.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124615\n",
      "\n",
      "Gene ensembl id ENSG00000124615 changed to uniprot id : Q9NZB8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164172\n",
      "\n",
      "Gene ensembl id ENSG00000164172 changed to uniprot id : O96033.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171723\n",
      "\n",
      "Gene ensembl id ENSG00000171723 changed to uniprot id : G3V4R0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000244681\n",
      "\n",
      "No uniprot id found for gene ENSG00000244681.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169814\n",
      "\n",
      "Gene ensembl id ENSG00000169814 changed to uniprot id : A0A2R8Y5J9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159267\n",
      "\n",
      "Gene ensembl id ENSG00000159267 changed to uniprot id : P50747.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131979\n",
      "\n",
      "Gene ensembl id ENSG00000131979 changed to uniprot id : P30793.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000150787\n",
      "\n",
      "Gene ensembl id ENSG00000150787 changed to uniprot id : E9PKY8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116096\n",
      "\n",
      "Gene ensembl id ENSG00000116096 changed to uniprot id : P35270.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132570\n",
      "\n",
      "Gene ensembl id ENSG00000132570 changed to uniprot id : Q9H0N5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166228\n",
      "\n",
      "Gene ensembl id ENSG00000166228 changed to uniprot id : P61457.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173221\n",
      "\n",
      "Gene ensembl id ENSG00000173221 changed to uniprot id : P35754.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000023572\n",
      "\n",
      "Gene ensembl id ENSG00000023572 changed to uniprot id : Q9NS18.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167996\n",
      "\n",
      "Gene ensembl id ENSG00000167996 changed to uniprot id : P02794.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165060\n",
      "\n",
      "Gene ensembl id ENSG00000165060 changed to uniprot id : Q16595.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000047457\n",
      "\n",
      "Gene ensembl id ENSG00000047457 changed to uniprot id : D6RE86.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163755\n",
      "\n",
      "Gene ensembl id ENSG00000163755 changed to uniprot id : Q969F9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000181867\n",
      "\n",
      "Gene ensembl id ENSG00000181867 changed to uniprot id : Q8N4E7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148218\n",
      "\n",
      "Gene ensembl id ENSG00000148218 changed to uniprot id : P13716.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000256269\n",
      "\n",
      "Gene ensembl id ENSG00000256269 changed to uniprot id : P08397.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000188690\n",
      "\n",
      "Gene ensembl id ENSG00000188690 changed to uniprot id : Q5T3L8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000126088\n",
      "\n",
      "Gene ensembl id ENSG00000126088 changed to uniprot id : P06132.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000080819\n",
      "\n",
      "Gene ensembl id ENSG00000080819 changed to uniprot id : P36551.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143224\n",
      "\n",
      "Gene ensembl id ENSG00000143224 changed to uniprot id : D3DVG2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000066926\n",
      "\n",
      "Gene ensembl id ENSG00000066926 changed to uniprot id : P22830.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000004961\n",
      "\n",
      "Gene ensembl id ENSG00000004961 changed to uniprot id : P53701.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000072042\n",
      "\n",
      "Gene ensembl id ENSG00000072042 changed to uniprot id : Q8TC12.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162496\n",
      "\n",
      "Gene ensembl id ENSG00000162496 changed to uniprot id : O75911.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000080511\n",
      "\n",
      "Gene ensembl id ENSG00000080511 changed to uniprot id : Q9NYR8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000073737\n",
      "\n",
      "Gene ensembl id ENSG00000073737 changed to uniprot id : Q9BPW9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135437\n",
      "\n",
      "Gene ensembl id ENSG00000135437 changed to uniprot id : Q92781.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000095596\n",
      "\n",
      "Gene ensembl id ENSG00000095596 changed to uniprot id : O43174.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121207\n",
      "\n",
      "Gene ensembl id ENSG00000121207 changed to uniprot id : O95237.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000114115\n",
      "\n",
      "Gene ensembl id ENSG00000114115 changed to uniprot id : P09455.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000159398\n",
      "\n",
      "Gene ensembl id ENSG00000159398 changed to uniprot id : Q6NT32.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198848\n",
      "\n",
      "Gene ensembl id ENSG00000198848 changed to uniprot id : P23141.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172828\n",
      "\n",
      "Gene ensembl id ENSG00000172828 changed to uniprot id : A0A0C4DFY7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138207\n",
      "\n",
      "Gene ensembl id ENSG00000138207 changed to uniprot id : Q5VY30.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000172831\n",
      "\n",
      "Gene ensembl id ENSG00000172831 changed to uniprot id : O00748.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000187630\n",
      "\n",
      "Gene ensembl id ENSG00000187630 changed to uniprot id : F6TD35.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139547\n",
      "\n",
      "Gene ensembl id ENSG00000139547 changed to uniprot id : O75452.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000003137\n",
      "\n",
      "Gene ensembl id ENSG00000003137 changed to uniprot id : Q9NR63.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000128918\n",
      "\n",
      "Gene ensembl id ENSG00000128918 changed to uniprot id : O94788.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165092\n",
      "\n",
      "Gene ensembl id ENSG00000165092 changed to uniprot id : P00352.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135002\n",
      "\n",
      "Gene ensembl id ENSG00000135002 changed to uniprot id : Q969G6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160688\n",
      "\n",
      "Gene ensembl id ENSG00000160688 changed to uniprot id : Q8NFF5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163352\n",
      "\n",
      "Gene ensembl id ENSG00000163352 changed to uniprot id : Q9Y5L5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000090013\n",
      "\n",
      "Gene ensembl id ENSG00000090013 changed to uniprot id : P30043.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129167\n",
      "\n",
      "Gene ensembl id ENSG00000129167 changed to uniprot id : P17752.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139287\n",
      "\n",
      "Gene ensembl id ENSG00000139287 changed to uniprot id : Q8IWU9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196511\n",
      "\n",
      "Gene ensembl id ENSG00000196511 changed to uniprot id : Q9H3S4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000259431\n",
      "\n",
      "Gene ensembl id ENSG00000259431 changed to uniprot id : Q9BU02.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135778\n",
      "\n",
      "Gene ensembl id ENSG00000135778 changed to uniprot id : Q9BSD7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151611\n",
      "\n",
      "Gene ensembl id ENSG00000151611 changed to uniprot id : Q8IVH4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139428\n",
      "\n",
      "Gene ensembl id ENSG00000139428 changed to uniprot id : Q96EY8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100092\n",
      "\n",
      "Gene ensembl id ENSG00000100092 changed to uniprot id : Q9Y3L3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000144362\n",
      "\n",
      "Gene ensembl id ENSG00000144362 changed to uniprot id : Q8TCD6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000241360\n",
      "\n",
      "Gene ensembl id ENSG00000241360 changed to uniprot id : Q96GD0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160209\n",
      "\n",
      "Gene ensembl id ENSG00000160209 changed to uniprot id : O00764.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108439\n",
      "\n",
      "Gene ensembl id ENSG00000108439 changed to uniprot id : Q9NVS9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000019186\n",
      "\n",
      "Gene ensembl id ENSG00000019186 changed to uniprot id : Q07973.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104808\n",
      "\n",
      "Gene ensembl id ENSG00000104808 changed to uniprot id : Q9UQ10.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117594\n",
      "\n",
      "Gene ensembl id ENSG00000117594 changed to uniprot id : P28845.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139914\n",
      "\n",
      "Gene ensembl id ENSG00000139914 changed to uniprot id : A5D6W6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167676\n",
      "\n",
      "Gene ensembl id ENSG00000167676 changed to uniprot id : A0A0J9YXN7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105355\n",
      "\n",
      "Gene ensembl id ENSG00000105355 changed to uniprot id : O60664.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213639\n",
      "\n",
      "Gene ensembl id ENSG00000213639 changed to uniprot id : P62140.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000072062\n",
      "\n",
      "Gene ensembl id ENSG00000072062 changed to uniprot id : P17612.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000186298\n",
      "\n",
      "Gene ensembl id ENSG00000186298 changed to uniprot id : A0A024RBP2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170323\n",
      "\n",
      "Gene ensembl id ENSG00000170323 changed to uniprot id : E7DVW4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142875\n",
      "\n",
      "Gene ensembl id ENSG00000142875 changed to uniprot id : P22694.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113396\n",
      "\n",
      "Gene ensembl id ENSG00000113396 changed to uniprot id : Q9Y2P4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163586\n",
      "\n",
      "Gene ensembl id ENSG00000163586 changed to uniprot id : P07148.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164687\n",
      "\n",
      "Gene ensembl id ENSG00000164687 changed to uniprot id : Q01469.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197416\n",
      "\n",
      "Gene ensembl id ENSG00000197416 changed to uniprot id : A6NFH5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000166747\n",
      "\n",
      "Gene ensembl id ENSG00000166747 changed to uniprot id : O43747.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165029\n",
      "\n",
      "Gene ensembl id ENSG00000165029 changed to uniprot id : O95477.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165269\n",
      "\n",
      "Gene ensembl id ENSG00000165269 changed to uniprot id : O14520.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165272\n",
      "\n",
      "Gene ensembl id ENSG00000165272 changed to uniprot id : Q92482.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000005471\n",
      "\n",
      "Gene ensembl id ENSG00000005471 changed to uniprot id : P21439.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139433\n",
      "\n",
      "Gene ensembl id ENSG00000139433 changed to uniprot id : F5GZ49.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138075\n",
      "\n",
      "Gene ensembl id ENSG00000138075 changed to uniprot id : Q9H222.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160179\n",
      "\n",
      "Gene ensembl id ENSG00000160179 changed to uniprot id : P45844.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143921\n",
      "\n",
      "Gene ensembl id ENSG00000143921 changed to uniprot id : Q9H221.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000145217\n",
      "\n",
      "Gene ensembl id ENSG00000145217 changed to uniprot id : Q9H2B4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100156\n",
      "\n",
      "Gene ensembl id ENSG00000100156 changed to uniprot id : O95907.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170190\n",
      "\n",
      "Gene ensembl id ENSG00000170190 changed to uniprot id : O15375.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000118596\n",
      "\n",
      "Gene ensembl id ENSG00000118596 changed to uniprot id : O60669.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141526\n",
      "\n",
      "Gene ensembl id ENSG00000141526 changed to uniprot id : O15427.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168679\n",
      "\n",
      "Gene ensembl id ENSG00000168679 changed to uniprot id : O15374.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155380\n",
      "\n",
      "Gene ensembl id ENSG00000155380 changed to uniprot id : P53985.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108932\n",
      "\n",
      "Gene ensembl id ENSG00000108932 changed to uniprot id : A0A024R8J3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197241\n",
      "\n",
      "Gene ensembl id ENSG00000197241 changed to uniprot id : Q6PXP3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000136856\n",
      "\n",
      "Gene ensembl id ENSG00000136856 changed to uniprot id : Q9NY64.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109667\n",
      "\n",
      "Gene ensembl id ENSG00000109667 changed to uniprot id : Q9NRM0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163581\n",
      "\n",
      "Gene ensembl id ENSG00000163581 changed to uniprot id : P11168.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000133460\n",
      "\n",
      "Gene ensembl id ENSG00000133460 changed to uniprot id : A0A087X018.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142583\n",
      "\n",
      "Gene ensembl id ENSG00000142583 changed to uniprot id : P22732.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111181\n",
      "\n",
      "Gene ensembl id ENSG00000111181 changed to uniprot id : P48065.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000132164\n",
      "\n",
      "Gene ensembl id ENSG00000132164 changed to uniprot id : P48066.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138074\n",
      "\n",
      "Gene ensembl id ENSG00000138074 changed to uniprot id : Q9Y289.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000003989\n",
      "\n",
      "Gene ensembl id ENSG00000003989 changed to uniprot id : P52569.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000059804\n",
      "\n",
      "Gene ensembl id ENSG00000059804 changed to uniprot id : P11169.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117394\n",
      "\n",
      "Gene ensembl id ENSG00000117394 changed to uniprot id : P11166.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000197496\n",
      "\n",
      "Gene ensembl id ENSG00000197496 changed to uniprot id : O95528.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151229\n",
      "\n",
      "Gene ensembl id ENSG00000151229 changed to uniprot id : Q96QE2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112077\n",
      "\n",
      "Gene ensembl id ENSG00000112077 changed to uniprot id : Q02094.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143595\n",
      "\n",
      "Gene ensembl id ENSG00000143595 changed to uniprot id : Q96PS8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167580\n",
      "\n",
      "Gene ensembl id ENSG00000167580 changed to uniprot id : P41181.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100170\n",
      "\n",
      "Gene ensembl id ENSG00000100170 changed to uniprot id : P13866.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000161798\n",
      "\n",
      "Gene ensembl id ENSG00000161798 changed to uniprot id : P55064.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000086159\n",
      "\n",
      "Gene ensembl id ENSG00000086159 changed to uniprot id : Q13520.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000171885\n",
      "\n",
      "Gene ensembl id ENSG00000171885 changed to uniprot id : P55087.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000240583\n",
      "\n",
      "Gene ensembl id ENSG00000240583 changed to uniprot id : P29972.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103375\n",
      "\n",
      "Gene ensembl id ENSG00000103375 changed to uniprot id : O94778.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124564\n",
      "\n",
      "Gene ensembl id ENSG00000124564 changed to uniprot id : O00476.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146039\n",
      "\n",
      "Gene ensembl id ENSG00000146039 changed to uniprot id : Q9Y2C5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157765\n",
      "\n",
      "Gene ensembl id ENSG00000157765 changed to uniprot id : O95436.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112337\n",
      "\n",
      "Gene ensembl id ENSG00000112337 changed to uniprot id : O00624.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124568\n",
      "\n",
      "Gene ensembl id ENSG00000124568 changed to uniprot id : Q14916.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131183\n",
      "\n",
      "Gene ensembl id ENSG00000131183 changed to uniprot id : Q06495.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168575\n",
      "\n",
      "Gene ensembl id ENSG00000168575 changed to uniprot id : Q08357.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000144136\n",
      "\n",
      "Gene ensembl id ENSG00000144136 changed to uniprot id : Q8WUM9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198569\n",
      "\n",
      "Gene ensembl id ENSG00000198569 changed to uniprot id : Q8N130.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151012\n",
      "\n",
      "Gene ensembl id ENSG00000151012 changed to uniprot id : Q9UPY5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000168003\n",
      "\n",
      "Gene ensembl id ENSG00000168003 changed to uniprot id : P08195.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000079215\n",
      "\n",
      "Gene ensembl id ENSG00000079215 changed to uniprot id : A0A087WT87.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110436\n",
      "\n",
      "Gene ensembl id ENSG00000110436 changed to uniprot id : P43004.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000162383\n",
      "\n",
      "Gene ensembl id ENSG00000162383 changed to uniprot id : F1T0D4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000106688\n",
      "\n",
      "Gene ensembl id ENSG00000106688 changed to uniprot id : P43005.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105143\n",
      "\n",
      "Gene ensembl id ENSG00000105143 changed to uniprot id : P48664.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000125454\n",
      "\n",
      "Gene ensembl id ENSG00000125454 changed to uniprot id : Q9HC21.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158296\n",
      "\n",
      "Gene ensembl id ENSG00000158296 changed to uniprot id : Q8WWT9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112053\n",
      "\n",
      "Gene ensembl id ENSG00000112053 changed to uniprot id : Q96RN1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000091138\n",
      "\n",
      "Gene ensembl id ENSG00000091138 changed to uniprot id : P40879.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155850\n",
      "\n",
      "Gene ensembl id ENSG00000155850 changed to uniprot id : P50443.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000070214\n",
      "\n",
      "Gene ensembl id ENSG00000070214 changed to uniprot id : Q8WWI5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143036\n",
      "\n",
      "Gene ensembl id ENSG00000143036 changed to uniprot id : Q8N4M1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112499\n",
      "\n",
      "Gene ensembl id ENSG00000112499 changed to uniprot id : O15244.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000204385\n",
      "\n",
      "Gene ensembl id ENSG00000204385 changed to uniprot id : A0A0G2JL76.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129353\n",
      "\n",
      "Gene ensembl id ENSG00000129353 changed to uniprot id : A0A088QCU6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137968\n",
      "\n",
      "Gene ensembl id ENSG00000137968 changed to uniprot id : Q8NCS7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000074803\n",
      "\n",
      "Gene ensembl id ENSG00000074803 changed to uniprot id : Q8IUN5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000064651\n",
      "\n",
      "Gene ensembl id ENSG00000064651 changed to uniprot id : P55011.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117834\n",
      "\n",
      "Gene ensembl id ENSG00000117834 changed to uniprot id : Q2M3M2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000181856\n",
      "\n",
      "Gene ensembl id ENSG00000181856 changed to uniprot id : P14672.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146477\n",
      "\n",
      "Gene ensembl id ENSG00000146477 changed to uniprot id : O75751.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112759\n",
      "\n",
      "Gene ensembl id ENSG00000112759 changed to uniprot id : Q99808.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174669\n",
      "\n",
      "Gene ensembl id ENSG00000174669 changed to uniprot id : Q14542.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000089057\n",
      "\n",
      "Gene ensembl id ENSG00000089057 changed to uniprot id : Q9UGH3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170482\n",
      "\n",
      "Gene ensembl id ENSG00000170482 changed to uniprot id : Q9UHI7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173262\n",
      "\n",
      "Gene ensembl id ENSG00000173262 changed to uniprot id : Q8TDB8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000146411\n",
      "\n",
      "Gene ensembl id ENSG00000146411 changed to uniprot id : Q8TD20.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000160326\n",
      "\n",
      "Gene ensembl id ENSG00000160326 changed to uniprot id : Q9UGQ3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140675\n",
      "\n",
      "Gene ensembl id ENSG00000140675 changed to uniprot id : P31639.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164638\n",
      "\n",
      "Gene ensembl id ENSG00000164638 changed to uniprot id : Q7RTT9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139209\n",
      "\n",
      "Gene ensembl id ENSG00000139209 changed to uniprot id : Q969I6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165349\n",
      "\n",
      "Gene ensembl id ENSG00000165349 changed to uniprot id : Q8WY07.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139514\n",
      "\n",
      "Gene ensembl id ENSG00000139514 changed to uniprot id : P30825.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000123643\n",
      "\n",
      "Gene ensembl id ENSG00000123643 changed to uniprot id : Q7Z2H8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000092068\n",
      "\n",
      "Gene ensembl id ENSG00000092068 changed to uniprot id : Q9UHI5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155465\n",
      "\n",
      "Gene ensembl id ENSG00000155465 changed to uniprot id : Q9UM01.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000167703\n",
      "\n",
      "Gene ensembl id ENSG00000167703 changed to uniprot id : Q8N370.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103257\n",
      "\n",
      "Gene ensembl id ENSG00000103257 changed to uniprot id : Q01650.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000149150\n",
      "\n",
      "Gene ensembl id ENSG00000149150 changed to uniprot id : O75387.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000112394\n",
      "\n",
      "Gene ensembl id ENSG00000112394 changed to uniprot id : Q8TF71.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130876\n",
      "\n",
      "Gene ensembl id ENSG00000130876 changed to uniprot id : Q9NS82.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164363\n",
      "\n",
      "Gene ensembl id ENSG00000164363 changed to uniprot id : Q96N87.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130164\n",
      "\n",
      "Gene ensembl id ENSG00000130164 changed to uniprot id : P01130.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105281\n",
      "\n",
      "Gene ensembl id ENSG00000105281 changed to uniprot id : Q15758.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111371\n",
      "\n",
      "Gene ensembl id ENSG00000111371 changed to uniprot id : Q9H2H9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115902\n",
      "\n",
      "Gene ensembl id ENSG00000115902 changed to uniprot id : P43007.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174358\n",
      "\n",
      "Gene ensembl id ENSG00000174358 changed to uniprot id : Q695T7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000087916\n",
      "\n",
      "No uniprot id found for gene ENSG00000087916.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000134294\n",
      "\n",
      "Gene ensembl id ENSG00000134294 changed to uniprot id : Q96QD8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000072041\n",
      "\n",
      "Gene ensembl id ENSG00000072041 changed to uniprot id : Q9H2J7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000017483\n",
      "\n",
      "Gene ensembl id ENSG00000017483 changed to uniprot id : Q8WUX1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000188338\n",
      "\n",
      "Gene ensembl id ENSG00000188338 changed to uniprot id : Q99624.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000011083\n",
      "\n",
      "Gene ensembl id ENSG00000011083 changed to uniprot id : Q99884.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163817\n",
      "\n",
      "Gene ensembl id ENSG00000163817 changed to uniprot id : Q9NP91.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000180251\n",
      "\n",
      "Gene ensembl id ENSG00000180251 changed to uniprot id : Q6AI14.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135740\n",
      "\n",
      "Gene ensembl id ENSG00000135740 changed to uniprot id : Q14940.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000090020\n",
      "\n",
      "Gene ensembl id ENSG00000090020 changed to uniprot id : P19634.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000066230\n",
      "\n",
      "Gene ensembl id ENSG00000066230 changed to uniprot id : P48764.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000181804\n",
      "\n",
      "Gene ensembl id ENSG00000181804 changed to uniprot id : Q8IVB4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115616\n",
      "\n",
      "Gene ensembl id ENSG00000115616 changed to uniprot id : Q9UBY0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000033867\n",
      "\n",
      "Gene ensembl id ENSG00000033867 changed to uniprot id : Q9Y6M7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000148942\n",
      "\n",
      "Gene ensembl id ENSG00000148942 changed to uniprot id : Q1EHB4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000158865\n",
      "\n",
      "Gene ensembl id ENSG00000158865 changed to uniprot id : Q8WWX8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131389\n",
      "\n",
      "Gene ensembl id ENSG00000131389 changed to uniprot id : P31641.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000196517\n",
      "\n",
      "Gene ensembl id ENSG00000196517 changed to uniprot id : P48067.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000188687\n",
      "\n",
      "Gene ensembl id ENSG00000188687 changed to uniprot id : Q9BY07.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138079\n",
      "\n",
      "Gene ensembl id ENSG00000138079 changed to uniprot id : Q07837.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000103064\n",
      "\n",
      "Gene ensembl id ENSG00000103064 changed to uniprot id : H3BNF5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138449\n",
      "\n",
      "Gene ensembl id ENSG00000138449 changed to uniprot id : Q9NP59.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000018280\n",
      "\n",
      "Gene ensembl id ENSG00000018280 changed to uniprot id : P49279.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000110911\n",
      "\n",
      "Gene ensembl id ENSG00000110911 changed to uniprot id : P49281.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165794\n",
      "\n",
      "Gene ensembl id ENSG00000165794 changed to uniprot id : Q9NP94.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141873\n",
      "\n",
      "Gene ensembl id ENSG00000141873 changed to uniprot id : Q9BRY0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000138821\n",
      "\n",
      "Gene ensembl id ENSG00000138821 changed to uniprot id : Q9C0K1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000104635\n",
      "\n",
      "Gene ensembl id ENSG00000104635 changed to uniprot id : Q15043.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141424\n",
      "\n",
      "Gene ensembl id ENSG00000141424 changed to uniprot id : Q13433.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139540\n",
      "\n",
      "Gene ensembl id ENSG00000139540 changed to uniprot id : Q6ZMH5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147804\n",
      "\n",
      "Gene ensembl id ENSG00000147804 changed to uniprot id : Q6P5W5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000140199\n",
      "\n",
      "Gene ensembl id ENSG00000140199 changed to uniprot id : Q9UHW9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113504\n",
      "\n",
      "Gene ensembl id ENSG00000113504 changed to uniprot id : Q9Y666.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124067\n",
      "\n",
      "Gene ensembl id ENSG00000124067 changed to uniprot id : Q9UP95.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124140\n",
      "\n",
      "Gene ensembl id ENSG00000124140 changed to uniprot id : Q9H2X9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000007216\n",
      "\n",
      "Gene ensembl id ENSG00000007216 changed to uniprot id : Q13183.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141485\n",
      "\n",
      "Gene ensembl id ENSG00000141485 changed to uniprot id : Q86YT5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147100\n",
      "\n",
      "Gene ensembl id ENSG00000147100 changed to uniprot id : P36021.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139155\n",
      "\n",
      "Gene ensembl id ENSG00000139155 changed to uniprot id : Q9NYB5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000176463\n",
      "\n",
      "Gene ensembl id ENSG00000176463 changed to uniprot id : Q9UIG8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000117479\n",
      "\n",
      "Gene ensembl id ENSG00000117479 changed to uniprot id : O60779.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000135917\n",
      "\n",
      "Gene ensembl id ENSG00000135917 changed to uniprot id : Q9BZV2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000174640\n",
      "\n",
      "Gene ensembl id ENSG00000174640 changed to uniprot id : Q92959.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137491\n",
      "\n",
      "Gene ensembl id ENSG00000137491 changed to uniprot id : A0A1B0GX35.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108576\n",
      "\n",
      "Gene ensembl id ENSG00000108576 changed to uniprot id : P31645.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000121270\n",
      "\n",
      "Gene ensembl id ENSG00000121270 changed to uniprot id : Q96J66.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000170385\n",
      "\n",
      "Gene ensembl id ENSG00000170385 changed to uniprot id : Q9Y6M5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000180773\n",
      "\n",
      "Gene ensembl id ENSG00000180773 changed to uniprot id : Q6YBV0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000091137\n",
      "\n",
      "Gene ensembl id ENSG00000091137 changed to uniprot id : O43511.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164889\n",
      "\n",
      "Gene ensembl id ENSG00000164889 changed to uniprot id : P04920.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000225697\n",
      "\n",
      "Gene ensembl id ENSG00000225697 changed to uniprot id : Q9BXS9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000050438\n",
      "\n",
      "Gene ensembl id ENSG00000050438 changed to uniprot id : Q2Y0W8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000036565\n",
      "\n",
      "Gene ensembl id ENSG00000036565 changed to uniprot id : P54219.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165646\n",
      "\n",
      "Gene ensembl id ENSG00000165646 changed to uniprot id : Q05940.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000173638\n",
      "\n",
      "Gene ensembl id ENSG00000173638 changed to uniprot id : P41440.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000076351\n",
      "\n",
      "Gene ensembl id ENSG00000076351 changed to uniprot id : Q96NT5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000081800\n",
      "\n",
      "Gene ensembl id ENSG00000081800 changed to uniprot id : Q9BZW2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000080493\n",
      "\n",
      "Gene ensembl id ENSG00000080493 changed to uniprot id : Q9Y6R1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000214617\n",
      "\n",
      "No uniprot id found for gene ENSG00000214617.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000004809\n",
      "\n",
      "Gene ensembl id ENSG00000004809 changed to uniprot id : Q86VW1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163393\n",
      "\n",
      "Gene ensembl id ENSG00000163393 changed to uniprot id : Q8IZD6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198743\n",
      "\n",
      "No uniprot id found for gene ENSG00000198743.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165970\n",
      "\n",
      "Gene ensembl id ENSG00000165970 changed to uniprot id : Q9Y345.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000142494\n",
      "\n",
      "Gene ensembl id ENSG00000142494 changed to uniprot id : Q96FL8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137204\n",
      "\n",
      "Gene ensembl id ENSG00000137204 changed to uniprot id : Q9Y694.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000129083\n",
      "\n",
      "Gene ensembl id ENSG00000129083 changed to uniprot id : P53618.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000095139\n",
      "\n",
      "Gene ensembl id ENSG00000095139 changed to uniprot id : P48444.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000105669\n",
      "\n",
      "Gene ensembl id ENSG00000105669 changed to uniprot id : O14579.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184432\n",
      "\n",
      "Gene ensembl id ENSG00000184432 changed to uniprot id : P35606.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000181789\n",
      "\n",
      "Gene ensembl id ENSG00000181789 changed to uniprot id : Q9Y678.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122218\n",
      "\n",
      "Gene ensembl id ENSG00000122218 changed to uniprot id : P53621.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183032\n",
      "\n",
      "Gene ensembl id ENSG00000183032 changed to uniprot id : Q9BQT8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102125\n",
      "\n",
      "Gene ensembl id ENSG00000102125 changed to uniprot id : F6Y2X3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175592\n",
      "\n",
      "Gene ensembl id ENSG00000175592 changed to uniprot id : P15407.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000004864\n",
      "\n",
      "Gene ensembl id ENSG00000004864 changed to uniprot id : Q9UJS0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115840\n",
      "\n",
      "Gene ensembl id ENSG00000115840 changed to uniprot id : O75746.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000183048\n",
      "\n",
      "Gene ensembl id ENSG00000183048 changed to uniprot id : A0A0S2Z3G3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115657\n",
      "\n",
      "Gene ensembl id ENSG00000115657 changed to uniprot id : Q9NP58.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108528\n",
      "\n",
      "Gene ensembl id ENSG00000108528 changed to uniprot id : Q02978.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000122912\n",
      "\n",
      "Gene ensembl id ENSG00000122912 changed to uniprot id : P16260.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000143158\n",
      "\n",
      "Gene ensembl id ENSG00000143158 changed to uniprot id : O95563.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000060762\n",
      "\n",
      "Gene ensembl id ENSG00000060762 changed to uniprot id : Q9Y5U8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100075\n",
      "\n",
      "Gene ensembl id ENSG00000100075 changed to uniprot id : P53007.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000075415\n",
      "\n",
      "Gene ensembl id ENSG00000075415 changed to uniprot id : Q00325.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000131748\n",
      "\n",
      "Gene ensembl id ENSG00000131748 changed to uniprot id : Q14849.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102743\n",
      "\n",
      "Gene ensembl id ENSG00000102743 changed to uniprot id : Q9Y619.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120329\n",
      "\n",
      "Gene ensembl id ENSG00000120329 changed to uniprot id : Q9BXI2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000182902\n",
      "\n",
      "Gene ensembl id ENSG00000182902 changed to uniprot id : Q9H1K4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000177542\n",
      "\n",
      "Gene ensembl id ENSG00000177542 changed to uniprot id : Q9H936.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000165449\n",
      "\n",
      "Gene ensembl id ENSG00000165449 changed to uniprot id : Q7RTY1.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155287\n",
      "\n",
      "Gene ensembl id ENSG00000155287 changed to uniprot id : Q96A46.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000147454\n",
      "\n",
      "Gene ensembl id ENSG00000147454 changed to uniprot id : Q9NYZ2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169100\n",
      "\n",
      "Gene ensembl id ENSG00000169100 changed to uniprot id : P12236.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000151729\n",
      "\n",
      "Gene ensembl id ENSG00000151729 changed to uniprot id : P12235.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000005022\n",
      "\n",
      "Gene ensembl id ENSG00000005022 changed to uniprot id : P05141.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102078\n",
      "\n",
      "Gene ensembl id ENSG00000102078 changed to uniprot id : O95258.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000153291\n",
      "\n",
      "Gene ensembl id ENSG00000153291 changed to uniprot id : O95847.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175567\n",
      "\n",
      "Gene ensembl id ENSG00000175567 changed to uniprot id : A0A024R5N5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000109424\n",
      "\n",
      "Gene ensembl id ENSG00000109424 changed to uniprot id : P25874.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000175564\n",
      "\n",
      "Gene ensembl id ENSG00000175564 changed to uniprot id : P55916.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000155368\n",
      "\n",
      "Gene ensembl id ENSG00000155368 changed to uniprot id : P07108.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000144741\n",
      "\n",
      "Gene ensembl id ENSG00000144741 changed to uniprot id : Q70HW3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100644\n",
      "\n",
      "Gene ensembl id ENSG00000100644 changed to uniprot id : A8MYV6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000100372\n",
      "\n",
      "Gene ensembl id ENSG00000100372 changed to uniprot id : O43808.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000164414\n",
      "\n",
      "Gene ensembl id ENSG00000164414 changed to uniprot id : P78382.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000213024\n",
      "\n",
      "Gene ensembl id ENSG00000213024 changed to uniprot id : P37198.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124789\n",
      "\n",
      "Gene ensembl id ENSG00000124789 changed to uniprot id : P49790.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113569\n",
      "\n",
      "Gene ensembl id ENSG00000113569 changed to uniprot id : O75694.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000108559\n",
      "\n",
      "Gene ensembl id ENSG00000108559 changed to uniprot id : Q99567.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000163002\n",
      "\n",
      "Gene ensembl id ENSG00000163002 changed to uniprot id : Q8NFH5.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000093000\n",
      "\n",
      "Gene ensembl id ENSG00000093000 changed to uniprot id : Q9UKX7.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000126883\n",
      "\n",
      "Gene ensembl id ENSG00000126883 changed to uniprot id : P35658.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000120253\n",
      "\n",
      "Gene ensembl id ENSG00000120253 changed to uniprot id : B4E301.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000139496\n",
      "\n",
      "Gene ensembl id ENSG00000139496 changed to uniprot id : Q9BVL2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000111581\n",
      "\n",
      "Gene ensembl id ENSG00000111581 changed to uniprot id : G3V1T4.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000115677\n",
      "\n",
      "Gene ensembl id ENSG00000115677 changed to uniprot id : C9JHN6.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141458\n",
      "\n",
      "Gene ensembl id ENSG00000141458 changed to uniprot id : O15118.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000119899\n",
      "\n",
      "Gene ensembl id ENSG00000119899 changed to uniprot id : Q9NRA2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000198246\n",
      "\n",
      "Gene ensembl id ENSG00000198246 changed to uniprot id : Q9BZD2.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000205060\n",
      "\n",
      "Gene ensembl id ENSG00000205060 changed to uniprot id : Q969S0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000130958\n",
      "\n",
      "Gene ensembl id ENSG00000130958 changed to uniprot id : Q76EJ3.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000102100\n",
      "\n",
      "Gene ensembl id ENSG00000102100 changed to uniprot id : P78381.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000181830\n",
      "\n",
      "Gene ensembl id ENSG00000181830 changed to uniprot id : Q96A29.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000157593\n",
      "\n",
      "Gene ensembl id ENSG00000157593 changed to uniprot id : Q8TB61.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000025800\n",
      "\n",
      "Gene ensembl id ENSG00000025800 changed to uniprot id : O60684.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000169180\n",
      "\n",
      "Gene ensembl id ENSG00000169180 changed to uniprot id : Q96QU8.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000113163\n",
      "\n",
      "Gene ensembl id ENSG00000113163 changed to uniprot id : A0A2R8Y5S9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000141179\n",
      "\n",
      "Gene ensembl id ENSG00000141179 changed to uniprot id : I3L3H0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000184304\n",
      "\n",
      "Gene ensembl id ENSG00000184304 changed to uniprot id : Q15139.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000124164\n",
      "\n",
      "Gene ensembl id ENSG00000124164 changed to uniprot id : O95292.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000101558\n",
      "\n",
      "Gene ensembl id ENSG00000101558 changed to uniprot id : Q9P0L0.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000137700\n",
      "\n",
      "Gene ensembl id ENSG00000137700 changed to uniprot id : A0A024R3H9.\n",
      "\n",
      "DEBUG : getting uniprot gene id for ENSG00000116704\n",
      "\n",
      "Gene ensembl id ENSG00000116704 changed to uniprot id : Q9NTN3.\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00097x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00157x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00228x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00242x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00266x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00267x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00268x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00269x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00402x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00545x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00648x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00932x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m00970x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01020x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01070x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01100x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01107x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01111x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01115x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01159x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01174x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01252x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01253x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01256x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01260x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01280x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01285x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01286x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01287x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01288x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01290x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01296x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01306x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01307x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01308x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01327x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01330x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01334x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01339x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01361x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01365x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01368x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01369x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01370x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01374x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01383x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01393x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01396x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01397x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01398x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01400x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01401x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01403x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01410x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01419x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01433x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01438x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01442x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01445x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01450x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01513x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01517x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01525x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01533x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01543x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01554x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01569x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01570x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01587x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01588x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01590x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01596x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01615x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01619x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01621x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01626x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01628x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01629x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01630x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01638x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01641x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01648x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01652x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01655x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01659x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01666x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01668x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01669x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01671x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01673x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01682x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01700x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01714x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01716x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01736x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01742x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01743x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01744x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01745x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01758x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01765x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01786x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01788x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01789x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01795x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01796x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01797x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01799x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01800x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01821x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01822x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01830x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01833x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01840x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01874x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01910x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01913x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01935x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01938x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01948x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01959x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01962x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01965x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01974x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01975x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01983x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01986x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01987x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m01988x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02001x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02016x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02023x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02026x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02027x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02034x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02038x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02039x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02040x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02041x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02046x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02047x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02049x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02054x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02104x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02105x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02108x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02118x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02119x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02120x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02125x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02136x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02141x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02145x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02147x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02161x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02167x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02170x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02171x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02184x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02193x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02200x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02237x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02278x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02288x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02303x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02332x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02337x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02348x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02352x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02353x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02354x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02357x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02360x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02362x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02366x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02369x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02387x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02389x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02394x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02403x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02418x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02426x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02445x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02450x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02453x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02470x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02471x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02510x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02519x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02556x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02560x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02578x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02614x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02617x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02630x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02631x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02642x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02658x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02661x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02672x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02680x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02691x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02692x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02724x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02746x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02751x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02754x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02770x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02772x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02785x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02786x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02789x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02815x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02819x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02833x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02842x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02885x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02896x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02897x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02924x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02926x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02930x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02943x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02945x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02946x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02949x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02961x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02962x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02963x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02967x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02980x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02982x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02983x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02985x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02993x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02996x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m02998x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03039x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03044x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03052x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03089x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03101x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03106x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03114x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03117x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03120x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03121x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03123x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03130x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03134x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03135x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03141x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03146x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03147x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03151x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03154x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_m03157x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_temp001x\n",
      "\n",
      "Changing gene_reaction_rule for reaction EX_temp006x\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3905\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3907\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4097\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4099\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4108\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4133\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4137\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4281\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4388\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4283\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8357\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4379\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4301\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4355\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4358\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4360\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4363\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4365\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4368\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4370\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4371\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4372\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4373\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4375\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4381\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4391\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4394\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4396\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4521\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6410\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6412\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7745\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7746\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7747\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7748\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7749\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8360\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3989\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4122\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4837\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5395\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5396\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9727\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5397\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5398\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5399\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5400\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5401\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8568\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8570\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8572\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8574\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8576\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8578\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8591\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8592\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8587\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8589\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3944\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4128\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4130\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4131\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4132\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4303\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4414\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4415\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4416\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4774\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4775\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7674\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8761\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8762\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8767\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4297\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4315\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4316\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4319\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4383\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4385\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4386\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4387\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4399\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4400\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4401\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4402\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4490\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4706\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8768\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4590\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4591\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4592\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4595\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8341\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8726\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8727\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8729\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6537\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1568\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3853\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3854\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3855\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3857\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3859\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4087\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4089\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4091\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4103\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4143\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4193\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8497\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8500\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8501\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8502\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8503\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8504\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8506\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8507\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8508\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8509\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8511\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8512\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8514\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8516\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4280\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0153\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3212\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3797\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4105\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4106\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4107\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4282\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4331\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4459\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4460\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4497\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4741\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9804\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1434\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1436\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3163\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4461\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4604\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5351\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4052\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4304\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4306\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4350\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4351\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4352\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4354\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4398\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4404\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4473\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4474\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4477\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4501\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4565\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4567\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4568\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4623\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4625\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4710\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4841\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8074\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8653\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3998\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4000\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4002\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4004\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4010\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4081\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4082\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4012\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4014\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4020\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4022\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4038\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4040\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4042\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4046\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4048\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4080\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4085\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4134\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4168\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4171\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4290\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4406\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4412\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4417\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4418\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4419\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4421\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4449\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4450\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4452\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4453\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4480\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4481\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4482\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4486\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4487\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4488\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4489\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4492\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4493\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4495\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4496\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4573\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4574\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4600\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4602\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4603\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4611\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4612\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4614\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4615\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4617\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4618\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4619\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4621\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4632\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4651\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4663\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4694\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4695\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4705\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4709\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4799\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4802\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4804\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4808\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4810\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4812\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4814\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5301\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5353\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6602\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6603\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7144\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4016\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4018\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4044\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4291\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4420\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4570\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4572\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4006\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4028\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4030\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4670\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4673\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4675\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6614\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6615\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6616\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4635\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3931\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3969\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3970\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4008\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4032\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4034\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4036\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4050\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4056\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4059\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4060\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4127\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4177\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4179\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4182\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4183\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4192\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4194\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4210\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4343\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4346\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4484\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4485\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4510\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4512\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4513\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4514\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4577\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4579\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4580\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4608\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4575\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4637\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4642\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4643\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4644\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4676\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5299\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5352\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6612\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6621\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6622\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6623\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6624\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6627\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8072\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3965\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3966\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3967\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3968\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3999\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4024\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4026\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4054\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4061\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4136\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4180\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4185\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4186\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4342\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4344\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4483\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4507\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4516\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4601\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4633\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4640\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4641\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4680\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4806\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6625\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6626\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7160\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7161\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7162\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7163\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7164\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7713\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7716\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7721\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7725\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7728\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7801\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7863\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7864\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7865\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7866\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7867\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7868\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7869\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7870\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7871\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7872\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7873\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7874\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7876\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7878\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7879\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7880\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7881\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7882\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7883\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7884\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7885\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7886\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7887\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7888\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7889\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7890\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7891\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7892\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7893\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8083\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8085\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8087\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8443\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8444\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8448\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8449\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8450\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8451\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8452\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8453\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8454\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8455\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8456\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8457\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8458\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8459\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8460\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8462\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8463\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8464\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8465\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8466\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8467\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8468\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8469\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8470\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8471\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8472\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8473\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8474\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8478\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8479\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8482\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8483\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8485\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8486\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8487\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8488\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8489\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3802\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3804\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3811\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3813\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3822\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3827\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3829\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3831\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3862\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3865\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3870\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3873\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8654\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3890\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3892\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9802\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3899\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3903\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4109\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4114\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4115\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4118\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4172\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4196\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4197\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4287\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4693\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6780\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7641\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7642\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8626\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8628\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3806\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3807\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3816\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3819\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3820\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3833\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3835\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3837\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3838\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3895\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3897\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3956\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3993\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4073\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4075\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4077\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4190\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4191\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4212\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4422\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4423\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4424\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4583\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4606\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4607\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4776\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4777\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4778\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4779\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4780\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4781\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4784\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4785\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4786\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4787\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5384\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6929\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6930\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6931\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6932\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6933\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6934\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6935\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6936\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6938\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6939\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6941\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6942\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6943\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6944\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6945\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6946\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6947\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6948\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6949\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6950\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6951\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6952\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6956\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6957\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6958\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6959\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6961\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6963\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6964\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8096\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8097\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8098\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8426\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8427\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8432\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8603\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8604\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8605\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8609\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8610\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8611\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4285\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0457\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0460\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3752\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3770\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3784\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3839\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3841\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3843\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3845\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3847\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3849\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3852\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3856\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3860\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3883\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3901\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3939\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3974\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4198\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4200\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4348\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4467\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4584\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4696\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4697\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4698\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4699\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4700\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4742\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4788\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4791\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4792\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5392\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5393\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6409\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7702\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8433\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8434\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8435\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8436\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8437\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8439\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8440\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8441\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8442\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9718\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9486\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4428\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4429\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4430\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4431\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4437\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4658\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4660\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4712\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5336\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5337\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5338\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5339\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5340\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8783\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8784\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8786\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4239\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4288\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4596\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4597\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4599\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4667\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4668\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4737\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4739\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6415\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6974\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6975\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6976\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6977\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6978\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6979\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6980\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6981\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6982\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6983\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6984\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6985\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8017\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8018\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8019\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8021\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8025\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8026\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8027\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8029\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3164\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3166\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3885\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4250\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8091\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8092\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8563\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8564\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8566\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3935\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4681\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4682\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4683\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4685\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4686\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4687\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7689\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8529\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8530\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8534\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8535\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3206\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3208\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3211\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3213\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3215\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3747\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3744\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3757\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3761\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3777\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3778\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3795\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4797\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6417\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6419\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6420\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6421\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6422\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6923\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3765\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8088\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8787\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3940\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4214\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4216\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4222\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4225\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4242\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4243\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4244\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4246\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6707\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6713\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6714\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6715\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6716\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6717\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6718\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6719\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6730\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6731\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6734\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6736\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6738\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6740\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6745\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6746\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6747\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6748\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6749\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6750\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6751\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6752\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6753\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6754\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6755\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6756\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6758\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6760\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6761\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6762\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6763\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6764\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6766\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6767\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6768\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6772\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6774\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6776\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6778\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6782\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6783\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6784\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6785\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6786\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6787\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6788\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6789\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6790\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6791\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6800\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6801\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6808\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6813\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6815\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6826\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6827\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6829\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6831\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6834\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6835\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6836\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6837\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6838\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6839\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6841\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6844\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6845\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6849\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6850\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6851\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6854\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6855\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7628\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7756\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8539\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3875\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3877\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3879\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3881\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3912\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3917\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3928\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3995\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4189\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4302\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4323\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5385\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5386\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5387\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5388\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5389\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5391\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6515\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6518\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6519\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6520\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8062\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8066\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8067\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8068\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8069\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8641\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3868\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3869\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3889\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3933\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3937\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4116\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4120\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4121\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4175\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4176\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4324\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4326\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4328\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4329\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4708\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8769\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8770\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4079\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4330\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4425\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4427\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4689\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4735\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7991\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7992\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7993\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7128\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7129\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7130\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7131\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7133\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7134\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7135\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7136\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7137\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7138\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7139\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7140\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7141\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8640\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3748\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3751\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3753\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3755\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3759\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3763\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3767\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3769\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3773\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3775\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3780\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3783\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3785\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3790\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3792\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3796\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3823\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6416\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3986\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3988\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4124\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4158\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4159\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4299\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4300\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4494\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4524\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4525\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4526\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4527\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4528\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4529\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4530\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4532\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4536\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4628\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4629\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4631\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7696\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7697\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8371\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8375\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8377\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8672\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5130\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5131\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5132\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5133\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5134\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5135\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5136\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5137\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5138\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5139\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5140\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5141\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5142\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5143\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5145\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5146\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5147\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5148\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5149\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5150\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7171\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7174\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7175\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7180\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7181\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7436\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7438\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7440\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8257\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1532\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7254\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7255\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7256\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7261\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7263\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7264\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7265\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7266\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7267\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7268\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7269\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7270\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7271\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7274\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7275\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7276\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7277\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7278\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7279\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7280\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7281\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7285\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7286\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7287\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7288\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7289\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7290\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7291\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7293\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7300\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7308\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7309\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7310\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7311\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7312\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7313\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7314\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7315\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7316\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7317\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7318\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7319\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7320\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7321\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7322\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7323\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7324\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7325\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7327\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7328\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7332\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7333\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7334\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7428\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7429\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7574\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7575\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7576\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7577\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7578\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7580\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7582\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7585\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7586\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7587\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8691\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8693\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8694\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5151\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5166\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5169\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5170\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5171\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5172\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7617\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7618\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7621\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5258\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5259\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5282\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5284\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5285\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5286\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5287\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5288\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5407\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3861\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3866\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3915\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4084\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4187\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4701\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4840\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8662\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8663\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8667\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8668\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0710\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3787\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3957\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3958\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4111\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4112\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4113\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4139\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4141\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4145\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4147\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4149\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4152\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4209\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4408\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4410\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4454\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4456\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4458\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4465\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4585\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4586\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4587\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4588\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4589\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4652\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5294\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5297\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6411\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6413\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6414\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8743\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8772\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8773\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8775\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8777\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8778\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8780\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8782\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3975\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3977\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3979\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6911\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6912\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6914\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6916\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6918\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6921\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3960\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3980\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3982\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4767\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8410\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8413\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8415\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0156\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0165\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0168\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0171\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0174\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0177\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0180\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0184\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0188\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0192\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0196\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0200\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0204\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0209\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0213\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0217\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0226\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0233\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0237\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0241\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0245\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0249\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0255\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0259\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0263\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0267\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0271\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0275\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0279\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0283\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0289\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0293\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0297\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0301\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0305\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0309\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0313\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0319\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0323\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0327\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0331\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0337\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0341\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0345\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0349\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0353\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0357\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0361\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0365\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0369\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0373\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0377\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0381\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0385\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0389\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0393\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0397\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0401\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0405\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0409\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0413\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0417\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0421\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0425\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0429\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0433\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0437\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2942\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2943\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2944\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2945\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2946\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2947\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2948\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2949\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2951\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2952\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2954\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2955\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2956\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2957\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2959\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2961\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2962\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2963\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2964\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2965\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2966\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2967\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2968\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2969\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2970\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2971\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2972\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2973\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2974\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2975\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2976\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2977\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2978\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2979\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2980\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2981\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2982\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2983\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2984\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2985\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2986\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2987\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2988\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2989\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2990\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2991\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2992\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2994\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2996\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2998\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2999\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3000\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3001\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3002\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3003\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2152\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2153\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2154\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2155\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2156\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2157\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2158\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2159\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2160\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2161\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2162\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2163\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2164\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2165\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2166\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2167\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2168\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2169\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2170\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2171\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2173\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2174\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2175\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2176\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2178\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2179\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2180\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2181\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4156\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4295\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2227\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2228\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2229\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2230\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2231\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2232\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2233\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2234\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2235\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2236\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2237\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2238\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2239\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2240\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2241\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2242\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2243\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2244\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2245\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2246\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2247\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2249\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2250\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2251\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2252\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2254\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2255\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2256\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2257\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2305\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2307\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2309\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2311\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2315\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2317\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2319\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2321\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2324\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2326\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2328\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2330\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2332\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2334\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2336\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2338\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2342\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2343\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2344\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2345\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2347\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2348\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2349\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2350\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2353\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2354\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2355\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2356\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2361\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2362\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2363\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2364\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2190\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2191\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2193\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2194\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2201\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2202\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2203\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2204\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2205\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2208\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2209\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2210\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2211\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2212\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2213\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2214\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2215\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2217\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2218\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2219\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2259\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2260\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2261\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2262\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2263\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2264\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2265\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2266\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2267\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2268\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2269\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2270\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2281\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2282\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2284\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2286\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2287\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2292\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2293\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2294\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2295\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2296\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2359\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2365\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2288\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2289\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0709\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2150\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2151\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2172\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2177\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2182\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2248\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2253\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2258\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2440\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2441\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2455\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2456\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2457\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2458\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2459\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2460\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2463\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2464\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2465\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2466\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2467\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2468\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2469\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2470\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6398\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6400\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6401\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6402\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6403\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2472\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2475\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2478\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2480\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2482\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2484\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2486\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2489\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2491\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2493\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2497\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2499\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2501\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2503\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2505\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2510\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2511\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2512\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2513\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2515\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2516\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2518\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2520\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2522\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2524\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2526\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2528\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2530\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2533\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2535\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2537\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2540\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2541\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2542\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2543\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2544\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2546\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2548\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2549\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2551\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2553\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2555\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2556\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2557\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2560\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2561\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2562\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2563\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2570\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2571\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2572\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2573\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2574\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2576\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2577\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2578\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2579\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2580\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2582\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2584\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2368\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2371\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2374\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2376\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2378\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2380\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2383\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2387\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2389\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2395\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2397\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2399\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2401\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2403\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2408\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2409\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2410\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2411\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2415\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2417\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2419\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2421\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2423\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2426\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2428\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2430\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2433\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2434\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2435\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2436\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3444\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3446\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3447\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3448\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3449\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3450\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3451\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3452\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3453\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3454\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3455\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3456\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3457\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3458\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3459\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3461\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3462\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3463\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3464\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3465\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3466\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3467\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3468\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3469\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3470\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3471\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3472\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0931\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0932\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0933\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0934\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0935\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0936\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0937\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0938\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0939\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0940\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0942\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0943\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0944\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0945\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0946\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0947\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0948\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0949\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0950\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0951\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0953\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0954\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0955\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0956\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0957\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0958\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0960\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0963\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0964\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0969\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0973\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0976\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0979\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0980\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0981\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0991\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0992\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0993\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0994\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1002\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1003\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1004\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1006\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1007\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1008\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1020\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1021\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1022\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1023\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1025\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1026\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1027\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1028\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1029\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1033\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1037\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1039\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1043\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1048\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1049\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1050\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1054\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1055\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1057\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1058\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1059\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1061\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1065\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1066\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1067\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1068\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1069\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1070\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1071\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1072\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1074\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3015\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1080\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1081\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1084\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1085\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1088\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1091\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1098\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1100\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1101\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1102\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1103\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1105\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1106\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1107\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1108\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1109\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1110\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1111\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1112\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1116\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1117\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1119\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1120\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1121\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1122\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1123\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1124\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1125\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1126\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1127\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1128\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1129\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1130\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1131\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1132\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1133\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1134\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1137\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1138\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1139\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1140\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1146\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1147\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1148\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1149\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1150\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1151\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1152\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1153\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1154\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1155\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1156\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1157\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1158\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1159\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1160\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1161\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1162\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1163\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1164\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1165\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1166\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1167\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1168\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1170\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1171\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1186\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1190\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1191\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1192\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1193\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1194\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1195\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1196\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1200\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1201\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1202\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1203\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1204\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1205\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1206\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1207\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1208\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1209\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1210\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1211\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1212\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1213\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1228\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1229\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1230\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1231\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1232\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1233\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1234\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1235\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1236\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1237\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1238\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1239\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1241\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1244\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1245\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1246\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1248\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1249\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1250\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1251\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1253\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1254\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1255\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1256\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1257\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1258\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1259\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1260\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1261\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1262\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1263\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1264\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1265\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1266\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1270\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1271\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1272\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1273\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1274\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1275\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1276\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1277\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1278\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1279\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1280\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1281\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1282\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1287\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1288\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1289\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1290\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1291\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1292\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1293\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1294\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1295\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1296\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1297\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1298\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1299\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1300\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1301\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1303\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1304\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4796\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8546\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8548\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8549\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8556\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8560\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0154\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0189\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0193\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0197\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0201\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0206\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0210\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0214\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0223\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0230\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0234\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0238\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0242\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0246\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0250\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0256\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0260\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0264\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0268\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0272\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0276\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0280\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0284\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0290\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0294\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0298\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0302\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0306\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0310\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0314\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0320\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0324\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0328\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0332\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0338\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0342\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0346\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0350\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0354\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0358\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0362\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0366\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0370\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0374\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0378\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0382\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0386\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0390\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0394\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0398\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0402\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0406\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0410\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0414\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0418\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0422\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0426\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0430\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0434\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0438\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3475\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0159\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2591\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2594\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2599\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2602\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2605\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2608\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2611\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2614\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2620\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2623\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2626\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2633\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2636\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2642\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2648\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2651\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2654\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2657\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2660\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2663\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2666\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2669\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2672\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2675\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2678\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2681\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2684\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2687\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2690\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2693\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2699\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2702\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2705\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2708\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2711\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2714\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2717\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2720\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2726\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2730\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2733\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2736\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2739\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2742\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2746\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2752\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2755\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2759\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2762\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2768\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2771\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2774\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2877\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3032\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3521\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0160\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0161\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0162\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0163\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2592\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2593\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2596\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2598\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2600\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2601\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2603\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2604\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2606\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2607\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2609\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2610\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2612\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2613\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2616\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2618\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2621\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2622\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2624\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2625\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2629\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2630\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2634\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2635\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2638\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2640\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2644\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2646\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2649\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2650\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2652\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2653\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2655\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2656\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2658\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2659\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2661\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2662\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2664\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2665\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2667\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2668\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2670\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2671\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2673\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2674\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2676\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2677\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2679\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2680\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2682\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2683\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2685\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2686\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2688\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2689\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2691\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2692\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2695\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2697\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2700\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2701\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2703\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2704\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2706\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2707\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2709\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2710\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2712\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2713\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2715\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2716\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2718\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2719\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2722\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2724\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2727\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2729\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2731\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2732\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2734\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2735\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2737\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2738\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2740\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2741\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2744\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2745\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2748\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2750\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2753\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2754\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2757\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2758\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2760\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2761\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2764\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2766\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2769\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2770\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2772\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2773\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2775\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2776\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3519\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3520\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3027\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3028\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3030\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3033\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3034\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3035\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3037\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2787\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2788\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2789\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2790\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2791\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2792\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2793\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2794\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2795\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2796\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2797\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2798\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2799\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2800\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2801\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2803\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2805\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2806\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2807\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2809\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2811\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2812\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2813\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2814\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2815\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2816\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2817\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2819\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2821\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2822\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2823\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2824\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2825\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2827\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2829\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2830\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2831\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2832\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2833\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2834\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2835\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2836\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2837\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2838\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2839\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2840\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2841\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2842\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2843\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2844\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2845\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2846\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2847\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2848\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2849\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2850\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2851\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2852\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2853\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2854\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2855\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2856\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2857\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2859\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2861\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2862\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2863\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2864\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2865\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2866\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2867\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2868\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2869\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2870\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2871\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2872\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2873\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2874\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2875\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2876\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2878\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2879\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2880\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2882\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2884\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2886\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2888\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2890\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2892\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2894\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2896\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2897\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2898\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2899\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2900\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2901\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2902\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2903\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2904\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2905\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2906\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2907\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2908\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2909\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2910\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2911\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2912\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2913\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2914\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2915\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2916\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2917\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2918\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2919\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2920\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2921\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2922\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2923\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2924\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2925\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2926\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2927\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2928\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2929\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2930\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2931\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2932\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0001\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0002\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0003\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0005\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0604\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0605\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0665\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0667\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0668\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0669\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0670\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0671\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0672\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0679\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0680\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0681\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0682\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0683\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0684\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7588\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3053\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3057\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3058\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3059\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3060\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3062\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3063\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3064\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3065\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3066\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3067\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3068\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3069\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3070\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3071\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3072\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3073\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3074\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3075\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3076\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3077\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3078\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3079\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3080\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3081\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3082\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3083\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3084\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3085\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3086\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3087\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3088\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3089\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3090\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3091\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3092\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3093\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3094\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3095\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3096\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3097\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3098\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3099\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3100\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3101\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3102\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3103\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3104\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3326\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3327\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3328\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3329\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3330\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3331\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3332\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3333\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3334\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3335\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3336\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3337\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3338\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3339\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3340\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3341\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3342\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3343\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3344\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3345\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3346\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3347\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3348\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3349\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3350\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3351\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3352\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3353\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3355\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3364\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3365\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3367\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3368\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3369\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3370\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3372\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3373\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3375\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3480\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3481\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3482\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3484\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3486\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3488\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3489\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3498\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3501\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3503\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3505\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3506\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3508\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3509\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3510\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3511\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3512\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3513\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3514\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3515\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3517\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3301\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3302\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3304\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3305\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3306\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3307\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3309\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3310\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3311\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3312\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3314\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3315\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3316\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3317\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3319\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3320\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3321\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3322\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3323\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3107\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3108\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3109\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3110\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3111\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3112\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3113\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3114\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3115\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3116\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3117\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3118\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3121\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3122\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3123\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3125\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3128\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3129\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3130\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3132\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3135\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3136\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3137\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3139\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3142\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3143\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3144\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3146\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3149\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3150\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3151\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3153\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3156\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3157\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3158\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3160\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3398\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3406\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3407\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3408\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3409\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3411\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3413\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3414\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3416\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3421\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3422\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3423\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3424\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3425\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3426\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9719\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3427\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3428\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3429\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3430\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3431\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3432\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3433\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3170\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3171\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3172\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3173\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3174\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3175\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3176\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3177\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3178\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3179\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3180\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3181\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3182\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3183\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3184\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3185\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3186\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3187\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3188\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3189\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3190\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3191\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3192\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3193\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3194\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3195\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3196\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3197\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3198\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3199\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3200\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3201\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3202\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3203\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3204\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3205\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3396\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3240\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3241\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3242\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3243\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3244\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3245\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3246\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3247\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3250\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3252\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3254\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3256\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3258\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3260\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3262\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3264\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3272\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3356\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3357\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3358\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3359\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3360\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3361\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3362\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3363\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3218\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3219\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3220\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3221\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3222\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3223\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3224\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3225\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3226\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3227\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3228\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3229\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3230\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3231\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3232\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3233\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3234\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3235\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3236\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3237\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3239\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3275\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3277\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3278\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3279\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3280\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3281\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3282\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3283\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3284\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3285\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3286\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3287\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3288\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3290\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3292\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3293\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3294\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3296\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3298\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1174\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1175\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1176\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1177\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1178\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1179\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1180\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1181\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1182\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1183\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1184\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1216\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1217\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1218\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1219\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1220\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1221\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1222\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1223\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1224\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1225\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1226\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3522\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3523\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3524\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3525\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3526\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3527\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3528\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3529\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3530\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3531\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3532\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3533\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3534\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1573\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6901\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6903\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6904\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6905\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6906\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6907\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6908\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6909\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6910\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7165\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7166\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7167\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7168\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7169\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7170\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1644\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1651\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1920\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1924\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1926\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1927\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1940\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1948\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1990\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1991\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1992\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2002\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2003\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2004\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2005\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2007\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4766\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7648\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7930\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7935\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7940\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7941\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7943\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7945\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7948\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7950\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7952\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7955\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7958\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7959\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7962\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7965\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7968\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2041\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7970\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7971\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7974\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7976\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7978\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7980\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7984\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7987\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1944\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1952\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1953\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1962\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1963\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1973\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1974\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1976\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1977\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2014\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2015\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2016\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2017\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2018\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2019\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2025\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1440\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1445\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1448\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1454\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1465\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1467\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1470\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1473\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1477\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1478\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1479\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1484\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1490\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1493\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1494\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1495\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1496\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1500\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1502\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1503\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1504\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1505\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1509\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1512\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1516\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1519\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1526\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1533\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1557\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1558\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1565\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1535\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1536\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1538\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1539\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1540\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1543\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1544\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1545\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1546\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1547\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1548\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1549\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1550\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1551\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1552\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1553\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1437\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1451\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1457\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1460\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1531\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1570\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1576\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1577\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3105\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4630\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2029\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2030\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2031\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2032\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2033\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2034\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2036\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2038\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2042\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2043\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2044\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2045\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2048\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2052\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2053\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2054\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2058\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2059\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2060\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2061\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2063\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2068\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2069\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2070\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2071\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2072\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2073\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2074\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2075\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2076\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2078\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2083\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2084\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2085\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2086\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2088\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2089\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2095\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2096\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2097\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2098\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2099\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2100\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2106\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2107\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2108\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2109\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2110\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2111\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2112\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2113\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3537\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3540\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3545\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3546\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3551\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3552\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3574\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3577\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3581\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3586\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3588\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3589\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3625\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3626\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3627\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3628\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3629\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3630\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3631\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3632\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3633\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3634\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3635\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3636\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3637\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3638\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3639\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3640\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3641\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3642\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3643\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3644\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3645\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3646\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3647\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3648\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3649\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3650\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3651\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3652\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3653\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3654\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3655\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3656\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3657\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3658\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3659\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3660\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3661\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3662\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3663\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3664\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3665\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3666\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3667\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3668\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3669\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3670\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3671\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3672\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3673\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3674\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3675\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3676\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3677\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3678\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3679\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3680\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3681\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3682\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3683\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3684\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3685\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3686\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3687\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3688\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3689\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3690\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3691\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3692\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3693\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3694\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3695\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3696\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3697\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3698\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3699\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3700\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3701\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3703\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3704\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3705\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3706\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3707\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3708\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3709\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3710\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3711\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3712\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3713\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3714\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3715\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3716\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3717\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3718\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3719\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3720\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3721\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3722\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3723\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3724\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3725\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3726\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3727\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3728\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3729\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3732\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3733\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3734\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3735\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3736\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3737\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3738\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3739\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3740\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3741\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0715\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0716\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0735\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0736\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0738\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0741\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0744\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0746\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0748\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0750\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0753\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0754\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0758\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0760\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0761\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0762\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0765\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0766\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0767\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0773\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0775\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0779\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0781\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0783\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0795\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8147\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8148\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8149\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8150\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8151\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8152\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8155\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8165\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8166\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8167\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8168\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8172\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8173\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8175\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8178\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8179\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8180\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8184\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8185\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8186\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8188\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8189\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8190\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8191\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8192\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8194\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8198\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8201\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0786\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8206\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8209\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8211\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8212\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8217\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8219\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8220\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8224\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8226\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8227\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8228\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8233\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8235\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8238\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8245\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8246\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8248\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8249\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8250\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8251\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0453\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0456\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0458\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0459\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0486\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0487\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0488\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0489\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0490\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0491\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0492\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0493\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0494\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0495\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0496\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0497\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0498\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0499\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0500\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0501\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0502\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0503\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0504\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0505\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0506\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0507\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0508\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0509\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0510\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0511\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0512\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0513\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0514\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0515\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0516\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0517\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0518\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0519\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0520\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0521\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0522\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0523\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0524\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0525\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0526\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0527\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0528\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0529\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0530\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0531\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0532\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0533\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0534\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0535\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0536\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0537\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0538\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0539\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0540\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0541\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0542\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0543\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0544\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0588\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0596\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0673\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0674\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0675\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0676\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0677\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0678\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4296\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4390\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5254\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7599\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0448\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0450\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0468\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0478\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0479\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0481\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0579\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0580\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0581\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0582\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0584\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0586\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0589\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0590\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0591\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0592\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0593\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0594\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0597\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0598\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0599\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0600\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0601\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0602\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0607\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0610\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0613\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0614\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0615\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0616\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0622\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0623\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0625\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0627\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0629\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0630\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0632\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0633\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0634\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0635\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0636\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0638\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0642\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0643\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0644\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0645\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0646\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0648\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0651\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0653\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0654\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0657\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0660\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4627\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4838\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4839\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7591\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7594\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7597\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7601\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7611\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7613\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7755\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8518\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8523\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8525\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0815\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0816\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0817\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0819\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0820\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0821\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0823\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0824\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0825\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0827\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0828\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0830\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0837\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0838\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0841\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0842\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0844\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0845\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0847\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0848\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0849\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0850\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0855\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0856\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0803\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0807\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0809\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0810\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0811\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0812\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0859\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0873\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0875\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0876\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0889\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0890\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0891\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0892\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0893\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0894\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0895\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0898\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0899\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0900\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0906\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0910\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0911\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0787\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0790\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0792\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0794\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0797\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0804\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0808\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0826\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0832\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0835\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0836\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0920\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0924\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0925\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0926\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8379\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7185\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8380\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7186\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8381\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7187\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8383\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8384\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8385\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8387\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8388\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8389\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8390\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8391\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8392\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8393\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8394\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8395\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8396\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8397\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8398\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8399\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8401\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8402\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8403\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8404\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8405\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8406\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8407\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1981\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1985\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1988\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1312\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1313\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1321\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1325\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1326\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1327\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1329\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1339\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1347\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1367\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1370\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1379\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1382\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1384\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1387\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1394\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1395\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1398\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0703\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7598\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7600\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7602\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7603\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7604\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7614\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7615\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0025\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0030\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7200\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7202\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7203\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7205\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7206\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7207\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7208\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7209\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7210\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7211\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7212\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7213\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7214\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7215\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7216\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7217\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7218\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7219\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7491\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7492\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7493\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7494\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7495\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7496\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7497\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7498\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7509\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7519\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7520\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7521\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7522\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7534\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7535\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7536\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7537\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7551\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7552\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7553\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7501\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7502\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7503\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7504\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7505\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7506\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7507\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7513\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7514\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7515\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7516\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7517\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7525\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7526\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7527\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7528\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7529\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7530\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7531\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7532\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7541\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7542\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7543\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7544\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7545\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7546\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7547\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7548\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7549\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7557\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7558\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7559\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7560\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7561\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7562\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7563\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7564\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7565\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7566\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7225\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7226\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7227\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7228\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7229\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7230\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7231\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7232\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7233\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7234\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7235\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7236\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7237\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7238\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7239\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7240\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7241\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7242\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7243\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7244\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7245\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7246\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7247\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7248\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7249\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7250\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7571\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7572\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7573\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7335\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7336\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7337\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7338\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7339\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7340\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7341\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7342\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7343\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7344\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7345\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7346\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7347\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7348\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7349\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7350\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7351\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7352\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7353\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7354\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7355\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7356\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7357\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7358\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7359\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7360\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7361\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7362\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7363\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7364\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7365\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7366\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7367\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7368\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7369\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7370\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7371\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7442\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7443\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7444\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7445\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7446\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7447\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7448\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7449\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7450\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7451\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7452\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7471\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7472\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7473\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7474\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7475\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7476\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7477\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7478\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7479\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7480\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7481\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7373\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7375\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7376\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7377\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7378\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7379\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7380\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7381\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7382\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7383\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7384\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7385\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7386\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7387\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7388\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7389\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7390\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7391\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7392\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7393\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7394\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7395\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7396\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7397\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7398\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7399\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7400\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7401\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7402\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7403\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7404\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7405\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7406\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7407\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7408\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7409\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7410\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7411\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7412\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7413\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7414\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7415\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7416\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7417\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7418\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7419\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7420\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7421\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7422\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7423\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7424\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7425\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7426\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7427\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7455\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7456\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7457\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7458\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7459\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7460\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7461\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7462\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7463\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7464\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7465\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7466\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7467\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7468\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7469\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7484\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7485\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7486\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7487\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7488\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7489\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1581\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1584\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1585\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1590\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1593\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1595\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1598\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1599\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1604\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1605\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1608\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1609\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1610\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1611\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1613\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1614\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1619\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1620\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1623\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1624\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1625\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1627\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1630\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1632\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1635\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1637\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1638\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1639\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1642\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1646\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1652\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1653\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1659\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1660\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1662\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1663\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1665\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1666\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1673\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1676\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1678\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1679\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1681\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1682\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1684\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1685\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1687\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1689\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1691\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1692\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1693\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1694\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1695\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1696\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1697\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1699\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1700\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1703\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1704\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1706\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1708\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1710\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1718\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1726\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1727\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1729\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1730\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1735\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1737\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1738\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1739\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1740\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1741\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1742\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1743\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1744\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1745\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1746\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1747\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1748\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1749\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1750\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1751\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1752\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1753\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1754\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1756\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1758\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1759\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1760\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1761\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1762\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1766\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1767\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1768\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1769\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1770\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1771\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1784\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1786\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1790\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1792\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1794\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1796\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1797\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1798\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1800\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1805\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1806\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1807\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1810\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1811\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1813\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1832\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1833\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1846\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3799\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3908\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3910\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4679\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8065\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1847\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1849\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1852\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1853\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1859\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1868\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1870\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1872\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1874\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1875\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1876\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1877\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1878\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1879\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1880\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1881\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1882\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1883\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1884\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1885\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1886\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1887\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1888\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1889\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1890\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1891\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1892\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1893\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1894\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1895\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1896\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1897\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8262\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8263\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8264\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8267\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8268\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8271\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8274\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8275\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8276\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8277\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8278\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8279\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8280\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8284\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8285\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8318\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8325\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0712\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4228\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4251\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4252\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4254\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4255\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4257\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4260\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4261\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4262\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4263\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4264\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4265\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4269\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4270\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4271\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4276\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4278\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4662\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7623\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7625\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8790\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8791\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3871\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4058\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4498\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4500\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4714\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4715\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4716\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4717\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4718\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4723\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4725\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4727\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4730\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4731\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4732\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4733\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4734\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7731\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0663\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4308\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6539\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6540\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6542\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6543\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6544\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6545\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6546\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6547\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6548\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6549\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6550\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6551\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6552\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6553\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6554\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6555\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6556\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6557\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6558\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6559\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6560\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6561\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6562\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6563\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6564\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6565\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6566\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6567\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6568\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6569\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6570\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6571\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6572\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6573\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6574\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6575\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6576\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6579\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6581\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6583\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6585\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6587\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6591\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6592\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6595\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7652\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7654\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7655\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7656\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8799\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8800\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8803\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8804\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8805\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8807\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8813\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8815\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8816\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8818\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8821\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8822\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8823\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8824\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8830\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8832\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8833\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8835\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8836\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3923\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3925\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3929\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4332\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4333\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4335\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4336\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4338\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4340\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4440\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4442\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4444\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4503\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4505\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4654\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4655\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4656\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4665\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9726\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4666\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7145\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7146\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7147\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7908\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7909\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7910\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7911\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7912\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7913\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7916\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7919\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7920\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7921\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7922\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7925\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8105\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8106\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8107\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8108\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8109\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8110\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8112\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8113\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8114\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8115\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8116\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8117\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8129\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8130\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8132\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8133\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8135\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8136\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8137\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8138\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8139\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8140\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8141\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8142\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8143\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8144\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8758\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7661\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7662\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7663\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7668\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7669\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7670\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7671\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7672\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7673\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4160\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4162\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4163\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4165\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4166\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4167\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4169\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4170\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4523\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4539\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4540\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4541\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4542\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4543\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4544\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4816\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4817\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4818\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4833\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4834\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4835\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4836\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8538\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8738\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8739\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6396\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3991\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4657\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4744\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4746\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4748\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4750\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4752\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4755\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4757\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4758\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4762\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4771\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4772\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6630\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6631\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6632\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6633\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6634\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6635\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6636\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6638\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6639\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6641\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6642\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6643\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6644\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6645\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6646\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6647\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6648\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6649\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6650\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6651\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6652\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6653\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6654\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6655\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6656\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6657\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6658\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6659\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6664\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6665\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6668\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6669\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6671\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6674\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6675\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6676\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6677\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6678\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6679\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6680\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6682\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6687\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6690\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6691\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6692\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6693\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6700\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6701\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6702\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6703\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8698\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8709\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8710\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8711\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6506\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6507\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6508\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6509\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6511\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4537\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4545\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4548\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4558\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4559\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4560\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4204\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4206\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4207\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4208\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8613\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8615\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8616\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4064\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4065\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4066\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4067\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4068\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4069\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4070\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4071\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8102\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8724\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8725\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2114\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2117\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2128\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2129\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2130\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2131\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2132\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2133\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2134\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2135\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2136\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2137\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2138\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2139\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2140\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2142\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2143\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2144\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2145\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7996\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7999\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8008\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6426\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6427\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6428\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6429\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6432\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6433\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6434\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6435\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6436\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6441\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6442\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6443\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6444\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6447\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6448\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6450\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6451\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6453\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6464\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6465\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6466\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6467\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6470\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6471\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6472\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6473\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6476\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6477\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6478\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6479\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6482\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6484\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6486\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6490\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6492\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6495\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6496\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6992\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6993\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6994\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6995\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6996\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6997\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6998\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6999\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7000\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7001\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7002\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7003\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7004\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7005\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7013\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7014\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7017\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7019\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7021\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7022\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7023\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7024\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7025\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7026\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7027\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7028\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7029\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7030\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7033\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7034\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7035\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7036\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7038\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7039\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7040\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7041\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7042\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7048\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7050\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7052\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7053\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7054\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7055\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7056\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7057\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7058\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7059\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7060\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7061\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7065\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7066\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7078\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7079\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7080\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7081\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7082\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7083\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7084\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7088\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7089\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7099\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7100\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7688\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8059\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6535\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6536\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9569\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3955\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3984\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4201\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4202\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5406\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8751\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8752\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9488\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9489\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9493\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9494\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9516\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9517\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9577\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9578\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9581\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9582\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9583\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9584\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0010\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0014\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0031\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0032\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0545\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0546\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0547\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0548\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0549\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0550\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0551\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0553\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0556\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0559\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0560\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0561\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0689\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5233\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5238\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5247\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0004\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0006\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0007\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0008\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0009\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0015\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0018\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0019\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0155\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0164\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0167\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0170\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0173\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0176\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0179\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0183\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0187\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0190\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0191\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0194\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0195\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0198\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0199\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0202\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0203\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0207\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0208\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0211\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0212\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0215\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0216\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0224\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0225\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0231\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0232\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0235\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0236\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0239\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0240\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0243\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0244\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0247\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0248\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0253\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0254\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0257\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0258\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0261\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0262\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0265\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0266\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0269\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0270\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0273\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0274\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0277\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0278\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0281\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0282\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0287\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0288\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0291\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0292\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0295\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0296\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0299\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0300\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0303\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0304\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0307\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0308\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0311\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0312\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0317\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0318\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0321\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0322\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0325\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0326\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0329\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0330\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0335\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0336\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0339\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0340\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0343\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0344\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0347\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0348\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0351\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0352\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0355\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0356\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0359\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0360\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0363\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0364\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0367\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0368\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0371\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0372\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0375\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0376\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0379\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0380\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0383\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0384\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0387\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0388\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0391\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0392\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0395\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0396\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0399\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0400\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0403\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0404\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0407\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0408\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0411\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0412\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0415\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0416\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0419\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0420\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0423\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0424\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0427\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0428\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0431\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0432\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0435\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0436\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0439\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0440\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0444\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0470\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0476\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0770\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1082\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1086\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1898\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1899\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1900\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1901\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1902\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1903\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1904\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1905\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1906\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1907\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1908\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1909\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1911\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1914\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1919\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3858\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3959\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4249\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4321\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4691\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4721\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4844\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4849\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4858\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4861\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4873\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4885\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4896\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4910\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4911\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4912\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4919\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4928\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4931\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4932\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4933\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4935\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4938\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4946\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4947\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4954\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4956\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4973\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4975\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4976\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4994\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4996\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4999\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5000\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5003\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5008\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5010\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5018\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5021\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5023\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5029\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5032\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5034\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5035\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5037\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5042\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5068\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5070\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5071\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5073\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5074\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5076\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5077\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5078\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5079\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5080\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5082\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5084\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5085\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5087\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5088\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5089\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5091\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5092\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5094\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5198\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5232\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5237\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5241\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5246\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5295\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5296\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5303\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5305\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5307\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5308\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5310\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5311\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5313\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5314\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5315\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5316\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5317\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5318\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5319\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5320\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5322\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5324\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5326\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5328\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5330\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5333\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5413\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5429\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5430\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5432\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5440\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5442\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5443\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5444\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5445\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5446\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5447\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5450\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5451\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5454\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5455\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5458\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5459\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5460\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5461\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5462\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5463\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5464\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5465\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5466\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5467\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5468\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5469\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5470\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5471\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5472\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5473\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5474\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5475\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5476\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5477\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5478\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5479\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5480\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5481\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5482\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5483\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5484\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5485\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5486\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5487\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5488\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5489\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5490\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5491\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5492\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5493\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5494\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5495\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5496\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5497\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5498\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5499\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5500\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5501\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5502\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5503\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5504\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5505\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5506\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5507\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5508\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5509\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5510\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5511\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5512\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5513\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5514\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5515\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5516\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5517\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5518\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5519\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5520\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5521\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5522\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5523\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5524\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5525\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5526\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5527\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5528\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5529\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5530\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5531\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5532\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5533\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5534\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5535\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5536\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5537\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5538\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5539\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5540\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5541\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5542\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5543\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5544\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5545\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5546\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5547\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5548\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5549\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5550\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5551\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5552\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5553\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5554\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5555\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5556\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5557\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5558\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5559\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5560\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5561\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5562\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5563\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5564\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5565\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5566\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5567\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5568\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5569\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5570\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5571\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5572\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5573\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5574\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5575\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5576\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5577\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5578\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5579\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5580\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5581\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5582\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5583\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5584\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5585\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5586\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5587\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5588\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5589\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5590\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5591\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5592\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5593\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5594\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5595\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5596\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5597\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5598\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5599\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5600\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5601\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5602\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5603\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5604\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5605\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5606\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5607\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5608\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5609\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5610\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5611\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5612\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5613\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5614\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5615\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5616\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5617\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5618\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5619\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5620\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5621\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5622\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5623\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5624\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5625\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5626\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5627\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5628\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5629\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5630\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5631\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5632\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5633\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5634\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5635\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5636\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5637\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5638\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5639\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5640\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5641\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5642\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5643\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5644\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5645\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5646\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5647\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5648\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5649\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5650\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5651\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5652\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5653\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5654\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5655\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5656\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5657\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5658\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5659\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5660\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5661\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5662\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5663\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5664\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5665\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5666\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5667\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5668\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5669\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5670\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5671\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5672\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5673\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5674\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5675\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5676\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5677\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5678\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5679\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5680\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5681\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5682\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5683\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5684\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5685\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5686\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5687\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5688\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5689\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5690\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5691\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5692\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5693\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5694\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5695\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5696\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5697\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5698\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5699\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5700\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5701\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5702\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5703\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5704\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5705\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5706\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5707\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5708\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5709\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5710\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5711\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5712\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5713\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5714\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5715\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5716\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5717\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5718\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5719\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5720\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5721\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5722\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5723\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5724\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5725\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5726\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5727\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5728\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5729\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5730\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5731\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5732\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5733\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5734\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5735\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5736\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5737\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5738\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5739\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5740\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5741\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5742\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5743\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5744\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5745\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5746\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5747\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5748\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5749\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5750\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5751\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5752\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5753\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5754\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5755\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5756\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5757\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5758\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5759\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5760\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5761\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5762\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5763\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5764\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5765\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5766\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5767\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5768\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5769\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5770\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5771\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5772\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5773\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5774\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5775\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5776\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5777\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5778\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5779\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5780\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5781\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5782\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5783\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5784\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5785\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5786\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5787\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5788\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5789\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5790\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5791\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5792\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5793\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5794\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5795\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5796\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5797\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5798\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5799\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5800\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5801\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5802\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5803\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5804\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5805\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5806\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5807\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5808\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5809\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5810\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5811\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5812\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5813\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5814\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5815\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5816\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5817\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5818\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5819\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5820\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5821\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5822\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5823\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5824\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5825\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5826\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5827\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5828\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5829\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5830\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5831\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5832\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5833\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5834\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5835\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5836\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5837\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5838\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5839\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5840\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5841\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5842\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5843\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5844\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5845\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5846\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5847\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5848\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5849\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5850\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5851\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5852\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5853\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5854\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5855\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5856\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5857\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5858\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5859\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5860\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5861\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5862\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5863\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5864\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5865\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5866\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5867\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5868\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5869\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5870\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5871\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5872\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5873\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5874\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5875\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5876\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5877\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5878\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5879\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5880\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5881\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5882\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5883\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5884\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5885\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5886\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5887\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5888\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5889\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5890\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5891\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5892\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5893\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5894\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5895\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5896\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5897\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5898\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5899\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5900\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5901\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5902\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5903\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5904\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5905\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5906\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5907\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5908\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5909\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5910\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5911\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5912\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5913\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5914\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5915\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5916\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5917\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5918\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5919\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5920\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5921\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5922\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5923\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5924\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5925\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5926\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5927\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5928\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5929\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5930\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5931\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5932\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5933\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5934\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5935\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5936\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5937\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5938\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5939\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5940\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5941\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5942\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5943\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5944\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5945\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5946\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5947\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5948\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5949\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5950\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5951\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5952\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5953\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5954\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5955\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5956\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5957\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5958\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5959\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5960\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5961\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5962\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5963\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5964\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5965\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5966\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5967\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5968\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5969\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5970\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5971\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5972\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5973\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5974\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5975\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5976\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5977\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5978\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5979\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5980\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5981\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5982\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5983\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5984\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5985\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5986\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5987\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5989\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5990\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5992\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5994\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5996\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5997\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5998\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6000\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6001\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6002\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6003\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6004\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6005\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6006\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6007\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6008\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6009\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6010\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6011\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6012\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6013\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6014\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6015\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6016\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6017\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6018\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6019\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6020\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6021\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6022\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6023\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6024\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6025\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6026\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6027\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6028\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6029\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6030\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6031\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6032\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6033\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6034\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6035\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6036\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6037\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6038\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6039\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6040\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6041\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6042\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6043\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6044\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6045\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6046\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6047\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6048\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6049\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6050\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6051\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6052\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6053\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6054\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6055\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6058\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6059\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6091\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6096\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6098\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6101\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6129\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6190\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6241\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6243\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6244\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6245\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6246\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6247\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6248\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6249\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6250\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6254\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6257\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6258\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6275\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6366\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6367\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6368\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6369\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6370\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6371\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6372\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6373\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6374\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6375\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6377\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6379\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6380\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6381\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6382\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6383\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6384\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6392\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6406\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6424\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6439\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6462\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6474\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6505\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6516\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6525\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6526\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6527\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6530\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6531\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6532\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6533\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6534\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6732\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7109\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7111\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7113\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7115\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7117\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7119\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7121\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7123\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7125\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7127\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7633\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7634\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7635\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7636\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7637\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7639\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7643\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7644\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7645\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7650\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7651\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7666\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7667\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7682\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7683\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7684\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7685\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7686\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7687\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7690\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7691\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7692\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7735\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7787\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7896\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7898\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7900\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7902\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7904\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7905\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7951\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7964\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7967\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7982\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7986\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7990\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7994\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8057\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8075\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8076\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8077\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8081\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8225\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8340\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8354\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8364\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8374\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8408\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8429\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8567\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8586\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8612\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8629\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8633\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8658\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8670\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8676\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8677\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8678\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8687\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8733\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8734\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8735\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8736\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8760\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8846\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8847\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8850\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8867\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8872\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8876\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8878\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8889\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8901\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8911\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8912\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8917\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8925\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8929\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8930\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8934\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8935\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8939\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8940\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8941\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8942\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8943\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8944\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8945\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8946\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8947\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9173\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9176\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9178\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9183\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9184\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9185\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9190\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9191\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9192\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9197\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9198\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9590\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9591\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9592\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9593\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9594\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9595\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9596\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9597\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9598\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9599\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9600\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9601\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9602\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9603\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9604\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9605\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9606\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9607\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9608\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9609\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9611\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9613\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9616\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9619\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9623\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9624\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9643\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9731\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4312\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8022\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0158\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0446\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0578\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0587\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0962\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1145\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1187\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1243\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1572\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1602\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1925\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1936\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2116\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2141\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2590\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3762\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3825\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3863\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3864\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3954\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3971\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4237\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4692\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4720\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4729\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4738\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4743\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4754\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4756\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4760\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4843\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4851\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4852\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4854\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4855\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4862\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4864\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4865\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4867\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4868\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4870\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4871\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4872\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4898\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4905\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4914\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4922\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4926\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4940\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4944\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4964\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4971\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4972\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4977\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4997\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5006\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5015\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5016\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5031\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5033\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5043\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5046\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5096\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5099\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5101\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5105\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5107\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5109\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5118\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5122\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5225\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5227\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5292\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5304\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5348\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5349\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5411\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5420\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6276\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6277\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6286\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6287\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6288\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6289\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6290\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6291\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6292\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6293\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6294\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6295\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6296\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6297\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6298\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6301\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6302\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6307\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6308\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6313\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6314\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6317\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6318\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6321\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6323\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6324\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6327\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6328\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6330\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6331\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6332\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6333\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6334\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6335\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6336\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6337\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6338\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6339\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6340\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6341\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6342\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6343\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6391\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6431\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6438\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6446\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6454\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6469\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6481\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6521\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6522\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6618\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6620\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6890\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6902\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7638\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7719\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7758\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7769\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7804\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7806\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7808\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7810\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7812\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7814\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7815\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7816\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7818\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7820\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7822\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7824\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7825\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7826\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7827\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7828\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7829\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7830\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7831\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7832\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7833\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7834\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7835\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7836\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7837\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7838\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7839\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7840\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7841\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7842\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7843\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7844\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7845\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7846\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7847\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7848\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7849\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7850\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7851\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7852\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7853\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7854\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7995\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8009\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8356\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8365\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8411\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8438\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8505\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8510\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8513\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8562\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8614\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8627\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8657\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8741\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8742\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8776\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8785\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0713\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1104\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1144\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1188\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1242\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1286\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2407\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2509\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2575\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3007\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3008\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3011\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3013\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3014\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3017\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3018\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3019\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3020\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3021\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3022\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3023\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3024\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3025\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3473\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3850\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4393\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4468\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4930\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5452\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6928\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6986\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7590\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7596\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7640\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7646\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7707\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7788\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8016\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8414\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8756\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9678\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2564\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4274\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4277\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4535\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6617\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6619\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7653\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7664\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7765\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7767\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7792\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7796\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8024\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8028\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8082\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8412\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8476\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8639\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8789\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8828\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8840\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8844\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8638\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0791\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0831\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0929\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1917\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3987\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4907\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5047\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5048\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5049\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5050\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5051\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5052\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5053\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5054\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5055\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5056\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5057\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5058\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5059\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5060\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5061\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5062\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5063\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5064\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5065\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5066\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5067\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5197\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5224\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5226\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5228\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7196\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7224\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7374\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7430\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7454\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7483\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7500\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7512\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7524\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7540\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7556\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7570\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7581\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7584\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7711\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7714\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7715\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7718\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7720\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7722\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7724\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7726\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7727\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7729\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7730\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7762\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7799\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7924\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8125\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8127\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8200\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8208\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8216\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8590\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8753\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8858\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8887\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8908\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0612\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3597\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5251\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5252\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7198\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7284\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0731\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0734\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0918\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1916\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4125\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7201\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7204\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7433\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7434\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7435\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7675\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7693\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7733\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7740\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7743\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8084\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8086\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8146\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8193\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8199\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8205\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8207\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8496\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8661\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7432\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7307\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1809\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0461\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0467\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0664\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0730\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1035\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1269\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1311\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1672\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2554\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4765\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4856\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4860\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4878\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4880\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4881\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4894\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4906\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4942\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_5027\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6385\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6386\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6388\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6425\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6440\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6463\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6475\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6629\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_6797\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7262\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7272\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7273\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7649\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7789\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7944\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7947\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7949\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7963\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7966\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7981\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7983\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7985\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7989\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8058\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8234\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8239\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8594\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8851\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_8915\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9177\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9732\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9725\n",
      "\n",
      "Changing gene_reaction_rule for reaction cofactors_vitamins\n",
      "\n",
      "Changing gene_reaction_rule for reaction vitaminA\n",
      "\n",
      "Changing gene_reaction_rule for reaction vitaminD\n",
      "\n",
      "Changing gene_reaction_rule for reaction vitaminE\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4953\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3106\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_3947\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0685\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0686\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_2393\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_4783\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7777\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7779\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_1095\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_0020\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7108\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7110\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7112\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7114\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7116\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7118\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7120\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7122\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7124\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_7126\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9024\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9025\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9027\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9028\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9029\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9030\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9031\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9808\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9809\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9810\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9811\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9812\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9813\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9814\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9815\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9816\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9033\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9034\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9035\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9036\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9037\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9038\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9039\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9040\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9041\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9042\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9043\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9044\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9045\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9046\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9047\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9048\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9049\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9051\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9052\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9053\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9054\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9055\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9058\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9061\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9062\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9063\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9064\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9065\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9066\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9067\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9068\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9069\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9070\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9071\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9072\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9073\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9074\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9075\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9076\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9077\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9078\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9079\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9080\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9081\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9083\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9084\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9085\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9086\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9087\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9088\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9091\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9092\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9093\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9094\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9095\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9096\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9097\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9098\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9099\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9100\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9101\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9102\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9105\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9106\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9107\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9109\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9110\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9111\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9113\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9114\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9115\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9116\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9117\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9118\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9119\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9120\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9121\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9122\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9124\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9125\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9126\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9129\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9132\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9133\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9134\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9135\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9136\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9137\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9139\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9140\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9141\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9143\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9145\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9146\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9150\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9151\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9152\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9153\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9154\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9155\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9156\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9158\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9159\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9160\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9161\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9162\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9165\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9166\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9167\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9168\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9169\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9201\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9203\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9204\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9205\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9206\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9207\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9208\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9216\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9219\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9220\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9230\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9232\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9234\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9235\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9236\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9238\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9239\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9240\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9243\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9247\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9254\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9255\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9257\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9259\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9260\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9262\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9264\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9269\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9270\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9271\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9272\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9273\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9275\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9279\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9280\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9281\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9282\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9283\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9284\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9285\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9286\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9287\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9290\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9292\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9293\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9295\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9296\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9297\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9300\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9301\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9302\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9303\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9305\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9310\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9312\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9315\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9316\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9317\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9318\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9330\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9336\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9338\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9340\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9341\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9343\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9346\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9348\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9350\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9351\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9352\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9354\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9359\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9360\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9361\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9362\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9363\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9364\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9366\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9367\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9368\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9370\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9372\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9383\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9386\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9394\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9396\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9397\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9398\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9399\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9404\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9407\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9411\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9412\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9415\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9416\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9417\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9418\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9421\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9423\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9424\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9426\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9427\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9430\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9435\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9436\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9438\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9439\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9440\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9441\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9447\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9448\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9450\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9452\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9453\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9456\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9457\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9458\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9460\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9682\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9683\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9685\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9692\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9693\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9695\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9715\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9730\n",
      "\n",
      "Changing gene_reaction_rule for reaction HMR_9736\n",
      "\n",
      "Changing gene_reaction_rule for reaction biomass_components\n"
     ]
    }
   ],
   "source": [
    "Hep_G2_unip = mc.convert_ids(Hep_G2)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "8fac3a62",
   "metadata": {},
   "source": [
    "---\n",
    "#### Ajout de la réaction de biomasse fonctionnelle\n",
    "\n",
    "[Fichier](./Model_correction.py)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "0d949349",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'EX_temp001x' since it already exists.\n"
     ]
    }
   ],
   "source": [
    "Hep_G2_unip_ab = mc.add_biomass_reaction(Hep_G2_unip, Hep_G2_dir+\"utils_biomass_metabolites.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "49bba1f1",
   "metadata": {},
   "outputs": [],
   "source": [
    "Hep_G2_unip_ab = mc.fix_formulas(Hep_G2_unip_ab)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "2464ed06",
   "metadata": {},
   "source": [
    "---\n",
    "#### Enregistrement du modèle corrigé"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "b3659a73",
   "metadata": {},
   "outputs": [],
   "source": [
    "cobra.io.save_json_model(Hep_G2_unip_ab, Hep_G2_dir+\"Hep-G2_json.json\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "7d4bb9f6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Q92506 already found somewhere else.\n",
      "P12532 already found somewhere else.\n",
      "O96024 already found somewhere else.\n",
      "Q99943 already found somewhere else.\n",
      "P07327 already found somewhere else.\n",
      "P00326 already found somewhere else.\n",
      "P28332 already found somewhere else.\n",
      "P08319 already found somewhere else.\n",
      "B4DFI7 already found somewhere else.\n",
      "P00325 already found somewhere else.\n",
      "Q8N4Q0 already found somewhere else.\n",
      "P40394 already found somewhere else.\n",
      "D6RAY0 already found somewhere else.\n",
      "P14550 already found somewhere else.\n",
      "Q9NR19 already found somewhere else.\n",
      "Q9H6R3 already found somewhere else.\n",
      "Q9NUB1 already found somewhere else.\n",
      "O00330 already found somewhere else.\n",
      "P10515 already found somewhere else.\n",
      "P11177 already found somewhere else.\n",
      "P08559 already found somewhere else.\n",
      "P09622 already found somewhere else.\n",
      "P29803 already found somewhere else.\n",
      "Q6ZMR3 already found somewhere else.\n",
      "Q8IX04 already found somewhere else.\n",
      "P07195 already found somewhere else.\n",
      "Q9BYZ2 already found somewhere else.\n",
      "P07864 already found somewhere else.\n",
      "P47895 already found somewhere else.\n",
      "P30838 already found somewhere else.\n",
      "P48448 already found somewhere else.\n",
      "P43353 already found somewhere else.\n",
      "P49419 already found somewhere else.\n",
      "P30837 already found somewhere else.\n",
      "P51648 already found somewhere else.\n",
      "P05091 already found somewhere else.\n",
      "P08237 already found somewhere else.\n",
      "F8WEU2 already found somewhere else.\n",
      "O43822 already found somewhere else.\n",
      "Q01813 already found somewhere else.\n",
      "P05062 already found somewhere else.\n",
      "P09972 already found somewhere else.\n",
      "P04075 already found somewhere else.\n",
      "Q9P1Z3 already found somewhere else.\n",
      "Q969K7 already found somewhere else.\n",
      "P14618 already found somewhere else.\n",
      "P30613 already found somewhere else.\n",
      "P09104 already found somewhere else.\n",
      "A0A024R4F1 already found somewhere else.\n",
      "P13929 already found somewhere else.\n",
      "A0A024R782 already found somewhere else.\n",
      "Q8N0Y7 already found somewhere else.\n",
      "P18669 already found somewhere else.\n",
      "P15259 already found somewhere else.\n",
      "P07205 already found somewhere else.\n",
      "Q5JRA6 already found somewhere else.\n",
      "P00558 already found somewhere else.\n",
      "P14621 already found somewhere else.\n",
      "P07311 already found somewhere else.\n",
      "P04406 already found somewhere else.\n",
      "O14556 already found somewhere else.\n",
      "P06744 already found somewhere else.\n",
      "P60174 already found somewhere else.\n",
      "P35557 already found somewhere else.\n",
      "Q9BRR6 already found somewhere else.\n",
      "P52790 already found somewhere else.\n",
      "P52789 already found somewhere else.\n",
      "P19367 already found somewhere else.\n",
      "Q2TB90 already found somewhere else.\n",
      "Q96G03 already found somewhere else.\n",
      "P36871 already found somewhere else.\n",
      "Q15124 already found somewhere else.\n",
      "Q9NQR9 already found somewhere else.\n",
      "P35575 already found somewhere else.\n",
      "K7ENK1 already found somewhere else.\n",
      "Q96C23 already found somewhere else.\n",
      "P04040 already found somewhere else.\n",
      "P20807 already found somewhere else.\n",
      "H3BRQ6 already found somewhere else.\n",
      "P10253 already found somewhere else.\n",
      "O60701 already found somewhere else.\n",
      "O43280 already found somewhere else.\n",
      "Q32M88 already found somewhere else.\n",
      "O15488 already found somewhere else.\n",
      "G5E9W8 already found somewhere else.\n",
      "P13807 already found somewhere else.\n",
      "P54840 already found somewhere else.\n",
      "Q9UQK1 already found somewhere else.\n",
      "Q86XI6 already found somewhere else.\n",
      "Q04446 already found somewhere else.\n",
      "P11216 already found somewhere else.\n",
      "P11217 already found somewhere else.\n",
      "P06737 already found somewhere else.\n",
      "P35573 already found somewhere else.\n",
      "P0DUB6 already found somewhere else.\n",
      "P0DTE8 already found somewhere else.\n",
      "P0DTE7 already found somewhere else.\n",
      "P04746 already found somewhere else.\n",
      "P19961 already found somewhere else.\n",
      "C9J3M0 already found somewhere else.\n",
      "Q5QPP1 already found somewhere else.\n",
      "P51570 already found somewhere else.\n",
      "Q01415 already found somewhere else.\n",
      "P07902 already found somewhere else.\n",
      "P14410 already found somewhere else.\n",
      "Q96M60 already found somewhere else.\n",
      "Q6UWU2 already found somewhere else.\n",
      "P09848 already found somewhere else.\n",
      "F8WEN1 already found somewhere else.\n",
      "A0A3B3IRU3 already found somewhere else.\n",
      "O60909 already found somewhere else.\n",
      "P15291 already found somewhere else.\n",
      "P00709 already found somewhere else.\n",
      "P50053 already found somewhere else.\n",
      "Q16877 already found somewhere else.\n",
      "O60825 already found somewhere else.\n",
      "Q16875 already found somewhere else.\n",
      "I1Z9G3 already found somewhere else.\n",
      "Q00796 already found somewhere else.\n",
      "O60218 already found somewhere else.\n",
      "P15121 already found somewhere else.\n",
      "B4DYB8 already found somewhere else.\n",
      "H3BM92 already found somewhere else.\n",
      "A0A024R1P3 already found somewhere else.\n",
      "Q92871 already found somewhere else.\n",
      "A0A804HJC3 already found somewhere else.\n",
      "Q5XPI4 already found somewhere else.\n",
      "A0A024R329 already found somewhere else.\n",
      "O60547 already found somewhere else.\n",
      "E9PLH9 already found somewhere else.\n",
      "Q59H18 already found somewhere else.\n",
      "Q8N0W3 already found somewhere else.\n",
      "Q9NQ88 already found somewhere else.\n",
      "Q7Z4W1 already found somewhere else.\n",
      "Q9BY49 already found somewhere else.\n",
      "Q9BV79 already found somewhere else.\n",
      "O75191 already found somewhere else.\n",
      "O43488 already found somewhere else.\n",
      "Q9H477 already found somewhere else.\n",
      "Q9Y673 already found somewhere else.\n",
      "Q8NBZ7 already found somewhere else.\n",
      "I3NI24 already found somewhere else.\n",
      "Q04760 already found somewhere else.\n",
      "Q16775 already found somewhere else.\n",
      "B4DED4 already found somewhere else.\n",
      "Q86WU2 already found somewhere else.\n",
      "Q6P9G0 already found somewhere else.\n",
      "E9PND9 already found somewhere else.\n",
      "P23368 already found somewhere else.\n",
      "P48163 already found somewhere else.\n",
      "Q9UIY3 already found somewhere else.\n",
      "Q16822 already found somewhere else.\n",
      "P11498 already found somewhere else.\n",
      "Q9UBQ7 already found somewhere else.\n",
      "Q13825 already found somewhere else.\n",
      "P00338 already found somewhere else.\n",
      "Q9H845 already found somewhere else.\n",
      "P45954 already found somewhere else.\n",
      "Q709F0 already found somewhere else.\n",
      "P16219 already found somewhere else.\n",
      "Q9UKU7 already found somewhere else.\n",
      "P11310 already found somewhere else.\n",
      "Q6JQN1 already found somewhere else.\n",
      "O95822 already found somewhere else.\n",
      "Q02252 already found somewhere else.\n",
      "Q15392 already found somewhere else.\n",
      "Q6NVY1 already found somewhere else.\n",
      "Q9HBH5 already found somewhere else.\n",
      "Q8NEX9 already found somewhere else.\n",
      "P80365 already found somewhere else.\n",
      "Q9BUT1 already found somewhere else.\n",
      "Q9BPX1 already found somewhere else.\n",
      "A0A024R7X6 already found somewhere else.\n",
      "Q5I0G3 already found somewhere else.\n",
      "Q9H2F3 already found somewhere else.\n",
      "D6RCD0 already found somewhere else.\n",
      "Q9BUP3 already found somewhere else.\n",
      "Q8N4T8 already found somewhere else.\n",
      "Q8N3Y7 already found somewhere else.\n",
      "P17516 already found somewhere else.\n",
      "Q96NR8 already found somewhere else.\n",
      "Q04828 already found somewhere else.\n",
      "Q92506 already found somewhere else.\n",
      "Q13363 already found somewhere else.\n",
      "Q8NBN7 already found somewhere else.\n",
      "Q9BWD1 already found somewhere else.\n",
      "Q86V21 already found somewhere else.\n",
      "Q02338 already found somewhere else.\n",
      "P54868 already found somewhere else.\n",
      "Q01581 already found somewhere else.\n",
      "P11908 already found somewhere else.\n",
      "A0A6Q8PG31 already found somewhere else.\n",
      "P21108 already found somewhere else.\n",
      "P11413 already found somewhere else.\n",
      "Q9NXR7 already found somewhere else.\n",
      "P49247 already found somewhere else.\n",
      "Q9Y315 already found somewhere else.\n",
      "Q9H0I9 already found somewhere else.\n",
      "P29401 already found somewhere else.\n",
      "P51854 already found somewhere else.\n",
      "P52209 already found somewhere else.\n",
      "Q96AT9 already found somewhere else.\n",
      "P37837 already found somewhere else.\n",
      "O95479 already found somewhere else.\n",
      "O95336 already found somewhere else.\n",
      "O75355 already found somewhere else.\n",
      "O75354 already found somewhere else.\n",
      "K7EIP9 already found somewhere else.\n",
      "Q9Y227 already found somewhere else.\n",
      "A0A024R6D3 already found somewhere else.\n",
      "P49961 already found somewhere else.\n",
      "Q5MY95 already found somewhere else.\n",
      "Q9Y5L3 already found somewhere else.\n",
      "P51828 already found somewhere else.\n",
      "Q02153 already found somewhere else.\n",
      "Q8NFM4 already found somewhere else.\n",
      "Q02846 already found somewhere else.\n",
      "Q96PN6 already found somewhere else.\n",
      "P51841 already found somewhere else.\n",
      "O60266 already found somewhere else.\n",
      "O95622 already found somewhere else.\n",
      "P16066 already found somewhere else.\n",
      "P25092 already found somewhere else.\n",
      "Q6PJR4 already found somewhere else.\n",
      "Q08828 already found somewhere else.\n",
      "Q08462 already found somewhere else.\n",
      "O43306 already found somewhere else.\n",
      "P20594 already found somewhere else.\n",
      "O60503 already found somewhere else.\n",
      "P33402 already found somewhere else.\n",
      "P40145 already found somewhere else.\n",
      "Q96MA6 already found somewhere else.\n",
      "Q9UIJ7 already found somewhere else.\n",
      "Q9Y6K8 already found somewhere else.\n",
      "P00568 already found somewhere else.\n",
      "P27144 already found somewhere else.\n",
      "P54819 already found somewhere else.\n",
      "Q96M32 already found somewhere else.\n",
      "Q8TCD5 already found somewhere else.\n",
      "Q9BXI3 already found somewhere else.\n",
      "P49902 already found somewhere else.\n",
      "Q9NPB1 already found somewhere else.\n",
      "P15309 already found somewhere else.\n",
      "Q96P26 already found somewhere else.\n",
      "P21589 already found somewhere else.\n",
      "Q969T7 already found somewhere else.\n",
      "A0A024RA81 already found somewhere else.\n",
      "P27707 already found somewhere else.\n",
      "P55263 already found somewhere else.\n",
      "A0A1B1UZQ1 already found somewhere else.\n",
      "Q9HCR9 already found somewhere else.\n",
      "O95263 already found somewhere else.\n",
      "P51160 already found somewhere else.\n",
      "G3V1C5 already found somewhere else.\n",
      "Q14123 already found somewhere else.\n",
      "O76074 already found somewhere else.\n",
      "O60658 already found somewhere else.\n",
      "Q01064 already found somewhere else.\n",
      "O43924 already found somewhere else.\n",
      "O00408 already found somewhere else.\n",
      "Q13956 already found somewhere else.\n",
      "P35913 already found somewhere else.\n",
      "Q13946 already found somewhere else.\n",
      "Q14432 already found somewhere else.\n",
      "P27815 already found somewhere else.\n",
      "F1T0K3 already found somewhere else.\n",
      "Q08493 already found somewhere else.\n",
      "Q08499 already found somewhere else.\n",
      "Q13370 already found somewhere else.\n",
      "P18545 already found somewhere else.\n",
      "Q9NP56 already found somewhere else.\n",
      "P54750 already found somewhere else.\n",
      "O76083 already found somewhere else.\n",
      "Q07343 already found somewhere else.\n",
      "Q16774 already found somewhere else.\n",
      "P31939 already found somewhere else.\n",
      "C9J381 already found somewhere else.\n",
      "P12268 already found somewhere else.\n",
      "P30520 already found somewhere else.\n",
      "Q8N142 already found somewhere else.\n",
      "P49915 already found somewhere else.\n",
      "A0A5F9UK94 already found somewhere else.\n",
      "Q01432 already found somewhere else.\n",
      "P23109 already found somewhere else.\n",
      "P07741 already found somewhere else.\n",
      "Q9BY32 already found somewhere else.\n",
      "A8K4H7 already found somewhere else.\n",
      "A0A096LP92 already found somewhere else.\n",
      "Q9UPQ9 already found somewhere else.\n",
      "P00492 already found somewhere else.\n",
      "P36959 already found somewhere else.\n",
      "Q9P2T1 already found somewhere else.\n",
      "O15254 already found somewhere else.\n",
      "P49748 already found somewhere else.\n",
      "P00813 already found somewhere else.\n",
      "Q9NZK5 already found somewhere else.\n",
      "P00491 already found somewhere else.\n",
      "Q16854 already found somewhere else.\n",
      "E5KRG5 already found somewhere else.\n",
      "Q7LG56 already found somewhere else.\n",
      "P23921 already found somewhere else.\n",
      "P31350 already found somewhere else.\n",
      "P10599 already found somewhere else.\n",
      "P22102 already found somewhere else.\n",
      "P22234 already found somewhere else.\n",
      "O15067 already found somewhere else.\n",
      "Q9HA47 already found somewhere else.\n",
      "Q9BZX2 already found somewhere else.\n",
      "Q9NWZ5 already found somewhere else.\n",
      "P22079 already found somewhere else.\n",
      "P30041 already found somewhere else.\n",
      "P11678 already found somewhere else.\n",
      "A1KZ92 already found somewhere else.\n",
      "Q9UKK9 already found somewhere else.\n",
      "P56597 already found somewhere else.\n",
      "Q13232 already found somewhere else.\n",
      "A0A0C4DG91 already found somewhere else.\n",
      "Q32Q12 already found somewhere else.\n",
      "P15531 already found somewhere else.\n",
      "Q9Y5B8 already found somewhere else.\n",
      "O00746 already found somewhere else.\n",
      "P22392 already found somewhere else.\n",
      "Q16881 already found somewhere else.\n",
      "Q86VQ6 already found somewhere else.\n",
      "Q9NNW7 already found somewhere else.\n",
      "P30085 already found somewhere else.\n",
      "P17812 already found somewhere else.\n",
      "Q9NRF8 already found somewhere else.\n",
      "P27708 already found somewhere else.\n",
      "A8K5J1 already found somewhere else.\n",
      "O14638 already found somewhere else.\n",
      "P22413 already found somewhere else.\n",
      "O95848 already found somewhere else.\n",
      "O75063 already found somewhere else.\n",
      "Q16555 already found somewhere else.\n",
      "Q14195 already found somewhere else.\n",
      "Q14117 already found somewhere else.\n",
      "P32320 already found somewhere else.\n",
      "Q9GZX7 already found somewhere else.\n",
      "O95045 already found somewhere else.\n",
      "Q16831 already found somewhere else.\n",
      "Q8WW27 already found somewhere else.\n",
      "Q02127 already found somewhere else.\n",
      "P23919 already found somewhere else.\n",
      "P33316 already found somewhere else.\n",
      "P04818 already found somewhere else.\n",
      "P04183 already found somewhere else.\n",
      "O00142 already found somewhere else.\n",
      "Q5EBM0 already found somewhere else.\n",
      "A0A384ME17 already found somewhere else.\n",
      "Q9UQ16 already found somewhere else.\n",
      "Q969S9 already found somewhere else.\n",
      "P13639 already found somewhere else.\n",
      "Q05639 already found somewhere else.\n",
      "B4DDQ3 already found somewhere else.\n",
      "Q05193 already found somewhere else.\n",
      "Q7Z2Z2 already found somewhere else.\n",
      "P68104 already found somewhere else.\n",
      "Q96RP9 already found somewhere else.\n",
      "O95340 already found somewhere else.\n",
      "O43252 already found somewhere else.\n",
      "D6R9P0 already found somewhere else.\n",
      "Q9UNA4 already found somewhere else.\n",
      "P06746 already found somewhere else.\n",
      "D6RJD0 already found somewhere else.\n",
      "P56282 already found somewhere else.\n",
      "Q14181 already found somewhere else.\n",
      "Q9BQG0 already found somewhere else.\n",
      "Q9Y253 already found somewhere else.\n",
      "Q15054 already found somewhere else.\n",
      "P09884 already found somewhere else.\n",
      "P49005 already found somewhere else.\n",
      "O75417 already found somewhere else.\n",
      "Q9NRJ5 already found somewhere else.\n",
      "Q6PIY7 already found somewhere else.\n",
      "Q9BT43 already found somewhere else.\n",
      "Q9NVV4 already found somewhere else.\n",
      "P19388 already found somewhere else.\n",
      "O15514 already found somewhere else.\n",
      "Q9BWT3 already found somewhere else.\n",
      "P52434 already found somewhere else.\n",
      "Q6ZS17 already found somewhere else.\n",
      "P61218 already found somewhere else.\n",
      "A0A6Q8PGB0 already found somewhere else.\n",
      "Q9GZS1 already found somewhere else.\n",
      "O14802 already found somewhere else.\n",
      "O15160 already found somewhere else.\n",
      "Q9NW08 already found somewhere else.\n",
      "P36954 already found somewhere else.\n",
      "P0DPB6 already found somewhere else.\n",
      "P62487 already found somewhere else.\n",
      "Q9H1D9 already found somewhere else.\n",
      "O00411 already found somewhere else.\n",
      "P30876 already found somewhere else.\n",
      "P52435 already found somewhere else.\n",
      "O95602 already found somewhere else.\n",
      "Q8TCS8 already found somewhere else.\n",
      "O60671 already found somewhere else.\n",
      "P00367 already found somewhere else.\n",
      "P49448 already found somewhere else.\n",
      "P00966 already found somewhere else.\n",
      "O75575 already found somewhere else.\n",
      "P04424 already found somewhere else.\n",
      "A0A024RAC7 already found somewhere else.\n",
      "P00505 already found somewhere else.\n",
      "Q8NHS2 already found somewhere else.\n",
      "P17174 already found somewhere else.\n",
      "P07954 already found somewhere else.\n",
      "Q7L266 already found somewhere else.\n",
      "Q86U10 already found somewhere else.\n",
      "P00390 already found somewhere else.\n",
      "P31327 already found somewhere else.\n",
      "Q5TDP6 already found somewhere else.\n",
      "A8YXX4 already found somewhere else.\n",
      "B7Z509 already found somewhere else.\n",
      "G5E9X4 already found somewhere else.\n",
      "P24298 already found somewhere else.\n",
      "P08243 already found somewhere else.\n",
      "Q8TD30 already found somewhere else.\n",
      "Q9NQR4 already found somewhere else.\n",
      "Q16773 already found somewhere else.\n",
      "Q6YP21 already found somewhere else.\n",
      "Q8N5Z0 already found somewhere else.\n",
      "P80404 already found somewhere else.\n",
      "P17735 already found somewhere else.\n",
      "Q99489 already found somewhere else.\n",
      "O94903 already found somewhere else.\n",
      "Q96HD9 already found somewhere else.\n",
      "P45381 already found somewhere else.\n",
      "P04181 already found somewhere else.\n",
      "P78540 already found somewhere else.\n",
      "P05089 already found somewhere else.\n",
      "Q96C36 already found somewhere else.\n",
      "B5MD87 already found somewhere else.\n",
      "A0A024R8U9 already found somewhere else.\n",
      "E7EQL6 already found somewhere else.\n",
      "A0A590UKC3 already found somewhere else.\n",
      "P54886 already found somewhere else.\n",
      "Q9Y2D0 already found somewhere else.\n",
      "P35218 already found somewhere else.\n",
      "P35228 already found somewhere else.\n",
      "P29474 already found somewhere else.\n",
      "P29475 already found somewhere else.\n",
      "Q5VXN5 already found somewhere else.\n",
      "P19623 already found somewhere else.\n",
      "P52788 already found somewhere else.\n",
      "Q96A70 already found somewhere else.\n",
      "P11926 already found somewhere else.\n",
      "Q16853 already found somewhere else.\n",
      "O75106 already found somewhere else.\n",
      "P19801 already found somewhere else.\n",
      "Q9BSE5 already found somewhere else.\n",
      "V9HWB2 already found somewhere else.\n",
      "A5YM72 already found somewhere else.\n",
      "Q96KN2 already found somewhere else.\n",
      "Q96KP4 already found somewhere else.\n",
      "P13674 already found somewhere else.\n",
      "Q7Z4N8 already found somewhere else.\n",
      "Q86XE5 already found somewhere else.\n",
      "Q13126 already found somewhere else.\n",
      "Q96F10 already found somewhere else.\n",
      "P21673 already found somewhere else.\n",
      "Q9NWM0 already found somewhere else.\n",
      "Q6QHF9 already found somewhere else.\n",
      "P12277 already found somewhere else.\n",
      "P17540 already found somewhere else.\n",
      "P06732 already found somewhere else.\n",
      "P12532 already found somewhere else.\n",
      "Q9UL12 already found somewhere else.\n",
      "P27338 already found somewhere else.\n",
      "Q53YE7 already found somewhere else.\n",
      "P51649 already found somewhere else.\n",
      "Q5VV43 already found somewhere else.\n",
      "A1LQE8 already found somewhere else.\n",
      "A0A286YER3 already found somewhere else.\n",
      "Q8NBM4 already found somewhere else.\n",
      "Q9Y617 already found somewhere else.\n",
      "P78330 already found somewhere else.\n",
      "P34896 already found somewhere else.\n",
      "O75600 already found somewhere else.\n",
      "Q9P0Z9 already found somewhere else.\n",
      "Q96GA7 already found somewhere else.\n",
      "Q9GZT4 already found somewhere else.\n",
      "P20132 already found somewhere else.\n",
      "V9HW60 already found somewhere else.\n",
      "P22557 already found somewhere else.\n",
      "Q9UBL6 already found somewhere else.\n",
      "P13196 already found somewhere else.\n",
      "P21549 already found somewhere else.\n",
      "Q8NE62 already found somewhere else.\n",
      "Q93088 already found somewhere else.\n",
      "Q9H2M3 already found somewhere else.\n",
      "Q9UI17 already found somewhere else.\n",
      "Q8NCN5 already found somewhere else.\n",
      "Q9BYV1 already found somewhere else.\n",
      "V9HW06 already found somewhere else.\n",
      "Q5JU23 already found somewhere else.\n",
      "Q92820 already found somewhere else.\n",
      "P23434 already found somewhere else.\n",
      "P48728 already found somewhere else.\n",
      "P23378 already found somewhere else.\n",
      "P20711 already found somewhere else.\n",
      "P19113 already found somewhere else.\n",
      "P50135 already found somewhere else.\n",
      "Q4VB95 already found somewhere else.\n",
      "A0A0G2JQF2 already found somewhere else.\n",
      "Q96NU7 already found somewhere else.\n",
      "Q96N76 already found somewhere else.\n",
      "Q9Y530 already found somewhere else.\n",
      "Q8IXJ6 already found somewhere else.\n",
      "Q9BQ69 already found somewhere else.\n",
      "Q96EB6 already found somewhere else.\n",
      "Q9NTG7 already found somewhere else.\n",
      "Q02083 already found somewhere else.\n",
      "Q9NXA8 already found somewhere else.\n",
      "Q96HA8 already found somewhere else.\n",
      "Q12887 already found somewhere else.\n",
      "Q53H54 already found somewhere else.\n",
      "Q86SQ9 already found somewhere else.\n",
      "Q9HB03 already found somewhere else.\n",
      "Q9NXB9 already found somewhere else.\n",
      "A0A024RD35 already found somewhere else.\n",
      "Q9BW60 already found somewhere else.\n",
      "Q9H5J4 already found somewhere else.\n",
      "D6RBM2 already found somewhere else.\n",
      "Q9GZR5 already found somewhere else.\n",
      "P36957 already found somewhere else.\n",
      "Q9ULD0 already found somewhere else.\n",
      "Q96HY7 already found somewhere else.\n",
      "Q02218 already found somewhere else.\n",
      "Q9UDR5 already found somewhere else.\n",
      "Q8NBX0 already found somewhere else.\n",
      "O15047 already found somewhere else.\n",
      "Q9UPS6 already found somewhere else.\n",
      "Q9H5I1 already found somewhere else.\n",
      "Q8IZD2 already found somewhere else.\n",
      "Q4FZB7 already found somewhere else.\n",
      "Q8TEK3 already found somewhere else.\n",
      "O96028 already found somewhere else.\n",
      "Q9H7B4 already found somewhere else.\n",
      "H0YEU4 already found somewhere else.\n",
      "Q96T68 already found somewhere else.\n",
      "Q9BZ95 already found somewhere else.\n",
      "P61009 already found somewhere else.\n",
      "Q9Y6A9 already found somewhere else.\n",
      "Q15005 already found somewhere else.\n",
      "Q9NVH6 already found somewhere else.\n",
      "O75936 already found somewhere else.\n",
      "Q5T4B2 already found somewhere else.\n",
      "O00469 already found somewhere else.\n",
      "O60568 already found somewhere else.\n",
      "Q8IYK4 already found somewhere else.\n",
      "Q92800 already found somewhere else.\n",
      "P40939 already found somewhere else.\n",
      "P30084 already found somewhere else.\n",
      "Q16836 already found somewhere else.\n",
      "A0A0S2Z410 already found somewhere else.\n",
      "P24752 already found somewhere else.\n",
      "Q16678 already found somewhere else.\n",
      "P04798 already found somewhere else.\n",
      "P05177 already found somewhere else.\n",
      "P33261 already found somewhere else.\n",
      "P35914 already found somewhere else.\n",
      "P50225 already found somewhere else.\n",
      "P50226 already found somewhere else.\n",
      "P55084 already found somewhere else.\n",
      "P42765 already found somewhere else.\n",
      "P05165 already found somewhere else.\n",
      "P05166 already found somewhere else.\n",
      "Q96PE7 already found somewhere else.\n",
      "P22033 already found somewhere else.\n",
      "P54687 already found somewhere else.\n",
      "O15382 already found somewhere else.\n",
      "P31937 already found somewhere else.\n",
      "Q5VVL7 already found somewhere else.\n",
      "P21953 already found somewhere else.\n",
      "P12694 already found somewhere else.\n",
      "P00439 already found somewhere else.\n",
      "D6RA50 already found somewhere else.\n",
      "P14902 already found somewhere else.\n",
      "Q63HM1 already found somewhere else.\n",
      "Q16719 already found somewhere else.\n",
      "Q92947 already found somewhere else.\n",
      "Q96E66 already found somewhere else.\n",
      "P21964 already found somewhere else.\n",
      "A0A0B4J1R4 already found somewhere else.\n",
      "Q93099 already found somewhere else.\n",
      "A0A0C4DFM0 already found somewhere else.\n",
      "P16930 already found somewhere else.\n",
      "O95050 already found somewhere else.\n",
      "P19224 already found somewhere else.\n",
      "P22310 already found somewhere else.\n",
      "Q9HAW9 already found somewhere else.\n",
      "P36537 already found somewhere else.\n",
      "P06133 already found somewhere else.\n",
      "P0DTE4 already found somewhere else.\n",
      "P54855 already found somewhere else.\n",
      "P16662 already found somewhere else.\n",
      "Q9BY64 already found somewhere else.\n",
      "O75310 already found somewhere else.\n",
      "Q5DT02 already found somewhere else.\n",
      "P22309 already found somewhere else.\n",
      "Q6UWM9 already found somewhere else.\n",
      "O75795 already found somewhere else.\n",
      "P07202 already found somewhere else.\n",
      "P05164 already found somewhere else.\n",
      "P46439 already found somewhere else.\n",
      "P10620 already found somewhere else.\n",
      "Q03013 already found somewhere else.\n",
      "P78417 already found somewhere else.\n",
      "P08263 already found somewhere else.\n",
      "P09211 already found somewhere else.\n",
      "Q9Y2Q3 already found somewhere else.\n",
      "Q99735 already found somewhere else.\n",
      "Q16772 already found somewhere else.\n",
      "P09210 already found somewhere else.\n",
      "Q9H4Y5 already found somewhere else.\n",
      "P09488 already found somewhere else.\n",
      "Q7RTV2 already found somewhere else.\n",
      "P28161 already found somewhere else.\n",
      "O60760 already found somewhere else.\n",
      "P49895 already found somewhere else.\n",
      "A0A024R6J8 already found somewhere else.\n",
      "O43704 already found somewhere else.\n",
      "O95803 already found somewhere else.\n",
      "C9IZ77 already found somewhere else.\n",
      "Q9Y2C2 already found somewhere else.\n",
      "Q8IZP7 already found somewhere else.\n",
      "O43529 already found somewhere else.\n",
      "O75897 already found somewhere else.\n",
      "Q7L1S5 already found somewhere else.\n",
      "Q7LGA3 already found somewhere else.\n",
      "Q96A11 already found somewhere else.\n",
      "Q9NS84 already found somewhere else.\n",
      "P52849 already found somewhere else.\n",
      "Q8NCG5 already found somewhere else.\n",
      "P52848 already found somewhere else.\n",
      "Q9Y4C5 already found somewhere else.\n",
      "O00338 already found somewhere else.\n",
      "A0A590UJL0 already found somewhere else.\n",
      "B2C6G4 already found somewhere else.\n",
      "P31153 already found somewhere else.\n",
      "Q00266 already found somewhere else.\n",
      "Q9NZL9 already found somewhere else.\n",
      "O43865 already found somewhere else.\n",
      "Q96HN2 already found somewhere else.\n",
      "P23526 already found somewhere else.\n",
      "P35520 already found somewhere else.\n",
      "P32929 already found somewhere else.\n",
      "Q99707 already found somewhere else.\n",
      "Q16762 already found somewhere else.\n",
      "P25325 already found somewhere else.\n",
      "Q674R7 already found somewhere else.\n",
      "P51687 already found somewhere else.\n",
      "Q9BV20 already found somewhere else.\n",
      "Q96GX9 already found somewhere else.\n",
      "Q9UHY7 already found somewhere else.\n",
      "Q9BV57 already found somewhere else.\n",
      "P40925 already found somewhere else.\n",
      "P40926 already found somewhere else.\n",
      "Q9Y6K1 already found somewhere else.\n",
      "Q7Z3E4 already found somewhere else.\n",
      "O14841 already found somewhere else.\n",
      "A0A140VJJ2 already found somewhere else.\n",
      "Q06830 already found somewhere else.\n",
      "P22352 already found somewhere else.\n",
      "P18283 already found somewhere else.\n",
      "P59796 already found somewhere else.\n",
      "O75715 already found somewhere else.\n",
      "A6NIW5 already found somewhere else.\n",
      "P30048 already found somewhere else.\n",
      "P07203 already found somewhere else.\n",
      "P36969 already found somewhere else.\n",
      "Q96SL4 already found somewhere else.\n",
      "P19440 already found somewhere else.\n",
      "Q14390 already found somewhere else.\n",
      "P36269 already found somewhere else.\n",
      "B4DPG6 already found somewhere else.\n",
      "Q9UJ14 already found somewhere else.\n",
      "P48507 already found somewhere else.\n",
      "Q14TF0 already found somewhere else.\n",
      "A0A0S2Z4J7 already found somewhere else.\n",
      "P15144 already found somewhere else.\n",
      "P55786 already found somewhere else.\n",
      "P28838 already found somewhere else.\n",
      "O75223 already found somewhere else.\n",
      "Q96I15 already found somewhere else.\n",
      "Q99518 already found somewhere else.\n",
      "P31513 already found somewhere else.\n",
      "P49326 already found somewhere else.\n",
      "Q01740 already found somewhere else.\n",
      "P31512 already found somewhere else.\n",
      "Q06278 already found somewhere else.\n",
      "P26440 already found somewhere else.\n",
      "Q9HCC0 already found somewhere else.\n",
      "Q96RQ3 already found somewhere else.\n",
      "Q5URX0 already found somewhere else.\n",
      "H3BTD4 already found somewhere else.\n",
      "Q3KQV9 already found somewhere else.\n",
      "Q16222 already found somewhere else.\n",
      "Q8TDQ7 already found somewhere else.\n",
      "P46926 already found somewhere else.\n",
      "Q06210 already found somewhere else.\n",
      "A0A0S2Z4X9 already found somewhere else.\n",
      "Q9UJ70 already found somewhere else.\n",
      "Q9Y223 already found somewhere else.\n",
      "P51606 already found somewhere else.\n",
      "Q9NR45 already found somewhere else.\n",
      "Q8TBE9 already found somewhere else.\n",
      "Q8NFW8 already found somewhere else.\n",
      "Q96EK6 already found somewhere else.\n",
      "Q9Y303 already found somewhere else.\n",
      "O95394 already found somewhere else.\n",
      "O95455 already found somewhere else.\n",
      "Q9H8J5 already found somewhere else.\n",
      "Q13231 already found somewhere else.\n",
      "Q9BZP6 already found somewhere else.\n",
      "Q9Y2Z4 already found somewhere else.\n",
      "P54577 already found somewhere else.\n",
      "P49588 already found somewhere else.\n",
      "Q5JTZ9 already found somewhere else.\n",
      "Q5T160 already found somewhere else.\n",
      "Q96I59 already found somewhere else.\n",
      "O43776 already found somewhere else.\n",
      "A0A3B3IT89 already found somewhere else.\n",
      "P14868 already found somewhere else.\n",
      "Q9HA77 already found somewhere else.\n",
      "P47897 already found somewhere else.\n",
      "Q5JPH6 already found somewhere else.\n",
      "P07814 already found somewhere else.\n",
      "A0A6Q8PGW4 already found somewhere else.\n",
      "B4E1C5 already found somewhere else.\n",
      "B4DQ67 already found somewhere else.\n",
      "Q9NSE4 already found somewhere else.\n",
      "P41252 already found somewhere else.\n",
      "Q9P2J5 already found somewhere else.\n",
      "Q15031 already found somewhere else.\n",
      "Q15046 already found somewhere else.\n",
      "Q96GW9 already found somewhere else.\n",
      "P56192 already found somewhere else.\n",
      "O95363 already found somewhere else.\n",
      "Q7L3T8 already found somewhere else.\n",
      "Q9NP81 already found somewhere else.\n",
      "Q9BW92 already found somewhere else.\n",
      "P26639 already found somewhere else.\n",
      "A0A024R6K8 already found somewhere else.\n",
      "P26640 already found somewhere else.\n",
      "A2ABL6 already found somewhere else.\n",
      "Q16842 already found somewhere else.\n",
      "Q11201 already found somewhere else.\n",
      "Q6ZMB0 already found somewhere else.\n",
      "O95395 already found somewhere else.\n",
      "Q9P109 already found somewhere else.\n",
      "Q02742 already found somewhere else.\n",
      "O43286 already found somewhere else.\n",
      "Q8N3T1 already found somewhere else.\n",
      "Q9NY28 already found somewhere else.\n",
      "Q10472 already found somewhere else.\n",
      "Q8NCW6 already found somewhere else.\n",
      "Q10471 already found somewhere else.\n",
      "Q58A54 already found somewhere else.\n",
      "Q14435 already found somewhere else.\n",
      "Q7Z7M9 already found somewhere else.\n",
      "Q96EU7 already found somewhere else.\n",
      "Q9NS00 already found somewhere else.\n",
      "A0A024R3R2 already found somewhere else.\n",
      "P14324 already found somewhere else.\n",
      "Q86YN1 already found somewhere else.\n",
      "Q5BKT4 already found somewhere else.\n",
      "Q9UPQ8 already found somewhere else.\n",
      "Q9H3H5 already found somewhere else.\n",
      "Q96F25 already found somewhere else.\n",
      "Q9BT22 already found somewhere else.\n",
      "Q9H553 already found somewhere else.\n",
      "Q2TAA5 already found somewhere else.\n",
      "Q9P2X0 already found somewhere else.\n",
      "O60762 already found somewhere else.\n",
      "Q92685 already found somewhere else.\n",
      "Q9H6U8 already found somewhere else.\n",
      "Q9BV10 already found somewhere else.\n",
      "Q9Y672 already found somewhere else.\n",
      "Q9BVK2 already found somewhere else.\n",
      "Q5I7T1 already found somewhere else.\n",
      "P61803 already found somewhere else.\n",
      "P04843 already found somewhere else.\n",
      "P39656 already found somewhere else.\n",
      "D6RDV0 already found somewhere else.\n",
      "Q8TCJ2 already found somewhere else.\n",
      "P46977 already found somewhere else.\n",
      "P04844 already found somewhere else.\n",
      "Q13724 already found somewhere else.\n",
      "Q14697 already found somewhere else.\n",
      "Q9UKM7 already found somewhere else.\n",
      "Q9NR34 already found somewhere else.\n",
      "O60476 already found somewhere else.\n",
      "P33908 already found somewhere else.\n",
      "P26572 already found somewhere else.\n",
      "A0A0C4DGL1 already found somewhere else.\n",
      "Q16706 already found somewhere else.\n",
      "Q10469 already found somewhere else.\n",
      "Q9UBM8 already found somewhere else.\n",
      "Q9UM21 already found somewhere else.\n",
      "Q9UQ53 already found somewhere else.\n",
      "Q09328 already found somewhere else.\n",
      "Q9BYC5 already found somewhere else.\n",
      "P15907 already found somewhere else.\n",
      "Q96JF0 already found somewhere else.\n",
      "Q9NTJ4 already found somewhere else.\n",
      "O00754 already found somewhere else.\n",
      "O00462 already found somewhere else.\n",
      "P20933 already found somewhere else.\n",
      "Q8NFI3 already found somewhere else.\n",
      "E9PMZ3 already found somewhere else.\n",
      "Q8WWR8 already found somewhere else.\n",
      "X6R8A1 already found somewhere else.\n",
      "Q99519 already found somewhere else.\n",
      "Q9Y3R4 already found somewhere else.\n",
      "P34059 already found somewhere else.\n",
      "P04066 already found somewhere else.\n",
      "Q9BTY2 already found somewhere else.\n",
      "P30530 already found somewhere else.\n",
      "P06213 already found somewhere else.\n",
      "P43405 already found somewhere else.\n",
      "P08069 already found somewhere else.\n",
      "P07333 already found somewhere else.\n",
      "P11362 already found somewhere else.\n",
      "Q6NVW1 already found somewhere else.\n",
      "P06241 already found somewhere else.\n",
      "Q16620 already found somewhere else.\n",
      "P34925 already found somewhere else.\n",
      "Q04912 already found somewhere else.\n",
      "P07332 already found somewhere else.\n",
      "P36888 already found somewhere else.\n",
      "E7ERN0 already found somewhere else.\n",
      "Q01973 already found somewhere else.\n",
      "P29597 already found somewhere else.\n",
      "P42679 already found somewhere else.\n",
      "P23458 already found somewhere else.\n",
      "P42685 already found somewhere else.\n",
      "P42680 already found somewhere else.\n",
      "P07947 already found somewhere else.\n",
      "X5D7M5 already found somewhere else.\n",
      "P21802 already found somewhere else.\n",
      "Q13882 already found somewhere else.\n",
      "P42684 already found somewhere else.\n",
      "P29322 already found somewhere else.\n",
      "Q13163 already found somewhere else.\n",
      "Q15303 already found somewhere else.\n",
      "P41240 already found somewhere else.\n",
      "P36507 already found somewhere else.\n",
      "E5RH01 already found somewhere else.\n",
      "Q96KB5 already found somewhere else.\n",
      "Q9HAZ1 already found somewhere else.\n",
      "Q92630 already found somewhere else.\n",
      "A1MA61 already found somewhere else.\n",
      "P14616 already found somewhere else.\n",
      "Q08881 already found somewhere else.\n",
      "P52564 already found somewhere else.\n",
      "O14733 already found somewhere else.\n",
      "P00519 already found somewhere else.\n",
      "P21709 already found somewhere else.\n",
      "P04626 already found somewhere else.\n",
      "A0A7P0T9L5 already found somewhere else.\n",
      "Q9NR20 already found somewhere else.\n",
      "P33981 already found somewhere else.\n",
      "P54753 already found somewhere else.\n",
      "Q07912 already found somewhere else.\n",
      "P16591 already found somewhere else.\n",
      "P09769 already found somewhere else.\n",
      "P22455 already found somewhere else.\n",
      "Q15569 already found somewhere else.\n",
      "A0A024R7T2 already found somewhere else.\n",
      "P29317 already found somewhere else.\n",
      "Q8NCJ5 already found somewhere else.\n",
      "P54760 already found somewhere else.\n",
      "P10721 already found somewhere else.\n",
      "A0A8I5QJL9 already found somewhere else.\n",
      "A0A024R728 already found somewhere else.\n",
      "H3BRE4 already found somewhere else.\n",
      "P00533 already found somewhere else.\n",
      "P51813 already found somewhere else.\n",
      "I6LM06 already found somewhere else.\n",
      "P29320 already found somewhere else.\n",
      "P21860 already found somewhere else.\n",
      "P51451 already found somewhere else.\n",
      "O15146 already found somewhere else.\n",
      "Q13470 already found somewhere else.\n",
      "P16234 already found somewhere else.\n",
      "O60674 already found somewhere else.\n",
      "Q12866 already found somewhere else.\n",
      "P04629 already found somewhere else.\n",
      "P09619 already found somewhere else.\n",
      "Q9H792 already found somewhere else.\n",
      "B4DXM2 already found somewhere else.\n",
      "Q8IWU2 already found somewhere else.\n",
      "P49759 already found somewhere else.\n",
      "P18031 already found somewhere else.\n",
      "P43378 already found somewhere else.\n",
      "Q8WUK0 already found somewhere else.\n",
      "P28827 already found somewhere else.\n",
      "P26045 already found somewhere else.\n",
      "Q68DX3 already found somewhere else.\n",
      "P54829 already found somewhere else.\n",
      "P10586 already found somewhere else.\n",
      "Q16825 already found somewhere else.\n",
      "Q15256 already found somewhere else.\n",
      "P23467 already found somewhere else.\n",
      "P30304 already found somewhere else.\n",
      "Q12974 already found somewhere else.\n",
      "Q15678 already found somewhere else.\n",
      "A0A0B4J2F5 already found somewhere else.\n",
      "O00167 already found somewhere else.\n",
      "Q68J44 already found somewhere else.\n",
      "Q16828 already found somewhere else.\n",
      "Q8NEJ0 already found somewhere else.\n",
      "P18433 already found somewhere else.\n",
      "P23468 already found somewhere else.\n",
      "P24666 already found somewhere else.\n",
      "P28562 already found somewhere else.\n",
      "A2A3K4 already found somewhere else.\n",
      "F5H5X4 already found somewhere else.\n",
      "P23469 already found somewhere else.\n",
      "Q9HD43 already found somewhere else.\n",
      "Q92729 already found somewhere else.\n",
      "P17706 already found somewhere else.\n",
      "Q12923 already found somewhere else.\n",
      "Q9UNH5 already found somewhere else.\n",
      "Q6B8I1 already found somewhere else.\n",
      "Q99952 already found somewhere else.\n",
      "Q13615 already found somewhere else.\n",
      "E9PMT0 already found somewhere else.\n",
      "O95677 already found somewhere else.\n",
      "Q12913 already found somewhere else.\n",
      "Q16667 already found somewhere else.\n",
      "Q9BVJ7 already found somewhere else.\n",
      "Q05209 already found somewhere else.\n",
      "Q76I76 already found somewhere else.\n",
      "Q8WTR2 already found somewhere else.\n",
      "Q8WYL5 already found somewhere else.\n",
      "S4R378 already found somewhere else.\n",
      "Q9H3S7 already found somewhere else.\n",
      "P51452 already found somewhere else.\n",
      "Q99956 already found somewhere else.\n",
      "P53634 already found somewhere else.\n",
      "O60911 already found somewhere else.\n",
      "Q9UBR2 already found somewhere else.\n",
      "A0A7I2YQA2 already found somewhere else.\n",
      "P43235 already found somewhere else.\n",
      "P14091 already found somewhere else.\n",
      "P56202 already found somewhere else.\n",
      "O14773 already found somewhere else.\n",
      "A0A7I2YQE6 already found somewhere else.\n",
      "Q9UBX1 already found somewhere else.\n",
      "P07339 already found somewhere else.\n",
      "E9PSG5 already found somewhere else.\n",
      "P43234 already found somewhere else.\n",
      "Q6Y1H2 already found somewhere else.\n",
      "B0YJ81 already found somewhere else.\n",
      "Q8N1Q1 already found somewhere else.\n",
      "P35219 already found somewhere else.\n",
      "P00918 already found somewhere else.\n",
      "P22748 already found somewhere else.\n",
      "V9HWE3 already found somewhere else.\n",
      "O43570 already found somewhere else.\n",
      "P43166 already found somewhere else.\n",
      "Q13564 already found somewhere else.\n",
      "Q16790 already found somewhere else.\n",
      "P23280 already found somewhere else.\n",
      "A8K3J4 already found somewhere else.\n",
      "P07451 already found somewhere else.\n",
      "O95861 already found somewhere else.\n",
      "Q9NX62 already found somewhere else.\n",
      "Q9P2R7 already found somewhere else.\n",
      "Q96I99 already found somewhere else.\n",
      "P53597 already found somewhere else.\n",
      "P48735 already found somewhere else.\n",
      "O75874 already found somewhere else.\n",
      "P55809 already found somewhere else.\n",
      "Q9BYC2 already found somewhere else.\n",
      "Q68D72 already found somewhere else.\n",
      "A0A087WZN1 already found somewhere else.\n",
      "G5E9Q7 already found somewhere else.\n",
      "A0A024RB75 already found somewhere else.\n",
      "Q14678 already found somewhere else.\n",
      "P53396 already found somewhere else.\n",
      "P48200 already found somewhere else.\n",
      "P21399 already found somewhere else.\n",
      "Q99798 already found somewhere else.\n",
      "Q99643 already found somewhere else.\n",
      "P21912 already found somewhere else.\n",
      "P31040 already found somewhere else.\n",
      "G3V173 already found somewhere else.\n",
      "Q8N0X4 already found somewhere else.\n",
      "Q9H008 already found somewhere else.\n",
      "Q9H2U2 already found somewhere else.\n",
      "Q15181 already found somewhere else.\n",
      "P38117 already found somewhere else.\n",
      "A0A8I5KUF3 already found somewhere else.\n",
      "A0A804HI48 already found somewhere else.\n",
      "Q02221 already found somewhere else.\n",
      "Q14061 already found somewhere else.\n",
      "P13073 already found somewhere else.\n",
      "P09669 already found somewhere else.\n",
      "P10606 already found somewhere else.\n",
      "Q7GIM7 already found somewhere else.\n",
      "A0A1X7RBF7 already found somewhere else.\n",
      "P14406 already found somewhere else.\n",
      "P24311 already found somewhere else.\n",
      "P24310 already found somewhere else.\n",
      "Q7KZN9 already found somewhere else.\n",
      "Q8TF08 already found somewhere else.\n",
      "Q53XN1 already found somewhere else.\n",
      "Q96KJ9 already found somewhere else.\n",
      "P14854 already found somewhere else.\n",
      "P15954 already found somewhere else.\n",
      "P20674 already found somewhere else.\n",
      "P00395 already found somewhere else.\n",
      "Q7Z4L0 already found somewhere else.\n",
      "O14548 already found somewhere else.\n",
      "Q6YFQ2 already found somewhere else.\n",
      "P12074 already found somewhere else.\n",
      "Q9UI12 already found somewhere else.\n",
      "P30049 already found somewhere else.\n",
      "O75964 already found somewhere else.\n",
      "A4D1K0 already found somewhere else.\n",
      "Q9P241 already found somewhere else.\n",
      "O75449 already found somewhere else.\n",
      "Q9NVP1 already found somewhere else.\n",
      "Q9Y230 already found somewhere else.\n",
      "P48047 already found somewhere else.\n",
      "E9PBH4 already found somewhere else.\n",
      "P13569 already found somewhere else.\n",
      "Q96A05 already found somewhere else.\n",
      "P05026 already found somewhere else.\n",
      "Q99437 already found somewhere else.\n",
      "Q9BUQ8 already found somewhere else.\n",
      "Q9BYX4 already found somewhere else.\n",
      "Q8N5M1 already found somewhere else.\n",
      "O14983 already found somewhere else.\n",
      "F2Z2H1 already found somewhere else.\n",
      "B7Z641 already found somewhere else.\n",
      "Q9Y4B4 already found somewhere else.\n",
      "Q9Y6V7 already found somewhere else.\n",
      "F6S2B7 already found somewhere else.\n",
      "Q8IVL1 already found somewhere else.\n",
      "Q9BX63 already found somewhere else.\n",
      "Q06055 already found somewhere else.\n",
      "P98194 already found somewhere else.\n",
      "P26196 already found somewhere else.\n",
      "O60231 already found somewhere else.\n",
      "Q9NZC9 already found somewhere else.\n",
      "Q9Y5K8 already found somewhere else.\n",
      "Q14839 already found somewhere else.\n",
      "O43520 already found somewhere else.\n",
      "I3L2G1 already found somewhere else.\n",
      "O60312 already found somewhere else.\n",
      "O95670 already found somewhere else.\n",
      "Q9NXL9 already found somewhere else.\n",
      "A0A024R968 already found somewhere else.\n",
      "P61421 already found somewhere else.\n",
      "P06576 already found somewhere else.\n",
      "Q7L2E3 already found somewhere else.\n",
      "P15313 already found somewhere else.\n",
      "P14415 already found somewhere else.\n",
      "P19447 already found somewhere else.\n",
      "Q15904 already found somewhere else.\n",
      "A0A024R791 already found somewhere else.\n",
      "P25705 already found somewhere else.\n",
      "A0A5F9ZGZ2 already found somewhere else.\n",
      "P56381 already found somewhere else.\n",
      "Q9UHI6 already found somewhere else.\n",
      "P36542 already found somewhere else.\n",
      "A0A024R2E4 already found somewhere else.\n",
      "B7ZKW0 already found somewhere else.\n",
      "Q8NB49 already found somewhere else.\n",
      "Q9NY93 already found somewhere else.\n",
      "P20648 already found somewhere else.\n",
      "Q8NEY4 already found somewhere else.\n",
      "B7Z6D5 already found somewhere else.\n",
      "Q9GZR7 already found somewhere else.\n",
      "Q5TC12 already found somewhere else.\n",
      "Q9BW62 already found somewhere else.\n",
      "P13984 already found somewhere else.\n",
      "P24539 already found somewhere else.\n",
      "P51530 already found somewhere else.\n",
      "P46100 already found somewhere else.\n",
      "Q8TDI0 already found somewhere else.\n",
      "Q9NXZ2 already found somewhere else.\n",
      "Q92499 already found somewhere else.\n",
      "Q92620 already found somewhere else.\n",
      "P56134 already found somewhere else.\n",
      "P18859 already found somewhere else.\n",
      "A0A024RBC7 already found somewhere else.\n",
      "P36543 already found somewhere else.\n",
      "A0A0G2JQU7 already found somewhere else.\n",
      "A0A024R991 already found somewhere else.\n",
      "P17844 already found somewhere else.\n",
      "P21283 already found somewhere else.\n",
      "Q8IYT4 already found somewhere else.\n",
      "O00148 already found somewhere else.\n",
      "P38919 already found somewhere else.\n",
      "P50993 already found somewhere else.\n",
      "Q8IY37 already found somewhere else.\n",
      "Q9H0S4 already found somewhere else.\n",
      "P27449 already found somewhere else.\n",
      "P56385 already found somewhere else.\n",
      "P05496 already found somewhere else.\n",
      "Q3L8U1 already found somewhere else.\n",
      "Q7Z4Y8 already found somewhere else.\n",
      "O15342 already found somewhere else.\n",
      "E5KRP5 already found somewhere else.\n",
      "P42285 already found somewhere else.\n",
      "Q93084 already found somewhere else.\n",
      "P21281 already found somewhere else.\n",
      "Q8NFZ0 already found somewhere else.\n",
      "Q8TE96 already found somewhere else.\n",
      "Q7L014 already found somewhere else.\n",
      "P05023 already found somewhere else.\n",
      "O75348 already found somewhere else.\n",
      "P00846 already found somewhere else.\n",
      "P33991 already found somewhere else.\n",
      "Q86XP3 already found somewhere else.\n",
      "Q08211 already found somewhere else.\n",
      "Q8NCM8 already found somewhere else.\n",
      "Q8TBM3 already found somewhere else.\n",
      "A0A024R8M9 already found somewhere else.\n",
      "P48201 already found somewhere else.\n",
      "Q9H611 already found somewhere else.\n",
      "Q13488 already found somewhere else.\n",
      "O75947 already found somewhere else.\n",
      "P51164 already found somewhere else.\n",
      "Q8TF62 already found somewhere else.\n",
      "Q9Y2Q0 already found somewhere else.\n",
      "P13637 already found somewhere else.\n",
      "P38606 already found somewhere else.\n",
      "P46063 already found somewhere else.\n",
      "Q13733 already found somewhere else.\n",
      "O14646 already found somewhere else.\n",
      "P54709 already found somewhere else.\n",
      "Q96LB4 already found somewhere else.\n",
      "P07919 already found somewhere else.\n",
      "P08574 already found somewhere else.\n",
      "P31930 already found somewhere else.\n",
      "Q9UDW1 already found somewhere else.\n",
      "P22695 already found somewhere else.\n",
      "O14949 already found somewhere else.\n",
      "O14957 already found somewhere else.\n",
      "P47985 already found somewhere else.\n",
      "P14927 already found somewhere else.\n",
      "P00156 already found somewhere else.\n",
      "E9PIZ8 already found somewhere else.\n",
      "O95299 already found somewhere else.\n",
      "O43678 already found somewhere else.\n",
      "P56181 already found somewhere else.\n",
      "B4DJ81 already found somewhere else.\n",
      "P03905 already found somewhere else.\n",
      "Q9NRX3 already found somewhere else.\n",
      "P19404 already found somewhere else.\n",
      "O75438 already found somewhere else.\n",
      "O00217 already found somewhere else.\n",
      "Q9P0J0 already found somewhere else.\n",
      "O75251 already found somewhere else.\n",
      "O75306 already found somewhere else.\n",
      "O15239 already found somewhere else.\n",
      "P51970 already found somewhere else.\n",
      "A0A087WYD0 already found somewhere else.\n",
      "P03897 already found somewhere else.\n",
      "R4GN43 already found somewhere else.\n",
      "Q86Y39 already found somewhere else.\n",
      "P03923 already found somewhere else.\n",
      "O43920 already found somewhere else.\n",
      "O95139 already found somewhere else.\n",
      "O43676 already found somewhere else.\n",
      "O14561 already found somewhere else.\n",
      "Q9Y6M9 already found somewhere else.\n",
      "A8K761 already found somewhere else.\n",
      "Q16795 already found somewhere else.\n",
      "P03915 already found somewhere else.\n",
      "O75489 already found somewhere else.\n",
      "P03891 already found somewhere else.\n",
      "O95168 already found somewhere else.\n",
      "P03886 already found somewhere else.\n",
      "O75380 already found somewhere else.\n",
      "O43677 already found somewhere else.\n",
      "P49821 already found somewhere else.\n",
      "Q16718 already found somewhere else.\n",
      "Q9NX14 already found somewhere else.\n",
      "O95167 already found somewhere else.\n",
      "O00483 already found somewhere else.\n",
      "O95298 already found somewhere else.\n",
      "O95178 already found somewhere else.\n",
      "A0A0S2Z433 already found somewhere else.\n",
      "P03901 already found somewhere else.\n",
      "P17568 already found somewhere else.\n",
      "Q9UI09 already found somewhere else.\n",
      "P00441 already found somewhere else.\n",
      "P08294 already found somewhere else.\n",
      "Q08AH1 already found somewhere else.\n",
      "Q6NUN0 already found somewhere else.\n",
      "P0C7M7 already found somewhere else.\n",
      "Q53FZ2 already found somewhere else.\n",
      "Q08AH3 already found somewhere else.\n",
      "O60488 already found somewhere else.\n",
      "O14975 already found somewhere else.\n",
      "Q5FVE4 already found somewhere else.\n",
      "Q9ULC5 already found somewhere else.\n",
      "O95573 already found somewhere else.\n",
      "P33121 already found somewhere else.\n",
      "Q9UKU0 already found somewhere else.\n",
      "P49753 already found somewhere else.\n",
      "Q8N9L9 already found somewhere else.\n",
      "Q9NWU1 already found somewhere else.\n",
      "P49327 already found somewhere else.\n",
      "O00763 already found somewhere else.\n",
      "Q53GQ0 already found somewhere else.\n",
      "Q9P035 already found somewhere else.\n",
      "Q9BW72 already found somewhere else.\n",
      "Q9NZ01 already found somewhere else.\n",
      "Q86SK9 already found somewhere else.\n",
      "O00767 already found somewhere else.\n",
      "A0A087WWN9 already found somewhere else.\n",
      "O95864 already found somewhere else.\n",
      "A0A0A0MR51 already found somewhere else.\n",
      "Q9Y5Q0 already found somewhere else.\n",
      "A0A0G2JHB6 already found somewhere else.\n",
      "Q14032 already found somewhere else.\n",
      "Q86TX2 already found somewhere else.\n",
      "O00154 already found somewhere else.\n",
      "P50897 already found somewhere else.\n",
      "Q8IVS2 already found somewhere else.\n",
      "Q9NV23 already found somewhere else.\n",
      "P16050 already found somewhere else.\n",
      "P33260 already found somewhere else.\n",
      "Q9HBI6 already found somewhere else.\n",
      "P20813 already found somewhere else.\n",
      "Q9HB55 already found somewhere else.\n",
      "P11511 already found somewhere else.\n",
      "P11509 already found somewhere else.\n",
      "Q86W10 already found somewhere else.\n",
      "P20853 already found somewhere else.\n",
      "P10632 already found somewhere else.\n",
      "P24903 already found somewhere else.\n",
      "Q9HCS2 already found somewhere else.\n",
      "P24462 already found somewhere else.\n",
      "P05181 already found somewhere else.\n",
      "P08684 already found somewhere else.\n",
      "P20815 already found somewhere else.\n",
      "Q16696 already found somewhere else.\n",
      "S5RV20 already found somewhere else.\n",
      "Q6NWU0 already found somewhere else.\n",
      "P07099 already found somewhere else.\n",
      "P34913 already found somewhere else.\n",
      "Q9Y234 already found somewhere else.\n",
      "A6NK58 already found somewhere else.\n",
      "O43766 already found somewhere else.\n",
      "Q15067 already found somewhere else.\n",
      "Q08426 already found somewhere else.\n",
      "A0A2R8Y674 already found somewhere else.\n",
      "Q9ULV0 already found somewhere else.\n",
      "P09110 already found somewhere else.\n",
      "P09917 already found somewhere else.\n",
      "P16152 already found somewhere else.\n",
      "O75828 already found somewhere else.\n",
      "Q9BTZ2 already found somewhere else.\n",
      "P09960 already found somewhere else.\n",
      "P78329 already found somewhere else.\n",
      "P35354 already found somewhere else.\n",
      "P23219 already found somewhere else.\n",
      "Q99426 already found somewhere else.\n",
      "O14684 already found somewhere else.\n",
      "Q9H7Z7 already found somewhere else.\n",
      "P28330 already found somewhere else.\n",
      "Q16698 already found somewhere else.\n",
      "A0A0J9YY83 already found somewhere else.\n",
      "O75521 already found somewhere else.\n",
      "Q13011 already found somewhere else.\n",
      "P42126 already found somewhere else.\n",
      "P18054 already found somewhere else.\n",
      "B7Z225 already found somewhere else.\n",
      "Q96CM8 already found somewhere else.\n",
      "O14734 already found somewhere else.\n",
      "Q4G176 already found somewhere else.\n",
      "Q6P1M0 already found somewhere else.\n",
      "Q4L235 already found somewhere else.\n",
      "Q9UHK6 already found somewhere else.\n",
      "Q5K4L6 already found somewhere else.\n",
      "Q14914 already found somewhere else.\n",
      "Q16873 already found somewhere else.\n",
      "Q9H4A9 already found somewhere else.\n",
      "Q5FWF5 already found somewhere else.\n",
      "Q8IUH5 already found somewhere else.\n",
      "Q9UHE5 already found somewhere else.\n",
      "Q969W1 already found somewhere else.\n",
      "Q9NST1 already found somewhere else.\n",
      "Q7L5N7 already found somewhere else.\n",
      "Q8IZN3 already found somewhere else.\n",
      "C9J4E9 already found somewhere else.\n",
      "Q9Y397 already found somewhere else.\n",
      "Q8N8M0 already found somewhere else.\n",
      "Q6ZPD8 already found somewhere else.\n",
      "Q8IUH4 already found somewhere else.\n",
      "Q5W0Z9 already found somewhere else.\n",
      "Q93015 already found somewhere else.\n",
      "Q6UX98 already found somewhere else.\n",
      "Q96MV8 already found somewhere else.\n",
      "A0A075B6R5 already found somewhere else.\n",
      "Q9H7X0 already found somewhere else.\n",
      "Q9NYG2 already found somewhere else.\n",
      "Q92604 already found somewhere else.\n",
      "Q9C0B5 already found somewhere else.\n",
      "Q8NCC3 already found somewhere else.\n",
      "P41227 already found somewhere else.\n",
      "Q56NI9 already found somewhere else.\n",
      "Q86UY6 already found somewhere else.\n",
      "Q6ZWT7 already found somewhere else.\n",
      "Q9Y371 already found somewhere else.\n",
      "Q6P1A2 already found somewhere else.\n",
      "Q8WTX9 already found somewhere else.\n",
      "Q6ZNC8 already found somewhere else.\n",
      "Q8NF37 already found somewhere else.\n",
      "Q9NPG8 already found somewhere else.\n",
      "Q643R3 already found somewhere else.\n",
      "Q6UWP7 already found somewhere else.\n",
      "B3KXV3 already found somewhere else.\n",
      "Q9ULC8 already found somewhere else.\n",
      "O15498 already found somewhere else.\n",
      "Q92523 already found somewhere else.\n",
      "P50416 already found somewhere else.\n",
      "Q8TCG5 already found somewhere else.\n",
      "O43772 already found somewhere else.\n",
      "P43155 already found somewhere else.\n",
      "P23786 already found somewhere else.\n",
      "Q9UKG9 already found somewhere else.\n",
      "Q8N8R3 already found somewhere else.\n",
      "Q8IVM8 already found somewhere else.\n",
      "O15245 already found somewhere else.\n",
      "Q9H015 already found somewhere else.\n",
      "O76082 already found somewhere else.\n",
      "H7C0L1 already found somewhere else.\n",
      "F8W1M3 already found somewhere else.\n",
      "Q9Y6T7 already found somewhere else.\n",
      "Q13574 already found somewhere else.\n",
      "Q86XP1 already found somewhere else.\n",
      "P52824 already found somewhere else.\n",
      "P52429 already found somewhere else.\n",
      "P49619 already found somewhere else.\n",
      "O75912 already found somewhere else.\n",
      "O75907 already found somewhere else.\n",
      "Q96PD7 already found somewhere else.\n",
      "Q96PD6 already found somewhere else.\n",
      "Q3SYC2 already found somewhere else.\n",
      "Q86VF5 already found somewhere else.\n",
      "P22760 already found somewhere else.\n",
      "P16233 already found somewhere else.\n",
      "Q9Y5X9 already found somewhere else.\n",
      "P07098 already found somewhere else.\n",
      "Q05469 already found somewhere else.\n",
      "Q17RR3 already found somewhere else.\n",
      "P19835 already found somewhere else.\n",
      "P54315 already found somewhere else.\n",
      "P11150 already found somewhere else.\n",
      "P41247 already found somewhere else.\n",
      "Q99685 already found somewhere else.\n",
      "Q9BV23 already found somewhere else.\n",
      "O15228 already found somewhere else.\n",
      "O14832 already found somewhere else.\n",
      "B3KPX4 already found somewhere else.\n",
      "Q86YH6 already found somewhere else.\n",
      "Q5T2R2 already found somewhere else.\n",
      "A0A1D8H0A6 already found somewhere else.\n",
      "Q9Y2Z9 already found somewhere else.\n",
      "Q9NZJ6 already found somewhere else.\n",
      "Q5HYK3 already found somewhere else.\n",
      "Q99807 already found somewhere else.\n",
      "B4DL54 already found somewhere else.\n",
      "P49354 already found somewhere else.\n",
      "O60725 already found somewhere else.\n",
      "Q9UHG3 already found somewhere else.\n",
      "Q9Y3M8 already found somewhere else.\n",
      "P59095 already found somewhere else.\n",
      "Q9NQZ5 already found somewhere else.\n",
      "Q9NSY2 already found somewhere else.\n",
      "Q9P2P6 already found somewhere else.\n",
      "Q96DR4 already found somewhere else.\n",
      "Q6ZWL3 already found somewhere else.\n",
      "P08686 already found somewhere else.\n",
      "P05093 already found somewhere else.\n",
      "P42330 already found somewhere else.\n",
      "P52895 already found somewhere else.\n",
      "P26439 already found somewhere else.\n",
      "P14060 already found somewhere else.\n",
      "P05108 already found somewhere else.\n",
      "P37059 already found somewhere else.\n",
      "O14756 already found somewhere else.\n",
      "P56937 already found somewhere else.\n",
      "P14061 already found somewhere else.\n",
      "Q06520 already found somewhere else.\n",
      "O00204 already found somewhere else.\n",
      "Q9H8P0 already found somewhere else.\n",
      "P51857 already found somewhere else.\n",
      "P04035 already found somewhere else.\n",
      "F5H163 already found somewhere else.\n",
      "Q15126 already found somewhere else.\n",
      "Q13907 already found somewhere else.\n",
      "Q9BXS1 already found somewhere else.\n",
      "Q6IAX1 already found somewhere else.\n",
      "Q14534 already found somewhere else.\n",
      "P48449 already found somewhere else.\n",
      "Q16850 already found somewhere else.\n",
      "A0A024R577 already found somewhere else.\n",
      "Q96IV6 already found somewhere else.\n",
      "Q15800 already found somewhere else.\n",
      "Q15738 already found somewhere else.\n",
      "A0A024QYX0 already found somewhere else.\n",
      "O75845 already found somewhere else.\n",
      "B4E1K5 already found somewhere else.\n",
      "P53602 already found somewhere else.\n",
      "Q02318 already found somewhere else.\n",
      "B7Z212 already found somewhere else.\n",
      "P51689 already found somewhere else.\n",
      "P38571 already found somewhere else.\n",
      "P35610 already found somewhere else.\n",
      "O75908 already found somewhere else.\n",
      "Q6QHC5 already found somewhere else.\n",
      "Q13510 already found somewhere else.\n",
      "Q5QJU3 already found somewhere else.\n",
      "Q8TDN7 already found somewhere else.\n",
      "Q8NHU3 already found somewhere else.\n",
      "Q86VZ5 already found somewhere else.\n",
      "A0A8I5QJP4 already found somewhere else.\n",
      "O15270 already found somewhere else.\n",
      "Q06136 already found somewhere else.\n",
      "Q9NYA1 already found somewhere else.\n",
      "Q9NRA0 already found somewhere else.\n",
      "O43688 already found somewhere else.\n",
      "O14494 already found somewhere else.\n",
      "Q14693 already found somewhere else.\n",
      "Q92539 already found somewhere else.\n",
      "O14495 already found somewhere else.\n",
      "O95470 already found somewhere else.\n",
      "Q9HA82 already found somewhere else.\n",
      "Q8IU89 already found somewhere else.\n",
      "Q6ZMG9 already found somewhere else.\n",
      "P27544 already found somewhere else.\n",
      "Q96G23 already found somewhere else.\n",
      "Q8N5B7 already found somewhere else.\n",
      "O15121 already found somewhere else.\n",
      "Q8TCT0 already found somewhere else.\n",
      "Q16739 already found somewhere else.\n",
      "P04062 already found somewhere else.\n",
      "Q9HCG7 already found somewhere else.\n",
      "Q9UNA3 already found somewhere else.\n",
      "O60513 already found somewhere else.\n",
      "Q9Y2A9 already found somewhere else.\n",
      "Q6UX72 already found somewhere else.\n",
      "Q76KP1 already found somewhere else.\n",
      "Q9Y5Z6 already found somewhere else.\n",
      "Q8N5D6 already found somewhere else.\n",
      "Q9Y231 already found somewhere else.\n",
      "Q9H1C3 already found somewhere else.\n",
      "Q6L9W6 already found somewhere else.\n",
      "Q68CQ7 already found somewhere else.\n",
      "Q8NHY0 already found somewhere else.\n",
      "A0A087WY64 already found somewhere else.\n",
      "A0A6I8PU65 already found somewhere else.\n",
      "Q495W5 already found somewhere else.\n",
      "Q9BYG0 already found somewhere else.\n",
      "Q92521 already found somewhere else.\n",
      "O15294 already found somewhere else.\n",
      "I3L232 already found somewhere else.\n",
      "A0A8I5KR25 already found somewhere else.\n",
      "Q86VD9 already found somewhere else.\n",
      "Q9UBV7 already found somewhere else.\n",
      "Q9UBX8 already found somewhere else.\n",
      "Q7Z4J2 already found somewhere else.\n",
      "Q9NYU2 already found somewhere else.\n",
      "Q9Y2C3 already found somewhere else.\n",
      "A6NG13 already found somewhere else.\n",
      "Q9NY97 already found somewhere else.\n",
      "A0A024R7E2 already found somewhere else.\n",
      "Q8NEB5 already found somewhere else.\n",
      "Q5VZY2 already found somewhere else.\n",
      "O60906 already found somewhere else.\n",
      "Q9NY59 already found somewhere else.\n",
      "P17405 already found somewhere else.\n",
      "Q9NPC4 already found somewhere else.\n",
      "O75752 already found somewhere else.\n",
      "Q00973 already found somewhere else.\n",
      "O96024 already found somewhere else.\n",
      "Q969X2 already found somewhere else.\n",
      "Q9UJ37 already found somewhere else.\n",
      "Q92185 already found somewhere else.\n",
      "O15466 already found somewhere else.\n",
      "Q9UNP4 already found somewhere else.\n",
      "P17050 already found somewhere else.\n",
      "Q99999 already found somewhere else.\n",
      "A0A0C4DFZ2 already found somewhere else.\n",
      "O00400 already found somewhere else.\n",
      "Q9BX95 already found somewhere else.\n",
      "Q5VW52 already found somewhere else.\n",
      "A0A024RDG5 already found somewhere else.\n",
      "Q86UL3 already found somewhere else.\n",
      "Q9NUQ2 already found somewhere else.\n",
      "Q9NRZ5 already found somewhere else.\n",
      "O15120 already found somewhere else.\n",
      "Q9NRZ7 already found somewhere else.\n",
      "Q99943 already found somewhere else.\n",
      "Q53H12 already found somewhere else.\n",
      "Q8TBG4 already found somewhere else.\n",
      "P09923 already found somewhere else.\n",
      "P05187 already found somewhere else.\n",
      "P10696 already found somewhere else.\n",
      "A0A024RAB4 already found somewhere else.\n",
      "Q3MJ16 already found somewhere else.\n",
      "P14555 already found somewhere else.\n",
      "Q68DD2 already found somewhere else.\n",
      "Q9UP65 already found somewhere else.\n",
      "Q9BZM1 already found somewhere else.\n",
      "P39877 already found somewhere else.\n",
      "Q9BS22 already found somewhere else.\n",
      "Q86XP0 already found somewhere else.\n",
      "Q9UNK4 already found somewhere else.\n",
      "P47712 already found somewhere else.\n",
      "P53816 already found somewhere else.\n",
      "O15496 already found somewhere else.\n",
      "Q9NZK7 already found somewhere else.\n",
      "A0A590UJZ2 already found somewhere else.\n",
      "Q9NZ20 already found somewhere else.\n",
      "Q6P1J6 already found somewhere else.\n",
      "Q9Y6K0 already found somewhere else.\n",
      "Q6ZS86 already found somewhere else.\n",
      "Q14409 already found somewhere else.\n",
      "Q14410 already found somewhere else.\n",
      "P32189 already found somewhere else.\n",
      "Q8WUD6 already found somewhere else.\n",
      "A0A024R138 already found somewhere else.\n",
      "Q8N335 already found somewhere else.\n",
      "O95674 already found somewhere else.\n",
      "Q92903 already found somewhere else.\n",
      "Q32NB8 already found somewhere else.\n",
      "B3KY94 already found somewhere else.\n",
      "A0A024R1K5 already found somewhere else.\n",
      "Q9BVG9 already found somewhere else.\n",
      "P48651 already found somewhere else.\n",
      "Q8N2A8 already found somewhere else.\n",
      "O14939 already found somewhere else.\n",
      "Q13393 already found somewhere else.\n",
      "A0A024R0Q4 already found somewhere else.\n",
      "Q6ZV29 already found somewhere else.\n",
      "Q05315 already found somewhere else.\n",
      "Q8IY17 already found somewhere else.\n",
      "A0A024R746 already found somewhere else.\n",
      "Q9UBD9 already found somewhere else.\n",
      "Q6IAQ1 already found somewhere else.\n",
      "Q9UHV8 already found somewhere else.\n",
      "P04180 already found somewhere else.\n",
      "Q9NPB8 already found somewhere else.\n",
      "P35790 already found somewhere else.\n",
      "Q9Y259 already found somewhere else.\n",
      "P49585 already found somewhere else.\n",
      "Q9Y5K3 already found somewhere else.\n",
      "Q9NVF9 already found somewhere else.\n",
      "Q9HBU6 already found somewhere else.\n",
      "Q99447 already found somewhere else.\n",
      "Q9UBM1 already found somewhere else.\n",
      "O00116 already found somewhere else.\n",
      "A0A024R0L6 already found somewhere else.\n",
      "Q99487 already found somewhere else.\n",
      "P68402 already found somewhere else.\n",
      "P43034 already found somewhere else.\n",
      "Q3LXA3 already found somewhere else.\n",
      "K7EQ20 already found somewhere else.\n",
      "K7ENC0 already found somewhere else.\n",
      "P51993 already found somewhere else.\n",
      "Q9Y274 already found somewhere else.\n",
      "O43505 already found somewhere else.\n",
      "Q10981 already found somewhere else.\n",
      "Q6IZA2 already found somewhere else.\n",
      "Q8N0V5 already found somewhere else.\n",
      "P17900 already found somewhere else.\n",
      "Q16880 already found somewhere else.\n",
      "A0A2K4ZA02 already found somewhere else.\n",
      "P57054 already found somewhere else.\n",
      "Q96I23 already found somewhere else.\n",
      "Q92535 already found somewhere else.\n",
      "Q9BRB3 already found somewhere else.\n",
      "Q9Y2B2 already found somewhere else.\n",
      "Q8TBF5 already found somewhere else.\n",
      "A0A1W2PNH8 already found somewhere else.\n",
      "Q5H8A4 already found somewhere else.\n",
      "Q07326 already found somewhere else.\n",
      "Q96S52 already found somewhere else.\n",
      "O43292 already found somewhere else.\n",
      "Q9H490 already found somewhere else.\n",
      "Q9BYV7 already found somewhere else.\n",
      "O95059 already found somewhere else.\n",
      "Q96GP6 already found somewhere else.\n",
      "O00519 already found somewhere else.\n",
      "Q8WVX9 already found somewhere else.\n",
      "Q96K12 already found somewhere else.\n",
      "Q86Y38 already found somewhere else.\n",
      "B4DT06 already found somewhere else.\n",
      "Q96L58 already found somewhere else.\n",
      "Q9P2W7 already found somewhere else.\n",
      "Q9NPZ5 already found somewhere else.\n",
      "Q8N6G5 already found somewhere else.\n",
      "Q8TDX6 already found somewhere else.\n",
      "Q9UBQ6 already found somewhere else.\n",
      "O43909 already found somewhere else.\n",
      "Q16394 already found somewhere else.\n",
      "Q92935 already found somewhere else.\n",
      "O94923 already found somewhere else.\n",
      "A0A024R860 already found somewhere else.\n",
      "Q9NPF2 already found somewhere else.\n",
      "Q8IZ52 already found somewhere else.\n",
      "Q9P2E5 already found somewhere else.\n",
      "Q7LGC8 already found somewhere else.\n",
      "Q7LFX5 already found somewhere else.\n",
      "P15848 already found somewhere else.\n",
      "O43820 already found somewhere else.\n",
      "Q2M3T9 already found somewhere else.\n",
      "Q12794 already found somewhere else.\n",
      "P08236 already found somewhere else.\n",
      "Q9UEF7 already found somewhere else.\n",
      "O60597 already found somewhere else.\n",
      "Q9Y251 already found somewhere else.\n",
      "Q8WWQ2 already found somewhere else.\n",
      "P15586 already found somewhere else.\n",
      "P51688 already found somewhere else.\n",
      "E5RJC4 already found somewhere else.\n",
      "P54802 already found somewhere else.\n",
      "P35475 already found somewhere else.\n",
      "Q11203 already found somewhere else.\n",
      "O43916 already found somewhere else.\n",
      "P22680 already found somewhere else.\n",
      "Q9UNU6 already found somewhere else.\n",
      "Q9Y2P5 already found somewhere else.\n",
      "P22307 already found somewhere else.\n",
      "P33897 already found somewhere else.\n",
      "Q76G19 already found somewhere else.\n",
      "Q6P2M8 already found somewhere else.\n",
      "P51572 already found somewhere else.\n",
      "P51571 already found somewhere else.\n",
      "P48029 already found somewhere else.\n",
      "Q99424 already found somewhere else.\n",
      "Q9Y6A2 already found somewhere else.\n",
      "Q9NYL5 already found somewhere else.\n",
      "O95992 already found somewhere else.\n",
      "Q16878 already found somewhere else.\n",
      "Q05329 already found somewhere else.\n",
      "A0A024RAX7 already found somewhere else.\n",
      "Q99259 already found somewhere else.\n",
      "Q96SZ5 already found somewhere else.\n",
      "O15438 already found somewhere else.\n",
      "O95342 already found somewhere else.\n",
      "A0A0A0MS99 already found somewhere else.\n",
      "Q92887 already found somewhere else.\n",
      "Q86UW2 already found somewhere else.\n",
      "Q86UW1 already found somewhere else.\n",
      "O15439 already found somewhere else.\n",
      "P82251 already found somewhere else.\n",
      "Q9NPD5 already found somewhere else.\n",
      "B4DJE6 already found somewhere else.\n",
      "Q96BD0 already found somewhere else.\n",
      "Q9Y6L6 already found somewhere else.\n",
      "W6JLF7 already found somewhere else.\n",
      "P46952 already found somewhere else.\n",
      "Q15274 already found somewhere else.\n",
      "Q6XQN6 already found somewhere else.\n",
      "Q9BQG2 already found somewhere else.\n",
      "A0A2R8YFG2 already found somewhere else.\n",
      "Q9BZQ4 already found somewhere else.\n",
      "Q9HAN9 already found somewhere else.\n",
      "Q6IA69 already found somewhere else.\n",
      "P43490 already found somewhere else.\n",
      "Q9NWW6 already found somewhere else.\n",
      "O95544 already found somewhere else.\n",
      "Q4G0N4 already found somewhere else.\n",
      "Q9BZG2 already found somewhere else.\n",
      "P13686 already found somewhere else.\n",
      "Q8TE99 already found somewhere else.\n",
      "Q9NPH0 already found somewhere else.\n",
      "Q13423 already found somewhere else.\n",
      "P52961 already found somewhere else.\n",
      "Q96L15 already found somewhere else.\n",
      "Q13057 already found somewhere else.\n",
      "H0Y9E4 already found somewhere else.\n",
      "Q9BZ23 already found somewhere else.\n",
      "A0A8C8KBT8 already found somewhere else.\n",
      "Q9H999 already found somewhere else.\n",
      "O95498 already found somewhere else.\n",
      "O95497 already found somewhere else.\n",
      "Q9HAB8 already found somewhere else.\n",
      "H3BU63 already found somewhere else.\n",
      "B7Z7D2 already found somewhere else.\n",
      "P16885 already found somewhere else.\n",
      "Q9UPR0 already found somewhere else.\n",
      "Q9BRC7 already found somewhere else.\n",
      "O75038 already found somewhere else.\n",
      "P19174 already found somewhere else.\n",
      "Q01970 already found somewhere else.\n",
      "A0A8I5KXD7 already found somewhere else.\n",
      "Q9BVT6 already found somewhere else.\n",
      "Q9P212 already found somewhere else.\n",
      "Q8N7S5 already found somewhere else.\n",
      "Q15111 already found somewhere else.\n",
      "Q0VAA5 already found somewhere else.\n",
      "Q4KWH8 already found somewhere else.\n",
      "Q9NQ66 already found somewhere else.\n",
      "Q8N3E9 already found somewhere else.\n",
      "P51178 already found somewhere else.\n",
      "Q9NPH2 already found somewhere else.\n",
      "Q9UGB7 already found somewhere else.\n",
      "O14732 already found somewhere else.\n",
      "P29218 already found somewhere else.\n",
      "Q8NEB9 already found somewhere else.\n",
      "Q13496 already found somewhere else.\n",
      "Q9C0I1 already found somewhere else.\n",
      "Q9Y2I7 already found somewhere else.\n",
      "F8VW52 already found somewhere else.\n",
      "A0A6Q8PGT1 already found somewhere else.\n",
      "Q32ZL2 already found somewhere else.\n",
      "Q8IY26 already found somewhere else.\n",
      "A0A6Q8PG49 already found somewhere else.\n",
      "Q9NTJ5 already found somewhere else.\n",
      "Q86UY8 already found somewhere else.\n",
      "Q9Y216 already found somewhere else.\n",
      "Q99570 already found somewhere else.\n",
      "Q9Y217 already found somewhere else.\n",
      "Q10713 already found somewhere else.\n",
      "Q9NRX4 already found somewhere else.\n",
      "A0A024RB48 already found somewhere else.\n",
      "Q8IWX5 already found somewhere else.\n",
      "Q13613 already found somewhere else.\n",
      "Q8IXW5 already found somewhere else.\n",
      "O15357 already found somewhere else.\n",
      "Q05D32 already found somewhere else.\n",
      "Q9Y2H2 already found somewhere else.\n",
      "Q86V88 already found somewhere else.\n",
      "P42356 already found somewhere else.\n",
      "Q8TCG2 already found somewhere else.\n",
      "Q9BTU6 already found somewhere else.\n",
      "Q9UBF8 already found somewhere else.\n",
      "Q8WYR1 already found somewhere else.\n",
      "O00750 already found somewhere else.\n",
      "O00443 already found somewhere else.\n",
      "F5H369 already found somewhere else.\n",
      "O15327 already found somewhere else.\n",
      "Q96PE3 already found somewhere else.\n",
      "P60484 already found somewhere else.\n",
      "O60331 already found somewhere else.\n",
      "Q99755 already found somewhere else.\n",
      "O14986 already found somewhere else.\n",
      "Q01968 already found somewhere else.\n",
      "O15056 already found somewhere else.\n",
      "O43426 already found somewhere else.\n",
      "Q9NRR6 already found somewhere else.\n",
      "P32019 already found somewhere else.\n",
      "Q86T03 already found somewhere else.\n",
      "Q8N4L2 already found somewhere else.\n",
      "P48426 already found somewhere else.\n",
      "Q8TBX8 already found somewhere else.\n",
      "O00329 already found somewhere else.\n",
      "P42338 already found somewhere else.\n",
      "A0A024R720 already found somewhere else.\n",
      "P42336 already found somewhere else.\n",
      "O00459 already found somewhere else.\n",
      "P27986 already found somewhere else.\n",
      "Q9UBK5 already found somewhere else.\n",
      "Q92569 already found somewhere else.\n",
      "Q14642 already found somewhere else.\n",
      "B5MBZ3 already found somewhere else.\n",
      "P49441 already found somewhere else.\n",
      "P23677 already found somewhere else.\n",
      "P27987 already found somewhere else.\n",
      "Q96DU7 already found somewhere else.\n",
      "Q9UNW1 already found somewhere else.\n",
      "Q92835 already found somewhere else.\n",
      "Q13572 already found somewhere else.\n",
      "Q8NFU5 already found somewhere else.\n",
      "Q9H8X2 already found somewhere else.\n",
      "Q6PFW1 already found somewhere else.\n",
      "B2RCP4 already found somewhere else.\n",
      "A0A087WZV0 already found somewhere else.\n",
      "P09417 already found somewhere else.\n",
      "B4DM58 already found somewhere else.\n",
      "O75891 already found somewhere else.\n",
      "Q3SY69 already found somewhere else.\n",
      "P11586 already found somewhere else.\n",
      "P13995 already found somewhere else.\n",
      "Q9H903 already found somewhere else.\n",
      "Q6UB35 already found somewhere else.\n",
      "E7EU94 already found somewhere else.\n",
      "P49914 already found somewhere else.\n",
      "Q9NZB8 already found somewhere else.\n",
      "O96033 already found somewhere else.\n",
      "G3V4R0 already found somewhere else.\n",
      "A0A2R8Y5J9 already found somewhere else.\n",
      "P50747 already found somewhere else.\n",
      "P30793 already found somewhere else.\n",
      "E9PKY8 already found somewhere else.\n",
      "P35270 already found somewhere else.\n",
      "Q9H0N5 already found somewhere else.\n",
      "P61457 already found somewhere else.\n",
      "Q9NS18 already found somewhere else.\n",
      "P35754 already found somewhere else.\n",
      "D6RE86 already found somewhere else.\n",
      "P02794 already found somewhere else.\n",
      "Q16595 already found somewhere else.\n",
      "Q969F9 already found somewhere else.\n",
      "Q8N4E7 already found somewhere else.\n",
      "P13716 already found somewhere else.\n",
      "P08397 already found somewhere else.\n",
      "Q5T3L8 already found somewhere else.\n",
      "P06132 already found somewhere else.\n",
      "P36551 already found somewhere else.\n",
      "D3DVG2 already found somewhere else.\n",
      "P22830 already found somewhere else.\n",
      "P53701 already found somewhere else.\n",
      "Q9BPW9 already found somewhere else.\n",
      "Q8TC12 already found somewhere else.\n",
      "Q9NYR8 already found somewhere else.\n",
      "O75911 already found somewhere else.\n",
      "Q92781 already found somewhere else.\n",
      "O43174 already found somewhere else.\n",
      "O95237 already found somewhere else.\n",
      "P09455 already found somewhere else.\n",
      "P23141 already found somewhere else.\n",
      "O00748 already found somewhere else.\n",
      "Q6NT32 already found somewhere else.\n",
      "Q5VY30 already found somewhere else.\n",
      "A0A0C4DFY7 already found somewhere else.\n",
      "O75452 already found somewhere else.\n",
      "F6TD35 already found somewhere else.\n",
      "Q9NR63 already found somewhere else.\n",
      "P00352 already found somewhere else.\n",
      "O94788 already found somewhere else.\n",
      "Q969G6 already found somewhere else.\n",
      "Q9Y5L5 already found somewhere else.\n",
      "Q8NFF5 already found somewhere else.\n",
      "P30043 already found somewhere else.\n",
      "Q8IWU9 already found somewhere else.\n",
      "P17752 already found somewhere else.\n",
      "Q9H3S4 already found somewhere else.\n",
      "Q9BU02 already found somewhere else.\n",
      "Q9BSD7 already found somewhere else.\n",
      "Q8IVH4 already found somewhere else.\n",
      "Q96EY8 already found somewhere else.\n",
      "Q96GD0 already found somewhere else.\n",
      "Q8TCD6 already found somewhere else.\n",
      "Q9Y3L3 already found somewhere else.\n",
      "O00764 already found somewhere else.\n",
      "Q9NVS9 already found somewhere else.\n",
      "Q07973 already found somewhere else.\n",
      "Q9UQ10 already found somewhere else.\n",
      "P28845 already found somewhere else.\n",
      "O60664 already found somewhere else.\n",
      "A5D6W6 already found somewhere else.\n",
      "A0A0J9YXN7 already found somewhere else.\n",
      "A0A024RBP2 already found somewhere else.\n",
      "P22694 already found somewhere else.\n",
      "E7DVW4 already found somewhere else.\n",
      "P17612 already found somewhere else.\n",
      "P62140 already found somewhere else.\n",
      "A6NFH5 already found somewhere else.\n",
      "Q9Y2P4 already found somewhere else.\n",
      "P07148 already found somewhere else.\n",
      "Q01469 already found somewhere else.\n",
      "O95477 already found somewhere else.\n",
      "O43747 already found somewhere else.\n",
      "Q92482 already found somewhere else.\n",
      "O14520 already found somewhere else.\n",
      "P21439 already found somewhere else.\n",
      "F5GZ49 already found somewhere else.\n",
      "P45844 already found somewhere else.\n",
      "Q9H221 already found somewhere else.\n",
      "Q9H222 already found somewhere else.\n",
      "Q9H2B4 already found somewhere else.\n",
      "A0A024R8J3 already found somewhere else.\n",
      "O15427 already found somewhere else.\n",
      "O15374 already found somewhere else.\n",
      "P53985 already found somewhere else.\n",
      "O95907 already found somewhere else.\n",
      "O60669 already found somewhere else.\n",
      "O15375 already found somewhere else.\n",
      "Q6PXP3 already found somewhere else.\n",
      "Q9NRM0 already found somewhere else.\n",
      "P22732 already found somewhere else.\n",
      "P11168 already found somewhere else.\n",
      "Q9NY64 already found somewhere else.\n",
      "A0A087X018 already found somewhere else.\n",
      "P48066 already found somewhere else.\n",
      "P48065 already found somewhere else.\n",
      "Q9Y289 already found somewhere else.\n",
      "P52569 already found somewhere else.\n",
      "O95528 already found somewhere else.\n",
      "P11169 already found somewhere else.\n",
      "P11166 already found somewhere else.\n",
      "Q96QE2 already found somewhere else.\n",
      "Q02094 already found somewhere else.\n",
      "P29972 already found somewhere else.\n",
      "P41181 already found somewhere else.\n",
      "P13866 already found somewhere else.\n",
      "O94778 already found somewhere else.\n",
      "P55087 already found somewhere else.\n",
      "Q96PS8 already found somewhere else.\n",
      "Q13520 already found somewhere else.\n",
      "P55064 already found somewhere else.\n",
      "Q9Y2C5 already found somewhere else.\n",
      "Q14916 already found somewhere else.\n",
      "O95436 already found somewhere else.\n",
      "O00476 already found somewhere else.\n",
      "Q06495 already found somewhere else.\n",
      "O00624 already found somewhere else.\n",
      "Q8WUM9 already found somewhere else.\n",
      "Q08357 already found somewhere else.\n",
      "Q8N130 already found somewhere else.\n",
      "Q9UPY5 already found somewhere else.\n",
      "P08195 already found somewhere else.\n",
      "A0A087WT87 already found somewhere else.\n",
      "F1T0D4 already found somewhere else.\n",
      "P43005 already found somewhere else.\n",
      "P48664 already found somewhere else.\n",
      "P43004 already found somewhere else.\n",
      "Q9HC21 already found somewhere else.\n",
      "Q8WWT9 already found somewhere else.\n",
      "P50443 already found somewhere else.\n",
      "Q96RN1 already found somewhere else.\n",
      "P40879 already found somewhere else.\n",
      "Q8WWI5 already found somewhere else.\n",
      "A0A0G2JL76 already found somewhere else.\n",
      "A0A088QCU6 already found somewhere else.\n",
      "O15244 already found somewhere else.\n",
      "Q8NCS7 already found somewhere else.\n",
      "Q8N4M1 already found somewhere else.\n",
      "P55011 already found somewhere else.\n",
      "Q8IUN5 already found somewhere else.\n",
      "Q2M3M2 already found somewhere else.\n",
      "P14672 already found somewhere else.\n",
      "O75751 already found somewhere else.\n",
      "Q14542 already found somewhere else.\n",
      "Q99808 already found somewhere else.\n",
      "Q9UGH3 already found somewhere else.\n",
      "Q9UHI7 already found somewhere else.\n",
      "Q8TD20 already found somewhere else.\n",
      "Q9UGQ3 already found somewhere else.\n",
      "P31639 already found somewhere else.\n",
      "Q8TDB8 already found somewhere else.\n",
      "Q7RTT9 already found somewhere else.\n",
      "P30825 already found somewhere else.\n",
      "Q969I6 already found somewhere else.\n",
      "Q8WY07 already found somewhere else.\n",
      "Q9UM01 already found somewhere else.\n",
      "Q01650 already found somewhere else.\n",
      "Q9UHI5 already found somewhere else.\n",
      "Q7Z2H8 already found somewhere else.\n",
      "Q8N370 already found somewhere else.\n",
      "O75387 already found somewhere else.\n",
      "Q8TF71 already found somewhere else.\n",
      "Q9NS82 already found somewhere else.\n",
      "Q96N87 already found somewhere else.\n",
      "P01130 already found somewhere else.\n",
      "Q15758 already found somewhere else.\n",
      "Q96QD8 already found somewhere else.\n",
      "Q9H2H9 already found somewhere else.\n",
      "P43007 already found somewhere else.\n",
      "Q695T7 already found somewhere else.\n",
      "Q9H2J7 already found somewhere else.\n",
      "Q99624 already found somewhere else.\n",
      "Q8WUX1 already found somewhere else.\n",
      "Q99884 already found somewhere else.\n",
      "Q9NP91 already found somewhere else.\n",
      "Q8IVB4 already found somewhere else.\n",
      "P48764 already found somewhere else.\n",
      "Q14940 already found somewhere else.\n",
      "P19634 already found somewhere else.\n",
      "Q6AI14 already found somewhere else.\n",
      "Q9UBY0 already found somewhere else.\n",
      "Q9Y6M7 already found somewhere else.\n",
      "Q1EHB4 already found somewhere else.\n",
      "Q8WWX8 already found somewhere else.\n",
      "P31641 already found somewhere else.\n",
      "P48067 already found somewhere else.\n",
      "Q9BY07 already found somewhere else.\n",
      "Q07837 already found somewhere else.\n",
      "H3BNF5 already found somewhere else.\n",
      "P49281 already found somewhere else.\n",
      "P49279 already found somewhere else.\n",
      "Q9NP59 already found somewhere else.\n",
      "Q9NP94 already found somewhere else.\n",
      "Q9C0K1 already found somewhere else.\n",
      "Q6ZMH5 already found somewhere else.\n",
      "Q6P5W5 already found somewhere else.\n",
      "Q13433 already found somewhere else.\n",
      "Q15043 already found somewhere else.\n",
      "Q9BRY0 already found somewhere else.\n",
      "Q9H2X9 already found somewhere else.\n",
      "Q9UP95 already found somewhere else.\n",
      "Q9UHW9 already found somewhere else.\n",
      "Q9Y666 already found somewhere else.\n",
      "Q86YT5 already found somewhere else.\n",
      "Q13183 already found somewhere else.\n",
      "P36021 already found somewhere else.\n",
      "Q9NYB5 already found somewhere else.\n",
      "Q9UIG8 already found somewhere else.\n",
      "O60779 already found somewhere else.\n",
      "Q9BZV2 already found somewhere else.\n",
      "Q92959 already found somewhere else.\n",
      "A0A1B0GX35 already found somewhere else.\n",
      "P31645 already found somewhere else.\n",
      "Q96J66 already found somewhere else.\n",
      "Q9Y6M5 already found somewhere else.\n",
      "Q6YBV0 already found somewhere else.\n",
      "P04920 already found somewhere else.\n",
      "O43511 already found somewhere else.\n",
      "Q9BXS9 already found somewhere else.\n",
      "Q2Y0W8 already found somewhere else.\n",
      "P54219 already found somewhere else.\n",
      "Q05940 already found somewhere else.\n",
      "P41440 already found somewhere else.\n",
      "Q96NT5 already found somewhere else.\n",
      "Q9BZW2 already found somewhere else.\n",
      "Q9Y6R1 already found somewhere else.\n",
      "Q86VW1 already found somewhere else.\n",
      "Q8IZD6 already found somewhere else.\n",
      "Q9Y345 already found somewhere else.\n",
      "Q96FL8 already found somewhere else.\n",
      "Q9Y694 already found somewhere else.\n",
      "P35606 already found somewhere else.\n",
      "Q9Y678 already found somewhere else.\n",
      "P53618 already found somewhere else.\n",
      "O14579 already found somewhere else.\n",
      "P53621 already found somewhere else.\n",
      "P48444 already found somewhere else.\n",
      "Q9BQT8 already found somewhere else.\n",
      "F6Y2X3 already found somewhere else.\n",
      "P15407 already found somewhere else.\n",
      "Q9UJS0 already found somewhere else.\n",
      "O75746 already found somewhere else.\n",
      "A0A0S2Z3G3 already found somewhere else.\n",
      "Q9NP58 already found somewhere else.\n",
      "Q02978 already found somewhere else.\n",
      "P16260 already found somewhere else.\n",
      "O95563 already found somewhere else.\n",
      "Q9Y5U8 already found somewhere else.\n",
      "P53007 already found somewhere else.\n",
      "Q00325 already found somewhere else.\n",
      "Q14849 already found somewhere else.\n",
      "Q9BXI2 already found somewhere else.\n",
      "Q9Y619 already found somewhere else.\n",
      "Q9H1K4 already found somewhere else.\n",
      "Q9H936 already found somewhere else.\n",
      "Q7RTY1 already found somewhere else.\n",
      "Q96A46 already found somewhere else.\n",
      "Q9NYZ2 already found somewhere else.\n",
      "P12236 already found somewhere else.\n",
      "P12235 already found somewhere else.\n",
      "P05141 already found somewhere else.\n",
      "O95258 already found somewhere else.\n",
      "P55916 already found somewhere else.\n",
      "O95847 already found somewhere else.\n",
      "P25874 already found somewhere else.\n",
      "A0A024R5N5 already found somewhere else.\n",
      "P07108 already found somewhere else.\n",
      "Q70HW3 already found somewhere else.\n",
      "A8MYV6 already found somewhere else.\n",
      "O43808 already found somewhere else.\n",
      "P78382 already found somewhere else.\n",
      "B4E301 already found somewhere else.\n",
      "G3V1T4 already found somewhere else.\n",
      "Q99567 already found somewhere else.\n",
      "O75694 already found somewhere else.\n",
      "P49790 already found somewhere else.\n",
      "P35658 already found somewhere else.\n",
      "Q9UKX7 already found somewhere else.\n",
      "Q9BVL2 already found somewhere else.\n",
      "Q8NFH5 already found somewhere else.\n",
      "P37198 already found somewhere else.\n",
      "C9JHN6 already found somewhere else.\n",
      "O15118 already found somewhere else.\n",
      "Q9NRA2 already found somewhere else.\n",
      "Q9BZD2 already found somewhere else.\n",
      "Q969S0 already found somewhere else.\n",
      "Q76EJ3 already found somewhere else.\n",
      "P78381 already found somewhere else.\n",
      "Q96A29 already found somewhere else.\n",
      "Q8TB61 already found somewhere else.\n",
      "O60684 already found somewhere else.\n",
      "Q96QU8 already found somewhere else.\n",
      "A0A2R8Y5S9 already found somewhere else.\n",
      "I3L3H0 already found somewhere else.\n",
      "Q15139 already found somewhere else.\n",
      "Q9P0L0 already found somewhere else.\n",
      "O95292 already found somewhere else.\n",
      "A0A024R3H9 already found somewhere else.\n",
      "Q9NTN3 already found somewhere else.\n"
     ]
    }
   ],
   "source": [
    "mc.remove_gene_dupes(Hep_G2_dir+\"Hep-G2_json.json\", Hep_G2_dir+\"Hep-G2_json_nodup.json\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "f7461ddf",
   "metadata": {},
   "outputs": [],
   "source": [
    "json_temp_model = cobra.io.load_json_model(Hep_G2_dir+\"Hep-G2_json_nodup.json\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "7e9af2cf",
   "metadata": {},
   "outputs": [],
   "source": [
    "cobra.io.write_sbml_model(json_temp_model, Hep_G2_dir+\"Hep-G2_complete.xml\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "eb17ad61",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "No objective coefficients in model. Unclear what should be optimized\n"
     ]
    }
   ],
   "source": [
    "model_hep_g2= cobra.io.read_sbml_model(Hep_G2_dir+\"Hep-G2_complete.xml\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "57978f5a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(-inf, inf)\n",
      "(0.0, inf)\n",
      "(0.0, inf)\n"
     ]
    }
   ],
   "source": [
    "for r in model_hep_g2.reactions :\n",
    "    print(r.bounds)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "0e35cb2b",
   "metadata": {},
   "outputs": [],
   "source": [
    "model_hep_g2.solver = \"glpk\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "df6f745d",
   "metadata": {},
   "outputs": [
    {
     "ename": "OptimizationError",
     "evalue": "Solver status is 'unbounded'.",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mOptimizationError\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[21], line 2\u001b[0m\n\u001b[1;32m      1\u001b[0m model_hep_g2\u001b[39m.\u001b[39mobjective \u001b[39m=\u001b[39m \u001b[39m\"\u001b[39m\u001b[39martificial_biomass\u001b[39m\u001b[39m\"\u001b[39m\n\u001b[0;32m----> 2\u001b[0m sol \u001b[39m=\u001b[39m model_hep_g2\u001b[39m.\u001b[39;49moptimize()\n\u001b[1;32m      3\u001b[0m sol\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/core/model.py:1236\u001b[0m, in \u001b[0;36mModel.optimize\u001b[0;34m(self, objective_sense, raise_error)\u001b[0m\n\u001b[1;32m   1232\u001b[0m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mobjective\u001b[39m.\u001b[39mdirection \u001b[39m=\u001b[39m {\u001b[39m\"\u001b[39m\u001b[39mmaximize\u001b[39m\u001b[39m\"\u001b[39m: \u001b[39m\"\u001b[39m\u001b[39mmax\u001b[39m\u001b[39m\"\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39mminimize\u001b[39m\u001b[39m\"\u001b[39m: \u001b[39m\"\u001b[39m\u001b[39mmin\u001b[39m\u001b[39m\"\u001b[39m}\u001b[39m.\u001b[39mget(\n\u001b[1;32m   1233\u001b[0m     objective_sense, original_direction\n\u001b[1;32m   1234\u001b[0m )\n\u001b[1;32m   1235\u001b[0m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mslim_optimize()\n\u001b[0;32m-> 1236\u001b[0m solution \u001b[39m=\u001b[39m get_solution(\u001b[39mself\u001b[39;49m, raise_error\u001b[39m=\u001b[39;49mraise_error)\n\u001b[1;32m   1237\u001b[0m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mobjective\u001b[39m.\u001b[39mdirection \u001b[39m=\u001b[39m original_direction\n\u001b[1;32m   1238\u001b[0m \u001b[39mreturn\u001b[39;00m solution\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/core/solution.py:166\u001b[0m, in \u001b[0;36mget_solution\u001b[0;34m(model, reactions, metabolites, raise_error)\u001b[0m\n\u001b[1;32m    138\u001b[0m \u001b[39mdef\u001b[39;00m \u001b[39mget_solution\u001b[39m(\n\u001b[1;32m    139\u001b[0m     model: \u001b[39m\"\u001b[39m\u001b[39mModel\u001b[39m\u001b[39m\"\u001b[39m,\n\u001b[1;32m    140\u001b[0m     reactions: Optional[Iterable[\u001b[39m\"\u001b[39m\u001b[39mReaction\u001b[39m\u001b[39m\"\u001b[39m]] \u001b[39m=\u001b[39m \u001b[39mNone\u001b[39;00m,\n\u001b[1;32m    141\u001b[0m     metabolites: Optional[Iterable[\u001b[39m\"\u001b[39m\u001b[39mMetabolite\u001b[39m\u001b[39m\"\u001b[39m]] \u001b[39m=\u001b[39m \u001b[39mNone\u001b[39;00m,\n\u001b[1;32m    142\u001b[0m     raise_error: \u001b[39mbool\u001b[39m \u001b[39m=\u001b[39m \u001b[39mFalse\u001b[39;00m,\n\u001b[1;32m    143\u001b[0m ) \u001b[39m-\u001b[39m\u001b[39m>\u001b[39m Solution:\n\u001b[1;32m    144\u001b[0m     \u001b[39m\"\"\"\u001b[39;00m\n\u001b[1;32m    145\u001b[0m \u001b[39m    Generate a solution representation of the current solver state.\u001b[39;00m\n\u001b[1;32m    146\u001b[0m \n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    164\u001b[0m \n\u001b[1;32m    165\u001b[0m \u001b[39m    \"\"\"\u001b[39;00m\n\u001b[0;32m--> 166\u001b[0m     check_solver_status(model\u001b[39m.\u001b[39;49msolver\u001b[39m.\u001b[39;49mstatus, raise_error\u001b[39m=\u001b[39;49mraise_error)\n\u001b[1;32m    167\u001b[0m     \u001b[39mif\u001b[39;00m reactions \u001b[39mis\u001b[39;00m \u001b[39mNone\u001b[39;00m:\n\u001b[1;32m    168\u001b[0m         reactions \u001b[39m=\u001b[39m model\u001b[39m.\u001b[39mreactions\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/util/solver.py:560\u001b[0m, in \u001b[0;36mcheck_solver_status\u001b[0;34m(status, raise_error)\u001b[0m\n\u001b[1;32m    556\u001b[0m     \u001b[39mraise\u001b[39;00m OptimizationError(\n\u001b[1;32m    557\u001b[0m         \u001b[39m\"\u001b[39m\u001b[39mModel is not optimized yet or solver context has been switched.\u001b[39m\u001b[39m\"\u001b[39m\n\u001b[1;32m    558\u001b[0m     )\n\u001b[1;32m    559\u001b[0m \u001b[39melse\u001b[39;00m:\n\u001b[0;32m--> 560\u001b[0m     \u001b[39mraise\u001b[39;00m OptimizationError(\u001b[39mf\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mSolver status is \u001b[39m\u001b[39m'\u001b[39m\u001b[39m{\u001b[39;00mstatus\u001b[39m}\u001b[39;00m\u001b[39m'\u001b[39m\u001b[39m.\u001b[39m\u001b[39m\"\u001b[39m)\n",
      "\u001b[0;31mOptimizationError\u001b[0m: Solver status is 'unbounded'."
     ]
    }
   ],
   "source": [
    "model_hep_g2.objective = \"artificial_biomass\"\n",
    "sol = model_hep_g2.optimize()\n",
    "sol"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "33833a24",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(-1000.0, 1000.0)\n",
      "(0.0, 1000.0)\n"
     ]
    }
   ],
   "source": [
    "for reaction in model_hep_g2.reactions :\n",
    "    print(reaction.bounds)"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "id": "70afd8f3",
   "metadata": {},
   "source": [
    "### Manips\n",
    "\n",
    "The goal here is to show that the model is able to simulate a cancer cell's metabolism.\n",
    "\n",
    "First, we give the model the objective function \"artificial biomass\" : this is what will be maximized.\n",
    "Then, before modifying anything, we can look at how the model produces biomass and tweak what isn't right."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "e2de4b1a",
   "metadata": {},
   "outputs": [
    {
     "ename": "ValueError",
     "evalue": "invalid objective",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/core/model.py:1315\u001b[0m, in \u001b[0;36mModel.objective\u001b[0;34m(self, value)\u001b[0m\n\u001b[1;32m   1314\u001b[0m \u001b[39mtry\u001b[39;00m:\n\u001b[0;32m-> 1315\u001b[0m     reactions \u001b[39m=\u001b[39m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mreactions\u001b[39m.\u001b[39;49mget_by_any(value)\n\u001b[1;32m   1316\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mKeyError\u001b[39;00m:\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/core/dictlist.py:109\u001b[0m, in \u001b[0;36mDictList.get_by_any\u001b[0;34m(self, iterable)\u001b[0m\n\u001b[1;32m    108\u001b[0m     iterable \u001b[39m=\u001b[39m [iterable]\n\u001b[0;32m--> 109\u001b[0m \u001b[39mreturn\u001b[39;00m [get_item(item) \u001b[39mfor\u001b[39;00m item \u001b[39min\u001b[39;00m iterable]\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/core/dictlist.py:109\u001b[0m, in \u001b[0;36m<listcomp>\u001b[0;34m(.0)\u001b[0m\n\u001b[1;32m    108\u001b[0m     iterable \u001b[39m=\u001b[39m [iterable]\n\u001b[0;32m--> 109\u001b[0m \u001b[39mreturn\u001b[39;00m [get_item(item) \u001b[39mfor\u001b[39;00m item \u001b[39min\u001b[39;00m iterable]\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/core/dictlist.py:101\u001b[0m, in \u001b[0;36mDictList.get_by_any.<locals>.get_item\u001b[0;34m(item)\u001b[0m\n\u001b[1;32m    100\u001b[0m \u001b[39melif\u001b[39;00m \u001b[39misinstance\u001b[39m(item, \u001b[39mstr\u001b[39m):\n\u001b[0;32m--> 101\u001b[0m     \u001b[39mreturn\u001b[39;00m \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49mget_by_id(item)\n\u001b[1;32m    102\u001b[0m \u001b[39melif\u001b[39;00m item \u001b[39min\u001b[39;00m \u001b[39mself\u001b[39m:\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/core/dictlist.py:75\u001b[0m, in \u001b[0;36mDictList.get_by_id\u001b[0;34m(self, id)\u001b[0m\n\u001b[1;32m     74\u001b[0m \u001b[39m\"\"\"Return the element with a matching id.\"\"\"\u001b[39;00m\n\u001b[0;32m---> 75\u001b[0m \u001b[39mreturn\u001b[39;00m \u001b[39mlist\u001b[39m\u001b[39m.\u001b[39m\u001b[39m__getitem__\u001b[39m(\u001b[39mself\u001b[39m, \u001b[39mself\u001b[39;49m\u001b[39m.\u001b[39;49m_dict[\u001b[39mid\u001b[39;49m])\n",
      "\u001b[0;31mKeyError\u001b[0m: 'artificial_biomass'",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mValueError\u001b[0m                                Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[7], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m Hep_G2\u001b[39m.\u001b[39mobjective \u001b[39m=\u001b[39m \u001b[39m\"\u001b[39m\u001b[39martificial_biomass\u001b[39m\u001b[39m\"\u001b[39m\n\u001b[1;32m      3\u001b[0m Hep_G2\u001b[39m.\u001b[39msummary()\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/core/model.py:1317\u001b[0m, in \u001b[0;36mModel.objective\u001b[0;34m(self, value)\u001b[0m\n\u001b[1;32m   1315\u001b[0m     reactions \u001b[39m=\u001b[39m \u001b[39mself\u001b[39m\u001b[39m.\u001b[39mreactions\u001b[39m.\u001b[39mget_by_any(value)\n\u001b[1;32m   1316\u001b[0m \u001b[39mexcept\u001b[39;00m \u001b[39mKeyError\u001b[39;00m:\n\u001b[0;32m-> 1317\u001b[0m     \u001b[39mraise\u001b[39;00m \u001b[39mValueError\u001b[39;00m(\u001b[39m\"\u001b[39m\u001b[39minvalid objective\u001b[39m\u001b[39m\"\u001b[39m)\n\u001b[1;32m   1318\u001b[0m value \u001b[39m=\u001b[39m {rxn: \u001b[39m1\u001b[39m \u001b[39mfor\u001b[39;00m rxn \u001b[39min\u001b[39;00m reactions}\n\u001b[1;32m   1319\u001b[0m \u001b[39m# TODO - check that it is reset with context.\u001b[39;00m\n",
      "\u001b[0;31mValueError\u001b[0m: invalid objective"
     ]
    }
   ],
   "source": [
    "Hep_G2.objective = \"artificial_biomass\"\n",
    "\n",
    "Hep_G2.summary()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "5ce74e28",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "hmr_7438 = iHep.reactions.HMR_7438.genes\n",
    "#(Hep_G2.reactions.get_by_id(\"HMR_4388\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "7ed8694d",
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'iHep' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[3], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m iHep\u001b[39m.\u001b[39mreactions\u001b[39m.\u001b[39mHMR_7438\u001b[39m.\u001b[39mgene_reaction_rule\n",
      "\u001b[0;31mNameError\u001b[0m: name 'iHep' is not defined"
     ]
    }
   ],
   "source": [
    "iHep.reactions.HMR_7438.gene_reaction_rule"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "312b44c7",
   "metadata": {},
   "outputs": [],
   "source": [
    "Hep_G2.reactions.EX_temp001x.bounds = (0.0,1000)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "03abb0e4",
   "metadata": {},
   "source": [
    "### Addition of an artificial biomass reaction to Hep_G2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "e29da7a0",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'EX_temp001x' since it already exists.\n"
     ]
    }
   ],
   "source": [
    "biomass_metabolites = './Hep-G2/Hep-G2.xml-5a91f1955e8a9c8a5c5d2ff4a1737c21/utils_biomass_metabolites.txt'\n",
    "biomass_components_manual = []\n",
    "with open(biomass_metabolites, 'r') as input_metabolites :\n",
    "    metabolites_list = input_metabolites.readlines()\n",
    "    \n",
    "    for metabolite_to_add in metabolites_list :\n",
    "        biomass_components_manual.append(metabolite_to_add.strip('\\n'))\n",
    "\n",
    "reaction = cobra.Reaction('artificial_biomass')\n",
    "reaction.lower_bound = 0.0\n",
    "reaction.upper_bound = 1000.0\n",
    "temp001x= cobra.Metabolite(\n",
    "    'temp001x',\n",
    "    formula='',\n",
    "    name='biomass_',\n",
    "    compartment='C_c')\n",
    "#Ajout de la sortie de la biomasse\n",
    "output_biom = cobra.Reaction('EX_temp001x')\n",
    "output_biom.lower_bound = 0.0\n",
    "output_biom.upper_bound = 1000.0\n",
    "\n",
    "output_biom.add_metabolites({temp001x :-1.0})\n",
    "Hep_G2.add_reactions([output_biom])\n",
    "Hep_G2.add_reactions([reaction])\n",
    "Hep_G2.reactions.artificial_biomass.add_metabolites({temp001x : 1.0})\n",
    "\n",
    "for metabolite in biomass_components_manual :\n",
    "    Hep_G2.reactions.artificial_biomass.add_metabolites({Hep_G2.metabolites.get_by_id(str(metabolite)) : -1.0})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "649a17b9",
   "metadata": {},
   "outputs": [],
   "source": [
    "Hep_G2.remove_metabolites(Hep_G2.metabolites.get_by_id('m01371c'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "605337cc",
   "metadata": {},
   "outputs": [],
   "source": [
    "abm = []\n",
    "for metabolite in Hep_G2.reactions.artificial_biomass.metabolites :\n",
    "    abm.append(metabolite.id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "c4ee9967",
   "metadata": {},
   "outputs": [],
   "source": [
    "bm = []\n",
    "for metabolite in Hep_G2.reactions.biomass_components.metabolites:\n",
    "    bm.append(metabolite.id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "7139a2c2",
   "metadata": {},
   "outputs": [],
   "source": [
    "iHep_bm = []\n",
    "for metabolite in iHep.reactions.biomass_components.metabolites :\n",
    "    iHep_bm.append(metabolite.id)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f3a2ce46",
   "metadata": {},
   "source": [
    "Metabolites that are in the **artificial_biomass** reaction but not in **iHep's biomass reaction**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "21939dd0",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "temp001x biomass\n"
     ]
    }
   ],
   "source": [
    "for m in abm :\n",
    "    if m not in iHep_bm :\n",
    "        print(m, Hep_G2.metabolites.get_by_id(m).name)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9d7e9ecb",
   "metadata": {},
   "source": [
    "Metabolites that are in **iHep's biomass reaction** but not in the **artificial_biomass** reaction"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "90a71a5a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "m01602c cofactors and vitamins\n"
     ]
    }
   ],
   "source": [
    "for m in iHep_bm :\n",
    "    if m not in abm :\n",
    "        print(m, Hep_G2.metabolites.get_by_id(m).name)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3bbbefce",
   "metadata": {},
   "source": [
    "Metabolites that are in **Hep_G2's original biomass reaction** but not in the **artificial_biomass reaction**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "b47e4d0a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "m01602c cofactors and vitamins\n"
     ]
    }
   ],
   "source": [
    "for m in bm : \n",
    "    if m not in abm : \n",
    "        print(m, Hep_G2.metabolites.get_by_id(m).name)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b77a8a5b",
   "metadata": {},
   "source": [
    "Metabolites that are in the **artificial_biomass reaction** but not in **Hep_G2's original biomass reaction**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "318d1a97",
   "metadata": {},
   "outputs": [],
   "source": [
    "for m in abm : \n",
    "    if m not in bm : \n",
    "        print(m,Hep_G2.metabolites.get_by_id(m).name)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b16f5646",
   "metadata": {},
   "source": [
    "## Hep_G2's biomass reaction :"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "3038f164",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "HMR_3905, (0.0, inf)\n",
      "HMR_3907, (0.0, inf)\n",
      "HMR_4097, (0.0, inf)\n",
      "HMR_4099, (0.0, inf)\n",
      "HMR_4108, (0.0, inf)\n",
      "HMR_4133, (0.0, inf)\n",
      "HMR_4137, (0.0, inf)\n",
      "HMR_4283, (0.0, inf)\n",
      "HMR_8357, (0.0, inf)\n",
      "HMR_4379, (0.0, inf)\n",
      "HMR_4301, (0.0, inf)\n",
      "HMR_4355, (0.0, inf)\n",
      "HMR_4358, (0.0, inf)\n",
      "HMR_4360, (0.0, inf)\n",
      "HMR_4370, (0.0, inf)\n",
      "HMR_4371, (0.0, inf)\n",
      "HMR_4372, (0.0, inf)\n",
      "HMR_4394, (0.0, inf)\n",
      "HMR_4521, (0.0, inf)\n",
      "HMR_6410, (0.0, inf)\n",
      "HMR_6412, (0.0, inf)\n",
      "HMR_7746, (0.0, inf)\n",
      "HMR_7747, (0.0, inf)\n",
      "HMR_8360, (0.0, inf)\n",
      "HMR_3989, (0.0, inf)\n",
      "HMR_4122, (0.0, inf)\n",
      "HMR_4837, (0.0, inf)\n",
      "HMR_5395, (0.0, inf)\n",
      "HMR_5396, (0.0, inf)\n",
      "HMR_9727, (0.0, inf)\n",
      "HMR_5397, (0.0, inf)\n",
      "HMR_5398, (0.0, inf)\n",
      "HMR_5399, (0.0, inf)\n",
      "HMR_5400, (0.0, inf)\n",
      "HMR_5401, (0.0, inf)\n",
      "HMR_8568, (0.0, inf)\n",
      "HMR_8570, (0.0, inf)\n",
      "HMR_8572, (0.0, inf)\n",
      "HMR_8574, (0.0, inf)\n",
      "HMR_8576, (0.0, inf)\n",
      "HMR_8578, (0.0, inf)\n",
      "HMR_8591, (0.0, inf)\n",
      "HMR_8592, (0.0, inf)\n",
      "HMR_8587, (0.0, inf)\n",
      "HMR_8589, (0.0, inf)\n",
      "HMR_3944, (0.0, inf)\n",
      "HMR_4130, (0.0, inf)\n",
      "HMR_4132, (0.0, inf)\n",
      "HMR_4303, (0.0, inf)\n",
      "HMR_4414, (0.0, inf)\n",
      "HMR_4415, (0.0, inf)\n",
      "HMR_4416, (0.0, inf)\n",
      "HMR_7674, (0.0, inf)\n",
      "HMR_8761, (0.0, inf)\n",
      "HMR_8767, (0.0, inf)\n",
      "HMR_4297, (0.0, inf)\n",
      "HMR_4319, (0.0, inf)\n",
      "HMR_4385, (0.0, inf)\n",
      "HMR_4386, (0.0, inf)\n",
      "HMR_4387, (0.0, inf)\n",
      "HMR_4399, (0.0, inf)\n",
      "HMR_4400, (0.0, inf)\n",
      "HMR_4490, (0.0, inf)\n",
      "HMR_4706, (0.0, inf)\n",
      "HMR_8768, (0.0, inf)\n",
      "HMR_4590, (0.0, inf)\n",
      "HMR_4591, (0.0, inf)\n",
      "HMR_8341, (0.0, inf)\n",
      "HMR_8726, (0.0, inf)\n",
      "HMR_8727, (0.0, inf)\n",
      "HMR_8729, (0.0, inf)\n",
      "HMR_1568, (0.0, inf)\n",
      "HMR_3853, (0.0, inf)\n",
      "HMR_3854, (0.0, inf)\n",
      "HMR_3855, (0.0, inf)\n",
      "HMR_3857, (0.0, inf)\n",
      "HMR_3859, (0.0, inf)\n",
      "HMR_4087, (0.0, inf)\n",
      "HMR_4089, (0.0, inf)\n",
      "HMR_4091, (0.0, inf)\n",
      "HMR_4103, (0.0, inf)\n",
      "HMR_4143, (0.0, inf)\n",
      "HMR_4193, (0.0, inf)\n",
      "HMR_8497, (0.0, inf)\n",
      "HMR_8500, (0.0, inf)\n",
      "HMR_8501, (0.0, inf)\n",
      "HMR_8503, (0.0, inf)\n",
      "HMR_8504, (0.0, inf)\n",
      "HMR_8506, (0.0, inf)\n",
      "HMR_8507, (0.0, inf)\n",
      "HMR_8508, (0.0, inf)\n",
      "HMR_8509, (0.0, inf)\n",
      "HMR_8511, (0.0, inf)\n",
      "HMR_8514, (0.0, inf)\n",
      "HMR_8516, (0.0, inf)\n",
      "HMR_4105, (0.0, inf)\n",
      "HMR_4106, (0.0, inf)\n",
      "HMR_4107, (0.0, inf)\n",
      "HMR_4282, (0.0, inf)\n",
      "HMR_4497, (0.0, inf)\n",
      "HMR_4741, (0.0, inf)\n",
      "HMR_1436, (0.0, inf)\n",
      "HMR_3163, (0.0, inf)\n",
      "HMR_4604, (0.0, inf)\n",
      "HMR_4052, (0.0, inf)\n",
      "HMR_4304, (0.0, inf)\n",
      "HMR_4473, (0.0, inf)\n",
      "HMR_4474, (0.0, inf)\n",
      "HMR_4623, (0.0, inf)\n",
      "HMR_4625, (0.0, inf)\n",
      "HMR_8074, (0.0, inf)\n",
      "HMR_3998, (0.0, inf)\n",
      "HMR_4000, (0.0, inf)\n",
      "HMR_4010, (0.0, inf)\n",
      "HMR_4081, (0.0, inf)\n",
      "HMR_4082, (0.0, inf)\n",
      "HMR_4012, (0.0, inf)\n",
      "HMR_4014, (0.0, inf)\n",
      "HMR_4022, (0.0, inf)\n",
      "HMR_4040, (0.0, inf)\n",
      "HMR_4042, (0.0, inf)\n",
      "HMR_4046, (0.0, inf)\n",
      "HMR_4048, (0.0, inf)\n",
      "HMR_4080, (0.0, inf)\n",
      "HMR_4134, (0.0, inf)\n",
      "HMR_4168, (0.0, inf)\n",
      "HMR_4171, (0.0, inf)\n",
      "HMR_4290, (0.0, inf)\n",
      "HMR_4406, (0.0, inf)\n",
      "HMR_4417, (0.0, inf)\n",
      "HMR_4419, (0.0, inf)\n",
      "HMR_4421, (0.0, inf)\n",
      "HMR_4449, (0.0, inf)\n",
      "HMR_4450, (0.0, inf)\n",
      "HMR_4453, (0.0, inf)\n",
      "HMR_4481, (0.0, inf)\n",
      "HMR_4482, (0.0, inf)\n",
      "HMR_4486, (0.0, inf)\n",
      "HMR_4487, (0.0, inf)\n",
      "HMR_4488, (0.0, inf)\n",
      "HMR_4489, (0.0, inf)\n",
      "HMR_4492, (0.0, inf)\n",
      "HMR_4493, (0.0, inf)\n",
      "HMR_4495, (0.0, inf)\n",
      "HMR_4496, (0.0, inf)\n",
      "HMR_4573, (0.0, inf)\n",
      "HMR_4600, (0.0, inf)\n",
      "HMR_4602, (0.0, inf)\n",
      "HMR_4611, (0.0, inf)\n",
      "HMR_4612, (0.0, inf)\n",
      "HMR_4614, (0.0, inf)\n",
      "HMR_4615, (0.0, inf)\n",
      "HMR_4617, (0.0, inf)\n",
      "HMR_4618, (0.0, inf)\n",
      "HMR_4619, (0.0, inf)\n",
      "HMR_4621, (0.0, inf)\n",
      "HMR_4632, (0.0, inf)\n",
      "HMR_4694, (0.0, inf)\n",
      "HMR_4705, (0.0, inf)\n",
      "HMR_4799, (0.0, inf)\n",
      "HMR_4802, (0.0, inf)\n",
      "HMR_4808, (0.0, inf)\n",
      "HMR_4810, (0.0, inf)\n",
      "HMR_5301, (0.0, inf)\n",
      "HMR_7144, (0.0, inf)\n",
      "HMR_4420, (0.0, inf)\n",
      "HMR_4572, (0.0, inf)\n",
      "HMR_4675, (0.0, inf)\n",
      "HMR_3931, (0.0, inf)\n",
      "HMR_3969, (0.0, inf)\n",
      "HMR_3970, (0.0, inf)\n",
      "HMR_4032, (0.0, inf)\n",
      "HMR_4034, (0.0, inf)\n",
      "HMR_4036, (0.0, inf)\n",
      "HMR_4056, (0.0, inf)\n",
      "HMR_4059, (0.0, inf)\n",
      "HMR_4060, (0.0, inf)\n",
      "HMR_4127, (0.0, inf)\n",
      "HMR_4177, (0.0, inf)\n",
      "HMR_4179, (0.0, inf)\n",
      "HMR_4182, (0.0, inf)\n",
      "HMR_4183, (0.0, inf)\n",
      "HMR_4192, (0.0, inf)\n",
      "HMR_4194, (0.0, inf)\n",
      "HMR_4210, (0.0, inf)\n",
      "HMR_4343, (0.0, inf)\n",
      "HMR_4346, (0.0, inf)\n",
      "HMR_4484, (0.0, inf)\n",
      "HMR_4510, (0.0, inf)\n",
      "HMR_4513, (0.0, inf)\n",
      "HMR_4514, (0.0, inf)\n",
      "HMR_4580, (0.0, inf)\n",
      "HMR_4642, (0.0, inf)\n",
      "HMR_4643, (0.0, inf)\n",
      "HMR_4676, (0.0, inf)\n",
      "HMR_5299, (0.0, inf)\n",
      "HMR_5352, (0.0, inf)\n",
      "HMR_6622, (0.0, inf)\n",
      "HMR_6623, (0.0, inf)\n",
      "HMR_6624, (0.0, inf)\n",
      "HMR_8072, (0.0, inf)\n",
      "HMR_3965, (0.0, inf)\n",
      "HMR_3966, (0.0, inf)\n",
      "HMR_3999, (0.0, inf)\n",
      "HMR_4054, (0.0, inf)\n",
      "HMR_4061, (0.0, inf)\n",
      "HMR_4136, (0.0, inf)\n",
      "HMR_4180, (0.0, inf)\n",
      "HMR_4185, (0.0, inf)\n",
      "HMR_4186, (0.0, inf)\n",
      "HMR_4342, (0.0, inf)\n",
      "HMR_4483, (0.0, inf)\n",
      "HMR_4507, (0.0, inf)\n",
      "HMR_4516, (0.0, inf)\n",
      "HMR_4601, (0.0, inf)\n",
      "HMR_4633, (0.0, inf)\n",
      "HMR_4640, (0.0, inf)\n",
      "HMR_4641, (0.0, inf)\n",
      "HMR_4806, (0.0, inf)\n",
      "HMR_6625, (0.0, inf)\n",
      "HMR_6626, (0.0, inf)\n",
      "HMR_7160, (0.0, inf)\n",
      "HMR_7161, (0.0, inf)\n",
      "HMR_7162, (0.0, inf)\n",
      "HMR_7163, (0.0, inf)\n",
      "HMR_7164, (0.0, inf)\n",
      "HMR_7713, (0.0, inf)\n",
      "HMR_7716, (0.0, inf)\n",
      "HMR_7721, (0.0, inf)\n",
      "HMR_7725, (0.0, inf)\n",
      "HMR_7728, (0.0, inf)\n",
      "HMR_7879, (0.0, inf)\n",
      "HMR_7880, (0.0, inf)\n",
      "HMR_8083, (0.0, inf)\n",
      "HMR_8085, (0.0, inf)\n",
      "HMR_8087, (0.0, inf)\n",
      "HMR_8443, (0.0, inf)\n",
      "HMR_8444, (0.0, inf)\n",
      "HMR_8473, (0.0, inf)\n",
      "HMR_8474, (0.0, inf)\n",
      "HMR_8482, (0.0, inf)\n",
      "HMR_8483, (0.0, inf)\n",
      "HMR_8485, (0.0, inf)\n",
      "HMR_8487, (0.0, inf)\n",
      "HMR_8488, (0.0, inf)\n",
      "HMR_8489, (0.0, inf)\n",
      "HMR_3811, (0.0, inf)\n",
      "HMR_3822, (0.0, inf)\n",
      "HMR_3831, (0.0, inf)\n",
      "HMR_3862, (0.0, inf)\n",
      "HMR_3865, (0.0, inf)\n",
      "HMR_3870, (0.0, inf)\n",
      "HMR_3873, (0.0, inf)\n",
      "HMR_8654, (0.0, inf)\n",
      "HMR_3890, (0.0, inf)\n",
      "HMR_3892, (0.0, inf)\n",
      "HMR_9802, (0.0, inf)\n",
      "HMR_3903, (0.0, inf)\n",
      "HMR_4114, (0.0, inf)\n",
      "HMR_4115, (0.0, inf)\n",
      "HMR_4118, (0.0, inf)\n",
      "HMR_4172, (0.0, inf)\n",
      "HMR_4196, (0.0, inf)\n",
      "HMR_4197, (0.0, inf)\n",
      "HMR_4287, (0.0, inf)\n",
      "HMR_6780, (0.0, inf)\n",
      "HMR_7641, (0.0, inf)\n",
      "HMR_8626, (0.0, inf)\n",
      "HMR_8628, (0.0, inf)\n",
      "HMR_3816, (0.0, inf)\n",
      "HMR_3833, (0.0, inf)\n",
      "HMR_3835, (0.0, inf)\n",
      "HMR_3837, (0.0, inf)\n",
      "HMR_3895, (0.0, inf)\n",
      "HMR_3897, (0.0, inf)\n",
      "HMR_3956, (0.0, inf)\n",
      "HMR_3993, (0.0, inf)\n",
      "HMR_4073, (0.0, inf)\n",
      "HMR_4075, (0.0, inf)\n",
      "HMR_4077, (0.0, inf)\n",
      "HMR_4191, (0.0, inf)\n",
      "HMR_4212, (0.0, inf)\n",
      "HMR_4422, (0.0, inf)\n",
      "HMR_4423, (0.0, inf)\n",
      "HMR_4424, (0.0, inf)\n",
      "HMR_4606, (0.0, inf)\n",
      "HMR_4607, (0.0, inf)\n",
      "HMR_4776, (0.0, inf)\n",
      "HMR_4777, (0.0, inf)\n",
      "HMR_4778, (0.0, inf)\n",
      "HMR_4779, (0.0, inf)\n",
      "HMR_4780, (0.0, inf)\n",
      "HMR_4781, (0.0, inf)\n",
      "HMR_4784, (0.0, inf)\n",
      "HMR_4785, (0.0, inf)\n",
      "HMR_4786, (0.0, inf)\n",
      "HMR_4787, (0.0, inf)\n",
      "HMR_5384, (0.0, inf)\n",
      "HMR_6931, (0.0, inf)\n",
      "HMR_6932, (0.0, inf)\n",
      "HMR_6933, (0.0, inf)\n",
      "HMR_6934, (0.0, inf)\n",
      "HMR_6935, (0.0, inf)\n",
      "HMR_6936, (0.0, inf)\n",
      "HMR_6941, (0.0, inf)\n",
      "HMR_6942, (0.0, inf)\n",
      "HMR_6943, (0.0, inf)\n",
      "HMR_6944, (0.0, inf)\n",
      "HMR_6945, (0.0, inf)\n",
      "HMR_6946, (0.0, inf)\n",
      "HMR_6947, (0.0, inf)\n",
      "HMR_6948, (0.0, inf)\n",
      "HMR_6949, (0.0, inf)\n",
      "HMR_6950, (0.0, inf)\n",
      "HMR_6951, (0.0, inf)\n",
      "HMR_6952, (0.0, inf)\n",
      "HMR_6958, (0.0, inf)\n",
      "HMR_6959, (0.0, inf)\n",
      "HMR_6961, (0.0, inf)\n",
      "HMR_6963, (0.0, inf)\n",
      "HMR_6964, (0.0, inf)\n",
      "HMR_8098, (0.0, inf)\n",
      "HMR_8426, (0.0, inf)\n",
      "HMR_8432, (0.0, inf)\n",
      "HMR_8603, (0.0, inf)\n",
      "HMR_8604, (0.0, inf)\n",
      "HMR_8605, (0.0, inf)\n",
      "HMR_8609, (0.0, inf)\n",
      "HMR_8610, (0.0, inf)\n",
      "HMR_8611, (0.0, inf)\n",
      "HMR_4285, (0.0, inf)\n",
      "HMR_0457, (0.0, inf)\n",
      "HMR_0460, (0.0, inf)\n",
      "HMR_3843, (0.0, inf)\n",
      "HMR_3849, (0.0, inf)\n",
      "HMR_3852, (0.0, inf)\n",
      "HMR_3856, (0.0, inf)\n",
      "HMR_3860, (0.0, inf)\n",
      "HMR_3883, (0.0, inf)\n",
      "HMR_3901, (0.0, inf)\n",
      "HMR_3974, (0.0, inf)\n",
      "HMR_4198, (0.0, inf)\n",
      "HMR_4200, (0.0, inf)\n",
      "HMR_4348, (0.0, inf)\n",
      "HMR_4467, (0.0, inf)\n",
      "HMR_4696, (0.0, inf)\n",
      "HMR_4697, (0.0, inf)\n",
      "HMR_4698, (0.0, inf)\n",
      "HMR_4699, (0.0, inf)\n",
      "HMR_4742, (0.0, inf)\n",
      "HMR_4788, (0.0, inf)\n",
      "HMR_4791, (0.0, inf)\n",
      "HMR_5392, (0.0, inf)\n",
      "HMR_5393, (0.0, inf)\n",
      "HMR_8433, (0.0, inf)\n",
      "HMR_8434, (0.0, inf)\n",
      "HMR_8435, (0.0, inf)\n",
      "HMR_8436, (0.0, inf)\n",
      "HMR_8439, (0.0, inf)\n",
      "HMR_8440, (0.0, inf)\n",
      "HMR_8441, (0.0, inf)\n",
      "HMR_8442, (0.0, inf)\n",
      "HMR_9718, (0.0, inf)\n",
      "HMR_9486, (0.0, inf)\n",
      "HMR_4428, (0.0, inf)\n",
      "HMR_4429, (0.0, inf)\n",
      "HMR_4430, (0.0, inf)\n",
      "HMR_4431, (0.0, inf)\n",
      "HMR_4437, (0.0, inf)\n",
      "HMR_4660, (0.0, inf)\n",
      "HMR_4712, (0.0, inf)\n",
      "HMR_5338, (0.0, inf)\n",
      "HMR_5340, (0.0, inf)\n",
      "HMR_8783, (0.0, inf)\n",
      "HMR_8784, (0.0, inf)\n",
      "HMR_8786, (0.0, inf)\n",
      "HMR_4239, (0.0, inf)\n",
      "HMR_4288, (0.0, inf)\n",
      "HMR_4599, (0.0, inf)\n",
      "HMR_4667, (0.0, inf)\n",
      "HMR_4668, (0.0, inf)\n",
      "HMR_4737, (0.0, inf)\n",
      "HMR_4739, (0.0, inf)\n",
      "HMR_6415, (0.0, inf)\n",
      "HMR_6974, (0.0, inf)\n",
      "HMR_6975, (0.0, inf)\n",
      "HMR_6976, (0.0, inf)\n",
      "HMR_6977, (0.0, inf)\n",
      "HMR_6978, (0.0, inf)\n",
      "HMR_6979, (0.0, inf)\n",
      "HMR_6980, (0.0, inf)\n",
      "HMR_6981, (0.0, inf)\n",
      "HMR_6982, (0.0, inf)\n",
      "HMR_6983, (0.0, inf)\n",
      "HMR_6984, (0.0, inf)\n",
      "HMR_6985, (0.0, inf)\n",
      "HMR_8017, (0.0, inf)\n",
      "HMR_8018, (0.0, inf)\n",
      "HMR_8019, (0.0, inf)\n",
      "HMR_8021, (0.0, inf)\n",
      "HMR_8025, (0.0, inf)\n",
      "HMR_8026, (0.0, inf)\n",
      "HMR_8027, (0.0, inf)\n",
      "HMR_8029, (0.0, inf)\n",
      "HMR_3164, (0.0, inf)\n",
      "HMR_3166, (0.0, inf)\n",
      "HMR_4250, (0.0, inf)\n",
      "HMR_8091, (0.0, inf)\n",
      "HMR_8092, (0.0, inf)\n",
      "HMR_8563, (0.0, inf)\n",
      "HMR_8564, (0.0, inf)\n",
      "HMR_8566, (0.0, inf)\n",
      "HMR_4681, (0.0, inf)\n",
      "HMR_4682, (0.0, inf)\n",
      "HMR_4683, (0.0, inf)\n",
      "HMR_4685, (0.0, inf)\n",
      "HMR_7689, (0.0, inf)\n",
      "HMR_8529, (0.0, inf)\n",
      "HMR_8534, (0.0, inf)\n",
      "HMR_8535, (0.0, inf)\n",
      "HMR_3206, (0.0, inf)\n",
      "HMR_3208, (0.0, inf)\n",
      "HMR_3211, (0.0, inf)\n",
      "HMR_3213, (0.0, inf)\n",
      "HMR_3215, (0.0, inf)\n",
      "HMR_3747, (0.0, inf)\n",
      "HMR_3744, (0.0, inf)\n",
      "HMR_3757, (0.0, inf)\n",
      "HMR_3761, (0.0, inf)\n",
      "HMR_3777, (0.0, inf)\n",
      "HMR_3778, (0.0, inf)\n",
      "HMR_3795, (0.0, inf)\n",
      "HMR_4797, (0.0, inf)\n",
      "HMR_6419, (0.0, inf)\n",
      "HMR_6420, (0.0, inf)\n",
      "HMR_6421, (0.0, inf)\n",
      "HMR_6422, (0.0, inf)\n",
      "HMR_6923, (0.0, inf)\n",
      "HMR_3765, (0.0, inf)\n",
      "HMR_8787, (0.0, inf)\n",
      "HMR_3940, (0.0, inf)\n",
      "HMR_4214, (0.0, inf)\n",
      "HMR_4216, (0.0, inf)\n",
      "HMR_4225, (0.0, inf)\n",
      "HMR_4242, (0.0, inf)\n",
      "HMR_4243, (0.0, inf)\n",
      "HMR_4244, (0.0, inf)\n",
      "HMR_6707, (0.0, inf)\n",
      "HMR_6713, (0.0, inf)\n",
      "HMR_6714, (0.0, inf)\n",
      "HMR_6715, (0.0, inf)\n",
      "HMR_6716, (0.0, inf)\n",
      "HMR_6717, (0.0, inf)\n",
      "HMR_6718, (0.0, inf)\n",
      "HMR_6719, (0.0, inf)\n",
      "HMR_6730, (0.0, inf)\n",
      "HMR_6731, (0.0, inf)\n",
      "HMR_6736, (0.0, inf)\n",
      "HMR_6740, (0.0, inf)\n",
      "HMR_6745, (0.0, inf)\n",
      "HMR_6746, (0.0, inf)\n",
      "HMR_6747, (0.0, inf)\n",
      "HMR_6748, (0.0, inf)\n",
      "HMR_6749, (0.0, inf)\n",
      "HMR_6750, (0.0, inf)\n",
      "HMR_6751, (0.0, inf)\n",
      "HMR_6752, (0.0, inf)\n",
      "HMR_6753, (0.0, inf)\n",
      "HMR_6754, (0.0, inf)\n",
      "HMR_6758, (0.0, inf)\n",
      "HMR_6760, (0.0, inf)\n",
      "HMR_6761, (0.0, inf)\n",
      "HMR_6762, (0.0, inf)\n",
      "HMR_6763, (0.0, inf)\n",
      "HMR_6764, (0.0, inf)\n",
      "HMR_6766, (0.0, inf)\n",
      "HMR_6767, (0.0, inf)\n",
      "HMR_6768, (0.0, inf)\n",
      "HMR_6772, (0.0, inf)\n",
      "HMR_6774, (0.0, inf)\n",
      "HMR_6776, (0.0, inf)\n",
      "HMR_6778, (0.0, inf)\n",
      "HMR_6782, (0.0, inf)\n",
      "HMR_6783, (0.0, inf)\n",
      "HMR_6784, (0.0, inf)\n",
      "HMR_6785, (0.0, inf)\n",
      "HMR_6786, (0.0, inf)\n",
      "HMR_6787, (0.0, inf)\n",
      "HMR_6788, (0.0, inf)\n",
      "HMR_6789, (0.0, inf)\n",
      "HMR_6790, (0.0, inf)\n",
      "HMR_6791, (0.0, inf)\n",
      "HMR_6801, (0.0, inf)\n",
      "HMR_6829, (0.0, inf)\n",
      "HMR_6831, (0.0, inf)\n",
      "HMR_6837, (0.0, inf)\n",
      "HMR_6838, (0.0, inf)\n",
      "HMR_6839, (0.0, inf)\n",
      "HMR_6844, (0.0, inf)\n",
      "HMR_6845, (0.0, inf)\n",
      "HMR_6850, (0.0, inf)\n",
      "HMR_6851, (0.0, inf)\n",
      "HMR_7628, (0.0, inf)\n",
      "HMR_7756, (0.0, inf)\n",
      "HMR_8539, (0.0, inf)\n",
      "HMR_3875, (0.0, inf)\n",
      "HMR_3877, (0.0, inf)\n",
      "HMR_3879, (0.0, inf)\n",
      "HMR_3881, (0.0, inf)\n",
      "HMR_3917, (0.0, inf)\n",
      "HMR_3928, (0.0, inf)\n",
      "HMR_4189, (0.0, inf)\n",
      "HMR_4302, (0.0, inf)\n",
      "HMR_4323, (0.0, inf)\n",
      "HMR_5385, (0.0, inf)\n",
      "HMR_5386, (0.0, inf)\n",
      "HMR_5387, (0.0, inf)\n",
      "HMR_5388, (0.0, inf)\n",
      "HMR_5389, (0.0, inf)\n",
      "HMR_5391, (0.0, inf)\n",
      "HMR_6515, (0.0, inf)\n",
      "HMR_6518, (0.0, inf)\n",
      "HMR_8062, (0.0, inf)\n",
      "HMR_8641, (0.0, inf)\n",
      "HMR_3868, (0.0, inf)\n",
      "HMR_3869, (0.0, inf)\n",
      "HMR_3889, (0.0, inf)\n",
      "HMR_3937, (0.0, inf)\n",
      "HMR_4116, (0.0, inf)\n",
      "HMR_4120, (0.0, inf)\n",
      "HMR_4121, (0.0, inf)\n",
      "HMR_4175, (0.0, inf)\n",
      "HMR_4176, (0.0, inf)\n",
      "HMR_4324, (0.0, inf)\n",
      "HMR_4326, (0.0, inf)\n",
      "HMR_4328, (0.0, inf)\n",
      "HMR_4329, (0.0, inf)\n",
      "HMR_4708, (0.0, inf)\n",
      "HMR_8769, (0.0, inf)\n",
      "HMR_8770, (0.0, inf)\n",
      "HMR_4079, (0.0, inf)\n",
      "HMR_4425, (0.0, inf)\n",
      "HMR_4427, (0.0, inf)\n",
      "HMR_4689, (0.0, inf)\n",
      "HMR_7991, (0.0, inf)\n",
      "HMR_7992, (0.0, inf)\n",
      "HMR_7993, (0.0, inf)\n",
      "HMR_7128, (0.0, inf)\n",
      "HMR_7129, (0.0, inf)\n",
      "HMR_7130, (0.0, inf)\n",
      "HMR_7135, (0.0, inf)\n",
      "HMR_7138, (0.0, inf)\n",
      "HMR_7141, (0.0, inf)\n",
      "HMR_8640, (0.0, inf)\n",
      "HMR_3748, (0.0, inf)\n",
      "HMR_3751, (0.0, inf)\n",
      "HMR_3753, (0.0, inf)\n",
      "HMR_3755, (0.0, inf)\n",
      "HMR_3759, (0.0, inf)\n",
      "HMR_3763, (0.0, inf)\n",
      "HMR_3767, (0.0, inf)\n",
      "HMR_3773, (0.0, inf)\n",
      "HMR_3780, (0.0, inf)\n",
      "HMR_3785, (0.0, inf)\n",
      "HMR_3790, (0.0, inf)\n",
      "HMR_6416, (0.0, inf)\n",
      "HMR_3986, (0.0, inf)\n",
      "HMR_3988, (0.0, inf)\n",
      "HMR_4124, (0.0, inf)\n",
      "HMR_4158, (0.0, inf)\n",
      "HMR_4299, (0.0, inf)\n",
      "HMR_4300, (0.0, inf)\n",
      "HMR_4494, (0.0, inf)\n",
      "HMR_4524, (0.0, inf)\n",
      "HMR_4525, (0.0, inf)\n",
      "HMR_4526, (0.0, inf)\n",
      "HMR_4528, (0.0, inf)\n",
      "HMR_4529, (0.0, inf)\n",
      "HMR_4530, (0.0, inf)\n",
      "HMR_4628, (0.0, inf)\n",
      "HMR_4629, (0.0, inf)\n",
      "HMR_4631, (0.0, inf)\n",
      "HMR_7696, (0.0, inf)\n",
      "HMR_7697, (0.0, inf)\n",
      "HMR_8371, (0.0, inf)\n",
      "HMR_8375, (0.0, inf)\n",
      "HMR_8377, (0.0, inf)\n",
      "HMR_8672, (0.0, inf)\n",
      "HMR_5130, (0.0, inf)\n",
      "HMR_5131, (0.0, inf)\n",
      "HMR_5132, (0.0, inf)\n",
      "HMR_5133, (0.0, inf)\n",
      "HMR_5134, (0.0, inf)\n",
      "HMR_5135, (0.0, inf)\n",
      "HMR_5136, (0.0, inf)\n",
      "HMR_5137, (0.0, inf)\n",
      "HMR_5138, (0.0, inf)\n",
      "HMR_5139, (0.0, inf)\n",
      "HMR_5140, (0.0, inf)\n",
      "HMR_5141, (0.0, inf)\n",
      "HMR_5142, (0.0, inf)\n",
      "HMR_5143, (0.0, inf)\n",
      "HMR_5145, (0.0, inf)\n",
      "HMR_5146, (0.0, inf)\n",
      "HMR_5147, (0.0, inf)\n",
      "HMR_5148, (0.0, inf)\n",
      "HMR_5149, (0.0, inf)\n",
      "HMR_5150, (0.0, inf)\n",
      "HMR_7171, (0.0, inf)\n",
      "HMR_7174, (0.0, inf)\n",
      "HMR_7175, (0.0, inf)\n",
      "HMR_7180, (0.0, inf)\n",
      "HMR_7181, (0.0, inf)\n",
      "HMR_7436, (0.0, inf)\n",
      "HMR_7438, (0.0, inf)\n",
      "HMR_7440, (0.0, inf)\n",
      "HMR_8257, (0.0, inf)\n",
      "HMR_7254, (0.0, inf)\n",
      "HMR_7255, (0.0, inf)\n",
      "HMR_7256, (0.0, inf)\n",
      "HMR_7261, (0.0, inf)\n",
      "HMR_7263, (0.0, inf)\n",
      "HMR_7264, (0.0, inf)\n",
      "HMR_7265, (0.0, inf)\n",
      "HMR_7266, (0.0, inf)\n",
      "HMR_7267, (0.0, inf)\n",
      "HMR_7268, (0.0, inf)\n",
      "HMR_7269, (0.0, inf)\n",
      "HMR_7270, (0.0, inf)\n",
      "HMR_7271, (0.0, inf)\n",
      "HMR_7274, (0.0, inf)\n",
      "HMR_7275, (0.0, inf)\n",
      "HMR_7276, (0.0, inf)\n",
      "HMR_7277, (0.0, inf)\n",
      "HMR_7278, (0.0, inf)\n",
      "HMR_7279, (0.0, inf)\n",
      "HMR_7280, (0.0, inf)\n",
      "HMR_7281, (0.0, inf)\n",
      "HMR_7285, (0.0, inf)\n",
      "HMR_7286, (0.0, inf)\n",
      "HMR_7287, (0.0, inf)\n",
      "HMR_7288, (0.0, inf)\n",
      "HMR_7289, (0.0, inf)\n",
      "HMR_7290, (0.0, inf)\n",
      "HMR_7291, (0.0, inf)\n",
      "HMR_7293, (0.0, inf)\n",
      "HMR_7300, (0.0, inf)\n",
      "HMR_7308, (0.0, inf)\n",
      "HMR_7309, (0.0, inf)\n",
      "HMR_7310, (0.0, inf)\n",
      "HMR_7311, (0.0, inf)\n",
      "HMR_7312, (0.0, inf)\n",
      "HMR_7313, (0.0, inf)\n",
      "HMR_7314, (0.0, inf)\n",
      "HMR_7315, (0.0, inf)\n",
      "HMR_7316, (0.0, inf)\n",
      "HMR_7317, (0.0, inf)\n",
      "HMR_7318, (0.0, inf)\n",
      "HMR_7319, (0.0, inf)\n",
      "HMR_7320, (0.0, inf)\n",
      "HMR_7321, (0.0, inf)\n",
      "HMR_7322, (0.0, inf)\n",
      "HMR_7323, (0.0, inf)\n",
      "HMR_7324, (0.0, inf)\n",
      "HMR_7325, (0.0, inf)\n",
      "HMR_7327, (0.0, inf)\n",
      "HMR_7328, (0.0, inf)\n",
      "HMR_7332, (0.0, inf)\n",
      "HMR_7333, (0.0, inf)\n",
      "HMR_7334, (0.0, inf)\n",
      "HMR_7428, (0.0, inf)\n",
      "HMR_7429, (0.0, inf)\n",
      "HMR_7574, (0.0, inf)\n",
      "HMR_7575, (0.0, inf)\n",
      "HMR_7576, (0.0, inf)\n",
      "HMR_7577, (0.0, inf)\n",
      "HMR_7578, (0.0, inf)\n",
      "HMR_7580, (0.0, inf)\n",
      "HMR_7582, (0.0, inf)\n",
      "HMR_7585, (0.0, inf)\n",
      "HMR_7586, (0.0, inf)\n",
      "HMR_7587, (0.0, inf)\n",
      "HMR_8691, (0.0, inf)\n",
      "HMR_8693, (0.0, inf)\n",
      "HMR_8694, (0.0, inf)\n",
      "HMR_5151, (0.0, inf)\n",
      "HMR_5166, (0.0, inf)\n",
      "HMR_5169, (0.0, inf)\n",
      "HMR_5170, (0.0, inf)\n",
      "HMR_5171, (0.0, inf)\n",
      "HMR_5172, (0.0, inf)\n",
      "HMR_7617, (0.0, inf)\n",
      "HMR_7618, (0.0, inf)\n",
      "HMR_7621, (0.0, inf)\n",
      "HMR_5258, (0.0, inf)\n",
      "HMR_5259, (0.0, inf)\n",
      "HMR_5282, (0.0, inf)\n",
      "HMR_5284, (0.0, inf)\n",
      "HMR_5285, (0.0, inf)\n",
      "HMR_5286, (0.0, inf)\n",
      "HMR_5287, (0.0, inf)\n",
      "HMR_5288, (0.0, inf)\n",
      "HMR_3861, (0.0, inf)\n",
      "HMR_3866, (0.0, inf)\n",
      "HMR_3915, (0.0, inf)\n",
      "HMR_4084, (0.0, inf)\n",
      "HMR_4187, (0.0, inf)\n",
      "HMR_4701, (0.0, inf)\n",
      "HMR_4840, (0.0, inf)\n",
      "HMR_0710, (0.0, inf)\n",
      "HMR_3787, (0.0, inf)\n",
      "HMR_3957, (0.0, inf)\n",
      "HMR_3958, (0.0, inf)\n",
      "HMR_4111, (0.0, inf)\n",
      "HMR_4112, (0.0, inf)\n",
      "HMR_4113, (0.0, inf)\n",
      "HMR_4149, (0.0, inf)\n",
      "HMR_4209, (0.0, inf)\n",
      "HMR_4465, (0.0, inf)\n",
      "HMR_5294, (0.0, inf)\n",
      "HMR_5297, (0.0, inf)\n",
      "HMR_6411, (0.0, inf)\n",
      "HMR_6413, (0.0, inf)\n",
      "HMR_8772, (0.0, inf)\n",
      "HMR_8775, (0.0, inf)\n",
      "HMR_8777, (0.0, inf)\n",
      "HMR_8778, (0.0, inf)\n",
      "HMR_8780, (0.0, inf)\n",
      "HMR_8782, (0.0, inf)\n",
      "HMR_3975, (0.0, inf)\n",
      "HMR_3977, (0.0, inf)\n",
      "HMR_3979, (0.0, inf)\n",
      "HMR_6912, (0.0, inf)\n",
      "HMR_6914, (0.0, inf)\n",
      "HMR_6916, (0.0, inf)\n",
      "HMR_6918, (0.0, inf)\n",
      "HMR_6921, (0.0, inf)\n",
      "HMR_3960, (0.0, inf)\n",
      "HMR_3980, (0.0, inf)\n",
      "HMR_3982, (0.0, inf)\n",
      "HMR_4767, (0.0, inf)\n",
      "HMR_8410, (0.0, inf)\n",
      "HMR_8413, (0.0, inf)\n",
      "HMR_8415, (0.0, inf)\n",
      "HMR_0263, (0.0, inf)\n",
      "HMR_2959, (0.0, inf)\n",
      "HMR_2152, (0.0, inf)\n",
      "HMR_2153, (0.0, inf)\n",
      "HMR_2154, (0.0, inf)\n",
      "HMR_2155, (0.0, inf)\n",
      "HMR_2156, (0.0, inf)\n",
      "HMR_2157, (0.0, inf)\n",
      "HMR_2158, (0.0, inf)\n",
      "HMR_2159, (0.0, inf)\n",
      "HMR_2160, (0.0, inf)\n",
      "HMR_2161, (0.0, inf)\n",
      "HMR_2162, (0.0, inf)\n",
      "HMR_2163, (0.0, inf)\n",
      "HMR_2164, (0.0, inf)\n",
      "HMR_2165, (0.0, inf)\n",
      "HMR_2166, (0.0, inf)\n",
      "HMR_2167, (0.0, inf)\n",
      "HMR_2168, (0.0, inf)\n",
      "HMR_2169, (0.0, inf)\n",
      "HMR_2170, (0.0, inf)\n",
      "HMR_2171, (0.0, inf)\n",
      "HMR_2173, (0.0, inf)\n",
      "HMR_2174, (0.0, inf)\n",
      "HMR_2175, (0.0, inf)\n",
      "HMR_2176, (0.0, inf)\n",
      "HMR_2178, (0.0, inf)\n",
      "HMR_2179, (0.0, inf)\n",
      "HMR_2180, (0.0, inf)\n",
      "HMR_2181, (0.0, inf)\n",
      "HMR_4156, (0.0, inf)\n",
      "HMR_4295, (0.0, inf)\n",
      "HMR_2227, (0.0, inf)\n",
      "HMR_2228, (0.0, inf)\n",
      "HMR_2229, (0.0, inf)\n",
      "HMR_2230, (0.0, inf)\n",
      "HMR_2231, (0.0, inf)\n",
      "HMR_2232, (0.0, inf)\n",
      "HMR_2233, (0.0, inf)\n",
      "HMR_2234, (0.0, inf)\n",
      "HMR_2235, (0.0, inf)\n",
      "HMR_2236, (0.0, inf)\n",
      "HMR_2237, (0.0, inf)\n",
      "HMR_2238, (0.0, inf)\n",
      "HMR_2239, (0.0, inf)\n",
      "HMR_2240, (0.0, inf)\n",
      "HMR_2241, (0.0, inf)\n",
      "HMR_2242, (0.0, inf)\n",
      "HMR_2243, (0.0, inf)\n",
      "HMR_2244, (0.0, inf)\n",
      "HMR_2245, (0.0, inf)\n",
      "HMR_2246, (0.0, inf)\n",
      "HMR_2247, (0.0, inf)\n",
      "HMR_2249, (0.0, inf)\n",
      "HMR_2250, (0.0, inf)\n",
      "HMR_2251, (0.0, inf)\n",
      "HMR_2252, (0.0, inf)\n",
      "HMR_2254, (0.0, inf)\n",
      "HMR_2255, (0.0, inf)\n",
      "HMR_2256, (0.0, inf)\n",
      "HMR_2257, (0.0, inf)\n",
      "HMR_2305, (0.0, inf)\n",
      "HMR_2307, (0.0, inf)\n",
      "HMR_2309, (0.0, inf)\n",
      "HMR_2311, (0.0, inf)\n",
      "HMR_2315, (0.0, inf)\n",
      "HMR_2317, (0.0, inf)\n",
      "HMR_2319, (0.0, inf)\n",
      "HMR_2321, (0.0, inf)\n",
      "HMR_2324, (0.0, inf)\n",
      "HMR_2326, (0.0, inf)\n",
      "HMR_2328, (0.0, inf)\n",
      "HMR_2330, (0.0, inf)\n",
      "HMR_2332, (0.0, inf)\n",
      "HMR_2334, (0.0, inf)\n",
      "HMR_2336, (0.0, inf)\n",
      "HMR_2338, (0.0, inf)\n",
      "HMR_2342, (0.0, inf)\n",
      "HMR_2343, (0.0, inf)\n",
      "HMR_2344, (0.0, inf)\n",
      "HMR_2345, (0.0, inf)\n",
      "HMR_2347, (0.0, inf)\n",
      "HMR_2348, (0.0, inf)\n",
      "HMR_2349, (0.0, inf)\n",
      "HMR_2350, (0.0, inf)\n",
      "HMR_2353, (0.0, inf)\n",
      "HMR_2354, (0.0, inf)\n",
      "HMR_2355, (0.0, inf)\n",
      "HMR_2356, (0.0, inf)\n",
      "HMR_2361, (0.0, inf)\n",
      "HMR_2362, (0.0, inf)\n",
      "HMR_2363, (0.0, inf)\n",
      "HMR_2364, (0.0, inf)\n",
      "HMR_2190, (0.0, inf)\n",
      "HMR_2191, (0.0, inf)\n",
      "HMR_2193, (0.0, inf)\n",
      "HMR_2194, (0.0, inf)\n",
      "HMR_2201, (0.0, inf)\n",
      "HMR_2202, (0.0, inf)\n",
      "HMR_2203, (0.0, inf)\n",
      "HMR_2204, (0.0, inf)\n",
      "HMR_2205, (0.0, inf)\n",
      "HMR_2208, (0.0, inf)\n",
      "HMR_2209, (0.0, inf)\n",
      "HMR_2210, (0.0, inf)\n",
      "HMR_2211, (0.0, inf)\n",
      "HMR_2212, (0.0, inf)\n",
      "HMR_2213, (0.0, inf)\n",
      "HMR_2214, (0.0, inf)\n",
      "HMR_2215, (0.0, inf)\n",
      "HMR_2217, (0.0, inf)\n",
      "HMR_2218, (0.0, inf)\n",
      "HMR_2219, (0.0, inf)\n",
      "HMR_2259, (0.0, inf)\n",
      "HMR_2260, (0.0, inf)\n",
      "HMR_2261, (0.0, inf)\n",
      "HMR_2262, (0.0, inf)\n",
      "HMR_2263, (0.0, inf)\n",
      "HMR_2264, (0.0, inf)\n",
      "HMR_2265, (0.0, inf)\n",
      "HMR_2266, (0.0, inf)\n",
      "HMR_2267, (0.0, inf)\n",
      "HMR_2268, (0.0, inf)\n",
      "HMR_2269, (0.0, inf)\n",
      "HMR_2270, (0.0, inf)\n",
      "HMR_2281, (0.0, inf)\n",
      "HMR_2282, (0.0, inf)\n",
      "HMR_2284, (0.0, inf)\n",
      "HMR_2286, (0.0, inf)\n",
      "HMR_2287, (0.0, inf)\n",
      "HMR_2292, (0.0, inf)\n",
      "HMR_2293, (0.0, inf)\n",
      "HMR_2294, (0.0, inf)\n",
      "HMR_2295, (0.0, inf)\n",
      "HMR_2296, (0.0, inf)\n",
      "HMR_2359, (0.0, inf)\n",
      "HMR_2365, (0.0, inf)\n",
      "HMR_2288, (0.0, inf)\n",
      "HMR_2289, (0.0, inf)\n",
      "HMR_0709, (0.0, inf)\n",
      "HMR_2150, (0.0, inf)\n",
      "HMR_2151, (0.0, inf)\n",
      "HMR_2172, (0.0, inf)\n",
      "HMR_2177, (0.0, inf)\n",
      "HMR_2182, (0.0, inf)\n",
      "HMR_2248, (0.0, inf)\n",
      "HMR_2253, (0.0, inf)\n",
      "HMR_2258, (0.0, inf)\n",
      "HMR_2440, (0.0, inf)\n",
      "HMR_2441, (0.0, inf)\n",
      "HMR_2455, (0.0, inf)\n",
      "HMR_2456, (0.0, inf)\n",
      "HMR_2457, (0.0, inf)\n",
      "HMR_2458, (0.0, inf)\n",
      "HMR_2464, (0.0, inf)\n",
      "HMR_2468, (0.0, inf)\n",
      "HMR_2470, (0.0, inf)\n",
      "HMR_6398, (0.0, inf)\n",
      "HMR_6400, (0.0, inf)\n",
      "HMR_6401, (0.0, inf)\n",
      "HMR_6402, (0.0, inf)\n",
      "HMR_6403, (0.0, inf)\n",
      "HMR_2472, (0.0, inf)\n",
      "HMR_2475, (0.0, inf)\n",
      "HMR_2478, (0.0, inf)\n",
      "HMR_2480, (0.0, inf)\n",
      "HMR_2482, (0.0, inf)\n",
      "HMR_2484, (0.0, inf)\n",
      "HMR_2486, (0.0, inf)\n",
      "HMR_2489, (0.0, inf)\n",
      "HMR_2491, (0.0, inf)\n",
      "HMR_2493, (0.0, inf)\n",
      "HMR_2497, (0.0, inf)\n",
      "HMR_2499, (0.0, inf)\n",
      "HMR_2501, (0.0, inf)\n",
      "HMR_2503, (0.0, inf)\n",
      "HMR_2505, (0.0, inf)\n",
      "HMR_2510, (0.0, inf)\n",
      "HMR_2511, (0.0, inf)\n",
      "HMR_2512, (0.0, inf)\n",
      "HMR_2513, (0.0, inf)\n",
      "HMR_2515, (0.0, inf)\n",
      "HMR_2516, (0.0, inf)\n",
      "HMR_2518, (0.0, inf)\n",
      "HMR_2520, (0.0, inf)\n",
      "HMR_2522, (0.0, inf)\n",
      "HMR_2524, (0.0, inf)\n",
      "HMR_2526, (0.0, inf)\n",
      "HMR_2528, (0.0, inf)\n",
      "HMR_2530, (0.0, inf)\n",
      "HMR_2533, (0.0, inf)\n",
      "HMR_2535, (0.0, inf)\n",
      "HMR_2537, (0.0, inf)\n",
      "HMR_2540, (0.0, inf)\n",
      "HMR_2541, (0.0, inf)\n",
      "HMR_2542, (0.0, inf)\n",
      "HMR_2543, (0.0, inf)\n",
      "HMR_2544, (0.0, inf)\n",
      "HMR_2553, (0.0, inf)\n",
      "HMR_2555, (0.0, inf)\n",
      "HMR_2557, (0.0, inf)\n",
      "HMR_2561, (0.0, inf)\n",
      "HMR_2562, (0.0, inf)\n",
      "HMR_2563, (0.0, inf)\n",
      "HMR_2570, (0.0, inf)\n",
      "HMR_2572, (0.0, inf)\n",
      "HMR_2573, (0.0, inf)\n",
      "HMR_2574, (0.0, inf)\n",
      "HMR_2576, (0.0, inf)\n",
      "HMR_2578, (0.0, inf)\n",
      "HMR_2579, (0.0, inf)\n",
      "HMR_2580, (0.0, inf)\n",
      "HMR_2582, (0.0, inf)\n",
      "HMR_2368, (0.0, inf)\n",
      "HMR_2371, (0.0, inf)\n",
      "HMR_2374, (0.0, inf)\n",
      "HMR_2376, (0.0, inf)\n",
      "HMR_2378, (0.0, inf)\n",
      "HMR_2380, (0.0, inf)\n",
      "HMR_2383, (0.0, inf)\n",
      "HMR_2387, (0.0, inf)\n",
      "HMR_2389, (0.0, inf)\n",
      "HMR_2395, (0.0, inf)\n",
      "HMR_2397, (0.0, inf)\n",
      "HMR_2399, (0.0, inf)\n",
      "HMR_2401, (0.0, inf)\n",
      "HMR_2403, (0.0, inf)\n",
      "HMR_2408, (0.0, inf)\n",
      "HMR_2409, (0.0, inf)\n",
      "HMR_2410, (0.0, inf)\n",
      "HMR_2411, (0.0, inf)\n",
      "HMR_2415, (0.0, inf)\n",
      "HMR_2417, (0.0, inf)\n",
      "HMR_2419, (0.0, inf)\n",
      "HMR_2421, (0.0, inf)\n",
      "HMR_2423, (0.0, inf)\n",
      "HMR_2426, (0.0, inf)\n",
      "HMR_2428, (0.0, inf)\n",
      "HMR_2430, (0.0, inf)\n",
      "HMR_2433, (0.0, inf)\n",
      "HMR_2434, (0.0, inf)\n",
      "HMR_2435, (0.0, inf)\n",
      "HMR_2436, (0.0, inf)\n",
      "HMR_3444, (0.0, inf)\n",
      "HMR_3446, (0.0, inf)\n",
      "HMR_3447, (0.0, inf)\n",
      "HMR_3448, (0.0, inf)\n",
      "HMR_3449, (0.0, inf)\n",
      "HMR_3450, (0.0, inf)\n",
      "HMR_3451, (0.0, inf)\n",
      "HMR_3452, (0.0, inf)\n",
      "HMR_3453, (0.0, inf)\n",
      "HMR_3456, (0.0, inf)\n",
      "HMR_3458, (0.0, inf)\n",
      "HMR_3459, (0.0, inf)\n",
      "HMR_3461, (0.0, inf)\n",
      "HMR_3462, (0.0, inf)\n",
      "HMR_3463, (0.0, inf)\n",
      "HMR_3464, (0.0, inf)\n",
      "HMR_3465, (0.0, inf)\n",
      "HMR_3466, (0.0, inf)\n",
      "HMR_3467, (0.0, inf)\n",
      "HMR_3470, (0.0, inf)\n",
      "HMR_3472, (0.0, inf)\n",
      "HMR_0931, (0.0, inf)\n",
      "HMR_0932, (0.0, inf)\n",
      "HMR_0933, (0.0, inf)\n",
      "HMR_0934, (0.0, inf)\n",
      "HMR_0935, (0.0, inf)\n",
      "HMR_0936, (0.0, inf)\n",
      "HMR_0937, (0.0, inf)\n",
      "HMR_0938, (0.0, inf)\n",
      "HMR_0939, (0.0, inf)\n",
      "HMR_0940, (0.0, inf)\n",
      "HMR_0942, (0.0, inf)\n",
      "HMR_0943, (0.0, inf)\n",
      "HMR_0944, (0.0, inf)\n",
      "HMR_0945, (0.0, inf)\n",
      "HMR_0946, (0.0, inf)\n",
      "HMR_0947, (0.0, inf)\n",
      "HMR_0948, (0.0, inf)\n",
      "HMR_0949, (0.0, inf)\n",
      "HMR_0950, (0.0, inf)\n",
      "HMR_0951, (0.0, inf)\n",
      "HMR_0953, (0.0, inf)\n",
      "HMR_0954, (0.0, inf)\n",
      "HMR_0955, (0.0, inf)\n",
      "HMR_0956, (0.0, inf)\n",
      "HMR_0957, (0.0, inf)\n",
      "HMR_0958, (0.0, inf)\n",
      "HMR_0960, (0.0, inf)\n",
      "HMR_0963, (0.0, inf)\n",
      "HMR_0964, (0.0, inf)\n",
      "HMR_0969, (0.0, inf)\n",
      "HMR_0976, (0.0, inf)\n",
      "HMR_0981, (0.0, inf)\n",
      "HMR_1029, (0.0, inf)\n",
      "HMR_1033, (0.0, inf)\n",
      "HMR_1037, (0.0, inf)\n",
      "HMR_1039, (0.0, inf)\n",
      "HMR_1059, (0.0, inf)\n",
      "HMR_1061, (0.0, inf)\n",
      "HMR_1065, (0.0, inf)\n",
      "HMR_1066, (0.0, inf)\n",
      "HMR_1067, (0.0, inf)\n",
      "HMR_1068, (0.0, inf)\n",
      "HMR_1069, (0.0, inf)\n",
      "HMR_1070, (0.0, inf)\n",
      "HMR_1071, (0.0, inf)\n",
      "HMR_1072, (0.0, inf)\n",
      "HMR_3015, (0.0, inf)\n",
      "HMR_1080, (0.0, inf)\n",
      "HMR_1081, (0.0, inf)\n",
      "HMR_1084, (0.0, inf)\n",
      "HMR_1085, (0.0, inf)\n",
      "HMR_1100, (0.0, inf)\n",
      "HMR_1101, (0.0, inf)\n",
      "HMR_1102, (0.0, inf)\n",
      "HMR_1103, (0.0, inf)\n",
      "HMR_1105, (0.0, inf)\n",
      "HMR_1106, (0.0, inf)\n",
      "HMR_1107, (0.0, inf)\n",
      "HMR_1108, (0.0, inf)\n",
      "HMR_1109, (0.0, inf)\n",
      "HMR_1110, (0.0, inf)\n",
      "HMR_1111, (0.0, inf)\n",
      "HMR_1112, (0.0, inf)\n",
      "HMR_1126, (0.0, inf)\n",
      "HMR_1127, (0.0, inf)\n",
      "HMR_1128, (0.0, inf)\n",
      "HMR_1137, (0.0, inf)\n",
      "HMR_1138, (0.0, inf)\n",
      "HMR_1139, (0.0, inf)\n",
      "HMR_1140, (0.0, inf)\n",
      "HMR_1161, (0.0, inf)\n",
      "HMR_1162, (0.0, inf)\n",
      "HMR_1163, (0.0, inf)\n",
      "HMR_1164, (0.0, inf)\n",
      "HMR_1165, (0.0, inf)\n",
      "HMR_1166, (0.0, inf)\n",
      "HMR_1167, (0.0, inf)\n",
      "HMR_1168, (0.0, inf)\n",
      "HMR_1170, (0.0, inf)\n",
      "HMR_1171, (0.0, inf)\n",
      "HMR_1204, (0.0, inf)\n",
      "HMR_1205, (0.0, inf)\n",
      "HMR_1206, (0.0, inf)\n",
      "HMR_1207, (0.0, inf)\n",
      "HMR_1208, (0.0, inf)\n",
      "HMR_1209, (0.0, inf)\n",
      "HMR_1210, (0.0, inf)\n",
      "HMR_1211, (0.0, inf)\n",
      "HMR_1212, (0.0, inf)\n",
      "HMR_1213, (0.0, inf)\n",
      "HMR_1228, (0.0, inf)\n",
      "HMR_1229, (0.0, inf)\n",
      "HMR_1230, (0.0, inf)\n",
      "HMR_1231, (0.0, inf)\n",
      "HMR_1232, (0.0, inf)\n",
      "HMR_1233, (0.0, inf)\n",
      "HMR_1234, (0.0, inf)\n",
      "HMR_1235, (0.0, inf)\n",
      "HMR_1236, (0.0, inf)\n",
      "HMR_1237, (0.0, inf)\n",
      "HMR_1238, (0.0, inf)\n",
      "HMR_1239, (0.0, inf)\n",
      "HMR_1244, (0.0, inf)\n",
      "HMR_1245, (0.0, inf)\n",
      "HMR_1246, (0.0, inf)\n",
      "HMR_1253, (0.0, inf)\n",
      "HMR_1254, (0.0, inf)\n",
      "HMR_1255, (0.0, inf)\n",
      "HMR_1256, (0.0, inf)\n",
      "HMR_1257, (0.0, inf)\n",
      "HMR_1258, (0.0, inf)\n",
      "HMR_1259, (0.0, inf)\n",
      "HMR_1260, (0.0, inf)\n",
      "HMR_1261, (0.0, inf)\n",
      "HMR_1262, (0.0, inf)\n",
      "HMR_1270, (0.0, inf)\n",
      "HMR_1271, (0.0, inf)\n",
      "HMR_1272, (0.0, inf)\n",
      "HMR_1273, (0.0, inf)\n",
      "HMR_1274, (0.0, inf)\n",
      "HMR_1275, (0.0, inf)\n",
      "HMR_1276, (0.0, inf)\n",
      "HMR_1277, (0.0, inf)\n",
      "HMR_1278, (0.0, inf)\n",
      "HMR_1279, (0.0, inf)\n",
      "HMR_1287, (0.0, inf)\n",
      "HMR_1288, (0.0, inf)\n",
      "HMR_1289, (0.0, inf)\n",
      "HMR_1291, (0.0, inf)\n",
      "HMR_1292, (0.0, inf)\n",
      "HMR_1293, (0.0, inf)\n",
      "HMR_1294, (0.0, inf)\n",
      "HMR_1296, (0.0, inf)\n",
      "HMR_1299, (0.0, inf)\n",
      "HMR_1300, (0.0, inf)\n",
      "HMR_1301, (0.0, inf)\n",
      "HMR_1303, (0.0, inf)\n",
      "HMR_1304, (0.0, inf)\n",
      "HMR_4796, (0.0, inf)\n",
      "HMR_8556, (0.0, inf)\n",
      "HMR_0154, (0.0, inf)\n",
      "HMR_0189, (0.0, inf)\n",
      "HMR_0193, (0.0, inf)\n",
      "HMR_0197, (0.0, inf)\n",
      "HMR_0201, (0.0, inf)\n",
      "HMR_0206, (0.0, inf)\n",
      "HMR_0210, (0.0, inf)\n",
      "HMR_0214, (0.0, inf)\n",
      "HMR_0223, (0.0, inf)\n",
      "HMR_0230, (0.0, inf)\n",
      "HMR_0234, (0.0, inf)\n",
      "HMR_0238, (0.0, inf)\n",
      "HMR_0242, (0.0, inf)\n",
      "HMR_0246, (0.0, inf)\n",
      "HMR_0250, (0.0, inf)\n",
      "HMR_0256, (0.0, inf)\n",
      "HMR_0260, (0.0, inf)\n",
      "HMR_0264, (0.0, inf)\n",
      "HMR_0268, (0.0, inf)\n",
      "HMR_0272, (0.0, inf)\n",
      "HMR_0276, (0.0, inf)\n",
      "HMR_0280, (0.0, inf)\n",
      "HMR_0284, (0.0, inf)\n",
      "HMR_0290, (0.0, inf)\n",
      "HMR_0294, (0.0, inf)\n",
      "HMR_0298, (0.0, inf)\n",
      "HMR_0302, (0.0, inf)\n",
      "HMR_0306, (0.0, inf)\n",
      "HMR_0310, (0.0, inf)\n",
      "HMR_0314, (0.0, inf)\n",
      "HMR_0320, (0.0, inf)\n",
      "HMR_0324, (0.0, inf)\n",
      "HMR_0328, (0.0, inf)\n",
      "HMR_0332, (0.0, inf)\n",
      "HMR_0338, (0.0, inf)\n",
      "HMR_0342, (0.0, inf)\n",
      "HMR_0346, (0.0, inf)\n",
      "HMR_0350, (0.0, inf)\n",
      "HMR_0354, (0.0, inf)\n",
      "HMR_0358, (0.0, inf)\n",
      "HMR_0362, (0.0, inf)\n",
      "HMR_0366, (0.0, inf)\n",
      "HMR_0370, (0.0, inf)\n",
      "HMR_0374, (0.0, inf)\n",
      "HMR_0378, (0.0, inf)\n",
      "HMR_0382, (0.0, inf)\n",
      "HMR_0386, (0.0, inf)\n",
      "HMR_0390, (0.0, inf)\n",
      "HMR_0394, (0.0, inf)\n",
      "HMR_0398, (0.0, inf)\n",
      "HMR_0402, (0.0, inf)\n",
      "HMR_0406, (0.0, inf)\n",
      "HMR_0410, (0.0, inf)\n",
      "HMR_0414, (0.0, inf)\n",
      "HMR_0418, (0.0, inf)\n",
      "HMR_0422, (0.0, inf)\n",
      "HMR_0426, (0.0, inf)\n",
      "HMR_0430, (0.0, inf)\n",
      "HMR_0434, (0.0, inf)\n",
      "HMR_0438, (0.0, inf)\n",
      "HMR_3475, (0.0, inf)\n",
      "HMR_2592, (0.0, inf)\n",
      "HMR_2600, (0.0, inf)\n",
      "HMR_2745, (0.0, inf)\n",
      "HMR_2825, (0.0, inf)\n",
      "HMR_2829, (0.0, inf)\n",
      "HMR_2830, (0.0, inf)\n",
      "HMR_2833, (0.0, inf)\n",
      "HMR_2834, (0.0, inf)\n",
      "HMR_0001, (0.0, inf)\n",
      "HMR_0002, (0.0, inf)\n",
      "HMR_0003, (0.0, inf)\n",
      "HMR_0005, (0.0, inf)\n",
      "HMR_0604, (0.0, inf)\n",
      "HMR_0605, (0.0, inf)\n",
      "HMR_0665, (0.0, inf)\n",
      "HMR_0667, (0.0, inf)\n",
      "HMR_0668, (0.0, inf)\n",
      "HMR_0669, (0.0, inf)\n",
      "HMR_0670, (0.0, inf)\n",
      "HMR_0671, (0.0, inf)\n",
      "HMR_0672, (0.0, inf)\n",
      "HMR_0679, (0.0, inf)\n",
      "HMR_0680, (0.0, inf)\n",
      "HMR_0681, (0.0, inf)\n",
      "HMR_0682, (0.0, inf)\n",
      "HMR_0683, (0.0, inf)\n",
      "HMR_0684, (0.0, inf)\n",
      "HMR_7588, (0.0, inf)\n",
      "HMR_3053, (0.0, inf)\n",
      "HMR_3057, (0.0, inf)\n",
      "HMR_3058, (0.0, inf)\n",
      "HMR_3059, (0.0, inf)\n",
      "HMR_3060, (0.0, inf)\n",
      "HMR_3062, (0.0, inf)\n",
      "HMR_3063, (0.0, inf)\n",
      "HMR_3064, (0.0, inf)\n",
      "HMR_3065, (0.0, inf)\n",
      "HMR_3066, (0.0, inf)\n",
      "HMR_3067, (0.0, inf)\n",
      "HMR_3068, (0.0, inf)\n",
      "HMR_3069, (0.0, inf)\n",
      "HMR_3070, (0.0, inf)\n",
      "HMR_3071, (0.0, inf)\n",
      "HMR_3072, (0.0, inf)\n",
      "HMR_3073, (0.0, inf)\n",
      "HMR_3074, (0.0, inf)\n",
      "HMR_3075, (0.0, inf)\n",
      "HMR_3076, (0.0, inf)\n",
      "HMR_3077, (0.0, inf)\n",
      "HMR_3078, (0.0, inf)\n",
      "HMR_3079, (0.0, inf)\n",
      "HMR_3080, (0.0, inf)\n",
      "HMR_3081, (0.0, inf)\n",
      "HMR_3082, (0.0, inf)\n",
      "HMR_3083, (0.0, inf)\n",
      "HMR_3084, (0.0, inf)\n",
      "HMR_3085, (0.0, inf)\n",
      "HMR_3086, (0.0, inf)\n",
      "HMR_3087, (0.0, inf)\n",
      "HMR_3088, (0.0, inf)\n",
      "HMR_3089, (0.0, inf)\n",
      "HMR_3090, (0.0, inf)\n",
      "HMR_3091, (0.0, inf)\n",
      "HMR_3092, (0.0, inf)\n",
      "HMR_3093, (0.0, inf)\n",
      "HMR_3094, (0.0, inf)\n",
      "HMR_3095, (0.0, inf)\n",
      "HMR_3096, (0.0, inf)\n",
      "HMR_3097, (0.0, inf)\n",
      "HMR_3098, (0.0, inf)\n",
      "HMR_3099, (0.0, inf)\n",
      "HMR_3100, (0.0, inf)\n",
      "HMR_3101, (0.0, inf)\n",
      "HMR_3102, (0.0, inf)\n",
      "HMR_3103, (0.0, inf)\n",
      "HMR_3104, (0.0, inf)\n",
      "HMR_3326, (0.0, inf)\n",
      "HMR_3327, (0.0, inf)\n",
      "HMR_3328, (0.0, inf)\n",
      "HMR_3329, (0.0, inf)\n",
      "HMR_3330, (0.0, inf)\n",
      "HMR_3331, (0.0, inf)\n",
      "HMR_3332, (0.0, inf)\n",
      "HMR_3333, (0.0, inf)\n",
      "HMR_3334, (0.0, inf)\n",
      "HMR_3335, (0.0, inf)\n",
      "HMR_3336, (0.0, inf)\n",
      "HMR_3337, (0.0, inf)\n",
      "HMR_3338, (0.0, inf)\n",
      "HMR_3339, (0.0, inf)\n",
      "HMR_3340, (0.0, inf)\n",
      "HMR_3341, (0.0, inf)\n",
      "HMR_3342, (0.0, inf)\n",
      "HMR_3343, (0.0, inf)\n",
      "HMR_3344, (0.0, inf)\n",
      "HMR_3345, (0.0, inf)\n",
      "HMR_3346, (0.0, inf)\n",
      "HMR_3347, (0.0, inf)\n",
      "HMR_3348, (0.0, inf)\n",
      "HMR_3349, (0.0, inf)\n",
      "HMR_3350, (0.0, inf)\n",
      "HMR_3351, (0.0, inf)\n",
      "HMR_3352, (0.0, inf)\n",
      "HMR_3353, (0.0, inf)\n",
      "HMR_3364, (0.0, inf)\n",
      "HMR_3365, (0.0, inf)\n",
      "HMR_3367, (0.0, inf)\n",
      "HMR_3368, (0.0, inf)\n",
      "HMR_3369, (0.0, inf)\n",
      "HMR_3370, (0.0, inf)\n",
      "HMR_3372, (0.0, inf)\n",
      "HMR_3373, (0.0, inf)\n",
      "HMR_3484, (0.0, inf)\n",
      "HMR_3486, (0.0, inf)\n",
      "HMR_3488, (0.0, inf)\n",
      "HMR_3498, (0.0, inf)\n",
      "HMR_3501, (0.0, inf)\n",
      "HMR_3503, (0.0, inf)\n",
      "HMR_3505, (0.0, inf)\n",
      "HMR_3506, (0.0, inf)\n",
      "HMR_3508, (0.0, inf)\n",
      "HMR_3509, (0.0, inf)\n",
      "HMR_3510, (0.0, inf)\n",
      "HMR_3511, (0.0, inf)\n",
      "HMR_3512, (0.0, inf)\n",
      "HMR_3513, (0.0, inf)\n",
      "HMR_3514, (0.0, inf)\n",
      "HMR_3515, (0.0, inf)\n",
      "HMR_3301, (0.0, inf)\n",
      "HMR_3302, (0.0, inf)\n",
      "HMR_3304, (0.0, inf)\n",
      "HMR_3305, (0.0, inf)\n",
      "HMR_3306, (0.0, inf)\n",
      "HMR_3307, (0.0, inf)\n",
      "HMR_3309, (0.0, inf)\n",
      "HMR_3310, (0.0, inf)\n",
      "HMR_3311, (0.0, inf)\n",
      "HMR_3312, (0.0, inf)\n",
      "HMR_3314, (0.0, inf)\n",
      "HMR_3315, (0.0, inf)\n",
      "HMR_3316, (0.0, inf)\n",
      "HMR_3317, (0.0, inf)\n",
      "HMR_3319, (0.0, inf)\n",
      "HMR_3320, (0.0, inf)\n",
      "HMR_3321, (0.0, inf)\n",
      "HMR_3322, (0.0, inf)\n",
      "HMR_3107, (0.0, inf)\n",
      "HMR_3108, (0.0, inf)\n",
      "HMR_3109, (0.0, inf)\n",
      "HMR_3110, (0.0, inf)\n",
      "HMR_3111, (0.0, inf)\n",
      "HMR_3112, (0.0, inf)\n",
      "HMR_3113, (0.0, inf)\n",
      "HMR_3114, (0.0, inf)\n",
      "HMR_3115, (0.0, inf)\n",
      "HMR_3116, (0.0, inf)\n",
      "HMR_3117, (0.0, inf)\n",
      "HMR_3118, (0.0, inf)\n",
      "HMR_3121, (0.0, inf)\n",
      "HMR_3122, (0.0, inf)\n",
      "HMR_3123, (0.0, inf)\n",
      "HMR_3125, (0.0, inf)\n",
      "HMR_3128, (0.0, inf)\n",
      "HMR_3129, (0.0, inf)\n",
      "HMR_3130, (0.0, inf)\n",
      "HMR_3132, (0.0, inf)\n",
      "HMR_3135, (0.0, inf)\n",
      "HMR_3136, (0.0, inf)\n",
      "HMR_3137, (0.0, inf)\n",
      "HMR_3139, (0.0, inf)\n",
      "HMR_3142, (0.0, inf)\n",
      "HMR_3143, (0.0, inf)\n",
      "HMR_3144, (0.0, inf)\n",
      "HMR_3146, (0.0, inf)\n",
      "HMR_3149, (0.0, inf)\n",
      "HMR_3150, (0.0, inf)\n",
      "HMR_3151, (0.0, inf)\n",
      "HMR_3153, (0.0, inf)\n",
      "HMR_3156, (0.0, inf)\n",
      "HMR_3157, (0.0, inf)\n",
      "HMR_3158, (0.0, inf)\n",
      "HMR_3160, (0.0, inf)\n",
      "HMR_3398, (0.0, inf)\n",
      "HMR_3406, (0.0, inf)\n",
      "HMR_3407, (0.0, inf)\n",
      "HMR_3408, (0.0, inf)\n",
      "HMR_3409, (0.0, inf)\n",
      "HMR_3411, (0.0, inf)\n",
      "HMR_3413, (0.0, inf)\n",
      "HMR_3414, (0.0, inf)\n",
      "HMR_3416, (0.0, inf)\n",
      "HMR_3421, (0.0, inf)\n",
      "HMR_3422, (0.0, inf)\n",
      "HMR_3423, (0.0, inf)\n",
      "HMR_3424, (0.0, inf)\n",
      "HMR_3425, (0.0, inf)\n",
      "HMR_3426, (0.0, inf)\n",
      "HMR_9719, (0.0, inf)\n",
      "HMR_3427, (0.0, inf)\n",
      "HMR_3428, (0.0, inf)\n",
      "HMR_3429, (0.0, inf)\n",
      "HMR_3430, (0.0, inf)\n",
      "HMR_3431, (0.0, inf)\n",
      "HMR_3432, (0.0, inf)\n",
      "HMR_3433, (0.0, inf)\n",
      "HMR_3170, (0.0, inf)\n",
      "HMR_3171, (0.0, inf)\n",
      "HMR_3172, (0.0, inf)\n",
      "HMR_3173, (0.0, inf)\n",
      "HMR_3174, (0.0, inf)\n",
      "HMR_3175, (0.0, inf)\n",
      "HMR_3176, (0.0, inf)\n",
      "HMR_3177, (0.0, inf)\n",
      "HMR_3178, (0.0, inf)\n",
      "HMR_3179, (0.0, inf)\n",
      "HMR_3180, (0.0, inf)\n",
      "HMR_3181, (0.0, inf)\n",
      "HMR_3182, (0.0, inf)\n",
      "HMR_3183, (0.0, inf)\n",
      "HMR_3184, (0.0, inf)\n",
      "HMR_3185, (0.0, inf)\n",
      "HMR_3186, (0.0, inf)\n",
      "HMR_3187, (0.0, inf)\n",
      "HMR_3188, (0.0, inf)\n",
      "HMR_3189, (0.0, inf)\n",
      "HMR_3190, (0.0, inf)\n",
      "HMR_3191, (0.0, inf)\n",
      "HMR_3192, (0.0, inf)\n",
      "HMR_3193, (0.0, inf)\n",
      "HMR_3194, (0.0, inf)\n",
      "HMR_3195, (0.0, inf)\n",
      "HMR_3196, (0.0, inf)\n",
      "HMR_3197, (0.0, inf)\n",
      "HMR_3198, (0.0, inf)\n",
      "HMR_3199, (0.0, inf)\n",
      "HMR_3200, (0.0, inf)\n",
      "HMR_3201, (0.0, inf)\n",
      "HMR_3202, (0.0, inf)\n",
      "HMR_3203, (0.0, inf)\n",
      "HMR_3204, (0.0, inf)\n",
      "HMR_3205, (0.0, inf)\n",
      "HMR_3396, (0.0, inf)\n",
      "HMR_3240, (0.0, inf)\n",
      "HMR_3241, (0.0, inf)\n",
      "HMR_3242, (0.0, inf)\n",
      "HMR_3243, (0.0, inf)\n",
      "HMR_3244, (0.0, inf)\n",
      "HMR_3245, (0.0, inf)\n",
      "HMR_3246, (0.0, inf)\n",
      "HMR_3247, (0.0, inf)\n",
      "HMR_3250, (0.0, inf)\n",
      "HMR_3252, (0.0, inf)\n",
      "HMR_3254, (0.0, inf)\n",
      "HMR_3256, (0.0, inf)\n",
      "HMR_3258, (0.0, inf)\n",
      "HMR_3260, (0.0, inf)\n",
      "HMR_3262, (0.0, inf)\n",
      "HMR_3264, (0.0, inf)\n",
      "HMR_3356, (0.0, inf)\n",
      "HMR_3357, (0.0, inf)\n",
      "HMR_3358, (0.0, inf)\n",
      "HMR_3359, (0.0, inf)\n",
      "HMR_3360, (0.0, inf)\n",
      "HMR_3361, (0.0, inf)\n",
      "HMR_3362, (0.0, inf)\n",
      "HMR_3363, (0.0, inf)\n",
      "HMR_3218, (0.0, inf)\n",
      "HMR_3219, (0.0, inf)\n",
      "HMR_3220, (0.0, inf)\n",
      "HMR_3221, (0.0, inf)\n",
      "HMR_3222, (0.0, inf)\n",
      "HMR_3223, (0.0, inf)\n",
      "HMR_3224, (0.0, inf)\n",
      "HMR_3225, (0.0, inf)\n",
      "HMR_3226, (0.0, inf)\n",
      "HMR_3227, (0.0, inf)\n",
      "HMR_3228, (0.0, inf)\n",
      "HMR_3229, (0.0, inf)\n",
      "HMR_3230, (0.0, inf)\n",
      "HMR_3231, (0.0, inf)\n",
      "HMR_3232, (0.0, inf)\n",
      "HMR_3233, (0.0, inf)\n",
      "HMR_3234, (0.0, inf)\n",
      "HMR_3235, (0.0, inf)\n",
      "HMR_3236, (0.0, inf)\n",
      "HMR_3237, (0.0, inf)\n",
      "HMR_3275, (0.0, inf)\n",
      "HMR_3277, (0.0, inf)\n",
      "HMR_3278, (0.0, inf)\n",
      "HMR_3279, (0.0, inf)\n",
      "HMR_3280, (0.0, inf)\n",
      "HMR_3281, (0.0, inf)\n",
      "HMR_3282, (0.0, inf)\n",
      "HMR_3283, (0.0, inf)\n",
      "HMR_3284, (0.0, inf)\n",
      "HMR_3285, (0.0, inf)\n",
      "HMR_3286, (0.0, inf)\n",
      "HMR_3287, (0.0, inf)\n",
      "HMR_3288, (0.0, inf)\n",
      "HMR_3290, (0.0, inf)\n",
      "HMR_3292, (0.0, inf)\n",
      "HMR_3293, (0.0, inf)\n",
      "HMR_3294, (0.0, inf)\n",
      "HMR_3296, (0.0, inf)\n",
      "HMR_1174, (0.0, inf)\n",
      "HMR_1175, (0.0, inf)\n",
      "HMR_1176, (0.0, inf)\n",
      "HMR_1177, (0.0, inf)\n",
      "HMR_1178, (0.0, inf)\n",
      "HMR_1179, (0.0, inf)\n",
      "HMR_1180, (0.0, inf)\n",
      "HMR_1182, (0.0, inf)\n",
      "HMR_1183, (0.0, inf)\n",
      "HMR_1184, (0.0, inf)\n",
      "HMR_1216, (0.0, inf)\n",
      "HMR_1217, (0.0, inf)\n",
      "HMR_1218, (0.0, inf)\n",
      "HMR_1223, (0.0, inf)\n",
      "HMR_1224, (0.0, inf)\n",
      "HMR_1225, (0.0, inf)\n",
      "HMR_1226, (0.0, inf)\n",
      "HMR_3522, (0.0, inf)\n",
      "HMR_3523, (0.0, inf)\n",
      "HMR_3524, (0.0, inf)\n",
      "HMR_3525, (0.0, inf)\n",
      "HMR_3526, (0.0, inf)\n",
      "HMR_3527, (0.0, inf)\n",
      "HMR_3528, (0.0, inf)\n",
      "HMR_3529, (0.0, inf)\n",
      "HMR_3530, (0.0, inf)\n",
      "HMR_3531, (0.0, inf)\n",
      "HMR_3532, (0.0, inf)\n",
      "HMR_3533, (0.0, inf)\n",
      "HMR_3534, (0.0, inf)\n",
      "HMR_1573, (0.0, inf)\n",
      "HMR_6901, (0.0, inf)\n",
      "HMR_6903, (0.0, inf)\n",
      "HMR_6904, (0.0, inf)\n",
      "HMR_6905, (0.0, inf)\n",
      "HMR_6906, (0.0, inf)\n",
      "HMR_6907, (0.0, inf)\n",
      "HMR_6908, (0.0, inf)\n",
      "HMR_6909, (0.0, inf)\n",
      "HMR_6910, (0.0, inf)\n",
      "HMR_7165, (0.0, inf)\n",
      "HMR_7166, (0.0, inf)\n",
      "HMR_7167, (0.0, inf)\n",
      "HMR_7168, (0.0, inf)\n",
      "HMR_7169, (0.0, inf)\n",
      "HMR_7170, (0.0, inf)\n",
      "HMR_1644, (0.0, inf)\n",
      "HMR_1920, (0.0, inf)\n",
      "HMR_1924, (0.0, inf)\n",
      "HMR_1926, (0.0, inf)\n",
      "HMR_1927, (0.0, inf)\n",
      "HMR_1940, (0.0, inf)\n",
      "HMR_1948, (0.0, inf)\n",
      "HMR_1990, (0.0, inf)\n",
      "HMR_1991, (0.0, inf)\n",
      "HMR_1992, (0.0, inf)\n",
      "HMR_2005, (0.0, inf)\n",
      "HMR_2007, (0.0, inf)\n",
      "HMR_4766, (0.0, inf)\n",
      "HMR_7648, (0.0, inf)\n",
      "HMR_7930, (0.0, inf)\n",
      "HMR_7935, (0.0, inf)\n",
      "HMR_7940, (0.0, inf)\n",
      "HMR_7941, (0.0, inf)\n",
      "HMR_7945, (0.0, inf)\n",
      "HMR_7948, (0.0, inf)\n",
      "HMR_7950, (0.0, inf)\n",
      "HMR_7952, (0.0, inf)\n",
      "HMR_7955, (0.0, inf)\n",
      "HMR_7958, (0.0, inf)\n",
      "HMR_7959, (0.0, inf)\n",
      "HMR_7962, (0.0, inf)\n",
      "HMR_7965, (0.0, inf)\n",
      "HMR_7968, (0.0, inf)\n",
      "HMR_2041, (0.0, inf)\n",
      "HMR_7970, (0.0, inf)\n",
      "HMR_7971, (0.0, inf)\n",
      "HMR_7974, (0.0, inf)\n",
      "HMR_7976, (0.0, inf)\n",
      "HMR_7978, (0.0, inf)\n",
      "HMR_7980, (0.0, inf)\n",
      "HMR_7984, (0.0, inf)\n",
      "HMR_7987, (0.0, inf)\n",
      "HMR_1944, (0.0, inf)\n",
      "HMR_1952, (0.0, inf)\n",
      "HMR_1953, (0.0, inf)\n",
      "HMR_1962, (0.0, inf)\n",
      "HMR_1963, (0.0, inf)\n",
      "HMR_1973, (0.0, inf)\n",
      "HMR_1974, (0.0, inf)\n",
      "HMR_1976, (0.0, inf)\n",
      "HMR_1977, (0.0, inf)\n",
      "HMR_2016, (0.0, inf)\n",
      "HMR_2017, (0.0, inf)\n",
      "HMR_2018, (0.0, inf)\n",
      "HMR_2019, (0.0, inf)\n",
      "HMR_1440, (0.0, inf)\n",
      "HMR_1445, (0.0, inf)\n",
      "HMR_1454, (0.0, inf)\n",
      "HMR_1467, (0.0, inf)\n",
      "HMR_1470, (0.0, inf)\n",
      "HMR_1473, (0.0, inf)\n",
      "HMR_1477, (0.0, inf)\n",
      "HMR_1478, (0.0, inf)\n",
      "HMR_1479, (0.0, inf)\n",
      "HMR_1484, (0.0, inf)\n",
      "HMR_1490, (0.0, inf)\n",
      "HMR_1493, (0.0, inf)\n",
      "HMR_1494, (0.0, inf)\n",
      "HMR_1495, (0.0, inf)\n",
      "HMR_1496, (0.0, inf)\n",
      "HMR_1502, (0.0, inf)\n",
      "HMR_1503, (0.0, inf)\n",
      "HMR_1504, (0.0, inf)\n",
      "HMR_1505, (0.0, inf)\n",
      "HMR_1509, (0.0, inf)\n",
      "HMR_1512, (0.0, inf)\n",
      "HMR_1516, (0.0, inf)\n",
      "HMR_1519, (0.0, inf)\n",
      "HMR_1526, (0.0, inf)\n",
      "HMR_1533, (0.0, inf)\n",
      "HMR_1557, (0.0, inf)\n",
      "HMR_1558, (0.0, inf)\n",
      "HMR_1565, (0.0, inf)\n",
      "HMR_1535, (0.0, inf)\n",
      "HMR_1536, (0.0, inf)\n",
      "HMR_1538, (0.0, inf)\n",
      "HMR_1539, (0.0, inf)\n",
      "HMR_1540, (0.0, inf)\n",
      "HMR_1543, (0.0, inf)\n",
      "HMR_1544, (0.0, inf)\n",
      "HMR_1545, (0.0, inf)\n",
      "HMR_1546, (0.0, inf)\n",
      "HMR_1547, (0.0, inf)\n",
      "HMR_1548, (0.0, inf)\n",
      "HMR_1549, (0.0, inf)\n",
      "HMR_1550, (0.0, inf)\n",
      "HMR_1551, (0.0, inf)\n",
      "HMR_1552, (0.0, inf)\n",
      "HMR_1553, (0.0, inf)\n",
      "HMR_1437, (0.0, inf)\n",
      "HMR_1451, (0.0, inf)\n",
      "HMR_1457, (0.0, inf)\n",
      "HMR_1460, (0.0, inf)\n",
      "HMR_1531, (0.0, inf)\n",
      "HMR_1570, (0.0, inf)\n",
      "HMR_1576, (0.0, inf)\n",
      "HMR_1577, (0.0, inf)\n",
      "HMR_4630, (0.0, inf)\n",
      "HMR_2029, (0.0, inf)\n",
      "HMR_2030, (0.0, inf)\n",
      "HMR_2031, (0.0, inf)\n",
      "HMR_2032, (0.0, inf)\n",
      "HMR_2033, (0.0, inf)\n",
      "HMR_2034, (0.0, inf)\n",
      "HMR_2036, (0.0, inf)\n",
      "HMR_2038, (0.0, inf)\n",
      "HMR_2042, (0.0, inf)\n",
      "HMR_2043, (0.0, inf)\n",
      "HMR_2053, (0.0, inf)\n",
      "HMR_2054, (0.0, inf)\n",
      "HMR_3537, (0.0, inf)\n",
      "HMR_3540, (0.0, inf)\n",
      "HMR_3545, (0.0, inf)\n",
      "HMR_3546, (0.0, inf)\n",
      "HMR_3551, (0.0, inf)\n",
      "HMR_3552, (0.0, inf)\n",
      "HMR_3574, (0.0, inf)\n",
      "HMR_3577, (0.0, inf)\n",
      "HMR_3581, (0.0, inf)\n",
      "HMR_3586, (0.0, inf)\n",
      "HMR_3588, (0.0, inf)\n",
      "HMR_3589, (0.0, inf)\n",
      "HMR_3625, (0.0, inf)\n",
      "HMR_3626, (0.0, inf)\n",
      "HMR_3627, (0.0, inf)\n",
      "HMR_3628, (0.0, inf)\n",
      "HMR_3629, (0.0, inf)\n",
      "HMR_3630, (0.0, inf)\n",
      "HMR_3631, (0.0, inf)\n",
      "HMR_3632, (0.0, inf)\n",
      "HMR_3633, (0.0, inf)\n",
      "HMR_3634, (0.0, inf)\n",
      "HMR_3635, (0.0, inf)\n",
      "HMR_3636, (0.0, inf)\n",
      "HMR_3637, (0.0, inf)\n",
      "HMR_3638, (0.0, inf)\n",
      "HMR_3639, (0.0, inf)\n",
      "HMR_3640, (0.0, inf)\n",
      "HMR_3641, (0.0, inf)\n",
      "HMR_3642, (0.0, inf)\n",
      "HMR_3643, (0.0, inf)\n",
      "HMR_3644, (0.0, inf)\n",
      "HMR_3645, (0.0, inf)\n",
      "HMR_3646, (0.0, inf)\n",
      "HMR_3647, (0.0, inf)\n",
      "HMR_3648, (0.0, inf)\n",
      "HMR_3649, (0.0, inf)\n",
      "HMR_3650, (0.0, inf)\n",
      "HMR_3651, (0.0, inf)\n",
      "HMR_3652, (0.0, inf)\n",
      "HMR_3653, (0.0, inf)\n",
      "HMR_3654, (0.0, inf)\n",
      "HMR_3655, (0.0, inf)\n",
      "HMR_3656, (0.0, inf)\n",
      "HMR_3657, (0.0, inf)\n",
      "HMR_3658, (0.0, inf)\n",
      "HMR_3659, (0.0, inf)\n",
      "HMR_3660, (0.0, inf)\n",
      "HMR_3661, (0.0, inf)\n",
      "HMR_3662, (0.0, inf)\n",
      "HMR_3663, (0.0, inf)\n",
      "HMR_3664, (0.0, inf)\n",
      "HMR_3665, (0.0, inf)\n",
      "HMR_3666, (0.0, inf)\n",
      "HMR_3667, (0.0, inf)\n",
      "HMR_3668, (0.0, inf)\n",
      "HMR_3669, (0.0, inf)\n",
      "HMR_3670, (0.0, inf)\n",
      "HMR_3671, (0.0, inf)\n",
      "HMR_3672, (0.0, inf)\n",
      "HMR_3673, (0.0, inf)\n",
      "HMR_3674, (0.0, inf)\n",
      "HMR_3675, (0.0, inf)\n",
      "HMR_3676, (0.0, inf)\n",
      "HMR_3677, (0.0, inf)\n",
      "HMR_3678, (0.0, inf)\n",
      "HMR_3679, (0.0, inf)\n",
      "HMR_3680, (0.0, inf)\n",
      "HMR_3681, (0.0, inf)\n",
      "HMR_3682, (0.0, inf)\n",
      "HMR_3683, (0.0, inf)\n",
      "HMR_3684, (0.0, inf)\n",
      "HMR_3685, (0.0, inf)\n",
      "HMR_3686, (0.0, inf)\n",
      "HMR_3687, (0.0, inf)\n",
      "HMR_3688, (0.0, inf)\n",
      "HMR_3689, (0.0, inf)\n",
      "HMR_3690, (0.0, inf)\n",
      "HMR_3691, (0.0, inf)\n",
      "HMR_3692, (0.0, inf)\n",
      "HMR_3693, (0.0, inf)\n",
      "HMR_3694, (0.0, inf)\n",
      "HMR_3695, (0.0, inf)\n",
      "HMR_3696, (0.0, inf)\n",
      "HMR_3697, (0.0, inf)\n",
      "HMR_3698, (0.0, inf)\n",
      "HMR_3699, (0.0, inf)\n",
      "HMR_3700, (0.0, inf)\n",
      "HMR_3701, (0.0, inf)\n",
      "HMR_3703, (0.0, inf)\n",
      "HMR_3704, (0.0, inf)\n",
      "HMR_3705, (0.0, inf)\n",
      "HMR_3706, (0.0, inf)\n",
      "HMR_3707, (0.0, inf)\n",
      "HMR_3708, (0.0, inf)\n",
      "HMR_3709, (0.0, inf)\n",
      "HMR_3710, (0.0, inf)\n",
      "HMR_3711, (0.0, inf)\n",
      "HMR_3712, (0.0, inf)\n",
      "HMR_3713, (0.0, inf)\n",
      "HMR_3714, (0.0, inf)\n",
      "HMR_3715, (0.0, inf)\n",
      "HMR_3716, (0.0, inf)\n",
      "HMR_3717, (0.0, inf)\n",
      "HMR_3718, (0.0, inf)\n",
      "HMR_3719, (0.0, inf)\n",
      "HMR_3720, (0.0, inf)\n",
      "HMR_3721, (0.0, inf)\n",
      "HMR_3722, (0.0, inf)\n",
      "HMR_3723, (0.0, inf)\n",
      "HMR_3724, (0.0, inf)\n",
      "HMR_3725, (0.0, inf)\n",
      "HMR_3726, (0.0, inf)\n",
      "HMR_3727, (0.0, inf)\n",
      "HMR_3728, (0.0, inf)\n",
      "HMR_3729, (0.0, inf)\n",
      "HMR_3732, (0.0, inf)\n",
      "HMR_3733, (0.0, inf)\n",
      "HMR_3734, (0.0, inf)\n",
      "HMR_3735, (0.0, inf)\n",
      "HMR_3736, (0.0, inf)\n",
      "HMR_3737, (0.0, inf)\n",
      "HMR_3738, (0.0, inf)\n",
      "HMR_3739, (0.0, inf)\n",
      "HMR_3740, (0.0, inf)\n",
      "HMR_3741, (0.0, inf)\n",
      "HMR_0715, (0.0, inf)\n",
      "HMR_0716, (0.0, inf)\n",
      "HMR_0735, (0.0, inf)\n",
      "HMR_0736, (0.0, inf)\n",
      "HMR_0738, (0.0, inf)\n",
      "HMR_0741, (0.0, inf)\n",
      "HMR_0744, (0.0, inf)\n",
      "HMR_0746, (0.0, inf)\n",
      "HMR_0748, (0.0, inf)\n",
      "HMR_0750, (0.0, inf)\n",
      "HMR_0754, (0.0, inf)\n",
      "HMR_0758, (0.0, inf)\n",
      "HMR_0760, (0.0, inf)\n",
      "HMR_0761, (0.0, inf)\n",
      "HMR_0762, (0.0, inf)\n",
      "HMR_0765, (0.0, inf)\n",
      "HMR_0766, (0.0, inf)\n",
      "HMR_0767, (0.0, inf)\n",
      "HMR_0773, (0.0, inf)\n",
      "HMR_0775, (0.0, inf)\n",
      "HMR_0779, (0.0, inf)\n",
      "HMR_0781, (0.0, inf)\n",
      "HMR_0783, (0.0, inf)\n",
      "HMR_0795, (0.0, inf)\n",
      "HMR_8147, (0.0, inf)\n",
      "HMR_8148, (0.0, inf)\n",
      "HMR_8149, (0.0, inf)\n",
      "HMR_8150, (0.0, inf)\n",
      "HMR_8151, (0.0, inf)\n",
      "HMR_8152, (0.0, inf)\n",
      "HMR_8155, (0.0, inf)\n",
      "HMR_8165, (0.0, inf)\n",
      "HMR_8166, (0.0, inf)\n",
      "HMR_8167, (0.0, inf)\n",
      "HMR_8168, (0.0, inf)\n",
      "HMR_8172, (0.0, inf)\n",
      "HMR_8173, (0.0, inf)\n",
      "HMR_8175, (0.0, inf)\n",
      "HMR_8178, (0.0, inf)\n",
      "HMR_8179, (0.0, inf)\n",
      "HMR_8180, (0.0, inf)\n",
      "HMR_8184, (0.0, inf)\n",
      "HMR_8185, (0.0, inf)\n",
      "HMR_8186, (0.0, inf)\n",
      "HMR_8188, (0.0, inf)\n",
      "HMR_8189, (0.0, inf)\n",
      "HMR_8190, (0.0, inf)\n",
      "HMR_8191, (0.0, inf)\n",
      "HMR_8192, (0.0, inf)\n",
      "HMR_8194, (0.0, inf)\n",
      "HMR_8198, (0.0, inf)\n",
      "HMR_8201, (0.0, inf)\n",
      "HMR_0786, (0.0, inf)\n",
      "HMR_8206, (0.0, inf)\n",
      "HMR_8209, (0.0, inf)\n",
      "HMR_8211, (0.0, inf)\n",
      "HMR_8212, (0.0, inf)\n",
      "HMR_8217, (0.0, inf)\n",
      "HMR_8219, (0.0, inf)\n",
      "HMR_8220, (0.0, inf)\n",
      "HMR_8224, (0.0, inf)\n",
      "HMR_8226, (0.0, inf)\n",
      "HMR_8227, (0.0, inf)\n",
      "HMR_8228, (0.0, inf)\n",
      "HMR_8233, (0.0, inf)\n",
      "HMR_8235, (0.0, inf)\n",
      "HMR_8238, (0.0, inf)\n",
      "HMR_8245, (0.0, inf)\n",
      "HMR_8246, (0.0, inf)\n",
      "HMR_8248, (0.0, inf)\n",
      "HMR_8249, (0.0, inf)\n",
      "HMR_8250, (0.0, inf)\n",
      "HMR_8251, (0.0, inf)\n",
      "HMR_0456, (0.0, inf)\n",
      "HMR_0458, (0.0, inf)\n",
      "HMR_0459, (0.0, inf)\n",
      "HMR_0486, (0.0, inf)\n",
      "HMR_0487, (0.0, inf)\n",
      "HMR_0488, (0.0, inf)\n",
      "HMR_0489, (0.0, inf)\n",
      "HMR_0490, (0.0, inf)\n",
      "HMR_0491, (0.0, inf)\n",
      "HMR_0492, (0.0, inf)\n",
      "HMR_0493, (0.0, inf)\n",
      "HMR_0494, (0.0, inf)\n",
      "HMR_0495, (0.0, inf)\n",
      "HMR_0496, (0.0, inf)\n",
      "HMR_0497, (0.0, inf)\n",
      "HMR_0498, (0.0, inf)\n",
      "HMR_0499, (0.0, inf)\n",
      "HMR_0500, (0.0, inf)\n",
      "HMR_0501, (0.0, inf)\n",
      "HMR_0502, (0.0, inf)\n",
      "HMR_0503, (0.0, inf)\n",
      "HMR_0504, (0.0, inf)\n",
      "HMR_0505, (0.0, inf)\n",
      "HMR_0506, (0.0, inf)\n",
      "HMR_0507, (0.0, inf)\n",
      "HMR_0508, (0.0, inf)\n",
      "HMR_0509, (0.0, inf)\n",
      "HMR_0510, (0.0, inf)\n",
      "HMR_0511, (0.0, inf)\n",
      "HMR_0512, (0.0, inf)\n",
      "HMR_0513, (0.0, inf)\n",
      "HMR_0514, (0.0, inf)\n",
      "HMR_0515, (0.0, inf)\n",
      "HMR_0516, (0.0, inf)\n",
      "HMR_0517, (0.0, inf)\n",
      "HMR_0518, (0.0, inf)\n",
      "HMR_0519, (0.0, inf)\n",
      "HMR_0520, (0.0, inf)\n",
      "HMR_0521, (0.0, inf)\n",
      "HMR_0522, (0.0, inf)\n",
      "HMR_0523, (0.0, inf)\n",
      "HMR_0524, (0.0, inf)\n",
      "HMR_0525, (0.0, inf)\n",
      "HMR_0526, (0.0, inf)\n",
      "HMR_0527, (0.0, inf)\n",
      "HMR_0528, (0.0, inf)\n",
      "HMR_0529, (0.0, inf)\n",
      "HMR_0530, (0.0, inf)\n",
      "HMR_0531, (0.0, inf)\n",
      "HMR_0532, (0.0, inf)\n",
      "HMR_0533, (0.0, inf)\n",
      "HMR_0534, (0.0, inf)\n",
      "HMR_0535, (0.0, inf)\n",
      "HMR_0536, (0.0, inf)\n",
      "HMR_0537, (0.0, inf)\n",
      "HMR_0538, (0.0, inf)\n",
      "HMR_0539, (0.0, inf)\n",
      "HMR_0540, (0.0, inf)\n",
      "HMR_0541, (0.0, inf)\n",
      "HMR_0542, (0.0, inf)\n",
      "HMR_0543, (0.0, inf)\n",
      "HMR_0544, (0.0, inf)\n",
      "HMR_0588, (0.0, inf)\n",
      "HMR_0596, (0.0, inf)\n",
      "HMR_0673, (0.0, inf)\n",
      "HMR_0674, (0.0, inf)\n",
      "HMR_0675, (0.0, inf)\n",
      "HMR_0676, (0.0, inf)\n",
      "HMR_0677, (0.0, inf)\n",
      "HMR_0678, (0.0, inf)\n",
      "HMR_4296, (0.0, inf)\n",
      "HMR_5254, (0.0, inf)\n",
      "HMR_7599, (0.0, inf)\n",
      "HMR_0448, (0.0, inf)\n",
      "HMR_0450, (0.0, inf)\n",
      "HMR_0468, (0.0, inf)\n",
      "HMR_0478, (0.0, inf)\n",
      "HMR_0479, (0.0, inf)\n",
      "HMR_0481, (0.0, inf)\n",
      "HMR_0579, (0.0, inf)\n",
      "HMR_0580, (0.0, inf)\n",
      "HMR_0581, (0.0, inf)\n",
      "HMR_0582, (0.0, inf)\n",
      "HMR_0584, (0.0, inf)\n",
      "HMR_0586, (0.0, inf)\n",
      "HMR_0589, (0.0, inf)\n",
      "HMR_0590, (0.0, inf)\n",
      "HMR_0591, (0.0, inf)\n",
      "HMR_0592, (0.0, inf)\n",
      "HMR_0593, (0.0, inf)\n",
      "HMR_0594, (0.0, inf)\n",
      "HMR_0597, (0.0, inf)\n",
      "HMR_0598, (0.0, inf)\n",
      "HMR_0599, (0.0, inf)\n",
      "HMR_0600, (0.0, inf)\n",
      "HMR_0601, (0.0, inf)\n",
      "HMR_0602, (0.0, inf)\n",
      "HMR_0610, (0.0, inf)\n",
      "HMR_0613, (0.0, inf)\n",
      "HMR_0614, (0.0, inf)\n",
      "HMR_0615, (0.0, inf)\n",
      "HMR_0616, (0.0, inf)\n",
      "HMR_0622, (0.0, inf)\n",
      "HMR_0623, (0.0, inf)\n",
      "HMR_0625, (0.0, inf)\n",
      "HMR_0627, (0.0, inf)\n",
      "HMR_0629, (0.0, inf)\n",
      "HMR_0630, (0.0, inf)\n",
      "HMR_0632, (0.0, inf)\n",
      "HMR_0633, (0.0, inf)\n",
      "HMR_0634, (0.0, inf)\n",
      "HMR_0635, (0.0, inf)\n",
      "HMR_0636, (0.0, inf)\n",
      "HMR_0638, (0.0, inf)\n",
      "HMR_0642, (0.0, inf)\n",
      "HMR_0643, (0.0, inf)\n",
      "HMR_0644, (0.0, inf)\n",
      "HMR_0645, (0.0, inf)\n",
      "HMR_0646, (0.0, inf)\n",
      "HMR_0648, (0.0, inf)\n",
      "HMR_0651, (0.0, inf)\n",
      "HMR_0653, (0.0, inf)\n",
      "HMR_0654, (0.0, inf)\n",
      "HMR_0657, (0.0, inf)\n",
      "HMR_0660, (0.0, inf)\n",
      "HMR_4627, (0.0, inf)\n",
      "HMR_4838, (0.0, inf)\n",
      "HMR_4839, (0.0, inf)\n",
      "HMR_7594, (0.0, inf)\n",
      "HMR_7597, (0.0, inf)\n",
      "HMR_7601, (0.0, inf)\n",
      "HMR_7611, (0.0, inf)\n",
      "HMR_7613, (0.0, inf)\n",
      "HMR_7755, (0.0, inf)\n",
      "HMR_8518, (0.0, inf)\n",
      "HMR_8523, (0.0, inf)\n",
      "HMR_8525, (0.0, inf)\n",
      "HMR_0815, (0.0, inf)\n",
      "HMR_0816, (0.0, inf)\n",
      "HMR_0817, (0.0, inf)\n",
      "HMR_0819, (0.0, inf)\n",
      "HMR_0820, (0.0, inf)\n",
      "HMR_0821, (0.0, inf)\n",
      "HMR_0823, (0.0, inf)\n",
      "HMR_0824, (0.0, inf)\n",
      "HMR_0825, (0.0, inf)\n",
      "HMR_0827, (0.0, inf)\n",
      "HMR_0828, (0.0, inf)\n",
      "HMR_0830, (0.0, inf)\n",
      "HMR_0837, (0.0, inf)\n",
      "HMR_0838, (0.0, inf)\n",
      "HMR_0841, (0.0, inf)\n",
      "HMR_0842, (0.0, inf)\n",
      "HMR_0844, (0.0, inf)\n",
      "HMR_0845, (0.0, inf)\n",
      "HMR_0847, (0.0, inf)\n",
      "HMR_0848, (0.0, inf)\n",
      "HMR_0849, (0.0, inf)\n",
      "HMR_0850, (0.0, inf)\n",
      "HMR_0855, (0.0, inf)\n",
      "HMR_0856, (0.0, inf)\n",
      "HMR_0803, (0.0, inf)\n",
      "HMR_0807, (0.0, inf)\n",
      "HMR_0809, (0.0, inf)\n",
      "HMR_0810, (0.0, inf)\n",
      "HMR_0811, (0.0, inf)\n",
      "HMR_0812, (0.0, inf)\n",
      "HMR_0859, (0.0, inf)\n",
      "HMR_0875, (0.0, inf)\n",
      "HMR_0876, (0.0, inf)\n",
      "HMR_0889, (0.0, inf)\n",
      "HMR_0890, (0.0, inf)\n",
      "HMR_0891, (0.0, inf)\n",
      "HMR_0892, (0.0, inf)\n",
      "HMR_0893, (0.0, inf)\n",
      "HMR_0894, (0.0, inf)\n",
      "HMR_0895, (0.0, inf)\n",
      "HMR_0898, (0.0, inf)\n",
      "HMR_0899, (0.0, inf)\n",
      "HMR_0900, (0.0, inf)\n",
      "HMR_0906, (0.0, inf)\n",
      "HMR_0910, (0.0, inf)\n",
      "HMR_0911, (0.0, inf)\n",
      "HMR_0787, (0.0, inf)\n",
      "HMR_0790, (0.0, inf)\n",
      "HMR_0792, (0.0, inf)\n",
      "HMR_0794, (0.0, inf)\n",
      "HMR_0797, (0.0, inf)\n",
      "HMR_0804, (0.0, inf)\n",
      "HMR_0808, (0.0, inf)\n",
      "HMR_0826, (0.0, inf)\n",
      "HMR_0832, (0.0, inf)\n",
      "HMR_0835, (0.0, inf)\n",
      "HMR_0836, (0.0, inf)\n",
      "HMR_0920, (0.0, inf)\n",
      "HMR_0924, (0.0, inf)\n",
      "HMR_0925, (0.0, inf)\n",
      "HMR_0926, (0.0, inf)\n",
      "HMR_8379, (0.0, inf)\n",
      "HMR_7185, (0.0, inf)\n",
      "HMR_8380, (0.0, inf)\n",
      "HMR_7186, (0.0, inf)\n",
      "HMR_8381, (0.0, inf)\n",
      "HMR_7187, (0.0, inf)\n",
      "HMR_8383, (0.0, inf)\n",
      "HMR_8384, (0.0, inf)\n",
      "HMR_8385, (0.0, inf)\n",
      "HMR_8387, (0.0, inf)\n",
      "HMR_8388, (0.0, inf)\n",
      "HMR_8389, (0.0, inf)\n",
      "HMR_8390, (0.0, inf)\n",
      "HMR_8391, (0.0, inf)\n",
      "HMR_8392, (0.0, inf)\n",
      "HMR_8393, (0.0, inf)\n",
      "HMR_8394, (0.0, inf)\n",
      "HMR_8395, (0.0, inf)\n",
      "HMR_8396, (0.0, inf)\n",
      "HMR_8397, (0.0, inf)\n",
      "HMR_8398, (0.0, inf)\n",
      "HMR_8399, (0.0, inf)\n",
      "HMR_8401, (0.0, inf)\n",
      "HMR_8402, (0.0, inf)\n",
      "HMR_8403, (0.0, inf)\n",
      "HMR_8404, (0.0, inf)\n",
      "HMR_8405, (0.0, inf)\n",
      "HMR_8406, (0.0, inf)\n",
      "HMR_8407, (0.0, inf)\n",
      "HMR_1981, (0.0, inf)\n",
      "HMR_1985, (0.0, inf)\n",
      "HMR_1988, (0.0, inf)\n",
      "HMR_1312, (0.0, inf)\n",
      "HMR_1313, (0.0, inf)\n",
      "HMR_1325, (0.0, inf)\n",
      "HMR_1326, (0.0, inf)\n",
      "HMR_1327, (0.0, inf)\n",
      "HMR_0703, (0.0, inf)\n",
      "HMR_7598, (0.0, inf)\n",
      "HMR_7600, (0.0, inf)\n",
      "HMR_7602, (0.0, inf)\n",
      "HMR_7603, (0.0, inf)\n",
      "HMR_7614, (0.0, inf)\n",
      "HMR_7615, (0.0, inf)\n",
      "HMR_0025, (0.0, inf)\n",
      "HMR_0030, (0.0, inf)\n",
      "HMR_7200, (0.0, inf)\n",
      "HMR_7202, (0.0, inf)\n",
      "HMR_7203, (0.0, inf)\n",
      "HMR_7205, (0.0, inf)\n",
      "HMR_7206, (0.0, inf)\n",
      "HMR_7207, (0.0, inf)\n",
      "HMR_7208, (0.0, inf)\n",
      "HMR_7209, (0.0, inf)\n",
      "HMR_7210, (0.0, inf)\n",
      "HMR_7211, (0.0, inf)\n",
      "HMR_7212, (0.0, inf)\n",
      "HMR_7213, (0.0, inf)\n",
      "HMR_7214, (0.0, inf)\n",
      "HMR_7215, (0.0, inf)\n",
      "HMR_7216, (0.0, inf)\n",
      "HMR_7217, (0.0, inf)\n",
      "HMR_7218, (0.0, inf)\n",
      "HMR_7219, (0.0, inf)\n",
      "HMR_7491, (0.0, inf)\n",
      "HMR_7492, (0.0, inf)\n",
      "HMR_7493, (0.0, inf)\n",
      "HMR_7494, (0.0, inf)\n",
      "HMR_7495, (0.0, inf)\n",
      "HMR_7496, (0.0, inf)\n",
      "HMR_7497, (0.0, inf)\n",
      "HMR_7498, (0.0, inf)\n",
      "HMR_7509, (0.0, inf)\n",
      "HMR_7519, (0.0, inf)\n",
      "HMR_7520, (0.0, inf)\n",
      "HMR_7521, (0.0, inf)\n",
      "HMR_7522, (0.0, inf)\n",
      "HMR_7534, (0.0, inf)\n",
      "HMR_7535, (0.0, inf)\n",
      "HMR_7536, (0.0, inf)\n",
      "HMR_7537, (0.0, inf)\n",
      "HMR_7551, (0.0, inf)\n",
      "HMR_7552, (0.0, inf)\n",
      "HMR_7553, (0.0, inf)\n",
      "HMR_7501, (0.0, inf)\n",
      "HMR_7502, (0.0, inf)\n",
      "HMR_7503, (0.0, inf)\n",
      "HMR_7504, (0.0, inf)\n",
      "HMR_7505, (0.0, inf)\n",
      "HMR_7506, (0.0, inf)\n",
      "HMR_7507, (0.0, inf)\n",
      "HMR_7513, (0.0, inf)\n",
      "HMR_7514, (0.0, inf)\n",
      "HMR_7515, (0.0, inf)\n",
      "HMR_7516, (0.0, inf)\n",
      "HMR_7517, (0.0, inf)\n",
      "HMR_7525, (0.0, inf)\n",
      "HMR_7526, (0.0, inf)\n",
      "HMR_7527, (0.0, inf)\n",
      "HMR_7528, (0.0, inf)\n",
      "HMR_7529, (0.0, inf)\n",
      "HMR_7530, (0.0, inf)\n",
      "HMR_7531, (0.0, inf)\n",
      "HMR_7532, (0.0, inf)\n",
      "HMR_7541, (0.0, inf)\n",
      "HMR_7542, (0.0, inf)\n",
      "HMR_7543, (0.0, inf)\n",
      "HMR_7544, (0.0, inf)\n",
      "HMR_7545, (0.0, inf)\n",
      "HMR_7546, (0.0, inf)\n",
      "HMR_7547, (0.0, inf)\n",
      "HMR_7548, (0.0, inf)\n",
      "HMR_7549, (0.0, inf)\n",
      "HMR_7557, (0.0, inf)\n",
      "HMR_7558, (0.0, inf)\n",
      "HMR_7559, (0.0, inf)\n",
      "HMR_7560, (0.0, inf)\n",
      "HMR_7561, (0.0, inf)\n",
      "HMR_7562, (0.0, inf)\n",
      "HMR_7563, (0.0, inf)\n",
      "HMR_7564, (0.0, inf)\n",
      "HMR_7565, (0.0, inf)\n",
      "HMR_7566, (0.0, inf)\n",
      "HMR_7225, (0.0, inf)\n",
      "HMR_7226, (0.0, inf)\n",
      "HMR_7227, (0.0, inf)\n",
      "HMR_7228, (0.0, inf)\n",
      "HMR_7229, (0.0, inf)\n",
      "HMR_7230, (0.0, inf)\n",
      "HMR_7231, (0.0, inf)\n",
      "HMR_7232, (0.0, inf)\n",
      "HMR_7233, (0.0, inf)\n",
      "HMR_7234, (0.0, inf)\n",
      "HMR_7235, (0.0, inf)\n",
      "HMR_7236, (0.0, inf)\n",
      "HMR_7237, (0.0, inf)\n",
      "HMR_7238, (0.0, inf)\n",
      "HMR_7239, (0.0, inf)\n",
      "HMR_7240, (0.0, inf)\n",
      "HMR_7241, (0.0, inf)\n",
      "HMR_7242, (0.0, inf)\n",
      "HMR_7243, (0.0, inf)\n",
      "HMR_7244, (0.0, inf)\n",
      "HMR_7245, (0.0, inf)\n",
      "HMR_7246, (0.0, inf)\n",
      "HMR_7247, (0.0, inf)\n",
      "HMR_7248, (0.0, inf)\n",
      "HMR_7249, (0.0, inf)\n",
      "HMR_7250, (0.0, inf)\n",
      "HMR_7571, (0.0, inf)\n",
      "HMR_7572, (0.0, inf)\n",
      "HMR_7573, (0.0, inf)\n",
      "HMR_7335, (0.0, inf)\n",
      "HMR_7336, (0.0, inf)\n",
      "HMR_7337, (0.0, inf)\n",
      "HMR_7338, (0.0, inf)\n",
      "HMR_7339, (0.0, inf)\n",
      "HMR_7340, (0.0, inf)\n",
      "HMR_7341, (0.0, inf)\n",
      "HMR_7342, (0.0, inf)\n",
      "HMR_7343, (0.0, inf)\n",
      "HMR_7344, (0.0, inf)\n",
      "HMR_7345, (0.0, inf)\n",
      "HMR_7346, (0.0, inf)\n",
      "HMR_7347, (0.0, inf)\n",
      "HMR_7348, (0.0, inf)\n",
      "HMR_7349, (0.0, inf)\n",
      "HMR_7350, (0.0, inf)\n",
      "HMR_7351, (0.0, inf)\n",
      "HMR_7352, (0.0, inf)\n",
      "HMR_7353, (0.0, inf)\n",
      "HMR_7354, (0.0, inf)\n",
      "HMR_7355, (0.0, inf)\n",
      "HMR_7356, (0.0, inf)\n",
      "HMR_7357, (0.0, inf)\n",
      "HMR_7358, (0.0, inf)\n",
      "HMR_7359, (0.0, inf)\n",
      "HMR_7360, (0.0, inf)\n",
      "HMR_7361, (0.0, inf)\n",
      "HMR_7362, (0.0, inf)\n",
      "HMR_7363, (0.0, inf)\n",
      "HMR_7364, (0.0, inf)\n",
      "HMR_7365, (0.0, inf)\n",
      "HMR_7366, (0.0, inf)\n",
      "HMR_7367, (0.0, inf)\n",
      "HMR_7368, (0.0, inf)\n",
      "HMR_7369, (0.0, inf)\n",
      "HMR_7370, (0.0, inf)\n",
      "HMR_7371, (0.0, inf)\n",
      "HMR_7442, (0.0, inf)\n",
      "HMR_7443, (0.0, inf)\n",
      "HMR_7444, (0.0, inf)\n",
      "HMR_7445, (0.0, inf)\n",
      "HMR_7446, (0.0, inf)\n",
      "HMR_7447, (0.0, inf)\n",
      "HMR_7448, (0.0, inf)\n",
      "HMR_7449, (0.0, inf)\n",
      "HMR_7450, (0.0, inf)\n",
      "HMR_7451, (0.0, inf)\n",
      "HMR_7452, (0.0, inf)\n",
      "HMR_7471, (0.0, inf)\n",
      "HMR_7472, (0.0, inf)\n",
      "HMR_7473, (0.0, inf)\n",
      "HMR_7474, (0.0, inf)\n",
      "HMR_7475, (0.0, inf)\n",
      "HMR_7476, (0.0, inf)\n",
      "HMR_7477, (0.0, inf)\n",
      "HMR_7478, (0.0, inf)\n",
      "HMR_7479, (0.0, inf)\n",
      "HMR_7480, (0.0, inf)\n",
      "HMR_7481, (0.0, inf)\n",
      "HMR_7373, (0.0, inf)\n",
      "HMR_7375, (0.0, inf)\n",
      "HMR_7376, (0.0, inf)\n",
      "HMR_7377, (0.0, inf)\n",
      "HMR_7378, (0.0, inf)\n",
      "HMR_7379, (0.0, inf)\n",
      "HMR_7380, (0.0, inf)\n",
      "HMR_7381, (0.0, inf)\n",
      "HMR_7382, (0.0, inf)\n",
      "HMR_7383, (0.0, inf)\n",
      "HMR_7384, (0.0, inf)\n",
      "HMR_7385, (0.0, inf)\n",
      "HMR_7386, (0.0, inf)\n",
      "HMR_7387, (0.0, inf)\n",
      "HMR_7388, (0.0, inf)\n",
      "HMR_7389, (0.0, inf)\n",
      "HMR_7390, (0.0, inf)\n",
      "HMR_7391, (0.0, inf)\n",
      "HMR_7392, (0.0, inf)\n",
      "HMR_7393, (0.0, inf)\n",
      "HMR_7394, (0.0, inf)\n",
      "HMR_7395, (0.0, inf)\n",
      "HMR_7396, (0.0, inf)\n",
      "HMR_7397, (0.0, inf)\n",
      "HMR_7398, (0.0, inf)\n",
      "HMR_7399, (0.0, inf)\n",
      "HMR_7400, (0.0, inf)\n",
      "HMR_7401, (0.0, inf)\n",
      "HMR_7402, (0.0, inf)\n",
      "HMR_7403, (0.0, inf)\n",
      "HMR_7404, (0.0, inf)\n",
      "HMR_7405, (0.0, inf)\n",
      "HMR_7406, (0.0, inf)\n",
      "HMR_7407, (0.0, inf)\n",
      "HMR_7408, (0.0, inf)\n",
      "HMR_7409, (0.0, inf)\n",
      "HMR_7410, (0.0, inf)\n",
      "HMR_7411, (0.0, inf)\n",
      "HMR_7412, (0.0, inf)\n",
      "HMR_7413, (0.0, inf)\n",
      "HMR_7414, (0.0, inf)\n",
      "HMR_7415, (0.0, inf)\n",
      "HMR_7416, (0.0, inf)\n",
      "HMR_7417, (0.0, inf)\n",
      "HMR_7418, (0.0, inf)\n",
      "HMR_7419, (0.0, inf)\n",
      "HMR_7420, (0.0, inf)\n",
      "HMR_7421, (0.0, inf)\n",
      "HMR_7422, (0.0, inf)\n",
      "HMR_7423, (0.0, inf)\n",
      "HMR_7424, (0.0, inf)\n",
      "HMR_7425, (0.0, inf)\n",
      "HMR_7426, (0.0, inf)\n",
      "HMR_7427, (0.0, inf)\n",
      "HMR_7455, (0.0, inf)\n",
      "HMR_7456, (0.0, inf)\n",
      "HMR_7457, (0.0, inf)\n",
      "HMR_7458, (0.0, inf)\n",
      "HMR_7459, (0.0, inf)\n",
      "HMR_7460, (0.0, inf)\n",
      "HMR_7461, (0.0, inf)\n",
      "HMR_7462, (0.0, inf)\n",
      "HMR_7463, (0.0, inf)\n",
      "HMR_7464, (0.0, inf)\n",
      "HMR_7465, (0.0, inf)\n",
      "HMR_7466, (0.0, inf)\n",
      "HMR_7467, (0.0, inf)\n",
      "HMR_7468, (0.0, inf)\n",
      "HMR_7469, (0.0, inf)\n",
      "HMR_7484, (0.0, inf)\n",
      "HMR_7485, (0.0, inf)\n",
      "HMR_7486, (0.0, inf)\n",
      "HMR_7487, (0.0, inf)\n",
      "HMR_7488, (0.0, inf)\n",
      "HMR_7489, (0.0, inf)\n",
      "HMR_1581, (0.0, inf)\n",
      "HMR_1584, (0.0, inf)\n",
      "HMR_1585, (0.0, inf)\n",
      "HMR_1590, (0.0, inf)\n",
      "HMR_1595, (0.0, inf)\n",
      "HMR_1598, (0.0, inf)\n",
      "HMR_1599, (0.0, inf)\n",
      "HMR_1604, (0.0, inf)\n",
      "HMR_1605, (0.0, inf)\n",
      "HMR_1608, (0.0, inf)\n",
      "HMR_1609, (0.0, inf)\n",
      "HMR_1610, (0.0, inf)\n",
      "HMR_1611, (0.0, inf)\n",
      "HMR_1613, (0.0, inf)\n",
      "HMR_1614, (0.0, inf)\n",
      "HMR_1624, (0.0, inf)\n",
      "HMR_1625, (0.0, inf)\n",
      "HMR_1627, (0.0, inf)\n",
      "HMR_1632, (0.0, inf)\n",
      "HMR_1635, (0.0, inf)\n",
      "HMR_1637, (0.0, inf)\n",
      "HMR_1638, (0.0, inf)\n",
      "HMR_1639, (0.0, inf)\n",
      "HMR_1642, (0.0, inf)\n",
      "HMR_1646, (0.0, inf)\n",
      "HMR_1652, (0.0, inf)\n",
      "HMR_1653, (0.0, inf)\n",
      "HMR_1659, (0.0, inf)\n",
      "HMR_1660, (0.0, inf)\n",
      "HMR_1662, (0.0, inf)\n",
      "HMR_1663, (0.0, inf)\n",
      "HMR_1665, (0.0, inf)\n",
      "HMR_1666, (0.0, inf)\n",
      "HMR_1673, (0.0, inf)\n",
      "HMR_1676, (0.0, inf)\n",
      "HMR_1678, (0.0, inf)\n",
      "HMR_1681, (0.0, inf)\n",
      "HMR_1682, (0.0, inf)\n",
      "HMR_1685, (0.0, inf)\n",
      "HMR_1687, (0.0, inf)\n",
      "HMR_1689, (0.0, inf)\n",
      "HMR_1691, (0.0, inf)\n",
      "HMR_1693, (0.0, inf)\n",
      "HMR_1697, (0.0, inf)\n",
      "HMR_1699, (0.0, inf)\n",
      "HMR_1700, (0.0, inf)\n",
      "HMR_1704, (0.0, inf)\n",
      "HMR_1706, (0.0, inf)\n",
      "HMR_1708, (0.0, inf)\n",
      "HMR_1710, (0.0, inf)\n",
      "HMR_1718, (0.0, inf)\n",
      "HMR_1729, (0.0, inf)\n",
      "HMR_1730, (0.0, inf)\n",
      "HMR_1735, (0.0, inf)\n",
      "HMR_1737, (0.0, inf)\n",
      "HMR_1738, (0.0, inf)\n",
      "HMR_1739, (0.0, inf)\n",
      "HMR_1740, (0.0, inf)\n",
      "HMR_1741, (0.0, inf)\n",
      "HMR_1742, (0.0, inf)\n",
      "HMR_1743, (0.0, inf)\n",
      "HMR_1744, (0.0, inf)\n",
      "HMR_1745, (0.0, inf)\n",
      "HMR_1746, (0.0, inf)\n",
      "HMR_1747, (0.0, inf)\n",
      "HMR_1748, (0.0, inf)\n",
      "HMR_1749, (0.0, inf)\n",
      "HMR_1750, (0.0, inf)\n",
      "HMR_1751, (0.0, inf)\n",
      "HMR_1752, (0.0, inf)\n",
      "HMR_1753, (0.0, inf)\n",
      "HMR_1754, (0.0, inf)\n",
      "HMR_1756, (0.0, inf)\n",
      "HMR_1758, (0.0, inf)\n",
      "HMR_1759, (0.0, inf)\n",
      "HMR_1760, (0.0, inf)\n",
      "HMR_1761, (0.0, inf)\n",
      "HMR_1762, (0.0, inf)\n",
      "HMR_1766, (0.0, inf)\n",
      "HMR_1767, (0.0, inf)\n",
      "HMR_1768, (0.0, inf)\n",
      "HMR_1769, (0.0, inf)\n",
      "HMR_1770, (0.0, inf)\n",
      "HMR_1771, (0.0, inf)\n",
      "HMR_1784, (0.0, inf)\n",
      "HMR_1786, (0.0, inf)\n",
      "HMR_1794, (0.0, inf)\n",
      "HMR_1796, (0.0, inf)\n",
      "HMR_1797, (0.0, inf)\n",
      "HMR_1798, (0.0, inf)\n",
      "HMR_1800, (0.0, inf)\n",
      "HMR_1805, (0.0, inf)\n",
      "HMR_1810, (0.0, inf)\n",
      "HMR_1811, (0.0, inf)\n",
      "HMR_1846, (0.0, inf)\n",
      "HMR_3799, (0.0, inf)\n",
      "HMR_3908, (0.0, inf)\n",
      "HMR_3910, (0.0, inf)\n",
      "HMR_8065, (0.0, inf)\n",
      "HMR_1847, (0.0, inf)\n",
      "HMR_1849, (0.0, inf)\n",
      "HMR_1853, (0.0, inf)\n",
      "HMR_1868, (0.0, inf)\n",
      "HMR_1870, (0.0, inf)\n",
      "HMR_1872, (0.0, inf)\n",
      "HMR_1874, (0.0, inf)\n",
      "HMR_1878, (0.0, inf)\n",
      "HMR_1895, (0.0, inf)\n",
      "HMR_1896, (0.0, inf)\n",
      "HMR_1897, (0.0, inf)\n",
      "HMR_8262, (0.0, inf)\n",
      "HMR_8263, (0.0, inf)\n",
      "HMR_8264, (0.0, inf)\n",
      "HMR_8267, (0.0, inf)\n",
      "HMR_8268, (0.0, inf)\n",
      "HMR_8271, (0.0, inf)\n",
      "HMR_8274, (0.0, inf)\n",
      "HMR_8275, (0.0, inf)\n",
      "HMR_8276, (0.0, inf)\n",
      "HMR_8277, (0.0, inf)\n",
      "HMR_8278, (0.0, inf)\n",
      "HMR_8279, (0.0, inf)\n",
      "HMR_8280, (0.0, inf)\n",
      "HMR_8284, (0.0, inf)\n",
      "HMR_8285, (0.0, inf)\n",
      "HMR_8318, (0.0, inf)\n",
      "HMR_8325, (0.0, inf)\n",
      "HMR_0712, (0.0, inf)\n",
      "HMR_4228, (0.0, inf)\n",
      "HMR_4251, (0.0, inf)\n",
      "HMR_4252, (0.0, inf)\n",
      "HMR_4255, (0.0, inf)\n",
      "HMR_4260, (0.0, inf)\n",
      "HMR_4261, (0.0, inf)\n",
      "HMR_4263, (0.0, inf)\n",
      "HMR_4264, (0.0, inf)\n",
      "HMR_4265, (0.0, inf)\n",
      "HMR_4269, (0.0, inf)\n",
      "HMR_4270, (0.0, inf)\n",
      "HMR_4271, (0.0, inf)\n",
      "HMR_4278, (0.0, inf)\n",
      "HMR_4662, (0.0, inf)\n",
      "HMR_7623, (0.0, inf)\n",
      "HMR_7625, (0.0, inf)\n",
      "HMR_8790, (0.0, inf)\n",
      "HMR_3871, (0.0, inf)\n",
      "HMR_4058, (0.0, inf)\n",
      "HMR_4498, (0.0, inf)\n",
      "HMR_4500, (0.0, inf)\n",
      "HMR_4714, (0.0, inf)\n",
      "HMR_4715, (0.0, inf)\n",
      "HMR_4716, (0.0, inf)\n",
      "HMR_4718, (0.0, inf)\n",
      "HMR_4723, (0.0, inf)\n",
      "HMR_4725, (0.0, inf)\n",
      "HMR_4732, (0.0, inf)\n",
      "HMR_4733, (0.0, inf)\n",
      "HMR_4734, (0.0, inf)\n",
      "HMR_7731, (0.0, inf)\n",
      "HMR_0663, (0.0, inf)\n",
      "HMR_4308, (0.0, inf)\n",
      "HMR_6539, (0.0, inf)\n",
      "HMR_6540, (0.0, inf)\n",
      "HMR_6542, (0.0, inf)\n",
      "HMR_6543, (0.0, inf)\n",
      "HMR_6544, (0.0, inf)\n",
      "HMR_6545, (0.0, inf)\n",
      "HMR_6546, (0.0, inf)\n",
      "HMR_6547, (0.0, inf)\n",
      "HMR_6548, (0.0, inf)\n",
      "HMR_6549, (0.0, inf)\n",
      "HMR_6550, (0.0, inf)\n",
      "HMR_6551, (0.0, inf)\n",
      "HMR_6552, (0.0, inf)\n",
      "HMR_6553, (0.0, inf)\n",
      "HMR_6554, (0.0, inf)\n",
      "HMR_6555, (0.0, inf)\n",
      "HMR_6556, (0.0, inf)\n",
      "HMR_6557, (0.0, inf)\n",
      "HMR_6558, (0.0, inf)\n",
      "HMR_6559, (0.0, inf)\n",
      "HMR_6560, (0.0, inf)\n",
      "HMR_6561, (0.0, inf)\n",
      "HMR_6562, (0.0, inf)\n",
      "HMR_6563, (0.0, inf)\n",
      "HMR_6564, (0.0, inf)\n",
      "HMR_6565, (0.0, inf)\n",
      "HMR_6566, (0.0, inf)\n",
      "HMR_6567, (0.0, inf)\n",
      "HMR_6568, (0.0, inf)\n",
      "HMR_6569, (0.0, inf)\n",
      "HMR_6570, (0.0, inf)\n",
      "HMR_6571, (0.0, inf)\n",
      "HMR_6572, (0.0, inf)\n",
      "HMR_6573, (0.0, inf)\n",
      "HMR_6574, (0.0, inf)\n",
      "HMR_6576, (0.0, inf)\n",
      "HMR_6579, (0.0, inf)\n",
      "HMR_6581, (0.0, inf)\n",
      "HMR_6583, (0.0, inf)\n",
      "HMR_6585, (0.0, inf)\n",
      "HMR_6587, (0.0, inf)\n",
      "HMR_6591, (0.0, inf)\n",
      "HMR_6592, (0.0, inf)\n",
      "HMR_6595, (0.0, inf)\n",
      "HMR_7652, (0.0, inf)\n",
      "HMR_7654, (0.0, inf)\n",
      "HMR_7655, (0.0, inf)\n",
      "HMR_7656, (0.0, inf)\n",
      "HMR_8799, (0.0, inf)\n",
      "HMR_8800, (0.0, inf)\n",
      "HMR_8803, (0.0, inf)\n",
      "HMR_8804, (0.0, inf)\n",
      "HMR_8805, (0.0, inf)\n",
      "HMR_8807, (0.0, inf)\n",
      "HMR_8813, (0.0, inf)\n",
      "HMR_8815, (0.0, inf)\n",
      "HMR_8816, (0.0, inf)\n",
      "HMR_8818, (0.0, inf)\n",
      "HMR_8821, (0.0, inf)\n",
      "HMR_8822, (0.0, inf)\n",
      "HMR_8823, (0.0, inf)\n",
      "HMR_8824, (0.0, inf)\n",
      "HMR_8830, (0.0, inf)\n",
      "HMR_8832, (0.0, inf)\n",
      "HMR_8833, (0.0, inf)\n",
      "HMR_8835, (0.0, inf)\n",
      "HMR_8836, (0.0, inf)\n",
      "HMR_3923, (0.0, inf)\n",
      "HMR_4336, (0.0, inf)\n",
      "HMR_4338, (0.0, inf)\n",
      "HMR_4340, (0.0, inf)\n",
      "HMR_4656, (0.0, inf)\n",
      "HMR_4665, (0.0, inf)\n",
      "HMR_9726, (0.0, inf)\n",
      "HMR_4666, (0.0, inf)\n",
      "HMR_7145, (0.0, inf)\n",
      "HMR_7146, (0.0, inf)\n",
      "HMR_7147, (0.0, inf)\n",
      "HMR_7908, (0.0, inf)\n",
      "HMR_7909, (0.0, inf)\n",
      "HMR_7910, (0.0, inf)\n",
      "HMR_7911, (0.0, inf)\n",
      "HMR_7912, (0.0, inf)\n",
      "HMR_7913, (0.0, inf)\n",
      "HMR_7916, (0.0, inf)\n",
      "HMR_7919, (0.0, inf)\n",
      "HMR_7920, (0.0, inf)\n",
      "HMR_7921, (0.0, inf)\n",
      "HMR_7922, (0.0, inf)\n",
      "HMR_7925, (0.0, inf)\n",
      "HMR_8105, (0.0, inf)\n",
      "HMR_8106, (0.0, inf)\n",
      "HMR_8107, (0.0, inf)\n",
      "HMR_8108, (0.0, inf)\n",
      "HMR_8109, (0.0, inf)\n",
      "HMR_8110, (0.0, inf)\n",
      "HMR_8112, (0.0, inf)\n",
      "HMR_8113, (0.0, inf)\n",
      "HMR_8114, (0.0, inf)\n",
      "HMR_8115, (0.0, inf)\n",
      "HMR_8116, (0.0, inf)\n",
      "HMR_8117, (0.0, inf)\n",
      "HMR_8129, (0.0, inf)\n",
      "HMR_8130, (0.0, inf)\n",
      "HMR_8132, (0.0, inf)\n",
      "HMR_8133, (0.0, inf)\n",
      "HMR_8135, (0.0, inf)\n",
      "HMR_8136, (0.0, inf)\n",
      "HMR_8137, (0.0, inf)\n",
      "HMR_8138, (0.0, inf)\n",
      "HMR_8139, (0.0, inf)\n",
      "HMR_8140, (0.0, inf)\n",
      "HMR_8141, (0.0, inf)\n",
      "HMR_8142, (0.0, inf)\n",
      "HMR_8143, (0.0, inf)\n",
      "HMR_8758, (0.0, inf)\n",
      "HMR_7661, (0.0, inf)\n",
      "HMR_7662, (0.0, inf)\n",
      "HMR_7663, (0.0, inf)\n",
      "HMR_7668, (0.0, inf)\n",
      "HMR_7669, (0.0, inf)\n",
      "HMR_7670, (0.0, inf)\n",
      "HMR_7671, (0.0, inf)\n",
      "HMR_7672, (0.0, inf)\n",
      "HMR_7673, (0.0, inf)\n",
      "HMR_4160, (0.0, inf)\n",
      "HMR_4162, (0.0, inf)\n",
      "HMR_4163, (0.0, inf)\n",
      "HMR_4166, (0.0, inf)\n",
      "HMR_4167, (0.0, inf)\n",
      "HMR_4169, (0.0, inf)\n",
      "HMR_4170, (0.0, inf)\n",
      "HMR_4816, (0.0, inf)\n",
      "HMR_8538, (0.0, inf)\n",
      "HMR_8738, (0.0, inf)\n",
      "HMR_8739, (0.0, inf)\n",
      "HMR_6396, (0.0, inf)\n",
      "HMR_3991, (0.0, inf)\n",
      "HMR_4657, (0.0, inf)\n",
      "HMR_4744, (0.0, inf)\n",
      "HMR_4746, (0.0, inf)\n",
      "HMR_4748, (0.0, inf)\n",
      "HMR_4750, (0.0, inf)\n",
      "HMR_4752, (0.0, inf)\n",
      "HMR_4755, (0.0, inf)\n",
      "HMR_4757, (0.0, inf)\n",
      "HMR_4758, (0.0, inf)\n",
      "HMR_4762, (0.0, inf)\n",
      "HMR_4772, (0.0, inf)\n",
      "HMR_6630, (0.0, inf)\n",
      "HMR_6631, (0.0, inf)\n",
      "HMR_6632, (0.0, inf)\n",
      "HMR_6633, (0.0, inf)\n",
      "HMR_6634, (0.0, inf)\n",
      "HMR_6635, (0.0, inf)\n",
      "HMR_6636, (0.0, inf)\n",
      "HMR_6639, (0.0, inf)\n",
      "HMR_6641, (0.0, inf)\n",
      "HMR_6642, (0.0, inf)\n",
      "HMR_6643, (0.0, inf)\n",
      "HMR_6645, (0.0, inf)\n",
      "HMR_6646, (0.0, inf)\n",
      "HMR_6647, (0.0, inf)\n",
      "HMR_6648, (0.0, inf)\n",
      "HMR_6649, (0.0, inf)\n",
      "HMR_6650, (0.0, inf)\n",
      "HMR_6651, (0.0, inf)\n",
      "HMR_6653, (0.0, inf)\n",
      "HMR_6654, (0.0, inf)\n",
      "HMR_6655, (0.0, inf)\n",
      "HMR_6656, (0.0, inf)\n",
      "HMR_6657, (0.0, inf)\n",
      "HMR_6658, (0.0, inf)\n",
      "HMR_6664, (0.0, inf)\n",
      "HMR_6665, (0.0, inf)\n",
      "HMR_6669, (0.0, inf)\n",
      "HMR_6677, (0.0, inf)\n",
      "HMR_6678, (0.0, inf)\n",
      "HMR_6679, (0.0, inf)\n",
      "HMR_6680, (0.0, inf)\n",
      "HMR_6682, (0.0, inf)\n",
      "HMR_6687, (0.0, inf)\n",
      "HMR_6700, (0.0, inf)\n",
      "HMR_6701, (0.0, inf)\n",
      "HMR_6702, (0.0, inf)\n",
      "HMR_6703, (0.0, inf)\n",
      "HMR_6506, (0.0, inf)\n",
      "HMR_6507, (0.0, inf)\n",
      "HMR_6508, (0.0, inf)\n",
      "HMR_6509, (0.0, inf)\n",
      "HMR_6511, (0.0, inf)\n",
      "HMR_4537, (0.0, inf)\n",
      "HMR_4545, (0.0, inf)\n",
      "HMR_4558, (0.0, inf)\n",
      "HMR_4559, (0.0, inf)\n",
      "HMR_4560, (0.0, inf)\n",
      "HMR_4204, (0.0, inf)\n",
      "HMR_4206, (0.0, inf)\n",
      "HMR_4207, (0.0, inf)\n",
      "HMR_4208, (0.0, inf)\n",
      "HMR_8613, (0.0, inf)\n",
      "HMR_4064, (0.0, inf)\n",
      "HMR_4065, (0.0, inf)\n",
      "HMR_4066, (0.0, inf)\n",
      "HMR_4067, (0.0, inf)\n",
      "HMR_4068, (0.0, inf)\n",
      "HMR_4069, (0.0, inf)\n",
      "HMR_4070, (0.0, inf)\n",
      "HMR_8102, (0.0, inf)\n",
      "HMR_8724, (0.0, inf)\n",
      "HMR_8725, (0.0, inf)\n",
      "HMR_2114, (0.0, inf)\n",
      "HMR_2117, (0.0, inf)\n",
      "HMR_2128, (0.0, inf)\n",
      "HMR_2129, (0.0, inf)\n",
      "HMR_2131, (0.0, inf)\n",
      "HMR_2132, (0.0, inf)\n",
      "HMR_2133, (0.0, inf)\n",
      "HMR_2134, (0.0, inf)\n",
      "HMR_2135, (0.0, inf)\n",
      "HMR_2136, (0.0, inf)\n",
      "HMR_2137, (0.0, inf)\n",
      "HMR_2138, (0.0, inf)\n",
      "HMR_2139, (0.0, inf)\n",
      "HMR_2140, (0.0, inf)\n",
      "HMR_2142, (0.0, inf)\n",
      "HMR_2143, (0.0, inf)\n",
      "HMR_2144, (0.0, inf)\n",
      "HMR_2145, (0.0, inf)\n",
      "HMR_7996, (0.0, inf)\n",
      "HMR_7999, (0.0, inf)\n",
      "HMR_8008, (0.0, inf)\n",
      "HMR_6426, (0.0, inf)\n",
      "HMR_6427, (0.0, inf)\n",
      "HMR_6428, (0.0, inf)\n",
      "HMR_6429, (0.0, inf)\n",
      "HMR_6432, (0.0, inf)\n",
      "HMR_6433, (0.0, inf)\n",
      "HMR_6434, (0.0, inf)\n",
      "HMR_6435, (0.0, inf)\n",
      "HMR_6436, (0.0, inf)\n",
      "HMR_6441, (0.0, inf)\n",
      "HMR_6442, (0.0, inf)\n",
      "HMR_6443, (0.0, inf)\n",
      "HMR_6444, (0.0, inf)\n",
      "HMR_6447, (0.0, inf)\n",
      "HMR_6448, (0.0, inf)\n",
      "HMR_6450, (0.0, inf)\n",
      "HMR_6451, (0.0, inf)\n",
      "HMR_6453, (0.0, inf)\n",
      "HMR_6464, (0.0, inf)\n",
      "HMR_6465, (0.0, inf)\n",
      "HMR_6466, (0.0, inf)\n",
      "HMR_6467, (0.0, inf)\n",
      "HMR_6470, (0.0, inf)\n",
      "HMR_6471, (0.0, inf)\n",
      "HMR_6472, (0.0, inf)\n",
      "HMR_6473, (0.0, inf)\n",
      "HMR_6476, (0.0, inf)\n",
      "HMR_6477, (0.0, inf)\n",
      "HMR_6478, (0.0, inf)\n",
      "HMR_6479, (0.0, inf)\n",
      "HMR_6482, (0.0, inf)\n",
      "HMR_6484, (0.0, inf)\n",
      "HMR_6486, (0.0, inf)\n",
      "HMR_6490, (0.0, inf)\n",
      "HMR_6495, (0.0, inf)\n",
      "HMR_6496, (0.0, inf)\n",
      "HMR_6992, (0.0, inf)\n",
      "HMR_6993, (0.0, inf)\n",
      "HMR_6994, (0.0, inf)\n",
      "HMR_6997, (0.0, inf)\n",
      "HMR_6998, (0.0, inf)\n",
      "HMR_6999, (0.0, inf)\n",
      "HMR_7000, (0.0, inf)\n",
      "HMR_7001, (0.0, inf)\n",
      "HMR_7002, (0.0, inf)\n",
      "HMR_7003, (0.0, inf)\n",
      "HMR_7004, (0.0, inf)\n",
      "HMR_7005, (0.0, inf)\n",
      "HMR_7013, (0.0, inf)\n",
      "HMR_7014, (0.0, inf)\n",
      "HMR_7017, (0.0, inf)\n",
      "HMR_7021, (0.0, inf)\n",
      "HMR_7022, (0.0, inf)\n",
      "HMR_7023, (0.0, inf)\n",
      "HMR_7024, (0.0, inf)\n",
      "HMR_7025, (0.0, inf)\n",
      "HMR_7026, (0.0, inf)\n",
      "HMR_7027, (0.0, inf)\n",
      "HMR_7028, (0.0, inf)\n",
      "HMR_7029, (0.0, inf)\n",
      "HMR_7030, (0.0, inf)\n",
      "HMR_7033, (0.0, inf)\n",
      "HMR_7034, (0.0, inf)\n",
      "HMR_7035, (0.0, inf)\n",
      "HMR_7036, (0.0, inf)\n",
      "HMR_7038, (0.0, inf)\n",
      "HMR_7039, (0.0, inf)\n",
      "HMR_7040, (0.0, inf)\n",
      "HMR_7041, (0.0, inf)\n",
      "HMR_7042, (0.0, inf)\n",
      "HMR_7050, (0.0, inf)\n",
      "HMR_7052, (0.0, inf)\n",
      "HMR_7053, (0.0, inf)\n",
      "HMR_7054, (0.0, inf)\n",
      "HMR_7055, (0.0, inf)\n",
      "HMR_7056, (0.0, inf)\n",
      "HMR_7057, (0.0, inf)\n",
      "HMR_7058, (0.0, inf)\n",
      "HMR_7059, (0.0, inf)\n",
      "HMR_7060, (0.0, inf)\n",
      "HMR_7061, (0.0, inf)\n",
      "HMR_7065, (0.0, inf)\n",
      "HMR_7066, (0.0, inf)\n",
      "HMR_7078, (0.0, inf)\n",
      "HMR_7079, (0.0, inf)\n",
      "HMR_7080, (0.0, inf)\n",
      "HMR_7081, (0.0, inf)\n",
      "HMR_7082, (0.0, inf)\n",
      "HMR_7083, (0.0, inf)\n",
      "HMR_7084, (0.0, inf)\n",
      "HMR_7088, (0.0, inf)\n",
      "HMR_7089, (0.0, inf)\n",
      "HMR_7099, (0.0, inf)\n",
      "HMR_7100, (0.0, inf)\n",
      "HMR_7688, (0.0, inf)\n",
      "HMR_8059, (0.0, inf)\n",
      "HMR_9569, (0.0, inf)\n",
      "HMR_3955, (0.0, inf)\n",
      "HMR_3984, (0.0, inf)\n",
      "HMR_4201, (0.0, inf)\n",
      "HMR_4202, (0.0, inf)\n",
      "HMR_8752, (0.0, inf)\n",
      "HMR_9488, (0.0, inf)\n",
      "HMR_9489, (0.0, inf)\n",
      "HMR_9493, (0.0, inf)\n",
      "HMR_9494, (0.0, inf)\n",
      "HMR_9516, (0.0, inf)\n",
      "HMR_9517, (0.0, inf)\n",
      "HMR_9577, (0.0, inf)\n",
      "HMR_9578, (0.0, inf)\n",
      "HMR_9582, (0.0, inf)\n",
      "HMR_9584, (0.0, inf)\n",
      "HMR_0010, (0.0, inf)\n",
      "HMR_0014, (0.0, inf)\n",
      "HMR_0031, (0.0, inf)\n",
      "HMR_0032, (0.0, inf)\n",
      "HMR_0545, (0.0, inf)\n",
      "HMR_0546, (0.0, inf)\n",
      "HMR_0547, (0.0, inf)\n",
      "HMR_0548, (0.0, inf)\n",
      "HMR_0549, (0.0, inf)\n",
      "HMR_0550, (0.0, inf)\n",
      "HMR_0551, (0.0, inf)\n",
      "HMR_0553, (0.0, inf)\n",
      "HMR_0556, (0.0, inf)\n",
      "HMR_0559, (0.0, inf)\n",
      "HMR_0560, (0.0, inf)\n",
      "HMR_0561, (0.0, inf)\n",
      "HMR_0689, (0.0, inf)\n",
      "HMR_5233, (0.0, inf)\n",
      "HMR_5238, (0.0, inf)\n",
      "HMR_5247, (0.0, inf)\n",
      "HMR_0004, (0.0, inf)\n",
      "HMR_0006, (0.0, inf)\n",
      "HMR_0007, (0.0, inf)\n",
      "HMR_0008, (0.0, inf)\n",
      "HMR_0009, (0.0, inf)\n",
      "HMR_0015, (0.0, inf)\n",
      "HMR_0019, (0.0, inf)\n",
      "HMR_0187, (0.0, inf)\n",
      "HMR_0190, (0.0, inf)\n",
      "HMR_0191, (0.0, inf)\n",
      "HMR_0194, (0.0, inf)\n",
      "HMR_0195, (0.0, inf)\n",
      "HMR_0198, (0.0, inf)\n",
      "HMR_0199, (0.0, inf)\n",
      "HMR_0202, (0.0, inf)\n",
      "HMR_0203, (0.0, inf)\n",
      "HMR_0207, (0.0, inf)\n",
      "HMR_0208, (0.0, inf)\n",
      "HMR_0211, (0.0, inf)\n",
      "HMR_0212, (0.0, inf)\n",
      "HMR_0215, (0.0, inf)\n",
      "HMR_0216, (0.0, inf)\n",
      "HMR_0224, (0.0, inf)\n",
      "HMR_0225, (0.0, inf)\n",
      "HMR_0231, (0.0, inf)\n",
      "HMR_0232, (0.0, inf)\n",
      "HMR_0235, (0.0, inf)\n",
      "HMR_0236, (0.0, inf)\n",
      "HMR_0239, (0.0, inf)\n",
      "HMR_0240, (0.0, inf)\n",
      "HMR_0243, (0.0, inf)\n",
      "HMR_0244, (0.0, inf)\n",
      "HMR_0247, (0.0, inf)\n",
      "HMR_0248, (0.0, inf)\n",
      "HMR_0253, (0.0, inf)\n",
      "HMR_0254, (0.0, inf)\n",
      "HMR_0257, (0.0, inf)\n",
      "HMR_0258, (0.0, inf)\n",
      "HMR_0261, (0.0, inf)\n",
      "HMR_0262, (0.0, inf)\n",
      "HMR_0265, (0.0, inf)\n",
      "HMR_0266, (0.0, inf)\n",
      "HMR_0269, (0.0, inf)\n",
      "HMR_0270, (0.0, inf)\n",
      "HMR_0273, (0.0, inf)\n",
      "HMR_0274, (0.0, inf)\n",
      "HMR_0277, (0.0, inf)\n",
      "HMR_0278, (0.0, inf)\n",
      "HMR_0281, (0.0, inf)\n",
      "HMR_0282, (0.0, inf)\n",
      "HMR_0287, (0.0, inf)\n",
      "HMR_0288, (0.0, inf)\n",
      "HMR_0291, (0.0, inf)\n",
      "HMR_0292, (0.0, inf)\n",
      "HMR_0295, (0.0, inf)\n",
      "HMR_0296, (0.0, inf)\n",
      "HMR_0299, (0.0, inf)\n",
      "HMR_0300, (0.0, inf)\n",
      "HMR_0303, (0.0, inf)\n",
      "HMR_0304, (0.0, inf)\n",
      "HMR_0307, (0.0, inf)\n",
      "HMR_0308, (0.0, inf)\n",
      "HMR_0311, (0.0, inf)\n",
      "HMR_0312, (0.0, inf)\n",
      "HMR_0317, (0.0, inf)\n",
      "HMR_0318, (0.0, inf)\n",
      "HMR_0321, (0.0, inf)\n",
      "HMR_0322, (0.0, inf)\n",
      "HMR_0325, (0.0, inf)\n",
      "HMR_0326, (0.0, inf)\n",
      "HMR_0329, (0.0, inf)\n",
      "HMR_0330, (0.0, inf)\n",
      "HMR_0335, (0.0, inf)\n",
      "HMR_0336, (0.0, inf)\n",
      "HMR_0339, (0.0, inf)\n",
      "HMR_0340, (0.0, inf)\n",
      "HMR_0343, (0.0, inf)\n",
      "HMR_0344, (0.0, inf)\n",
      "HMR_0347, (0.0, inf)\n",
      "HMR_0348, (0.0, inf)\n",
      "HMR_0351, (0.0, inf)\n",
      "HMR_0352, (0.0, inf)\n",
      "HMR_0355, (0.0, inf)\n",
      "HMR_0356, (0.0, inf)\n",
      "HMR_0359, (0.0, inf)\n",
      "HMR_0360, (0.0, inf)\n",
      "HMR_0363, (0.0, inf)\n",
      "HMR_0364, (0.0, inf)\n",
      "HMR_0367, (0.0, inf)\n",
      "HMR_0368, (0.0, inf)\n",
      "HMR_0371, (0.0, inf)\n",
      "HMR_0372, (0.0, inf)\n",
      "HMR_0375, (0.0, inf)\n",
      "HMR_0376, (0.0, inf)\n",
      "HMR_0379, (0.0, inf)\n",
      "HMR_0380, (0.0, inf)\n",
      "HMR_0383, (0.0, inf)\n",
      "HMR_0384, (0.0, inf)\n",
      "HMR_0387, (0.0, inf)\n",
      "HMR_0388, (0.0, inf)\n",
      "HMR_0391, (0.0, inf)\n",
      "HMR_0392, (0.0, inf)\n",
      "HMR_0395, (0.0, inf)\n",
      "HMR_0396, (0.0, inf)\n",
      "HMR_0399, (0.0, inf)\n",
      "HMR_0400, (0.0, inf)\n",
      "HMR_0403, (0.0, inf)\n",
      "HMR_0404, (0.0, inf)\n",
      "HMR_0407, (0.0, inf)\n",
      "HMR_0408, (0.0, inf)\n",
      "HMR_0411, (0.0, inf)\n",
      "HMR_0412, (0.0, inf)\n",
      "HMR_0415, (0.0, inf)\n",
      "HMR_0416, (0.0, inf)\n",
      "HMR_0419, (0.0, inf)\n",
      "HMR_0420, (0.0, inf)\n",
      "HMR_0423, (0.0, inf)\n",
      "HMR_0424, (0.0, inf)\n",
      "HMR_0427, (0.0, inf)\n",
      "HMR_0428, (0.0, inf)\n",
      "HMR_0431, (0.0, inf)\n",
      "HMR_0432, (0.0, inf)\n",
      "HMR_0435, (0.0, inf)\n",
      "HMR_0436, (0.0, inf)\n",
      "HMR_0439, (0.0, inf)\n",
      "HMR_0440, (0.0, inf)\n",
      "HMR_0470, (0.0, inf)\n",
      "HMR_0476, (0.0, inf)\n",
      "HMR_1082, (0.0, inf)\n",
      "HMR_1911, (0.0, inf)\n",
      "HMR_1914, (0.0, inf)\n",
      "HMR_1919, (0.0, inf)\n",
      "HMR_3858, (0.0, inf)\n",
      "HMR_4691, (0.0, inf)\n",
      "HMR_4721, (0.0, inf)\n",
      "HMR_4910, (0.0, inf)\n",
      "HMR_4911, (0.0, inf)\n",
      "HMR_4912, (0.0, inf)\n",
      "HMR_4928, (0.0, inf)\n",
      "HMR_4931, (0.0, inf)\n",
      "HMR_4932, (0.0, inf)\n",
      "HMR_4933, (0.0, inf)\n",
      "HMR_4935, (0.0, inf)\n",
      "HMR_4938, (0.0, inf)\n",
      "HMR_4946, (0.0, inf)\n",
      "HMR_4956, (0.0, inf)\n",
      "HMR_4975, (0.0, inf)\n",
      "HMR_4976, (0.0, inf)\n",
      "HMR_4996, (0.0, inf)\n",
      "HMR_5018, (0.0, inf)\n",
      "HMR_5021, (0.0, inf)\n",
      "HMR_5068, (0.0, inf)\n",
      "HMR_5071, (0.0, inf)\n",
      "HMR_5073, (0.0, inf)\n",
      "HMR_5076, (0.0, inf)\n",
      "HMR_5077, (0.0, inf)\n",
      "HMR_5079, (0.0, inf)\n",
      "HMR_5084, (0.0, inf)\n",
      "HMR_5087, (0.0, inf)\n",
      "HMR_5088, (0.0, inf)\n",
      "HMR_5091, (0.0, inf)\n",
      "HMR_5094, (0.0, inf)\n",
      "HMR_5232, (0.0, inf)\n",
      "HMR_5237, (0.0, inf)\n",
      "HMR_5241, (0.0, inf)\n",
      "HMR_5246, (0.0, inf)\n",
      "HMR_5295, (0.0, inf)\n",
      "HMR_5305, (0.0, inf)\n",
      "HMR_5307, (0.0, inf)\n",
      "HMR_5308, (0.0, inf)\n",
      "HMR_5310, (0.0, inf)\n",
      "HMR_5311, (0.0, inf)\n",
      "HMR_5313, (0.0, inf)\n",
      "HMR_5314, (0.0, inf)\n",
      "HMR_5315, (0.0, inf)\n",
      "HMR_5316, (0.0, inf)\n",
      "HMR_5317, (0.0, inf)\n",
      "HMR_5318, (0.0, inf)\n",
      "HMR_5319, (0.0, inf)\n",
      "HMR_5320, (0.0, inf)\n",
      "HMR_5322, (0.0, inf)\n",
      "HMR_5324, (0.0, inf)\n",
      "HMR_5326, (0.0, inf)\n",
      "HMR_5328, (0.0, inf)\n",
      "HMR_5330, (0.0, inf)\n",
      "HMR_5333, (0.0, inf)\n",
      "HMR_5413, (0.0, inf)\n",
      "HMR_5429, (0.0, inf)\n",
      "HMR_5430, (0.0, inf)\n",
      "HMR_5432, (0.0, inf)\n",
      "HMR_5440, (0.0, inf)\n",
      "HMR_5442, (0.0, inf)\n",
      "HMR_5443, (0.0, inf)\n",
      "HMR_5444, (0.0, inf)\n",
      "HMR_5445, (0.0, inf)\n",
      "HMR_5446, (0.0, inf)\n",
      "HMR_5447, (0.0, inf)\n",
      "HMR_5450, (0.0, inf)\n",
      "HMR_5451, (0.0, inf)\n",
      "HMR_5454, (0.0, inf)\n",
      "HMR_5455, (0.0, inf)\n",
      "HMR_5550, (0.0, inf)\n",
      "HMR_5587, (0.0, inf)\n",
      "HMR_5588, (0.0, inf)\n",
      "HMR_5589, (0.0, inf)\n",
      "HMR_5590, (0.0, inf)\n",
      "HMR_5591, (0.0, inf)\n",
      "HMR_5592, (0.0, inf)\n",
      "HMR_5593, (0.0, inf)\n",
      "HMR_5594, (0.0, inf)\n",
      "HMR_5595, (0.0, inf)\n",
      "HMR_5596, (0.0, inf)\n",
      "HMR_5597, (0.0, inf)\n",
      "HMR_5598, (0.0, inf)\n",
      "HMR_5599, (0.0, inf)\n",
      "HMR_5600, (0.0, inf)\n",
      "HMR_5601, (0.0, inf)\n",
      "HMR_5602, (0.0, inf)\n",
      "HMR_5603, (0.0, inf)\n",
      "HMR_5604, (0.0, inf)\n",
      "HMR_5605, (0.0, inf)\n",
      "HMR_5606, (0.0, inf)\n",
      "HMR_5607, (0.0, inf)\n",
      "HMR_5608, (0.0, inf)\n",
      "HMR_5609, (0.0, inf)\n",
      "HMR_5610, (0.0, inf)\n",
      "HMR_5611, (0.0, inf)\n",
      "HMR_5612, (0.0, inf)\n",
      "HMR_5613, (0.0, inf)\n",
      "HMR_5614, (0.0, inf)\n",
      "HMR_5615, (0.0, inf)\n",
      "HMR_5616, (0.0, inf)\n",
      "HMR_5617, (0.0, inf)\n",
      "HMR_5618, (0.0, inf)\n",
      "HMR_5619, (0.0, inf)\n",
      "HMR_5620, (0.0, inf)\n",
      "HMR_5621, (0.0, inf)\n",
      "HMR_5622, (0.0, inf)\n",
      "HMR_5623, (0.0, inf)\n",
      "HMR_5624, (0.0, inf)\n",
      "HMR_5625, (0.0, inf)\n",
      "HMR_5626, (0.0, inf)\n",
      "HMR_5627, (0.0, inf)\n",
      "HMR_5628, (0.0, inf)\n",
      "HMR_5629, (0.0, inf)\n",
      "HMR_5630, (0.0, inf)\n",
      "HMR_5631, (0.0, inf)\n",
      "HMR_5632, (0.0, inf)\n",
      "HMR_5633, (0.0, inf)\n",
      "HMR_5634, (0.0, inf)\n",
      "HMR_5635, (0.0, inf)\n",
      "HMR_5636, (0.0, inf)\n",
      "HMR_5637, (0.0, inf)\n",
      "HMR_5638, (0.0, inf)\n",
      "HMR_5639, (0.0, inf)\n",
      "HMR_5640, (0.0, inf)\n",
      "HMR_5641, (0.0, inf)\n",
      "HMR_5642, (0.0, inf)\n",
      "HMR_5643, (0.0, inf)\n",
      "HMR_5644, (0.0, inf)\n",
      "HMR_5645, (0.0, inf)\n",
      "HMR_5646, (0.0, inf)\n",
      "HMR_5647, (0.0, inf)\n",
      "HMR_5648, (0.0, inf)\n",
      "HMR_5649, (0.0, inf)\n",
      "HMR_5650, (0.0, inf)\n",
      "HMR_5651, (0.0, inf)\n",
      "HMR_5652, (0.0, inf)\n",
      "HMR_5653, (0.0, inf)\n",
      "HMR_5654, (0.0, inf)\n",
      "HMR_5655, (0.0, inf)\n",
      "HMR_5656, (0.0, inf)\n",
      "HMR_5657, (0.0, inf)\n",
      "HMR_5658, (0.0, inf)\n",
      "HMR_5659, (0.0, inf)\n",
      "HMR_5660, (0.0, inf)\n",
      "HMR_5661, (0.0, inf)\n",
      "HMR_5662, (0.0, inf)\n",
      "HMR_5663, (0.0, inf)\n",
      "HMR_5664, (0.0, inf)\n",
      "HMR_5665, (0.0, inf)\n",
      "HMR_5666, (0.0, inf)\n",
      "HMR_5667, (0.0, inf)\n",
      "HMR_5668, (0.0, inf)\n",
      "HMR_5669, (0.0, inf)\n",
      "HMR_5670, (0.0, inf)\n",
      "HMR_5671, (0.0, inf)\n",
      "HMR_5672, (0.0, inf)\n",
      "HMR_5673, (0.0, inf)\n",
      "HMR_5674, (0.0, inf)\n",
      "HMR_5675, (0.0, inf)\n",
      "HMR_5676, (0.0, inf)\n",
      "HMR_5677, (0.0, inf)\n",
      "HMR_5678, (0.0, inf)\n",
      "HMR_5679, (0.0, inf)\n",
      "HMR_5680, (0.0, inf)\n",
      "HMR_5681, (0.0, inf)\n",
      "HMR_5682, (0.0, inf)\n",
      "HMR_5683, (0.0, inf)\n",
      "HMR_5684, (0.0, inf)\n",
      "HMR_5685, (0.0, inf)\n",
      "HMR_5686, (0.0, inf)\n",
      "HMR_5687, (0.0, inf)\n",
      "HMR_5688, (0.0, inf)\n",
      "HMR_5689, (0.0, inf)\n",
      "HMR_5690, (0.0, inf)\n",
      "HMR_5691, (0.0, inf)\n",
      "HMR_5692, (0.0, inf)\n",
      "HMR_5693, (0.0, inf)\n",
      "HMR_5694, (0.0, inf)\n",
      "HMR_5695, (0.0, inf)\n",
      "HMR_5696, (0.0, inf)\n",
      "HMR_5697, (0.0, inf)\n",
      "HMR_5698, (0.0, inf)\n",
      "HMR_5699, (0.0, inf)\n",
      "HMR_5700, (0.0, inf)\n",
      "HMR_5701, (0.0, inf)\n",
      "HMR_5702, (0.0, inf)\n",
      "HMR_5703, (0.0, inf)\n",
      "HMR_5704, (0.0, inf)\n",
      "HMR_5705, (0.0, inf)\n",
      "HMR_5706, (0.0, inf)\n",
      "HMR_5707, (0.0, inf)\n",
      "HMR_5708, (0.0, inf)\n",
      "HMR_5709, (0.0, inf)\n",
      "HMR_5710, (0.0, inf)\n",
      "HMR_5711, (0.0, inf)\n",
      "HMR_5712, (0.0, inf)\n",
      "HMR_5713, (0.0, inf)\n",
      "HMR_5714, (0.0, inf)\n",
      "HMR_5715, (0.0, inf)\n",
      "HMR_5716, (0.0, inf)\n",
      "HMR_5717, (0.0, inf)\n",
      "HMR_5718, (0.0, inf)\n",
      "HMR_5719, (0.0, inf)\n",
      "HMR_5720, (0.0, inf)\n",
      "HMR_5721, (0.0, inf)\n",
      "HMR_5722, (0.0, inf)\n",
      "HMR_5723, (0.0, inf)\n",
      "HMR_5724, (0.0, inf)\n",
      "HMR_5725, (0.0, inf)\n",
      "HMR_5726, (0.0, inf)\n",
      "HMR_5727, (0.0, inf)\n",
      "HMR_5728, (0.0, inf)\n",
      "HMR_5729, (0.0, inf)\n",
      "HMR_5730, (0.0, inf)\n",
      "HMR_5731, (0.0, inf)\n",
      "HMR_5732, (0.0, inf)\n",
      "HMR_5733, (0.0, inf)\n",
      "HMR_5734, (0.0, inf)\n",
      "HMR_5735, (0.0, inf)\n",
      "HMR_5736, (0.0, inf)\n",
      "HMR_5737, (0.0, inf)\n",
      "HMR_5738, (0.0, inf)\n",
      "HMR_5739, (0.0, inf)\n",
      "HMR_5740, (0.0, inf)\n",
      "HMR_5741, (0.0, inf)\n",
      "HMR_5742, (0.0, inf)\n",
      "HMR_5743, (0.0, inf)\n",
      "HMR_5744, (0.0, inf)\n",
      "HMR_5745, (0.0, inf)\n",
      "HMR_5746, (0.0, inf)\n",
      "HMR_5747, (0.0, inf)\n",
      "HMR_5748, (0.0, inf)\n",
      "HMR_5749, (0.0, inf)\n",
      "HMR_5750, (0.0, inf)\n",
      "HMR_5751, (0.0, inf)\n",
      "HMR_5752, (0.0, inf)\n",
      "HMR_5753, (0.0, inf)\n",
      "HMR_5754, (0.0, inf)\n",
      "HMR_5755, (0.0, inf)\n",
      "HMR_5756, (0.0, inf)\n",
      "HMR_5757, (0.0, inf)\n",
      "HMR_5758, (0.0, inf)\n",
      "HMR_5759, (0.0, inf)\n",
      "HMR_5760, (0.0, inf)\n",
      "HMR_5761, (0.0, inf)\n",
      "HMR_5762, (0.0, inf)\n",
      "HMR_5763, (0.0, inf)\n",
      "HMR_5764, (0.0, inf)\n",
      "HMR_5765, (0.0, inf)\n",
      "HMR_5766, (0.0, inf)\n",
      "HMR_5767, (0.0, inf)\n",
      "HMR_5768, (0.0, inf)\n",
      "HMR_5769, (0.0, inf)\n",
      "HMR_5770, (0.0, inf)\n",
      "HMR_5771, (0.0, inf)\n",
      "HMR_5772, (0.0, inf)\n",
      "HMR_5773, (0.0, inf)\n",
      "HMR_5774, (0.0, inf)\n",
      "HMR_5775, (0.0, inf)\n",
      "HMR_5776, (0.0, inf)\n",
      "HMR_5777, (0.0, inf)\n",
      "HMR_5778, (0.0, inf)\n",
      "HMR_5779, (0.0, inf)\n",
      "HMR_5780, (0.0, inf)\n",
      "HMR_5781, (0.0, inf)\n",
      "HMR_5782, (0.0, inf)\n",
      "HMR_5783, (0.0, inf)\n",
      "HMR_5784, (0.0, inf)\n",
      "HMR_5785, (0.0, inf)\n",
      "HMR_5786, (0.0, inf)\n",
      "HMR_5787, (0.0, inf)\n",
      "HMR_5788, (0.0, inf)\n",
      "HMR_5789, (0.0, inf)\n",
      "HMR_5790, (0.0, inf)\n",
      "HMR_5791, (0.0, inf)\n",
      "HMR_5792, (0.0, inf)\n",
      "HMR_5793, (0.0, inf)\n",
      "HMR_5794, (0.0, inf)\n",
      "HMR_5795, (0.0, inf)\n",
      "HMR_5796, (0.0, inf)\n",
      "HMR_5797, (0.0, inf)\n",
      "HMR_5798, (0.0, inf)\n",
      "HMR_5799, (0.0, inf)\n",
      "HMR_5800, (0.0, inf)\n",
      "HMR_5801, (0.0, inf)\n",
      "HMR_5802, (0.0, inf)\n",
      "HMR_5803, (0.0, inf)\n",
      "HMR_5804, (0.0, inf)\n",
      "HMR_5805, (0.0, inf)\n",
      "HMR_5806, (0.0, inf)\n",
      "HMR_5807, (0.0, inf)\n",
      "HMR_5808, (0.0, inf)\n",
      "HMR_5809, (0.0, inf)\n",
      "HMR_5810, (0.0, inf)\n",
      "HMR_5811, (0.0, inf)\n",
      "HMR_5812, (0.0, inf)\n",
      "HMR_5813, (0.0, inf)\n",
      "HMR_5814, (0.0, inf)\n",
      "HMR_5815, (0.0, inf)\n",
      "HMR_5816, (0.0, inf)\n",
      "HMR_5817, (0.0, inf)\n",
      "HMR_5818, (0.0, inf)\n",
      "HMR_5819, (0.0, inf)\n",
      "HMR_5820, (0.0, inf)\n",
      "HMR_5821, (0.0, inf)\n",
      "HMR_5822, (0.0, inf)\n",
      "HMR_5823, (0.0, inf)\n",
      "HMR_5824, (0.0, inf)\n",
      "HMR_5825, (0.0, inf)\n",
      "HMR_5826, (0.0, inf)\n",
      "HMR_5827, (0.0, inf)\n",
      "HMR_5828, (0.0, inf)\n",
      "HMR_5829, (0.0, inf)\n",
      "HMR_5830, (0.0, inf)\n",
      "HMR_5831, (0.0, inf)\n",
      "HMR_5832, (0.0, inf)\n",
      "HMR_5833, (0.0, inf)\n",
      "HMR_5834, (0.0, inf)\n",
      "HMR_5835, (0.0, inf)\n",
      "HMR_5836, (0.0, inf)\n",
      "HMR_5837, (0.0, inf)\n",
      "HMR_5838, (0.0, inf)\n",
      "HMR_5839, (0.0, inf)\n",
      "HMR_5840, (0.0, inf)\n",
      "HMR_5841, (0.0, inf)\n",
      "HMR_5842, (0.0, inf)\n",
      "HMR_5843, (0.0, inf)\n",
      "HMR_5844, (0.0, inf)\n",
      "HMR_5845, (0.0, inf)\n",
      "HMR_5846, (0.0, inf)\n",
      "HMR_5847, (0.0, inf)\n",
      "HMR_5848, (0.0, inf)\n",
      "HMR_5849, (0.0, inf)\n",
      "HMR_5850, (0.0, inf)\n",
      "HMR_5851, (0.0, inf)\n",
      "HMR_5852, (0.0, inf)\n",
      "HMR_5853, (0.0, inf)\n",
      "HMR_5854, (0.0, inf)\n",
      "HMR_5855, (0.0, inf)\n",
      "HMR_5856, (0.0, inf)\n",
      "HMR_5857, (0.0, inf)\n",
      "HMR_5858, (0.0, inf)\n",
      "HMR_5859, (0.0, inf)\n",
      "HMR_5860, (0.0, inf)\n",
      "HMR_5861, (0.0, inf)\n",
      "HMR_5862, (0.0, inf)\n",
      "HMR_5863, (0.0, inf)\n",
      "HMR_5864, (0.0, inf)\n",
      "HMR_5865, (0.0, inf)\n",
      "HMR_5866, (0.0, inf)\n",
      "HMR_5867, (0.0, inf)\n",
      "HMR_5868, (0.0, inf)\n",
      "HMR_5869, (0.0, inf)\n",
      "HMR_5870, (0.0, inf)\n",
      "HMR_5871, (0.0, inf)\n",
      "HMR_5872, (0.0, inf)\n",
      "HMR_5873, (0.0, inf)\n",
      "HMR_5874, (0.0, inf)\n",
      "HMR_5875, (0.0, inf)\n",
      "HMR_5876, (0.0, inf)\n",
      "HMR_5877, (0.0, inf)\n",
      "HMR_5878, (0.0, inf)\n",
      "HMR_5879, (0.0, inf)\n",
      "HMR_5880, (0.0, inf)\n",
      "HMR_5881, (0.0, inf)\n",
      "HMR_5882, (0.0, inf)\n",
      "HMR_5883, (0.0, inf)\n",
      "HMR_5884, (0.0, inf)\n",
      "HMR_5885, (0.0, inf)\n",
      "HMR_5886, (0.0, inf)\n",
      "HMR_5887, (0.0, inf)\n",
      "HMR_5888, (0.0, inf)\n",
      "HMR_5889, (0.0, inf)\n",
      "HMR_5890, (0.0, inf)\n",
      "HMR_5891, (0.0, inf)\n",
      "HMR_5892, (0.0, inf)\n",
      "HMR_5893, (0.0, inf)\n",
      "HMR_5894, (0.0, inf)\n",
      "HMR_5895, (0.0, inf)\n",
      "HMR_5896, (0.0, inf)\n",
      "HMR_5897, (0.0, inf)\n",
      "HMR_5898, (0.0, inf)\n",
      "HMR_5899, (0.0, inf)\n",
      "HMR_5900, (0.0, inf)\n",
      "HMR_5901, (0.0, inf)\n",
      "HMR_5902, (0.0, inf)\n",
      "HMR_5903, (0.0, inf)\n",
      "HMR_5904, (0.0, inf)\n",
      "HMR_5905, (0.0, inf)\n",
      "HMR_5906, (0.0, inf)\n",
      "HMR_5907, (0.0, inf)\n",
      "HMR_5908, (0.0, inf)\n",
      "HMR_5909, (0.0, inf)\n",
      "HMR_5910, (0.0, inf)\n",
      "HMR_5911, (0.0, inf)\n",
      "HMR_5912, (0.0, inf)\n",
      "HMR_5913, (0.0, inf)\n",
      "HMR_5914, (0.0, inf)\n",
      "HMR_5915, (0.0, inf)\n",
      "HMR_5916, (0.0, inf)\n",
      "HMR_5917, (0.0, inf)\n",
      "HMR_5918, (0.0, inf)\n",
      "HMR_5919, (0.0, inf)\n",
      "HMR_5920, (0.0, inf)\n",
      "HMR_5921, (0.0, inf)\n",
      "HMR_5922, (0.0, inf)\n",
      "HMR_5923, (0.0, inf)\n",
      "HMR_5924, (0.0, inf)\n",
      "HMR_5925, (0.0, inf)\n",
      "HMR_5926, (0.0, inf)\n",
      "HMR_5927, (0.0, inf)\n",
      "HMR_5928, (0.0, inf)\n",
      "HMR_5929, (0.0, inf)\n",
      "HMR_5930, (0.0, inf)\n",
      "HMR_5931, (0.0, inf)\n",
      "HMR_5932, (0.0, inf)\n",
      "HMR_5933, (0.0, inf)\n",
      "HMR_5934, (0.0, inf)\n",
      "HMR_5935, (0.0, inf)\n",
      "HMR_5936, (0.0, inf)\n",
      "HMR_5937, (0.0, inf)\n",
      "HMR_5938, (0.0, inf)\n",
      "HMR_5939, (0.0, inf)\n",
      "HMR_5940, (0.0, inf)\n",
      "HMR_5941, (0.0, inf)\n",
      "HMR_5942, (0.0, inf)\n",
      "HMR_5943, (0.0, inf)\n",
      "HMR_5944, (0.0, inf)\n",
      "HMR_5945, (0.0, inf)\n",
      "HMR_5946, (0.0, inf)\n",
      "HMR_5947, (0.0, inf)\n",
      "HMR_5948, (0.0, inf)\n",
      "HMR_5949, (0.0, inf)\n",
      "HMR_5950, (0.0, inf)\n",
      "HMR_5951, (0.0, inf)\n",
      "HMR_5952, (0.0, inf)\n",
      "HMR_5953, (0.0, inf)\n",
      "HMR_5954, (0.0, inf)\n",
      "HMR_5955, (0.0, inf)\n",
      "HMR_5956, (0.0, inf)\n",
      "HMR_5957, (0.0, inf)\n",
      "HMR_5958, (0.0, inf)\n",
      "HMR_5959, (0.0, inf)\n",
      "HMR_5960, (0.0, inf)\n",
      "HMR_5961, (0.0, inf)\n",
      "HMR_5962, (0.0, inf)\n",
      "HMR_5963, (0.0, inf)\n",
      "HMR_5964, (0.0, inf)\n",
      "HMR_5965, (0.0, inf)\n",
      "HMR_5966, (0.0, inf)\n",
      "HMR_5967, (0.0, inf)\n",
      "HMR_5968, (0.0, inf)\n",
      "HMR_5969, (0.0, inf)\n",
      "HMR_5970, (0.0, inf)\n",
      "HMR_5971, (0.0, inf)\n",
      "HMR_5972, (0.0, inf)\n",
      "HMR_5973, (0.0, inf)\n",
      "HMR_5974, (0.0, inf)\n",
      "HMR_5975, (0.0, inf)\n",
      "HMR_5976, (0.0, inf)\n",
      "HMR_5977, (0.0, inf)\n",
      "HMR_5978, (0.0, inf)\n",
      "HMR_5979, (0.0, inf)\n",
      "HMR_5980, (0.0, inf)\n",
      "HMR_5981, (0.0, inf)\n",
      "HMR_5982, (0.0, inf)\n",
      "HMR_5983, (0.0, inf)\n",
      "HMR_5984, (0.0, inf)\n",
      "HMR_5985, (0.0, inf)\n",
      "HMR_5986, (0.0, inf)\n",
      "HMR_5987, (0.0, inf)\n",
      "HMR_5989, (0.0, inf)\n",
      "HMR_5990, (0.0, inf)\n",
      "HMR_5992, (0.0, inf)\n",
      "HMR_5994, (0.0, inf)\n",
      "HMR_5996, (0.0, inf)\n",
      "HMR_5997, (0.0, inf)\n",
      "HMR_5998, (0.0, inf)\n",
      "HMR_6058, (0.0, inf)\n",
      "HMR_6059, (0.0, inf)\n",
      "HMR_6096, (0.0, inf)\n",
      "HMR_6098, (0.0, inf)\n",
      "HMR_6101, (0.0, inf)\n",
      "HMR_6129, (0.0, inf)\n",
      "HMR_6244, (0.0, inf)\n",
      "HMR_6246, (0.0, inf)\n",
      "HMR_6247, (0.0, inf)\n",
      "HMR_6248, (0.0, inf)\n",
      "HMR_6249, (0.0, inf)\n",
      "HMR_6275, (0.0, inf)\n",
      "HMR_6369, (0.0, inf)\n",
      "HMR_6370, (0.0, inf)\n",
      "HMR_6371, (0.0, inf)\n",
      "HMR_6372, (0.0, inf)\n",
      "HMR_6373, (0.0, inf)\n",
      "HMR_6374, (0.0, inf)\n",
      "HMR_6375, (0.0, inf)\n",
      "HMR_6377, (0.0, inf)\n",
      "HMR_6379, (0.0, inf)\n",
      "HMR_6380, (0.0, inf)\n",
      "HMR_6381, (0.0, inf)\n",
      "HMR_6382, (0.0, inf)\n",
      "HMR_6383, (0.0, inf)\n",
      "HMR_6384, (0.0, inf)\n",
      "HMR_6392, (0.0, inf)\n",
      "HMR_6406, (0.0, inf)\n",
      "HMR_6439, (0.0, inf)\n",
      "HMR_6525, (0.0, inf)\n",
      "HMR_6526, (0.0, inf)\n",
      "HMR_6527, (0.0, inf)\n",
      "HMR_6530, (0.0, inf)\n",
      "HMR_6531, (0.0, inf)\n",
      "HMR_6532, (0.0, inf)\n",
      "HMR_6533, (0.0, inf)\n",
      "HMR_6534, (0.0, inf)\n",
      "HMR_7109, (0.0, inf)\n",
      "HMR_7111, (0.0, inf)\n",
      "HMR_7113, (0.0, inf)\n",
      "HMR_7115, (0.0, inf)\n",
      "HMR_7117, (0.0, inf)\n",
      "HMR_7119, (0.0, inf)\n",
      "HMR_7121, (0.0, inf)\n",
      "HMR_7123, (0.0, inf)\n",
      "HMR_7125, (0.0, inf)\n",
      "HMR_7127, (0.0, inf)\n",
      "HMR_7633, (0.0, inf)\n",
      "HMR_7634, (0.0, inf)\n",
      "HMR_7635, (0.0, inf)\n",
      "HMR_7636, (0.0, inf)\n",
      "HMR_7637, (0.0, inf)\n",
      "HMR_7639, (0.0, inf)\n",
      "HMR_7645, (0.0, inf)\n",
      "HMR_7650, (0.0, inf)\n",
      "HMR_7651, (0.0, inf)\n",
      "HMR_7666, (0.0, inf)\n",
      "HMR_7667, (0.0, inf)\n",
      "HMR_7687, (0.0, inf)\n",
      "HMR_7691, (0.0, inf)\n",
      "HMR_7692, (0.0, inf)\n",
      "HMR_7787, (0.0, inf)\n",
      "HMR_7902, (0.0, inf)\n",
      "HMR_7904, (0.0, inf)\n",
      "HMR_7964, (0.0, inf)\n",
      "HMR_7967, (0.0, inf)\n",
      "HMR_7982, (0.0, inf)\n",
      "HMR_7986, (0.0, inf)\n",
      "HMR_7990, (0.0, inf)\n",
      "HMR_8075, (0.0, inf)\n",
      "HMR_8076, (0.0, inf)\n",
      "HMR_8077, (0.0, inf)\n",
      "HMR_8354, (0.0, inf)\n",
      "HMR_8364, (0.0, inf)\n",
      "HMR_8429, (0.0, inf)\n",
      "HMR_8612, (0.0, inf)\n",
      "HMR_8629, (0.0, inf)\n",
      "HMR_8633, (0.0, inf)\n",
      "HMR_8658, (0.0, inf)\n",
      "HMR_8670, (0.0, inf)\n",
      "HMR_8677, (0.0, inf)\n",
      "HMR_8687, (0.0, inf)\n",
      "HMR_8735, (0.0, inf)\n",
      "HMR_8736, (0.0, inf)\n",
      "HMR_8760, (0.0, inf)\n",
      "HMR_8847, (0.0, inf)\n",
      "HMR_8850, (0.0, inf)\n",
      "HMR_8872, (0.0, inf)\n",
      "HMR_8876, (0.0, inf)\n",
      "HMR_8889, (0.0, inf)\n",
      "HMR_8911, (0.0, inf)\n",
      "HMR_8912, (0.0, inf)\n",
      "HMR_8917, (0.0, inf)\n",
      "HMR_8925, (0.0, inf)\n",
      "HMR_8929, (0.0, inf)\n",
      "HMR_8930, (0.0, inf)\n",
      "HMR_8934, (0.0, inf)\n",
      "HMR_8935, (0.0, inf)\n",
      "HMR_8939, (0.0, inf)\n",
      "HMR_8940, (0.0, inf)\n",
      "HMR_8941, (0.0, inf)\n",
      "HMR_8942, (0.0, inf)\n",
      "HMR_8943, (0.0, inf)\n",
      "HMR_8944, (0.0, inf)\n",
      "HMR_8945, (0.0, inf)\n",
      "HMR_8946, (0.0, inf)\n",
      "HMR_8947, (0.0, inf)\n",
      "HMR_9190, (0.0, inf)\n",
      "HMR_9191, (0.0, inf)\n",
      "HMR_9197, (0.0, inf)\n",
      "HMR_9198, (0.0, inf)\n",
      "HMR_9590, (0.0, inf)\n",
      "HMR_9591, (0.0, inf)\n",
      "HMR_9592, (0.0, inf)\n",
      "HMR_9593, (0.0, inf)\n",
      "HMR_9594, (0.0, inf)\n",
      "HMR_9595, (0.0, inf)\n",
      "HMR_9596, (0.0, inf)\n",
      "HMR_9597, (0.0, inf)\n",
      "HMR_9598, (0.0, inf)\n",
      "HMR_9599, (0.0, inf)\n",
      "HMR_9600, (0.0, inf)\n",
      "HMR_9601, (0.0, inf)\n",
      "HMR_9602, (0.0, inf)\n",
      "HMR_9603, (0.0, inf)\n",
      "HMR_9604, (0.0, inf)\n",
      "HMR_9605, (0.0, inf)\n",
      "HMR_9606, (0.0, inf)\n",
      "HMR_9607, (0.0, inf)\n",
      "HMR_9608, (0.0, inf)\n",
      "HMR_9609, (0.0, inf)\n",
      "HMR_9611, (0.0, inf)\n",
      "HMR_9613, (0.0, inf)\n",
      "HMR_9616, (0.0, inf)\n",
      "HMR_9619, (0.0, inf)\n",
      "HMR_9623, (0.0, inf)\n",
      "HMR_9624, (0.0, inf)\n",
      "HMR_0578, (0.0, inf)\n",
      "HMR_2141, (0.0, inf)\n",
      "HMR_2590, (0.0, inf)\n",
      "HMR_3825, (0.0, inf)\n",
      "HMR_3864, (0.0, inf)\n",
      "HMR_3954, (0.0, inf)\n",
      "HMR_4754, (0.0, inf)\n",
      "HMR_4905, (0.0, inf)\n",
      "HMR_4926, (0.0, inf)\n",
      "HMR_4964, (0.0, inf)\n",
      "HMR_4977, (0.0, inf)\n",
      "HMR_4997, (0.0, inf)\n",
      "HMR_5006, (0.0, inf)\n",
      "HMR_5015, (0.0, inf)\n",
      "HMR_5016, (0.0, inf)\n",
      "HMR_5096, (0.0, inf)\n",
      "HMR_5101, (0.0, inf)\n",
      "HMR_5105, (0.0, inf)\n",
      "HMR_5107, (0.0, inf)\n",
      "HMR_5118, (0.0, inf)\n",
      "HMR_5122, (0.0, inf)\n",
      "HMR_5225, (0.0, inf)\n",
      "HMR_5348, (0.0, inf)\n",
      "HMR_5349, (0.0, inf)\n",
      "HMR_5420, (0.0, inf)\n",
      "HMR_6286, (0.0, inf)\n",
      "HMR_6287, (0.0, inf)\n",
      "HMR_6288, (0.0, inf)\n",
      "HMR_6289, (0.0, inf)\n",
      "HMR_6290, (0.0, inf)\n",
      "HMR_6291, (0.0, inf)\n",
      "HMR_6292, (0.0, inf)\n",
      "HMR_6293, (0.0, inf)\n",
      "HMR_6294, (0.0, inf)\n",
      "HMR_6295, (0.0, inf)\n",
      "HMR_6296, (0.0, inf)\n",
      "HMR_6301, (0.0, inf)\n",
      "HMR_6302, (0.0, inf)\n",
      "HMR_6307, (0.0, inf)\n",
      "HMR_6308, (0.0, inf)\n",
      "HMR_6313, (0.0, inf)\n",
      "HMR_6314, (0.0, inf)\n",
      "HMR_6317, (0.0, inf)\n",
      "HMR_6318, (0.0, inf)\n",
      "HMR_6321, (0.0, inf)\n",
      "HMR_6323, (0.0, inf)\n",
      "HMR_6324, (0.0, inf)\n",
      "HMR_6327, (0.0, inf)\n",
      "HMR_6332, (0.0, inf)\n",
      "HMR_6333, (0.0, inf)\n",
      "HMR_6334, (0.0, inf)\n",
      "HMR_6335, (0.0, inf)\n",
      "HMR_6336, (0.0, inf)\n",
      "HMR_6337, (0.0, inf)\n",
      "HMR_6338, (0.0, inf)\n",
      "HMR_6339, (0.0, inf)\n",
      "HMR_6340, (0.0, inf)\n",
      "HMR_6341, (0.0, inf)\n",
      "HMR_6342, (0.0, inf)\n",
      "HMR_6343, (0.0, inf)\n",
      "HMR_6431, (0.0, inf)\n",
      "HMR_6446, (0.0, inf)\n",
      "HMR_6454, (0.0, inf)\n",
      "HMR_6469, (0.0, inf)\n",
      "HMR_6481, (0.0, inf)\n",
      "HMR_7638, (0.0, inf)\n",
      "HMR_7815, (0.0, inf)\n",
      "HMR_7816, (0.0, inf)\n",
      "HMR_7825, (0.0, inf)\n",
      "HMR_7826, (0.0, inf)\n",
      "HMR_7827, (0.0, inf)\n",
      "HMR_7828, (0.0, inf)\n",
      "HMR_7829, (0.0, inf)\n",
      "HMR_7830, (0.0, inf)\n",
      "HMR_7831, (0.0, inf)\n",
      "HMR_7832, (0.0, inf)\n",
      "HMR_7833, (0.0, inf)\n",
      "HMR_7834, (0.0, inf)\n",
      "HMR_7835, (0.0, inf)\n",
      "HMR_7836, (0.0, inf)\n",
      "HMR_7837, (0.0, inf)\n",
      "HMR_7838, (0.0, inf)\n",
      "HMR_7839, (0.0, inf)\n",
      "HMR_7840, (0.0, inf)\n",
      "HMR_7841, (0.0, inf)\n",
      "HMR_7842, (0.0, inf)\n",
      "HMR_7843, (0.0, inf)\n",
      "HMR_7844, (0.0, inf)\n",
      "HMR_7845, (0.0, inf)\n",
      "HMR_7846, (0.0, inf)\n",
      "HMR_7847, (0.0, inf)\n",
      "HMR_7848, (0.0, inf)\n",
      "HMR_7849, (0.0, inf)\n",
      "HMR_7850, (0.0, inf)\n",
      "HMR_7851, (0.0, inf)\n",
      "HMR_7852, (0.0, inf)\n",
      "HMR_7853, (0.0, inf)\n",
      "HMR_7854, (0.0, inf)\n",
      "HMR_8614, (0.0, inf)\n",
      "HMR_8627, (0.0, inf)\n",
      "HMR_0713, (0.0, inf)\n",
      "HMR_1286, (0.0, inf)\n",
      "HMR_3008, (0.0, inf)\n",
      "HMR_3011, (0.0, inf)\n",
      "HMR_3013, (0.0, inf)\n",
      "HMR_3014, (0.0, inf)\n",
      "HMR_3017, (0.0, inf)\n",
      "HMR_3020, (0.0, inf)\n",
      "HMR_3023, (0.0, inf)\n",
      "HMR_3473, (0.0, inf)\n",
      "HMR_7646, (0.0, inf)\n",
      "HMR_8016, (0.0, inf)\n",
      "HMR_9678, (0.0, inf)\n",
      "HMR_8028, (0.0, inf)\n",
      "HMR_3987, (0.0, inf)\n",
      "HMR_4907, (0.0, inf)\n",
      "HMR_5047, (0.0, inf)\n",
      "HMR_5048, (0.0, inf)\n",
      "HMR_5049, (0.0, inf)\n",
      "HMR_5050, (0.0, inf)\n",
      "HMR_5051, (0.0, inf)\n",
      "HMR_5052, (0.0, inf)\n",
      "HMR_5053, (0.0, inf)\n",
      "HMR_5054, (0.0, inf)\n",
      "HMR_5055, (0.0, inf)\n",
      "HMR_5056, (0.0, inf)\n",
      "HMR_5057, (0.0, inf)\n",
      "HMR_5058, (0.0, inf)\n",
      "HMR_5059, (0.0, inf)\n",
      "HMR_5060, (0.0, inf)\n",
      "HMR_5061, (0.0, inf)\n",
      "HMR_5062, (0.0, inf)\n",
      "HMR_5063, (0.0, inf)\n",
      "HMR_5064, (0.0, inf)\n",
      "HMR_5065, (0.0, inf)\n",
      "HMR_5066, (0.0, inf)\n",
      "HMR_5067, (0.0, inf)\n",
      "HMR_5197, (0.0, inf)\n",
      "HMR_5224, (0.0, inf)\n",
      "HMR_5226, (0.0, inf)\n",
      "HMR_5228, (0.0, inf)\n",
      "HMR_7224, (0.0, inf)\n",
      "HMR_7374, (0.0, inf)\n",
      "HMR_7430, (0.0, inf)\n",
      "HMR_7454, (0.0, inf)\n",
      "HMR_7483, (0.0, inf)\n",
      "HMR_7500, (0.0, inf)\n",
      "HMR_7512, (0.0, inf)\n",
      "HMR_7524, (0.0, inf)\n",
      "HMR_7540, (0.0, inf)\n",
      "HMR_7556, (0.0, inf)\n",
      "HMR_7570, (0.0, inf)\n",
      "HMR_7581, (0.0, inf)\n",
      "HMR_7584, (0.0, inf)\n",
      "HMR_7799, (0.0, inf)\n",
      "HMR_7924, (0.0, inf)\n",
      "HMR_8125, (0.0, inf)\n",
      "HMR_8127, (0.0, inf)\n",
      "HMR_0612, (0.0, inf)\n",
      "HMR_3597, (0.0, inf)\n",
      "HMR_5251, (0.0, inf)\n",
      "HMR_5252, (0.0, inf)\n",
      "HMR_0731, (0.0, inf)\n",
      "HMR_7201, (0.0, inf)\n",
      "HMR_7435, (0.0, inf)\n",
      "HMR_8661, (0.0, inf)\n",
      "HMR_7307, (0.0, inf)\n",
      "HMR_1269, (0.0, inf)\n",
      "HMR_4860, (0.0, inf)\n",
      "HMR_4881, (0.0, inf)\n",
      "HMR_4906, (0.0, inf)\n",
      "HMR_6385, (0.0, inf)\n",
      "HMR_6386, (0.0, inf)\n",
      "HMR_7272, (0.0, inf)\n",
      "HMR_7273, (0.0, inf)\n",
      "HMR_7789, (0.0, inf)\n",
      "cofactors_vitamins, (0.0, inf)\n",
      "vitaminA, (0.0, inf)\n",
      "vitaminD, (0.0, inf)\n",
      "vitaminE, (0.0, inf)\n",
      "HMR_0685, (0.0, inf)\n",
      "HMR_0686, (0.0, inf)\n",
      "HMR_2393, (0.0, inf)\n",
      "HMR_4783, (0.0, inf)\n",
      "HMR_7777, (0.0, inf)\n",
      "HMR_7779, (0.0, inf)\n",
      "HMR_9024, (0.0, inf)\n",
      "HMR_9025, (0.0, inf)\n",
      "HMR_9027, (0.0, inf)\n",
      "HMR_9028, (0.0, inf)\n",
      "HMR_9029, (0.0, inf)\n",
      "HMR_9030, (0.0, inf)\n",
      "HMR_9031, (0.0, inf)\n",
      "HMR_9035, (0.0, inf)\n",
      "HMR_9036, (0.0, inf)\n",
      "HMR_9037, (0.0, inf)\n",
      "HMR_9038, (0.0, inf)\n",
      "HMR_9039, (0.0, inf)\n",
      "HMR_9040, (0.0, inf)\n",
      "HMR_9041, (0.0, inf)\n",
      "HMR_9042, (0.0, inf)\n",
      "HMR_9043, (0.0, inf)\n",
      "HMR_9044, (0.0, inf)\n",
      "HMR_9045, (0.0, inf)\n",
      "HMR_9046, (0.0, inf)\n",
      "HMR_9048, (0.0, inf)\n",
      "HMR_9049, (0.0, inf)\n",
      "HMR_9051, (0.0, inf)\n",
      "HMR_9052, (0.0, inf)\n",
      "HMR_9053, (0.0, inf)\n",
      "HMR_9054, (0.0, inf)\n",
      "HMR_9055, (0.0, inf)\n",
      "HMR_9058, (0.0, inf)\n",
      "biomass_components, (0.0, inf)\n",
      "artificial_biomass, (0.0, 1000.0)\n"
     ]
    }
   ],
   "source": [
    "for reaction in Hep_G2_unip_ab.reactions :\n",
    "    if reaction.bounds != (float('-inf'), float('inf')) :\n",
    "        print(f\"{reaction.id}, {reaction.bounds}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "a0b6486f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<h3>Objective</h3><p>1.0 artificial_biomass = 0.0</p><h4>Uptake</h4><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>Metabolite</th>\n",
       "      <th>Reaction</th>\n",
       "      <th>Flux</th>\n",
       "      <th>C-Number</th>\n",
       "      <th>C-Flux</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table><h4>Secretion</h4><table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>Metabolite</th>\n",
       "      <th>Reaction</th>\n",
       "      <th>Flux</th>\n",
       "      <th>C-Number</th>\n",
       "      <th>C-Flux</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>"
      ],
      "text/plain": [
       "<cobra.summary.model_summary.ModelSummary at 0x7f56b14e78b0>"
      ]
     },
     "execution_count": 62,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Hep_G2_unip_ab.objective = \"artificial_biomass\"\n",
    "#sol_Hep_G2 = Hep_G2_unip_ab.optimize()\n",
    "\n",
    "#sol_Hep_G2\n",
    "Hep_G2_unip_ab.summary()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "59ab92fc",
   "metadata": {},
   "source": [
    "### Réactions d'échange"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "9e8b1998",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "for reaction in Hep_G2_unip_ab.reactions :\n",
    "    if \"EX_\" in reaction.id :\n",
    "        if reaction.id != \"EX_m01286x\" and reaction.id != \"EX_m01965x\" and reaction.id != \"EX_m01252x\":\n",
    "            reaction.bounds = (0.0,0.0)\n",
    "            #print_reactions(reaction)\n",
    "            #print(\"\")\n",
    "            #print(f\"Flux : {reaction.flux}\\n\")\n",
    "            #print(f\"{reaction.bounds} --- {reaction.id}\")\n",
    "            #print(\"\\n\\n\")\n",
    "        else :\n",
    "            reaction.bounds = (float(\"-inf\"), float(\"inf\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "id": "2d73fb5a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "EX_temp001x: temp001x --> \n"
     ]
    }
   ],
   "source": [
    "print(Hep_G2_unip_ab.reactions.EX_temp001x)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "c02a6c9a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "EX_m01286x: m01286x <--  (-inf, 0.0)\n",
      "EX_m01965x: m01965x <--  (-inf, 0.0)\n"
     ]
    }
   ],
   "source": [
    "for reaction in Hep_G2_unip_ab.reactions :\n",
    "    if \"EX_\" in reaction.id :\n",
    "        metab = [m.id for m in reaction.metabolites]\n",
    "        if \"m01965x\" in metab or \"m01286x\" in metab :\n",
    "            \n",
    "            reaction.bounds = (float('-inf'),0.0)\n",
    "            print(reaction, reaction.bounds)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "dbb5bdbc",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Reaction identifier</strong></td><td>artificial_biomass</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Name</strong></td><td></td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x7fac785d18b0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Stoichiometry</strong></td>\n",
       "                <td>\n",
       "                    <p style='text-align:right'>m01307c + m01365c + m01369c + m01370c + m01450c + m01451r + m01589c + m01628c + m01721n + m01722n + m01974c + m01975c + m01986c + m02125c + m02184c + m02360c + m02392c + m02426c + m02471c + m02724c...</p>\n",
       "                    <p style='text-align:right'>alanine + arginine + asparagine + aspartate + cholesterol + cholesterol-ester pool + CL pool + cysteine + DNA + DNA-5-methylcytosine + glutamate + glutamine + glycine + histidine + isoleucine +...</p>\n",
       "                </td>\n",
       "            </tr><tr>\n",
       "                <td><strong>GPR</strong></td><td></td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Lower bound</strong></td><td>0.0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Upper bound</strong></td><td>1000.0</td>\n",
       "            </tr>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<Reaction artificial_biomass at 0x7fac785d18b0>"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Hep_G2.reactions.artificial_biomass"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "80c22195",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "biomass --- temp001x\n",
      "glutamine --- m01975c\n",
      "RNA --- m02847c\n",
      "tryptophan --- m03089c\n",
      "threonine --- m02993c\n",
      "proline --- m02770c\n",
      "lysine --- m02426c\n",
      "arginine --- m01365c\n",
      "serine --- m02896c\n",
      "tyrosine --- m03101c\n",
      "isoleucine --- m02184c\n",
      "asparagine --- m01369c\n",
      "cysteine --- m01628c\n",
      "methionine --- m02471c\n",
      "glycine --- m01986c\n",
      "aspartate --- m01370c\n",
      "glycogen --- m03161c\n",
      "phenylalanine --- m02724c\n",
      "alanine --- m01307c\n",
      "histidine --- m02125c\n",
      "valine --- m03135c\n",
      "leucine --- m02360c\n",
      "glutamate --- m01974c\n",
      "DNA --- m01721n\n",
      "DNA-5-methylcytosine --- m01722n\n",
      "cholesterol --- m01450c\n",
      "CL pool --- m01589c\n",
      "cholesterol-ester pool --- m01451r\n",
      "lipid droplet --- m02392c\n",
      "PI pool --- m02750c\n",
      "SM pool --- m02908c\n",
      "phosphatidate-LD-TAG pool --- m02733c\n"
     ]
    }
   ],
   "source": [
    "for biomass_metabolite in Hep_G2.reactions.artificial_biomass.metabolites :\n",
    "    print(biomass_metabolite.name,'---', biomass_metabolite.id)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "78be248e",
   "metadata": {},
   "source": [
    "### Vérification effet Warburg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "b9ae83cf",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "O2 :\n",
      "O2 --- m02630c --- C_c\n",
      "O2- --- m02631c --- C_c\n",
      "O2 --- m02630m --- C_m\n",
      "O2- --- m02631m --- C_m\n",
      "O2 --- m02630n --- C_n\n",
      "O2- --- m02631n --- C_n\n",
      "O2 --- m02630p --- C_p\n",
      "O2- --- m02631p --- C_p\n",
      "O2 --- m02630r --- C_r\n",
      "O2 --- m02630s --- C_s\n",
      "O2- --- m02631s --- C_s\n",
      "O2 --- m02630x --- C_x\n",
      "O2- --- m02631x --- C_x\n",
      "\n",
      "Reactions associées à O2\n",
      "\n",
      "HMR_4896 --- 236.39280919611895\n",
      "\n",
      "1.0 O2 <=> 1.0 O2\n",
      "\n",
      "1.0 m02630c <=> 1.0 m02630s\n",
      "\n",
      "####====================####\n",
      "\n",
      "HMR_8414 --- -3.744976978240768e-15\n",
      "\n",
      "1.0 O2- <=> 1.0 O2-\n",
      "\n",
      "1.0 m02631c <=> 1.0 m02631p\n",
      "\n",
      "####====================####\n",
      "\n",
      "HMR_6914 --- -37.0677259254506\n",
      "\n",
      "4.0 ferrocytochrome C + 8.0 H+ + 1.0 O2 <=> 4.0 ferricytochrome C + 4.0 H+ + 2.0 H2O\n",
      "\n",
      "4.0 m01826m + 8.0 m02039m + 1.0 m02630m <=> 4.0 m01824m + 4.0 m02039c + 2.0 m02040m\n",
      "\n",
      "####====================####\n",
      "\n",
      "HMR_8414 --- -3.744976978240768e-15\n",
      "\n",
      "1.0 O2- <=> 1.0 O2-\n",
      "\n",
      "1.0 m02631c <=> 1.0 m02631p\n",
      "\n",
      "####====================####\n",
      "\n",
      "HMR_9048 --- -232.75219564567547\n",
      "\n",
      "1.0 O2 <=> 1.0 O2\n",
      "\n",
      "1.0 m02630x <=> 1.0 m02630s\n",
      "\n",
      "####====================####\n",
      "\n",
      "HMR_4896 --- 236.39280919611895\n",
      "\n",
      "1.0 O2 <=> 1.0 O2\n",
      "\n",
      "1.0 m02630c <=> 1.0 m02630s\n",
      "\n",
      "####====================####\n",
      "\n",
      "HMR_9048 --- -232.75219564567547\n",
      "\n",
      "1.0 O2 <=> 1.0 O2\n",
      "\n",
      "1.0 m02630x <=> 1.0 m02630s\n",
      "\n",
      "####====================####\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Recherche des réactions impliquant l'O2\n",
    "O2_metab = []\n",
    "print(\"O2 :\")\n",
    "for metab in Hep_G2.metabolites :\n",
    "    if 'O2' in metab.name.split('_')[0][:2] and len(metab.name.split('_')[0]) < 6 :\n",
    "        O2_metab.append(metab)       \n",
    "        print(f'{metab.name} --- {metab.id} --- {metab.compartment}')\n",
    "print(\"\\nReactions associées à O2\\n\")\n",
    "for O2 in O2_metab :\n",
    "    for reac in O2.reactions :\n",
    "        if reac.flux != 0.0 and len(reac.compartments) >1:\n",
    "            print(f\"{reac.id} --- {reac.flux}\\n\")\n",
    "            print_reactions(reac)\n",
    "            print(\"\\n####====================####\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "0be9614a",
   "metadata": {},
   "outputs": [],
   "source": [
    "Hep_G2.reactions.HMR_4280.bounds = (0.0,0.0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "6891699b",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1.0 PEP + 1.0 UDP --> 1.0 pyruvate + 1.0 UTP\n",
      "\n",
      "1.0 m02696c + 1.0 m03106c --> 1.0 m02819c + 1.0 m03130c\n",
      "HMR_4210\n",
      "1.0 (R)-3-hydroxybutanoate + 1.0 pyruvate --> 1.0 (R)-3-hydroxybutanoate + 1.0 pyruvate\n",
      "\n",
      "1.0 m00157c + 1.0 m02819s --> 1.0 m00157s + 1.0 m02819c\n",
      "HMR_6021\n",
      "1.0 malate + 1.0 NADP+ --> 1.0 CO2 + 1.0 H+ + 1.0 NADPH + 1.0 pyruvate\n",
      "\n",
      "1.0 m02439c + 1.0 m02554c --> 1.0 m01596c + 1.0 m02039c + 1.0 m02555c + 1.0 m02819c\n",
      "HMR_4089\n",
      "1.0 serine --> 1.0 NH3 + 1.0 pyruvate\n",
      "\n",
      "1.0 m02896c --> 1.0 m02578c + 1.0 m02819c\n",
      "HMR_3883\n",
      "1.0 L-lactate + 1.0 pyruvate --> 1.0 L-lactate + 1.0 pyruvate\n",
      "\n",
      "1.0 m02403c + 1.0 m02819s --> 1.0 m02403s + 1.0 m02819c\n",
      "HMR_6018\n",
      "1.0 H2O + 1.0 salsolinol-1-carboxylate --> 1.0 dopamine + 1.0 pyruvate\n",
      "\n",
      "1.0 m02040c + 1.0 m02876c --> 1.0 m01736c + 1.0 m02819c\n",
      "HMR_6738\n",
      "1.0 H2O + 1.0 se-methyl-L-selenocysteine --> 1.0 methaneselenol + 1.0 NH3 + 1.0 pyruvate\n",
      "\n",
      "1.0 m02040c + 1.0 m02895c --> 1.0 m02469c + 1.0 m02578c + 1.0 m02819c\n",
      "HMR_7141\n",
      "1.0 dGDP + 1.0 PEP --> 1.0 dGTP + 1.0 pyruvate\n",
      "\n",
      "1.0 m01680c + 1.0 m02696c --> 1.0 m01688c + 1.0 m02819c\n",
      "HMR_4573\n",
      "1.0 H+ + 1.0 NADH + 1.0 pyruvate --> 1.0 L-lactate + 1.0 NAD+\n",
      "\n",
      "1.0 m02039c + 1.0 m02553c + 1.0 m02819c --> 1.0 m02403c + 1.0 m02552c\n",
      "HMR_4388\n",
      "1.0 GDP + 1.0 PEP --> 1.0 GTP + 1.0 pyruvate\n",
      "\n",
      "1.0 m01948c + 1.0 m02696c --> 1.0 m02034c + 1.0 m02819c\n",
      "HMR_4171\n",
      "1.0 mercaptopyruvate + 1.0 sulfite --> 1.0 pyruvate + 1.0 thiosulfate\n",
      "\n",
      "1.0 m02464c + 1.0 m02949c --> 1.0 m02819c + 1.0 m02991c\n",
      "HMR_6515\n",
      "1.0 dehydroalanine + 1.0 H2O --> 1.0 NH3 + 1.0 pyruvate\n",
      "\n",
      "1.0 m01654c + 1.0 m02040c --> 1.0 m02578c + 1.0 m02819c\n",
      "HMR_3974\n",
      "1.0 acetoacetate + 1.0 pyruvate --> 1.0 acetoacetate + 1.0 pyruvate\n",
      "\n",
      "1.0 m01253c + 1.0 m02819s --> 1.0 m01253s + 1.0 m02819c\n",
      "HMR_6000\n",
      "1.0 2-hydroxybutyrate + 1.0 pyruvate --> 1.0 2-hydroxybutyrate + 1.0 pyruvate\n",
      "\n",
      "1.0 m00648c + 1.0 m02819s --> 1.0 m00648s + 1.0 m02819c\n",
      "HMR_6007\n",
      "1.0 dTDP + 1.0 PEP --> 1.0 dTTP + 1.0 pyruvate\n",
      "\n",
      "1.0 m01747c + 1.0 m02696c --> 1.0 m01753c + 1.0 m02819c\n",
      "HMR_6627\n",
      "1.0 butyrate + 1.0 pyruvate --> 1.0 butyrate + 1.0 pyruvate\n",
      "\n",
      "1.0 m01410c + 1.0 m02819s --> 1.0 m01410s + 1.0 m02819c\n",
      "HMR_6019\n",
      "2.0 ferricytochrome C + 1.0 L-lactate --> 2.0 ferrocytochrome C + 2.0 H+ + 1.0 pyruvate\n",
      "\n",
      "2.0 m01824m + 1.0 m02403c --> 2.0 m01826m + 2.0 m02039c + 1.0 m02819c\n",
      "HMR_8514\n",
      "1.0 H+ + 1.0 pyruvate --> 1.0 H+ + 1.0 pyruvate\n",
      "\n",
      "1.0 m02039c + 1.0 m02819c --> 1.0 m02039p + 1.0 m02819p\n",
      "HMR_4930\n",
      "1.0 mercaptopyruvate + 1.0 sulfite --> 1.0 H2S2O3 + 1.0 pyruvate\n",
      "\n",
      "1.0 m02464c + 1.0 m02949c --> 1.0 m02043c + 1.0 m02819c\n",
      "HMR_3928\n",
      "1.0 hydrogen-cyanide + 1.0 mercaptopyruvate --> 1.0 pyruvate + 1.0 thiocyanate\n",
      "\n",
      "1.0 m02145c + 1.0 m02464c --> 1.0 m02819c + 1.0 m02986c\n",
      "HMR_6518\n",
      "1.0 H+ + 1.0 pyruvate --> 1.0 H+ + 1.0 pyruvate\n",
      "\n",
      "1.0 m02039c + 1.0 m02819c --> 1.0 m02039m + 1.0 m02819m\n",
      "HMR_4926\n",
      "1.0 pyruvate + 1.0 serine --> 1.0 alanine + 1.0 hydroxypyruvate\n",
      "\n",
      "1.0 m02819c + 1.0 m02896c --> 1.0 m01307c + 1.0 m02154c\n",
      "HMR_4198\n",
      "1.0 D-lactate + 2.0 ferricytochrome C --> 2.0 ferrocytochrome C + 2.0 H+ + 1.0 pyruvate\n",
      "\n",
      "1.0 m01716c + 2.0 m01824m --> 2.0 m01826m + 2.0 m02039c + 1.0 m02819c\n",
      "HMR_3859\n",
      "1.0 AKG + 1.0 pyruvate --> 1.0 AKG + 1.0 pyruvate\n",
      "\n",
      "1.0 m01306c + 1.0 m02819s --> 1.0 m01306s + 1.0 m02819c\n",
      "HMR_6014\n",
      "1.0 pyruvate + 1.0 retinoate --> 1.0 pyruvate + 1.0 retinoate\n",
      "\n",
      "1.0 m02819c + 1.0 m02833s --> 1.0 m02819s + 1.0 m02833c\n",
      "HMR_6020\n",
      "1.0 ADP + 1.0 PEP --> 1.0 ATP + 1.0 pyruvate\n",
      "\n",
      "1.0 m01285c + 1.0 m02696c --> 1.0 m01371c + 1.0 m02819c\n",
      "HMR_4358\n",
      "1.0 AKG + 1.0 alanine --> 1.0 glutamate + 1.0 pyruvate\n",
      "\n",
      "1.0 m01306c + 1.0 m01307c --> 1.0 m01974c + 1.0 m02819c\n",
      "HMR_3899\n",
      "1.0 H2O + 1.0 methylglyoxal + 1.0 NADP+ --> 1.0 H+ + 1.0 NADPH + 1.0 pyruvate\n",
      "\n",
      "1.0 m02040c + 1.0 m02475c + 1.0 m02554c --> 1.0 m02039c + 1.0 m02555c + 1.0 m02819c\n",
      "HMR_4360\n",
      "1.0 D-lactate + 1.0 NAD+ --> 1.0 H+ + 1.0 NADH + 1.0 pyruvate\n",
      "\n",
      "1.0 m01716c + 1.0 m02552c --> 1.0 m02039c + 1.0 m02553c + 1.0 m02819c\n",
      "HMR_8512\n",
      "1.0 formate + 1.0 pyruvate --> 1.0 formate + 1.0 pyruvate\n",
      "\n",
      "1.0 m01833c + 1.0 m02819s --> 1.0 m01833s + 1.0 m02819c\n",
      "HMR_6016\n",
      "1.0 propanoate + 1.0 pyruvate --> 1.0 propanoate + 1.0 pyruvate\n",
      "\n",
      "1.0 m02772c + 1.0 m02819s --> 1.0 m02772s + 1.0 m02819c\n",
      "HMR_6017\n",
      "1.0 cysteine + 1.0 H2O --> 1.0 H2S + 1.0 NH3 + 1.0 pyruvate\n",
      "\n",
      "1.0 m01628c + 1.0 m02040c --> 1.0 m02042c + 1.0 m02578c + 1.0 m02819c\n",
      "HMR_4302\n",
      "1.0 acetate + 1.0 pyruvate --> 1.0 acetate + 1.0 pyruvate\n",
      "\n",
      "1.0 m01252c + 1.0 m02819s --> 1.0 m01252s + 1.0 m02819c\n",
      "HMR_6015\n",
      "1.0 dADP + 1.0 PEP --> 1.0 dATP + 1.0 pyruvate\n",
      "\n",
      "1.0 m01637c + 1.0 m02696c --> 1.0 m01642c + 1.0 m02819c\n",
      "HMR_4421\n",
      "1.0 H+ + 1.0 pyruvate --> 1.0 H+ + 1.0 pyruvate\n",
      "\n",
      "1.0 m02039s + 1.0 m02819s --> 1.0 m02039c + 1.0 m02819c\n",
      "HMR_4928\n",
      "1.0 glutamine + 1.0 pyruvate --> 1.0 2-oxoglutaramate + 1.0 alanine\n",
      "\n",
      "1.0 m01975c + 1.0 m02819c --> 1.0 m00672c + 1.0 m01307c\n",
      "HMR_4196\n",
      "1.0 CDP + 1.0 PEP --> 1.0 CTP + 1.0 pyruvate\n",
      "\n",
      "1.0 m01424c + 1.0 m02696c --> 1.0 m01623c + 1.0 m02819c\n",
      "HMR_4193\n"
     ]
    }
   ],
   "source": [
    "for reaction in Hep_G2.metabolites.get_by_id('m02819c').reactions :\n",
    "    print_reactions(reaction)\n",
    "    print(reaction.id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "8ca3a02a",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "GLUCOSE :\n",
      "4,6-dideoxy-4-oxo-dTDP-D-glucose --- m00946c --- C_c\n",
      "ADP-glucose --- m01286c --- C_c\n",
      "ADP-glucose --- m01286s --- C_s\n",
      "ADP-glucose --- m01286x --- C_x\n",
      "beta-D-glucose --- m01388c --- C_c\n",
      "beta-D-glucose-6-phosphate --- m01389c --- C_c\n",
      "dTDP-glucose --- m01751c --- C_c\n",
      "glucose --- m01965c --- C_c\n",
      "glucose --- m01965g --- C_g\n",
      "glucose --- m01965l --- C_l\n",
      "glucose --- m01965r --- C_r\n",
      "glucose --- m01965s --- C_s\n",
      "glucose --- m01965x --- C_x\n",
      "glucose-1-phosphate --- m01967c --- C_c\n",
      "glucose-6-phosphate --- m01968c --- C_c\n",
      "glucose-6-phosphate --- m01968r --- C_r\n",
      "UDP-glucose --- m03108c --- C_c\n",
      "UDP-glucose --- m03108r --- C_r\n"
     ]
    }
   ],
   "source": [
    "# Recherche des réactions impliquant le lactate\n",
    "Lactate_metab = []\n",
    "print(\"GLUCOSE :\")\n",
    "for metab in Hep_G2_unip_ab.metabolites :\n",
    "    if 'glucose' in metab.name :\n",
    "        Lactate_metab.append(metab)       \n",
    "        print(f'{metab.name} --- {metab.id} --- {metab.compartment}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "52d612ee",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LACTATES :\n",
      "L-lactate --- m02403c --- C_c\n",
      "L-lactate --- m02403m --- C_m\n",
      "L-lactate --- m02403p --- C_p\n",
      "L-lactate --- m02403s --- C_s\n",
      "L-lactate --- m02403x --- C_x\n",
      "\n",
      "Reactions associées aux lactates\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Recherche des réactions impliquant le lactate\n",
    "Lactate_metab = []\n",
    "print(\"LACTATES :\")\n",
    "for metab in Hep_G2_unip_ab.metabolites :\n",
    "    if 'L-lactate' in metab.name :\n",
    "        Lactate_metab.append(metab)       \n",
    "        print(f'{metab.name} --- {metab.id} --- {metab.compartment}')\n",
    "print(\"\\nReactions associées aux lactates\\n\")\n",
    "for lac in Lactate_metab :\n",
    "    for reac in lac.reactions :\n",
    "        if reac.flux != 0.0 :\n",
    "            print(f\"ID : {reac.id} --- FLUX : {reac.flux}\\n\")\n",
    "            print_reactions(reac, reac.flux)\n",
    "            print(\"\\n####====================####\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "id": "97993115",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "L-Lactate : \n",
      "\n",
      "Réactions associées au L-Lactate\n",
      " :\n"
     ]
    }
   ],
   "source": [
    "metab_reactions_to_file(Hep_G2, \"L-Lactate\", \"lac_flux.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "id": "01536425",
   "metadata": {},
   "outputs": [],
   "source": [
    "def metab_reactions_to_file(model, metab_name, outfile) :\n",
    "    metab_list = []\n",
    "    \n",
    "    string = f\"{metab_name} : \\n\"\n",
    "    \n",
    "    for metab in model.metabolites :\n",
    "        if metab_name in metab.name :\n",
    "            metab_list.append(metab)\n",
    "            string += metab.name + \" --- \" + {metab.id} + \" --- \" + {metab.compartment} + \"\\n\"\n",
    "            print(string)\n",
    "    string += \"\\nRéactions associées au \" + metab_name + \"\\n :\"\n",
    "    \n",
    "    for m in metab_list :\n",
    "        for reac in m.reactions :\n",
    "            if reac.flux != 0.0 :\n",
    "                s1 = print_reactions(reac, reac.flux, False)[0]\n",
    "                s2 = print_reactions(reac, reac.flux, False)[1]\n",
    "                string += \"ID : \" + reac.id + \" --- FLUX : \" + reac.flux +\"\\n\"\n",
    "                string += s1 + \"\\n\" + s2 + \"\\n\"\n",
    "                string += \"\\n####====================####\\n\"\n",
    "    \n",
    "    with open(outfile, 'w') as out :\n",
    "        out.write(string)\n",
    "    print(string)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3cdaf41b",
   "metadata": {},
   "source": [
    "### Model from 2012 INIT"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "2324a984",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Model does not contain SBML fbc package information.\n",
      "SBML package 'layout' not supported by cobrapy, information is not parsed\n",
      "SBML package 'render' not supported by cobrapy, information is not parsed\n",
      "urn:miriam:9606 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17499 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17499 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17499 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17513 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17908 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18097 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18097 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C00999 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C00999 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C01609 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C02729 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C04253 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C04570 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C07292 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C07292 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C07292 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C07293 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C07293 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:kegg.compound:C07293 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28017 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m46 \"Starch\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28102 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m47 \"Amylose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16548 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m48 \"Chlordecone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m49 \"Protein-Tyrosines\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C00585 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m50 \"Protein tyrosine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C00585 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m51 \"Protein tyrosine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C00585 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m52 \"Protein tyrosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28516 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m53 \"1,2-Dihydronaphthalene-1,2-diol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28516 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m54 \"1,2-Dihydronaphthalene-1,2-diol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28516 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m55 \"1,2-Dihydronaphthalene-1,2-diol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:34048 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m56 \"1,2-Dihydroxy-3,4-epoxy-1,2,3,4-tetrahydronaphthalene\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m57 \"5-hydroxytryptophol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C01167 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m58 \"Protein tyrosine phosphate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C01167 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m59 \"Protein tyrosine phosphate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C01167 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m60 \"Protein tyrosine phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17890 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m61 \"Indole-3-ethanol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16765 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m62 \"Tryptamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16765 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m63 \"Tryptamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28790 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m64 \"Serotonin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28790 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m65 \"Serotonin\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16946 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m66 \"L-Kynurenine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28683 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m67 \"Kynurenine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1547 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m68 \"3-Hydroxykynurenine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17380 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m69 \"3-Hydroxy-L-kynurenine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17380 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m70 \"3-Hydroxy-L-kynurenine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m71 \"serotonin _ST_i_LT_O_ST_/i_LT_-sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28997 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m72 \"Deoxyinosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28997 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m73 \"Deoxyinosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18191 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m74 \"Thioredoxin disulfide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18191 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m75 \"Thioredoxin disulfide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18191 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m76 \"Thioredoxin disulfide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17596 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m77 \"Inosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17596 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m78 \"Inosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18107 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m79 \"Xanthosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18107 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m80 \"Xanthosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17489 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m81 \"3',5'-Cyclic AMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16356 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m82 \"3',5'-Cyclic GMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16356 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m83 \"3',5'-Cyclic GMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m84 \"3-_ST_i_LT_O_ST_/i_LT_-methyldopa\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15842 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m85 \"Orotidine 5'-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17202 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m86 \"IMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17202 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m87 \"IMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17202 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m88 \"IMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17202 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m89 \"IMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15652 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m90 \"Xanthosine 5'-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15652 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m91 \"Xanthosine 5'-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17256 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m92 \"Deoxyadenosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17256 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m93 \"Deoxyadenosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17256 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m94 \"Deoxyadenosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17256 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m95 \"Deoxyadenosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16335 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m96 \"Adenosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16335 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m97 \"Adenosine\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17172 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m98 \"Deoxyguanosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17172 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m99 \"Deoxyguanosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17172 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m100 \"Deoxyguanosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17172 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m101 \"Deoxyguanosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m102 \"Guanosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m103 \"Guanosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m104 \"Guanosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17381 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m105 \"Porphobilinogen\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17748 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m106 \"Thymidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17748 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m107 \"Thymidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17748 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m108 \"Thymidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17808 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m109 \"IDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17808 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m110 \"IDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17808 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m111 \"IDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17808 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m112 \"IDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15967 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m113 \"Thioredoxin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15967 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m114 \"Thioredoxin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15967 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m115 \"Thioredoxin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17709 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m116 \"Adenylyl sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17713 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m117 \"dAMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17713 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m118 \"dAMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17713 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m119 \"dAMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17713 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m120 \"dAMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16027 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m121 \"AMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16027 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m122 \"AMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16027 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m123 \"AMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16027 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m124 \"AMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16027 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m125 \"AMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16027 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m126 \"AMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16192 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m127 \"dGMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16192 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m128 \"dGMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16192 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m129 \"dGMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m130 \"GMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m131 \"GMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m132 \"GMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m133 \"GMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m134 \"GMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m135 \"GMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27928 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m136 \"Parathion\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27827 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m137 \"Paraoxon\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17013 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m138 \"dTMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17013 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m139 \"dTMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17013 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m140 \"dTMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17013 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m141 \"dTMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17013 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m142 \"dTMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16039 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m143 \"ITP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16039 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m144 \"ITP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16039 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m145 \"ITP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16039 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m146 \"ITP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18381 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m147 \"1-(5'-Phosphoribosyl)-5-formamido-4-imidazolecarboxamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16761 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m148 \"ADP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16761 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m149 \"ADP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16761 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m150 \"ADP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16761 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m151 \"ADP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16761 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m152 \"ADP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16761 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m153 \"ADP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17985 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m154 \"Adenosine 3',5'-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17985 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m155 \"Adenosine 3',5'-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17985 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m156 \"Adenosine 3',5'-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28862 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m157 \"dGDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28862 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m158 \"dGDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28862 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m159 \"dGDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17552 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m160 \"GDP\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17552 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m161 \"GDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17552 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m162 \"GDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17552 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m163 \"GDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17552 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m164 \"GDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17552 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m165 \"GDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17980 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m166 \"3'-Phosphoadenylyl sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17980 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m167 \"3'-Phosphoadenylyl sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17980 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m168 \"3'-Phosphoadenylyl sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16174 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m169 \"dADP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16174 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m170 \"dADP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16174 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m171 \"dADP\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05588 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m172 \"L-Metanephrine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05588 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m173 \"L-Metanephrine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18075 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m174 \"dTDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18075 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m175 \"dTDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18075 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m176 \"dTDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18075 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m177 \"dTDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18075 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m178 \"dTDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18075 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m179 \"dTDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16284 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m180 \"dATP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16284 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m181 \"dATP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16284 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m182 \"dATP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15422 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m183 \"ATP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15422 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m184 \"ATP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15422 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m185 \"ATP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15422 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m186 \"ATP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15422 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m187 \"ATP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15422 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m188 \"ATP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15422 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m189 \"ATP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16497 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m190 \"dGTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16497 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m191 \"dGTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16497 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m192 \"dGTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15996 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m193 \"GTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15996 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m194 \"GTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15996 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m195 \"GTP\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C06605 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m196 \"Aminoparathion\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18077 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m197 \"dTTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18077 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m198 \"dTTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18077 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m199 \"dTTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18077 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m200 \"dTTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18077 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m201 \"dTTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18077 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m202 \"dTTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16856 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m203 \"Glutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16856 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m204 \"Glutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16856 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m205 \"Glutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16856 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m206 \"Glutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16856 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m207 \"Glutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15840 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m208 \"N-Acylneuraminate 9-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:31529 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m209 \"Ecgonine methyl ester\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15682 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m210 \"N-(L-Arginino)succinate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1611049183 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m211 \"Phosphatidylcholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1611049183 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m212 \"Phosphatidylcholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1611049183 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m213 \"Phosphatidylcholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1611049183 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m214 \"Phosphatidylcholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16807 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m215 \"S-Acetyldihydrolipoamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m216 \"spermic acid 2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m217 \"spermic acid 2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m218 \"spermic acid 2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17211 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m219 \"Geranyl diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m220 \"spermine dialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m221 \"spermine dialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m222 \"spermine dialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m223 \"spermine dialdehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:36707 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m224 \"2-Acetyl-1-alkyl-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:36707 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m225 \"2-Acetyl-1-alkyl-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:36707 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m226 \"2-Acetyl-1-alkyl-sn-glycero-3-phosphocholine\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m227 \"deoxyhypusine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m228 \"spermine monoaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15746 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m229 \"Spermine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15746 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m230 \"Spermine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17184 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m231 \"Chlordecone alcohol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18344 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m232 \"4-Hydroxy-2-quinolinecarboxylic acid\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C02470 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m233 \"Xanthurenic acid\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C02470 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m234 \"Xanthurenic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16482 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m235 \"Naphthalene\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16482 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m236 \"Naphthalene\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16482 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m237 \"Naphthalene\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:10432 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m238 \"2-Naphthol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:33998 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m239 \"(1R,2S)-Naphthalene 1,2-oxide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:33998 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m240 \"(1R,2S)-Naphthalene 1,2-oxide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:33998 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m241 \"(1R,2S)-Naphthalene 1,2-oxide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:33998 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m242 \"(1R,2S)-Naphthalene 1,2-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14787 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m243 \"(1S,2R)-Naphthalene 1,2-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14787 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m244 \"(1S,2R)-Naphthalene 1,2-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14787 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m245 \"(1S,2R)-Naphthalene 1,2-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14787 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m246 \"(1S,2R)-Naphthalene 1,2-oxide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17435 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m247 \"Naphthalene-1,2-diol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18086 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m248 \"Indole-3-acetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18086 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m249 \"Indole-3-acetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16411 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m250 \"Indole-3-acetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16411 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m251 \"Indole-3-acetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:50157 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m252 \"5-Hydroxyindoleacetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:50157 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m253 \"5-Hydroxyindoleacetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27823 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m254 \"5-Hydroxyindoleacetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27823 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m255 \"5-Hydroxyindoleacetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17442 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m256 \"4-(2-Aminophenyl)-2,4-dioxobutanoate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05645 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m257 \"4-(2-Amino-3-hydroxyphenyl)-2,4-dioxobutanoate\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:kegg.compound:C05645 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m258 \"4-(2-Amino-3-hydroxyphenyl)-2,4-dioxobutanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16828 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m259 \"L-Tryptophan\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16828 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m260 \"L-Tryptophan\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16828 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m261 \"L-Tryptophan\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17780 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m262 \"5-Hydroxy-L-tryptophan\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28171 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m263 \"5-Hydroxytryptophan\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30249 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m264 \"L-Formylkynurenine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27748 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m265 \"Nicotinate D-ribonucleoside\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27748 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m266 \"Nicotinate D-ribonucleoside\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15927 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m267 \"N-Ribosylnicotinamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m268 \"nicotinamide riboside\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16171 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m269 \"Nicotinamide D-ribonucleotide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16171 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m270 \"Nicotinamide D-ribonucleotide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16171 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m271 \"Nicotinamide D-ribonucleotide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17509 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m272 \"5'-Methylthioadenosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15763 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m273 \"Nicotinate D-ribonucleotide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15763 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m274 \"Nicotinate D-ribonucleotide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15763 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m275 \"Nicotinate D-ribonucleotide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17615 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m276 \"1,2-Diacyl-3-beta-D-galactosyl-sn-glycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16749 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m277 \"1-Phosphatidyl-D-myo-inositol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16749 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m278 \"1-Phosphatidyl-D-myo-inositol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16749 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m279 \"1-Phosphatidyl-D-myo-inositol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16749 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m280 \"1-Phosphatidyl-D-myo-inositol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16749 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m281 \"1-Phosphatidyl-D-myo-inositol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m282 \"1-Phosphatidyl-1D-myo-inositol 4-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m283 \"1-Phosphatidyl-1D-myo-inositol 4-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m284 \"1-Phosphatidyl-1D-myo-inositol 4-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m285 \"1-Phosphatidyl-1D-myo-inositol 4-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m286 \"1-Phosphatidyl-1D-myo-inositol 4-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17283 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m287 \"1-Phosphatidyl-1D-myo-inositol 3-phosphate\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17283 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m288 \"1-Phosphatidyl-1D-myo-inositol 3-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17283 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m289 \"1-Phosphatidyl-1D-myo-inositol 3-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17283 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m290 \"1-Phosphatidyl-1D-myo-inositol 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m291 \"1-phosphatidyl-1D-myo-inositol 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m292 \"N-Acetyl-D-muramoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17012 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m293 \"N-Acetylneuraminate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1834828910 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m294 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1834828910 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m295 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1834828910 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m296 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1834828910 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m297 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1834828910 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m298 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16152 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m299 \"1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16152 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m300 \"1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16152 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m301 \"1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16152 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m302 \"1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16851 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m303 \"1-Phosphatidyl-1D-myo-inositol 3,5-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m304 \"1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16927 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m305 \"N6-(L-1,3-Dicarboxypropyl)-L-lysine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16927 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m306 \"N6-(L-1,3-Dicarboxypropyl)-L-lysine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16732 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m307 \"CDP-ethanolamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16732 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m308 \"CDP-ethanolamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16732 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m309 \"CDP-ethanolamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27438 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m310 \"N-Acetylneuraminate 9-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16618 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m311 \"Phosphatidylinositol-3,4,5-trisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16618 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m312 \"Phosphatidylinositol-3,4,5-trisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m313 \"1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16858 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m314 \"Pantetheine 4'-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m315 \"1,4-benzothiazine-o-quinonimine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m316 \"3,4-dihydro-1,4-benzothiazine-3-carboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m317 \"5-(S-L-cysteinyl)-dopaquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m318 \"5-S-cysteinyldopa\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18385 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m319 \"Thiamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18385 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m320 \"Thiamine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14865 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m321 \"2-(S-Glutathionyl)acetyl chloride\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14865 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m322 \"2-(S-Glutathionyl)acetyl chloride\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14871 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m323 \"S-(Formylmethyl)glutathione\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14871 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m324 \"S-(Formylmethyl)glutathione\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14871 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m325 \"S-(Formylmethyl)glutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:9532 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m326 \"Thiamin diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:9532 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m327 \"Thiamin diphosphate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14874 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m328 \"Glutathione episulfonium ion\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14874 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m329 \"Glutathione episulfonium ion\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14874 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m330 \"Glutathione episulfonium ion\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:9534 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m331 \"Thiamin triphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35896 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m332 \"S-(2-Hydroxyethyl)glutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17432 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m333 \"S-Succinyldihydrolipoamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m334 \"4(R)-hydroxy-dodec-6Z-enoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17992 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m335 \"Sucrose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:36219 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m336 \"Lactose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:36219 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m337 \"Lactose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:36219 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m338 \"Lactose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:4808 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m339 \"Epimelibiose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15769 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m340 \"(R)-4'-Phosphopantothenoyl-L-cysteine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C15658 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m341 \"6-(alpha-D-Glucosaminyl)-1D-myo-inositol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17577 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m342 \"S-(2-Methylpropanoyl)-dihydrolipoamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27527 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m343 \"Melibiitol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17312 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m344 \"N1-Acetylspermine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18319 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m345 \"1-(5'-Phosphoribosyl)-5-amino-4-(N-succinocarboxamide)-imidazole\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15694 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m346 \"(R)-S-Lactoylglutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15694 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m347 \"(R)-S-Lactoylglutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28391 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m348 \"[Dihydrolipoyllysine-residue succinyltransferase] S-glutaryldihydrolipoyllysine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28391 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m349 \"[Dihydrolipoyllysine-residue succinyltransferase] S-glutaryldihydrolipoyllysine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28692 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m350 \"S-(2-Methylbutanoyl)-dihydrolipoamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27462 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m351 \"S-(3-Methylbutanoyl)-dihydrolipoamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15919 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m352 \"N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16389 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m353 \"Ubiquinone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16389 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m354 \"Ubiquinone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17962 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m355 \"CDP-diacylglycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17962 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m356 \"CDP-diacylglycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16680 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m357 \"S-Adenosyl-L-homocysteine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17976 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m358 \"Ubiquinol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17976 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m359 \"Ubiquinol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m360 \"6(R)-hydroxy-tetradeca-2E,4E,8Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m361 \"6(S)-hydroxy-tetradeca-2E,4E,8Z-trienoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16082 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m362 \"UDP-D-xylose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16082 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m363 \"UDP-D-xylose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m364 \"beta-D-Glucuronosyl-(1-_LT_4)-N-acetyl-alpha-D-glucosaminylproteoglycan\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:978 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m365 \"2-(alpha-Hydroxyethyl)thiamine diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:978 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m366 \"2-(alpha-Hydroxyethyl)thiamine diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15625 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m367 \"S-Adenosylmethioninamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m368 \"6(R)-hydroxy-tetradeca-2E,8Z-dienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m369 \"6(R)-hydroxy-tetradeca-4E,8Z-dienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m370 \"6(S)-hydroxy-tetradeca-2E,8Z-dienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m371 \"6(S)-hydroxy-tetradeca-4E,8Z-dienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m372 \"3-oxo-6(R)-hydroxy-tetradec-8-cis-enoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m373 \"3-oxo-6(S)-hydroxy-tetradec-8Z-enoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28920 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m374 \"N-Acetylmuramoyl-Ala\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1611722783 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m375 \"beta-D-Galactosyl-1,3-N-acetyl-alpha-D-galactosaminyl-R\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1611722783 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m376 \"beta-D-Galactosyl-1,3-N-acetyl-alpha-D-galactosaminyl-R\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1611722783 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m377 \"beta-D-Galactosyl-1,3-N-acetyl-alpha-D-galactosaminyl-R\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16153 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m378 \"N-Acetyllactosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16153 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m379 \"N-Acetyllactosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16153 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m380 \"N-Acetyllactosamine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04755 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m381 \"Glycoprotein N-acetyl-D-glucosaminyl-phospho-D-mannose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m382 \"3(S),6(R)-dihydroxy-tetradec-8Z-enoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m383 \"3(S),6(S)-dihydroxy-tetradec-8Z-enoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16436 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m384 \"CDP-choline\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16436 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m385 \"CDP-choline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m386 \"3,5,3',5'-tetraiodothyroacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m387 \"3,5,3'-triiodothyroacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m388 \"3',5'-diiodothyroacetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18332 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m389 \"O-(4-Hydroxy-3,5-diidophenyl)-3,5-diiodo-L-tyrosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18332 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m390 \"O-(4-Hydroxy-3,5-diidophenyl)-3,5-diiodo-L-tyrosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18258 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m391 \"Triiodothyronine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18258 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m392 \"Triiodothyronine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18258 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m393 \"Triiodothyronine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m394 \"3,3',5' - triiodothyronine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m395 \"3,3',5' - triiodothyronine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m396 \"3,3'-diiodo-L-thyronine 4'-O-sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m397 \"3,3'-diiodo-L-thyronine 4'-O-sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m398 \"3,3'-diiodo-L-thyronine 4'-O-sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m399 \"3',5'-diiodo-L-thyronine 4'-O-sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m400 \"3,3'-diiodo-L-thyronine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m401 \"3,3'-diiodo-L-thyronine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m402 \"3,3'-diiodo-L-thyronine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m403 \"3',5'-diiodo-L-thyronine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m404 \"3'-monoiodo-L-thyronine 4'-O-sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m405 \"3'-monoiodo-L-thyronine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17200 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m406 \"UDP-glucuronate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17200 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m407 \"UDP-glucuronate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15414 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m408 \"S-Adenosyl-L-methionine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15414 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m409 \"S-Adenosyl-L-methionine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16960 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m410 \"ADP-ribose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16960 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m411 \"ADP-ribose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1806652249 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m412 \"UDP-glucose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1806652249 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m413 \"UDP-glucose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1806652249 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m414 \"UDP-glucose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18307 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m415 \"UDP-D-galactose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18307 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m416 \"UDP-D-galactose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18307 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m417 \"UDP-D-galactose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17407 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m418 \"trans,trans-Farnesyl diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17407 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m419 \"trans,trans-Farnesyl diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m420 \"3'-carboxy-alpha-chromanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m421 \"3'-carboxy-alpha-chromanol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16955 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m422 \"GDP-4-dehydro-6-deoxy-D-mannose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m423 \"8(R)-hydroxy-hexadeca-2E,4E,6E,10Z-tetraenoate\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m424 \"8(S)-hydroxy-hexadeca-2E,4E,6E,10Z-tetraenoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1463 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m425 \"3-Carboxy-1-hydroxypropyl-ThPP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17009 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m426 \"GDP-L-fucose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17009 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m427 \"GDP-L-fucose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15820 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m428 \"GDP-mannose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m429 \"8(R)-hydroxy-hexadeca-2E,6E,10Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m430 \"8(R)-hydroxy-hexadeca-4E,6E,10Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m431 \"8(S)-hydroxy-hexadeca-2E,6E,10Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m432 \"8(S)-hydroxy-hexadeca-4E,6E-10Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m433 \"3-oxo-8(R)-hydroxy-hexadeca-6E,10Z-dienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m434 \"3-oxo-8(S)-hydroxy-hexadeca-6E,10Z-dienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m435 \"3(S),8(R)-dihydroxy-6E,10Z-hexadecadienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m436 \"3(S),8(S)-dihydroxy-6E,10Z-hexadecadienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m437 \"palmitoleate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m438 \"palmitoleate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m439 \"palmitoleate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m440 \"palmitoleate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17600 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m441 \"Hexadecanal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17600 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m442 \"Hexadecanal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15756 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m443 \"Hexadecanoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15756 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m444 \"Hexadecanoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17015 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m445 \"Riboflavin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17015 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m446 \"Riboflavin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17621 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m447 \"FMN\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17621 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m448 \"FMN\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27958 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m449 \"Cocaine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17607 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m450 \"Reduced riboflavin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16048 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m451 \"Reduced FMN\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m452 \"12-oxo-(5Z,8Z,10E)-heptadecatrienoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28396 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m453 \"Digalactosyl-diacylglycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16264 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m454 \"UDP-N-acetyl-D-glucosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16264 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m455 \"UDP-N-acetyl-D-glucosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16264 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m456 \"UDP-N-acetyl-D-glucosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16264 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m457 \"UDP-N-acetyl-D-glucosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16650 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m458 \"UDP-N-acetyl-D-galactosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16287 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m459 \"UDP-N-acetyl-D-mannosamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m460 \"12S-HHT\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17049 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m461 \"6-(alpha-D-Glucosaminyl)-1-phosphatidyl-1D-myo-inositol\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17049 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m462 \"6-(alpha-D-Glucosaminyl)-1-phosphatidyl-1D-myo-inositol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17049 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m463 \"6-(alpha-D-Glucosaminyl)-1-phosphatidyl-1D-myo-inositol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m464 \"estrone-3,4-quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m465 \"estrone-3,4-quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m466 \"estrone-3,4-quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m467 \"estrone-3,4-quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m468 \"estrone-3,4-semiquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m469 \"estrone-3,4-semiquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m470 \"estrone-2,3-semiquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m471 \"estrone-2,3-semiquinone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17263 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m472 \"Estrone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17263 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m473 \"Estrone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17263 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m474 \"Estrone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17263 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m475 \"Estrone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1156 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m476 \"2-Hydroxyestrone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1156 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m477 \"2-Hydroxyestrone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1156 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m478 \"2-Hydroxyestrone\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05300 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m479 \"16alpha-Hydroxyestrone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m480 \"4-hydroxyestrone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m481 \"4-hydroxyestrone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m482 \"4-hydroxyestrone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m483 \"17beta-estradiol-3,4-quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m484 \"17beta-estradiol-3,4-quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m485 \"17beta-estradiol-3,4-quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m486 \"17beta-estradiol-3,4-quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m487 \"17beta-estradiol-3,4-semiquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m488 \"17beta-estradiol-3,4-semiquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m489 \"17beta-estradiol-2,3-semiquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m490 \"17beta-estradiol-2,3-semiquinone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m491 \"Estrone 3-sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m492 \"Estrone 3-sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m493 \"Estrone 3-sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m494 \"Estrone 3-sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m495 \"Estrone 3-sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m496 \"5-S-glutathionyl-aminochrome reduced\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16469 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m497 \"Estradiol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16469 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m498 \"Estradiol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27974 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m499 \"Estriol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28744 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m500 \"2-Hydroxyestradiol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28744 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m501 \"2-Hydroxyestradiol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28744 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m502 \"2-Hydroxyestradiol\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m503 \"4-hydroxy-17beta-estradiol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m504 \"4-hydroxy-17beta-estradiol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m505 \"4-hydroxy-17beta-estradiol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m506 \"omega-carboxy-trinor-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m507 \"omega-carboxy-trinor-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m508 \"5-S-glutathionyl-dopamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m509 \"3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m510 \"3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m511 \"3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m512 \"3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m513 \"3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m514 \"3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m515 \"3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m516 \"3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28661 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m517 \"(6Z,9Z,12Z)-Octadecatrienoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27432 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m518 \"(9Z,12Z,15Z)-Octadecatrienoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27432 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m519 \"(9Z,12Z,15Z)-Octadecatrienoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m520 \"13-hydroxy-(9Z,11E)-octadecadienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m521 \"9-hydroxyoctadecadienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m522 \"9-hydroxyoctadecadienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m523 \"9-hydroxyoctadecadienoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16634 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m524 \"Raffinose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m525 \"D-Gal alpha 1-_LT_6D-Gal alpha 1-_LT_6D-Glucose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17351 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m526 \"Linoleate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17351 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m527 \"Linoleate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17351 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m528 \"Linoleate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17351 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m529 \"Linoleate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14825 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m530 \"9(10)-EpOME\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14826 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m531 \"12(13)-EpOME\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15655 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m532 \"(9Z,11E)-(13S)-13-Hydroperoxyoctadeca-9,11-dienoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15655 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m533 \"(9Z,11E)-(13S)-13-Hydroperoxyoctadeca-9,11-dienoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:34498 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m534 \"9(S)-HPODE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:34498 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m535 \"9(S)-HPODE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m536 \"9,10-hydroxyoctadec-12(Z)-enoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m537 \"12,13-hydroxyoctadec-9(Z)-enoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16196 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m538 \"(9Z)-Octadecenoic acid\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C01530 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m539 \"Octadecanoic acid\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C01530 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m540 \"Octadecanoic acid\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C01530 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m541 \"Octadecanoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16393 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m542 \"Sphingosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16393 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m543 \"Sphingosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16393 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m544 \"Sphingosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16393 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m545 \"Sphingosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16393 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m546 \"Sphingosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17862 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m547 \"3-Dehydrosphinganine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17862 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m548 \"3-Dehydrosphinganine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17862 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m549 \"3-Dehydrosphinganine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:37550 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m550 \"Sphingosine 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:37550 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m551 \"Sphingosine 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:37550 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m552 \"Sphingosine 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16566 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m553 \"Sphinganine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16566 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m554 \"Sphinganine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16566 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m555 \"Sphinganine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16566 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m556 \"Sphinganine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16566 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m557 \"Sphinganine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16893 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m558 \"Sphinganine 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16893 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m559 \"Sphinganine 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16893 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m560 \"Sphinganine 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16893 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m561 \"Sphinganine 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15633 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m562 \"Dihydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15633 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m563 \"Dihydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15635 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m564 \"Tetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15635 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m565 \"Tetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05285 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m566 \"Adrenosterone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m567 \"5-S-glutathionyl-L-DOPA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m568 \"5-S-glutathionyl-adrenochrome hydroquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m569 \"13E-tetranor-16-carboxy-LTE4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16422 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m570 \"Androstenedione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16422 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m571 \"Androstenedione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27967 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m572 \"11beta-Hydroxyandrost-4-ene-3,17-dione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m573 \"omega-COOH-tetranor-LTE3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m574 \"5'-carboxy-alpha-chromanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m575 \"5'-carboxy-alpha-chromanol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17347 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m576 \"Testosterone\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17347 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m577 \"Testosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15994 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m578 \"5alpha-Androstane-3,17-dione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15994 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m579 \"5alpha-Androstane-3,17-dione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28689 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m580 \"Dehydroepiandrosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28689 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m581 \"Dehydroepiandrosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28689 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m582 \"Dehydroepiandrosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28689 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m583 \"Dehydroepiandrosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28689 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m584 \"Dehydroepiandrosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16985 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m585 \"5beta-Androstane-3,17-dione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16814 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m586 \"Dehydroepiandrosterone sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16814 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m587 \"Dehydroepiandrosterone sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16814 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m588 \"Dehydroepiandrosterone sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16814 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m589 \"Dehydroepiandrosterone sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16814 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m590 \"Dehydroepiandrosterone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m591 \"androsterone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m592 \"androsterone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m593 \"androsterone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m594 \"5alpha-dihydrotestosterone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m595 \"5alpha-dihydrotestosterone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m596 \"5alpha-dihydrotestosterone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m597 \"6-(_ST_i_LT_N_ST_/i_LT_-acetyl-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1219415935 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m598 \"N-Acetyl-D-glucosaminylphosphatidylinositol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16032 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m599 \"Androsterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16032 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m600 \"Androsterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16032 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m601 \"Androsterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16330 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m602 \"Dihydrotestosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16330 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m603 \"Dihydrotestosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16330 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m604 \"Dihydrotestosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28195 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m605 \"3alpha-Hydroxy-5beta-androstan-17-one\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28195 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m606 \"3alpha-Hydroxy-5beta-androstan-17-one\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1776152639 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m607 \"N-Acylsphingosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1776152639 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m608 \"N-Acylsphingosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1776152639 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m609 \"N-Acylsphingosine\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1776152639 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m610 \"N-Acylsphingosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1776152639 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m611 \"N-Acylsphingosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1776152639 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m612 \"N-Acylsphingosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16197 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m613 \"Ceramide 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16197 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m614 \"Ceramide 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16197 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m615 \"Ceramide 1-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m616 \"pristanate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m617 \"(2S,6R,10R)-pristanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:31488 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m618 \"Dihydroceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:31488 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m619 \"Dihydroceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:31488 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m620 \"Dihydroceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:31488 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m621 \"Dihydroceramide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m622 \"2(S)-pristanal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m623 \"2(R)-pristanal\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14849 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m624 \"Benzo[a]pyrene-9,10-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14849 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m625 \"Benzo[a]pyrene-9,10-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14849 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m626 \"Benzo[a]pyrene-9,10-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14850 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m627 \"Benzo[a]pyrene-7,8-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14850 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m628 \"Benzo[a]pyrene-7,8-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14850 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m629 \"Benzo[a]pyrene-7,8-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14851 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m630 \"Benzo[a]pyrene-4,5-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14851 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m631 \"Benzo[a]pyrene-4,5-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14851 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m632 \"Benzo[a]pyrene-4,5-oxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m633 \"3,5,3',5'-tetraiodothyroacetate-beta-D-glucuronoside\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14852 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m634 \"Benzo[a]pyrene-7,8-diol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14852 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m635 \"Benzo[a]pyrene-7,8-diol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14852 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m636 \"Benzo[a]pyrene-7,8-diol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m637 \"3,5,3'-triiodothyroacetate-beta-D-glucuronoside\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m638 \"3',5'-diiodothyroacetate-beta-D-glucuronoside\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15638 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m639 \"5,10-Methenyltetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15638 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m640 \"5,10-Methenyltetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15638 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m641 \"5,10-Methenyltetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1989 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m642 \"5,10-Methylenetetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1989 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m643 \"5,10-Methylenetetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15637 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m644 \"10-Formyltetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15637 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m645 \"10-Formyltetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15640 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m646 \"5-Formyltetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15640 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m647 \"5-Formyltetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15639 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m648 \"5-Formiminotetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15639 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m649 \"5-Formiminotetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14791 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m650 \"(1R)-Hydroxy-(2R)-glutathionyl-1,2-dihydronaphthalene\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14791 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m651 \"(1R)-Hydroxy-(2R)-glutathionyl-1,2-dihydronaphthalene\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14792 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m652 \"(1S)-Hydroxy-(2S)-glutathionyl-1,2-dihydronaphthalene\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14792 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m653 \"(1S)-Hydroxy-(2S)-glutathionyl-1,2-dihydronaphthalene\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14793 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m654 \"(1R)-Glutathionyl-(2R)-hydroxy-1,2-dihydronaphthalene\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14793 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m655 \"(1R)-Glutathionyl-(2R)-hydroxy-1,2-dihydronaphthalene\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15641 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m656 \"5-Methyltetrahydrofolate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15641 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m657 \"5-Methyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m658 \"12-oxo-20-carboxy-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m659 \"12-oxo-20-carboxy-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m660 \"12-oxo-20-carboxy-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m661 \"leukotriene A5\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m662 \"5-oxo-EPE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m663 \"5-oxo-EPE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m664 \"5(S),6(S)-epoxy-15(R)-HEPE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17898 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m665 \"Retinal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1536735291 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m666 \"Retinoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m667 \"20-carboxy-leukotriene-B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m668 \"20-carboxy-leukotriene-B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m669 \"20-carboxy-leukotriene-B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m670 \"20-carboxy-leukotriene-B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m671 \"12-oxo-10,11-dihydro-20-COOH-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m672 \"12-oxo-10,11-dihydro-20-COOH-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m673 \"12-oxo-10,11-dihydro-20-COOH-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m674 \"timnodonate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m675 \"timnodonate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m676 \"timnodonate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m677 \"5-oxo-(6E,8Z,11Z,14Z)-eicosatetraenoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m678 \"5-oxo-(6E,8Z,11Z,14Z)-eicosatetraenoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m679 \"15(R)-HEPE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m680 \"5-HEPE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m681 \"5-HEPE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m682 \"11R-HEPE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m683 \"18R-HEPE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m684 \"5(S),6(S)-epoxy-15(R)-hydroxyeicosatetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m685 \"5(S),6(S)-epoxy-15(S)-hydroxy-7E,9E,11Z,13E-eicosatetraenoic acid anion\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m686 \"5(S),6(S)-epoxy-15(S)-hydroxy-7E,9E,11Z,13E-eicosatetraenoic acid anion\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m687 \"5(S),6(S)-epoxy-15(S)-hydroxy-7E,9E,11Z,13E-eicosatetraenoic acid anion\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m688 \"delta12-Prostaglandin J2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m689 \"delta12-Prostaglandin J2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m690 \"12-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m691 \"12-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m692 \"12-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m693 \"12-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m694 \"5-oxo-6-trans-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m695 \"5-oxo-6-trans-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m696 \"5-oxo-6-trans-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m697 \"5-oxo-6-trans-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m698 \"5-oxo-6E-12-epi-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m699 \"5-hydroperoxy-EPA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m700 \"15(S)-hydroperoxy-EPE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m701 \"leukotriene B5\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m702 \"leukotriene B5\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m703 \"15(R)-hydroperoxy-EPE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m704 \"15(R)-hydroperoxy-EPE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m705 \"5-oxo-12(S)-hydroxy-eicosa-6E,8Z,10E,14Z-tetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m706 \"5-oxo-12(S)-hydroxy-eicosa-6E,8Z,10E,14Z-tetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m707 \"5-oxo-12(S)-hydroxy-eicosa-6E,8Z,10E,14Z-tetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m708 \"20-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m709 \"20-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m710 \"20-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m711 \"12-oxo-20-hydroxy-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m712 \"12-oxo-20-hydroxy-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m713 \"15-oxo-lipoxin A4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m714 \"PGH3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m715 \"15-epi-lipoxin A5\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m716 \"15-epi-lipoxin B5\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m717 \"20-hydroxy-leukotriene B5\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m718 \"12-oxo-20-dihydroxy-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m719 \"12-oxo-20-dihydroxy-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m720 \"11-dehydro-15-keto-TXB2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m721 \"12-oxo-20-trihydroxy-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m722 \"12-oxo-20-trihydroxy-leukotriene B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17336 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m723 \"Retinol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m724 \"13-cis-retinol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m725 \"13-cis-retinol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15651 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m726 \"Leukotriene A4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15651 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m727 \"Leukotriene A4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27820 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m728 \"Prostaglandin A2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27820 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m729 \"Prostaglandin A2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28099 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m730 \"Prostaglandin B2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27555 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m731 \"Prostaglandin C2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27485 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m732 \"Prostaglandin J2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27485 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m733 \"Prostaglandin J2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15547 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m734 \"(5Z,13E)-11alpha-Hydroxy-9,15-dioxoprost-13-enoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m735 \"20-COOH-10,11-dihydro-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m736 \"20-COOH-10,11-dihydro-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m737 \"20-COOH-10,11-dihydro-LTB4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16556 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m738 \"CMP-N-acetylneuraminate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16556 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m739 \"CMP-N-acetylneuraminate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16556 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m740 \"CMP-N-acetylneuraminate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m741 \"12-hydroxyeicosatetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m742 \"12-hydroxyeicosatetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m743 \"11-hydroxyeicosatetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m744 \"15(R)-Hydroxy-(5Z,8Z,11Z,13E)-eicosatetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m745 \"8-hydroxyeicosatetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m746 \"13-HETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m747 \"7-HETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m748 \"10-HETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m749 \"18-HETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m750 \"17-HETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m751 \"6-trans-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m752 \"6-trans-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m753 \"6-trans-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m754 \"11,12-dihydroxy-5(E),7(E),9(E),14(Z)-eicosatetraenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m755 \"6,7-dihydro-5-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m756 \"6,7-dihydro-5-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m757 \"6,7-dihydro-5-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m758 \"6,7-dihydro-5-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m759 \"6,7-dihydro-5-oxo-12-epi-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m760 \"6,7-dihydro-5-oxo-12-epi-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m761 \"6,7-dihydro-5-oxo-12-epi-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m762 \"6,7-dihydro-5-oxo-12-epi-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m763 \"20-hydroxy-5S-HETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m764 \"10,11-dihydro-12-oxo-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m765 \"10,11-dihydro-12-oxo-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m766 \"10,11-dihydro-12-oxo-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m767 \"9-deoxy-delta12-PGD2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m768 \"9-deoxy-delta12-PGD2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m769 \"5,12-DiHETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m770 \"5,12-DiHETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m771 \"5,12-DiHETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m772 \"5,12-DiHETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m773 \"12,20-DiHETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m774 \"14,15-DiHETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m775 \"15-epi-lipoxin A4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m776 \"15-epi-lipoxin B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m777 \"13,14-dihydro-15-oxo-lipoxin A4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m778 \"5,12,20-TriHETE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m779 \"15-oxo-PGE1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m780 \"(5Z,13E)-(15S)-11-_AND_alpha;,15-dihydroxy-9-oxoprosta-5,13-dienoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m781 \"11-dehydrothromboxane B2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m782 \"20-dihydroxyleukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m783 \"11-dehydro-13,14-dihydro-15-oxo-TXB2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m784 \"20-trihydroxy-leukotriene-B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17858 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m785 \"Glutathione disulfide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17858 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m786 \"Glutathione disulfide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17858 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m787 \"Glutathione disulfide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17858 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m788 \"Glutathione disulfide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15843 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m789 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15843 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m790 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15843 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m791 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15558 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m792 \"(15S)-15-Hydroxy-5,8,11-cis-13-trans-eicosatetraenoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15558 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m793 \"(15S)-15-Hydroxy-5,8,11-cis-13-trans-eicosatetraenoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28209 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m794 \"5(S)-HETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28209 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m795 \"5(S)-HETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28209 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m796 \"5(S)-HETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28209 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m797 \"5(S)-HETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:34185 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m798 \"19(S)-HETE\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14768 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m799 \"5,6-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14768 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m800 \"5,6-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14768 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m801 \"5,6-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14769 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m802 \"8,9-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14769 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m803 \"8,9-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14769 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m804 \"8,9-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14770 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m805 \"11,12-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14770 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m806 \"11,12-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14770 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m807 \"11,12-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14771 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m808 \"14,15-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14771 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m809 \"14,15-EET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14771 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m810 \"14,15-EET\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:34162 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m811 \"16(R)-HETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:34495 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m812 \"9(S)-HETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15647 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m813 \"Leukotriene B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15647 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m814 \"Leukotriene B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15647 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m815 \"Leukotriene B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15647 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m816 \"Leukotriene B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15545 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m817 \"(13E)-(15S)-15-Hydroxy-9-oxoprosta-10,13-dienoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15545 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m818 \"(13E)-(15S)-15-Hydroxy-9-oxoprosta-10,13-dienoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15545 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m819 \"(13E)-(15S)-15-Hydroxy-9-oxoprosta-10,13-dienoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15631 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m820 \"(5Z,9E,14Z)-(8xi,11R,12S)-11,12-Epoxy-8-hydroxyicosa-5,9,14-trienoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15631 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m821 \"(5Z,9E,14Z)-(8xi,11R,12S)-11,12-Epoxy-8-hydroxyicosa-5,9,14-trienoic acid\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15632 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m822 \"5(S)-HPETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15632 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m823 \"5(S)-HPETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15632 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m824 \"5(S)-HPETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15626 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m825 \"12(S)-HPETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15626 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m826 \"12(S)-HPETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15628 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m827 \"15(S)-HPETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15628 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m828 \"15(S)-HPETE\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15628 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m829 \"15(S)-HPETE\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14781 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m830 \"15H-11,12-EETA\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14813 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m831 \"11H-14,15-EETA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15554 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m832 \"Prostaglandin H2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15554 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m833 \"Prostaglandin H2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15554 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m834 \"Prostaglandin H2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15554 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m835 \"Prostaglandin H2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15551 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m836 \"Prostaglandin E2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15551 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m837 \"Prostaglandin E2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15551 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m838 \"Prostaglandin E2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15551 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m839 \"Prostaglandin E2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15555 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m840 \"Prostaglandin D2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15555 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m841 \"Prostaglandin D2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15555 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m842 \"Prostaglandin D2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15552 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m843 \"Prostaglandin I2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15552 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m844 \"Prostaglandin I2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15627 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m845 \"Thromboxane A2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15627 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m846 \"Thromboxane A2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15548 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m847 \"(13E)-11alpha-Hydroxy-9,15-dioxoprost-13-enoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15550 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m848 \"(5Z)-(15S)-11alpha-Hydroxy-9,15-dioxoprostanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15646 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m849 \"20-OH-Leukotriene B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15646 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m850 \"20-OH-Leukotriene B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15646 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m851 \"20-OH-Leukotriene B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:6498 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m852 \"Lipoxin A4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:6498 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m853 \"Lipoxin A4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:6498 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m854 \"Lipoxin A4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:6499 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m855 \"Lipoxin B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:6499 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m856 \"Lipoxin B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:6499 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m857 \"Lipoxin B4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27647 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m858 \"Prostaglandin G2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m859 \"ganglioside GM4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m860 \"12-HETrE\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m861 \"10,11-dihydro-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m862 \"10,11-dihydro-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m863 \"10,11-dihydro-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m864 \"10,11-dihydro-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m865 \"10,11-dihydro-12-epi-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m866 \"10,11-dihydro-12-epi-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m867 \"10,11-dihydro-12-epi-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m868 \"6,7-dihydro-12-epi-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m869 \"6,7-dihydro-12-epi-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m870 \"6,7-dihydro-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m871 \"6,7-dihydro-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m872 \"6,7-dihydro-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m873 \"6,7-dihydro-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m874 \"20-OH-10,11-dihydro-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m875 \"20-OH-10,11-dihydro-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m876 \"13,14-dihydro- lipoxin A4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m877 \"prostaglandin H1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m878 \"prostaglandin H1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m879 \"prostaglandin H1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m880 \"13,14-dihydro-15-oxo-PGE1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m881 \"10,11-dihydro-20-dihydroxy-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m882 \"10,11-dihydro-20-dihydroxy-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m883 \"prostaglandin G1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m884 \"10,11-dihydro-20-trihydroxy-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m885 \"10,11-dihydro-20-trihydroxy-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m886 \"trans,trans,cis-geranyl-geranyl-PP\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14772 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m887 \"5,6-DHET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14772 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m888 \"5,6-DHET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14772 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m889 \"5,6-DHET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14773 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m890 \"8,9-DHET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14773 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m891 \"8,9-DHET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14773 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m892 \"8,9-DHET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14774 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m893 \"11,12-DHET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14774 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m894 \"11,12-DHET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14774 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m895 \"11,12-DHET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14775 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m896 \"14,15-DHET\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14775 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m897 \"14,15-DHET\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:kegg.compound:C14775 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m898 \"14,15-DHET\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15553 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m899 \"Prostaglandin F2alpha\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15553 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m900 \"Prostaglandin F2alpha\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15544 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m901 \"Alprostadil\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15544 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m902 \"Alprostadil\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15630 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m903 \"(5Z,9E,14Z)-(8xi,11xi,12S)-8,11,12-Trihydroxyicosa-5,9,14-trienoate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14782 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m904 \"11,12,15-THETA\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14814 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m905 \"11,14,15-THETA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28728 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m906 \"Thromboxane B2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m907 \"13,14-dihydro-PGE1\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16091 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m908 \"Dolichol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:48861 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m909 \"Geranylgeranyl diphosphate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05859 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m910 \"Dehydrodolichol diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16214 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m911 \"Dolichyl phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16214 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m912 \"Dolichyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m913 \"arachidate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m914 \"arachidate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1628537257 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m915 \"Phytanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1628537257 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m916 \"Phytanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1628537257 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m917 \"Phytanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28822 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m918 \"Icosanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m919 \"3,5,3',5'-tetraiodo-L-thyronine-beta-D-glucuronoside\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m920 \"3,3',5'-triiodo-L-thyronine-beta-D-glucuronoside\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m921 \"3,5,3'-triiodo-L-thyronine-beta-D-glucuronoside\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m922 \"3',5'-diiodo-L-thyronine-beta-D-glucuronoside\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m923 \"3,3'-diiodo-L-thyronine-beta-D-glucuronoside\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18304 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m924 \"Deamino-NAD+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18304 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m925 \"Deamino-NAD+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15846 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m926 \"NAD+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15846 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m927 \"NAD+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15846 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m928 \"NAD+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15846 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m929 \"NAD+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15846 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m930 \"NAD+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15846 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m931 \"NAD+\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05490 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m932 \"11-Dehydrocorticosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16962 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m933 \"Cortisone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16908 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m934 \"NADH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16908 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m935 \"NADH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16908 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m936 \"NADH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16908 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m937 \"NADH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16908 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m938 \"NADH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16908 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m939 \"NADH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18009 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m940 \"NADP+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18009 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m941 \"NADP+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18009 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m942 \"NADP+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18009 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m943 \"NADP+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18009 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m944 \"NADP+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18009 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m945 \"NADP+\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m946 \"18-carboxy-dinor-LTE4\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C02075 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m947 \"Retinyl ester\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m948 \"NADPH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m949 \"NADPH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m950 \"NADPH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m951 \"NADPH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m952 \"NADPH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16474 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m953 \"NADPH\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17026 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m954 \"Progesterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17252 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m955 \"17alpha-Hydroxyprogesterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17252 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m956 \"17alpha-Hydroxyprogesterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16827 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m957 \"Corticosterone\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05497 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m958 \"21-Deoxycortisol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05497 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m959 \"21-Deoxycortisol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17650 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m960 \"Cortisol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m961 \"7'-carboxy-alpha-chromanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m962 \"7'-carboxy-alpha-chromanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m963 \"pregnenolone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m964 \"pregnenolone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m965 \"pregnenolone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m966 \"pregnenolone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m967 \"17alpha-hydroxypregnenolone sulfate\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m968 \"17alpha-hydroxypregnenolone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m969 \"17alpha-hydroxypregnenolone sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m970 \"17alpha-hydroxypregnenolone sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16581 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m971 \"Pregnenolone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16581 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m972 \"Pregnenolone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16581 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m973 \"Pregnenolone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16581 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m974 \"Pregnenolone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16581 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m975 \"Pregnenolone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m976 \"17alpha-Hydroxypregnenolone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m977 \"17alpha-Hydroxypregnenolone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m978 \"17alpha-Hydroxypregnenolone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m979 \"17alpha-Hydroxypregnenolone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m980 \"17alpha-Hydroxypregnenolone\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05478 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m981 \"3alpha,21-Dihydroxy-5beta-pregnane-11,20-dione\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04825 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m982 \"3-beta-D-Galactosyl-4-beta-D-galactosyl-O-beta-D-xylosylprotein\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05476 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m983 \"Tetrahydrocorticosterone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15468 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m984 \"Dephospho-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15827 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m985 \"N(omega)-(ADP-D-ribosyl)-L-arginine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15346 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m986 \"CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15346 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m987 \"CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15346 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m988 \"CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15346 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m989 \"CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15346 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m990 \"CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15346 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m991 \"CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15346 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m992 \"CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15346 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m993 \"CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m994 \"IsoGlobotriasyl ceramide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14863 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m995 \"2-(S-Glutathionyl)acetyl glutathione\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14863 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m996 \"2-(S-Glutathionyl)acetyl glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m997 \"N-Acetyl-alpha-D-glucosaminyl-(1-_LT_4)-beta-D-glucuronosyl-(1-_LT_4)-N- acetyl-alpha-D-glucosaminylproteoglycan\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17984 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m998 \"Acyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17984 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m999 \"Acyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17984 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1000 \"Acyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1001 \"15-oxo-Prostaglandin E2 ethanolamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15522 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1002 \"Formyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15876 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1003 \"beta-D-Galactosyl-1,3-(N-acetyl-beta-D-glucosaminyl-1,6)-N-acetyl-D- galactosaminyl-R\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:2700 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1004 \"Anandamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:2700 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1005 \"Anandamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1006 \"12(S)-hydroxyarachidonylethanolamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1007 \"15(S)-hydroxyarachidonylethanolamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1008 \"Prostaglandin E2 ethanolamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1009 \"Prostaglandin E2 ethanolamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1010 \"20-COOH-leukotriene E4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1011 \"20-oxo-leukotriene E4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1012 \"leukotriene-E4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15650 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1013 \"Leukotriene E4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15650 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1014 \"Leukotriene E4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28700 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1015 \"20-Hydroxyleukotriene E4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1016 \"1-lyso-2-arachidonoyl-phosphatidate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1017 \"1-lyso-2-arachidonoyl-phosphatidate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1018 \"1-lyso-2-arachidonoyl-phosphatidate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15351 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1019 \"Acetyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15351 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1020 \"Acetyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15351 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1021 \"Acetyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15351 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1022 \"Acetyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:52392 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1023 \"2-Arachidonoylglycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:52392 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1024 \"2-Arachidonoylglycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:52392 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1025 \"2-Arachidonoylglycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1026 \"12-hydroperoxyeicosatetraenoate glyceryl ester\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17490 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1027 \"L-Palmitoylcarnitine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28919 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1028 \"Estrone glucuronide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:36489 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1029 \"Estradiol-17beta 3-glucuronide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05504 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1030 \"16-Glucuronide-estriol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1031 \"nisinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1032 \"nisinate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15469 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1033 \"2,3-Dehydroacyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:50998 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1034 \"trans-2,3-Dehydroacyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15489 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1035 \"3-Oxoacyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1036 \"6,9,12,15,18-all-cis-tetracosapentaenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1037 \"9'-carboxy-alpha-chromanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1038 \"9'-carboxy-alpha-chromanol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15513 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1039 \"Propenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15531 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1040 \"Malonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15531 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1041 \"Malonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15531 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1042 \"Malonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15531 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1043 \"Malonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15531 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1044 \"Malonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15531 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1045 \"Malonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15451 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1046 \"(2S)-2-Methylacyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15450 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1047 \"(2R)-2-Methylacyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15455 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1048 \"(3S)-3-Hydroxyacyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15539 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1049 \"Propanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15539 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1050 \"Propanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15539 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1051 \"Propanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27762 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1052 \"3-Hydroxypropionyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16325 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1053 \"3alpha-Hydroxy-5beta-cholanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16755 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1054 \"Chenodeoxycholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16755 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1055 \"Chenodeoxycholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16755 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1056 \"Chenodeoxycholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16755 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1057 \"Chenodeoxycholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28834 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1058 \"3alpha,12alpha-Dihydroxy-5beta-cholanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28834 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1059 \"3alpha,12alpha-Dihydroxy-5beta-cholanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16359 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1060 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16359 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1061 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16359 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1062 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16359 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1063 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17164 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1064 \"Stachyose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17507 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1065 \"Psychosine sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17507 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1066 \"Psychosine sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16874 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1067 \"Psychosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16874 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1068 \"Psychosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16874 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1069 \"Psychosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1070 \"lignocerate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1071 \"lignocerate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17636 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1072 \"Sphingomyelin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17636 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1073 \"Sphingomyelin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28835 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1074 \"Testosterone glucuronide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1075 \"leukotriene-D4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1076 \"(_ST_i_LT_S_ST_/i_LT_)-3-hydroxy-isobutyryl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1077 \"simvastatin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28832 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1078 \"Androsterone glucuronide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:37451 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1079 \"Etiocholan-3alpha-ol-17-one 3-glucuronide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05951 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1080 \"Leukotriene D4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1547336926 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1081 \"Crotonoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27754 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1082 \"2-Methylprop-2-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1083 \"Acetoacetyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1084 \"Acetoacetyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15380 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1085 \"Succinyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15380 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1086 \"Succinyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15466 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1087 \"(S)-Methylmalonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15466 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1088 \"(S)-Methylmalonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15465 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1089 \"(R)-Methylmalonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15465 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1090 \"(R)-Methylmalonyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1091 \"simvastatin acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16565 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1092 \"alpha-N-Acetylneuraminyl-2,3-beta-D-galactosyl-1,3-N-acetyl-alpha-D- galactosaminyl-R\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16565 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1093 \"alpha-N-Acetylneuraminyl-2,3-beta-D-galactosyl-1,3-N-acetyl-alpha-D- galactosaminyl-R\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16565 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1094 \"alpha-N-Acetylneuraminyl-2,3-beta-D-galactosyl-1,3-N-acetyl-alpha-D- galactosaminyl-R\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17725 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1095 \"alpha-N-Acetylneuraminyl-2,6-beta-D-galactosyl-1,4-N-acetyl-beta-D- glucosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15517 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1096 \"Butanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15479 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1097 \"2-Methylpropanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15479 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1098 \"2-Methylpropanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15453 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1099 \"(S)-3-Hydroxybutanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15452 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1100 \"(R)-3-Hydroxybutanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:37050 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1101 \"3-Hydroxybutanoyl-CoA\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28259 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1102 \"(S)-3-Hydroxyisobutyryl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16818 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1103 \"all-trans-Pentaprenyl diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18318 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1104 \"Sulfatide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18318 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1105 \"Sulfatide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18318 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1106 \"Sulfatide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18368 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1107 \"Glucosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18368 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1108 \"Glucosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18368 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1109 \"Glucosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18368 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1110 \"Glucosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18390 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1111 \"Galactosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18390 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1112 \"Galactosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18390 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1113 \"Galactosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18390 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1114 \"Galactosylceramide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1115 \"11'-carboxy-alpha-chromanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1116 \"11'-carboxy-alpha-chromanol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15486 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1117 \"3-Methylcrotonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15478 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1118 \"2-Methylbut-2-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15476 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1119 \"2-Methylacetoacetyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15476 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1120 \"2-Methylacetoacetyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15476 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1121 \"2-Methylacetoacetyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15524 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1122 \"Glutaryl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15524 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1123 \"Glutaryl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:37998 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1124 \"Glycolithocholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:36274 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1125 \"Glycochenodeoxycholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17687 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1126 \"Glycocholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17687 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1127 \"Glycocholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15477 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1128 \"2-Methylbutanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15477 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1129 \"2-Methylbutanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15487 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1130 \"3-Methylbutanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15449 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1131 \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15449 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1132 \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28291 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1133 \"3-Hydroxyisovaleryl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:36259 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1134 \"Taurolithocholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:9410 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1135 \"Taurodeoxycholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:165259407 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1136 \"Taurochenodeoxycholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:165259407 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1137 \"Taurochenodeoxycholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28865 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1138 \"Taurocholate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C15516 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1139 \"Taurohyocholate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15812 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1140 \"Dolichyl beta-D-glucosyl phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15809 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1141 \"Dolichyl phosphate D-mannose\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15809 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1142 \"Dolichyl phosphate D-mannose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1143 \"Glucosyl-Ceramides\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16238 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1144 \"FAD\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16238 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1145 \"FAD\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16238 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1146 \"FAD\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16238 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1147 \"FAD\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17877 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1148 \"FADH2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17877 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1149 \"FADH2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17877 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1150 \"FADH2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1151 \"2-methylglutaconyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1152 \"4-methyl-trans-2-pentenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1153 \"3-oxo-4-methyl-pentanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1154 \"25-hydroxyvitamin D3-26,23-lactone\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04903 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1155 \"3-beta-D-Glucuronosyl-3-beta-D-galactosyl-4-beta-D-galactosyl-O- beta-D-xylosylprotein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1156 \"4-methyl-pentanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1157 \"3(S)-hydroxy-4-methyl-pentanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15488 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1158 \"3-Methylglutaconyl-CoA\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14144 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1159 \"5-Carboxy-2-pentenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27910 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1160 \"7-Dehydrodesmosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1161 \"cholesta-5,7,24-trien-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1162 \"cholesta-5,7,24-trien-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1163 \"5_AND_alpha;-cholesta-8,24-dien-3-one\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1164 \"25-hydroxyvitamin D3-26,23-lactol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1165 \"24-oxo-1alpha,25-dihydroxyvitamin D3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1166 \"24-oxo-1alpha,23,25-trihydroxyvitamin D3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28706 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1167 \"trans-Hex-2-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27648 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1168 \"3-Oxohexanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15467 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1169 \"(S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15467 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1170 \"(S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14145 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1171 \"(3S)-3-Hydroxyadipyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17759 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1172 \"Cholesta-5,7-dien-3beta-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17759 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1173 \"Cholesta-5,7-dien-3beta-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17759 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1174 \"Cholesta-5,7-dien-3beta-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17737 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1175 \"Desmosterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17737 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1176 \"Desmosterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17737 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1177 \"Desmosterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18252 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1178 \"Zymosterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18252 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1179 \"Zymosterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16290 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1180 \"5alpha-Cholesta-7,24-dien-3beta-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16290 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1181 \"5alpha-Cholesta-7,24-dien-3beta-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28940 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1182 \"Vitamin D3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28940 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1183 \"Vitamin D3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28940 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1184 \"Vitamin D3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1185 \"5_AND_alpha;-cholesta-8-en-3-one\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17933 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1186 \"Calcidiol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17899 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1187 \"7alpha-Hydroxycholest-4-en-3-one\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17899 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1188 \"7alpha-Hydroxycholest-4-en-3-one\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17823 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1189 \"Calcitriol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1190 \"24,25-Dihydroxyvitamin D3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1191 \"23S,25-dihydroxyvitamin D3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1192 \"7_AND_alpha;,12_AND_alpha;-dihydroxycholest-4-en-3-one\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1193 \"23S,25,26-trihydroxyvitamin D3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1194 \"1alpha,24R,25-trihydroxyvitamin D3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1195 \"calcitetrol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27540 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1196 \"Hexanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28276 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1197 \"(S)-Hydroxyhexanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16113 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1198 \"Cholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16113 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1199 \"Cholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16113 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1200 \"Cholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16113 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1201 \"Cholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16113 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1202 \"Cholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16113 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1203 \"Cholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17168 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1204 \"5alpha-Cholest-7-en-3beta-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17168 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1205 \"5alpha-Cholest-7-en-3beta-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16608 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1206 \"5alpha-Cholest-8-en-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1207 \"cholesta-8(9)-en-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1208 \"cholesta-8(9)-en-3beta-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17500 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1209 \"7alpha-Hydroxycholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17500 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1210 \"7alpha-Hydroxycholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1296 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1211 \"20alpha-Hydroxycholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1296 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1212 \"20alpha-Hydroxycholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1301 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1213 \"22beta-Hydroxycholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1301 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1214 \"22beta-Hydroxycholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:34310 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1215 \"Cerebrosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1216 \"27-Hydroxycholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27428 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1217 \"3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27428 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1218 \"3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27428 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1219 \"3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:2288 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1220 \"7alpha,12alpha-Dihydroxy-5beta-cholestan-3-one\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:783 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1221 \"17alpha,20alpha-Dihydroxycholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:783 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1222 \"17alpha,20alpha-Dihydroxycholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1294 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1223 \"20alpha,22beta-Dihydroxycholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18431 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1224 \"7alpha,27-Dihydroxycholesterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1225 \"20_AND_alpha;-22_AND_beta;-dihydroxy-cholesterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16577 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1226 \"3alpha,7alpha-Dihydroxy-5beta-cholestanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16577 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1227 \"3alpha,7alpha-Dihydroxy-5beta-cholestanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16577 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1228 \"3alpha,7alpha-Dihydroxy-5beta-cholestanate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1229 \"3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-27-al\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1230 \"3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-27-al\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1231 \"3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-27-al\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C18043 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1232 \"Cholesterol sulfate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C18043 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1233 \"Cholesterol sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1234 \"cholesterol sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1235 \"cholesterol sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18402 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1236 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoate\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18402 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1237 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18402 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1238 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1239 \"3alpha,7alpha,12alpha,25-tetrahydroxy-5beta-cholestane-24-one\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28540 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1240 \"3alpha,7alpha,26-Trihydroxy-5beta-cholestane\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16496 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1241 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestane\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16496 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1242 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestane\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1243 \"5beta-cholestane-3alpha,7alpha,12alpha,27-tetraol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1244 \"5beta-cholestane-3alpha,7alpha,12alpha,25-tetrol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1245 \"5beta-cholestane-3alpha,7alpha,12alpha,24S,25-pentol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1246 \"5beta-cholestane-3alpha,7alpha,12alpha,23,25-pentol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1247 \"5beta-cholestane-3alpha,7alpha,12alpha,25,27-pentol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1248 \"5beta-cholestane-3alpha,7alpha,12alpha,24R,25-pentol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1249 \"4-hydroxyestrone-2-S-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1250 \"4-hydroxyestrone-2-S-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1251 \"4-hydroxyestrone-2-S-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1252 \"4-hydroxy-17beta-estradiol-2-S-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1253 \"4-hydroxy-17beta-estradiol-2-S-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1254 \"4-hydroxy-17beta-estradiol-2-S-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1255 \"4_AND_alpha;-carboxy-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:50593 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1256 \"3-Keto-4-methylzymosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1257 \"4_AND_alpha;-formyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17002 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1258 \"Cholesterol ester\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17002 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1259 \"Cholesterol ester\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1260 \"4_AND_alpha;-carboxy-5_AND_alpha;-cholesta-8-en-3_AND_beta;-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16495 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1261 \"4alpha-Methyl-5alpha-cholest-7-en-3-one\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1949 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1262 \"4alpha-Methylzymosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1263 \"4_AND_alpha;-hydroxymethyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1264 \"4_AND_alpha;-formyl-5_AND_alpha;-cholesta-8-en-3_AND_beta;-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28432 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1265 \"4alpha-Methylcholesta-8-en-3beta-ol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05111 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1266 \"Methostenol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1267 \"4_AND_alpha;-hydroxymethyl-5_AND_alpha;-cholesta-8-en-3_AND_beta;-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18278 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1268 \"N-Acetyl-D-glucosaminyldiphosphodolichol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18278 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1269 \"N-Acetyl-D-glucosaminyldiphosphodolichol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1270 \"13'-carboxy-alpha-tocotrienol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14153 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1271 \"alpha-Tocotrienol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14153 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1272 \"alpha-Tocotrienol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14153 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1273 \"alpha-Tocotrienol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1274 \"13'-hydroxy-alpha-tocotrienol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17813 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1275 \"4,4-Dimethyl-5alpha-cholesta-8,14,24-trien-3beta-ol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1276 \"4_AND_alpha;-formyl-4_AND_beta;-methyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C15808 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1277 \"4alpha-Methylzymosterol-4-carboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1278 \"13'-carboxy-alpha-tocopherol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1279 \"13'-carboxy-alpha-tocopherol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27537 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1280 \"trans-Oct-2-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28264 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1281 \"3-Oxooctanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18364 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1282 \"14-Demethyllanosterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18364 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1283 \"14-Demethyllanosterol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1284 \"4_AND_alpha;-hydroxymethyl-4_AND_beta;-methyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18134 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1285 \"3beta-Hydroxy-4beta-methyl-5alpha-cholest-7-ene-4alpha-carboxylate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15533 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1286 \"Octanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15533 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1287 \"Octanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28632 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1288 \"(S)-Hydroxyoctanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18145 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1289 \"alpha-Tocopherol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18145 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1290 \"alpha-Tocopherol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18145 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1291 \"alpha-Tocopherol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1292 \"13'-hydroxy-alpha-tocopherol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1293 \"13'-hydroxy-_AND_alpha;-tocopherol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1294 \"mannosealpha1-6(ethanolaminephosphate-2)mannosealpha1-4(acyl)glucosaminylphosphatidylinositol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14857 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1295 \"1,1-Dichloroethylene epoxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14857 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1296 \"1,1-Dichloroethylene epoxide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16891 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1297 \"Glyoxylate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16995 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1298 \"Oxalate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14870 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1299 \"2-Bromoacetaldehyde\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14870 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1300 \"2-Bromoacetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28534 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1301 \"1,2-Dibromoethane\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28534 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1302 \"1,2-Dibromoethane\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15705 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1303 \"L-Amino acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15343 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1304 \"Acetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15343 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1305 \"Acetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1306 \"Acetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1307 \"Acetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1308 \"Acetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1309 \"Acetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1310 \"Acetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1311 \"Acetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17071 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1312 \"Glycolaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15428 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1313 \"Glycine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15428 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1314 \"Glycine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15428 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1315 \"Glycine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15428 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1316 \"Glycine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16236 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1317 \"Ethanol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16236 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1318 \"Ethanol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16000 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1319 \"Ethanolamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16000 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1320 \"Ethanolamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16668 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1321 \"Hypotaurine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16668 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1322 \"Hypotaurine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15891 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1323 \"Taurine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17553 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1324 \"Ethanolamine phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17553 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1325 \"Ethanolamine phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1326 \"12-oxo-c-LTB3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1327 \"12-oxo-c-LTB3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1328 \"12-oxo-c-LTB3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1329 \"12-oxo-c-LTB3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1330 \"S-(PGA2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1331 \"S-(PGA2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1332 \"S-(PGA2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1333 \"S-(PGJ2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1334 \"S-(PGJ2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1335 \"S-(9-deoxy-delta9,12-PGD2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1336 \"S-(9-deoxy-delta9,12-PGD2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1337 \"S-(9-deoxy-delta9,12-PGD2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1338 \"2,6-dimethyl-trans-2-heptenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1339 \"3-oxo-(2S)-methylisocapryloyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1340 \"10,11-dihydro-12R-hydroxy-leukotriene C4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1341 \"10,11-dihydro-12R-hydroxy-leukotriene C4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1342 \"10,11-dihydro-12R-hydroxy-leukotriene C4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1343 \"S-(9-hydroxy-PGA2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1344 \"S-(9-deoxy-delta12-PGD2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1345 \"S-(9-deoxy-delta12-PGD2)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1346 \"S-(9-deoxy-delta12-PGD2)-glutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16978 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1347 \"Leukotriene C4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16978 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1348 \"Leukotriene C4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1349 \"2(R),6-dimethyl-heptanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1350 \"2(S),6-dimethyl-heptanoyl-CoA\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1351 \"3(S)-hydroxy-2(S),6-dimethyl-heptanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C11304 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1352 \"S-(PGA1)-glutathione\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C11304 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1353 \"S-(PGA1)-glutathione\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C11304 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1354 \"S-(PGA1)-glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1355 \"S-(9-hydroxy-PGA1)-glutathione\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15440 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1356 \"Squalene\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15440 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1357 \"Squalene\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15441 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1358 \"(S)-2,3-Epoxysqualene\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15441 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1359 \"(S)-2,3-Epoxysqualene\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16521 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1360 \"Lanosterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16521 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1361 \"Lanosterol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17528 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1362 \"all-trans-Hexaprenyl diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15442 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1363 \"Presqualene diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15442 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1364 \"Presqualene diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1365 \"trans,cis-deca-2,4-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1366 \"trans,cis-deca-2,4-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1367 \"trans,cis-deca-2,4-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1368 \"trans-_AND_Delta;_ST_SUP_LT_2_ST_/SUP_LT_, cis-_AND_Delta;_ST_SUP_LT_4_ST_/SUP_LT_-decadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1369 \"cis-deca-4-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1370 \"cis-deca-4-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1371 \"trans-3-decenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1372 \"trans-3-decenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1373 \"trans-3-decenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:10723 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1374 \"trans-Dec-2-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:10723 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1375 \"trans-Dec-2-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:10723 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1376 \"trans-Dec-2-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28528 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1377 \"3-Oxodecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28493 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1378 \"Decanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28325 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1379 \"(S)-Hydroxydecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17950 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1380 \"beta-D-Galactosyl-1,4-beta-D-glucosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17950 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1381 \"beta-D-Galactosyl-1,4-beta-D-glucosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17950 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1382 \"beta-D-Galactosyl-1,4-beta-D-glucosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17950 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1383 \"beta-D-Galactosyl-1,4-beta-D-glucosylceramide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C06126 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1384 \"Digalactosylceramide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C06126 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1385 \"Digalactosylceramide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C06126 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1386 \"Digalactosylceramide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C06126 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1387 \"Digalactosylceramide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C03405 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1388 \"Lactosylceramide sulfate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C03405 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1389 \"Lactosylceramide sulfate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C06127 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1390 \"Digalactosylceramidesulfate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C06127 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1391 \"Digalactosylceramidesulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1392 \"4(R),8-dimethyl-trans-2-nonenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1393 \"3-oxo-4(R),8-dimethyl-nonanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1394 \"4(R),8-dimethyl-nonanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1395 \"4(R),8-dimethyl-nonanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1396 \"3(S)-hydroxy-4(R),8-dimethyl-nonanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17033 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1397 \"Biliverdin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16990 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1398 \"Bilirubin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16990 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1399 \"Bilirubin\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05791 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1400 \"D-Urobilinogen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1401 \"trans,cis-dodeca-2,5-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1402 \"trans,cis-dodeca-2,5-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1403 \"3-oxolaur-cis-5-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1404 \"3-oxolaur-cis-5-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1405 \"3-oxolaur-cis-5-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1406 \"3-oxolaur-6-cis-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1407 \"3-oxolaur-6-cis-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1408 \"cis-laur-5-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1409 \"cis-laur-5-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1410 \"cis-6-dodecenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1411 \"cis-6-dodecenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1412 \"cis-6-dodecenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1413 \"(3S)-3-hydroxydodec-cis-6-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1414 \"(3S)-3-hydroxydodec-cis-6-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1415 \"3(S)-3-hydroxydodecen-(5Z)-oyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1416 \"3(S)-3-hydroxydodecen-(5Z)-oyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1417 \"3(S)-3-hydroxydodecen-(5Z)-oyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28387 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1418 \"trans,cis-Lauro-2,6-dienoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28387 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1419 \"trans,cis-Lauro-2,6-dienoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28002 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1420 \"cis,cis-3,6-Dodecadienoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28002 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1421 \"cis,cis-3,6-Dodecadienoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27989 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1422 \"3-cis-Dodecenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27989 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1423 \"3-cis-Dodecenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27989 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1424 \"3-cis-Dodecenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15471 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1425 \"2-trans-Dodecenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15471 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1426 \"2-trans-Dodecenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27868 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1427 \"3-Oxododecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15521 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1428 \"Lauroyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27668 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1429 \"(S)-3-Hydroxydodecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17627 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1430 \"Heme\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17627 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1431 \"Heme\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17627 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1432 \"Heme\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15430 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1433 \"Protoporphyrin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15430 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1434 \"Protoporphyrin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15435 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1435 \"Protoporphyrinogen IX\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1436 \"trans-2-cis,cis-5,8-tetradecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1437 \"trans-2-cis,cis-5,8-tetradecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1438 \"(3E,5Z,8Z)-tetradecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1439 \"trans-2-cis,cis-4,8-tetradecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1440 \"3-oxo-cis,cis-5,8-tetradecadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1441 \"3-oxo-cis,cis-5,8-tetradecadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1442 \"cis,cis-myristo-5,8-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1443 \"cis,cis-myristo-5,8-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1444 \"trans,cis-myristo-2,7-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1445 \"trans,cis-myristo-2,7-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1446 \"trans-3-cis-8-tetradecadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1447 \"trans-2-cis-8-tetradecadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1448 \"3-oxomyrist-7-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1449 \"3-oxomyrist-7-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1450 \"3(S)-hydroxy-5Z,8Z-tetradecadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1451 \"3(S)-hydroxy-5Z,8Z-tetradecadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1452 \"3-oxo-cis-8-tetradecenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1453 \"alpha-N-Acetyl-D-glucosaminyl-(1-_LT_4)-beta-D-glucuronosyl-(1-_LT_3)- beta-D-galactosyl-(1-_LT_3)-beta-D-galactosyl-(1-_LT_4)-beta-D- xylosylproteoglycan\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1454 \"alpha-N-Acetyl-D-glucosaminyl-(1-_LT_4)-beta-D-glucuronosyl-(1-_LT_3)- beta-D-galactosyl-(1-_LT_3)-beta-D-galactosyl-(1-_LT_4)-beta-D- xylosylproteoglycan\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1455 \"cis-myrist-7-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1456 \"cis-myrist-7-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1457 \"(2S,6R,10R)-trimethyl-2E-hendecenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1458 \"cis-(3S)-hydroxytetradec-7-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1459 \"cis-(3S)-hydroxytetradec-7-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1460 \"(3S)-3-hydroxy-cis-8-tetradecenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1461 \"3-oxo-(2S,6R,10R)-trimethyl-hendecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1462 \"(2S,6R,10R)-trimethyl-hendecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1463 \"(2R,6R,10R)-trimethyl-hendecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1464 \"3(R)-hydroxy-(2S,6R,10)-trimethyl-hendecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27721 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1465 \"trans-Tetradec-2-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28726 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1466 \"3-Oxotetradecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17613 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1467 \"all-trans-Heptaprenyl diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15532 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1468 \"Tetradecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27466 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1469 \"(S)-3-Hydroxytetradecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1470 \"mannosealpha1-2mannosealpha1-6(ethanolaminephosphate-2)mannosealpha1-4glucosaminyl-acyl-phosphatidylinositol\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15439 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1471 \"Coproporphyrinogen III\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15439 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1472 \"Coproporphyrinogen III\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28607 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1473 \"Coproporphyrinogen I\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:12427 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1474 \"N,N'-Diacetylchitobiosyldiphosphodolichol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:12427 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1475 \"N,N'-Diacetylchitobiosyldiphosphodolichol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1476 \"2E,4Z,7Z,10Z-hexadecatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1477 \"trans,cis,cis-2,7,10-hexadecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1478 \"trans,cis,cis-2,7,10-hexadecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1479 \"trans,cis,cis-2,7,10-hexadecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1480 \"4Z,7Z,10Z-hexadecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1481 \"3Z,7Z,10Z-hexadecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1482 \"3Z,7Z,10Z-hexadecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1483 \"3-oxo-cis,cis-7,10-hexadecadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1484 \"3-oxo-cis,cis-7,10-hexadecadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1485 \"cis,cis-palmito-7,10-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1486 \"cis,cis-palmito-7,10-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1487 \"trans,cis-hexadeca-2,9-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1488 \"trans,cis-hexadeca-2,9-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1489 \"3-oxopalmitoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1490 \"3-oxopalmitoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1491 \"(3S)-3-hydroxy-cis,cis-palmito-7,10-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1492 \"(3S)-3-hydroxy-cis,cis-palmito-7,10-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1493 \"palmitoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1494 \"palmitoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1495 \"palmitoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1496 \"palmitoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1497 \"(4R,8R,12R)-trimethyl-2E-tridecenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1498 \"(S)-3-hydroxypalmitoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1499 \"(S)-3-hydroxypalmitoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1500 \"3-oxo-(4R,8R,12R)-trimethyl-tridecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1501 \"3(R)-hydroxy-(4R,8R,12R)-trimethyl-tridecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28935 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1502 \"trans-Hexadec-2-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15491 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1503 \"3-Oxopalmitoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15525 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1504 \"Palmitoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15525 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1505 \"Palmitoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15525 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1506 \"Palmitoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15525 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1507 \"Palmitoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15495 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1508 \"4,8,12-Trimethyltridecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27402 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1509 \"(S)-3-Hydroxyhexadecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:12310 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1510 \"Trihexosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:12310 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1511 \"Trihexosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:12310 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1512 \"Trihexosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:12310 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1513 \"Trihexosylceramide\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16427 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1514 \"Bilirubin-glucuronoside\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16427 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1515 \"Bilirubin-glucuronoside\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1516 \"18-CoA-18-oxo-dinorleukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1517 \"18-CoA-18-oxo-dinorleukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1518 \"18-CoA-18-oxo-dinorleukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1519 \"2E,6Z,9Z,12Z-octadecatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1520 \"stearidonoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1521 \"3-oxo-6Z,9Z,12Z-octadecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1522 \"3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1523 \"3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1524 \"3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1525 \"3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1526 \"3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1527 \"3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1528 \"trans,cis,cis-2,9,12-octadecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1529 \"trans,cis,cis-2,9,12-octadecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1530 \"alpha-linolenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1531 \"3-oxolinoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1532 \"3-oxolinoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1533 \"3(S)-hydroxy-6Z,9Z,12Z-octadecatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1534 \"3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1535 \"3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1536 \"3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1537 \"trans,cis-2,9-octadecadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1538 \"(3S)-3-hydroxylinoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1539 \"(3S)-3-hydroxylinoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1540 \"3-oxo-cis-9-octadecenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15508 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1541 \"gamma-Linolenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15508 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1542 \"gamma-Linolenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15508 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1543 \"gamma-Linolenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1544 \"3-oxooctadecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1545 \"3-oxooctadecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1546 \"3(S)-hydroxy-cis-9-octadecenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15530 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1547 \"Linoleoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15530 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1548 \"Linoleoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15530 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1549 \"Linoleoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15530 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1550 \"Linoleoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1551 \"3-hydroxyoctadecanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15534 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1552 \"Oleoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15534 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1553 \"Oleoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:50571 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1554 \"3-Oxostearoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17103 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1555 \"N-Acetyl-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-D- glucosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17103 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1556 \"N-Acetyl-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-D- glucosylceramide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15541 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1557 \"Stearoyl-CoA\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15541 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1558 \"Stearoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15541 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1559 \"Stearoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15541 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1560 \"Stearoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15725 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1561 \"1,3-Diaminopropane\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15725 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1562 \"1,3-Diaminopropane\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1830837080 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1563 \"Propenoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17158 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1564 \"Methylglyoxal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17158 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1565 \"Methylglyoxal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17158 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1566 \"Methylglyoxal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17158 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1567 \"Methylglyoxal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1568 \"malonic dialdehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32816 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1569 \"Pyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32816 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1570 \"Pyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32816 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1571 \"Pyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17960 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1572 \"3-Oxopropanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17960 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1573 \"3-Oxopropanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17960 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1574 \"3-Oxopropanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17960 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1575 \"3-Oxopropanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16208 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1576 \"Mercaptopyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16208 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1577 \"Mercaptopyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30841 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1578 \"Hydroxypyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30841 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1579 \"Hydroxypyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16992 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1580 \"2-Hydroxy-3-oxopropanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1665 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1581 \"3-Sulfinylpyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1665 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1582 \"3-Sulfinylpyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1802144897 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1583 \"Phosphoenolpyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1802144897 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1584 \"Phosphoenolpyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1802144897 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1585 \"Phosphoenolpyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1586 \"plasmenate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1587 \"plasmenate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30933 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1588 \"3-Phosphonooxypyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15412 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1589 \"(-)-Ureidoglycolate\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17153 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1590 \"Propanal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30768 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1591 \"Propanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30768 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1592 \"Propanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27957 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1593 \"Hydroxyacetone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18419 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1594 \"Lactaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:422 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1595 \"(S)-Lactate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:42111 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1596 \"(R)-Lactate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1597 \"D-Glyceraldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1598 \"D-Glyceraldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1599 \"D-Glyceraldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1600 \"D-Glyceraldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:33404 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1601 \"3-Hydroxypropanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32398 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1602 \"D-Glycerate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32398 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1603 \"D-Glycerate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32398 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1604 \"D-Glycerate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32398 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1605 \"D-Glycerate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17197 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1606 \"O-Alkylglycerone phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1607 \"alk-1-enyl-2-lyso-glycerophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1608 \"alk-1-enyl-2-lyso-glycerophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1609 \"alk-1-enyl-2-lyso-glycerophosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16344 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1610 \"Guanidinoacetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16344 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1611 \"Guanidinoacetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17906 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1612 \"Aminoacetone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17906 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1613 \"Aminoacetone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1614 \"3-Aminopropanal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1615 \"3-Aminopropanal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1616 \"3-Aminopropanal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16977 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1617 \"L-Alanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16977 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1618 \"L-Alanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16977 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1619 \"L-Alanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16977 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1620 \"L-Alanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16958 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1621 \"beta-Alanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16958 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1622 \"beta-Alanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16958 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1623 \"beta-Alanine\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15611 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1624 \"Sarcosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15611 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1625 \"Sarcosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15611 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1626 \"Sarcosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17561 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1627 \"L-Cysteine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17561 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1628 \"L-Cysteine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17561 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1629 \"L-Cysteine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17115 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1630 \"L-Serine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17115 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1631 \"L-Serine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17115 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1632 \"L-Serine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17115 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1633 \"L-Serine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17115 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1634 \"L-Serine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17115 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1635 \"L-Serine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16523 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1636 \"D-Serine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1637 \"3-Sulfino-L-alanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1638 \"3-Sulfino-L-alanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16345 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1639 \"3-Sulfino-L-alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1640 \"alkylglycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16108 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1641 \"Glycerone phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16108 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1642 \"Glycerone phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16108 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1643 \"Glycerone phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16108 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1644 \"Glycerone phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29052 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1645 \"D-Glyceraldehyde 3-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17138 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1646 \"DL-Glyceraldehyde 3-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17794 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1647 \"3-Phospho-D-glycerate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17835 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1648 \"2-Phospho-D-glycerate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17835 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1649 \"2-Phospho-D-glycerate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15811 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1650 \"O-Phospho-L-serine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16001 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1651 \"3-Phospho-D-glyceroyl phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17720 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1652 \"2,3-Bisphospho-D-glycerate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16997 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1653 \"Propane-1,2-diol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29002 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1654 \"(S)-Propane-1,2-diol\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17754 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1655 \"Glycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17754 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1656 \"Glycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17754 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1657 \"Glycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17754 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1658 \"Glycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17754 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1659 \"Glycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17106 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1660 \"1-Alkyl-sn-glycero-3-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15978 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1661 \"sn-Glycerol 3-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15978 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1662 \"sn-Glycerol 3-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15437 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1663 \"Uroporphyrinogen III\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28766 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1664 \"Uroporphyrinogen I\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28766 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1665 \"Uroporphyrinogen I\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16645 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1666 \"Hydroxymethylbilane\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1667 \"13E-tetranor-16-oxo-16-CoA-LTE4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1668 \"CoA-omega-COOH-tetranor-LTE3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1669 \"trans-2,3-dehydropristanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1670 \"(2S)-pristanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1671 \"(2R)-pristanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1672 \"3(R)-hydroxy-pristanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16275 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1673 \"all-trans-Octaprenyl diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15371 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1674 \"3-Oxopristanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1675 \"18E-20-oxo-20-CoA-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1676 \"18E-20-oxo-20-CoA-LTB4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1677 \"18,20-dioxo-20-CoA-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1678 \"18,20-dioxo-20-CoA-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1679 \"18,20-dioxo-20-CoA-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1680 \"20-CoA-20-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1681 \"20-CoA-20-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1682 \"20-CoA-20-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1683 \"20-CoA-20-oxo-leukotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1684 \"20-CoA-20-oxo-18R-hydroxyleucotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1685 \"20-CoA-20-oxo-18R-hydroxyleucotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1686 \"20-CoA-20-oxo-18R-hydroxyleucotriene B4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1687 \"2-trans-cis,cis,cis,cis-4,8,11,14-eicosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1688 \"2-trans-cis,cis,cis,cis-4,8,11,14-eicosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1689 \"eicosa-2E,8Z,11Z,14Z,17Z-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1690 \"eicosa-2E,8Z,11Z,14Z,17Z-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1691 \"eicosa-5,8,11,14,17-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1692 \"eicosa-5,8,11,14,17-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1693 \"trans-3-cis-5,8,11,14-eicosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1694 \"trans-3-cis-5,8,11,14-eicosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1695 \"3-keto-eicosa-8,11,14,17-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1696 \"5-oxo-12(S)-hydroxy-eicosa-2E,8E,10E,14Z-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1697 \"5-oxo-12(S)-hydroxy-eicosa-2E,8E,10E,14Z-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1698 \"5-oxo-12(R)-hydroxy-eicosa-2E,8E,10E,14Z-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1699 \"5-oxo-12(R)-hydroxy-eicosa-2E,8E,10E,14Z-tetraenoyl-CoA\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1700 \"3,5-dioxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1701 \"3,5-dioxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1702 \"3,5-dioxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1703 \"3,5-dioxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1704 \"3,5-dioxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1705 \"3,5-dioxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1706 \"(8Z,11Z,14Z,17Z)-eicosatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1707 \"(8Z,11Z,14Z,17Z)-eicosatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1708 \"trans-2-cis,cis,cis-8,11,14-eicosatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1709 \"trans-2-cis,cis,cis-8,11,14-eicosatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1710 \"trans-3-cis-8,11,14-eicosatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1711 \"3-oxo-dihomo-gama-linolenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1712 \"3-oxo-dihomo-gama-linolenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1713 \"3(S)-hydroxy-all-cis-8,11,14,17-eicosatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1714 \"3(S)-hydroxy-all-cis-8,11,14,17-eicosatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1715 \"5-oxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1716 \"5-oxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1717 \"5-oxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1718 \"5-oxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1719 \"5-oxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1720 \"5-oxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1721 \"5-oxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1722 \"5-oxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1723 \"5(S),12(R)-dihydroxy-eicosa-2,8-trans-6,14-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1724 \"5(S),12(R)-dihydroxy-eicosa-2,8-trans-6,14-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1725 \"3(S),12(S)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1726 \"3(S),12(S)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1727 \"3(S),12(S)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1728 \"3(S),12(R)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1729 \"3(S),12(R)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1730 \"3(S),12(R)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1731 \"3-oxo-5(S),12(R)-dihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1732 \"3-oxo-5(S),12(R)-dihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1733 \"3-oxo-5(S),12(R)-dihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1734 \"dihomo-gama-linolenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1735 \"trans,cis,cis-2,11,14-eicosatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1736 \"3(S)-hydroxy-dihomo-gama-linolenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1737 \"3-oxoeicosa-cis,cis-11,14-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1738 \"10,11-dihydro-LTB4-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1739 \"10,11-dihydro-LTB4-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1740 \"10,11-dihydro-LTB4-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1741 \"10,11-dihydro-LTB4-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1742 \"3(S),5(S),12(R)-trihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1743 \"3(S),5(S),12(R)-trihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1744 \"3(S),5(S),12(R)-trihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15514 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1745 \"Arachidonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15514 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1746 \"Arachidonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15514 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1747 \"Arachidonyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15514 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1748 \"Arachidonyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1749 \"cis,cis-11,14-eicosadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1750 \"trans,cis-2,11-eicosadienoyl-CoA\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1751 \"trans,cis-2,11-eicosadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1752 \"(3S)-hydroxy-eicosa-cis,cis-11,14-dienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1753 \"3-oxo-11-cis-eicosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1754 \"3-oxo-11-cis-eicosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1755 \"trans-eicos-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1756 \"11-cis-eicosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1757 \"11-cis-eicosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1758 \"3-oxoeicosanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1759 \"3(S)-hydroxy-11-cis-eicosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1760 \"3(S)-hydroxy-11-cis-eicosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1761 \"3-hydroxyeicosanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1762 \"3(S)-2-hydroxyphytanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15527 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1763 \"Eicosanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15527 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1764 \"Eicosanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15527 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1765 \"Eicosanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15538 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1766 \"Phytanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15538 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1767 \"Phytanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15475 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1768 \"2-Hydroxyphytanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1769 \"CoA-18-COOH-16E-dinor-LTE5\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1770 \"CoA-omega-COOH-dinor-LTE4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1771 \"16E-18-oxo-18-CoA-dinor-LTE4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1772 \"CoA-18-COOH-15E-dinor-LTE4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1773 \"16,18-oxo-18-CoA-dinor-LTE4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1774 \"16(S)-hydroxy-18-oxo-18-CoA-LTE4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18396 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1775 \"beta-D-Mannosyldiacetylchitobiosyldiphosphodolichol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1776 \"2-trans-7,10,13,16,19-all-cis-docosahexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1777 \"docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1778 \"docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1779 \"docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1780 \"3-oxo-docosa-7,10,13,16,19-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1781 \"docosa-7,10,13,16,19-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1782 \"adrenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1783 \"adrenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1784 \"adrenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1785 \"2E,7Z,10Z,13Z,16Z-docosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1786 \"trans,cis,cis,cis,cis-2,10,13,16,19-docosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1787 \"trans,cis,cis,cis,cis-2,10,13,16,19-docosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1788 \"3(S)-hydroxy-docosa-7,10,13,16,19-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1789 \"3-oxo-docosa-7,10,13,16-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1790 \"3-oxo-docosa-10,13,16,19-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1791 \"3-oxo-docosa-10,13,16,19-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1792 \"trans,cis,cis,cis-2,10,13,16-docosatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1793 \"all-cis-10,13,16,19-docosatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1794 \"all-cis-10,13,16,19-docosatetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1795 \"3(S)-hydroxy-docosa-7,10,13,16-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1796 \"3-oxo-docosa-cis,cis,cis-10,13,16-trienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1797 \"3(S)-hydroxy-docosa-10,13,16,19-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1798 \"3(S)-hydroxy-docosa-10,13,16,19-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1799 \"3(S)-hydroxy-10,13,16-all-cis-docosatrienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1800 \"trans,cis-2,13-docosadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1801 \"trans,cis-2,13-docosadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1802 \"3-oxo-13cis-docosenoyl-CoA\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1803 \"3-oxo-13cis-docosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1804 \"trans-docos-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1805 \"trans-docos-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1806 \"cis-13-docosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1807 \"cis-13-docosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1808 \"3-oxodocosanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1809 \"3-oxodocosanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1810 \"3(S)-hydroxy-13cis-docosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1811 \"3(S)-hydroxy-13cis-docosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1812 \"docosanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1813 \"3-hydroxydocosanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1814 \"18E-20-oxo-20-CoA-LTE4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1815 \"CoA-20-COOH-18-oxo-LTE4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1816 \"CoA-20-COOH-LTE4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1817 \"18(R)-hydroxy-20-oxo-20-CoA-LTE4\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18392 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1818 \"Bilirubin beta-diglucuronide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18392 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1819 \"Bilirubin beta-diglucuronide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1820 \"trans-2-all-cis-6,9,12,15,18,21-tetracosaheptaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1821 \"3-oxo-all-cis-6,9,12,15,18,21-tetracosahexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1822 \"2-trans-9,12,15,18,21-all-cis-tetracosahexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1823 \"2-trans-9,12,15,18,21-all-cis-tetracosahexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1824 \"tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1825 \"tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1826 \"tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1827 \"trans-2-all-cis-6,9,12,15,18-tetracosahexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1828 \"3-oxo-all-cis-6,9,12,15,18-tetracosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1829 \"3-oxo-all-cis-6,9,12,15,18-tetracosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1830 \"3-oxo-all-cis-6,9,12,15,18-tetracosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1831 \"(3S)-hydroxy-tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1832 \"tetracosa-9,12,15,18,21-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1833 \"tetracosa-9,12,15,18,21-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1834 \"2-trans-9,12,15,18-all-cis-tetracosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1835 \"2-trans-9,12,15,18-all-cis-tetracosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1836 \"tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1837 \"tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1838 \"trans,cis,cis,cis,cis-2,12,15,18,21-tetracosapentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1839 \"3(S)-hydroxy-tetracosa-9,12,15,18,21-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1840 \"3(S)-hydroxy-tetracosa-9,12,15,18,21-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1841 \"3(S)-hydroxy-tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1842 \"3-oxo-tetracosa-12,15,18,21-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1843 \"3-oxo-tetracosa-12,15,18,21-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1844 \"tetracosa-9,12,15,18-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1845 \"tetracosa-9,12,15,18-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1846 \"3(S)-hydroxy-tetracosa-12,15,18,21-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1847 \"3(S)-hydroxy-tetracosa-12,15,18,21-all-cis-tetraenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1848 \"deoxycholoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1849 \"deoxycholoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1850 \"trans,cis-2,15-tetracosadienoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1851 \"3-oxo-cis-15-tetracosaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1852 \"3-oxo-cis-15-tetracosaenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28701 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1853 \"Chenodeoxycholoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28701 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1854 \"Chenodeoxycholoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28701 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1855 \"Chenodeoxycholoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28701 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1856 \"Chenodeoxycholoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1857 \"Chenodeoxyglycocholoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15519 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1858 \"Choloyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15519 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1859 \"Choloyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15519 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1860 \"Choloyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1861 \"trans-tetracos-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1862 \"cis-15-tetracosenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1863 \"3-oxotetracosanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1864 \"3-oxotetracosanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1865 \"3(S)-hydroxy-cis-15-tetracosaenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1866 \"3(S)-hydroxy-cis-15-tetracosaenoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18144 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1867 \"all-trans-Nonaprenyl diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1868 \"tetracosanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1869 \"tetracosanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1870 \"3-hydroxytetracosanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18259 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1871 \"Globoside\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04910 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1872 \"1,3-beta-D-Galactosyl-N-acetyl-D-glucosaminyl-1,3-beta-D-galactosyl- 1,4-D-glucosylceramide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04910 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1873 \"1,3-beta-D-Galactosyl-N-acetyl-D-glucosaminyl-1,3-beta-D-galactosyl- 1,4-D-glucosylceramide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04922 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1874 \"beta-D-Galactosyl-1,4-N-acetyl-beta-D-glucosaminyl-1,3-beta-D- galactosyl-1,4-beta-D-glucosylceramide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04922 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1875 \"beta-D-Galactosyl-1,4-N-acetyl-beta-D-glucosaminyl-1,3-beta-D- galactosyl-1,4-beta-D-glucosylceramide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C06136 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1876 \"GA1\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C06136 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1877 \"GA1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1878 \"24-oxo-25(R)-trihydroxycoprostanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1879 \"24-oxo-25(R)-trihydroxycoprostanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1880 \"24(R),25(R)-varanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1881 \"24(R),25(R)-varanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1882 \"24(R),25(R)-varanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28533 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1883 \"3alpha,7alpha-Dihydroxy-5beta-24-oxocholestanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27505 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1884 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholest-24-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27505 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1885 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholest-24-enoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27379 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1886 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-24-oxocholestanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15494 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1887 \"3alpha,7alpha-Dihydroxy-5beta-cholestanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15494 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1888 \"3alpha,7alpha-Dihydroxy-5beta-cholestanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15493 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1889 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15493 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1890 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15493 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1891 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:37642 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m1892 \"(25R)-3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestan-26-oyl-CoA\">\n",
      "IOPub message rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_msg_rate_limit`.\n",
      "\n",
      "Current values:\n",
      "NotebookApp.iopub_msg_rate_limit=1000.0 (msgs/sec)\n",
      "NotebookApp.rate_limit_window=3.0 (secs)\n",
      "\n",
      "urn:miriam:urn:miriam:kegg.compound:C14453 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2269 \"4-Bromophenol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14839 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2270 \"Bromobenzene-3,4-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14839 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2271 \"Bromobenzene-3,4-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14839 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2272 \"Bromobenzene-3,4-oxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14843 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2273 \"4-Bromocatechol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14845 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2274 \"4-Bromophenol-2,3-epoxide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14845 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2275 \"4-Bromophenol-2,3-epoxide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15940 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2276 \"Nicotinate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16836 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2277 \"4-Nitrophenol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16836 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2278 \"4-Nitrophenol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C03254 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2279 \"Aryl dialkyl phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17855 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2280 \"Triacylglycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17855 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2281 \"Triacylglycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17855 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2282 \"Triacylglycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17855 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2283 \"Triacylglycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17154 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2284 \"Nicotinamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30817 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2285 \"Urocanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17440 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2286 \"4-Nitrophenyl phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15882 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2287 \"Phenol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17594 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2288 \"p-Benzenediol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C00850 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2289 \"Aryl sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32805 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2290 \"cis-Aconitate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32805 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2291 \"cis-Aconitate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32805 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2292 \"cis-Aconitate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32805 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2293 \"cis-Aconitate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2294 \"Dehydroascorbate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:7815 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2295 \"Oxalosuccinate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:7815 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2296 \"Oxalosuccinate\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:7815 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2297 \"Oxalosuccinate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14844 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2298 \"Bromobenzene-3,4-dihydrodiol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14844 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2299 \"Bromobenzene-3,4-dihydrodiol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C14844 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2300 \"Bromobenzene-3,4-dihydrodiol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17602 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2301 \"4-Hydroxyaniline\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15745 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2302 \"2-Aminomuconate semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2303 \"1D-_ST_i_LT_myo_ST_/i_LT_-inositol (1,2,3,4,6)-penta_ST_i_LT_kis_ST_/i_LT_phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2304 \"D-_ST_i_LT_myo_ST_/i_LT_-inositol (1,2,3,5,6) pentakisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2305 \"diphospho-1D-myo-inositol tetrakisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2306 \"3-diphospho-1D-myo-inositol (1,2,4,5,6)pentakisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2307 \"1,5-bisdiphosphoinositol-1D-_ST_i_LT_myo_ST_/i_LT_-inositol (2,3,4,6)tetrakisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2308 \"3,5-bisdiphosphoinositol-1D-_ST_i_LT_myo_ST_/i_LT_-inositol (2,3,4,6)tetrakisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15490 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2309 \"3-Oxoadipyl-CoA\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16504 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2310 \"Monodehydroascorbate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28104 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2311 \"Methylimidazole acetaldehyde\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C03633 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2312 \"Peptidylproline (omega=0)\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C03798 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2313 \"Peptidylproline (omega=180)\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:27384 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2314 \"4-Imidazolone-5-propanoate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C03636 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2315 \"Protein N5-alkylglutamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15753 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2316 \"2-Oxoadipate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15753 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2317 \"2-Oxoadipate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:2907338290 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2318 \"Ascorbate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:2907338290 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2319 \"Ascorbate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30769 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2320 \"Citrate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30769 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2321 \"Citrate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30769 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2322 \"Citrate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30769 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2323 \"Citrate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30769 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2324 \"Citrate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30887 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2325 \"Isocitrate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30887 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2326 \"Isocitrate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30887 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2327 \"Isocitrate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30887 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2328 \"Isocitrate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30887 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2329 \"Isocitrate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15622 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2330 \"(4R,5S)-4,5,6-Trihydroxy-2,3-dioxohexanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2331 \"N-acetyl-L-asparagine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15971 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2332 \"L-Histidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15971 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2333 \"L-Histidine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C02583 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2334 \"Protein glutamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:49015 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2335 \"(S)-2,3,4,5-Tetrahydropyridine-2-carboxylate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:49015 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2336 \"(S)-2,3,4,5-Tetrahydropyridine-2-carboxylate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28095 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2337 \"2-Amino-3-oxoadipate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2338 \"dehydrodolichyl-P\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2339 \"dehydrodolichol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C03740 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2340 \"(5-L-Glutamyl)-L-amino acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17999 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2341 \"Protein dithiol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2342 \"spermidic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2343 \"spermidic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2344 \"spermidic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16038 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2345 \"Phosphatidylethanolamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16038 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2346 \"Phosphatidylethanolamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16038 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2347 \"Phosphatidylethanolamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16038 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2348 \"Phosphatidylethanolamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16038 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2349 \"Phosphatidylethanolamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16173 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2350 \"N-Acyl-D-mannosamine 6-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17755 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2351 \"Cystathionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2352 \"spermidine dialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2353 \"spermidine dialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2354 \"spermidine dialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2355 \"spermidine dialdehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18349 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2356 \"5'-Phosphoribosylglycinamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16347 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2357 \"L-Carnitine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16347 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2358 \"L-Carnitine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2359 \"Sedoheptulose 7-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18020 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2360 \"4-Trimethylammoniobutanal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18020 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2361 \"4-Trimethylammoniobutanal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1941 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2362 \"4-Trimethylammoniobutanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1941 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2363 \"4-Trimethylammoniobutanoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15355 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2364 \"Acetylcholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15355 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2365 \"Acetylcholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15355 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2366 \"Acetylcholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17969 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2367 \"Sedoheptulose 1,7-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17969 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2368 \"Sedoheptulose 1,7-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2369 \"spermidine monoaldehyde 2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2370 \"spermidine monoaldehyde 1\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16610 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2371 \"Spermidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16610 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2372 \"Spermidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30746 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2373 \"Benzoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15793 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2374 \"3-Hydroxyanthranilate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:995 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2375 \"2-Amino-3-carboxymuconate semialdehyde\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05842 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2376 \"N1-Methyl-2-pyridone-5-carboxamide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05843 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2377 \"N1-Methyl-4-pyridone-5-carboxamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16797 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2378 \"1-Methylnicotinamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2379 \"_ST_i_LT_di-trans, poly-cis_ST_/i_LT_-polyprenyl diphosphate (C_ST_sub_LT_80_ST_/sub_LT_)\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2380 \"_ST_i_LT_N_ST_/i_LT_-acetylglucosaminyl-diphosphodolichol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2381 \"dopamine o-quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2382 \"dopamine o-quinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2383 \"dopamine o-quinone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18405 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2384 \"Pyridoxal phosphate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05576 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2385 \"3,4-Dihydroxyphenylethyleneglycol\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05576 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2386 \"3,4-Dihydroxyphenylethyleneglycol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18397 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2387 \"Phenethylamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18397 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2388 \"Phenethylamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18397 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2389 \"Phenethylamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15760 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2390 \"Tyramine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15760 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2391 \"Tyramine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15760 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2392 \"Tyramine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15760 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2393 \"Tyramine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18243 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2394 \"Dopamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18243 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2395 \"Dopamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18243 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2396 \"Dopamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18243 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2397 \"Dopamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16709 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2398 \"Pyridoxine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16709 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2399 \"Pyridoxine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18357 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2400 \"L-Noradrenaline\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18357 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2401 \"L-Noradrenaline\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:37946 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2402 \"Dopamine 3-O-sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16410 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2403 \"Pyridoxamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16410 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2404 \"Pyridoxamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:11750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2405 \"Phosphatidylserine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:11750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2406 \"Phosphatidylserine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:11750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2407 \"Phosphatidylserine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:11750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2408 \"Phosphatidylserine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:11750 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2409 \"Phosphatidylserine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28803 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2410 \"Pyridoxine phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18335 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2411 \"Pyridoxamine phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2412 \"[chondroitin]-N-acetyl-galactosamine \">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17515 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2413 \"gamma-L-Glutamyl-L-cysteine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28843 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2414 \"Aminoimidazole ribotide\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C02301 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2415 \"O-Acylcarnitine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15804 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2416 \"Lipoylprotein\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:37393 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2417 \"Phosphatidylglycerophosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30314 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2418 \"Lipoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30314 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2419 \"Lipoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18272 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2420 \"5'-Phosphoribosyl-N-formylglycinamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:506227 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2421 \"N-Acetyl-D-glucosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:506227 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2422 \"N-Acetyl-D-glucosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:506227 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2423 \"N-Acetyl-D-glucosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17122 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2424 \"N-Acetyl-D-mannosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28037 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2425 \"N-Acetyl-D-galactosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28037 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2426 \"N-Acetyl-D-galactosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28009 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2427 \"N-Acetyl-beta-D-glucosamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17460 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2428 \"Lipoamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17460 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2429 \"Lipoamide\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18413 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2430 \"2-(Formamido)-N1-(5'-phosphoribosyl)acetamidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15784 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2431 \"N-Acetyl-D-glucosamine 6-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15784 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2432 \"N-Acetyl-D-glucosamine 6-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:7125 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2433 \"N-Acetyl-D-glucosamine 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:7125 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2434 \"N-Acetyl-D-glucosamine 1-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28273 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2435 \"N-Acetyl-D-mannosamine 6-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16194 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2436 \"Dihydrolipoylprotein\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17694 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2437 \"Dihydrolipoamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17694 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2438 \"Dihydrolipoamide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2439 \"1-alkyl-2-acyl-sn-glycero-3-phosphocholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2440 \"1-alkyl-2-acyl-sn-glycero-3-phosphocholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2441 \"1-alkyl-2-acyl-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04317 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2442 \"1-Organyl-2-lyso-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04317 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2443 \"1-Organyl-2-lyso-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04317 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2444 \"1-Organyl-2-lyso-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04317 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2445 \"1-Organyl-2-lyso-sn-glycero-3-phosphocholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2446 \"1-alkyl-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16870 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2447 \"sn-glycero-3-Phosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16870 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2448 \"sn-glycero-3-Phosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16870 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2449 \"sn-glycero-3-Phosphocholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2450 \"dopaminochrome\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2451 \"dopaminochrome\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2452 \"dopaminochrome\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16424 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2453 \"Phenylacetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16424 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2454 \"Phenylacetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16424 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2455 \"Phenylacetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15621 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2456 \"4-Hydroxyphenylacetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15621 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2457 \"4-Hydroxyphenylacetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15621 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2458 \"4-Hydroxyphenylacetaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30745 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2459 \"Phenylacetic acid\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C15583 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2460 \"Phenyl acetate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04043 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2461 \"3,4-Dihydroxyphenylacetaldehyde\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C04043 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2462 \"3,4-Dihydroxyphenylacetaldehyde\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:kegg.compound:C04043 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2463 \"3,4-Dihydroxyphenylacetaldehyde\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05852 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2464 \"2-Hydroxyphenylacetate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05852 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2465 \"2-Hydroxyphenylacetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:31128 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2466 \"4-Hydroxyphenyl acetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:44747 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2467 \"Homogentisate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:44747 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2468 \"Homogentisate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:41941 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2469 \"3,4-Dihydroxyphenylacetate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05577 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2470 \"3,4-Dihydroxymandelaldehyde\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05577 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2471 \"3,4-Dihydroxymandelaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2472 \"3,4-dihydroxymandelate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:47904 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2473 \"4-Maleylacetoacetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30907 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2474 \"4-Fumarylacetoacetate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16562 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2475 \"2-Phenylacetamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17310 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2476 \"Pyridoxal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2477 \"(_ST_i_LT_N_ST_/i_LT_-acetylglucosaminyl)_ST_sub_LT_2_ST_/sub_LT_-diphosphodolichol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28111 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2478 \"3-Methoxy-4-hydroxyphenylacetaldehyde\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05582 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2479 \"Homovanillate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05583 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2480 \"3-Methoxy-4-hydroxyphenylglycolaldehyde\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05583 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2481 \"3-Methoxy-4-hydroxyphenylglycolaldehyde\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05583 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2482 \"3-Methoxy-4-hydroxyphenylglycolaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:2010627622 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2483 \"3-Methoxy-4-hydroxymandelate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:2010627622 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2484 \"3-Methoxy-4-hydroxymandelate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2485 \"1,2,3,4-tetrahydroisoquinoline\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C02355 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2486 \"2',3'-Cyclic UMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16095 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2487 \"Sepiapterin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17804 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2488 \"6-Pyruvoyltetrahydropterin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17804 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2489 \"6-Pyruvoyltetrahydropterin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17295 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2490 \"L-Phenylalanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17295 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2491 \"L-Phenylalanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17895 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2492 \"L-Tyrosine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17895 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2493 \"L-Tyrosine\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15765 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2494 \"3,4-Dihydroxy-L-phenylalanine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16450 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2495 \"Deoxyuridine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16450 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2496 \"Deoxyuridine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16704 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2497 \"Uridine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16704 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2498 \"Uridine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16704 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2499 \"Uridine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05594 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2500 \"3-Methoxy-4-hydroxyphenylethyleneglycol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17622 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2501 \"dUMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17622 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2502 \"dUMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16695 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2503 \"UMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16695 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2504 \"UMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16695 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2505 \"UMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16695 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2506 \"UMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16695 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2507 \"UMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18116 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2508 \"Pseudouridine 5'-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28895 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2509 \"3'-UMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15698 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2510 \"Deoxycytidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15698 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2511 \"Deoxycytidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15698 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2512 \"Deoxycytidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17562 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2513 \"Cytidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17562 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2514 \"Cytidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17562 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2515 \"Cytidine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C00268 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2516 \"Dihydrobiopterin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15375 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2517 \"7,8-Dihydrobiopterin\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17248 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2518 \"6-Lactoyl-5,6,7,8-tetrahydropterin\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05587 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2519 \"3-Methoxytyramine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05587 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2520 \"3-Methoxytyramine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28918 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2521 \"L-Adrenaline\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28918 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2522 \"L-Adrenaline\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05589 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2523 \"L-Normetanephrine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05589 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2524 \"L-Normetanephrine\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28850 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2525 \"dUDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28850 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2526 \"dUDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28850 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2527 \"dUDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17659 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2528 \"UDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17659 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2529 \"UDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17659 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2530 \"UDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17659 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2531 \"UDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17659 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2532 \"UDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17659 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2533 \"UDP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2534 \"O-beta-D-Xylosylprotein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2535 \"O-beta-D-Xylosylprotein\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15918 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2536 \"dCMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15918 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2537 \"dCMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15918 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2538 \"dCMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15918 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2539 \"dCMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17361 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2540 \"CMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17361 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2541 \"CMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17361 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2542 \"CMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17361 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2543 \"CMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17361 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2544 \"CMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17361 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2545 \"CMP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28413 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2546 \"1-(5-Phospho-D-ribosyl)-5-amino-4-imidazolecarboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2547 \"quinonoid dihydrobiopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2548 \"O2'-4a-cyclic-tetrahydrobiopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2549 \"1-phosphatidyl-myo-inositol 3,5-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2550 \"1-phosphatidyl-myo-inositol 3,5-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2551 \"1-phosphatidyl-myo-inositol 3,5-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2552 \"1-phosphatidyl-myo-inositol 3,5-bisphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17625 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2553 \"dUTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17625 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2554 \"dUTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15713 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2555 \"UTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15713 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2556 \"UTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15713 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2557 \"UTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15713 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2558 \"UTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15713 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2559 \"UTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15713 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2560 \"UTP\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C00705 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2561 \"dCDP\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C00705 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2562 \"dCDP\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C00705 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2563 \"dCDP\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C00705 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2564 \"dCDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17239 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2565 \"CDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17239 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2566 \"CDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17239 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2567 \"CDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17239 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2568 \"CDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17239 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2569 \"CDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17239 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2570 \"CDP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18406 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2571 \"1-(5'-Phosphoribosyl)-5-amino-4-imidazolecarboxamide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15372 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2572 \"Tetrahydrobiopterin\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C15522 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2573 \"4a-Hydroxytetrahydrobiopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2574 \"1-phosphatidyl-myo-inositol 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2575 \"1-phosphatidyl-myo-inositol 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2576 \"1-phosphatidyl-myo-inositol 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2577 \"1-phosphatidyl-myo-inositol 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2578 \"1-phosphatidyl-myo-inositol 5-phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16311 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2579 \"dCTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16311 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2580 \"dCTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16311 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2581 \"dCTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17677 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2582 \"CTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17677 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2583 \"CTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17677 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2584 \"CTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17677 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2585 \"CTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17677 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2586 \"CTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17677 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2587 \"CTP\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18372 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2588 \"2-Amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl)dihydropteridine triphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18372 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2589 \"2-Amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl)dihydropteridine triphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2590 \"tetrahydrobiopterin-4a-carbinolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2591 \"4-hydroperoxy-2-nonenal\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29032469057916 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2592 \"Pantothenate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29032469057916 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2593 \"Pantothenate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29032469057916 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2594 \"Pantothenate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15905 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2595 \"D-4'-Phosphopantothenate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15905 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2596 \"D-4'-Phosphopantothenate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29019 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2597 \"Nonanoic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15754 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2598 \"3-beta-D-Galactosyl-sn-glycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15754 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2599 \"3-beta-D-Galactosyl-sn-glycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15754 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2600 \"3-beta-D-Galactosyl-sn-glycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15754 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2601 \"3-beta-D-Galactosyl-sn-glycerol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16882 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2602 \"S-Aminomethyldihydrolipoylprotein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2603 \"1-acyl-2-lyso-glycerophosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17504 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2604 \"1-Acyl-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17504 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2605 \"1-Acyl-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17504 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2606 \"1-Acyl-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17504 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2607 \"1-Acyl-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16728 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2608 \"2-Acyl-sn-glycero-3-phosphocholine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15786 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2609 \"3-Hydroxy-N6,N6,N6-trimethyl-L-lysine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15786 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2610 \"3-Hydroxy-N6,N6,N6-trimethyl-L-lysine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17927 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2611 \"N1-Acetylspermidine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30851 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2612 \"Phenylpyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30851 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2613 \"Phenylpyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30851 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2614 \"Phenylpyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:32815 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2615 \"2-Hydroxy-3-phenylpropenoate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15999 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2616 \"3-(4-Hydroxyphenyl)pyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15999 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2617 \"3-(4-Hydroxyphenyl)pyruvate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15999 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2618 \"3-(4-Hydroxyphenyl)pyruvate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C05350 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2619 \"2-Hydroxy-3-(4-hydroxyphenyl)propenoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2620 \"adrenochrome\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2621 \"adrenochrome\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16852 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2622 \"Dopaquinone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16852 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2623 \"Dopaquinone\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29108 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2624 \"Calcium\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29108 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2625 \"Calcium\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2626 \"GD3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2627 \"Galactosylgloboside; SSEA-3; Gb5Cer\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2628 \"GM3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2629 \"GM3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2630 \"GM3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2631 \"GM1b\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2632 \"GM1b\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2633 \"GM1b\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2634 \"Monosialylgalactosylgloboside; MSGG; Monosialyl-Gb5; SSEA-4; V3NeuAc-Gb5Cer\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2635 \"one carbon unit\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16842 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2636 \"Formaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16842 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2637 \"Formaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16842 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2638 \"Formaldehyde\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30751 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2639 \"Formate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30751 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2640 \"Formate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30751 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2641 \"Formate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16199 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2642 \"Urea\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16199 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2643 \"Urea\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17672 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2644 \"Carbamoyl phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17672 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2645 \"Carbamoyl phosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17790 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2646 \"Methanol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17790 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2647 \"Methanol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16007 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2648 \"Methanethiol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16830 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2649 \"Methylamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16830 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2650 \"Methylamine\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18407 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2651 \"Hydrogen cyanide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29200 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2652 \"Thiocyanate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2653 \"Fatty acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2654 \"Fatty acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2655 \"Fatty acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2656 \"Fatty acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2657 \"Fatty acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35366 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2658 \"Fatty acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17996 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2659 \"Cl-\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17996 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2660 \"Cl-\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17514 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2661 \"Cyanide\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17245 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2662 \"CO\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2663 \"CO2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2664 \"CO2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2665 \"CO2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2666 \"CO2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2667 \"CO2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2668 \"CO2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2669 \"CO2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16526 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2670 \"CO2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35757 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2671 \"Carboxylate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35757 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2672 \"Carboxylate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35757 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2673 \"Carboxylate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35757 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2674 \"Carboxylate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35757 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2675 \"Carboxylate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:35757 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2676 \"Carboxylate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18248 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2677 \"Iron\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18248 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2678 \"Iron\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29033 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2679 \"Fe2+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29033 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2680 \"Fe2+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29033 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2681 \"Fe2+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29034 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2682 \"Fe3+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29034 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2683 \"Fe3+\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2684 \"Melibiose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2685 \"Glycogen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2686 \"Dextrin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2687 \"Chondroitin 6-sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2688 \"Chondroitin 4-sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2689 \"22748-GlcNAc1Cer1Gal3Glc1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2690 \"22748-GlcNAc1Cer1Gal3Glc1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2691 \"GD1b\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2692 \"GT1b\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2693 \"Fuc-Lc4Cer; III4-a-Fuc-Lc4Cer; Lea glycolipid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2694 \"Fuc-Lc4Cer; III4-a-Fuc-Lc4Cer; Lea glycolipid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2695 \"IV2Fuc-nLc4Cer; IV2-a-Fuc-nLc4Cer; Type IIH glycolipid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2696 \"22761-GlcNAc1LFuc1Cer1Gal3Glc1GalNAc1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2697 \"IV2Fuc,III4Fuc-Lc4Cer; IV2-a-Fuc,III4-a-Fuc-Lc4Cer; Leb glycolipid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2698 \"22749-GlcNAc1LFuc2Cer1Gal3Glc1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2699 \"22749-GlcNAc1LFuc2Cer1Gal3Glc1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2700 \"Type IIIH glycolipid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2701 \"nLc6Cer; i-antigen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2702 \"VI2Fuc-nLc6\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2703 \"22774-GlcNAc2LFuc1Cer1Gal4Glc1GalNAc1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2704 \"22775-GlcNAc2LFuc2Cer1Gal4Glc1GalNAc1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2705 \"22729-GlcNAc2Man5Asn1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2706 \"22728-GlcNAc2Man9Asn1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2707 \"22728-GlcNAc2Man9Asn1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2708 \"22727-GlcNAc2Man9Asn1Glc1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2709 \"22727-GlcNAc2Man9Asn1Glc1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2710 \"22842-GlcNAc2Man9Asn1Glc2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2711 \"22842-GlcNAc2Man9Asn1Glc2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2712 \"nLc7Cer\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2713 \"22731-GlcNAc3Man3Asn1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2714 \"22730-GlcNAc3Man5Asn1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2715 \"22733-GlcNAc4LFuc1Man3Asn1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2716 \"22733-GlcNAc4LFuc1Man3Asn1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2717 \"22734-GlcNAc4LFuc1Man3Asn1Gal2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2718 \"DS 3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2719 \"22732-GlcNAc4Man3Asn1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2720 \"22732-GlcNAc4Man3Asn1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2721 \"22737-GlcNAc5Man3Asn1\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2722 \"H+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2723 \"H+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2724 \"H+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2725 \"H+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2726 \"H+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2727 \"H+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2728 \"H+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15378 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2729 \"H+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18276 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2730 \"Hydrogen\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15377 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2731 \"H2O\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15377 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2732 \"H2O\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15377 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2733 \"H2O\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15377 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2734 \"H2O\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15377 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2735 \"H2O\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15377 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2736 \"H2O\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15377 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2737 \"H2O\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15377 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2738 \"H2O\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16240 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2739 \"H2O2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16240 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2740 \"H2O2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16240 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2741 \"H2O2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16240 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2742 \"H2O2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16240 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2743 \"H2O2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16240 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2744 \"H2O2\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16136 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2745 \"Hydrogen sulfide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:5587 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2746 \"H2S2O3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:48854 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2747 \"Sulfite\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:48854 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2748 \"Sulfite\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16189 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2749 \"Sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16189 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2750 \"Sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16189 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2751 \"Sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16189 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2752 \"Sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16189 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2753 \"Sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16189 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2754 \"Sulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18367 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2755 \"Orthophosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18367 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2756 \"Orthophosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18367 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2757 \"Orthophosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18367 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2758 \"Orthophosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18367 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2759 \"Orthophosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18367 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2760 \"Orthophosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:47266 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2761 \"Hydrobromic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:47266 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2762 \"Hydrobromic acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17883 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2763 \"Hydrochloric acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17883 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2764 \"Hydrochloric acid\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17544 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2765 \"HCO3-\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17544 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2766 \"HCO3-\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:17544 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2767 \"HCO3-\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2768 \"hydroxyl radical\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:30879 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2769 \"Alcohol\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:33541 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2770 \"Thiosulfate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16382 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2771 \"Iodide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16382 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2772 \"Iodide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29101 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2773 \"Sodium\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:29101 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2774 \"Sodium\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C01664 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2775 \"Alkylamine\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C16722 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2776 \"Acceptor\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C16722 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2777 \"Acceptor\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C16722 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2778 \"Acceptor\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16134 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2779 \"NH3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16134 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2780 \"NH3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16134 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2781 \"NH3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16134 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2782 \"NH3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16134 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2783 \"NH3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16134 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2784 \"NH3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16134 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2785 \"NH3\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28938 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2786 \"NH4+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:28938 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2787 \"NH4+\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:16480 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2788 \"Nitric oxide\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15379 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2789 \"Oxygen\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15379 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2790 \"Oxygen\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15379 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2791 \"Oxygen\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15379 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2792 \"Oxygen\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:15379 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2793 \"Oxygen\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18421 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2794 \"O2.-\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18421 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2795 \"O2.-\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18421 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2796 \"O2.-\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:18421 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2797 \"O2.-\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2798 \"hypochlorite\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1836129888 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2799 \"Diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1836129888 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2800 \"Diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1836129888 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2801 \"Diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1836129888 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2802 \"Diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1836129888 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2803 \"Diphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1803639949 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2804 \"Triphosphate\">\n",
      "urn:miriam:urn:miriam:obo.chebi:CHEBI:1803639949 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2805 \"Triphosphate\">\n",
      "urn:miriam:urn:miriam:kegg.compound:C02534 does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2806 \"Dialkyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2807 \"22724-PP-Dol1GlcNAc2Man3\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2808 \"31676-PP-Dol1GlcNAc2Man4\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2809 \"31679-PP-Dol1GlcNAc2Man6\">\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2810 \"Tn antigen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2811 \"22829-Ser1Xyl1Gal1\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_m2812 \"22829-Ser1Xyl1Gal1\">\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4 \"2-(alpha-Hydroxyethyl)thiamine diphosphate+CO2=Thiamindiphosphate+Pyruvate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4 \"2-(alpha-Hydroxyethyl)thiamine diphosphate+CO2=Thiamindiphosphate+Pyruvate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_11 \"2 Prostaglandin H2+Oxygen=2 Prostaglandin G2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_11 \"2 Prostaglandin H2+Oxygen=2 Prostaglandin G2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_13 \"Oxygen+4 Fe2++4 H+=4 Fe3++2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_13 \"Oxygen+4 Fe2++4 H+=4 Fe3++2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_14 \"Oxygen+4 Fe2++4 H+=4 Fe3++2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_14 \"Oxygen+4 Fe2++4 H+=4 Fe3++2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_17 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_17 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_18 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_18 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_19 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_19 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_20 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_20 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_21 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_21 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_22 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_22 \"ATP+H2O=ADP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_23 \"ATP=3',5'-Cyclic AMP+Pyrophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_23 \"ATP=3',5'-Cyclic AMP+Pyrophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_25 \"NAD++2 Glutathione=Oxidized glutathione+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_25 \"NAD++2 Glutathione=Oxidized glutathione+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_26 \"NADH+2 Ferricytochrome b5=NAD++2 Ferrocytochrome b5\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_26 \"NADH+2 Ferricytochrome b5=NAD++2 Ferrocytochrome b5\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_28 \"NADH+2 Ferricytochrome b5=NAD++2 Ferrocytochrome b5\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_28 \"NADH+2 Ferricytochrome b5=NAD++2 Ferrocytochrome b5\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_30 \"NAD++H2O=AMP+Nicotinamide D-ribonucleotide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_30 \"NAD++H2O=AMP+Nicotinamide D-ribonucleotide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_31 \"NAD++H2O=AMP+Nicotinamide D-ribonucleotide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_31 \"NAD++H2O=AMP+Nicotinamide D-ribonucleotide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_32 \"NAD++H2O=AMP+Nicotinamide D-ribonucleotide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_32 \"NAD++H2O=AMP+Nicotinamide D-ribonucleotide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_35 \"ATP+NAD+=ADP+NADP+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_35 \"ATP+NAD+=ADP+NADP+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_41 \"NADPH+NAD+=NADP++NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_41 \"NADPH+NAD+=NADP++NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_42 \"NADPH+NAD+=NADP++NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_42 \"NADPH+NAD+=NADP++NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_44 \"NADP++2 Glutathione=Oxidized glutathione+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_44 \"NADP++2 Glutathione=Oxidized glutathione+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_46 \"ADP+H2O=AMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_46 \"ADP+H2O=AMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_50 \"ATP+AMP=2 ADP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_50 \"ATP+AMP=2 ADP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_51 \"ATP+AMP=2 ADP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_51 \"ATP+AMP=2 ADP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_52 \"ATP+AMP=2 ADP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_52 \"ATP+AMP=2 ADP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_55 \"ATP+Nicotinamide D-ribonucleotide=Pyrophosphate+NAD+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_55 \"ATP+Nicotinamide D-ribonucleotide=Pyrophosphate+NAD+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_57 \"2 ATP+NH3+CO2+H2O=2 ADP+Orthophosphate+Carbamoylphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_57 \"2 ATP+NH3+CO2+H2O=2 ADP+Orthophosphate+Carbamoylphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_58 \"UDP+H2O=UMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_58 \"UDP+H2O=UMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_59 \"UDP+H2O=UMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_59 \"UDP+H2O=UMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_60 \"UDP+H2O=UMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_60 \"UDP+H2O=UMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_61 \"UDP+H2O=UMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_61 \"UDP+H2O=UMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_62 \"ATP+UDP=ADP+UTP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_62 \"ATP+UDP=ADP+UTP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_64 \"ATP+UDP=ADP+UTP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_64 \"ATP+UDP=ADP+UTP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_67 \"ATP+UMP=ADP+UDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_67 \"ATP+UMP=ADP+UDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_68 \"UTP+H2O=UDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_68 \"UTP+H2O=UDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_69 \"UTP+H2O=UDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_69 \"UTP+H2O=UDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_70 \"UTP+H2O=UDP+Orthophosphate\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_70 \"UTP+H2O=UDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_71 \"UTP+H2O=UDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_71 \"UTP+H2O=UDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_73 \"FAD+H2O=AMP+FMN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_73 \"FAD+H2O=AMP+FMN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_74 \"ATP+FMN=Pyrophosphate+FAD\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_74 \"ATP+FMN=Pyrophosphate+FAD\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_90 \"Orthophosphate+Pyrophosphate+S-Adenosyl-L-methionine=ATP +L-Methionine+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_90 \"Orthophosphate+Pyrophosphate+S-Adenosyl-L-methionine=ATP +L-Methionine+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_91 \"S-Adenosyl-L-methionine=S-Adenosylmethioninamine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_91 \"S-Adenosyl-L-methionine=S-Adenosylmethioninamine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_93 \"AMP+H2O=IMP+NH3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_93 \"AMP+H2O=IMP+NH3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_94 \"AMP+H2O=IMP+NH3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_94 \"AMP+H2O=IMP+NH3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_96 \"AMP+H2O=Adenosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_96 \"AMP+H2O=Adenosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_97 \"AMP+H2O=Adenosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_97 \"AMP+H2O=Adenosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_109 \"ATP+Pyruvate=ADP+Phosphoenolpyruvate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_109 \"ATP+Pyruvate=ADP+Phosphoenolpyruvate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_111 \"Pyruvate+CoA+NAD+=Acetyl-CoA+CO2+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_111 \"Pyruvate+CoA+NAD+=Acetyl-CoA+CO2+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_113 \"Pyruvate+CoA+NADP+=Acetyl-CoA+CO2+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_113 \"Pyruvate+CoA+NADP+=Acetyl-CoA+CO2+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_114 \"(S)-Malate+NAD+=Pyruvate+CO2+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_114 \"(S)-Malate+NAD+=Pyruvate+CO2+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_115 \"(S)-Malate+NAD+=Pyruvate+CO2+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_115 \"(S)-Malate+NAD+=Pyruvate+CO2+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_116 \"(S)-Malate+NADP+=Pyruvate+CO2+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_116 \"(S)-Malate+NADP+=Pyruvate+CO2+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_117 \"(S)-Malate+NADP+=Pyruvate+CO2+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_117 \"(S)-Malate+NADP+=Pyruvate+CO2+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_119 \"(S)-2-Acetolactate+CO2=2 Pyruvate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_119 \"(S)-2-Acetolactate+CO2=2 Pyruvate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_120 \"Acetyl-CoA+H2O=CoA+Acetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_120 \"Acetyl-CoA+H2O=CoA+Acetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_127 \"2 Acetyl-CoA=CoA+Acetoacetyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_127 \"2 Acetyl-CoA=CoA+Acetoacetyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_129 \"ATP+L-Glutamate=ADP+L-Glutamyl 5-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_129 \"ATP+L-Glutamate=ADP+L-Glutamyl 5-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_132 \"L-Glutamate 5-semialdehyde+NAD++H2O=L-Glutamate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_132 \"L-Glutamate 5-semialdehyde+NAD++H2O=L-Glutamate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_137 \"ATP+L-Glutamate+NH3=ADP+Orthophosphate+L-Glutamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_137 \"ATP+L-Glutamate+NH3=ADP+Orthophosphate+L-Glutamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_139 \"L-Glutamine+H2O=L-Glutamate+NH3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_139 \"L-Glutamine+H2O=L-Glutamate+NH3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_143 \"L-Glutamate=4-Aminobutanoate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_143 \"L-Glutamate=4-Aminobutanoate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_144 \"L-Glutamate=4-Aminobutanoate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_144 \"L-Glutamate=4-Aminobutanoate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_146 \"Oxalosuccinate=2-Oxoglutarate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_146 \"Oxalosuccinate=2-Oxoglutarate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_147 \"Oxalosuccinate=2-Oxoglutarate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_147 \"Oxalosuccinate=2-Oxoglutarate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_148 \"Oxalosuccinate=2-Oxoglutarate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_148 \"Oxalosuccinate=2-Oxoglutarate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_152 \"2 O2.-+2 H+=H2O2+Oxygen\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_152 \"2 O2.-+2 H+=H2O2+Oxygen\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_153 \"2 O2.-+2 H+=H2O2+Oxygen\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_153 \"2 O2.-+2 H+=H2O2+Oxygen\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_155 \"2 O2.-+2 H+=H2O2+Oxygen (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_155 \"2 O2.-+2 H+=H2O2+Oxygen (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_157 \"2 O2.-+2 H+=H2O2+Oxygen (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_157 \"2 O2.-+2 H+=H2O2+Oxygen (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_159 \"Pyridoxine phosphate+Oxygen=H2O2+Pyridoxal phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_159 \"Pyridoxine phosphate+Oxygen=H2O2+Pyridoxal phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_161 \"UDPglucose+H2O=UMP+D-Glucose 1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_161 \"UDPglucose+H2O=UMP+D-Glucose 1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_162 \"UDPglucose+H2O=UMP+D-Glucose 1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_162 \"UDPglucose+H2O=UMP+D-Glucose 1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_165 \"UDPglucose+Amylose=UDP+Amylose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_165 \"UDPglucose+Amylose=UDP+Amylose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_166 \"Protoporphyrin+Fe2+=Heme+2 H+ (reversible)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_166 \"Protoporphyrin+Fe2+=Heme+2 H+ (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_167 \"Protoporphyrin+Fe2+=Heme+2 H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_167 \"Protoporphyrin+Fe2+=Heme+2 H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_171 \"GDP+H2O=GMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_171 \"GDP+H2O=GMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_172 \"GDP+H2O=GMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_172 \"GDP+H2O=GMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_173 \"GDP+H2O=GMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_173 \"GDP+H2O=GMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_176 \"GDP+H2O=GMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_176 \"GDP+H2O=GMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_181 \"ATP+GDP=ADP+GTP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_181 \"ATP+GDP=ADP+GTP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_183 \"ATP+GDP=ADP+GTP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_183 \"ATP+GDP=ADP+GTP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_184 \"ATP+GMP=ADP+GDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_184 \"ATP+GMP=ADP+GDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_185 \"ATP+GMP=ADP+GDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_185 \"ATP+GMP=ADP+GDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_188 \"GTP+H2O=GDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_188 \"GTP+H2O=GDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_189 \"GTP+H2O=GDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_189 \"GTP+H2O=GDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_190 \"GTP+H2O=GDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_190 \"GTP+H2O=GDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_191 \"(S)-Malate+NAD+=Oxaloacetate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_191 \"(S)-Malate+NAD+=Oxaloacetate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_192 \"(S)-Malate+NAD+=Oxaloacetate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_192 \"(S)-Malate+NAD+=Oxaloacetate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_197 \"L-Aspartate+2-Oxoglutarate=Oxaloacetate+L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_197 \"L-Aspartate+2-Oxoglutarate=Oxaloacetate+L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_198 \"L-Aspartate+2-Oxoglutarate=Oxaloacetate+L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_198 \"L-Aspartate+2-Oxoglutarate=Oxaloacetate+L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_201 \"Citrate=Acetate+Oxaloacetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_201 \"Citrate=Acetate+Oxaloacetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_202 \"Citrate=Acetate+Oxaloacetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_202 \"Citrate=Acetate+Oxaloacetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_204 \"Glycine+H2O+Oxygen=Glyoxylate+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_204 \"Glycine+H2O+Oxygen=Glyoxylate+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_206 \"L-Alanine+Glyoxylate=Pyruvate+Glycine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_206 \"L-Alanine+Glyoxylate=Pyruvate+Glycine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_207 \"Acetyl-CoA+Glycine=CoA+L-2-Amino-3-oxobutanoic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_207 \"Acetyl-CoA+Glycine=CoA+L-2-Amino-3-oxobutanoic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_222 \"ATP+Succinate+CoA=ADP+Orthophosphate+Succinyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_222 \"ATP+Succinate+CoA=ADP+Orthophosphate+Succinyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_226 \"UDP-N-acetyl-D-glucosamine=UDP-N-acetyl-D-mannosamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_226 \"UDP-N-acetyl-D-glucosamine=UDP-N-acetyl-D-mannosamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_231 \"GTP+Pyruvate=GDP+Phosphoenolpyruvate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_231 \"GTP+Pyruvate=GDP+Phosphoenolpyruvate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_232 \"GTP+Oxaloacetate=GDP+Phosphoenolpyruvate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_232 \"GTP+Oxaloacetate=GDP+Phosphoenolpyruvate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_233 \"GTP+Oxaloacetate=GDP+Phosphoenolpyruvate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_233 \"GTP+Oxaloacetate=GDP+Phosphoenolpyruvate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_234 \"GTP+Succinate+CoA=GDP+Orthophosphate+Succinyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_234 \"GTP+Succinate+CoA=GDP+Orthophosphate+Succinyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_235 \"GTP=3',5'-Cyclic GMP+Pyrophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_235 \"GTP=3',5'-Cyclic GMP+Pyrophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_236 \"GTP=3',5'-Cyclic GMP+Pyrophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_236 \"GTP=3',5'-Cyclic GMP+Pyrophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_261 \"L-Aspartate=beta-Alanine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_261 \"L-Aspartate=beta-Alanine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_262 \"L-Aspartate=beta-Alanine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_262 \"L-Aspartate=beta-Alanine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_266 \"UDP-D-galactose+D-Glucose=UDP+Lactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_266 \"UDP-D-galactose+D-Glucose=UDP+Lactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_267 \"UDP-D-galactose+D-Glucose=UDP+Lactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_267 \"UDP-D-galactose+D-Glucose=UDP+Lactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_268 \"UDP-D-galactose+D-Glucose=UDP+Lactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_268 \"UDP-D-galactose+D-Glucose=UDP+Lactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_269 \"3'-Phosphoadenylyl sulfate+H2O=Sulfate+Adenosine3',5'-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_269 \"3'-Phosphoadenylyl sulfate+H2O=Sulfate+Adenosine3',5'-bisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_270 \"3'-Phosphoadenylyl sulfate+H2O=Sulfate+Adenosine3',5'-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_270 \"3'-Phosphoadenylyl sulfate+H2O=Sulfate+Adenosine3',5'-bisphosphate\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_273 \"CMP+H2O=Cytidine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_273 \"CMP+H2O=Cytidine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_276 \"CMP+H2O=Cytidine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_276 \"CMP+H2O=Cytidine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_277 \"CMP+H2O=Cytidine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_277 \"CMP+H2O=Cytidine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_278 \"ATP+CMP=ADP+CDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_278 \"ATP+CMP=ADP+CDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_279 \"ATP+CMP=ADP+CDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_279 \"ATP+CMP=ADP+CDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_281 \"CDP+H2O=CMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_281 \"CDP+H2O=CMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_282 \"CDP+H2O=CMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_282 \"CDP+H2O=CMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_283 \"CDP+H2O=CMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_283 \"CDP+H2O=CMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_284 \"CDP+H2O=CMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_284 \"CDP+H2O=CMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_289 \"ATP+Sulfate=Pyrophosphate+Adenylylsulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_289 \"ATP+Sulfate=Pyrophosphate+Adenylylsulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_290 \"Sulfite+Oxygen+H2O=Sulfate+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_290 \"Sulfite+Oxygen+H2O=Sulfate+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_291 \"Sulfite+Oxygen+H2O=Sulfate+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_291 \"Sulfite+Oxygen+H2O=Sulfate+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_293 \"FMN+H2O=Riboflavin+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_293 \"FMN+H2O=Riboflavin+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_294 \"ATP+Riboflavin=ADP+FMN\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_294 \"ATP+Riboflavin=ADP+FMN\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_296 \"NAD++L-Arginine=Nicotinamide +N(omega)-(ADP-D-ribosyl)-L-arginine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_296 \"NAD++L-Arginine=Nicotinamide +N(omega)-(ADP-D-ribosyl)-L-arginine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_305 \"L-Arginine+Glycine=L-Ornithine+Guanidinoacetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_305 \"L-Arginine+Glycine=L-Ornithine+Guanidinoacetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_306 \"L-Arginine=Agmatine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_306 \"L-Arginine=Agmatine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_307 \"CTP+H2O=CDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_307 \"CTP+H2O=CDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_308 \"CTP+H2O=CDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_308 \"CTP+H2O=CDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_309 \"CTP+H2O=CDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_309 \"CTP+H2O=CDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_310 \"CTP+H2O=CDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_310 \"CTP+H2O=CDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_313 \"ATP+CDP=ADP+CTP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_313 \"ATP+CDP=ADP+CTP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_319 \"L-Serine+Pyruvate=Hydroxypyruvate+L-Alanine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_319 \"L-Serine+Pyruvate=Hydroxypyruvate+L-Alanine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_325 \"Methanol+H2O2=Formaldehyde+2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_325 \"Methanol+H2O2=Formaldehyde+2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_330 \"Sarcosine+H2O+Oxygen=Glycine+Formaldehyde+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_330 \"Sarcosine+H2O+Oxygen=Glycine+Formaldehyde+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_333 \"ATP+Thiamin diphosphate=ADP+Thiamin triphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_333 \"ATP+Thiamin diphosphate=ADP+Thiamin triphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_336 \"2-Oxoglutarate+Thiamin diphosphate _ST_=_LT_3-Carboxy-1-hydroxypropyl-ThPP+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_336 \"2-Oxoglutarate+Thiamin diphosphate _ST_=_LT_3-Carboxy-1-hydroxypropyl-ThPP+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_343 \"2-Phospho-D-glycerate=Phosphoenolpyruvate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_343 \"2-Phospho-D-glycerate=Phosphoenolpyruvate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_344 \"2-Phospho-D-glycerate=Phosphoenolpyruvate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_344 \"2-Phospho-D-glycerate=Phosphoenolpyruvate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_345 \"UTP+Pyruvate=UDP+Phosphoenolpyruvate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_345 \"UTP+Pyruvate=UDP+Phosphoenolpyruvate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_348 \"L-Ornithine+2-Oxoglutarate=L-Glutamate 5-semialdehyde +L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_348 \"L-Ornithine+2-Oxoglutarate=L-Glutamate 5-semialdehyde +L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_350 \"L-Ornithine=Putrescine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_350 \"L-Ornithine=Putrescine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_354 \"L-Tryptophan=Tryptamine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_354 \"L-Tryptophan=Tryptamine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_357 \"L-Phenylalanine+2-Oxoglutarate=Phenylpyruvate+L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_357 \"L-Phenylalanine+2-Oxoglutarate=Phenylpyruvate+L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_358 \"L-Phenylalanine=2-Phenylacetamide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_358 \"L-Phenylalanine=2-Phenylacetamide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_360 \"L-Phenylalanine=Phenethylamine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_360 \"L-Phenylalanine=Phenethylamine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_362 \"2 trans,trans-Farnesyl diphosphate=Pyrophosphate+Presqualenediphosphate+H+\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_362 \"2 trans,trans-Farnesyl diphosphate=Pyrophosphate+Presqualenediphosphate+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_365 \"3-Oxopropanoate+CoA+NAD+=Acetyl-CoA+CO2+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_365 \"3-Oxopropanoate+CoA+NAD+=Acetyl-CoA+CO2+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_366 \"3-Oxopropanoate+CoA+NADP+=Acetyl-CoA+CO2+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_366 \"3-Oxopropanoate+CoA+NADP+=Acetyl-CoA+CO2+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_367 \"(S)-1-Pyrroline-5-carboxylate+NAD++2 H2O=L-Glutamate+NADH+ H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_367 \"(S)-1-Pyrroline-5-carboxylate+NAD++2 H2O=L-Glutamate+NADH+ H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_368 \"(S)-1-Pyrroline-5-carboxylate+NADP++2 H2O=L-Glutamate +NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_368 \"(S)-1-Pyrroline-5-carboxylate+NADP++2 H2O=L-Glutamate +NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_370 \"Isocitrate+NAD+=2-Oxoglutarate+CO2+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_370 \"Isocitrate+NAD+=2-Oxoglutarate+CO2+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_371 \"Acetaldehyde+NAD++H2O=Acetate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_371 \"Acetaldehyde+NAD++H2O=Acetate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_372 \"Acetaldehyde+NAD++H2O=Acetate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_372 \"Acetaldehyde+NAD++H2O=Acetate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_373 \"Acetaldehyde+NADP++H2O=Acetate+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_373 \"Acetaldehyde+NADP++H2O=Acetate+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_374 \"Acetaldehyde+NADP++H2O=Acetate+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_374 \"Acetaldehyde+NADP++H2O=Acetate+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_376 \"Succinate semialdehyde+NAD++H2O=Succinate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_376 \"Succinate semialdehyde+NAD++H2O=Succinate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_377 \"N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O=L-Lysine +2-Oxoglutarate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_377 \"N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O=L-Lysine +2-Oxoglutarate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_378 \"N6-(L-1,3-Dicarboxypropyl)-L-lysine+NADP++H2O=L-Lysine +2-Oxoglutarate+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_378 \"N6-(L-1,3-Dicarboxypropyl)-L-lysine+NADP++H2O=L-Lysine +2-Oxoglutarate+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_380 \"ITP+H2O=IDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_380 \"ITP+H2O=IDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_381 \"ITP+H2O=IDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_381 \"ITP+H2O=IDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_386 \"ATP+IDP=ADP+ITP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_386 \"ATP+IDP=ADP+ITP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_388 \"ATP+IDP=ADP+ITP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_388 \"ATP+IDP=ADP+ITP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_389 \"ITP+D-Glucose=IDP+D-Glucose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_389 \"ITP+D-Glucose=IDP+D-Glucose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_391 \"ITP+Oxaloacetate=IDP+Phosphoenolpyruvate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_391 \"ITP+Oxaloacetate=IDP+Phosphoenolpyruvate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_392 \"ITP+Oxaloacetate=IDP+Phosphoenolpyruvate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_392 \"ITP+Oxaloacetate=IDP+Phosphoenolpyruvate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_393 \"ITP+Succinate+CoA=IDP+Orthophosphate+Succinyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_393 \"ITP+Succinate+CoA=IDP+Orthophosphate+Succinyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_398 \"L-Tyrosine+2-Oxoglutarate=3-(4-Hydroxyphenyl)pyruvate +L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_398 \"L-Tyrosine+2-Oxoglutarate=3-(4-Hydroxyphenyl)pyruvate +L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_400 \"L-Tyrosine=Tyramine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_400 \"L-Tyrosine=Tyramine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_403 \"L-Tyrosine=Tyramine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_403 \"L-Tyrosine=Tyramine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_404 \"3-Oxopropanoate+CoA+NADP+=Malonyl-CoA+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_404 \"3-Oxopropanoate+CoA+NADP+=Malonyl-CoA+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_406 \"Ethanol+NADP+=Acetaldehyde+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_406 \"Ethanol+NADP+=Acetaldehyde+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_409 \"Ethanol+NAD+=Acetaldehyde+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_409 \"Ethanol+NAD+=Acetaldehyde+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_413 \"D-Mannose 6-phosphate=D-Fructose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_413 \"D-Mannose 6-phosphate=D-Fructose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_423 \"Succinyl-CoA+Acetyl-CoA=CoA+3-Oxoadipyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_423 \"Succinyl-CoA+Acetyl-CoA=CoA+3-Oxoadipyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_425 \"Succinyl-CoA+Glycine=5-Aminolevulinate+CoA+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_425 \"Succinyl-CoA+Glycine=5-Aminolevulinate+CoA+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_426 \"Succinyl-CoA+Glycine=2-Amino-3-oxoadipate+CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_426 \"Succinyl-CoA+Glycine=2-Amino-3-oxoadipate+CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_427 \"(R)-2-Methyl-3-oxopropanoyl-CoA=Succinyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_427 \"(R)-2-Methyl-3-oxopropanoyl-CoA=Succinyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_428 \"1L-myo-Inositol 1-phosphate=D-Glucose 6-phosphate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_428 \"1L-myo-Inositol 1-phosphate=D-Glucose 6-phosphate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_429 \"sn-Glycerol 3-phosphate+NAD+=Glycerone phosphate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_429 \"sn-Glycerol 3-phosphate+NAD+=Glycerone phosphate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_430 \"sn-Glycerol 3-phosphate+NAD+=Glycerone phosphate+NADH+H+\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_430 \"sn-Glycerol 3-phosphate+NAD+=Glycerone phosphate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_432 \"ATP+Glycerol=ADP+sn-Glycerol 3-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_432 \"ATP+Glycerol=ADP+sn-Glycerol 3-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_434 \"sn-Glycerol 3-phosphate+FAD=Glycerone phosphate+FADH2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_434 \"sn-Glycerol 3-phosphate+FAD=Glycerone phosphate+FADH2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_435 \"sn-Glycerol 3-phosphate+Acceptor=Glycerone phosphate+Reducedacceptor\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_435 \"sn-Glycerol 3-phosphate+Acceptor=Glycerone phosphate+Reducedacceptor\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_436 \"sn-Glycerol 3-phosphate+Acceptor=Glycerone phosphate+Reducedacceptor\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_436 \"sn-Glycerol 3-phosphate+Acceptor=Glycerone phosphate+Reducedacceptor\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_440 \"ATP+D-Fructose=ADP+beta-D-Fructose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_440 \"ATP+D-Fructose=ADP+beta-D-Fructose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_442 \"ITP+D-Fructose=IDP+D-Fructose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_442 \"ITP+D-Fructose=IDP+D-Fructose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_449 \"L-Cysteine+Oxygen=3-Sulfino-L-alanine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_449 \"L-Cysteine+Oxygen=3-Sulfino-L-alanine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_450 \"ATP+L-Glutamate+L-Cysteine=ADP+Orthophosphate +gamma-L-Glutamyl-L-cysteine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_450 \"ATP+L-Glutamate+L-Cysteine=ADP+Orthophosphate +gamma-L-Glutamyl-L-cysteine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_451 \"L-Cysteine+2-Oxoglutarate=Mercaptopyruvate+Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_451 \"L-Cysteine+2-Oxoglutarate=Mercaptopyruvate+Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_452 \"L-Cysteine+2-Oxoglutarate=Mercaptopyruvate+Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_452 \"L-Cysteine+2-Oxoglutarate=Mercaptopyruvate+Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_453 \"Cys-Gly+H2O=L-Cysteine+Glycine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_453 \"Cys-Gly+H2O=L-Cysteine+Glycine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_454 \"beta-Aminopropion aldehyde+NAD++H2O=beta-Alanine+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_454 \"beta-Aminopropion aldehyde+NAD++H2O=beta-Alanine+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_456 \"beta-Aminopropion aldehyde+NAD++H2O=beta-Alanine+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_456 \"beta-Aminopropion aldehyde+NAD++H2O=beta-Alanine+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_459 \"beta-Alanine+2-Oxoglutarate=3-Oxopropanoate+L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_459 \"beta-Alanine+2-Oxoglutarate=3-Oxopropanoate+L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_463 \"Propanoyl-CoA+Acetyl-CoA=CoA+2-Methylacetoacetyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_463 \"Propanoyl-CoA+Acetyl-CoA=CoA+2-Methylacetoacetyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_464 \"(S)-Methylmalonate semialdehyde+CoA+NAD+=Propanoyl-CoA+CO2+ NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_464 \"(S)-Methylmalonate semialdehyde+CoA+NAD+=Propanoyl-CoA+CO2+ NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_475 \"Tetrahydrofolate+Formate+ATP=ADP+Orthophosphate +10-Formyltetrahydrofolate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_475 \"Tetrahydrofolate+Formate+ATP=ADP+Orthophosphate +10-Formyltetrahydrofolate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_476 \"5,10-Methylenetetrahydrofolate+Glycine+H2O=Tetrahydrofolate+ L-Serine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_476 \"5,10-Methylenetetrahydrofolate+Glycine+H2O=Tetrahydrofolate+ L-Serine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_477 \"5,10-Methylenetetrahydrofolate+Glycine+H2O=Tetrahydrofolate+ L-Serine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_477 \"5,10-Methylenetetrahydrofolate+Glycine+H2O=Tetrahydrofolate+ L-Serine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_480 \"UDPglucose+alpha-D-Galactose 1-phosphate=D-Glucose 1-phosphate+ UDP-D-galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_480 \"UDPglucose+alpha-D-Galactose 1-phosphate=D-Glucose 1-phosphate+ UDP-D-galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_481 \"D-Glucose 1-phosphate=alpha-D-Glucose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_481 \"D-Glucose 1-phosphate=alpha-D-Glucose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_483 \"IDP+H2O=IMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_483 \"IDP+H2O=IMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_489 \"UMP+H2O=Uridine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_489 \"UMP+H2O=Uridine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_492 \"UMP+H2O=Uridine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_492 \"UMP+H2O=Uridine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_493 \"UMP+H2O=Uridine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_493 \"UMP+H2O=Uridine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_500 \"5,6-Dihydrouracil+NADP+=Uracil+NADPH+H+ (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_500 \"5,6-Dihydrouracil+NADP+=Uracil+NADPH+H+ (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_507 \"UDPglucose+Dolichyl phosphate=UDP+Dolichyl beta-D-glucosylphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_507 \"UDPglucose+Dolichyl phosphate=UDP+Dolichyl beta-D-glucosylphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_509 \"UDP-N-acetyl-D-glucosamine+Dolichyl phosphate=UMP +N-Acetyl-D-glucosaminyldiphosphodolichol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_509 \"UDP-N-acetyl-D-glucosamine+Dolichyl phosphate=UMP +N-Acetyl-D-glucosaminyldiphosphodolichol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_510 \"UDP-N-acetyl-D-glucosamine+Dolichyl phosphate=UMP +N-Acetyl-D-glucosaminyldiphosphodolichol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_510 \"UDP-N-acetyl-D-glucosamine+Dolichyl phosphate=UMP +N-Acetyl-D-glucosaminyldiphosphodolichol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_511 \"GDPmannose+Dolichyl phosphate=GDP+Dolichyl phosphateD-mannose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_511 \"GDPmannose+Dolichyl phosphate=GDP+Dolichyl phosphateD-mannose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_517 \"CTP+Dolichol=CDP+Dolichyl phosphate\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_517 \"CTP+Dolichol=CDP+Dolichyl phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_519 \"Acetyl-CoA+Choline=CoA+Acetylcholine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_519 \"Acetyl-CoA+Choline=CoA+Acetylcholine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_521 \"Choline+Acceptor=Betaine aldehyde+Reduced acceptor\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_521 \"Choline+Acceptor=Betaine aldehyde+Reduced acceptor\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_522 \"Acetylcholine+H2O=Choline+Acetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_522 \"Acetylcholine+H2O=Choline+Acetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_523 \"Acetylcholine+H2O=Choline+Acetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_523 \"Acetylcholine+H2O=Choline+Acetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_524 \"Acetylcholine+H2O=Choline+Acetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_524 \"Acetylcholine+H2O=Choline+Acetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_525 \"Acylcholine+H2O=Choline+Carboxylate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_525 \"Acylcholine+H2O=Choline+Carboxylate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_528 \"Glycerol+NAD+=D-Glyceraldehyde+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_528 \"Glycerol+NAD+=D-Glyceraldehyde+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_529 \"Glycerol+NAD+=D-Glyceraldehyde+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_529 \"Glycerol+NAD+=D-Glyceraldehyde+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_530 \"Glycerol+NAD+=D-Glyceraldehyde+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_530 \"Glycerol+NAD+=D-Glyceraldehyde+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_531 \"Glycerol+NADP+=D-Glyceraldehyde+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_531 \"Glycerol+NADP+=D-Glyceraldehyde+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_532 \"Glycerol+NADP+=D-Glyceraldehyde+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_532 \"Glycerol+NADP+=D-Glyceraldehyde+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_533 \"Glycerol+NADP+=D-Glyceraldehyde+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_533 \"Glycerol+NADP+=D-Glyceraldehyde+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_537 \"Uracil+D-Ribose 5-phosphate=Pseudouridine 5'-phosphate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_537 \"Uracil+D-Ribose 5-phosphate=Pseudouridine 5'-phosphate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_544 \"beta-D-Fructose 1,6-bisphosphate=Glycerone phosphate +(2R)-2-Hydroxy-3-(phosphonooxy)-propanal\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_544 \"beta-D-Fructose 1,6-bisphosphate=Glycerone phosphate +(2R)-2-Hydroxy-3-(phosphonooxy)-propanal\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_551 \"(S)-Malate=Fumarate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_551 \"(S)-Malate=Fumarate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_555 \"L-Leucine+2-Oxoglutarate=4-Methyl-2-oxopentanoate +L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_555 \"L-Leucine+2-Oxoglutarate=4-Methyl-2-oxopentanoate +L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_558 \"D-Galactose+NAD+=Galactitol+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_558 \"D-Galactose+NAD+=Galactitol+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_560 \"D-Galactose+NADP+=Galactitol+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_560 \"D-Galactose+NADP+=Galactitol+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_561 \"Lactose+H2O=D-Glucose+D-Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_561 \"Lactose+H2O=D-Glucose+D-Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_565 \"Melibiose+H2O=D-Galactose+D-Glucose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_565 \"Melibiose+H2O=D-Galactose+D-Glucose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_569 \"Raffinose+H2O=D-Galactose+Sucrose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_569 \"Raffinose+H2O=D-Galactose+Sucrose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_573 \"3-beta-D-Galactosyl-sn-glycerol+H2O=D-Galactose+Glycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_573 \"3-beta-D-Galactosyl-sn-glycerol+H2O=D-Galactose+Glycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_574 \"3-beta-D-Galactosyl-sn-glycerol+H2O=D-Galactose+Glycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_574 \"3-beta-D-Galactosyl-sn-glycerol+H2O=D-Galactose+Glycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_575 \"3-beta-D-Galactosyl-sn-glycerol+H2O=D-Galactose+Glycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_575 \"3-beta-D-Galactosyl-sn-glycerol+H2O=D-Galactose+Glycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_584 \"Isopentenyl diphosphate=Dimethylallyl diphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_584 \"Isopentenyl diphosphate=Dimethylallyl diphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_585 \"IMP+H2O=Inosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_585 \"IMP+H2O=Inosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_587 \"IMP+H2O=Inosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_587 \"IMP+H2O=Inosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_592 \"IMP+NAD++H2O=Xanthosine 5'-phosphate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_592 \"IMP+NAD++H2O=Xanthosine 5'-phosphate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_598 \"IMP+NH3+NADP+=GMP+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_598 \"IMP+NH3+NADP+=GMP+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_604 \"ATP+dADP=ADP+dATP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_604 \"ATP+dADP=ADP+dATP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_605 \"dATP+Pyruvate=dADP+Phosphoenolpyruvate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_605 \"dATP+Pyruvate=dADP+Phosphoenolpyruvate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_606 \"dATP+D-Glucose=dADP+D-Glucose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_606 \"dATP+D-Glucose=dADP+D-Glucose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_608 \"dATP+D-Fructose=dADP+D-Fructose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_608 \"dATP+D-Fructose=dADP+D-Fructose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_612 \"Acetyl-CoA+Putrescine=CoA+N-Acetylputrescine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_612 \"Acetyl-CoA+Putrescine=CoA+N-Acetylputrescine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_613 \"Agmatine+H2O=Putrescine+Urea\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_613 \"Agmatine+H2O=Putrescine+Urea\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_617 \"L-Histidine=Urocanate+NH3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_617 \"L-Histidine=Urocanate+NH3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_618 \"Butanoyl-CoA+NAD+=Crotonoyl-CoA+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_618 \"Butanoyl-CoA+NAD+=Crotonoyl-CoA+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_619 \"Butanoyl-CoA+FAD=FADH2+Crotonoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_619 \"Butanoyl-CoA+FAD=FADH2+Crotonoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_621 \"Acetyl-CoA+Butanoyl-CoA=CoA+3-Oxohexanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_621 \"Acetyl-CoA+Butanoyl-CoA=CoA+3-Oxohexanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_623 \"1L-myo-Inositol 1-phosphate+H2O=myo-Inositol+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_623 \"1L-myo-Inositol 1-phosphate+H2O=myo-Inositol+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_624 \"myo-Inositol 4-phosphate+H2O=myo-Inositol+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_624 \"myo-Inositol 4-phosphate+H2O=myo-Inositol+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_625 \"1D-myo-Inositol 3-phosphate+H2O=myo-Inositol+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_625 \"1D-myo-Inositol 3-phosphate+H2O=myo-Inositol+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_629 \"Reduced ferredoxin+NADP+=Oxidized ferredoxin+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_629 \"Reduced ferredoxin+NADP+=Oxidized ferredoxin+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_632 \"Acetyl-CoA+D-Glucosamine=CoA+N-Acetyl-D-glucosamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_632 \"Acetyl-CoA+D-Glucosamine=CoA+N-Acetyl-D-glucosamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_633 \"UDP-D-galactose+N-Acetyl-D-glucosamine=UDP +N-Acetyllactosamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_633 \"UDP-D-galactose+N-Acetyl-D-glucosamine=UDP +N-Acetyllactosamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_634 \"UDP-D-galactose+N-Acetyl-D-glucosamine=UDP +N-Acetyllactosamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_634 \"UDP-D-galactose+N-Acetyl-D-glucosamine=UDP +N-Acetyllactosamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_635 \"UDP-D-galactose+N-Acetyl-D-glucosamine=UDP +N-Acetyllactosamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_635 \"UDP-D-galactose+N-Acetyl-D-glucosamine=UDP +N-Acetyllactosamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_640 \"3-Methyl-2-oxobutanoic acid+CoA+NAD+=2-Methylpropanoyl-CoA +CO2+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_640 \"3-Methyl-2-oxobutanoic acid+CoA+NAD+=2-Methylpropanoyl-CoA +CO2+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_641 \"L-Valine+2-Oxoglutarate=3-Methyl-2-oxobutanoic acid +L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_641 \"L-Valine+2-Oxoglutarate=3-Methyl-2-oxobutanoic acid +L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_644 \"5,10-Methylenetetrahydrofolate+NAD+ _ST_=_LT_5,10-Methenyltetrahydrofolate+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_644 \"5,10-Methylenetetrahydrofolate+NAD+ _ST_=_LT_5,10-Methenyltetrahydrofolate+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_646 \"5,10-Methylenetetrahydrofolate+NADP+ _ST_=_LT_5,10-Methenyltetrahydrofolate+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_646 \"5,10-Methylenetetrahydrofolate+NADP+ _ST_=_LT_5,10-Methenyltetrahydrofolate+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_648 \"Glycine+Tetrahydrofolate+NAD+=5,10-Methylenetetrahydrofolate+ NH3+CO2+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_648 \"Glycine+Tetrahydrofolate+NAD+=5,10-Methylenetetrahydrofolate+ NH3+CO2+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_649 \"5,10-Methylenetetrahydrofolate+FADH2=5-Methyltetrahydrofolate+ FAD (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_649 \"5,10-Methylenetetrahydrofolate+FADH2=5-Methyltetrahydrofolate+ FAD (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_651 \"GMP+H2O=Guanosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_651 \"GMP+H2O=Guanosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_653 \"GMP+H2O=Guanosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_653 \"GMP+H2O=Guanosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_654 \"GMP+H2O=Guanosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_654 \"GMP+H2O=Guanosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_658 \"ATP+Xanthosine 5'-phosphate+NH3=AMP+Pyrophosphate+GMP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_658 \"ATP+Xanthosine 5'-phosphate+NH3=AMP+Pyrophosphate+GMP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_669 \"L-Proline+NADP+=(S)-1-Pyrroline-5-carboxylate+NADPH+H+ (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_669 \"L-Proline+NADP+=(S)-1-Pyrroline-5-carboxylate+NADPH+H+ (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_671 \"L-Proline+2-Oxoglutarate+Oxygen=trans-4-Hydroxy-L-proline +Succinate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_671 \"L-Proline+2-Oxoglutarate+Oxygen=trans-4-Hydroxy-L-proline +Succinate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_673 \"Glutathione+L-Amino acid=Cys-Gly+(5-L-Glutamyl)-L-amino acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_673 \"Glutathione+L-Amino acid=Cys-Gly+(5-L-Glutamyl)-L-amino acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_675 \"S-Adenosyl-L-methionine+Nicotinamide=S-Adenosyl-L-homocysteine+ 1-Methylnicotinamide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_675 \"S-Adenosyl-L-methionine+Nicotinamide=S-Adenosyl-L-homocysteine+ 1-Methylnicotinamide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_677 \"Palmitoyl-CoA+H2O=CoA+Hexadecanoic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_677 \"Palmitoyl-CoA+H2O=CoA+Hexadecanoic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_678 \"Palmitoyl-CoA+NADP+=trans-Hexadec-2-enoyl-CoA+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_678 \"Palmitoyl-CoA+NADP+=trans-Hexadec-2-enoyl-CoA+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_679 \"Palmitoyl-CoA+FAD=trans-Hexadec-2-enoyl-CoA+FADH2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_679 \"Palmitoyl-CoA+FAD=trans-Hexadec-2-enoyl-CoA+FADH2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_681 \"Palmitoyl-CoA+L-Serine=3-Dehydrosphinganine+CoA+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_681 \"Palmitoyl-CoA+L-Serine=3-Dehydrosphinganine+CoA+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_682 \"L-Serine+L-Homocysteine=Cystathionine+H2O\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_682 \"L-Serine+L-Homocysteine=Cystathionine+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_686 \"Phosphatidylcholine+H2O=Phosphatidate+Choline\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_686 \"Phosphatidylcholine+H2O=Phosphatidate+Choline\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_689 \"Phosphatidylcholine+H2O=Phosphatidate+Choline\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_689 \"Phosphatidylcholine+H2O=Phosphatidate+Choline\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_690 \"Phosphatidylcholine+H2O=1-Acyl-sn-glycero-3-phosphocholine +Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_690 \"Phosphatidylcholine+H2O=1-Acyl-sn-glycero-3-phosphocholine +Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_691 \"Phosphatidylcholine+H2O=1-Acyl-sn-glycero-3-phosphocholine +Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_691 \"Phosphatidylcholine+H2O=1-Acyl-sn-glycero-3-phosphocholine +Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_692 \"Phosphatidylcholine+H2O=1-Acyl-sn-glycero-3-phosphocholine +Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_692 \"Phosphatidylcholine+H2O=1-Acyl-sn-glycero-3-phosphocholine +Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_703 \"Citrate=Isocitrate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_703 \"Citrate=Isocitrate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_704 \"Citrate=Isocitrate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_704 \"Citrate=Isocitrate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_705 \"Citrate=Isocitrate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_705 \"Citrate=Isocitrate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_706 \"Citrate=Isocitrate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_706 \"Citrate=Isocitrate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_707 \"Citrate=cis-Aconitate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_707 \"Citrate=cis-Aconitate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_708 \"Citrate=cis-Aconitate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_708 \"Citrate=cis-Aconitate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_709 \"Citrate=cis-Aconitate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_709 \"Citrate=cis-Aconitate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_710 \"Citrate=cis-Aconitate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_710 \"Citrate=cis-Aconitate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_711 \"ATP+D-Mannose=ADP+D-Mannose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_711 \"ATP+D-Mannose=ADP+D-Mannose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_712 \"ITP+D-Mannose=IDP+D-Mannose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_712 \"ITP+D-Mannose=IDP+D-Mannose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_715 \"Epimelibiose+H2O=D-Mannose+D-Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_715 \"Epimelibiose+H2O=D-Mannose+D-Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_718 \"dATP+D-Mannose=dADP+D-Mannose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_718 \"dATP+D-Mannose=dADP+D-Mannose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_723 \"Glycerol+Fatty acid=Triacylglycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_723 \"Glycerol+Fatty acid=Triacylglycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_729 \"(S)-3-Hydroxy-3-methylglutaryl-CoA=Acetyl-CoA+Acetoacetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_729 \"(S)-3-Hydroxy-3-methylglutaryl-CoA=Acetyl-CoA+Acetoacetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_731 \"(R)-3-Hydroxybutanoate+NAD+=Acetoacetate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_731 \"(R)-3-Hydroxybutanoate+NAD+=Acetoacetate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_732 \"Acetoacetate+Fumarate=4-Fumarylacetoacetate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_732 \"Acetoacetate+Fumarate=4-Fumarylacetoacetate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_734 \"Triacylglycerol+H2O=Diacylglycerol+Carboxylate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_734 \"Triacylglycerol+H2O=Diacylglycerol+Carboxylate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_735 \"Triacylglycerol+H2O=Diacylglycerol+Carboxylate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_735 \"Triacylglycerol+H2O=Diacylglycerol+Carboxylate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_737 \"Phenylpyruvate+Oxygen=2-Hydroxyphenylacetate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_737 \"Phenylpyruvate+Oxygen=2-Hydroxyphenylacetate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_738 \"L-Glutamine+Phenylpyruvate=2-Oxoglutaramate+L-Phenylalanine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_738 \"L-Glutamine+Phenylpyruvate=2-Oxoglutaramate+L-Phenylalanine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_739 \"Phenylpyruvate=enol-Phenylpyruvate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_739 \"Phenylpyruvate=enol-Phenylpyruvate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_745 \"D-Glycerate+NADP+=Hydroxypyruvate+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_745 \"D-Glycerate+NADP+=Hydroxypyruvate+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_746 \"Hydroxypyruvate=2-Hydroxy-3-oxopropanoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_746 \"Hydroxypyruvate=2-Hydroxy-3-oxopropanoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_749 \"Carbamoyl phosphate+L-Ornithine=Orthophosphate+L-Citrulline\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_749 \"Carbamoyl phosphate+L-Ornithine=Orthophosphate+L-Citrulline\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_750 \"5'-Methylthioadenosine+Orthophosphate=Adenine +S-Methyl-5-thio-D-ribose 1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_750 \"5'-Methylthioadenosine+Orthophosphate=Adenine +S-Methyl-5-thio-D-ribose 1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_755 \"Xylitol+NADP+=D-Xylose+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_755 \"Xylitol+NADP+=D-Xylose+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_757 \"Cholesterol+NAD+=Cholesta-5,7-dien-3beta-ol+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_757 \"Cholesterol+NAD+=Cholesta-5,7-dien-3beta-ol+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_758 \"Cholesterol+NAD+=Cholesta-5,7-dien-3beta-ol+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_758 \"Cholesterol+NAD+=Cholesta-5,7-dien-3beta-ol+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_759 \"Cholesterol+NADPH+H++Oxygen=20alpha-Hydroxycholesterol +NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_759 \"Cholesterol+NADPH+H++Oxygen=20alpha-Hydroxycholesterol +NADP++H2O\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_760 \"Cholesterol+NADPH+H++Oxygen=20alpha-Hydroxycholesterol +NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_760 \"Cholesterol+NADPH+H++Oxygen=20alpha-Hydroxycholesterol +NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_762 \"Cholesterol+NADP+=Cholesta-5,7-dien-3beta-ol+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_762 \"Cholesterol+NADP+=Cholesta-5,7-dien-3beta-ol+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_763 \"Cholesterol+NADP+=Cholesta-5,7-dien-3beta-ol+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_763 \"Cholesterol+NADP+=Cholesta-5,7-dien-3beta-ol+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_764 \"Acyl-CoA+Cholesterol=CoA+Cholesterol ester\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_764 \"Acyl-CoA+Cholesterol=CoA+Cholesterol ester\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_765 \"Acyl-CoA+Cholesterol=CoA+Cholesterol ester\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_765 \"Acyl-CoA+Cholesterol=CoA+Cholesterol ester\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_767 \"Cholesterol+Oxygen+NADPH+H+=7alpha-Hydroxycholesterol +NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_767 \"Cholesterol+Oxygen+NADPH+H+=7alpha-Hydroxycholesterol +NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_768 \"Cholesterol+Oxygen+NADPH+H+=7alpha-Hydroxycholesterol +NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_768 \"Cholesterol+Oxygen+NADPH+H+=7alpha-Hydroxycholesterol +NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_775 \"N-Acylsphingosine+H2O=Carboxylate+Sphingosine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_775 \"N-Acylsphingosine+H2O=Carboxylate+Sphingosine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_776 \"N-Acylsphingosine+H2O=Carboxylate+Sphingosine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_776 \"N-Acylsphingosine+H2O=Carboxylate+Sphingosine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_777 \"N-Acylsphingosine+H2O=Carboxylate+Sphingosine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_777 \"N-Acylsphingosine+H2O=Carboxylate+Sphingosine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_778 \"N-Acylsphingosine+H2O=Carboxylate+Sphingosine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_778 \"N-Acylsphingosine+H2O=Carboxylate+Sphingosine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_780 \"N-Acylsphingosine+H2O=Fatty acid+Sphingosine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_780 \"N-Acylsphingosine+H2O=Fatty acid+Sphingosine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_781 \"N-Acylsphingosine+H2O=Fatty acid+Sphingosine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_781 \"N-Acylsphingosine+H2O=Fatty acid+Sphingosine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_783 \"N-Acylsphingosine+H2O=Fatty acid+Sphingosine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_783 \"N-Acylsphingosine+H2O=Fatty acid+Sphingosine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_786 \"UDPglucose+N-Acylsphingosine=UDP+Glucosylceramide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_786 \"UDPglucose+N-Acylsphingosine=UDP+Glucosylceramide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_787 \"UDPglucose+N-Acylsphingosine=UDP+Glucosylceramide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_787 \"UDPglucose+N-Acylsphingosine=UDP+Glucosylceramide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_789 \"Glucosylceramide+H2O=D-Glucose+N-Acylsphingosine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_789 \"Glucosylceramide+H2O=D-Glucose+N-Acylsphingosine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_791 \"Glucosylceramide+H2O=D-Glucose+N-Acylsphingosine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_791 \"Glucosylceramide+H2O=D-Glucose+N-Acylsphingosine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_797 \"2,3-Bisphospho-D-glycerate+H2O=3-Phospho-D-glycerate +Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_797 \"2,3-Bisphospho-D-glycerate+H2O=3-Phospho-D-glycerate +Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_799 \"beta-D-Glucose+NAD+=D-Glucono-1,5-lactone+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_799 \"beta-D-Glucose+NAD+=D-Glucono-1,5-lactone+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_800 \"beta-D-Glucose+NADP+=D-Glucono-1,5-lactone+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_800 \"beta-D-Glucose+NADP+=D-Glucono-1,5-lactone+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_803 \"ATP+dAMP=ADP+dADP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_803 \"ATP+dAMP=ADP+dADP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_804 \"ATP+dAMP=ADP+dADP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_804 \"ATP+dAMP=ADP+dADP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_805 \"ATP+dAMP=ADP+dADP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_805 \"ATP+dAMP=ADP+dADP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_813 \"N,N-Dimethylglycine+Acceptor+H2O=Sarcosine+Formaldehyde +Reduced acceptor\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_813 \"N,N-Dimethylglycine+Acceptor+H2O=Sarcosine+Formaldehyde +Reduced acceptor\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_814 \"N,N-Dimethylglycine+Acceptor+H2O=Sarcosine+Formaldehyde +Reduced acceptor\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_814 \"N,N-Dimethylglycine+Acceptor+H2O=Sarcosine+Formaldehyde +Reduced acceptor\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_816 \"dTMP+H2O=Thymidine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_816 \"dTMP+H2O=Thymidine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_820 \"dTMP+H2O=Thymidine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_820 \"dTMP+H2O=Thymidine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_821 \"Thymidine+Orthophosphate=Thymine+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_821 \"Thymidine+Orthophosphate=Thymine+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_822 \"Thymidine+Orthophosphate=Thymine+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_822 \"Thymidine+Orthophosphate=Thymine+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_825 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+2 Oxygen=Prostaglandin G2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_825 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+2 Oxygen=Prostaglandin G2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_826 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen=15(S)-HPETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_826 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen=15(S)-HPETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_830 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen=15(S)-HPETE\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_830 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen=15(S)-HPETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_831 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen=5(S)-HPETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_831 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen=5(S)-HPETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_833 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen=12(S)-HPETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_833 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen=12(S)-HPETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_837 \"ATP+beta-D-Glucose=ADP+beta-D-Glucose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_837 \"ATP+beta-D-Glucose=ADP+beta-D-Glucose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_839 \"alpha-D-Glucose=beta-D-Glucose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_839 \"alpha-D-Glucose=beta-D-Glucose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_846 \"4-Aminobutanoate+2-Oxoglutarate=Succinate semialdehyde +L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_846 \"4-Aminobutanoate+2-Oxoglutarate=Succinate semialdehyde +L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_847 \"4-Methyl-2-oxopentanoate+CoA+NAD+=3-Methylbutanoyl-CoA+CO2+ NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_847 \"4-Methyl-2-oxopentanoate+CoA+NAD+=3-Methylbutanoyl-CoA+CO2+ NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_851 \"5,10-Methenyltetrahydrofolate+H2O=10-Formyltetrahydrofolate +H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_851 \"5,10-Methenyltetrahydrofolate+H2O=10-Formyltetrahydrofolate +H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_852 \"Dimethylallyl diphosphate+Isopentenyl diphosphate _ST_=_LT_Pyrophosphate+Geranyl diphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_852 \"Dimethylallyl diphosphate+Isopentenyl diphosphate _ST_=_LT_Pyrophosphate+Geranyl diphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_854 \"dCMP+H2O=dUMP+NH3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_854 \"dCMP+H2O=dUMP+NH3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_855 \"dCMP+H2O=Deoxycytidine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_855 \"dCMP+H2O=Deoxycytidine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_856 \"dCMP+H2O=Deoxycytidine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_856 \"dCMP+H2O=Deoxycytidine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_857 \"dCMP+H2O=Deoxycytidine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_857 \"dCMP+H2O=Deoxycytidine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_862 \"ATP+Deoxycytidine=ADP+dCMP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_862 \"ATP+Deoxycytidine=ADP+dCMP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_864 \"dCDP+H2O=dCMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_864 \"dCDP+H2O=dCMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_866 \"Lactose+H2O=alpha-D-Glucose+D-Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_866 \"Lactose+H2O=alpha-D-Glucose+D-Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_878 \"Dihydrolipoamide+NAD+=Lipoamide+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_878 \"Dihydrolipoamide+NAD+=Lipoamide+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_879 \"Pyruvate+Lipoamide=S-Acetyldihydrolipoamide+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_879 \"Pyruvate+Lipoamide=S-Acetyldihydrolipoamide+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_881 \"3-Methyl-2-oxobutanoic acid+Lipoamide _ST_=_LT_S-(2-Methylpropanoyl)-dihydrolipoamide+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_881 \"3-Methyl-2-oxobutanoic acid+Lipoamide _ST_=_LT_S-(2-Methylpropanoyl)-dihydrolipoamide+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_883 \"4-Methyl-2-oxopentanoate+Lipoamide _ST_=_LT_S-(3-Methylbutanoyl)-dihydrolipoamide+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_883 \"4-Methyl-2-oxopentanoate+Lipoamide _ST_=_LT_S-(3-Methylbutanoyl)-dihydrolipoamide+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_897 \"D-Glyceraldehyde+NAD++H2O=D-Glycerate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_897 \"D-Glyceraldehyde+NAD++H2O=D-Glycerate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_898 \"D-Glyceraldehyde+NAD++H2O=D-Glycerate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_898 \"D-Glyceraldehyde+NAD++H2O=D-Glycerate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_899 \"D-Glyceraldehyde+NAD++H2O=D-Glycerate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_899 \"D-Glyceraldehyde+NAD++H2O=D-Glycerate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_900 \"D-Glyceraldehyde+NAD++H2O=D-Glycerate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_900 \"D-Glyceraldehyde+NAD++H2O=D-Glycerate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_902 \"L-Arabitol+NAD+=L-Arabinose+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_902 \"L-Arabitol+NAD+=L-Arabinose+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_903 \"L-Arabitol+NADP+=L-Arabinose+NADPH (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_903 \"L-Arabitol+NADP+=L-Arabinose+NADPH (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_904 \"L-Arabitol+NADP+=L-Arabinose+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_904 \"L-Arabitol+NADP+=L-Arabinose+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_906 \"Hypoxanthine+NAD++H2O=Xanthine+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_906 \"Hypoxanthine+NAD++H2O=Xanthine+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_907 \"Hypoxanthine+Oxygen+H2O=Xanthine+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_907 \"Hypoxanthine+Oxygen+H2O=Xanthine+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_908 \"Inosine+H2O=Hypoxanthine+D-Ribose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_908 \"Inosine+H2O=Hypoxanthine+D-Ribose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_910 \"(3S)-3-Hydroxyacyl-CoA+NAD+=3-Oxoacyl-CoA+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_910 \"(3S)-3-Hydroxyacyl-CoA+NAD+=3-Oxoacyl-CoA+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_912 \"ATP+alpha-D-Glucose=ADP+alpha-D-Glucose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_912 \"ATP+alpha-D-Glucose=ADP+alpha-D-Glucose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_913 \"D-Sorbitol+NADP+=alpha-D-Glucose+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_913 \"D-Sorbitol+NADP+=alpha-D-Glucose+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_918 \"Dextrin+H2O=alpha-D-Glucose+Dextrin\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_918 \"Dextrin+H2O=alpha-D-Glucose+Dextrin\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_924 \"CTP+Phosphatidate=Pyrophosphate+CDPdiacylglycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_924 \"CTP+Phosphatidate=Pyrophosphate+CDPdiacylglycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_925 \"CTP+Phosphatidate=Pyrophosphate+CDPdiacylglycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_925 \"CTP+Phosphatidate=Pyrophosphate+CDPdiacylglycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_935 \"Tetrahydrobiopterin+L-Tryptophan+Oxygen _ST_=_LT_5-Hydroxy-L-tryptophan+Dihydrobiopterin+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_935 \"Tetrahydrobiopterin+L-Tryptophan+Oxygen _ST_=_LT_5-Hydroxy-L-tryptophan+Dihydrobiopterin+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_940 \"Sedoheptulose 1,7-bisphosphate=Glycerone phosphate+D-Erythrose4-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_940 \"Sedoheptulose 1,7-bisphosphate=Glycerone phosphate+D-Erythrose4-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_941 \"Sedoheptulose 1,7-bisphosphate=Glycerone phosphate+D-Erythrose4-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_941 \"Sedoheptulose 1,7-bisphosphate=Glycerone phosphate+D-Erythrose4-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_944 \"5alpha-Androstane-3,17-dione+Acceptor=Androst-4-ene-3,17-dione+ Reduced acceptor (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_944 \"5alpha-Androstane-3,17-dione+Acceptor=Androst-4-ene-3,17-dione+ Reduced acceptor (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_946 \"5beta-Androstane-3,17-dione+Acceptor=Androst-4-ene-3,17-dione+ Reduced acceptor (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_946 \"5beta-Androstane-3,17-dione+Acceptor=Androst-4-ene-3,17-dione+ Reduced acceptor (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_957 \"ATP+dGDP=ADP+dGTP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_957 \"ATP+dGDP=ADP+dGTP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_958 \"dGTP+Pyruvate=dGDP+Phosphoenolpyruvate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_958 \"dGTP+Pyruvate=dGDP+Phosphoenolpyruvate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_959 \"ATP+Propanoyl-CoA+HCO3-=ADP+Orthophosphate +(S)-2-Methyl-3-oxopropanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_959 \"ATP+Propanoyl-CoA+HCO3-=ADP+Orthophosphate +(S)-2-Methyl-3-oxopropanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_962 \"(S)-Dihydroorotate+Oxygen=Orotate+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_962 \"(S)-Dihydroorotate+Oxygen=Orotate+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_963 \"(S)-Dihydroorotate+Oxygen=Orotate+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_963 \"(S)-Dihydroorotate+Oxygen=Orotate+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_965 \"Uridine+Orthophosphate=Uracil+alpha-D-Ribose 1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_965 \"Uridine+Orthophosphate=Uracil+alpha-D-Ribose 1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_967 \"Uridine+Orthophosphate=Uracil+alpha-D-Ribose 1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_967 \"Uridine+Orthophosphate=Uracil+alpha-D-Ribose 1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_978 \"CTP+Choline phosphate=Pyrophosphate+CDPcholine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_978 \"CTP+Choline phosphate=Pyrophosphate+CDPcholine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_983 \"Isocitrate+NADP+=Oxalosuccinate+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_983 \"Isocitrate+NADP+=Oxalosuccinate+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_984 \"Isocitrate+NADP+=Oxalosuccinate+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_984 \"Isocitrate+NADP+=Oxalosuccinate+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_985 \"Isocitrate+NADP+=Oxalosuccinate+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_985 \"Isocitrate+NADP+=Oxalosuccinate+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_986 \"Isocitrate=cis-Aconitate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_986 \"Isocitrate=cis-Aconitate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_987 \"Isocitrate=cis-Aconitate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_987 \"Isocitrate=cis-Aconitate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_988 \"Isocitrate=cis-Aconitate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_988 \"Isocitrate=cis-Aconitate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_989 \"Isocitrate=cis-Aconitate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_989 \"Isocitrate=cis-Aconitate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_990 \"L-Arabitol+NAD+=L-Xylulose+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_990 \"L-Arabitol+NAD+=L-Xylulose+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_991 \"Xylitol+NADP+=L-Xylulose+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_991 \"Xylitol+NADP+=L-Xylulose+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_995 \"Sphingosine+ATP=Sphingosine 1-phosphate+ADP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_995 \"Sphingosine+ATP=Sphingosine 1-phosphate+ADP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_996 \"Thiosulfate+Hydrogen cyanide=Sulfite+Thiocyanate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_996 \"Thiosulfate+Hydrogen cyanide=Sulfite+Thiocyanate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_997 \"2-Oxoadipate+CoA+NAD+=Glutaryl-CoA+CO2+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_997 \"2-Oxoadipate+CoA+NAD+=Glutaryl-CoA+CO2+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_998 \"L-2-Aminoadipate+2-Oxoglutarate=2-Oxoadipate+L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_998 \"L-2-Aminoadipate+2-Oxoglutarate=2-Oxoadipate+L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_999 \"2-Oxoadipate+Lipoamide=S-Glutaryldihydrolipoamide+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_999 \"2-Oxoadipate+Lipoamide=S-Glutaryldihydrolipoamide+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1002 \"L-Kynurenine+2-Oxoglutarate _ST_=_LT_4-(2-Aminophenyl)-2,4-dioxobutanoate+L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1002 \"L-Kynurenine+2-Oxoglutarate _ST_=_LT_4-(2-Aminophenyl)-2,4-dioxobutanoate+L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1010 \"ITP+D-Glucosamine=IDP+D-Glucosamine 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1010 \"ITP+D-Glucosamine=IDP+D-Glucosamine 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1012 \"dATP+D-Glucosamine=dADP+D-Glucosamine 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1012 \"dATP+D-Glucosamine=dADP+D-Glucosamine 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1015 \"dGMP+H2O=Deoxyguanosine+Orthophosphate\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1015 \"dGMP+H2O=Deoxyguanosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1016 \"dGMP+H2O=Deoxyguanosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1016 \"dGMP+H2O=Deoxyguanosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1018 \"dGMP+H2O=Deoxyguanosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1018 \"dGMP+H2O=Deoxyguanosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1020 \"Deoxyguanosine+Orthophosphate=Guanine+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1020 \"Deoxyguanosine+Orthophosphate=Guanine+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1021 \"(S)-3-Hydroxybutanoyl-CoA+NAD+=Acetoacetyl-CoA+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1021 \"(S)-3-Hydroxybutanoyl-CoA+NAD+=Acetoacetyl-CoA+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1024 \"4-Aminobutanal+NADP++H2O=4-Aminobutanoate+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1024 \"4-Aminobutanal+NADP++H2O=4-Aminobutanoate+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1025 \"4-Aminobutanal+NADP++H2O=4-Aminobutanoate+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1025 \"4-Aminobutanal+NADP++H2O=4-Aminobutanoate+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1028 \"L-Arginine+4-Aminobutanoate=L-Ornithine+4-Guanidinobutanoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1028 \"L-Arginine+4-Aminobutanoate=L-Ornithine+4-Guanidinobutanoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1029 \"L-Arginine+4-Aminobutanoate=L-Ornithine+4-Guanidinobutanoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1029 \"L-Arginine+4-Aminobutanoate=L-Ornithine+4-Guanidinobutanoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1031 \"Geranyl diphosphate+Isopentenyl diphosphate=Pyrophosphate +trans,trans-Farnesyl diphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1031 \"Geranyl diphosphate+Isopentenyl diphosphate=Pyrophosphate +trans,trans-Farnesyl diphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1032 \"Thioredoxin+NADP+=Oxidized thioredoxin+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1032 \"Thioredoxin+NADP+=Oxidized thioredoxin+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1033 \"Thioredoxin+NADP+=Oxidized thioredoxin+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1033 \"Thioredoxin+NADP+=Oxidized thioredoxin+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1034 \"Thioredoxin+NADP+=Oxidized thioredoxin+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1034 \"Thioredoxin+NADP+=Oxidized thioredoxin+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1035 \"dADP+Oxidized thioredoxin+H2O=Thioredoxin+ADP (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1035 \"dADP+Oxidized thioredoxin+H2O=Thioredoxin+ADP (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1036 \"dADP+Oxidized thioredoxin+H2O=Thioredoxin+ADP (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1036 \"dADP+Oxidized thioredoxin+H2O=Thioredoxin+ADP (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1037 \"dADP+Oxidized thioredoxin+H2O=Thioredoxin+ADP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1037 \"dADP+Oxidized thioredoxin+H2O=Thioredoxin+ADP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1038 \"dUDP+Oxidized thioredoxin+H2O=Thioredoxin+UDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1038 \"dUDP+Oxidized thioredoxin+H2O=Thioredoxin+UDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1039 \"dUDP+Oxidized thioredoxin+H2O=Thioredoxin+UDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1039 \"dUDP+Oxidized thioredoxin+H2O=Thioredoxin+UDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1040 \"dGDP+Oxidized thioredoxin+H2O=GDP+Thioredoxin (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1040 \"dGDP+Oxidized thioredoxin+H2O=GDP+Thioredoxin (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1041 \"dGDP+Oxidized thioredoxin+H2O=GDP+Thioredoxin\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1041 \"dGDP+Oxidized thioredoxin+H2O=GDP+Thioredoxin\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1043 \"dCDP+Oxidized thioredoxin+H2O=Thioredoxin+CDP (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1043 \"dCDP+Oxidized thioredoxin+H2O=Thioredoxin+CDP (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1044 \"dCDP+Oxidized thioredoxin+H2O=Thioredoxin+CDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1044 \"dCDP+Oxidized thioredoxin+H2O=Thioredoxin+CDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1048 \"D-Glucono-1,5-lactone 6-phosphate+H2O=6-Phospho-D-gluconate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1048 \"D-Glucono-1,5-lactone 6-phosphate+H2O=6-Phospho-D-gluconate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1050 \"3-Hydroxy-2-methylpropanoate+NAD+=2-Methyl-3-oxopropanoate +NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1050 \"3-Hydroxy-2-methylpropanoate+NAD+=2-Methyl-3-oxopropanoate +NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1051 \"(R)-3-Amino-2-methylpropanoate+2-Oxoglutarate _ST_=_LT_2-Methyl-3-oxopropanoate+L-Glutamate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1051 \"(R)-3-Amino-2-methylpropanoate+2-Oxoglutarate _ST_=_LT_2-Methyl-3-oxopropanoate+L-Glutamate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1052 \"(R)-3-Amino-2-methylpropanoate+Pyruvate _ST_=_LT_2-Methyl-3-oxopropanoate+L-Alanine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1052 \"(R)-3-Amino-2-methylpropanoate+Pyruvate _ST_=_LT_2-Methyl-3-oxopropanoate+L-Alanine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1053 \"Phosphatidylethanolamine+H2O=Ethanolamine+Phosphatidate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1053 \"Phosphatidylethanolamine+H2O=Ethanolamine+Phosphatidate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1056 \"Phosphatidylethanolamine+H2O=Ethanolamine+Phosphatidate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1056 \"Phosphatidylethanolamine+H2O=Ethanolamine+Phosphatidate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1057 \"Phosphatidylethanolamine+H2O _ST_=_LT_1-Acyl-sn-glycero-3-phosphoethanolamine+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1057 \"Phosphatidylethanolamine+H2O _ST_=_LT_1-Acyl-sn-glycero-3-phosphoethanolamine+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1058 \"Phosphatidylethanolamine+H2O _ST_=_LT_1-Acyl-sn-glycero-3-phosphoethanolamine+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1058 \"Phosphatidylethanolamine+H2O _ST_=_LT_1-Acyl-sn-glycero-3-phosphoethanolamine+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1059 \"Phosphatidylethanolamine+H2O _ST_=_LT_1-Acyl-sn-glycero-3-phosphoethanolamine+Fatty acid\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1059 \"Phosphatidylethanolamine+H2O _ST_=_LT_1-Acyl-sn-glycero-3-phosphoethanolamine+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1061 \"Phosphatidylserine=Phosphatidylethanolamine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1061 \"Phosphatidylserine=Phosphatidylethanolamine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1066 \"Acetyl-CoA+D-Glucosamine 6-phosphate=CoA +N-Acetyl-D-glucosamine 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1066 \"Acetyl-CoA+D-Glucosamine 6-phosphate=CoA +N-Acetyl-D-glucosamine 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1068 \"trans,trans-Farnesyl diphosphate+Isopentenyl diphosphate _ST_=_LT_Pyrophosphate+Geranylgeranyl diphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1068 \"trans,trans-Farnesyl diphosphate+Isopentenyl diphosphate _ST_=_LT_Pyrophosphate+Geranylgeranyl diphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1080 \"(S)-3-Hydroxy-3-methylglutaryl-CoA=3-Methylglutaconyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1080 \"(S)-3-Hydroxy-3-methylglutaryl-CoA=3-Methylglutaconyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1082 \"dAMP+H2O=Deoxyadenosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1082 \"dAMP+H2O=Deoxyadenosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1083 \"dAMP+H2O=Deoxyadenosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1083 \"dAMP+H2O=Deoxyadenosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1084 \"dAMP+H2O=Deoxyadenosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1084 \"dAMP+H2O=Deoxyadenosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1089 \"dTDP+H2O=dTMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1089 \"dTDP+H2O=dTMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1090 \"dTDP+H2O=dTMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1090 \"dTDP+H2O=dTMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1091 \"dTDP+H2O=dTMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1091 \"dTDP+H2O=dTMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1093 \"dTDP+H2O=dTMP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1093 \"dTDP+H2O=dTMP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1096 \"ATP+dTDP=ADP+dTTP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1096 \"ATP+dTDP=ADP+dTTP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1099 \"dTTP+H2O=dTDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1099 \"dTTP+H2O=dTDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1100 \"dTTP+H2O=dTDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1100 \"dTTP+H2O=dTDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1101 \"dTTP+H2O=dTDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1101 \"dTTP+H2O=dTDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1102 \"dTTP+H2O=dTDP+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1102 \"dTTP+H2O=dTDP+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1111 \"dUMP+5,10-Methylenetetrahydrofolate=Dihydrofolate+dTMP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1111 \"dUMP+5,10-Methylenetetrahydrofolate=Dihydrofolate+dTMP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1113 \"Xanthine+NAD++H2O=Urate+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1113 \"Xanthine+NAD++H2O=Urate+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1115 \"Xanthine+H2O+Oxygen=Urate+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1115 \"Xanthine+H2O+Oxygen=Urate+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1116 \"Starch+H2O=Dextrin+Starch\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1116 \"Starch+H2O=Dextrin+Starch\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1118 \"Starch+H2O=Amylose+alpha-D-Glucose (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1118 \"Starch+H2O=Amylose+alpha-D-Glucose (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1119 \"Amylose=Starch (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1119 \"Amylose=Starch (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1120 \"Starch+Orthophosphate=Amylose+D-Glucose 1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1120 \"Starch+Orthophosphate=Amylose+D-Glucose 1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1130 \"1H-Imidazole-4-ethanamine+Oxygen+H2O _ST_=_LT_Imidazole-4-acetaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1130 \"1H-Imidazole-4-ethanamine+Oxygen+H2O _ST_=_LT_Imidazole-4-acetaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1132 \"Ubiquinone+Succinate=Ubiquinol+Fumarate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1132 \"Ubiquinone+Succinate=Ubiquinol+Fumarate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1133 \"Ubiquinone+Succinate=Ubiquinol+Fumarate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1133 \"Ubiquinone+Succinate=Ubiquinol+Fumarate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1135 \"Ubiquinol=Ubiquinone+2 H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1135 \"Ubiquinol=Ubiquinone+2 H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1139 \"Tryptamine+H2O+Oxygen=Indole-3-acetaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1139 \"Tryptamine+H2O+Oxygen=Indole-3-acetaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1148 \"L-Isoleucine+2-Oxoglutarate=3-Methyl-2-oxopentanoate +L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1148 \"L-Isoleucine+2-Oxoglutarate=3-Methyl-2-oxopentanoate +L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1149 \"L-Isoleucine+2-Oxoglutarate=(S)-3-Methyl-2-oxopentanoic acid +L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1149 \"L-Isoleucine+2-Oxoglutarate=(S)-3-Methyl-2-oxopentanoic acid +L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1153 \"L-Pipecolate+Oxygen=2,3,4,5-Tetrahydropyridine-2-carboxylate +H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1153 \"L-Pipecolate+Oxygen=2,3,4,5-Tetrahydropyridine-2-carboxylate +H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1154 \"L-Pipecolate+FAD=FADH2 +2,3,4,5-Tetrahydropyridine-2-carboxylate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1154 \"L-Pipecolate+FAD=FADH2 +2,3,4,5-Tetrahydropyridine-2-carboxylate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1171 \"Pregnenolone+NADP+=Progesterone+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1171 \"Pregnenolone+NADP+=Progesterone+NADPH+H+\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1177 \"Stearoyl-CoA+Reduced acceptor+Oxygen=Oleoyl-CoA+Acceptor +2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1177 \"Stearoyl-CoA+Reduced acceptor+Oxygen=Oleoyl-CoA+Acceptor +2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1178 \"Stearoyl-CoA+Reduced acceptor+Oxygen=Oleoyl-CoA+Acceptor +2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1178 \"Stearoyl-CoA+Reduced acceptor+Oxygen=Oleoyl-CoA+Acceptor +2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1186 \"ATP+1,2-Diacyl-sn-glycerol=ADP+Phosphatidate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1186 \"ATP+1,2-Diacyl-sn-glycerol=ADP+Phosphatidate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1187 \"ATP+1,2-Diacyl-sn-glycerol=ADP+Phosphatidate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1187 \"ATP+1,2-Diacyl-sn-glycerol=ADP+Phosphatidate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1188 \"ATP+1,2-Diacyl-sn-glycerol=ADP+Phosphatidate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1188 \"ATP+1,2-Diacyl-sn-glycerol=ADP+Phosphatidate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1189 \"Phosphatidate+CoA=1-Acyl-sn-glycerol 3-phosphate+Acyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1189 \"Phosphatidate+CoA=1-Acyl-sn-glycerol 3-phosphate+Acyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1190 \"Phosphatidate+CoA=1-Acyl-sn-glycerol 3-phosphate+Acyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1190 \"Phosphatidate+CoA=1-Acyl-sn-glycerol 3-phosphate+Acyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1191 \"Phosphatidate+CoA=1-Acyl-sn-glycerol 3-phosphate+Acyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1191 \"Phosphatidate+CoA=1-Acyl-sn-glycerol 3-phosphate+Acyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1197 \"Triacylglycerol+H2O=1,2-Diacyl-sn-glycerol+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1197 \"Triacylglycerol+H2O=1,2-Diacyl-sn-glycerol+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1198 \"Triacylglycerol+H2O=1,2-Diacyl-sn-glycerol+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1198 \"Triacylglycerol+H2O=1,2-Diacyl-sn-glycerol+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1200 \"Triacylglycerol+CoA=1,2-Diacyl-sn-glycerol+Acyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1200 \"Triacylglycerol+CoA=1,2-Diacyl-sn-glycerol+Acyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1202 \"Prostaglandin H2=Prostaglandin E2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1202 \"Prostaglandin H2=Prostaglandin E2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1203 \"Prostaglandin H2=Prostaglandin E2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1203 \"Prostaglandin H2=Prostaglandin E2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1204 \"Prostaglandin H2 _ST_=_LT_(5Z,13E)-(15S)-9alpha,15-Dihydroxy-11-oxoprosta-5,13-dienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1204 \"Prostaglandin H2 _ST_=_LT_(5Z,13E)-(15S)-9alpha,15-Dihydroxy-11-oxoprosta-5,13-dienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1205 \"Prostaglandin H2 _ST_=_LT_(5Z,13E)-(15S)-9alpha,15-Dihydroxy-11-oxoprosta-5,13-dienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1205 \"Prostaglandin H2 _ST_=_LT_(5Z,13E)-(15S)-9alpha,15-Dihydroxy-11-oxoprosta-5,13-dienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1206 \"Prostaglandin H2 _ST_=_LT_(5Z,13E)-(15S)-9alpha,15-Dihydroxy-11-oxoprosta-5,13-dienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1206 \"Prostaglandin H2 _ST_=_LT_(5Z,13E)-(15S)-9alpha,15-Dihydroxy-11-oxoprosta-5,13-dienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1207 \"Prostaglandin H2=Prostaglandin I2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1207 \"Prostaglandin H2=Prostaglandin I2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1208 \"Prostaglandin H2=Prostaglandin I2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1208 \"Prostaglandin H2=Prostaglandin I2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1209 \"Prostaglandin H2=Thromboxane A2 (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1209 \"Prostaglandin H2=Thromboxane A2 (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1210 \"5,6-Dihydrouracil+H2O=3-Ureidopropionate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1210 \"5,6-Dihydrouracil+H2O=3-Ureidopropionate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1211 \"2-Amino-3-oxoadipate=5-Aminolevulinate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1211 \"2-Amino-3-oxoadipate=5-Aminolevulinate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1219 \"5-Formyltetrahydrofolate=5,10-Methenyltetrahydrofolate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1219 \"5-Formyltetrahydrofolate=5,10-Methenyltetrahydrofolate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1220 \"ATP+5-Formyltetrahydrofolate=ADP+Orthophosphate +5,10-Methenyltetrahydrofolate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1220 \"ATP+5-Formyltetrahydrofolate=ADP+Orthophosphate +5,10-Methenyltetrahydrofolate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1223 \"5-Formiminotetrahydrofolate=5,10-Methenyltetrahydrofolate+NH3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1223 \"5-Formiminotetrahydrofolate=5,10-Methenyltetrahydrofolate+NH3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1225 \"N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O=L-Glutamate +L-2-Aminoadipate 6-semialdehyde+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1225 \"N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O=L-Glutamate +L-2-Aminoadipate 6-semialdehyde+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1226 \"Nicotinamide D-ribonucleotide+H2O=N-Ribosylnicotinamide +Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1226 \"Nicotinamide D-ribonucleotide+H2O=N-Ribosylnicotinamide +Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1231 \"ATP+dCDP=ADP+dCTP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1231 \"ATP+dCDP=ADP+dCTP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1235 \"ATP+dUDP=ADP+dUTP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1235 \"ATP+dUDP=ADP+dUTP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1242 \"Estradiol-17beta+NAD+=Estrone+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1242 \"Estradiol-17beta+NAD+=Estrone+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1244 \"Estradiol-17beta+NADP+=Estrone+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1244 \"Estradiol-17beta+NADP+=Estrone+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1246 \"Estrone+UDPglucuronate=Estrone glucuronide+UDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1246 \"Estrone+UDPglucuronate=Estrone glucuronide+UDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1251 \"dCTP+Cytidine=dCDP+CMP\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1251 \"dCTP+Cytidine=dCDP+CMP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1255 \"Tyramine+H2O+Oxygen=4-Hydroxyphenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1255 \"Tyramine+H2O+Oxygen=4-Hydroxyphenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1256 \"Tyramine+H2O+Oxygen=4-Hydroxyphenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1256 \"Tyramine+H2O+Oxygen=4-Hydroxyphenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1257 \"Tyramine+H2O+Oxygen=4-Hydroxyphenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1257 \"Tyramine+H2O+Oxygen=4-Hydroxyphenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1261 \"Tyramine+Oxygen+NADH+H+=4-(2-Aminoethyl)-1,2-benzenediol +NAD++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1261 \"Tyramine+Oxygen+NADH+H+=4-(2-Aminoethyl)-1,2-benzenediol +NAD++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1266 \"Bilirubin+NADP+=Biliverdin+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1266 \"Bilirubin+NADP+=Biliverdin+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1283 \"Sphinganine 1-phosphate=Ethanolamine phosphate+Hexadecanal\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1283 \"Sphinganine 1-phosphate=Ethanolamine phosphate+Hexadecanal\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1284 \"Sphinganine 1-phosphate=Ethanolamine phosphate+Hexadecanal\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1284 \"Sphinganine 1-phosphate=Ethanolamine phosphate+Hexadecanal\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1285 \"3-Sulfino-L-alanine=Hypotaurine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1285 \"3-Sulfino-L-alanine=Hypotaurine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1286 \"3-Sulfino-L-alanine=Hypotaurine+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1286 \"3-Sulfino-L-alanine=Hypotaurine+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1292 \"Androsterone+NAD+=5alpha-Androstane-3,17-dione+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1292 \"Androsterone+NAD+=5alpha-Androstane-3,17-dione+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1293 \"Androsterone+NADP+=5alpha-Androstane-3,17-dione+NADPH (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1293 \"Androsterone+NADP+=5alpha-Androstane-3,17-dione+NADPH (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1296 \"Androsterone+UDPglucuronate=Androsterone glucuronide+UDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1296 \"Androsterone+UDPglucuronate=Androsterone glucuronide+UDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1300 \"Deoxyuridine+Orthophosphate=Uracil+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1300 \"Deoxyuridine+Orthophosphate=Uracil+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1306 \"ATP+Pyridoxamine=ADP+Pyridoxamine phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1306 \"ATP+Pyridoxamine=ADP+Pyridoxamine phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1307 \"17beta-Hydroxyandrostan-3-one+Acceptor=Testosterone+Reducedacceptor (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1307 \"17beta-Hydroxyandrostan-3-one+Acceptor=Testosterone+Reducedacceptor (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1308 \"17beta-Hydroxyandrostan-3-one+Acceptor=Testosterone+Reducedacceptor (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1308 \"17beta-Hydroxyandrostan-3-one+Acceptor=Testosterone+Reducedacceptor (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1318 \"Testosterone+UDPglucuronate=Testosterone glucuronide+UDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1318 \"Testosterone+UDPglucuronate=Testosterone glucuronide+UDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1322 \"3-(4-Hydroxyphenyl)pyruvate+Oxygen=Homogentisate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1322 \"3-(4-Hydroxyphenyl)pyruvate+Oxygen=Homogentisate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1324 \"Aminoacetone+H2O+Oxygen=Methylglyoxal+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1324 \"Aminoacetone+H2O+Oxygen=Methylglyoxal+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1326 \"(R)-S-Lactoylglutathione=Glutathione+Methylglyoxal\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1326 \"(R)-S-Lactoylglutathione=Glutathione+Methylglyoxal\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1328 \"Lactaldehyde+NAD+=Methylglyoxal+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1328 \"Lactaldehyde+NAD+=Methylglyoxal+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1329 \"L-Noradrenaline+H2O+Oxygen=3,4-Dihydroxymandelaldehyde+NH3+ H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1329 \"L-Noradrenaline+H2O+Oxygen=3,4-Dihydroxymandelaldehyde+NH3+ H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1339 \"Sphingomyelin+H2O=N-Acylsphingosine+Choline phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1339 \"Sphingomyelin+H2O=N-Acylsphingosine+Choline phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1341 \"Sphingomyelin+H2O=N-Acylsphingosine+Choline phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1341 \"Sphingomyelin+H2O=N-Acylsphingosine+Choline phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1343 \"4-Aminobutanal+NAD++H2O=4-Aminobutanoate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1343 \"4-Aminobutanal+NAD++H2O=4-Aminobutanoate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1344 \"4-Aminobutanal+NAD++H2O=4-Aminobutanoate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1344 \"4-Aminobutanal+NAD++H2O=4-Aminobutanoate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1347 \"Deoxyadenosine+Orthophosphate=Adenine+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1347 \"Deoxyadenosine+Orthophosphate=Adenine+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1348 \"Betaine aldehyde+FAD+H2O=FADH2+Betaine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1348 \"Betaine aldehyde+FAD+H2O=FADH2+Betaine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1349 \"D-Fructose 1-phosphate=Glycerone phosphate+D-Glyceraldehyde\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1349 \"D-Fructose 1-phosphate=Glycerone phosphate+D-Glyceraldehyde\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1351 \"Acetyl-CoA+Dihydrolipoamide=CoA+S-Acetyldihydrolipoamide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1351 \"Acetyl-CoA+Dihydrolipoamide=CoA+S-Acetyldihydrolipoamide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1353 \"Succinyl-CoA+Dihydrolipoamide=CoA+S-Succinyldihydrolipoamide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1353 \"Succinyl-CoA+Dihydrolipoamide=CoA+S-Succinyldihydrolipoamide\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1354 \"Glutaryl-CoA+Dihydrolipoamide=CoA+S-Glutaryldihydrolipoamide (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1354 \"Glutaryl-CoA+Dihydrolipoamide=CoA+S-Glutaryldihydrolipoamide (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1355 \"Glutaryl-CoA+Dihydrolipoamide=CoA+S-Glutaryldihydrolipoamide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1355 \"Glutaryl-CoA+Dihydrolipoamide=CoA+S-Glutaryldihydrolipoamide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1357 \"Propane-1,2-diol+NADP+=Lactaldehyde+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1357 \"Propane-1,2-diol+NADP+=Lactaldehyde+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1358 \"Prostaglandin E2+NAD+ _ST_=_LT_(5Z,13E)-11alpha-Hydroxy-9,15-dioxoprost-13-enoate+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1358 \"Prostaglandin E2+NAD+ _ST_=_LT_(5Z,13E)-11alpha-Hydroxy-9,15-dioxoprost-13-enoate+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1359 \"Prostaglandin F2alpha+NADP+=Prostaglandin E2+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1359 \"Prostaglandin F2alpha+NADP+=Prostaglandin E2+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1360 \"Prostaglandin F2alpha+NADP+=Prostaglandin E2+NADPH+H+ (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1360 \"Prostaglandin F2alpha+NADP+=Prostaglandin E2+NADPH+H+ (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1361 \"ATP+Protein tyrosine=ADP+Protein tyrosine phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1361 \"ATP+Protein tyrosine=ADP+Protein tyrosine phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1362 \"ATP+Protein tyrosine=ADP+Protein tyrosine phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1362 \"ATP+Protein tyrosine=ADP+Protein tyrosine phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1363 \"ATP+Protein tyrosine=ADP+Protein tyrosine phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1363 \"ATP+Protein tyrosine=ADP+Protein tyrosine phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1364 \"Protein tyrosine phosphate+H2O=Protein tyrosine +Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1364 \"Protein tyrosine phosphate+H2O=Protein tyrosine +Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1365 \"Protein tyrosine phosphate+H2O=Protein tyrosine +Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1365 \"Protein tyrosine phosphate+H2O=Protein tyrosine +Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1367 \"Protein tyrosine phosphate+H2O=Protein tyrosine +Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1367 \"Protein tyrosine phosphate+H2O=Protein tyrosine +Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1379 \"Phenethylamine+Oxygen+H2O=Phenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1379 \"Phenethylamine+Oxygen+H2O=Phenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1380 \"Phenethylamine+Oxygen+H2O=Phenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1380 \"Phenethylamine+Oxygen+H2O=Phenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1383 \"3-Sulfino-L-alanine+2-Oxoglutarate=3-Sulfinylpyruvate +Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1383 \"3-Sulfino-L-alanine+2-Oxoglutarate=3-Sulfinylpyruvate +Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1384 \"3-Sulfino-L-alanine+2-Oxoglutarate=3-Sulfinylpyruvate +Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1384 \"3-Sulfino-L-alanine+2-Oxoglutarate=3-Sulfinylpyruvate +Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1388 \"UDP-N-acetyl-D-glucosamine+1-Phosphatidyl-D-myo-inositol=UDP +N-Acetyl-D-glucosaminylphosphatidylinositol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1388 \"UDP-N-acetyl-D-glucosamine+1-Phosphatidyl-D-myo-inositol=UDP +N-Acetyl-D-glucosaminylphosphatidylinositol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1391 \"2-Methylpropanoyl-CoA+Oxygen=2-Methylprop-2-enoyl-CoA+2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1391 \"2-Methylpropanoyl-CoA+Oxygen=2-Methylprop-2-enoyl-CoA+2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1392 \"2-Methylpropanoyl-CoA+FAD=2-Methylprop-2-enoyl-CoA+FADH2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1392 \"2-Methylpropanoyl-CoA+FAD=2-Methylprop-2-enoyl-CoA+FADH2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1394 \"3-Hydroxy-L-kynurenine+H2O=3-Hydroxyanthranilate+L-Alanine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1394 \"3-Hydroxy-L-kynurenine+H2O=3-Hydroxyanthranilate+L-Alanine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1396 \"Indole-3-acetaldehyde+NAD++H2O=Indole-3-acetate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1396 \"Indole-3-acetaldehyde+NAD++H2O=Indole-3-acetate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1399 \"1,2-Diacyl-sn-glycerol+H2O=Acylglycerol+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1399 \"1,2-Diacyl-sn-glycerol+H2O=Acylglycerol+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1400 \"1,2-Diacyl-sn-glycerol+H2O=Acylglycerol+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1400 \"1,2-Diacyl-sn-glycerol+H2O=Acylglycerol+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1402 \"1,2-Diacyl-sn-glycerol+H2O=2-Acylglycerol+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1402 \"1,2-Diacyl-sn-glycerol+H2O=2-Acylglycerol+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1403 \"1,2-Diacyl-sn-glycerol+H2O=2-Acylglycerol+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1403 \"1,2-Diacyl-sn-glycerol+H2O=2-Acylglycerol+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1412 \"Xanthosine 5'-phosphate+H2O=Xanthosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1412 \"Xanthosine 5'-phosphate+H2O=Xanthosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1414 \"Xanthosine 5'-phosphate+H2O=Xanthosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1414 \"Xanthosine 5'-phosphate+H2O=Xanthosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1418 \"Cholesterol+NADPH+Oxygen=22beta-Hydroxycholesterol+NADP+ +H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1418 \"Cholesterol+NADPH+Oxygen=22beta-Hydroxycholesterol+NADP+ +H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1422 \"D-Fructose 2,6-bisphosphate+H2O=beta-D-Fructose 6-phosphate +Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1422 \"D-Fructose 2,6-bisphosphate+H2O=beta-D-Fructose 6-phosphate +Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1423 \"ATP+beta-D-Fructose 6-phosphate=ADP+D-Fructose2,6-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1423 \"ATP+beta-D-Fructose 6-phosphate=ADP+D-Fructose2,6-bisphosphate\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1426 \"alpha-D-Glucose 6-phosphate=beta-D-Glucose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1426 \"alpha-D-Glucose 6-phosphate=beta-D-Glucose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1429 \"alpha-D-Glucose 6-phosphate=beta-D-Fructose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1429 \"alpha-D-Glucose 6-phosphate=beta-D-Fructose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1431 \"1-Acyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_sn-glycero-3-Phosphocholine+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1431 \"1-Acyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_sn-glycero-3-Phosphocholine+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1432 \"1-Acyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_sn-glycero-3-Phosphocholine+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1432 \"1-Acyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_sn-glycero-3-Phosphocholine+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1433 \"1-Acyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_sn-glycero-3-Phosphocholine+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1433 \"1-Acyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_sn-glycero-3-Phosphocholine+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1435 \"2-Acyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_sn-glycero-3-Phosphocholine+Fatty acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1435 \"2-Acyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_sn-glycero-3-Phosphocholine+Fatty acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1438 \"Deoxyinosine+Orthophosphate=Hypoxanthine+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1438 \"Deoxyinosine+Orthophosphate=Hypoxanthine+2-Deoxy-D-ribose1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1441 \"2,3-Dehydroacyl-CoA+1-Acyl-sn-glycerol 3-phosphate=CoA +Phosphatidate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1441 \"2,3-Dehydroacyl-CoA+1-Acyl-sn-glycerol 3-phosphate=CoA +Phosphatidate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1443 \"(R)-2-Methyl-3-oxopropanoyl-CoA=(S)-2-Methyl-3-oxopropanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1443 \"(R)-2-Methyl-3-oxopropanoyl-CoA=(S)-2-Methyl-3-oxopropanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1447 \"N-Acylneuraminate 9-phosphate+Orthophosphate _ST_=_LT_N-Acyl-D-mannosamine 6-phosphate+Phosphoenolpyruvate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1447 \"N-Acylneuraminate 9-phosphate+Orthophosphate _ST_=_LT_N-Acyl-D-mannosamine 6-phosphate+Phosphoenolpyruvate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1450 \"ATP+3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate+CoA=AMP+ Pyrophosphate+Choloyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1450 \"ATP+3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate+CoA=AMP+ Pyrophosphate+Choloyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1455 \"Betaine+L-Homocysteine=N,N-Dimethylglycine+L-Methionine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1455 \"Betaine+L-Homocysteine=N,N-Dimethylglycine+L-Methionine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1457 \"Cortisol+NAD+=Cortisone+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1457 \"Cortisol+NAD+=Cortisone+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1459 \"Cortisol+NADP+=Cortisone+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1459 \"Cortisol+NADP+=Cortisone+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1477 \"ATP+D-Sorbitol=ADP+Sorbitol 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1477 \"ATP+D-Sorbitol=ADP+Sorbitol 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1479 \"ITP+D-Sorbitol=IDP+Sorbitol 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1479 \"ITP+D-Sorbitol=IDP+Sorbitol 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1481 \"dATP+D-Sorbitol=dADP+Sorbitol 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1481 \"dATP+D-Sorbitol=dADP+Sorbitol 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1483 \"S-Adenosylmethioninamine+Spermidine=5'-Methylthioadenosine +Spermine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1483 \"S-Adenosylmethioninamine+Spermidine=5'-Methylthioadenosine +Spermine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1485 \"Presqualene diphosphate+NADPH+H+=Pyrophosphate+Squalene +NADP+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1485 \"Presqualene diphosphate+NADPH+H+=Pyrophosphate+Squalene +NADP+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1486 \"Squalene+Oxygen+NADPH+H+=(S)-2,3-Epoxysqualene+NADP+ +H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1486 \"Squalene+Oxygen+NADPH+H+=(S)-2,3-Epoxysqualene+NADP+ +H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1495 \"3-(2-Aminoethyl)-1H-indol-5-ol+H2O+Oxygen _ST_=_LT_5-Hydroxyindoleacetaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1495 \"3-(2-Aminoethyl)-1H-indol-5-ol+H2O+Oxygen _ST_=_LT_5-Hydroxyindoleacetaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1499 \"4-Imidazolone-5-propanoate=Urocanate+H2O (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1499 \"4-Imidazolone-5-propanoate=Urocanate+H2O (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1502 \"L-Adrenaline+H2O+Oxygen=3,4-Dihydroxymandelaldehyde +Methylamine+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1502 \"L-Adrenaline+H2O+Oxygen=3,4-Dihydroxymandelaldehyde +Methylamine+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1503 \"L-Adrenaline+H2O+Oxygen=3,4-Dihydroxymandelaldehyde +Methylamine+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1503 \"L-Adrenaline+H2O+Oxygen=3,4-Dihydroxymandelaldehyde +Methylamine+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1508 \"Melibiitol+H2O=D-Sorbitol+D-Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1508 \"Melibiitol+H2O=D-Sorbitol+D-Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1522 \"ATP+Sphinganine=ADP+Sphinganine 1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1522 \"ATP+Sphinganine=ADP+Sphinganine 1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1524 \"Sphinganine+NADP+=3-Dehydrosphinganine+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1524 \"Sphinganine+NADP+=3-Dehydrosphinganine+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1525 \"Sphinganine+NADP+=3-Dehydrosphinganine+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1525 \"Sphinganine+NADP+=3-Dehydrosphinganine+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1527 \"Deamino-NAD++H2O=AMP+Nicotinate D-ribonucleotide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1527 \"Deamino-NAD++H2O=AMP+Nicotinate D-ribonucleotide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1528 \"Deamino-NAD++H2O=AMP+Nicotinate D-ribonucleotide\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1528 \"Deamino-NAD++H2O=AMP+Nicotinate D-ribonucleotide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1530 \"ATP+Nicotinate D-ribonucleotide=Pyrophosphate+Deamino-NAD+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1530 \"ATP+Nicotinate D-ribonucleotide=Pyrophosphate+Deamino-NAD+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1533 \"ATP+Pantothenate=ADP+D-4'-Phosphopantothenate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1533 \"ATP+Pantothenate=ADP+D-4'-Phosphopantothenate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1536 \"4-Nitrophenyl phosphate+H2O=4-Nitrophenol+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1536 \"4-Nitrophenyl phosphate+H2O=4-Nitrophenol+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1539 \"(S)-3-Hydroxybutanoyl-CoA=Crotonoyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1539 \"(S)-3-Hydroxybutanoyl-CoA=Crotonoyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1540 \"(R)-3-Hydroxybutanoyl-CoA=Crotonoyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1540 \"(R)-3-Hydroxybutanoyl-CoA=Crotonoyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1541 \"ATP+Pantetheine 4'-phosphate=Pyrophosphate+Dephospho-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1541 \"ATP+Pantetheine 4'-phosphate=Pyrophosphate+Dephospho-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1542 \"Dephospho-CoA+H2O=Pantetheine 4'-phosphate+AMP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1542 \"Dephospho-CoA+H2O=Pantetheine 4'-phosphate+AMP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1546 \"3-Hydroxypropionyl-CoA=Propenoyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1546 \"3-Hydroxypropionyl-CoA=Propenoyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1549 \"Leukotriene A4+H2O=Leukotriene B4\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1549 \"Leukotriene A4+H2O=Leukotriene B4\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1550 \"5(S)-HPETE=Leukotriene A4+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1550 \"5(S)-HPETE=Leukotriene A4+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1551 \"5(S)-HPETE=Leukotriene A4+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1551 \"5(S)-HPETE=Leukotriene A4+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1556 \"Estradiol-17beta+UDPglucuronate=Estradiol-17beta 3-glucuronide+ UDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1556 \"Estradiol-17beta+UDPglucuronate=Estradiol-17beta 3-glucuronide+ UDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1563 \"Mercaptopyruvate+Sulfite=H2S2O3+Pyruvate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1563 \"Mercaptopyruvate+Sulfite=H2S2O3+Pyruvate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1566 \"1,3-Diaminopropane+Oxygen+H2O=beta-Aminopropion aldehyde +NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1566 \"1,3-Diaminopropane+Oxygen+H2O=beta-Aminopropion aldehyde +NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1569 \"3-Hydroxypropionyl-CoA+Orthophosphate+ADP _ST_=_LT_3-Hydroxypropanoate+CoA+ATP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1569 \"3-Hydroxypropionyl-CoA+Orthophosphate+ADP _ST_=_LT_3-Hydroxypropanoate+CoA+ATP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1570 \"3-Hydroxypropanoate+CoA=3-Hydroxypropionyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1570 \"3-Hydroxypropanoate+CoA=3-Hydroxypropionyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1573 \"alpha-D-Fucoside+H2O=Alcohol+6-Deoxy-L-galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1573 \"alpha-D-Fucoside+H2O=Alcohol+6-Deoxy-L-galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1575 \"Hydroxymethylbilane=Uroporphyrinogen III+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1575 \"Hydroxymethylbilane=Uroporphyrinogen III+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1578 \"2-Methylbutanoyl-CoA+Oxygen=2-Methylbut-2-enoyl-CoA+2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1578 \"2-Methylbutanoyl-CoA+Oxygen=2-Methylbut-2-enoyl-CoA+2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1579 \"3-Methyl-2-oxopentanoate+CoA+NAD+=2-Methylbutanoyl-CoA+CO2+ NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1579 \"3-Methyl-2-oxopentanoate+CoA+NAD+=2-Methylbutanoyl-CoA+CO2+ NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1580 \"2-Methylbutanoyl-CoA+FAD=2-Methylbut-2-enoyl-CoA+FADH2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1580 \"2-Methylbutanoyl-CoA+FAD=2-Methylbut-2-enoyl-CoA+FADH2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1585 \"4-Maleylacetoacetate=4-Fumarylacetoacetate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1585 \"4-Maleylacetoacetate=4-Fumarylacetoacetate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1589 \"(S)-2,3-Epoxysqualene=Lanosterol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1589 \"(S)-2,3-Epoxysqualene=Lanosterol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1591 \"Coproporphyrinogen III+Oxygen=Protoporphyrinogen IX+2 CO2 +2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1591 \"Coproporphyrinogen III+Oxygen=Protoporphyrinogen IX+2 CO2 +2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1593 \"2 Protoporphyrinogen IX+3 Oxygen=2 Protoporphyrin+6 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1593 \"2 Protoporphyrinogen IX+3 Oxygen=2 Protoporphyrin+6 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1605 \"2-(alpha-Hydroxyethyl)thiamine diphosphate+Lipoamide _ST_=_LT_S-Acetyldihydrolipoamide+Thiamin diphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1605 \"2-(alpha-Hydroxyethyl)thiamine diphosphate+Lipoamide _ST_=_LT_S-Acetyldihydrolipoamide+Thiamin diphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1607 \"4-Trimethylammoniobutanal+NAD++H2O _ST_=_LT_4-Trimethylammoniobutanoate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1607 \"4-Trimethylammoniobutanal+NAD++H2O _ST_=_LT_4-Trimethylammoniobutanoate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1608 \"4-Trimethylammoniobutanal+NAD++H2O _ST_=_LT_4-Trimethylammoniobutanoate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1608 \"4-Trimethylammoniobutanal+NAD++H2O _ST_=_LT_4-Trimethylammoniobutanoate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1609 \"4-Trimethylammoniobutanal+Glycine _ST_=_LT_3-Hydroxy-N6,N6,N6-trimethyl-L-lysine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1609 \"4-Trimethylammoniobutanal+Glycine _ST_=_LT_3-Hydroxy-N6,N6,N6-trimethyl-L-lysine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1610 \"4-Trimethylammoniobutanal+Glycine _ST_=_LT_3-Hydroxy-N6,N6,N6-trimethyl-L-lysine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1610 \"4-Trimethylammoniobutanal+Glycine _ST_=_LT_3-Hydroxy-N6,N6,N6-trimethyl-L-lysine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1615 \"trans-4-Hydroxy-L-proline+FAD _ST_=_LT_L-1-Pyrroline-3-hydroxy-5-carboxylate+FADH2\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1615 \"trans-4-Hydroxy-L-proline+FAD _ST_=_LT_L-1-Pyrroline-3-hydroxy-5-carboxylate+FADH2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1624 \"Cholesta-5,7-dien-3beta-ol=Vitamin D3 (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1624 \"Cholesta-5,7-dien-3beta-ol=Vitamin D3 (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1626 \"L-Glutamate 5-semialdehyde+Orthophosphate+NADP+=L-Glutamyl5-phosphate+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1626 \"L-Glutamate 5-semialdehyde+Orthophosphate+NADP+=L-Glutamyl5-phosphate+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1628 \"3-Carboxy-1-hydroxypropyl-ThPP+Lipoamide _ST_=_LT_S-Succinyldihydrolipoamide+Thiamin diphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1628 \"3-Carboxy-1-hydroxypropyl-ThPP+Lipoamide _ST_=_LT_S-Succinyldihydrolipoamide+Thiamin diphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1631 \"beta-D-Glucose 6-phosphate=beta-D-Fructose 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1631 \"beta-D-Glucose 6-phosphate=beta-D-Fructose 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1641 \"17alpha-Hydroxypregnenolone+NADP+=17alpha-Hydroxyprogesterone+ NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1641 \"17alpha-Hydroxypregnenolone+NADP+=17alpha-Hydroxyprogesterone+ NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1643 \"17alpha-Hydroxyprogesterone+NADPH+H++Oxygen _ST_=_LT_21-Deoxycortisol+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1643 \"17alpha-Hydroxyprogesterone+NADPH+H++Oxygen _ST_=_LT_21-Deoxycortisol+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1645 \"3-(4-Hydroxyphenyl)pyruvate _ST_=_LT_2-Hydroxy-3-(4-hydroxyphenyl)propenoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1645 \"3-(4-Hydroxyphenyl)pyruvate _ST_=_LT_2-Hydroxy-3-(4-hydroxyphenyl)propenoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1648 \"Nicotinate D-ribonucleotide+H2O=Nicotinate D-ribonucleoside +Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1648 \"Nicotinate D-ribonucleotide+H2O=Nicotinate D-ribonucleoside +Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1650 \"Nicotinate D-ribonucleotide+H2O=Nicotinate D-ribonucleoside +Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1650 \"Nicotinate D-ribonucleotide+H2O=Nicotinate D-ribonucleoside +Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1653 \"beta-D-Galactosyl-1,4-beta-D-glucosylceramide+H2O _ST_=_LT_Glucosylceramide+beta-D-Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1653 \"beta-D-Galactosyl-1,4-beta-D-glucosylceramide+H2O _ST_=_LT_Glucosylceramide+beta-D-Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1655 \"beta-D-Galactosyl-1,4-beta-D-glucosylceramide+H2O _ST_=_LT_Glucosylceramide+beta-D-Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1655 \"beta-D-Galactosyl-1,4-beta-D-glucosylceramide+H2O _ST_=_LT_Glucosylceramide+beta-D-Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1656 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 4-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1656 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 4-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1657 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 4-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1657 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 4-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1659 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 4-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1659 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 4-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1660 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 3-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1660 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 3-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1661 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 3-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1661 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 3-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1662 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 3-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1662 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 3-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1663 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 3-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1663 \"ATP+1-Phosphatidyl-D-myo-inositol=ADP +1-Phosphatidyl-1D-myo-inositol 3-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1665 \"(S)-Methylmalonate semialdehyde+CoA+NAD+ _ST_=_LT_(R)-2-Methyl-3-oxopropanoyl-CoA+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1665 \"(S)-Methylmalonate semialdehyde+CoA+NAD+ _ST_=_LT_(R)-2-Methyl-3-oxopropanoyl-CoA+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1666 \"1D-myo-Inositol 1,4-bisphosphate+H2O=myo-Inositol 4-phosphate+ Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1666 \"1D-myo-Inositol 1,4-bisphosphate+H2O=myo-Inositol 4-phosphate+ Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1669 \"3beta-Hydroxyandrost-5-en-17-one 3-sulfate+H2O _ST_=_LT_3beta-Hydroxyandrost-5-en-17-one+Sulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1669 \"3beta-Hydroxyandrost-5-en-17-one 3-sulfate+H2O _ST_=_LT_3beta-Hydroxyandrost-5-en-17-one+Sulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1670 \"3beta-Hydroxyandrost-5-en-17-one 3-sulfate+H2O _ST_=_LT_3beta-Hydroxyandrost-5-en-17-one+Sulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1670 \"3beta-Hydroxyandrost-5-en-17-one 3-sulfate+H2O _ST_=_LT_3beta-Hydroxyandrost-5-en-17-one+Sulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1671 \"3beta-Hydroxyandrost-5-en-17-one 3-sulfate+H2O _ST_=_LT_3beta-Hydroxyandrost-5-en-17-one+Sulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1671 \"3beta-Hydroxyandrost-5-en-17-one 3-sulfate+H2O _ST_=_LT_3beta-Hydroxyandrost-5-en-17-one+Sulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1672 \"3beta-Hydroxyandrost-5-en-17-one 3-sulfate+H2O _ST_=_LT_3beta-Hydroxyandrost-5-en-17-one+Sulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1672 \"3beta-Hydroxyandrost-5-en-17-one 3-sulfate+H2O _ST_=_LT_3beta-Hydroxyandrost-5-en-17-one+Sulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1675 \"1-Acyl-sn-glycero-3-phosphoethanolamine+H2O=Fatty acid +sn-glycero-3-Phosphoethanolamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1675 \"1-Acyl-sn-glycero-3-phosphoethanolamine+H2O=Fatty acid +sn-glycero-3-Phosphoethanolamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1676 \"1-Acyl-sn-glycero-3-phosphoethanolamine+H2O=Fatty acid +sn-glycero-3-Phosphoethanolamine\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1676 \"1-Acyl-sn-glycero-3-phosphoethanolamine+H2O=Fatty acid +sn-glycero-3-Phosphoethanolamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1677 \"1-Acyl-sn-glycero-3-phosphoethanolamine+H2O=Fatty acid +sn-glycero-3-Phosphoethanolamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1677 \"1-Acyl-sn-glycero-3-phosphoethanolamine+H2O=Fatty acid +sn-glycero-3-Phosphoethanolamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1679 \"2-Acyl-sn-glycero-3-phosphoethanolamine+H2O=Fatty acid +sn-glycero-3-Phosphoethanolamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1679 \"2-Acyl-sn-glycero-3-phosphoethanolamine+H2O=Fatty acid +sn-glycero-3-Phosphoethanolamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1684 \"Glycine+Lipoylprotein=S-Aminomethyldihydrolipoylprotein+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1684 \"Glycine+Lipoylprotein=S-Aminomethyldihydrolipoylprotein+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1687 \"ATP+1D-myo-Inositol 1,3,4-trisphosphate=ADP+1D-myo-Inositol1,3,4,6-tetrakisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1687 \"ATP+1D-myo-Inositol 1,3,4-trisphosphate=ADP+1D-myo-Inositol1,3,4,6-tetrakisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1691 \"ATP+D-myo-Inositol 1,4,5-trisphosphate=ADP+1D-myo-Inositol1,3,4,5-tetrakisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1691 \"ATP+D-myo-Inositol 1,4,5-trisphosphate=ADP+1D-myo-Inositol1,3,4,5-tetrakisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1692 \"1D-myo-Inositol 1,3,4,5-tetrakisphosphate+H2O=D-myo-Inositol1,4,5-trisphosphate+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1692 \"1D-myo-Inositol 1,3,4,5-tetrakisphosphate+H2O=D-myo-Inositol1,4,5-trisphosphate+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1693 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+H2O _ST_=_LT_D-myo-Inositol 1,4,5-trisphosphate+1,2-Diacyl-sn-glycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1693 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+H2O _ST_=_LT_D-myo-Inositol 1,4,5-trisphosphate+1,2-Diacyl-sn-glycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1694 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+H2O _ST_=_LT_D-myo-Inositol 1,4,5-trisphosphate+1,2-Diacyl-sn-glycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1694 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+H2O _ST_=_LT_D-myo-Inositol 1,4,5-trisphosphate+1,2-Diacyl-sn-glycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1695 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+H2O _ST_=_LT_D-myo-Inositol 1,4,5-trisphosphate+1,2-Diacyl-sn-glycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1695 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+H2O _ST_=_LT_D-myo-Inositol 1,4,5-trisphosphate+1,2-Diacyl-sn-glycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1696 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+H2O _ST_=_LT_D-myo-Inositol 1,4,5-trisphosphate+1,2-Diacyl-sn-glycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1696 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+H2O _ST_=_LT_D-myo-Inositol 1,4,5-trisphosphate+1,2-Diacyl-sn-glycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1697 \"ATP+[Pyruvate dehydrogenase (lipoamide)]=ADP+[Pyruvatedehydrogenase (lipoamide)] phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1697 \"ATP+[Pyruvate dehydrogenase (lipoamide)]=ADP+[Pyruvatedehydrogenase (lipoamide)] phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1699 \"[Pyruvate dehydrogenase (lipoamide)] phosphate+H2O=[Pyruvatedehydrogenase (lipoamide)]+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1699 \"[Pyruvate dehydrogenase (lipoamide)] phosphate+H2O=[Pyruvatedehydrogenase (lipoamide)]+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1701 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1701 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1702 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1702 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1704 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1704 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1707 \"G00143+H2O=G00144+Acetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1707 \"G00143+H2O=G00144+Acetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1709 \"UDP-D-galactose+Lactosylceramide=UDP+Globotriaosylceramide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1709 \"UDP-D-galactose+Lactosylceramide=UDP+Globotriaosylceramide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1711 \"CMP-N-acetylneuraminate +beta-D-Galactosyl-1,4-beta-D-glucosylceramide=CMP+GM3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1711 \"CMP-N-acetylneuraminate +beta-D-Galactosyl-1,4-beta-D-glucosylceramide=CMP+GM3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1739 \"Calcidiol+Oxygen+NADPH=Calcitriol+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1739 \"Calcidiol+Oxygen+NADPH=Calcitriol+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1740 \"3-beta-D-Galactosyl-sn-glycerol+H2O=Galactose+Glycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1740 \"3-beta-D-Galactosyl-sn-glycerol+H2O=Galactose+Glycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1742 \"3-beta-D-Galactosyl-sn-glycerol+H2O=Galactose+Glycerol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1742 \"3-beta-D-Galactosyl-sn-glycerol+H2O=Galactose+Glycerol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1745 \"Linoleate+Oxygen _ST_=_LT_(9Z,11E)-(13S)-13-Hydroperoxyoctadeca-9,11-dienoic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1745 \"Linoleate+Oxygen _ST_=_LT_(9Z,11E)-(13S)-13-Hydroperoxyoctadeca-9,11-dienoic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1750 \"ATP+Phytanate+CoA=AMP+Pyrophosphate+Phytanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1750 \"ATP+Phytanate+CoA=AMP+Pyrophosphate+Phytanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1751 \"ATP+Phytanate+CoA=AMP+Pyrophosphate+Phytanoyl-CoA (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1751 \"ATP+Phytanate+CoA=AMP+Pyrophosphate+Phytanoyl-CoA (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1753 \"ATP+Protamine=ADP+O-Phosphoprotamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1753 \"ATP+Protamine=ADP+O-Phosphoprotamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1759 \"Stachyose+H2O=Raffinose+D-Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1759 \"Stachyose+H2O=Raffinose+D-Galactose\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1806 \"Lanosterol+Oxygen+NADPH+H+=14-Demethyllanosterol+Formate+ NADP+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1806 \"Lanosterol+Oxygen+NADPH+H+=14-Demethyllanosterol+Formate+ NADP+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1810 \"Chlordecone alcohol+NADP+=Chlordecone+NADPH (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1810 \"Chlordecone alcohol+NADP+=Chlordecone+NADPH (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1811 \"Choloyl-CoA+Glycine=CoA+Glycocholate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1811 \"Choloyl-CoA+Glycine=CoA+Glycocholate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1812 \"Choloyl-CoA+Glycine=CoA+Glycocholate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1812 \"Choloyl-CoA+Glycine=CoA+Glycocholate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1813 \"Propanoyl-CoA+Choloyl-CoA=CoA +3alpha,7alpha,12alpha-Trihydroxy-5beta-24-oxocholestanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1813 \"Propanoyl-CoA+Choloyl-CoA=CoA +3alpha,7alpha,12alpha-Trihydroxy-5beta-24-oxocholestanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1814 \"Choloyl-CoA+Taurine=CoA+Taurocholate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1814 \"Choloyl-CoA+Taurine=CoA+Taurocholate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1816 \"O-(4-Hydroxy-3,5-diidophenyl)-3,5-diiodo-L-tyrosine+Reducedacceptor=Triiodothyronine+Iodide+Acceptor+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1816 \"O-(4-Hydroxy-3,5-diidophenyl)-3,5-diiodo-L-tyrosine+Reducedacceptor=Triiodothyronine+Iodide+Acceptor+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1819 \"Octanoyl-CoA+NADP+=trans-Oct-2-enoyl-CoA+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1819 \"Octanoyl-CoA+NADP+=trans-Oct-2-enoyl-CoA+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1820 \"Octanoyl-CoA+FAD=trans-Oct-2-enoyl-CoA+FADH2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1820 \"Octanoyl-CoA+FAD=trans-Oct-2-enoyl-CoA+FADH2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1821 \"Octanoyl-CoA+Acetyl-CoA=CoA+3-Oxodecanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1821 \"Octanoyl-CoA+Acetyl-CoA=CoA+3-Oxodecanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1830 \"Linoleoyl-CoA+Reduced acceptor+Oxygen=gamma-Linolenoyl-CoA +Acceptor+2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1830 \"Linoleoyl-CoA+Reduced acceptor+Oxygen=gamma-Linolenoyl-CoA +Acceptor+2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1831 \"Linoleoyl-CoA+Reduced acceptor+Oxygen=gamma-Linolenoyl-CoA +Acceptor+2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1831 \"Linoleoyl-CoA+Reduced acceptor+Oxygen=gamma-Linolenoyl-CoA +Acceptor+2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1833 \"Dihydrolipoylprotein+NADP+=Lipoylprotein+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1833 \"Dihydrolipoylprotein+NADP+=Lipoylprotein+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1837 \"Corticosterone+NAD+=11-Dehydrocorticosterone+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1837 \"Corticosterone+NAD+=11-Dehydrocorticosterone+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1839 \"Corticosterone+NADP+=11-Dehydrocorticosterone+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1839 \"Corticosterone+NADP+=11-Dehydrocorticosterone+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1844 \"Lauroyl-CoA+NADP+=2-trans-Dodecenoyl-CoA+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1844 \"Lauroyl-CoA+NADP+=2-trans-Dodecenoyl-CoA+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1845 \"Lauroyl-CoA+FAD=2-trans-Dodecenoyl-CoA+FADH2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1845 \"Lauroyl-CoA+FAD=2-trans-Dodecenoyl-CoA+FADH2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1846 \"Lauroyl-CoA+Acetyl-CoA=CoA+3-Oxotetradecanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1846 \"Lauroyl-CoA+Acetyl-CoA=CoA+3-Oxotetradecanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1850 \"Leukotriene B4+NADPH+H++Oxygen=20-OH-Leukotriene B4 +NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1850 \"Leukotriene B4+NADPH+H++Oxygen=20-OH-Leukotriene B4 +NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1851 \"Leukotriene C4+H2O=Leukotriene D4+Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1851 \"Leukotriene C4+H2O=Leukotriene D4+Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1852 \"(S)-Methylmalonate semialdehyde+NAD++H2O=Methylmalonate +NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1852 \"(S)-Methylmalonate semialdehyde+NAD++H2O=Methylmalonate +NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1853 \"(S)-Methylmalonate semialdehyde+NAD++H2O=Methylmalonate +NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1853 \"(S)-Methylmalonate semialdehyde+NAD++H2O=Methylmalonate +NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1873 \"20alpha,22beta-Dihydroxycholesterol+Ferredoxin+Oxygen _ST_=_LT_4-Methylpentanal+Pregnenolone+Ferredoxin\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1873 \"20alpha,22beta-Dihydroxycholesterol+Ferredoxin+Oxygen _ST_=_LT_4-Methylpentanal+Pregnenolone+Ferredoxin\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1883 \"ATP+Chenodeoxycholate+CoA=AMP+Pyrophosphate +Chenodeoxycholoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1883 \"ATP+Chenodeoxycholate+CoA=AMP+Pyrophosphate +Chenodeoxycholoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1884 \"Estrone 3-sulfate+H2O=Sulfate+Estrone\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1884 \"Estrone 3-sulfate+H2O=Sulfate+Estrone\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1885 \"Estrone 3-sulfate+H2O=Sulfate+Estrone\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1885 \"Estrone 3-sulfate+H2O=Sulfate+Estrone\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1886 \"Estrone 3-sulfate+H2O=Sulfate+Estrone\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1886 \"Estrone 3-sulfate+H2O=Sulfate+Estrone\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1887 \"Estrone 3-sulfate+H2O=Sulfate+Estrone\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1887 \"Estrone 3-sulfate+H2O=Sulfate+Estrone\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1889 \"Protein glutamine+Alkylamine=Protein N5-alkylglutamine+NH3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1889 \"Protein glutamine+Alkylamine=Protein N5-alkylglutamine+NH3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1893 \"Tetradecanoyl-CoA+NADP+=trans-Tetradec-2-enoyl-CoA+NADPH +H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1893 \"Tetradecanoyl-CoA+NADP+=trans-Tetradec-2-enoyl-CoA+NADPH +H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1894 \"Tetradecanoyl-CoA+FAD=trans-Tetradec-2-enoyl-CoA+FADH2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1894 \"Tetradecanoyl-CoA+FAD=trans-Tetradec-2-enoyl-CoA+FADH2\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1895 \"Tetradecanoyl-CoA+Acetyl-CoA=CoA+3-Oxopalmitoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1895 \"Tetradecanoyl-CoA+Acetyl-CoA=CoA+3-Oxopalmitoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1901 \"3'-Phosphoadenylyl sulfate+Galactosylceramide=Adenosine3',5'-bisphosphate+Sulfatide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1901 \"3'-Phosphoadenylyl sulfate+Galactosylceramide=Adenosine3',5'-bisphosphate+Sulfatide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1902 \"3'-Phosphoadenylyl sulfate+Galactosylceramide=Adenosine3',5'-bisphosphate+Sulfatide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1902 \"3'-Phosphoadenylyl sulfate+Galactosylceramide=Adenosine3',5'-bisphosphate+Sulfatide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1907 \"Digalactosylceramide+H2O=Galactosylceramide+Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1907 \"Digalactosylceramide+H2O=Galactosylceramide+Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1908 \"Digalactosylceramide+H2O=Galactosylceramide+Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1908 \"Digalactosylceramide+H2O=Galactosylceramide+Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1909 \"Digalactosylceramide+H2O=Galactosylceramide+Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1909 \"Digalactosylceramide+H2O=Galactosylceramide+Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1910 \"N-Acetylputrescine+H2O+Oxygen=N4-Acetylaminobutanal+NH3 +H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1910 \"N-Acetylputrescine+H2O+Oxygen=N4-Acetylaminobutanal+NH3 +H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1911 \"N-Acetylputrescine+H2O+Oxygen=N4-Acetylaminobutanal+NH3 +H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1911 \"N-Acetylputrescine+H2O+Oxygen=N4-Acetylaminobutanal+NH3 +H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1913 \"N-Methylputrescine+Oxygen=1-Methylpyrrolinium+H2O2+NH3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1913 \"N-Methylputrescine+Oxygen=1-Methylpyrrolinium+H2O2+NH3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1920 \"Imidazole-4-acetaldehyde+NAD++H2O=Imidazole-4-acetate+NADH+ H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1920 \"Imidazole-4-acetaldehyde+NAD++H2O=Imidazole-4-acetate+NADH+ H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1921 \"Imidazole-4-acetaldehyde+NAD++H2O=Imidazole-4-acetate+NADH+ H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1921 \"Imidazole-4-acetaldehyde+NAD++H2O=Imidazole-4-acetate+NADH+ H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1927 \"1-Methylnicotinamide+2 Oxygen+2 H2O _ST_=_LT_N1-Methyl-4-pyridone-5-carboxamide +N1-Methyl-2-pyridone-5-carboxamide+2 H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1927 \"1-Methylnicotinamide+2 Oxygen+2 H2O _ST_=_LT_N1-Methyl-4-pyridone-5-carboxamide +N1-Methyl-2-pyridone-5-carboxamide+2 H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1928 \"3-Methylbutanoyl-CoA+FAD=3-Methylcrotonyl-CoA+FADH2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1928 \"3-Methylbutanoyl-CoA+FAD=3-Methylcrotonyl-CoA+FADH2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1929 \"3-Methylbutanoyl-CoA+Acceptor=3-Methylcrotonyl-CoA+Reducedacceptor (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1929 \"3-Methylbutanoyl-CoA+Acceptor=3-Methylcrotonyl-CoA+Reducedacceptor (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1931 \"N-Acetylmuramoyl-Ala+H2O=N-Acetyl-D-muramoate+L-Alanine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1931 \"N-Acetylmuramoyl-Ala+H2O=N-Acetyl-D-muramoate+L-Alanine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1934 \"Tetrahydrofolate+S-Aminomethyldihydrolipoylprotein _ST_=_LT_5,10-Methylenetetrahydrofolate+NH3+Dihydrolipoylprotein\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1934 \"Tetrahydrofolate+S-Aminomethyldihydrolipoylprotein _ST_=_LT_5,10-Methylenetetrahydrofolate+NH3+Dihydrolipoylprotein\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1935 \"3-Hydroxyisovaleryl-CoA=3-Methylcrotonyl-CoA+H2O (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1935 \"3-Hydroxyisovaleryl-CoA=3-Methylcrotonyl-CoA+H2O (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1940 \"(S)-3-Hydroxydodecanoyl-CoA=2-trans-Dodecenoyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1940 \"(S)-3-Hydroxydodecanoyl-CoA=2-trans-Dodecenoyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1941 \"3-Hydroxy-L-kynurenine+2-Oxoglutarate _ST_=_LT_4-(2-Amino-3-hydroxyphenyl)-2,4-dioxobutanoate+L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1941 \"3-Hydroxy-L-kynurenine+2-Oxoglutarate _ST_=_LT_4-(2-Amino-3-hydroxyphenyl)-2,4-dioxobutanoate+L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1942 \"3-Hydroxy-L-kynurenine+2-Oxoglutarate _ST_=_LT_4-(2-Amino-3-hydroxyphenyl)-2,4-dioxobutanoate+L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1942 \"3-Hydroxy-L-kynurenine+2-Oxoglutarate _ST_=_LT_4-(2-Amino-3-hydroxyphenyl)-2,4-dioxobutanoate+L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1949 \"H2O+IV3GalNAca-Gb4Cer=N-Acetyl-D-galactosamine+Globoside\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1949 \"H2O+IV3GalNAca-Gb4Cer=N-Acetyl-D-galactosamine+Globoside\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1952 \"H2O+Globoside=N-Acetyl-D-galactosamine+Globotriaosylceramide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1952 \"H2O+Globoside=N-Acetyl-D-galactosamine+Globotriaosylceramide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1954 \"L-3-Amino-isobutanoate+2-Oxoglutarate=(S)-Methylmalonatesemialdehyde+L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1954 \"L-3-Amino-isobutanoate+2-Oxoglutarate=(S)-Methylmalonatesemialdehyde+L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1958 \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA+NAD+ _ST_=_LT_2-Methylacetoacetyl-CoA+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1958 \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA+NAD+ _ST_=_LT_2-Methylacetoacetyl-CoA+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1959 \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA=2-Methylbut-2-enoyl-CoA +H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1959 \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA=2-Methylbut-2-enoyl-CoA +H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1967 \"2-Methylprop-2-enoyl-CoA+H2O=(S)-3-Hydroxyisobutyryl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1967 \"2-Methylprop-2-enoyl-CoA+H2O=(S)-3-Hydroxyisobutyryl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1969 \"3-Methyl-2-oxopentanoate+Lipoamide _ST_=_LT_S-(2-Methylbutanoyl)-dihydrolipoamide+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1969 \"3-Methyl-2-oxopentanoate+Lipoamide _ST_=_LT_S-(2-Methylbutanoyl)-dihydrolipoamide+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1981 \"Peptidylproline (omega=180)=Peptidylproline (omega=0)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1981 \"Peptidylproline (omega=180)=Peptidylproline (omega=0)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1986 \"6-Lactoyl-5,6,7,8-tetrahydropterin+NADP+ _ST_=_LT_6-Pyruvoyltetrahydropterin+NADPH+H+ (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1986 \"6-Lactoyl-5,6,7,8-tetrahydropterin+NADP+ _ST_=_LT_6-Pyruvoyltetrahydropterin+NADPH+H+ (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1988 \"2-Amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl)dihydropteridinetriphosphate+H2O=6-Pyruvoyltetrahydropterin+Triphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1988 \"2-Amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl)dihydropteridinetriphosphate+H2O=6-Pyruvoyltetrahydropterin+Triphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1994 \"4-(2-Aminoethyl)-1,2-benzenediol+H2O+Oxygen _ST_=_LT_3,4-Dihydroxyphenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1994 \"4-(2-Aminoethyl)-1,2-benzenediol+H2O+Oxygen _ST_=_LT_3,4-Dihydroxyphenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1996 \"4-(2-Aminoethyl)-1,2-benzenediol+H2O+Oxygen _ST_=_LT_3,4-Dihydroxyphenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1996 \"4-(2-Aminoethyl)-1,2-benzenediol+H2O+Oxygen _ST_=_LT_3,4-Dihydroxyphenylacetaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_1999 \"3alpha-Hydroxy-5beta-androstan-17-one+NADP+ _ST_=_LT_5beta-Androstane-3,17-dione+NADPH (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_1999 \"3alpha-Hydroxy-5beta-androstan-17-one+NADP+ _ST_=_LT_5beta-Androstane-3,17-dione+NADPH (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2009 \"2-Amino-3-carboxymuconate semialdehyde=2-Aminomuconatesemialdehyde+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2009 \"2-Amino-3-carboxymuconate semialdehyde=2-Aminomuconatesemialdehyde+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2011 \"5alpha-Cholest-8-en-3beta-ol+CO2+4 H2O+3 NADP+=Methostenol+ 3 Oxygen+3 NADPH+3 H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2011 \"5alpha-Cholest-8-en-3beta-ol+CO2+4 H2O+3 NADP+=Methostenol+ 3 Oxygen+3 NADPH+3 H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2014 \"3alpha-Hydroxy-5beta-androstan-17-one+UDPglucuronate _ST_=_LT_Etiocholan-3alpha-ol-17-one 3-glucuronide+UDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2014 \"3alpha-Hydroxy-5beta-androstan-17-one+UDPglucuronate _ST_=_LT_Etiocholan-3alpha-ol-17-one 3-glucuronide+UDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2020 \"D-myo-Inositol 3,4-bisphosphate+H2O=1D-myo-Inositol3-phosphate+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2020 \"D-myo-Inositol 3,4-bisphosphate+H2O=1D-myo-Inositol3-phosphate+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2021 \"Glycoprotein N-acetyl-D-glucosaminyl-phospho-D-mannose+H2O _ST_=_LT_N-Acetyl-D-glucosamine+Glycoprotein phospho-D-mannose (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2021 \"Glycoprotein N-acetyl-D-glucosaminyl-phospho-D-mannose+H2O _ST_=_LT_N-Acetyl-D-glucosamine+Glycoprotein phospho-D-mannose (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2031 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+H2O _ST_=_LT_1-Phosphatidyl-1D-myo-inositol 4-phosphate+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2031 \"1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+H2O _ST_=_LT_1-Phosphatidyl-1D-myo-inositol 4-phosphate+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2036 \"Electron-transferring flavoprotein+Propanoyl-CoA=Reducedelectron-transferring flavoprotein+Propenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2036 \"Electron-transferring flavoprotein+Propanoyl-CoA=Reducedelectron-transferring flavoprotein+Propenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2038 \"Reduced electron-transferring flavoprotein+Ubiquinone _ST_=_LT_Electron-transferring flavoprotein+Ubiquinol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2038 \"Reduced electron-transferring flavoprotein+Ubiquinone _ST_=_LT_Electron-transferring flavoprotein+Ubiquinol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2040 \"L-1-Pyrroline-3-hydroxy-5-carboxylate+NAD++2 H2O _ST_=_LT_L-erythro-4-Hydroxyglutamate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2040 \"L-1-Pyrroline-3-hydroxy-5-carboxylate+NAD++2 H2O _ST_=_LT_L-erythro-4-Hydroxyglutamate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2041 \"L-1-Pyrroline-3-hydroxy-5-carboxylate+NADP++2 H2O _ST_=_LT_L-erythro-4-Hydroxyglutamate+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2041 \"L-1-Pyrroline-3-hydroxy-5-carboxylate+NADP++2 H2O _ST_=_LT_L-erythro-4-Hydroxyglutamate+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2043 \"1-Alkyl-2-acetyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_1-Organyl-2-lyso-sn-glycero-3-phosphocholine+Acetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2043 \"1-Alkyl-2-acetyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_1-Organyl-2-lyso-sn-glycero-3-phosphocholine+Acetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2044 \"1-Alkyl-2-acetyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_1-Organyl-2-lyso-sn-glycero-3-phosphocholine+Acetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2044 \"1-Alkyl-2-acetyl-sn-glycero-3-phosphocholine+H2O _ST_=_LT_1-Organyl-2-lyso-sn-glycero-3-phosphocholine+Acetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2048 \"Digalactosyl-diacylglycerol+H2O _ST_=_LT_1,2-Diacyl-3-beta-D-galactosyl-sn-glycerol+Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2048 \"Digalactosyl-diacylglycerol+H2O _ST_=_LT_1,2-Diacyl-3-beta-D-galactosyl-sn-glycerol+Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2052 \"3beta-Hydroxy-4beta-methyl-5alpha-cholest-7-ene-4alpha-carboxylate +NAD+=4alpha-Methyl-5alpha-cholest-7-en-3-one+CO2+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2052 \"3beta-Hydroxy-4beta-methyl-5alpha-cholest-7-ene-4alpha-carboxylate +NAD+=4alpha-Methyl-5alpha-cholest-7-en-3-one+CO2+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2054 \"UDP-N-acetyl-D-glucosamine+Lactosylceramide=UDP+Lc3Cer\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2054 \"UDP-N-acetyl-D-glucosamine+Lactosylceramide=UDP+Lc3Cer\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2055 \"UDP-N-acetyl-D-glucosamine+Lactosylceramide=UDP+Lc3Cer\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2055 \"UDP-N-acetyl-D-glucosamine+Lactosylceramide=UDP+Lc3Cer\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2056 \"UDP-N-acetyl-D-glucosamine +N-Acetyl-D-glucosaminyldiphosphodolichol=UDP +N,N'-Chitobiosyldiphosphodolichol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2056 \"UDP-N-acetyl-D-glucosamine +N-Acetyl-D-glucosaminyldiphosphodolichol=UDP +N,N'-Chitobiosyldiphosphodolichol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2057 \"UDP-N-acetyl-D-glucosamine +N-Acetyl-D-glucosaminyldiphosphodolichol=UDP +N,N'-Chitobiosyldiphosphodolichol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2057 \"UDP-N-acetyl-D-glucosamine +N-Acetyl-D-glucosaminyldiphosphodolichol=UDP +N,N'-Chitobiosyldiphosphodolichol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2062 \"GDP-D-mannose+N,N'-Chitobiosyldiphosphodolichol=GDP+G00003\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2062 \"GDP-D-mannose+N,N'-Chitobiosyldiphosphodolichol=GDP+G00003\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2064 \"3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al+NAD++H2O _ST_=_LT_3alpha,7alpha-Dihydroxy-5beta-cholestanate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2064 \"3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al+NAD++H2O _ST_=_LT_3alpha,7alpha-Dihydroxy-5beta-cholestanate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2065 \"3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al+NAD++H2O _ST_=_LT_3alpha,7alpha-Dihydroxy-5beta-cholestanate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2065 \"3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al+NAD++H2O _ST_=_LT_3alpha,7alpha-Dihydroxy-5beta-cholestanate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2066 \"3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al+NAD++H2O _ST_=_LT_3alpha,7alpha-Dihydroxy-5beta-cholestanate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2066 \"3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al+NAD++H2O _ST_=_LT_3alpha,7alpha-Dihydroxy-5beta-cholestanate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2069 \"Phosphatidylinositol-3,4,5-trisphosphate+H2O _ST_=_LT_1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2069 \"Phosphatidylinositol-3,4,5-trisphosphate+H2O _ST_=_LT_1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2080 \"ATP+1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate=ADP +Phosphatidylinositol-3,4,5-trisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2080 \"ATP+1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate=ADP +Phosphatidylinositol-3,4,5-trisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2081 \"Propanoyl-CoA+Chenodeoxycholoyl-CoA=CoA +3alpha,7alpha-Dihydroxy-5beta-cholestanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2081 \"Propanoyl-CoA+Chenodeoxycholoyl-CoA=CoA +3alpha,7alpha-Dihydroxy-5beta-cholestanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2082 \"Alprostadil+NADP+ _ST_=_LT_(13E)-11alpha-Hydroxy-9,15-dioxoprost-13-enoate+NADPH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2082 \"Alprostadil+NADP+ _ST_=_LT_(13E)-11alpha-Hydroxy-9,15-dioxoprost-13-enoate+NADPH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2083 \"(5Z)-(15S)-11alpha-Hydroxy-9,15-dioxoprostanoate+NAD+ _ST_=_LT_(5Z)-(15S)-11alpha-Hydroxy-9,15-dioxoprosta-13-enoate+NADH (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2083 \"(5Z)-(15S)-11alpha-Hydroxy-9,15-dioxoprostanoate+NAD+ _ST_=_LT_(5Z)-(15S)-11alpha-Hydroxy-9,15-dioxoprosta-13-enoate+NADH (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2095 \"ATP+3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoate+CoA _ST_=_LT_AMP+Pyrophosphate +3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2095 \"ATP+3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoate+CoA _ST_=_LT_AMP+Pyrophosphate +3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2099 \"CMP-N-acetylneuraminate+GM3=CMP+GD3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2099 \"CMP-N-acetylneuraminate+GM3=CMP+GD3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2102 \"GM2+H2O=GM3+N-Acetyl-D-galactosamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2102 \"GM2+H2O=GM3+N-Acetyl-D-galactosamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2103 \"GM2+H2O=GM3+N-Acetyl-D-galactosamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2103 \"GM2+H2O=GM3+N-Acetyl-D-galactosamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2104 \"CMP-N-acetylneuraminate+T antigen=CMP+G00026\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2104 \"CMP-N-acetylneuraminate+T antigen=CMP+G00026\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2105 \"CMP-N-acetylneuraminate+T antigen=CMP+G00026\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2105 \"CMP-N-acetylneuraminate+T antigen=CMP+G00026\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2106 \"CMP-N-acetylneuraminate+T antigen=CMP+G00026\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2106 \"CMP-N-acetylneuraminate+T antigen=CMP+G00026\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2116 \"GM1+H2O=GM2+Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2116 \"GM1+H2O=GM2+Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2118 \"GM1+H2O=GM2+Galactose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2118 \"GM1+H2O=GM2+Galactose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2123 \"CMP-N-acetylneuraminate +D-Galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-glucosylceramide=CMP +N-Acetylneuraminyl-D-galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-glucosylceramide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2123 \"CMP-N-acetylneuraminate +D-Galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-glucosylceramide=CMP +N-Acetylneuraminyl-D-galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-glucosylceramide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2127 \"UDP-N-acetyl-D-glucosamine+Paragloboside=UDP+nLc5Cer\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2127 \"UDP-N-acetyl-D-glucosamine+Paragloboside=UDP+nLc5Cer\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2131 \"14-Demethyllanosterol+6 Oxygen+6 NADPH+6 H+=Zymosterol+2CO2+8 H2O+6 NADP+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2131 \"14-Demethyllanosterol+6 Oxygen+6 NADPH+6 H+=Zymosterol+2CO2+8 H2O+6 NADP+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2132 \"14-Demethyllanosterol+6 Oxygen+6 NADPH+6 H+=Zymosterol+2CO2+8 H2O+6 NADP+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2132 \"14-Demethyllanosterol+6 Oxygen+6 NADPH+6 H+=Zymosterol+2CO2+8 H2O+6 NADP+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2133 \"4alpha-Methylcholesta-8-en-3beta-ol=Methostenol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2133 \"4alpha-Methylcholesta-8-en-3beta-ol=Methostenol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2135 \"(S)-2-Acetolactate+Thiamin diphosphate _ST_=_LT_2-(alpha-Hydroxyethyl)thiamine diphosphate+Pyruvate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2135 \"(S)-2-Acetolactate+Thiamin diphosphate _ST_=_LT_2-(alpha-Hydroxyethyl)thiamine diphosphate+Pyruvate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2138 \"N-Methylhistamine+H2O+Oxygen=Methylimidazole acetaldehyde +NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2138 \"N-Methylhistamine+H2O+Oxygen=Methylimidazole acetaldehyde +NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2141 \"17alpha,20alpha-Dihydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_17alpha-Hydroxypregnenolone+4-Methylpentanal+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2141 \"17alpha,20alpha-Dihydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_17alpha-Hydroxypregnenolone+4-Methylpentanal+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2142 \"17alpha,20alpha-Dihydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_17alpha-Hydroxypregnenolone+4-Methylpentanal+NADP++H2O\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2142 \"17alpha,20alpha-Dihydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_17alpha-Hydroxypregnenolone+4-Methylpentanal+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2150 \"Estriol+NAD+=16alpha-Hydroxyestrone+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2150 \"Estriol+NAD+=16alpha-Hydroxyestrone+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2152 \"Estriol+NADP+=16alpha-Hydroxyestrone+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2152 \"Estriol+NADP+=16alpha-Hydroxyestrone+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2154 \"UDPglucuronate+Estriol=UDP+16-Glucuronide-estriol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2154 \"UDPglucuronate+Estriol=UDP+16-Glucuronide-estriol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2164 \"(2S)-2-Methylacyl-CoA=(2R)-2-Methylacyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2164 \"(2S)-2-Methylacyl-CoA=(2R)-2-Methylacyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2169 \"(S)-3-Hydroxyhexadecanoyl-CoA+NAD+=3-Oxopalmitoyl-CoA+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2169 \"(S)-3-Hydroxyhexadecanoyl-CoA+NAD+=3-Oxopalmitoyl-CoA+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2170 \"(S)-3-Hydroxyhexadecanoyl-CoA=trans-Hexadec-2-enoyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2170 \"(S)-3-Hydroxyhexadecanoyl-CoA=trans-Hexadec-2-enoyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2171 \"(S)-3-Hydroxytetradecanoyl-CoA+NAD+=3-Oxotetradecanoyl-CoA +NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2171 \"(S)-3-Hydroxytetradecanoyl-CoA+NAD+=3-Oxotetradecanoyl-CoA +NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2172 \"(S)-3-Hydroxytetradecanoyl-CoA=trans-Tetradec-2-enoyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2172 \"(S)-3-Hydroxytetradecanoyl-CoA=trans-Tetradec-2-enoyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2173 \"(S)-3-Hydroxydodecanoyl-CoA+NAD+=3-Oxododecanoyl-CoA+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2173 \"(S)-3-Hydroxydodecanoyl-CoA+NAD+=3-Oxododecanoyl-CoA+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2174 \"Decanoyl-CoA+Acetyl-CoA=CoA+3-Oxododecanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2174 \"Decanoyl-CoA+Acetyl-CoA=CoA+3-Oxododecanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2175 \"(S)-Hydroxydecanoyl-CoA+NAD+=3-Oxodecanoyl-CoA+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2175 \"(S)-Hydroxydecanoyl-CoA+NAD+=3-Oxodecanoyl-CoA+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2176 \"(S)-Hydroxydecanoyl-CoA=trans-Dec-2-enoyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2176 \"(S)-Hydroxydecanoyl-CoA=trans-Dec-2-enoyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2177 \"(S)-Hydroxyoctanoyl-CoA+NAD+=3-Oxooctanoyl-CoA+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2177 \"(S)-Hydroxyoctanoyl-CoA+NAD+=3-Oxooctanoyl-CoA+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2178 \"(S)-Hydroxyoctanoyl-CoA=trans-Oct-2-enoyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2178 \"(S)-Hydroxyoctanoyl-CoA=trans-Oct-2-enoyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2179 \"Hexanoyl-CoA+Acetyl-CoA=CoA+3-Oxooctanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2179 \"Hexanoyl-CoA+Acetyl-CoA=CoA+3-Oxooctanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2180 \"(S)-Hydroxyhexanoyl-CoA+NAD+=3-Oxohexanoyl-CoA+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2180 \"(S)-Hydroxyhexanoyl-CoA+NAD+=3-Oxohexanoyl-CoA+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2181 \"(S)-Hydroxyhexanoyl-CoA=trans-Hex-2-enoyl-CoA+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2181 \"(S)-Hydroxyhexanoyl-CoA=trans-Hex-2-enoyl-CoA+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2182 \"Hexanoyl-CoA+FAD=trans-Hex-2-enoyl-CoA+FADH2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2182 \"Hexanoyl-CoA+FAD=trans-Hex-2-enoyl-CoA+FADH2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2183 \"Decanoyl-CoA+NADP+=trans-Dec-2-enoyl-CoA+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2183 \"Decanoyl-CoA+NADP+=trans-Dec-2-enoyl-CoA+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2184 \"Decanoyl-CoA+FAD=trans-Dec-2-enoyl-CoA+FADH2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2184 \"Decanoyl-CoA+FAD=trans-Dec-2-enoyl-CoA+FADH2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2186 \"11beta-Hydroxyandrost-4-ene-3,17-dione+NAD+=Adrenosterone +NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2186 \"11beta-Hydroxyandrost-4-ene-3,17-dione+NAD+=Adrenosterone +NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2188 \"11beta-Hydroxyandrost-4-ene-3,17-dione+NADP+=Adrenosterone +NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2188 \"11beta-Hydroxyandrost-4-ene-3,17-dione+NADP+=Adrenosterone +NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2192 \"ATP+beta-D-Fructose 6-phosphate=ADP+beta-D-Fructose1,6-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2192 \"ATP+beta-D-Fructose 6-phosphate=ADP+beta-D-Fructose1,6-bisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2197 \"Zymosterol=5alpha-Cholesta-7,24-dien-3beta-ol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2197 \"Zymosterol=5alpha-Cholesta-7,24-dien-3beta-ol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2198 \"Zymosterol=5alpha-Cholesta-7,24-dien-3beta-ol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2198 \"Zymosterol=5alpha-Cholesta-7,24-dien-3beta-ol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2200 \"3alpha,7alpha,26-Trihydroxy-5beta-cholestane+NAD+ _ST_=_LT_3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al+NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2200 \"3alpha,7alpha,26-Trihydroxy-5beta-cholestane+NAD+ _ST_=_LT_3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al+NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2205 \"Propanoyl-CoA+Chenodeoxyglycocholoyl-CoA=CoA +3alpha,7alpha-Dihydroxy-5beta-24-oxocholestanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2205 \"Propanoyl-CoA+Chenodeoxyglycocholoyl-CoA=CoA +3alpha,7alpha-Dihydroxy-5beta-24-oxocholestanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2210 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestane+NAD+ _ST_=_LT_7alpha,12alpha-Dihydroxy-5beta-cholestan-3-one+NADH+H+ (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2210 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestane+NAD+ _ST_=_LT_7alpha,12alpha-Dihydroxy-5beta-cholestan-3-one+NADH+H+ (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2225 \"Tetrahydrocorticosterone+NAD+ _ST_=_LT_3alpha,21-Dihydroxy-5beta-pregnane-11,20-dione+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2225 \"Tetrahydrocorticosterone+NAD+ _ST_=_LT_3alpha,21-Dihydroxy-5beta-pregnane-11,20-dione+NADH+H+\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2227 \"Tetrahydrocorticosterone+NADP+ _ST_=_LT_3alpha,21-Dihydroxy-5beta-pregnane-11,20-dione+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2227 \"Tetrahydrocorticosterone+NADP+ _ST_=_LT_3alpha,21-Dihydroxy-5beta-pregnane-11,20-dione+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2245 \"20alpha-Hydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_17alpha,20alpha-Dihydroxycholesterol+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2245 \"20alpha-Hydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_17alpha,20alpha-Dihydroxycholesterol+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2246 \"20alpha-Hydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_17alpha,20alpha-Dihydroxycholesterol+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2246 \"20alpha-Hydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_17alpha,20alpha-Dihydroxycholesterol+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2248 \"20alpha-Hydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_20alpha,22beta-Dihydroxycholesterol+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2248 \"20alpha-Hydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_20alpha,22beta-Dihydroxycholesterol+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2250 \"22beta-Hydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_20alpha,22beta-Dihydroxycholesterol+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2250 \"22beta-Hydroxycholesterol+NADPH+H++Oxygen _ST_=_LT_20alpha,22beta-Dihydroxycholesterol+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2251 \"Sulfatide+H2O=Galactosylceramide+Sulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2251 \"Sulfatide+H2O=Galactosylceramide+Sulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2252 \"Sulfatide+H2O=Galactosylceramide+Sulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2252 \"Sulfatide+H2O=Galactosylceramide+Sulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2278 \"3-Methoxytyramine+H2O+Oxygen _ST_=_LT_3-Methoxy-4-hydroxyphenylacetaldehyde+H2O2+NH3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2278 \"3-Methoxytyramine+H2O+Oxygen _ST_=_LT_3-Methoxy-4-hydroxyphenylacetaldehyde+H2O2+NH3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2279 \"3-Methoxy-4-hydroxyphenylglycolaldehyde+NAD++H2O _ST_=_LT_3-Methoxy-4-hydroxymandelate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2279 \"3-Methoxy-4-hydroxyphenylglycolaldehyde+NAD++H2O _ST_=_LT_3-Methoxy-4-hydroxymandelate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2280 \"3-Methoxy-4-hydroxyphenylglycolaldehyde+NAD++H2O _ST_=_LT_3-Methoxy-4-hydroxymandelate+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2280 \"3-Methoxy-4-hydroxyphenylglycolaldehyde+NAD++H2O _ST_=_LT_3-Methoxy-4-hydroxymandelate+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2282 \"3-Methoxy-4-hydroxyphenylglycolaldehyde+NADP++H2O _ST_=_LT_3-Methoxy-4-hydroxymandelate+NADPH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2282 \"3-Methoxy-4-hydroxyphenylglycolaldehyde+NADP++H2O _ST_=_LT_3-Methoxy-4-hydroxymandelate+NADPH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2283 \"L-Normetanephrine+H2O+Oxygen _ST_=_LT_3-Methoxy-4-hydroxyphenylglycolaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2283 \"L-Normetanephrine+H2O+Oxygen _ST_=_LT_3-Methoxy-4-hydroxyphenylglycolaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2284 \"L-Normetanephrine+H2O+Oxygen _ST_=_LT_3-Methoxy-4-hydroxyphenylglycolaldehyde+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2284 \"L-Normetanephrine+H2O+Oxygen _ST_=_LT_3-Methoxy-4-hydroxyphenylglycolaldehyde+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2286 \"L-Metanephrine+H2O+Oxygen _ST_=_LT_3-Methoxy-4-hydroxyphenylglycolaldehyde+H2O2+Methylamine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2286 \"L-Metanephrine+H2O+Oxygen _ST_=_LT_3-Methoxy-4-hydroxyphenylglycolaldehyde+H2O2+Methylamine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2287 \"5-Hydroxyindoleacetaldehyde+NAD++H2O=5-Hydroxyindoleacetate+ H++NADH\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2287 \"5-Hydroxyindoleacetaldehyde+NAD++H2O=5-Hydroxyindoleacetate+ H++NADH\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2324 \"Uroporphyrinogen I=Coproporphyrinogen I+4 CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2324 \"Uroporphyrinogen I=Coproporphyrinogen I+4 CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2325 \"Bilirubin beta-diglucuronide+2 H2O+6 H+=D-Urobilinogen+2D-Glucuronate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2325 \"Bilirubin beta-diglucuronide+2 H2O+6 H+=D-Urobilinogen+2D-Glucuronate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2329 \"GDP-D-mannose+G00003=GDP+G00004\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2329 \"GDP-D-mannose+G00003=GDP+G00004\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2340 \"N4-Acetylaminobutanal+NAD++H2O=4-Acetamidobutanoate+NADH +H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2340 \"N4-Acetylaminobutanal+NAD++H2O=4-Acetamidobutanoate+NADH +H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2341 \"N4-Acetylaminobutanal+NAD++H2O=4-Acetamidobutanoate+NADH +H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2341 \"N4-Acetylaminobutanal+NAD++H2O=4-Acetamidobutanoate+NADH +H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2343 \"L-erythro-4-Hydroxyglutamate+NAD+=L-4-Hydroxyglutamatesemialdehyde+NADH+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2343 \"L-erythro-4-Hydroxyglutamate+NAD+=L-4-Hydroxyglutamatesemialdehyde+NADH+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2345 \"L-erythro-4-Hydroxyglutamate+2-Oxoglutarate _ST_=_LT_D-4-Hydroxy-2-oxoglutarate+L-Glutamate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2345 \"L-erythro-4-Hydroxyglutamate+2-Oxoglutarate _ST_=_LT_D-4-Hydroxy-2-oxoglutarate+L-Glutamate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2346 \"(S)-3-Hydroxyisobutyryl-CoA+H2O=CoA+(S)-3-Hydroxyisobutyrate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2346 \"(S)-3-Hydroxyisobutyryl-CoA+H2O=CoA+(S)-3-Hydroxyisobutyrate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2347 \"(S)-3-Hydroxyisobutyrate+NAD+=(S)-Methylmalonate semialdehyde+ NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2347 \"(S)-3-Hydroxyisobutyrate+NAD+=(S)-Methylmalonate semialdehyde+ NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2349 \"3'-Phosphoadenylyl sulfate+Digalactosylceramide=Adenosine3',5'-bisphosphate+Digalactosylceramidesulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2349 \"3'-Phosphoadenylyl sulfate+Digalactosylceramide=Adenosine3',5'-bisphosphate+Digalactosylceramidesulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2350 \"3'-Phosphoadenylyl sulfate+Digalactosylceramide=Adenosine3',5'-bisphosphate+Digalactosylceramidesulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2350 \"3'-Phosphoadenylyl sulfate+Digalactosylceramide=Adenosine3',5'-bisphosphate+Digalactosylceramidesulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2359 \"CMP-N-acetylneuraminate+GD1b=CMP+GT1b\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2359 \"CMP-N-acetylneuraminate+GD1b=CMP+GT1b\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2371 \"Diacylglycerol+H2O=Acylglycerol+Carboxylate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2371 \"Diacylglycerol+H2O=Acylglycerol+Carboxylate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2372 \"Diacylglycerol+H2O=Acylglycerol+Carboxylate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2372 \"Diacylglycerol+H2O=Acylglycerol+Carboxylate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2383 \"4,8,12-Trimethyltridecanoyl-CoA+Propanoyl-CoA _ST_=_LT_3-Oxopristanoyl-CoA+CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2383 \"4,8,12-Trimethyltridecanoyl-CoA+Propanoyl-CoA _ST_=_LT_3-Oxopristanoyl-CoA+CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2386 \"2-Deoxy-5-keto-D-gluconic acid 6-phosphate=3-Oxopropanoate +Glycerone phosphate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2386 \"2-Deoxy-5-keto-D-gluconic acid 6-phosphate=3-Oxopropanoate +Glycerone phosphate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2387 \"2-Deoxy-5-keto-D-gluconic acid 6-phosphate=3-Oxopropanoate +Glycerone phosphate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2387 \"2-Deoxy-5-keto-D-gluconic acid 6-phosphate=3-Oxopropanoate +Glycerone phosphate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2388 \"2-Deoxy-5-keto-D-gluconic acid 6-phosphate=3-Oxopropanoate +Glycerone phosphate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2388 \"2-Deoxy-5-keto-D-gluconic acid 6-phosphate=3-Oxopropanoate +Glycerone phosphate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2390 \"Parathion+H2O=Diethylthiophosphoric acid+4-Nitrophenol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2390 \"Parathion+H2O=Diethylthiophosphoric acid+4-Nitrophenol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2392 \"Paraoxon+H2O=Diethylphosphoric acid+4-Nitrophenol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2392 \"Paraoxon+H2O=Diethylphosphoric acid+4-Nitrophenol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2394 \"D-Gal alpha 1-_LT_6D-Gal alpha 1-_LT_6D-Glucose+H2O=D-Galactose +Melibiose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2394 \"D-Gal alpha 1-_LT_6D-Gal alpha 1-_LT_6D-Glucose+H2O=D-Galactose +Melibiose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2408 \"Crotonoyl-CoA+H2O=3-Hydroxybutanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2408 \"Crotonoyl-CoA+H2O=3-Hydroxybutanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2412 \"14-Demethyllanosterol+NADP+ _ST_=_LT_4,4-Dimethyl-5alpha-cholesta-8,14,24-trien-3beta-ol+NADPH+H+ (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2412 \"14-Demethyllanosterol+NADP+ _ST_=_LT_4,4-Dimethyl-5alpha-cholesta-8,14,24-trien-3beta-ol+NADPH+H+ (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2417 \"Reduced riboflavin+NADP+=Riboflavin+NADPH+H+ (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2417 \"Reduced riboflavin+NADP+=Riboflavin+NADPH+H+ (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2418 \"Phytanoyl-CoA+2-Oxoglutarate+Oxygen=2-Hydroxyphytanoyl-CoA +Succinate+CO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2418 \"Phytanoyl-CoA+2-Oxoglutarate+Oxygen=2-Hydroxyphytanoyl-CoA +Succinate+CO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2435 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2435 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2436 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2436 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2437 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2437 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2438 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2438 \"ATP+1-Phosphatidyl-1D-myo-inositol 4-phosphate=ADP +1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2441 \"ATP+myo-Inositol hexakisphosphate=ADP+1-Diphosinositolpentakisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2441 \"ATP+myo-Inositol hexakisphosphate=ADP+1-Diphosinositolpentakisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2444 \"ATP+D-myo-Inositol 1,4,5-trisphosphate=ADP+1D-myo-Inositol1,4,5,6-tetrakisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2444 \"ATP+D-myo-Inositol 1,4,5-trisphosphate=ADP+1D-myo-Inositol1,4,5,6-tetrakisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2445 \"ATP+1D-myo-Inositol 1,4,5,6-tetrakisphosphate=ADP +1D-myo-Inositol 1,3,4,5,6-pentakisphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2445 \"ATP+1D-myo-Inositol 1,4,5,6-tetrakisphosphate=ADP +1D-myo-Inositol 1,3,4,5,6-pentakisphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2454 \"UDP-N-acetyl-D-galactosamine+Protein serine=UDP+Tn antigen\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2454 \"UDP-N-acetyl-D-galactosamine+Protein serine=UDP+Tn antigen\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2473 \"UDP-D-xylose+Protein serine=UDP+G00154\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2473 \"UDP-D-xylose+Protein serine=UDP+G00154\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2474 \"UDP-D-galactose+G00154=UDP+G00155\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2474 \"UDP-D-galactose+G00154=UDP+G00155\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2475 \"UDP-D-galactose+G00154=UDP+G00155\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2475 \"UDP-D-galactose+G00154=UDP+G00155\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2476 \"UDP-D-galactose+G00155=UDP+G00156\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2476 \"UDP-D-galactose+G00155=UDP+G00156\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2489 \"CMP-N-acetylneuraminate+GD3=CMP+GT3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2489 \"CMP-N-acetylneuraminate+GD3=CMP+GT3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2498 \"CMP-N-acetylneuraminate+GA1=CMP+GM1b\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2498 \"CMP-N-acetylneuraminate+GA1=CMP+GM1b\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2499 \"CMP-N-acetylneuraminate+GA1=CMP+GM1b\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2499 \"CMP-N-acetylneuraminate+GA1=CMP+GM1b\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2503 \"H2O+Globotriaosylceramide=alpha-D-Galactose+Lactosylceramide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2503 \"H2O+Globotriaosylceramide=alpha-D-Galactose+Lactosylceramide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2504 \"H2O+Globotriaosylceramide=alpha-D-Galactose+Lactosylceramide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2504 \"H2O+Globotriaosylceramide=alpha-D-Galactose+Lactosylceramide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2505 \"H2O+Globotriaosylceramide=alpha-D-Galactose+Lactosylceramide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2505 \"H2O+Globotriaosylceramide=alpha-D-Galactose+Lactosylceramide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2509 \"CMP-N-acetylneuraminate+Galactosylgloboside=CMP +Monosialylgalactosylgloboside\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2509 \"CMP-N-acetylneuraminate+Galactosylgloboside=CMP +Monosialylgalactosylgloboside\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2514 \"UDP-N-acetyl-D-glucosamine+nLc6Cer=UDP+nLc7Cer\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2514 \"UDP-N-acetyl-D-glucosamine+nLc6Cer=UDP+nLc7Cer\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2517 \"UDP-D-galactose+Lc3Cer=UDP+Paragloboside\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2517 \"UDP-D-galactose+Lc3Cer=UDP+Paragloboside\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2522 \"H2O+G00171=D-Glucose+G00010\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2522 \"H2O+G00171=D-Glucose+G00010\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2523 \"H2O+G00171=D-Glucose+G00010\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2523 \"H2O+G00171=D-Glucose+G00010\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2525 \"H2O+G00010=D-Glucose+G00011\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2525 \"H2O+G00010=D-Glucose+G00011\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2526 \"H2O+G00010=D-Glucose+G00011\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2526 \"H2O+G00010=D-Glucose+G00011\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2528 \"G00011+4 H2O=G00012+4 D-Mannose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2528 \"G00011+4 H2O=G00012+4 D-Mannose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2533 \"G00013+2 H2O=G00014+2 D-Mannose\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2533 \"G00013+2 H2O=G00014+2 D-Mannose\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2539 \"UDP-N-acetyl-D-glucosamine+G00015=UDP+G00020\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2539 \"UDP-N-acetyl-D-glucosamine+G00015=UDP+G00020\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2542 \"GDP-L-fucose+G00015=GDP+G00016\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2542 \"GDP-L-fucose+G00015=GDP+G00016\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2544 \"2 UDP-D-galactose+G00016=2 UDP+G00017\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2544 \"2 UDP-D-galactose+G00016=2 UDP+G00017\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2547 \"2 CMP-N-acetylneuraminate+G00017=2 CMP+DS 3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2547 \"2 CMP-N-acetylneuraminate+G00017=2 CMP+DS 3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2556 \"GDP-L-fucose+Paragloboside=GDP+IV2Fuc-nLc4Cer\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2556 \"GDP-L-fucose+Paragloboside=GDP+IV2Fuc-nLc4Cer\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2564 \"GDP-L-fucose+G00057=GDP+Type IIIH glycolipid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2564 \"GDP-L-fucose+G00057=GDP+Type IIIH glycolipid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2567 \"UDP-D-galactose+nLc5Cer=UDP+nLc6Cer\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2567 \"UDP-D-galactose+nLc5Cer=UDP+nLc6Cer\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2569 \"GDP-L-fucose+nLc6Cer=GDP+VI2Fuc-nLc6\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2569 \"GDP-L-fucose+nLc6Cer=GDP+VI2Fuc-nLc6\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2577 \"GDP-L-fucose+G00073=GDP+G00074\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2577 \"GDP-L-fucose+G00073=GDP+G00074\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2589 \"G00005+GDP-D-mannose=G10526+GDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2589 \"G00005+GDP-D-mannose=G10526+GDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2590 \"G10526+GDP-D-mannose=G00006+GDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2590 \"G10526+GDP-D-mannose=G00006+GDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2594 \"GDP-L-fucose+Lc4Cer=GDP+Fuc-Lc4Cer (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2594 \"GDP-L-fucose+Lc4Cer=GDP+Fuc-Lc4Cer (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2597 \"GDP-L-fucose+Lc4Cer=GDP+IV2Fuc-Lc4Cer\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2597 \"GDP-L-fucose+Lc4Cer=GDP+IV2Fuc-Lc4Cer\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2599 \"GDP-L-fucose+Lc4Cer=GDP+IV2Fuc-Lc4Cer\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2599 \"GDP-L-fucose+Lc4Cer=GDP+IV2Fuc-Lc4Cer\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2603 \"GDP-L-fucose+IV2Fuc-Lc4Cer=GDP+IV2Fuc,III4Fuc-Lc4Cer\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2603 \"GDP-L-fucose+IV2Fuc-Lc4Cer=GDP+IV2Fuc,III4Fuc-Lc4Cer\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2606 \"GDP-L-fucose+Type IB glycolipid=GDP+G00040 (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2606 \"GDP-L-fucose+Type IB glycolipid=GDP+G00040 (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2619 \"GDP-L-fucose+G00038=GDP+Type IB glycolipid (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2619 \"GDP-L-fucose+G00038=GDP+Type IB glycolipid (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2620 \"GDP-L-fucose+G00038=GDP+Type IB glycolipid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2620 \"GDP-L-fucose+G00038=GDP+Type IB glycolipid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2649 \"GDP-D-mannose+G00004=GDP+G00005\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2649 \"GDP-D-mannose+G00004=GDP+G00005\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2651 \"Dolichyl phosphate D-mannose+G00006=Dolichyl phosphate +G10595\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2651 \"Dolichyl phosphate D-mannose+G00006=Dolichyl phosphate +G10595\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2670 \"Sphingosine 1-phosphate=Ethanolamine phosphate+Hexadecanal\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2670 \"Sphingosine 1-phosphate=Ethanolamine phosphate+Hexadecanal\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2671 \"Dihydroceramide+H2O=Fatty acid+Sphinganine (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2671 \"Dihydroceramide+H2O=Fatty acid+Sphinganine (reversible)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2672 \"Dihydroceramide+H2O=Fatty acid+Sphinganine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2672 \"Dihydroceramide+H2O=Fatty acid+Sphinganine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2673 \"Dihydroceramide+H2O=Fatty acid+Sphinganine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2673 \"Dihydroceramide+H2O=Fatty acid+Sphinganine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2675 \"Dihydroceramide+H2O=Fatty acid+Sphinganine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2675 \"Dihydroceramide+H2O=Fatty acid+Sphinganine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2676 \"Sphinganine 1-phosphate+H2O=Sphinganine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2676 \"Sphinganine 1-phosphate+H2O=Sphinganine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2677 \"Sphinganine 1-phosphate+H2O=Sphinganine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2677 \"Sphinganine 1-phosphate+H2O=Sphinganine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2678 \"Sphinganine 1-phosphate+H2O=Sphinganine+Orthophosphate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2678 \"Sphinganine 1-phosphate+H2O=Sphinganine+Orthophosphate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2679 \"Sphingosine 1-phosphate+H2O=Sphingosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2679 \"Sphingosine 1-phosphate+H2O=Sphingosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2680 \"Sphingosine 1-phosphate+H2O=Sphingosine+Orthophosphate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2680 \"Sphingosine 1-phosphate+H2O=Sphingosine+Orthophosphate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2682 \"Ceramide 1-phosphate+H2O=N-Acylsphingosine+Orthophosphate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2682 \"Ceramide 1-phosphate+H2O=N-Acylsphingosine+Orthophosphate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2683 \"Ceramide 1-phosphate+H2O=N-Acylsphingosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2683 \"Ceramide 1-phosphate+H2O=N-Acylsphingosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2684 \"Ceramide 1-phosphate+H2O=N-Acylsphingosine+Orthophosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2684 \"Ceramide 1-phosphate+H2O=N-Acylsphingosine+Orthophosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2693 \"Cocaine+H2O=Ecgonine methyl ester+Benzoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2693 \"Cocaine+H2O=Ecgonine methyl ester+Benzoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2697 \"Cadaverine+H2O+Oxygen=5-Aminopentanal+NH3+H2O2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2697 \"Cadaverine+H2O+Oxygen=5-Aminopentanal+NH3+H2O2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2700 \"Choline phosphate+H2O=Choline+Orthophosphate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2700 \"Choline phosphate+H2O=Choline+Orthophosphate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2702 \"4-Hydroxyphenyl acetate+H2O=Hydroquinone+Acetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2702 \"4-Hydroxyphenyl acetate+H2O=Hydroquinone+Acetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2704 \"(3S)-3-Hydroxyadipyl-CoA+NAD+=3-Oxoadipyl-CoA+NADH+H+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2704 \"(3S)-3-Hydroxyadipyl-CoA+NAD+=3-Oxoadipyl-CoA+NADH+H+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2706 \"5-Carboxy-2-pentenoyl-CoA+H2O=(3S)-3-Hydroxyadipyl-CoA (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2706 \"5-Carboxy-2-pentenoyl-CoA+H2O=(3S)-3-Hydroxyadipyl-CoA (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2709 \"trans-Hex-2-enoyl-CoA+NADPH+H+=Hexanoyl-CoA+NADP+\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2709 \"trans-Hex-2-enoyl-CoA+NADPH+H+=Hexanoyl-CoA+NADP+\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2711 \"Naphthalene+NADPH+Oxygen+H+=(1R,2S)-Naphthalene 1,2-oxide+ NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2711 \"Naphthalene+NADPH+Oxygen+H+=(1R,2S)-Naphthalene 1,2-oxide+ NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2712 \"Naphthalene+NADPH+Oxygen+H+=(1S,2R)-Naphthalene 1,2-oxide+ NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2712 \"Naphthalene+NADPH+Oxygen+H+=(1S,2R)-Naphthalene 1,2-oxide+ NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2713 \"Naphthalene+NADPH+Oxygen+H+=(1S,2R)-Naphthalene 1,2-oxide+ NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2713 \"Naphthalene+NADPH+Oxygen+H+=(1S,2R)-Naphthalene 1,2-oxide+ NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2715 \"(1R,2S)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1R)-Hydroxy-(2R)-glutathionyl-1,2-dihydronaphthalene\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2715 \"(1R,2S)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1R)-Hydroxy-(2R)-glutathionyl-1,2-dihydronaphthalene\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2716 \"(1R,2S)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1R)-Hydroxy-(2R)-glutathionyl-1,2-dihydronaphthalene\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2716 \"(1R,2S)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1R)-Hydroxy-(2R)-glutathionyl-1,2-dihydronaphthalene\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2718 \"(1S,2R)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1R)-Glutathionyl-(2R)-hydroxy-1,2-dihydronaphthalene\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2718 \"(1S,2R)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1R)-Glutathionyl-(2R)-hydroxy-1,2-dihydronaphthalene\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2719 \"(1S,2R)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1R)-Glutathionyl-(2R)-hydroxy-1,2-dihydronaphthalene\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2719 \"(1S,2R)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1R)-Glutathionyl-(2R)-hydroxy-1,2-dihydronaphthalene\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2721 \"(1S,2R)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1S)-Hydroxy-(2S)-glutathionyl-1,2-dihydronaphthalene\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2721 \"(1S,2R)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1S)-Hydroxy-(2S)-glutathionyl-1,2-dihydronaphthalene\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2722 \"(1S,2R)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1S)-Hydroxy-(2S)-glutathionyl-1,2-dihydronaphthalene\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2722 \"(1S,2R)-Naphthalene 1,2-oxide+Glutathione _ST_=_LT_(1S)-Hydroxy-(2S)-glutathionyl-1,2-dihydronaphthalene\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2729 \"(1R,2S)-Naphthalene 1,2-oxide=beta-Naphthol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2729 \"(1R,2S)-Naphthalene 1,2-oxide=beta-Naphthol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2731 \"(1R,2S)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2731 \"(1R,2S)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2732 \"(1R,2S)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2732 \"(1R,2S)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2733 \"(1R,2S)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2733 \"(1R,2S)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2734 \"(1S,2R)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2734 \"(1S,2R)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2735 \"(1S,2R)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2735 \"(1S,2R)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2736 \"(1S,2R)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2736 \"(1S,2R)-Naphthalene 1,2-oxide+H2O _ST_=_LT_1,2-Dihydronaphthalene-1,2-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2740 \"1,2-Dihydronaphthalene-1,2-diol+NADPH+Oxygen+H+ _ST_=_LT_1,2-Dihydroxy-3,4-epoxy-1,2,3,4-tetrahydronaphthalene+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2740 \"1,2-Dihydronaphthalene-1,2-diol+NADPH+Oxygen+H+ _ST_=_LT_1,2-Dihydroxy-3,4-epoxy-1,2,3,4-tetrahydronaphthalene+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2745 \"beta-Naphthol+NADPH+Oxygen+H+=Naphthalene-1,2-diol+NADP++ H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2745 \"beta-Naphthol+NADPH+Oxygen+H+=Naphthalene-1,2-diol+NADP++ H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2768 \"2 Glutathione+5(S)-HPETE=Oxidized glutathione+5(S)-HETE +H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2768 \"2 Glutathione+5(S)-HPETE=Oxidized glutathione+5(S)-HETE +H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2769 \"2 Glutathione+5(S)-HPETE=Oxidized glutathione+5(S)-HETE +H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2769 \"2 Glutathione+5(S)-HPETE=Oxidized glutathione+5(S)-HETE +H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2772 \"2 Glutathione+15(S)-HPETE=Oxidized glutathione +(15S)-15-Hydroxy-5,8,11-cis-13-trans-eicosatetraenoate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2772 \"2 Glutathione+15(S)-HPETE=Oxidized glutathione +(15S)-15-Hydroxy-5,8,11-cis-13-trans-eicosatetraenoate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2773 \"2 Glutathione+15(S)-HPETE=Oxidized glutathione +(15S)-15-Hydroxy-5,8,11-cis-13-trans-eicosatetraenoate+H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2773 \"2 Glutathione+15(S)-HPETE=Oxidized glutathione +(15S)-15-Hydroxy-5,8,11-cis-13-trans-eicosatetraenoate+H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2779 \"15(S)-HPETE=15H-11,12-EETA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2779 \"15(S)-HPETE=15H-11,12-EETA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2781 \"15(S)-HPETE=11H-14,15-EETA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2781 \"15(S)-HPETE=11H-14,15-EETA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2783 \"15H-11,12-EETA+H2O=11,12,15-THETA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2783 \"15H-11,12-EETA+H2O=11,12,15-THETA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2785 \"11H-14,15-EETA+H2O=11,14,15-THETA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2785 \"11H-14,15-EETA+H2O=11,14,15-THETA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2787 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_19(S)-HETE+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2787 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_19(S)-HETE+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2789 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_14,15-EET+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2789 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_14,15-EET+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2791 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_11,12-EET+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2791 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_11,12-EET+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2793 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_8,9-EET+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2793 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_8,9-EET+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2795 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_5,6-EET+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2795 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_5,6-EET+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2797 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_16(R)-HETE+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2797 \"(5Z,8Z,11Z,14Z)-Icosatetraenoic acid+Oxygen+NADPH+H+ _ST_=_LT_16(R)-HETE+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2798 \"Linoleate+Oxygen+NADPH+H+=9(10)-EpOME+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2798 \"Linoleate+Oxygen+NADPH+H+=9(10)-EpOME+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2800 \"Linoleate+Oxygen+NADPH+H+=12(13)-EpOME+NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2800 \"Linoleate+Oxygen+NADPH+H+=12(13)-EpOME+NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2808 \"Linoleate+Reduced acceptor+Oxygen _ST_=_LT_(6Z,9Z,12Z)-Octadecatrienoic acid+Acceptor+2 H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2808 \"Linoleate+Reduced acceptor+Oxygen _ST_=_LT_(6Z,9Z,12Z)-Octadecatrienoic acid+Acceptor+2 H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2824 \"Bromobenzene-3,4-oxide+H2O=Bromobenzene-3,4-dihydrodiol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2824 \"Bromobenzene-3,4-oxide+H2O=Bromobenzene-3,4-dihydrodiol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2825 \"Bromobenzene-3,4-oxide+H2O=Bromobenzene-3,4-dihydrodiol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2825 \"Bromobenzene-3,4-oxide+H2O=Bromobenzene-3,4-dihydrodiol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2826 \"Bromobenzene-3,4-oxide+H2O=Bromobenzene-3,4-dihydrodiol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2826 \"Bromobenzene-3,4-oxide+H2O=Bromobenzene-3,4-dihydrodiol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2830 \"Bromobenzene-3,4-oxide=4-Bromophenol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2830 \"Bromobenzene-3,4-oxide=4-Bromophenol\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2833 \"4-Bromophenol+NADPH+Oxygen+H+=4-Bromophenol-2,3-epoxide +NADP++H2O\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2833 \"4-Bromophenol+NADPH+Oxygen+H+=4-Bromophenol-2,3-epoxide +NADP++H2O\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2834 \"4-Bromophenol-2,3-epoxide=4-Bromocatechol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2834 \"4-Bromophenol-2,3-epoxide=4-Bromocatechol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2836 \"Bromobenzene-3,4-dihydrodiol=4-Bromocatechol (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2836 \"Bromobenzene-3,4-dihydrodiol=4-Bromocatechol (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2843 \"Benzo[a]pyrene-7,8-oxide+H2O=Benzo[a]pyrene-7,8-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2843 \"Benzo[a]pyrene-7,8-oxide+H2O=Benzo[a]pyrene-7,8-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2844 \"Benzo[a]pyrene-7,8-oxide+H2O=Benzo[a]pyrene-7,8-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2844 \"Benzo[a]pyrene-7,8-oxide+H2O=Benzo[a]pyrene-7,8-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2845 \"Benzo[a]pyrene-7,8-oxide+H2O=Benzo[a]pyrene-7,8-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2845 \"Benzo[a]pyrene-7,8-oxide+H2O=Benzo[a]pyrene-7,8-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2865 \"1,1-Dichloroethylene epoxide+Glutathione _ST_=_LT_2-(S-Glutathionyl)acetyl chloride+HCl\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2865 \"1,1-Dichloroethylene epoxide+Glutathione _ST_=_LT_2-(S-Glutathionyl)acetyl chloride+HCl\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2866 \"1,1-Dichloroethylene epoxide+Glutathione _ST_=_LT_2-(S-Glutathionyl)acetyl chloride+HCl\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2866 \"1,1-Dichloroethylene epoxide+Glutathione _ST_=_LT_2-(S-Glutathionyl)acetyl chloride+HCl\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2871 \"2-(S-Glutathionyl)acetyl chloride+Glutathione _ST_=_LT_2-(S-Glutathionyl)acetyl glutathione+HCl\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2871 \"2-(S-Glutathionyl)acetyl chloride+Glutathione _ST_=_LT_2-(S-Glutathionyl)acetyl glutathione+HCl\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2872 \"2-(S-Glutathionyl)acetyl chloride+Glutathione _ST_=_LT_2-(S-Glutathionyl)acetyl glutathione+HCl\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2872 \"2-(S-Glutathionyl)acetyl chloride+Glutathione _ST_=_LT_2-(S-Glutathionyl)acetyl glutathione+HCl\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2893 \"14,15-EET+H2O=14,15-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2893 \"14,15-EET+H2O=14,15-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2894 \"14,15-EET+H2O=14,15-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2894 \"14,15-EET+H2O=14,15-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2895 \"14,15-EET+H2O=14,15-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2895 \"14,15-EET+H2O=14,15-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2896 \"11,12-EET+H2O=11,12-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2896 \"11,12-EET+H2O=11,12-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2897 \"11,12-EET+H2O=11,12-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2897 \"11,12-EET+H2O=11,12-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2898 \"11,12-EET+H2O=11,12-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2898 \"11,12-EET+H2O=11,12-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2899 \"8,9-EET+H2O=8,9-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2899 \"8,9-EET+H2O=8,9-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2900 \"8,9-EET+H2O=8,9-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2900 \"8,9-EET+H2O=8,9-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2901 \"8,9-EET+H2O=8,9-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2901 \"8,9-EET+H2O=8,9-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2902 \"5,6-EET+H2O=5,6-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2902 \"5,6-EET+H2O=5,6-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2903 \"5,6-EET+H2O=5,6-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2903 \"5,6-EET+H2O=5,6-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2904 \"5,6-EET+H2O=5,6-DHET\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2904 \"5,6-EET+H2O=5,6-DHET\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2906 \"1,2-Dibromoethane+NADPH+Oxygen+H+=2-Bromoacetaldehyde +NADP++H2O+Hydrobromic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2906 \"1,2-Dibromoethane+NADPH+Oxygen+H+=2-Bromoacetaldehyde +NADP++H2O+Hydrobromic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2908 \"1,2-Dibromoethane+Glutathione=Glutathione episulfonium ion+2Hydrobromic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2908 \"1,2-Dibromoethane+Glutathione=Glutathione episulfonium ion+2Hydrobromic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2909 \"1,2-Dibromoethane+Glutathione=Glutathione episulfonium ion+2Hydrobromic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2909 \"1,2-Dibromoethane+Glutathione=Glutathione episulfonium ion+2Hydrobromic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2913 \"2-Bromoacetaldehyde+Glutathione=S-(Formylmethyl)glutathione +Hydrobromic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2913 \"2-Bromoacetaldehyde+Glutathione=S-(Formylmethyl)glutathione +Hydrobromic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2914 \"2-Bromoacetaldehyde+Glutathione=S-(Formylmethyl)glutathione +Hydrobromic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2914 \"2-Bromoacetaldehyde+Glutathione=S-(Formylmethyl)glutathione +Hydrobromic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2915 \"Glutathione episulfonium ion+H2O=S-(2-Hydroxyethyl)glutathione (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2915 \"Glutathione episulfonium ion+H2O=S-(2-Hydroxyethyl)glutathione (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2918 \"S-(Formylmethyl)glutathione=S-(2-Hydroxyethyl)glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2918 \"S-(Formylmethyl)glutathione=S-(2-Hydroxyethyl)glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2922 \"UDP-N-acetyl-D-glucosamine+G00157=UDP+G00162\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2922 \"UDP-N-acetyl-D-glucosamine+G00157=UDP+G00162\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2924 \"G00144+H2O=6-(alpha-D-glucosaminyl)-1D-myo-inositol +Phosphatidate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2924 \"G00144+H2O=6-(alpha-D-glucosaminyl)-1D-myo-inositol +Phosphatidate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2925 \"O(,2)+L-tyrosine+L-dopa=H(,2)O+L-dopa+Dopaquinone\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2925 \"O(,2)+L-tyrosine+L-dopa=H(,2)O+L-dopa+Dopaquinone\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2926 \"O(,2)+L-tyrosine+L-dopa=H(,2)O+L-dopa+Dopaquinone\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2926 \"O(,2)+L-tyrosine+L-dopa=H(,2)O+L-dopa+Dopaquinone\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2930 \"acetyl-CoA+D-glucosamine 1-phosphate=CoA+`N`-acetyl-D-glucosamine 1-phosphate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2930 \"acetyl-CoA+D-glucosamine 1-phosphate=CoA+`N`-acetyl-D-glucosamine 1-phosphate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2931 \"acetyl-CoA+D-glucosamine 1-phosphate=CoA+`N`-acetyl-D-glucosamine 1-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2931 \"acetyl-CoA+D-glucosamine 1-phosphate=CoA+`N`-acetyl-D-glucosamine 1-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2932 \"ATP+D-gluconate=ADP+6-phospho-D-gluconate+H('+) (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2932 \"ATP+D-gluconate=ADP+6-phospho-D-gluconate+H('+) (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2934 \"cyanide+H('+)+3-mercaptopyruvate=pyruvate+thiocyanate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2934 \"cyanide+H('+)+3-mercaptopyruvate=pyruvate+thiocyanate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2935 \"D-glucosamine 1-phosphate=D-glucosamine 6-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2935 \"D-glucosamine 1-phosphate=D-glucosamine 6-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2936 \"arachidonate+ATP+CoA=AMP+arachidonoyl-CoA+PP(,i)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2936 \"arachidonate+ATP+CoA=AMP+arachidonoyl-CoA+PP(,i)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2937 \"arachidonate+ATP+CoA=AMP+arachidonoyl-CoA+PP(,i)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2937 \"arachidonate+ATP+CoA=AMP+arachidonoyl-CoA+PP(,i)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2938 \"H(,2)O+stearoyl-CoA=CoA+H('+)+stearate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2938 \"H(,2)O+stearoyl-CoA=CoA+H('+)+stearate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2940 \"H(,2)O+stearoyl-CoA=CoA+H('+)+stearate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2940 \"H(,2)O+stearoyl-CoA=CoA+H('+)+stearate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2943 \"CoA+a-ketoglutarate+NAD('+)=CO(,2)+NADH+succinyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2943 \"CoA+a-ketoglutarate+NAD('+)=CO(,2)+NADH+succinyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2944 \"CoA+a-ketoglutarate+NAD('+)=CO(,2)+NADH+succinyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2944 \"CoA+a-ketoglutarate+NAD('+)=CO(,2)+NADH+succinyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2945 \"L-gulonate+NADP('+)=D-glucuronate+NADPH+H('+)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2945 \"L-gulonate+NADP('+)=D-glucuronate+NADPH+H('+)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2950 \"dATP+thymidine 5'-phosphate=dADP+TDP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2950 \"dATP+thymidine 5'-phosphate=dADP+TDP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_2952 \"dATP+TDP=dADP+TTP (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_2952 \"dATP+TDP=dADP+TTP (reversible)\">'\n",
      "IOPub message rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_msg_rate_limit`.\n",
      "\n",
      "Current values:\n",
      "NotebookApp.iopub_msg_rate_limit=1000.0 (msgs/sec)\n",
      "NotebookApp.rate_limit_window=3.0 (secs)\n",
      "\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4177 \"androsterone+H('+)+SO4=H(,2)O+androsterone sulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4178 \"androsterone+H('+)+SO4=H(,2)O+androsterone sulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4178 \"androsterone+H('+)+SO4=H(,2)O+androsterone sulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4179 \"androsterone+H('+)+SO4=H(,2)O+androsterone sulfate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4179 \"androsterone+H('+)+SO4=H(,2)O+androsterone sulfate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4180 \"malonyl-CoA+oleoyl-CoA+H('+)=CO(,2)+CoA+3-oxo-11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4180 \"malonyl-CoA+oleoyl-CoA+H('+)=CO(,2)+CoA+3-oxo-11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4181 \"malonyl-CoA+oleoyl-CoA+H('+)=CO(,2)+CoA+3-oxo-11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4181 \"malonyl-CoA+oleoyl-CoA+H('+)=CO(,2)+CoA+3-oxo-11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4182 \"malonyl-CoA+H('+)+11-`cis`-eicosenoyl-CoA=CO(,2)+CoA+3-oxo-13`cis`-docosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4182 \"malonyl-CoA+H('+)+11-`cis`-eicosenoyl-CoA=CO(,2)+CoA+3-oxo-13`cis`-docosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4183 \"malonyl-CoA+H('+)+11-`cis`-eicosenoyl-CoA=CO(,2)+CoA+3-oxo-13`cis`-docosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4183 \"malonyl-CoA+H('+)+11-`cis`-eicosenoyl-CoA=CO(,2)+CoA+3-oxo-13`cis`-docosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4184 \"malonyl-CoA+H('+)+`cis`-13-docosenoyl-CoA=CO(,2)+CoA+3-oxo-`cis`-15-tetracosaenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4184 \"malonyl-CoA+H('+)+`cis`-13-docosenoyl-CoA=CO(,2)+CoA+3-oxo-`cis`-15-tetracosaenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4185 \"malonyl-CoA+H('+)+`cis`-13-docosenoyl-CoA=CO(,2)+CoA+3-oxo-`cis`-15-tetracosaenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4185 \"malonyl-CoA+H('+)+`cis`-13-docosenoyl-CoA=CO(,2)+CoA+3-oxo-`cis`-15-tetracosaenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4186 \"NADPH+H('+)+3-oxo-11-`cis`-eicosenoyl-CoA=NADP('+)+3(`S`)-hydroxy-11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4186 \"NADPH+H('+)+3-oxo-11-`cis`-eicosenoyl-CoA=NADP('+)+3(`S`)-hydroxy-11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4187 \"NADPH+H('+)+3-oxo-11-`cis`-eicosenoyl-CoA=NADP('+)+3(`S`)-hydroxy-11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4187 \"NADPH+H('+)+3-oxo-11-`cis`-eicosenoyl-CoA=NADP('+)+3(`S`)-hydroxy-11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4188 \"3(`S`)-hydroxy-11-`cis`-eicosenoyl-CoA=H(,2)O+`trans`,`cis`-2,11-eicosadienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4188 \"3(`S`)-hydroxy-11-`cis`-eicosenoyl-CoA=H(,2)O+`trans`,`cis`-2,11-eicosadienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4189 \"3(`S`)-hydroxy-11-`cis`-eicosenoyl-CoA=H(,2)O+`trans`,`cis`-2,11-eicosadienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4189 \"3(`S`)-hydroxy-11-`cis`-eicosenoyl-CoA=H(,2)O+`trans`,`cis`-2,11-eicosadienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4190 \"NADPH+H('+)+`trans`,`cis`-2,11-eicosadienoyl-CoA=NADP('+)+11-`cis`-eicosenoyl-CoA\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4190 \"NADPH+H('+)+`trans`,`cis`-2,11-eicosadienoyl-CoA=NADP('+)+11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4191 \"NADPH+H('+)+`trans`,`cis`-2,11-eicosadienoyl-CoA=NADP('+)+11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4191 \"NADPH+H('+)+`trans`,`cis`-2,11-eicosadienoyl-CoA=NADP('+)+11-`cis`-eicosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4192 \"NADPH+H('+)+3-oxo-13`cis`-docosenoyl-CoA=NADP('+)+3(`S`)-hydroxy-13`cis`-docosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4192 \"NADPH+H('+)+3-oxo-13`cis`-docosenoyl-CoA=NADP('+)+3(`S`)-hydroxy-13`cis`-docosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4193 \"NADPH+H('+)+3-oxo-13`cis`-docosenoyl-CoA=NADP('+)+3(`S`)-hydroxy-13`cis`-docosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4193 \"NADPH+H('+)+3-oxo-13`cis`-docosenoyl-CoA=NADP('+)+3(`S`)-hydroxy-13`cis`-docosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4194 \"3(`S`)-hydroxy-13`cis`-docosenoyl-CoA=H(,2)O+`trans`,`cis`-2,13-docosadienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4194 \"3(`S`)-hydroxy-13`cis`-docosenoyl-CoA=H(,2)O+`trans`,`cis`-2,13-docosadienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4195 \"3(`S`)-hydroxy-13`cis`-docosenoyl-CoA=H(,2)O+`trans`,`cis`-2,13-docosadienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4195 \"3(`S`)-hydroxy-13`cis`-docosenoyl-CoA=H(,2)O+`trans`,`cis`-2,13-docosadienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4196 \"NADPH+H('+)+`trans`,`cis`-2,13-docosadienoyl-CoA=NADP('+)+`cis`-13-docosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4196 \"NADPH+H('+)+`trans`,`cis`-2,13-docosadienoyl-CoA=NADP('+)+`cis`-13-docosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4197 \"NADPH+H('+)+`trans`,`cis`-2,13-docosadienoyl-CoA=NADP('+)+`cis`-13-docosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4197 \"NADPH+H('+)+`trans`,`cis`-2,13-docosadienoyl-CoA=NADP('+)+`cis`-13-docosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4201 \"NADPH+H('+)+3-oxo-`cis`-15-tetracosaenoyl-CoA=NADP('+)+3(`S`)-hydroxy-`cis`-15-tetracosaenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4201 \"NADPH+H('+)+3-oxo-`cis`-15-tetracosaenoyl-CoA=NADP('+)+3(`S`)-hydroxy-`cis`-15-tetracosaenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4202 \"NADPH+H('+)+3-oxo-`cis`-15-tetracosaenoyl-CoA=NADP('+)+3(`S`)-hydroxy-`cis`-15-tetracosaenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4202 \"NADPH+H('+)+3-oxo-`cis`-15-tetracosaenoyl-CoA=NADP('+)+3(`S`)-hydroxy-`cis`-15-tetracosaenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4204 \"3(`S`)-hydroxy-`cis`-15-tetracosaenoyl-CoA=H(,2)O+`trans`,`cis`-2,15-tetracosadienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4204 \"3(`S`)-hydroxy-`cis`-15-tetracosaenoyl-CoA=H(,2)O+`trans`,`cis`-2,15-tetracosadienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4206 \"NADPH+H('+)+`trans`,`cis`-2,15-tetracosadienoyl-CoA=NADP('+)+`cis`-15-tetracosenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4206 \"NADPH+H('+)+`trans`,`cis`-2,15-tetracosadienoyl-CoA=NADP('+)+`cis`-15-tetracosenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4211 \"NADPH+O(,2)+H('+)+stearoyl-CoA=2 H(,2)O+NADP('+)+oleoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4211 \"NADPH+O(,2)+H('+)+stearoyl-CoA=2 H(,2)O+NADP('+)+oleoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4213 \"malonyl-CoA+palmitoleoyl-CoA+H('+)=CO(,2)+CoA+3-oxo-`cis`-9-octadecenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4213 \"malonyl-CoA+palmitoleoyl-CoA+H('+)=CO(,2)+CoA+3-oxo-`cis`-9-octadecenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4215 \"NADPH+H('+)+3-oxo-`cis`-9-octadecenoyl-CoA=NADP('+)+3(`S`)-hydroxy-`cis`-9-octadecenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4215 \"NADPH+H('+)+3-oxo-`cis`-9-octadecenoyl-CoA=NADP('+)+3(`S`)-hydroxy-`cis`-9-octadecenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4217 \"3(`S`)-hydroxy-`cis`-9-octadecenoyl-CoA=H(,2)O+`trans`,`cis`-2,9-octadecadienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4217 \"3(`S`)-hydroxy-`cis`-9-octadecenoyl-CoA=H(,2)O+`trans`,`cis`-2,9-octadecadienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4219 \"NADPH+H('+)+`trans`,`cis`-2,9-octadecadienoyl-CoA=NADP('+)+oleoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4219 \"NADPH+H('+)+`trans`,`cis`-2,9-octadecadienoyl-CoA=NADP('+)+oleoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4222 \"palmitoyl-CoA+NADPH+O(,2)+H('+)=2 H(,2)O+NADP('+)+palmitoleoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4222 \"palmitoyl-CoA+NADPH+O(,2)+H('+)=2 H(,2)O+NADP('+)+palmitoleoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4225 \"H(,2)O+24-`trans`-trihydroxycoprostenoyl-CoA=24(`R`),25(`R`)-varanoyl-CoA (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4225 \"H(,2)O+24-`trans`-trihydroxycoprostenoyl-CoA=24(`R`),25(`R`)-varanoyl-CoA (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4226 \"H(,2)O+24-`trans`-trihydroxycoprostenoyl-CoA=24(`R`),25(`R`)-varanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4226 \"H(,2)O+24-`trans`-trihydroxycoprostenoyl-CoA=24(`R`),25(`R`)-varanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4227 \"CoA+24-oxo-25(`R`)-trihydroxycoprostanoyl-CoA=propionyl-CoA+choloyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4227 \"CoA+24-oxo-25(`R`)-trihydroxycoprostanoyl-CoA=propionyl-CoA+choloyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4229 \"NAD('+)+24(`R`),25(`R`)-varanoyl-CoA=NADH+H('+)+24-oxo-25(`R`)-trihydroxycoprostanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4229 \"NAD('+)+24(`R`),25(`R`)-varanoyl-CoA=NADH+H('+)+24-oxo-25(`R`)-trihydroxycoprostanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4230 \"NAD('+)+24(`R`),25(`R`)-varanoyl-CoA=NADH+H('+)+24-oxo-25(`R`)-trihydroxycoprostanoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4230 \"NAD('+)+24(`R`),25(`R`)-varanoyl-CoA=NADH+H('+)+24-oxo-25(`R`)-trihydroxycoprostanoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4240 \"H(,2)O+12,13-epoxyoctadec-9(`Z`)-enoate=12,13-hydroxyoctadec-9(`Z`)-enoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4240 \"H(,2)O+12,13-epoxyoctadec-9(`Z`)-enoate=12,13-hydroxyoctadec-9(`Z`)-enoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4242 \"H(,2)O+9,10-epoxyoctadec-12(`Z`)-enoate=9,10-hydroxyoctadec-12(`Z`)-enoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4242 \"H(,2)O+9,10-epoxyoctadec-12(`Z`)-enoate=9,10-hydroxyoctadec-12(`Z`)-enoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4255 \"CDP-ethanolamine+1,2-diacyl-`sn`-glycerol=CMP+phosphatidylethanolamine+H('+) (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4255 \"CDP-ethanolamine+1,2-diacyl-`sn`-glycerol=CMP+phosphatidylethanolamine+H('+) (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4256 \"CDP-ethanolamine+1,2-diacyl-`sn`-glycerol=CMP+phosphatidylethanolamine+H('+)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4256 \"CDP-ethanolamine+1,2-diacyl-`sn`-glycerol=CMP+phosphatidylethanolamine+H('+)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4262 \"H(,2)O+phosphatidylinositol 4-phosphate=L-a-Phosphatidylinositol+Pi\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4262 \"H(,2)O+phosphatidylinositol 4-phosphate=L-a-Phosphatidylinositol+Pi\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4263 \"H(,2)O+L-a-Phosphatidylinositol=`myo`-inositol+phosphatidate+H('+)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4263 \"H(,2)O+L-a-Phosphatidylinositol=`myo`-inositol+phosphatidate+H('+)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4264 \"H(,2)O+L-a-Phosphatidylinositol=`myo`-inositol+phosphatidate+H('+)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4264 \"H(,2)O+L-a-Phosphatidylinositol=`myo`-inositol+phosphatidate+H('+)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4265 \"H(,2)O+L-a-Phosphatidylinositol=`myo`-inositol+phosphatidate+H('+)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4265 \"H(,2)O+L-a-Phosphatidylinositol=`myo`-inositol+phosphatidate+H('+)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4272 \"arachidonate+FADH(,2)+O(,2)=FAD+H(,2)O+5,6-epoxy-(8Z,11Z,14Z)-eicosatrienoic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4272 \"arachidonate+FADH(,2)+O(,2)=FAD+H(,2)O+5,6-epoxy-(8Z,11Z,14Z)-eicosatrienoic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4274 \"arachidonate+FADH(,2)+O(,2)=FAD+H(,2)O+11,12-epoxy-(5`Z`,8`Z`,14`Z`)-eicosatrienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4274 \"arachidonate+FADH(,2)+O(,2)=FAD+H(,2)O+11,12-epoxy-(5`Z`,8`Z`,14`Z`)-eicosatrienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4276 \"arachidonate+FADH(,2)+O(,2)=FAD+H(,2)O+8,9-epoxy-(5Z,11Z,14Z)-eicosatrienoic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4276 \"arachidonate+FADH(,2)+O(,2)=FAD+H(,2)O+8,9-epoxy-(5Z,11Z,14Z)-eicosatrienoic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4278 \"arachidonate+FADH(,2)+O(,2)=FAD+H(,2)O+14,15-epoxy-(5`Z`,8`Z`,11`Z`)-eicosatrienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4278 \"arachidonate+FADH(,2)+O(,2)=FAD+H(,2)O+14,15-epoxy-(5`Z`,8`Z`,11`Z`)-eicosatrienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4281 \"phosphatidylethanolamine+L-serine=ethanolamine+L-a-Phosphatidylserine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4281 \"phosphatidylethanolamine+L-serine=ethanolamine+L-a-Phosphatidylserine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4282 \"phosphatidylcholine+L-serine=choline+L-a-Phosphatidylserine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4282 \"phosphatidylcholine+L-serine=choline+L-a-Phosphatidylserine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4283 \"H(,2)O+L-a-Phosphatidylserine=phosphatidate+H('+)+L-serine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4283 \"H(,2)O+L-a-Phosphatidylserine=phosphatidate+H('+)+L-serine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4284 \"H(,2)O+L-a-Phosphatidylserine=phosphatidate+H('+)+L-serine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4284 \"H(,2)O+L-a-Phosphatidylserine=phosphatidate+H('+)+L-serine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4285 \"H(,2)O+L-a-Phosphatidylserine=phosphatidate+H('+)+L-serine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4285 \"H(,2)O+L-a-Phosphatidylserine=phosphatidate+H('+)+L-serine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4290 \"H(,2)O+1-acyl-2-lyso-phosphatidylserine=H('+)+glycerophosphoserine+carboxylate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4290 \"H(,2)O+1-acyl-2-lyso-phosphatidylserine=H('+)+glycerophosphoserine+carboxylate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4291 \"H(,2)O+1-acyl-2-lyso-phosphatidylserine=H('+)+glycerophosphoserine+carboxylate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4291 \"H(,2)O+1-acyl-2-lyso-phosphatidylserine=H('+)+glycerophosphoserine+carboxylate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4292 \"H(,2)O+1-acyl-2-lyso-phosphatidylserine=H('+)+glycerophosphoserine+carboxylate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4292 \"H(,2)O+1-acyl-2-lyso-phosphatidylserine=H('+)+glycerophosphoserine+carboxylate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4293 \"H('+)+1-acyl-2-lyso-phosphatidylserine+carboxylate=H(,2)O+L-a-Phosphatidylserine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4293 \"H('+)+1-acyl-2-lyso-phosphatidylserine+carboxylate=H(,2)O+L-a-Phosphatidylserine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4294 \"H('+)+1-acyl-2-lyso-phosphatidylserine+carboxylate=H(,2)O+L-a-Phosphatidylserine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4294 \"H('+)+1-acyl-2-lyso-phosphatidylserine+carboxylate=H(,2)O+L-a-Phosphatidylserine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4295 \"H('+)+1-acyl-2-lyso-phosphatidylserine+carboxylate=H(,2)O+L-a-Phosphatidylserine\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4295 \"H('+)+1-acyl-2-lyso-phosphatidylserine+carboxylate=H(,2)O+L-a-Phosphatidylserine\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4297 \"NADP('+)+6-`trans`-leukotriene B(,4)=NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4) (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4297 \"NADP('+)+6-`trans`-leukotriene B(,4)=NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4) (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4298 \"NADP('+)+6-`trans`-leukotriene B(,4)=NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4) (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4298 \"NADP('+)+6-`trans`-leukotriene B(,4)=NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4) (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4299 \"NADP('+)+6-`trans`-leukotriene B(,4)=NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4) (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4299 \"NADP('+)+6-`trans`-leukotriene B(,4)=NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4) (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4300 \"NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4)=NADP('+)+6,7-dihydro-5-oxo-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4300 \"NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4)=NADP('+)+6,7-dihydro-5-oxo-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4301 \"NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4)=NADP('+)+6,7-dihydro-5-oxo-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4301 \"NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4)=NADP('+)+6,7-dihydro-5-oxo-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4302 \"NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4)=NADP('+)+6,7-dihydro-5-oxo-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4302 \"NADPH+H('+)+5-oxo-6-`trans`-leukotriene B(,4)=NADP('+)+6,7-dihydro-5-oxo-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4304 \"NADPH+H('+)+6,7-dihydro-5-oxo-leukotriene B(,4)=NADP('+)+6,7-dihydro-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4304 \"NADPH+H('+)+6,7-dihydro-5-oxo-leukotriene B(,4)=NADP('+)+6,7-dihydro-leukotriene B(,4)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4305 \"NADPH+H('+)+6,7-dihydro-5-oxo-leukotriene B(,4)=NADP('+)+6,7-dihydro-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4305 \"NADPH+H('+)+6,7-dihydro-5-oxo-leukotriene B(,4)=NADP('+)+6,7-dihydro-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4306 \"NADPH+H('+)+6,7-dihydro-5-oxo-leukotriene B(,4)=NADP('+)+6,7-dihydro-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4306 \"NADPH+H('+)+6,7-dihydro-5-oxo-leukotriene B(,4)=NADP('+)+6,7-dihydro-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4308 \"NADP('+)+6,7-dihydro-leukotriene B(,4)=NADPH+H('+)+6,7-dihydro-5-oxo-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4308 \"NADP('+)+6,7-dihydro-leukotriene B(,4)=NADPH+H('+)+6,7-dihydro-5-oxo-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4311 \"NADPH+H('+)+5-oxo-6`E`-12-`epi`-LTB(,4)=NADP('+)+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4311 \"NADPH+H('+)+5-oxo-6`E`-12-`epi`-LTB(,4)=NADP('+)+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4313 \"NADPH+H('+)+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)=NADP('+)+6,7-dihydro-12-`epi`-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4313 \"NADPH+H('+)+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)=NADP('+)+6,7-dihydro-12-`epi`-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4320 \"NADP('+)+6,7-dihydro-12-`epi`-LTB(,4)=NADPH+H('+)+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4320 \"NADP('+)+6,7-dihydro-12-`epi`-LTB(,4)=NADPH+H('+)+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4321 \"GDP+reduced glutaredoxin=dGDP+H(,2)O+glutaredoxin (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4321 \"GDP+reduced glutaredoxin=dGDP+H(,2)O+glutaredoxin (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4322 \"GDP+reduced glutaredoxin=dGDP+H(,2)O+glutaredoxin\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4322 \"GDP+reduced glutaredoxin=dGDP+H(,2)O+glutaredoxin\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4323 \"isobutyryl-CoA+D-Dihydro-6,8-dithiooctamide=CoA+6-`S`-(2-methylpropanoyl)-D-dihydro-6,8-dithiooctamide\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4323 \"isobutyryl-CoA+D-Dihydro-6,8-dithiooctamide=CoA+6-`S`-(2-methylpropanoyl)-D-dihydro-6,8-dithiooctamide\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4327 \"2 glutathione+9-HpODE=H(,2)O+Oxidized glutathione+9-hydroxyoctadecadienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4327 \"2 glutathione+9-HpODE=H(,2)O+Oxidized glutathione+9-hydroxyoctadecadienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4330 \"reduced acceptor+9-HpODE=acceptor+H(,2)O+9-hydroxyoctadecadienoate (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4330 \"reduced acceptor+9-HpODE=acceptor+H(,2)O+9-hydroxyoctadecadienoate (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4331 \"O(,2)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=H(,2)O(,2)+5-oxo-12(`R`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4331 \"O(,2)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=H(,2)O(,2)+5-oxo-12(`R`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4332 \"O(,2)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=H(,2)O(,2)+5-oxo-12(`R`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4332 \"O(,2)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=H(,2)O(,2)+5-oxo-12(`R`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4333 \"ATP+CoA+6,7-dihydro-5-oxo-leukotriene B(,4)=AMP+PP(,i)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4333 \"ATP+CoA+6,7-dihydro-5-oxo-leukotriene B(,4)=AMP+PP(,i)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4334 \"ATP+CoA+6,7-dihydro-5-oxo-leukotriene B(,4)=AMP+PP(,i)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4334 \"ATP+CoA+6,7-dihydro-5-oxo-leukotriene B(,4)=AMP+PP(,i)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4335 \"ATP+CoA+6,7-dihydro-5-oxo-leukotriene B(,4)=AMP+PP(,i)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4335 \"ATP+CoA+6,7-dihydro-5-oxo-leukotriene B(,4)=AMP+PP(,i)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4336 \"ATP+CoA+6,7-dihydro-5-oxo-leukotriene B(,4)=AMP+PP(,i)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4336 \"ATP+CoA+6,7-dihydro-5-oxo-leukotriene B(,4)=AMP+PP(,i)+5-oxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4337 \"CoA+3,5-dioxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=acetyl-CoA+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4337 \"CoA+3,5-dioxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=acetyl-CoA+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4338 \"CoA+3,5-dioxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=acetyl-CoA+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4338 \"CoA+3,5-dioxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=acetyl-CoA+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4339 \"H(,2)O+5-oxo-12(`R`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA=3(`S`),12(`R`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4339 \"H(,2)O+5-oxo-12(`R`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA=3(`S`),12(`R`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4340 \"H(,2)O+5-oxo-12(`R`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA=3(`S`),12(`R`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4340 \"H(,2)O+5-oxo-12(`R`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA=3(`S`),12(`R`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4341 \"NAD('+)+3(`S`),12(`R`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4341 \"NAD('+)+3(`S`),12(`R`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4342 \"NAD('+)+3(`S`),12(`R`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4342 \"NAD('+)+3(`S`),12(`R`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4343 \"NAD('+)+3(`S`),12(`R`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4343 \"NAD('+)+3(`S`),12(`R`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`R`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4344 \"H(,2)O+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA=CoA+H('+)+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4344 \"H(,2)O+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA=CoA+H('+)+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4345 \"H(,2)O+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA=CoA+H('+)+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4345 \"H(,2)O+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA=CoA+H('+)+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4346 \"H(,2)O+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA=CoA+H('+)+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4346 \"H(,2)O+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA=CoA+H('+)+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4347 \"NAD('+)+3(`S`),12(`S`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4347 \"NAD('+)+3(`S`),12(`S`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4348 \"NAD('+)+3(`S`),12(`S`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4348 \"NAD('+)+3(`S`),12(`S`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4349 \"NAD('+)+3(`S`),12(`S`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4349 \"NAD('+)+3(`S`),12(`S`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=NADH+H('+)+3,5-dioxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4352 \"CoA+3,5-dioxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=acetyl-CoA+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4352 \"CoA+3,5-dioxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=acetyl-CoA+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4353 \"CoA+3,5-dioxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=acetyl-CoA+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4353 \"CoA+3,5-dioxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA=acetyl-CoA+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4354 \"H(,2)O+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA=CoA+H('+)+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4354 \"H(,2)O+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA=CoA+H('+)+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4356 \"H(,2)O+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA=CoA+H('+)+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4356 \"H(,2)O+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoyl-CoA=CoA+H('+)+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4357 \"H(,2)O+5-oxo-12(`S`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA=3(`S`),12(`S`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4357 \"H(,2)O+5-oxo-12(`S`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA=3(`S`),12(`S`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4358 \"H(,2)O+5-oxo-12(`S`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA=3(`S`),12(`S`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4358 \"H(,2)O+5-oxo-12(`S`)-hydroxy-eicosa-2`E`,8`E`,10`E`,14`Z`-tetraenoyl-CoA=3(`S`),12(`S`)-dihydroxy-5-oxo-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4359 \"ATP+CoA+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)=AMP+PP(,i)+5-oxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4359 \"ATP+CoA+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)=AMP+PP(,i)+5-oxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4360 \"ATP+CoA+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)=AMP+PP(,i)+5-oxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4360 \"ATP+CoA+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)=AMP+PP(,i)+5-oxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4361 \"ATP+CoA+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)=AMP+PP(,i)+5-oxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4361 \"ATP+CoA+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)=AMP+PP(,i)+5-oxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4362 \"ATP+CoA+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)=AMP+PP(,i)+5-oxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4362 \"ATP+CoA+6,7-dihydro-5-oxo-12-`epi`-LTB(,4)=AMP+PP(,i)+5-oxo-12(`S`)-hydroxy-eicosa-8`E`,10`E`,14`Z`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4363 \"NADH+2 H('+)+3a,7a,12a-trihydroxy-5b-cholestanoic acid anion=H(,2)O+NAD('+)+3a,7a,12a-trihydroxy-5b-cholestan-27-al\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4363 \"NADH+2 H('+)+3a,7a,12a-trihydroxy-5b-cholestanoic acid anion=H(,2)O+NAD('+)+3a,7a,12a-trihydroxy-5b-cholestan-27-al\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4364 \"NADH+2 H('+)+3a,7a,12a-trihydroxy-5b-cholestanoic acid anion=H(,2)O+NAD('+)+3a,7a,12a-trihydroxy-5b-cholestan-27-al\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4364 \"NADH+2 H('+)+3a,7a,12a-trihydroxy-5b-cholestanoic acid anion=H(,2)O+NAD('+)+3a,7a,12a-trihydroxy-5b-cholestan-27-al\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4365 \"NADH+2 H('+)+3a,7a,12a-trihydroxy-5b-cholestanoic acid anion=H(,2)O+NAD('+)+3a,7a,12a-trihydroxy-5b-cholestan-27-al\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4365 \"NADH+2 H('+)+3a,7a,12a-trihydroxy-5b-cholestanoic acid anion=H(,2)O+NAD('+)+3a,7a,12a-trihydroxy-5b-cholestan-27-al\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4372 \"H(,2)O+O(,2)+spermidine monoaldehyde 1=H(,2)O(,2)+NH(,4)('+)+spermidine dialdehyde\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4372 \"H(,2)O+O(,2)+spermidine monoaldehyde 1=H(,2)O(,2)+NH(,4)('+)+spermidine dialdehyde\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4374 \"2 H(,2)O+2 NAD('+)+spermine dialdehyde=2 NADH+4 H('+)+spermic acid 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4374 \"2 H(,2)O+2 NAD('+)+spermine dialdehyde=2 NADH+4 H('+)+spermic acid 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4375 \"2 H(,2)O+2 NAD('+)+spermine dialdehyde=2 NADH+4 H('+)+spermic acid 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4375 \"2 H(,2)O+2 NAD('+)+spermine dialdehyde=2 NADH+4 H('+)+spermic acid 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4376 \"2 H(,2)O+2 NAD('+)+spermine dialdehyde=2 NADH+4 H('+)+spermic acid 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4376 \"2 H(,2)O+2 NAD('+)+spermine dialdehyde=2 NADH+4 H('+)+spermic acid 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4377 \"2 H(,2)O+2 NAD('+)+spermidine dialdehyde=2 NADH+4 H('+)+spermidic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4377 \"2 H(,2)O+2 NAD('+)+spermidine dialdehyde=2 NADH+4 H('+)+spermidic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4378 \"2 H(,2)O+2 NAD('+)+spermidine dialdehyde=2 NADH+4 H('+)+spermidic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4378 \"2 H(,2)O+2 NAD('+)+spermidine dialdehyde=2 NADH+4 H('+)+spermidic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4379 \"2 H(,2)O+2 NAD('+)+spermidine dialdehyde=2 NADH+4 H('+)+spermidic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4379 \"2 H(,2)O+2 NAD('+)+spermidine dialdehyde=2 NADH+4 H('+)+spermidic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4381 \"2 NADPH+2 O(,2)+H('+)+13'-hydroxy-a-tocotrienol=3 H(,2)O+2 NADP('+)+13'-carboxy-a-tocotrienol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4381 \"2 NADPH+2 O(,2)+H('+)+13'-hydroxy-a-tocotrienol=3 H(,2)O+2 NADP('+)+13'-carboxy-a-tocotrienol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4382 \"dimethylallyl pyrophosphate+farnesyl pyrophosphate=PP(,i)+`trans,trans,cis`-geranyl-geranyl-PP\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4382 \"dimethylallyl pyrophosphate+farnesyl pyrophosphate=PP(,i)+`trans,trans,cis`-geranyl-geranyl-PP\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4383 \"H(,2)O+1-alkyl-2-lyso-`sn`-glycero-3-phosphocholine=choline+H('+)+1-alkyl-2-lyso-`sn`-glycero-3-phosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4383 \"H(,2)O+1-alkyl-2-lyso-`sn`-glycero-3-phosphocholine=choline+H('+)+1-alkyl-2-lyso-`sn`-glycero-3-phosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4389 \"CoA+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate=acetyl-CoA+8(`R`)-hydroxy-hexadeca-4`E`,6`E`,10`Z`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4389 \"CoA+3-oxo-10(`R`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate=acetyl-CoA+8(`R`)-hydroxy-hexadeca-4`E`,6`E`,10`Z`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4390 \"CoA+3-oxo-8(`R`)-hydroxy-hexadeca-6`E`,10`Z`-dienoate=acetyl-CoA+6(`R`)-hydroxy-tetradeca-4`E`,8`Z`-dienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4390 \"CoA+3-oxo-8(`R`)-hydroxy-hexadeca-6`E`,10`Z`-dienoate=acetyl-CoA+6(`R`)-hydroxy-tetradeca-4`E`,8`Z`-dienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4391 \"O(,2)+8(`R`)-hydroxy-hexadeca-4`E`,6`E`,10`Z`-trienoate=H(,2)O(,2)+8(`R`)-hydroxy-hexadeca-2`E`,4`E`,6`E`,10`Z`-tetraenoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4391 \"O(,2)+8(`R`)-hydroxy-hexadeca-4`E`,6`E`,10`Z`-trienoate=H(,2)O(,2)+8(`R`)-hydroxy-hexadeca-2`E`,4`E`,6`E`,10`Z`-tetraenoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4392 \"H(,2)O+8(`R`)-hydroxy-hexadeca-2`E`,6`E`,10`Z`-trienoate=3(`S`),8(`R`)-dihydroxy-6`E`,10`Z`-hexadecadienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4392 \"H(,2)O+8(`R`)-hydroxy-hexadeca-2`E`,6`E`,10`Z`-trienoate=3(`S`),8(`R`)-dihydroxy-6`E`,10`Z`-hexadecadienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4393 \"NAD('+)+3(`S`),8(`R`)-dihydroxy-6`E`,10`Z`-hexadecadienoate=NADH+H('+)+3-oxo-8(`R`)-hydroxy-hexadeca-6`E`,10`Z`-dienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4393 \"NAD('+)+3(`S`),8(`R`)-dihydroxy-6`E`,10`Z`-hexadecadienoate=NADH+H('+)+3-oxo-8(`R`)-hydroxy-hexadeca-6`E`,10`Z`-dienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4394 \"CoA+3-oxo-6(`R`)-hydroxy-tetradec-8-`cis`-enoate=acetyl-CoA+4(`R`)-hydroxy-dodec-6`Z`-enoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4394 \"CoA+3-oxo-6(`R`)-hydroxy-tetradec-8-`cis`-enoate=acetyl-CoA+4(`R`)-hydroxy-dodec-6`Z`-enoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4395 \"O(,2)+6(`R`)-hydroxy-tetradeca-4`E`,8`Z`-dienoate=H(,2)O(,2)+6(`R`)-hydroxy-tetradeca-2`E`,4`E`,8`Z`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4395 \"O(,2)+6(`R`)-hydroxy-tetradeca-4`E`,8`Z`-dienoate=H(,2)O(,2)+6(`R`)-hydroxy-tetradeca-2`E`,4`E`,8`Z`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4396 \"H(,2)O+6(`R`)-hydroxy-tetradeca-2`E`,8`Z`-dienoate=3(`S`),6(`R`)-dihydroxy-tetradec-8`Z`-enoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4396 \"H(,2)O+6(`R`)-hydroxy-tetradeca-2`E`,8`Z`-dienoate=3(`S`),6(`R`)-dihydroxy-tetradec-8`Z`-enoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4397 \"NADPH+H('+)+8(`R`)-hydroxy-hexadeca-2`E`,4`E`,6`E`,10`Z`-tetraenoate=NADP('+)+8(`R`)-hydroxy-hexadeca-2`E`,6`E`,10`Z`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4397 \"NADPH+H('+)+8(`R`)-hydroxy-hexadeca-2`E`,4`E`,6`E`,10`Z`-tetraenoate=NADP('+)+8(`R`)-hydroxy-hexadeca-2`E`,6`E`,10`Z`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4398 \"NADPH+H('+)+6(`R`)-hydroxy-tetradeca-2`E`,4`E`,8`Z`-trienoate=NADP('+)+6(`R`)-hydroxy-tetradeca-2`E`,8`Z`-dienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4398 \"NADPH+H('+)+6(`R`)-hydroxy-tetradeca-2`E`,4`E`,8`Z`-trienoate=NADP('+)+6(`R`)-hydroxy-tetradeca-2`E`,8`Z`-dienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4399 \"NAD('+)+3(`S`),6(`R`)-dihydroxy-tetradec-8`Z`-enoate=NADH+H('+)+3-oxo-6(`R`)-hydroxy-tetradec-8-`cis`-enoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4399 \"NAD('+)+3(`S`),6(`R`)-dihydroxy-tetradec-8`Z`-enoate=NADH+H('+)+3-oxo-6(`R`)-hydroxy-tetradec-8-`cis`-enoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4400 \"CoA+3-oxo-8(`S`)-hydroxy-hexadeca-6`E`,10`Z`-dienoate=acetyl-CoA+6(`S`)-hydroxy-tetradeca-4`E`,8`Z`-dienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4400 \"CoA+3-oxo-8(`S`)-hydroxy-hexadeca-6`E`,10`Z`-dienoate=acetyl-CoA+6(`S`)-hydroxy-tetradeca-4`E`,8`Z`-dienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4402 \"CoA+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate=acetyl-CoA+8(`S`)-hydroxy-hexadeca-4`E`,6`E`-10`Z`-trienoate\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4402 \"CoA+3-oxo-10(`S`)-hydroxy-octadeca-6`E`,8`E`,12`Z`-trienoate=acetyl-CoA+8(`S`)-hydroxy-hexadeca-4`E`,6`E`-10`Z`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4404 \"H(,2)O+8(`S`)-hydroxy-hexadeca-2`E`,6`E`,10`Z`-trienoate=3(`S`),8(`S`)-dihydroxy-6`E`,10`Z`-hexadecadienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4404 \"H(,2)O+8(`S`)-hydroxy-hexadeca-2`E`,6`E`,10`Z`-trienoate=3(`S`),8(`S`)-dihydroxy-6`E`,10`Z`-hexadecadienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4405 \"NAD('+)+3(`S`),8(`S`)-dihydroxy-6`E`,10`Z`-hexadecadienoate=NADH+H('+)+3-oxo-8(`S`)-hydroxy-hexadeca-6`E`,10`Z`-dienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4405 \"NAD('+)+3(`S`),8(`S`)-dihydroxy-6`E`,10`Z`-hexadecadienoate=NADH+H('+)+3-oxo-8(`S`)-hydroxy-hexadeca-6`E`,10`Z`-dienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4407 \"H(,2)O+6(`S`)-hydroxy-tetradeca-2`E`,8`Z`-dienoate=3(`S`),6(`S`)-dihydroxy-tetradec-8`Z`-enoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4407 \"H(,2)O+6(`S`)-hydroxy-tetradeca-2`E`,8`Z`-dienoate=3(`S`),6(`S`)-dihydroxy-tetradec-8`Z`-enoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4408 \"NAD('+)+3(`S`),6(`S`)-dihydroxy-tetradec-8`Z`-enoate=NADH+H('+)+3-oxo-6(`S`)-hydroxy-tetradec-8`Z`-enoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4408 \"NAD('+)+3(`S`),6(`S`)-dihydroxy-tetradec-8`Z`-enoate=NADH+H('+)+3-oxo-6(`S`)-hydroxy-tetradec-8`Z`-enoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4409 \"NADPH+H('+)+8(`S`)-hydroxy-hexadeca-2`E`,4`E`,6`E`,10`Z`-tetraenoate=NADP('+)+8(`S`)-hydroxy-hexadeca-2`E`,6`E`,10`Z`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4409 \"NADPH+H('+)+8(`S`)-hydroxy-hexadeca-2`E`,4`E`,6`E`,10`Z`-tetraenoate=NADP('+)+8(`S`)-hydroxy-hexadeca-2`E`,6`E`,10`Z`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4410 \"NADPH+H('+)+6(`S`)-hydroxy-tetradeca-2`E`,4`E`,8`Z`-trienoate=NADP('+)+6(`S`)-hydroxy-tetradeca-2`E`,8`Z`-dienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4410 \"NADPH+H('+)+6(`S`)-hydroxy-tetradeca-2`E`,4`E`,8`Z`-trienoate=NADP('+)+6(`S`)-hydroxy-tetradeca-2`E`,8`Z`-dienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4416 \"H(,2)O+13(`S`)-hydroperoxy-(9`E`,11`E`)-octadecadienoate=nonanate+4-hydroperoxy-2-nonenal\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4416 \"H(,2)O+13(`S`)-hydroperoxy-(9`E`,11`E`)-octadecadienoate=nonanate+4-hydroperoxy-2-nonenal\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4423 \"2 glutathione+13(`S`)-hydroperoxy-(9`E`,11`E`)-octadecadienoate=H(,2)O+Oxidized glutathione+13-hydroxy-(9`Z`,11`E`)-octadecadienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4423 \"2 glutathione+13(`S`)-hydroperoxy-(9`E`,11`E`)-octadecadienoate=H(,2)O+Oxidized glutathione+13-hydroxy-(9`Z`,11`E`)-octadecadienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4427 \"prostaglandin J(,2)=D12-Prostaglandin J(,2) (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4427 \"prostaglandin J(,2)=D12-Prostaglandin J(,2) (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4428 \"prostaglandin J(,2)=D12-Prostaglandin J(,2)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4428 \"prostaglandin J(,2)=D12-Prostaglandin J(,2)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4437 \"NADH+H('+)+12-oxo-leukotriene B(,4)=NAD('+)+10,11-dihydro-12-oxo-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4437 \"NADH+H('+)+12-oxo-leukotriene B(,4)=NAD('+)+10,11-dihydro-12-oxo-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4441 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4441 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4442 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4442 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4443 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4443 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4444 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-12-epi-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4444 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-12-epi-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4445 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-12-epi-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4445 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-12-epi-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4446 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-12-epi-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4446 \"NADPH+H('+)+10,11-dihydro-12-oxo-LTB(,4)=NADP('+)+10,11-dihydro-12-epi-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4447 \"NADPH+O(,2)+H('+)+10,11-dihydro-leukotriene B(,4)=H(,2)O+NADP('+)+20-OH-10,11-dihydro-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4447 \"NADPH+O(,2)+H('+)+10,11-dihydro-leukotriene B(,4)=H(,2)O+NADP('+)+20-OH-10,11-dihydro-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4448 \"NADPH+O(,2)+H('+)+10,11-dihydro-leukotriene B(,4)=H(,2)O+NADP('+)+20-OH-10,11-dihydro-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4448 \"NADPH+O(,2)+H('+)+10,11-dihydro-leukotriene B(,4)=H(,2)O+NADP('+)+20-OH-10,11-dihydro-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4449 \"NADPH+O(,2)+H('+)+10,11-dihydro-20-dihydroxy-LTB(,4)=H(,2)O+NADP('+)+10,11-dihydro-20-trihydroxy-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4449 \"NADPH+O(,2)+H('+)+10,11-dihydro-20-dihydroxy-LTB(,4)=H(,2)O+NADP('+)+10,11-dihydro-20-trihydroxy-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4450 \"NADPH+O(,2)+H('+)+10,11-dihydro-20-dihydroxy-LTB(,4)=H(,2)O+NADP('+)+10,11-dihydro-20-trihydroxy-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4450 \"NADPH+O(,2)+H('+)+10,11-dihydro-20-dihydroxy-LTB(,4)=H(,2)O+NADP('+)+10,11-dihydro-20-trihydroxy-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4451 \"NADPH+O(,2)+H('+)+20-OH-10,11-dihydro-leukotriene B(,4)=H(,2)O+NADP('+)+10,11-dihydro-20-dihydroxy-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4451 \"NADPH+O(,2)+H('+)+20-OH-10,11-dihydro-leukotriene B(,4)=H(,2)O+NADP('+)+10,11-dihydro-20-dihydroxy-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4452 \"NADPH+O(,2)+H('+)+20-OH-10,11-dihydro-leukotriene B(,4)=H(,2)O+NADP('+)+10,11-dihydro-20-dihydroxy-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4452 \"NADPH+O(,2)+H('+)+20-OH-10,11-dihydro-leukotriene B(,4)=H(,2)O+NADP('+)+10,11-dihydro-20-dihydroxy-LTB(,4)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4453 \"10,11-dihydro-20-trihydroxy-leukotriene B(,4)=H(,2)O+H('+)+20-COOH-10,11-dihydro-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4453 \"10,11-dihydro-20-trihydroxy-leukotriene B(,4)=H(,2)O+H('+)+20-COOH-10,11-dihydro-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4460 \"CoA+3-oxo-5(`S`),12(`R`)-dihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA=acetyl-CoA+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4460 \"CoA+3-oxo-5(`S`),12(`R`)-dihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA=acetyl-CoA+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4461 \"CoA+3-oxo-5(`S`),12(`R`)-dihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA=acetyl-CoA+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4461 \"CoA+3-oxo-5(`S`),12(`R`)-dihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA=acetyl-CoA+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4462 \"H(,2)O+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoyl-CoA=CoA+H('+)+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4462 \"H(,2)O+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoyl-CoA=CoA+H('+)+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4464 \"H(,2)O+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoyl-CoA=CoA+H('+)+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4464 \"H(,2)O+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoyl-CoA=CoA+H('+)+3(`S`),10(`R`)-OH-octadeca-6-`trans`-4,12-`cis`-trienoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4465 \"O(,2)+10,11-dihydro-LTB(,4)-CoA=H(,2)O(,2)+5(`S`),12(`R`)-dihydroxy-eicosa-2,8-`trans`-6,14-`cis`-tetraenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4465 \"O(,2)+10,11-dihydro-LTB(,4)-CoA=H(,2)O(,2)+5(`S`),12(`R`)-dihydroxy-eicosa-2,8-`trans`-6,14-`cis`-tetraenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4466 \"O(,2)+10,11-dihydro-LTB(,4)-CoA=H(,2)O(,2)+5(`S`),12(`R`)-dihydroxy-eicosa-2,8-`trans`-6,14-`cis`-tetraenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4466 \"O(,2)+10,11-dihydro-LTB(,4)-CoA=H(,2)O(,2)+5(`S`),12(`R`)-dihydroxy-eicosa-2,8-`trans`-6,14-`cis`-tetraenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4467 \"ATP+CoA+10,11-dihydro-leukotriene B(,4)=AMP+PP(,i)+10,11-dihydro-LTB(,4)-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4467 \"ATP+CoA+10,11-dihydro-leukotriene B(,4)=AMP+PP(,i)+10,11-dihydro-LTB(,4)-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4468 \"ATP+CoA+10,11-dihydro-leukotriene B(,4)=AMP+PP(,i)+10,11-dihydro-LTB(,4)-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4468 \"ATP+CoA+10,11-dihydro-leukotriene B(,4)=AMP+PP(,i)+10,11-dihydro-LTB(,4)-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4469 \"ATP+CoA+10,11-dihydro-leukotriene B(,4)=AMP+PP(,i)+10,11-dihydro-LTB(,4)-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4469 \"ATP+CoA+10,11-dihydro-leukotriene B(,4)=AMP+PP(,i)+10,11-dihydro-LTB(,4)-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4470 \"ATP+CoA+10,11-dihydro-leukotriene B(,4)=AMP+PP(,i)+10,11-dihydro-LTB(,4)-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4470 \"ATP+CoA+10,11-dihydro-leukotriene B(,4)=AMP+PP(,i)+10,11-dihydro-LTB(,4)-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4471 \"H(,2)O+5(`S`),12(`R`)-dihydroxy-eicosa-2,8-`trans`-6,14-`cis`-tetraenoyl-CoA=3(`S`),5(`S`),12(`R`)-trihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4471 \"H(,2)O+5(`S`),12(`R`)-dihydroxy-eicosa-2,8-`trans`-6,14-`cis`-tetraenoyl-CoA=3(`S`),5(`S`),12(`R`)-trihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4472 \"H(,2)O+5(`S`),12(`R`)-dihydroxy-eicosa-2,8-`trans`-6,14-`cis`-tetraenoyl-CoA=3(`S`),5(`S`),12(`R`)-trihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4472 \"H(,2)O+5(`S`),12(`R`)-dihydroxy-eicosa-2,8-`trans`-6,14-`cis`-tetraenoyl-CoA=3(`S`),5(`S`),12(`R`)-trihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4473 \"NAD('+)+3(`S`),5(`S`),12(`R`)-trihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA=NADH+H('+)+3-oxo-5(`S`),12(`R`)-dihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4473 \"NAD('+)+3(`S`),5(`S`),12(`R`)-trihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA=NADH+H('+)+3-oxo-5(`S`),12(`R`)-dihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4474 \"NAD('+)+3(`S`),5(`S`),12(`R`)-trihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA=NADH+H('+)+3-oxo-5(`S`),12(`R`)-dihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4474 \"NAD('+)+3(`S`),5(`S`),12(`R`)-trihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA=NADH+H('+)+3-oxo-5(`S`),12(`R`)-dihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4475 \"NAD('+)+3(`S`),5(`S`),12(`R`)-trihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA=NADH+H('+)+3-oxo-5(`S`),12(`R`)-dihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4475 \"NAD('+)+3(`S`),5(`S`),12(`R`)-trihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA=NADH+H('+)+3-oxo-5(`S`),12(`R`)-dihydroxy-eicosa-8-`trans`-6,14-`cis`-trienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4492 \"NADPH+O(,2)+H('+)+a-tocotrienol=H(,2)O+NADP('+)+13'-hydroxy-a-tocotrienol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4492 \"NADPH+O(,2)+H('+)+a-tocotrienol=H(,2)O+NADP('+)+13'-hydroxy-a-tocotrienol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4494 \"H(,2)O+1-alkyl-2-acetyl-`sn`-glycerol=acetate+H('+)+alkylglycerol (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4494 \"H(,2)O+1-alkyl-2-acetyl-`sn`-glycerol=acetate+H('+)+alkylglycerol (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4496 \"5-oxo-12(`S`)-hydroxy-eicosa-6`E`,8`Z`,10`E`,14`Z`-tetraenoate=5-oxo-6-`trans`-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4496 \"5-oxo-12(`S`)-hydroxy-eicosa-6`E`,8`Z`,10`E`,14`Z`-tetraenoate=5-oxo-6-`trans`-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4497 \"NADP('+)+5,12-DiHETE=NADPH+H('+)+5-oxo-12(`S`)-hydroxy-eicosa-6`E`,8`Z`,10`E`,14`Z`-tetraenoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4497 \"NADP('+)+5,12-DiHETE=NADPH+H('+)+5-oxo-12(`S`)-hydroxy-eicosa-6`E`,8`Z`,10`E`,14`Z`-tetraenoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4498 \"NADP('+)+5,12-DiHETE=NADPH+H('+)+5-oxo-12(`S`)-hydroxy-eicosa-6`E`,8`Z`,10`E`,14`Z`-tetraenoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4498 \"NADP('+)+5,12-DiHETE=NADPH+H('+)+5-oxo-12(`S`)-hydroxy-eicosa-6`E`,8`Z`,10`E`,14`Z`-tetraenoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4499 \"NADP('+)+5,12-DiHETE=NADPH+H('+)+5-oxo-12(`S`)-hydroxy-eicosa-6`E`,8`Z`,10`E`,14`Z`-tetraenoate\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4499 \"NADP('+)+5,12-DiHETE=NADPH+H('+)+5-oxo-12(`S`)-hydroxy-eicosa-6`E`,8`Z`,10`E`,14`Z`-tetraenoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4501 \"ATP+2 CoA+2 H(,2)O+NAD('+)+O(,2)+13'-carboxy-a-tocopherol=AMP+CoA+H(,2)O(,2)+NADH+PP(,i)+propionyl-CoA+2 H('+)+11'-carboxy-a-chromanol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4501 \"ATP+2 CoA+2 H(,2)O+NAD('+)+O(,2)+13'-carboxy-a-tocopherol=AMP+CoA+H(,2)O(,2)+NADH+PP(,i)+propionyl-CoA+2 H('+)+11'-carboxy-a-chromanol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4502 \"ATP+2 CoA+2 H(,2)O+NAD('+)+O(,2)+7'-carboxy-a-chromanol=acetyl-CoA+AMP+CoA+H(,2)O(,2)+NADH+PP(,i)+2 H('+)+5'-carboxy-a-chromanol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4502 \"ATP+2 CoA+2 H(,2)O+NAD('+)+O(,2)+7'-carboxy-a-chromanol=acetyl-CoA+AMP+CoA+H(,2)O(,2)+NADH+PP(,i)+2 H('+)+5'-carboxy-a-chromanol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4503 \"ATP+2 CoA+2 H(,2)O+NAD('+)+O(,2)+7'-carboxy-a-chromanol=acetyl-CoA+AMP+CoA+H(,2)O(,2)+NADH+PP(,i)+2 H('+)+5'-carboxy-a-chromanol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4503 \"ATP+2 CoA+2 H(,2)O+NAD('+)+O(,2)+7'-carboxy-a-chromanol=acetyl-CoA+AMP+CoA+H(,2)O(,2)+NADH+PP(,i)+2 H('+)+5'-carboxy-a-chromanol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4505 \"5-hydroxyeicosatetraenoate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+20-hydroxy-5`S`-HETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4505 \"5-hydroxyeicosatetraenoate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+20-hydroxy-5`S`-HETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4508 \"5-hydroxyeicosatetraenoate+NADP('+)=NADPH+H('+)+5-oxo-(6`E`,8`Z`,11`Z`,14`Z`)-eicosatetraenoic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4508 \"5-hydroxyeicosatetraenoate+NADP('+)=NADPH+H('+)+5-oxo-(6`E`,8`Z`,11`Z`,14`Z`)-eicosatetraenoic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4509 \"5-hydroxyeicosatetraenoate+NADP('+)=NADPH+H('+)+5-oxo-(6`E`,8`Z`,11`Z`,14`Z`)-eicosatetraenoic acid\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4509 \"5-hydroxyeicosatetraenoate+NADP('+)=NADPH+H('+)+5-oxo-(6`E`,8`Z`,11`Z`,14`Z`)-eicosatetraenoic acid\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4511 \"thromboxane B(,2)=H(,2)+11-dehydrothromboxane B2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4511 \"thromboxane B(,2)=H(,2)+11-dehydrothromboxane B2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4513 \"NADH+H('+)+11-dehydro-15-keto-TXB(,2)=NAD('+)+11-dehydro-13,14-dihydro-15-oxo-TXB(,2)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4513 \"NADH+H('+)+11-dehydro-15-keto-TXB(,2)=NAD('+)+11-dehydro-13,14-dihydro-15-oxo-TXB(,2)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4514 \"NAD('+)+11-dehydrothromboxane B2=NADH+H('+)+11-dehydro-15-keto-TXB(,2)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4514 \"NAD('+)+11-dehydrothromboxane B2=NADH+H('+)+11-dehydro-15-keto-TXB(,2)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4516 \"NADH+H('+)+13,14-dihydro-15-oxo-PGE(,1)=NAD('+)+13,14-dihydro-PGE(,1)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4516 \"NADH+H('+)+13,14-dihydro-15-oxo-PGE(,1)=NAD('+)+13,14-dihydro-PGE(,1)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4517 \"NAD('+)+prostaglandin E(,1)=NADH+H('+)+15-oxo-PGE(,1)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4517 \"NAD('+)+prostaglandin E(,1)=NADH+H('+)+15-oxo-PGE(,1)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4518 \"NADH+H('+)+15-oxo-PGE(,1)=NAD('+)+13,14-dihydro-15-oxo-PGE(,1)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4518 \"NADH+H('+)+15-oxo-PGE(,1)=NAD('+)+13,14-dihydro-15-oxo-PGE(,1)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4519 \"O(,2)+timnodonate=5-hydroperoxy-EPA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4519 \"O(,2)+timnodonate=5-hydroperoxy-EPA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4521 \"5-hydroperoxy-EPA=H(,2)O+leukotriene A(,5)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4521 \"5-hydroperoxy-EPA=H(,2)O+leukotriene A(,5)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4522 \"O(,2)+timnodonate=15(`S`)-hydroperoxy-EPE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4522 \"O(,2)+timnodonate=15(`S`)-hydroperoxy-EPE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4525 \"O(,2)+timnodonate=15(`R`)-hydroperoxy-EPE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4525 \"O(,2)+timnodonate=15(`R`)-hydroperoxy-EPE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4526 \"O(,2)+timnodonate=15(`R`)-hydroperoxy-EPE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4526 \"O(,2)+timnodonate=15(`R`)-hydroperoxy-EPE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4529 \"reduced acceptor+15(`R`)-hydroperoxy-EPE=acceptor+H(,2)O+15(`R`)-HEPE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4529 \"reduced acceptor+15(`R`)-hydroperoxy-EPE=acceptor+H(,2)O+15(`R`)-HEPE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4531 \"H(,2)O+leukotriene A(,5)=leukotriene B(,5)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4531 \"H(,2)O+leukotriene A(,5)=leukotriene B(,5)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4533 \"2 H('+)+5-hydroperoxy-EPA=H(,2)O+5-HEPE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4533 \"2 H('+)+5-hydroperoxy-EPA=H(,2)O+5-HEPE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4540 \"2 O(,2)+reduced acceptor+timnodonate=acceptor+H(,2)O+PGH(,3)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4540 \"2 O(,2)+reduced acceptor+timnodonate=acceptor+H(,2)O+PGH(,3)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4542 \"O(,2)+15(`R`)-HEPE=H(,2)O+5(`S`),6(`S`)-epoxy-15(`R`)-HEPE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4542 \"O(,2)+15(`R`)-HEPE=H(,2)O+5(`S`),6(`S`)-epoxy-15(`R`)-HEPE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4545 \"H(,2)O+5(`S`),6(`S`)-epoxy-15(`R`)-HEPE=15-`epi`-lipoxin B(,5)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4545 \"H(,2)O+5(`S`),6(`S`)-epoxy-15(`R`)-HEPE=15-`epi`-lipoxin B(,5)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4548 \"H(,2)O+5(`S`),6(`S`)-epoxy-15(`R`)-HEPE=15-`epi`-lipoxin A(,5)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4548 \"H(,2)O+5(`S`),6(`S`)-epoxy-15(`R`)-HEPE=15-`epi`-lipoxin A(,5)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4550 \"NADP('+)+5-HEPE=NADPH+H('+)+5-oxo-EPE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4550 \"NADP('+)+5-HEPE=NADPH+H('+)+5-oxo-EPE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4551 \"NADP('+)+5-HEPE=NADPH+H('+)+5-oxo-EPE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4551 \"NADP('+)+5-HEPE=NADPH+H('+)+5-oxo-EPE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4556 \"O(,2)+reduced acceptor+timnodonate=acceptor+H(,2)O+18`R`-HEPE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4556 \"O(,2)+reduced acceptor+timnodonate=acceptor+H(,2)O+18`R`-HEPE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4559 \"O(,2)+reduced acceptor+timnodonate=acceptor+H(,2)O+11`R`-HEPE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4559 \"O(,2)+reduced acceptor+timnodonate=acceptor+H(,2)O+11`R`-HEPE\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4562 \"NADPH+O(,2)+H('+)+leukotriene B(,5)=H(,2)O+NADP('+)+20-hydroxy-leukotriene B(,5)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4562 \"NADPH+O(,2)+H('+)+leukotriene B(,5)=H(,2)O+NADP('+)+20-hydroxy-leukotriene B(,5)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4574 \"NADPH+O(,2)+H('+)+5,12-DiHETE=H(,2)O+NADP('+)+5,12,20-TriHETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4574 \"NADPH+O(,2)+H('+)+5,12-DiHETE=H(,2)O+NADP('+)+5,12,20-TriHETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4575 \"3-methylthiopropionate=acrylate+methanethiol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4575 \"3-methylthiopropionate=acrylate+methanethiol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4580 \"H(,2)O+hepoxilin A(,3)=trioxilin A(,3)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4580 \"H(,2)O+hepoxilin A(,3)=trioxilin A(,3)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4583 \"12-hydroperoxyeicosatetraenoate=hepoxilin A(,3) (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4583 \"12-hydroperoxyeicosatetraenoate=hepoxilin A(,3) (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4584 \"12-hydroperoxyeicosatetraenoate=hepoxilin A(,3)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4584 \"12-hydroperoxyeicosatetraenoate=hepoxilin A(,3)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4592 \"12-hydroxyeicosatetraenoate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+12,20-DiHETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4592 \"12-hydroxyeicosatetraenoate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+12,20-DiHETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4594 \"5-hydroxyeicosatetraenoate+O(,2)+2 H('+)=H(,2)O+5,12-DiHETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4594 \"5-hydroxyeicosatetraenoate+O(,2)+2 H('+)=H(,2)O+5,12-DiHETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4599 \"O(,2)+2 H('+)+5-oxo-(6`E`,8`Z`,11`Z`,14`Z`)-eicosatetraenoic acid=H(,2)O+5-oxo-12(`S`)-hydroxy-eicosa-6`E`,8`Z`,10`E`,14`Z`-tetraenoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4599 \"O(,2)+2 H('+)+5-oxo-(6`E`,8`Z`,11`Z`,14`Z`)-eicosatetraenoic acid=H(,2)O+5-oxo-12(`S`)-hydroxy-eicosa-6`E`,8`Z`,10`E`,14`Z`-tetraenoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4605 \"glutathione+9-deoxy-D('12)-PGD(,2)=S-(9-deoxy-D('12)-PGD(,2))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4605 \"glutathione+9-deoxy-D('12)-PGD(,2)=S-(9-deoxy-D('12)-PGD(,2))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4606 \"glutathione+9-deoxy-D('12)-PGD(,2)=S-(9-deoxy-D('12)-PGD(,2))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4606 \"glutathione+9-deoxy-D('12)-PGD(,2)=S-(9-deoxy-D('12)-PGD(,2))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4608 \"glutathione+prostaglandin J(,2)=S-(PGJ(,2))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4608 \"glutathione+prostaglandin J(,2)=S-(PGJ(,2))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4609 \"glutathione+prostaglandin J(,2)=S-(PGJ(,2))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4609 \"glutathione+prostaglandin J(,2)=S-(PGJ(,2))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4611 \"glutathione+D12-Prostaglandin J(,2)=`S`-(9-deoxy-D('9,12)-PGD(,2))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4611 \"glutathione+D12-Prostaglandin J(,2)=`S`-(9-deoxy-D('9,12)-PGD(,2))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4612 \"glutathione+D12-Prostaglandin J(,2)=`S`-(9-deoxy-D('9,12)-PGD(,2))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4612 \"glutathione+D12-Prostaglandin J(,2)=`S`-(9-deoxy-D('9,12)-PGD(,2))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4613 \"prostaglandin D(,2)+2 H('+)=H(,2)O+9-deoxy-D('12)-PGD(,2)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4613 \"prostaglandin D(,2)+2 H('+)=H(,2)O+9-deoxy-D('12)-PGD(,2)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4616 \"leukotriene E(,4)+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+20-hydroxy-leukotriene E(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4616 \"leukotriene E(,4)+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+20-hydroxy-leukotriene E(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4617 \"NAD('+)+20-hydroxy-leukotriene E(,4)=NADH+H('+)+20-oxo-leukotriene E(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4617 \"NAD('+)+20-hydroxy-leukotriene E(,4)=NADH+H('+)+20-oxo-leukotriene E(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4618 \"H(,2)O+NAD('+)+20-oxo-leukotriene E(,4)=NADH+2 H('+)+20-COOH-leukotriene E(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4618 \"H(,2)O+NAD('+)+20-oxo-leukotriene E(,4)=NADH+2 H('+)+20-COOH-leukotriene E(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4619 \"NAD('+)+prostaglandin E(,1)=NADH+prostaglandin E(,2)+H('+)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4619 \"NAD('+)+prostaglandin E(,1)=NADH+prostaglandin E(,2)+H('+)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4621 \"glutathione+prostaglandin A(,2)=S-(PGA(,2))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4621 \"glutathione+prostaglandin A(,2)=S-(PGA(,2))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4622 \"glutathione+prostaglandin A(,2)=S-(PGA(,2))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4622 \"glutathione+prostaglandin A(,2)=S-(PGA(,2))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4623 \"NADPH+H('+)+S-(PGA(,2))-glutathione=NADP('+)+S-(9-hydroxy-PGA(,2))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4623 \"NADPH+H('+)+S-(PGA(,2))-glutathione=NADP('+)+S-(9-hydroxy-PGA(,2))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4625 \"glutathione+prostaglandin A(,1)=S-(PGA(,1))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4625 \"glutathione+prostaglandin A(,1)=S-(PGA(,1))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4626 \"glutathione+prostaglandin A(,1)=S-(PGA(,1))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4626 \"glutathione+prostaglandin A(,1)=S-(PGA(,1))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4627 \"glutathione+prostaglandin A(,1)=S-(PGA(,1))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4627 \"glutathione+prostaglandin A(,1)=S-(PGA(,1))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4628 \"NADPH+H('+)+S-(PGA(,1))-glutathione=NADP('+)+S-(9-hydroxy-PGA(,1))-glutathione\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4628 \"NADPH+H('+)+S-(PGA(,1))-glutathione=NADP('+)+S-(9-hydroxy-PGA(,1))-glutathione\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4629 \"CoA+18,20-dioxo-20-CoA-leukotriene B(,4)=acetyl-CoA+18-CoA-18-oxo-dinorleukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4629 \"CoA+18,20-dioxo-20-CoA-leukotriene B(,4)=acetyl-CoA+18-CoA-18-oxo-dinorleukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4630 \"CoA+18,20-dioxo-20-CoA-leukotriene B(,4)=acetyl-CoA+18-CoA-18-oxo-dinorleukotriene B(,4)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4630 \"CoA+18,20-dioxo-20-CoA-leukotriene B(,4)=acetyl-CoA+18-CoA-18-oxo-dinorleukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4631 \"H(,2)O+18-CoA-18-oxo-dinorleukotriene B(,4)=CoA+H('+)+o-carboxy-trinor-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4631 \"H(,2)O+18-CoA-18-oxo-dinorleukotriene B(,4)=CoA+H('+)+o-carboxy-trinor-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4633 \"H(,2)O+18-CoA-18-oxo-dinorleukotriene B(,4)=CoA+H('+)+o-carboxy-trinor-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4633 \"H(,2)O+18-CoA-18-oxo-dinorleukotriene B(,4)=CoA+H('+)+o-carboxy-trinor-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4634 \"O(,2)+20-CoA-20-oxo-leukotriene B(,4)=H(,2)O(,2)+18`E`-20-oxo-20-CoA-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4634 \"O(,2)+20-CoA-20-oxo-leukotriene B(,4)=H(,2)O(,2)+18`E`-20-oxo-20-CoA-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4635 \"O(,2)+20-CoA-20-oxo-leukotriene B(,4)=H(,2)O(,2)+18`E`-20-oxo-20-CoA-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4635 \"O(,2)+20-CoA-20-oxo-leukotriene B(,4)=H(,2)O(,2)+18`E`-20-oxo-20-CoA-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4636 \"ATP+CoA+20-carboxy-leukotriene-B(,4)=AMP+PP(,i)+20-CoA-20-oxo-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4636 \"ATP+CoA+20-carboxy-leukotriene-B(,4)=AMP+PP(,i)+20-CoA-20-oxo-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4637 \"ATP+CoA+20-carboxy-leukotriene-B(,4)=AMP+PP(,i)+20-CoA-20-oxo-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4637 \"ATP+CoA+20-carboxy-leukotriene-B(,4)=AMP+PP(,i)+20-CoA-20-oxo-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4638 \"ATP+CoA+20-carboxy-leukotriene-B(,4)=AMP+PP(,i)+20-CoA-20-oxo-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4638 \"ATP+CoA+20-carboxy-leukotriene-B(,4)=AMP+PP(,i)+20-CoA-20-oxo-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4639 \"ATP+CoA+20-carboxy-leukotriene-B(,4)=AMP+PP(,i)+20-CoA-20-oxo-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4639 \"ATP+CoA+20-carboxy-leukotriene-B(,4)=AMP+PP(,i)+20-CoA-20-oxo-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4640 \"H(,2)O+18`E`-20-oxo-20-CoA-LTB(,4)=20-CoA-20-oxo-18`R`-hydroxyleucotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4640 \"H(,2)O+18`E`-20-oxo-20-CoA-LTB(,4)=20-CoA-20-oxo-18`R`-hydroxyleucotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4641 \"H(,2)O+18`E`-20-oxo-20-CoA-LTB(,4)=20-CoA-20-oxo-18`R`-hydroxyleucotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4641 \"H(,2)O+18`E`-20-oxo-20-CoA-LTB(,4)=20-CoA-20-oxo-18`R`-hydroxyleucotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4642 \"NAD('+)+20-CoA-20-oxo-18`R`-hydroxyleucotriene B(,4)=NADH+H('+)+18,20-dioxo-20-CoA-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4642 \"NAD('+)+20-CoA-20-oxo-18`R`-hydroxyleucotriene B(,4)=NADH+H('+)+18,20-dioxo-20-CoA-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4643 \"NAD('+)+20-CoA-20-oxo-18`R`-hydroxyleucotriene B(,4)=NADH+H('+)+18,20-dioxo-20-CoA-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4643 \"NAD('+)+20-CoA-20-oxo-18`R`-hydroxyleucotriene B(,4)=NADH+H('+)+18,20-dioxo-20-CoA-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4644 \"NAD('+)+20-CoA-20-oxo-18`R`-hydroxyleucotriene B(,4)=NADH+H('+)+18,20-dioxo-20-CoA-leukotriene B(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4644 \"NAD('+)+20-CoA-20-oxo-18`R`-hydroxyleucotriene B(,4)=NADH+H('+)+18,20-dioxo-20-CoA-leukotriene B(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4647 \"prostaglandin C(,2)=prostaglandin B(,2) (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4647 \"prostaglandin C(,2)=prostaglandin B(,2) (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4648 \"prostaglandin E(,1)=H(,2)O+prostaglandin A(,1)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4648 \"prostaglandin E(,1)=H(,2)O+prostaglandin A(,1)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4653 \"NADH+H('+)+CoA-20-COOH-18-oxo-LTE(,4)=NAD('+)+18(`R`)-hydroxy-20-oxo-20-CoA-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4653 \"NADH+H('+)+CoA-20-COOH-18-oxo-LTE(,4)=NAD('+)+18(`R`)-hydroxy-20-oxo-20-CoA-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4654 \"NADH+H('+)+16,18-oxo-18-CoA-dinor-LTE(,4)=NAD('+)+16(`S`)-hydroxy-18-oxo-18-CoA-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4654 \"NADH+H('+)+16,18-oxo-18-CoA-dinor-LTE(,4)=NAD('+)+16(`S`)-hydroxy-18-oxo-18-CoA-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4655 \"H(,2)O+CoA-o-COOH-dinor-LTE(,4)=CoA+H('+)+18-carboxy-dinor-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4655 \"H(,2)O+CoA-o-COOH-dinor-LTE(,4)=CoA+H('+)+18-carboxy-dinor-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4656 \"CoA+CoA-20-COOH-18-oxo-LTE(,4)=acetyl-CoA+CoA-o-COOH-dinor-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4656 \"CoA+CoA-20-COOH-18-oxo-LTE(,4)=acetyl-CoA+CoA-o-COOH-dinor-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4657 \"O(,2)+CoA-20-COOH-LTE(,4)=H(,2)O(,2)+18`E`-20-oxo-20-CoA-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4657 \"O(,2)+CoA-20-COOH-LTE(,4)=H(,2)O(,2)+18`E`-20-oxo-20-CoA-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4658 \"ATP+CoA+20-COOH-leukotriene E(,4)=AMP+PP(,i)+CoA-20-COOH-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4658 \"ATP+CoA+20-COOH-leukotriene E(,4)=AMP+PP(,i)+CoA-20-COOH-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4659 \"H(,2)O+18`E`-20-oxo-20-CoA-LTE(,4)=18(`R`)-hydroxy-20-oxo-20-CoA-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4659 \"H(,2)O+18`E`-20-oxo-20-CoA-LTE(,4)=18(`R`)-hydroxy-20-oxo-20-CoA-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4660 \"H(,2)O+CoA-o-COOH-tetranor-LTE(,3)=CoA+H('+)+o-COOH-tetranor-LTE(,3)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4660 \"H(,2)O+CoA-o-COOH-tetranor-LTE(,3)=CoA+H('+)+o-COOH-tetranor-LTE(,3)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4661 \"O(,2)+CoA-o-COOH-dinor-LTE(,4)=H(,2)O(,2)+CoA-18-COOH-16`E`-dinor-LTE(,5)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4661 \"O(,2)+CoA-o-COOH-dinor-LTE(,4)=H(,2)O(,2)+CoA-18-COOH-16`E`-dinor-LTE(,5)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4662 \"NADPH+H('+)+CoA-18-COOH-16`E`-dinor-LTE(,5)=NADP('+)+CoA-18-COOH-15`E`-dinor-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4662 \"NADPH+H('+)+CoA-18-COOH-16`E`-dinor-LTE(,5)=NADP('+)+CoA-18-COOH-15`E`-dinor-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4663 \"CoA-18-COOH-15`E`-dinor-LTE(,4)=16`E`-18-oxo-18-CoA-dinor-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4663 \"CoA-18-COOH-15`E`-dinor-LTE(,4)=16`E`-18-oxo-18-CoA-dinor-LTE(,4)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4664 \"H(,2)O+16`E`-18-oxo-18-CoA-dinor-LTE(,4)=16(`S`)-hydroxy-18-oxo-18-CoA-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4664 \"H(,2)O+16`E`-18-oxo-18-CoA-dinor-LTE(,4)=16(`S`)-hydroxy-18-oxo-18-CoA-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4665 \"CoA+16,18-oxo-18-CoA-dinor-LTE(,4)=acetyl-CoA+CoA-o-COOH-tetranor-LTE(,3)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4665 \"CoA+16,18-oxo-18-CoA-dinor-LTE(,4)=acetyl-CoA+CoA-o-COOH-tetranor-LTE(,3)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4666 \"O(,2)+CoA-o-COOH-tetranor-LTE(,3)=H(,2)O(,2)+13`E`-tetranor-16-oxo-16-CoA-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4666 \"O(,2)+CoA-o-COOH-tetranor-LTE(,3)=H(,2)O(,2)+13`E`-tetranor-16-oxo-16-CoA-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4667 \"H(,2)O+13`E`-tetranor-16-oxo-16-CoA-LTE(,4)=CoA+H('+)+13`E`-tetranor-16-carboxy-LTE(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4667 \"H(,2)O+13`E`-tetranor-16-oxo-16-CoA-LTE(,4)=CoA+H('+)+13`E`-tetranor-16-carboxy-LTE(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4668 \"prostaglandin H(,1)=prostaglandin E(,1) (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4668 \"prostaglandin H(,1)=prostaglandin E(,1) (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4669 \"prostaglandin H(,1)=prostaglandin E(,1)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4669 \"prostaglandin H(,1)=prostaglandin E(,1)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4671 \"reduced acceptor+prostaglandin G(,1)=acceptor+H(,2)O+prostaglandin H(,1)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4671 \"reduced acceptor+prostaglandin G(,1)=acceptor+H(,2)O+prostaglandin H(,1)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4672 \"dolichol+NADP('+)=NADPH+H('+)+dehydrodolichol (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4672 \"dolichol+NADP('+)=NADPH+H('+)+dehydrodolichol (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4673 \"dimethylallyl pyrophosphate+`trans,trans,cis`-geranyl-geranyl-PP=PP(,i)+dehydrodolichyl diphosphate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4673 \"dimethylallyl pyrophosphate+`trans,trans,cis`-geranyl-geranyl-PP=PP(,i)+dehydrodolichyl diphosphate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4674 \"H(,2)O+dehydrodolichyl diphosphate=Pi+dehydrodolichyl-P\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4674 \"H(,2)O+dehydrodolichyl diphosphate=Pi+dehydrodolichyl-P\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4675 \"H(,2)O+dehydrodolichyl-P=Pi+dehydrodolichol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4675 \"H(,2)O+dehydrodolichyl-P=Pi+dehydrodolichol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4676 \"NADPH+H('+)+12-oxo-20-carboxy-leukotriene B(,4)=NADP('+)+12-oxo-10,11-dihydro-20-COOH-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4676 \"NADPH+H('+)+12-oxo-20-carboxy-leukotriene B(,4)=NADP('+)+12-oxo-10,11-dihydro-20-COOH-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4677 \"NADPH+H('+)+12-oxo-20-carboxy-leukotriene B(,4)=NADP('+)+12-oxo-10,11-dihydro-20-COOH-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4677 \"NADPH+H('+)+12-oxo-20-carboxy-leukotriene B(,4)=NADP('+)+12-oxo-10,11-dihydro-20-COOH-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4678 \"NADPH+H('+)+12-oxo-20-carboxy-leukotriene B(,4)=NADP('+)+12-oxo-10,11-dihydro-20-COOH-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4678 \"NADPH+H('+)+12-oxo-20-carboxy-leukotriene B(,4)=NADP('+)+12-oxo-10,11-dihydro-20-COOH-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4679 \"NADPH+H('+)+20-carboxy-leukotriene-B(,4)=NADP('+)+20-COOH-10,11-dihydro-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4679 \"NADPH+H('+)+20-carboxy-leukotriene-B(,4)=NADP('+)+20-COOH-10,11-dihydro-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4680 \"NADPH+H('+)+20-carboxy-leukotriene-B(,4)=NADP('+)+20-COOH-10,11-dihydro-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4680 \"NADPH+H('+)+20-carboxy-leukotriene-B(,4)=NADP('+)+20-COOH-10,11-dihydro-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4681 \"NADPH+H('+)+20-carboxy-leukotriene-B(,4)=NADP('+)+20-COOH-10,11-dihydro-LTB(,4)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4681 \"NADPH+H('+)+20-carboxy-leukotriene-B(,4)=NADP('+)+20-COOH-10,11-dihydro-LTB(,4)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4683 \"O(,2)+`cis`-6-dodecenoyl-CoA=H(,2)O(,2)+`trans`,`cis`-dodeca-2,6-dienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4683 \"O(,2)+`cis`-6-dodecenoyl-CoA=H(,2)O(,2)+`trans`,`cis`-dodeca-2,6-dienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4684 \"O(,2)+`cis`-6-dodecenoyl-CoA=H(,2)O(,2)+`trans`,`cis`-dodeca-2,6-dienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4684 \"O(,2)+`cis`-6-dodecenoyl-CoA=H(,2)O(,2)+`trans`,`cis`-dodeca-2,6-dienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4686 \"O(,2)+`cis`-deca-4-enoyl-CoA=H(,2)O(,2)+`trans`,`cis`-deca-2,4-dienoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4686 \"O(,2)+`cis`-deca-4-enoyl-CoA=H(,2)O(,2)+`trans`,`cis`-deca-2,4-dienoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4687 \"NADPH+H('+)+`trans`,`cis`-deca-2,4-dienoyl-CoA=NADP('+)+`trans`-3-decenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4687 \"NADPH+H('+)+`trans`,`cis`-deca-2,4-dienoyl-CoA=NADP('+)+`trans`-3-decenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4688 \"NADPH+H('+)+`trans`,`cis`-deca-2,4-dienoyl-CoA=NADP('+)+`trans`-3-decenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4688 \"NADPH+H('+)+`trans`,`cis`-deca-2,4-dienoyl-CoA=NADP('+)+`trans`-3-decenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4689 \"NADPH+H('+)+`trans`,`cis`-deca-2,4-dienoyl-CoA=NADP('+)+`trans`-3-decenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4689 \"NADPH+H('+)+`trans`,`cis`-deca-2,4-dienoyl-CoA=NADP('+)+`trans`-3-decenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4690 \"`trans`-deca-2-enoyl-CoA=`trans`-3-decenoyl-CoA\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4690 \"`trans`-deca-2-enoyl-CoA=`trans`-3-decenoyl-CoA\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4700 \"1-hydroxymethylbilane=H(,2)O+uroporphyrinogen I\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4700 \"1-hydroxymethylbilane=H(,2)O+uroporphyrinogen I\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4701 \"12-hydroxyeicosatetraenoate+O(,2)+2 H('+)=H(,2)O+11,12-dihydroxy-5(`E`),7(`E`),9(`E`),14(`Z`)-eicosatetraenoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4701 \"12-hydroxyeicosatetraenoate+O(,2)+2 H('+)=H(,2)O+11,12-dihydroxy-5(`E`),7(`E`),9(`E`),14(`Z`)-eicosatetraenoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4702 \"12-hydroxyeicosatetraenoate+NADH+H('+)=NAD('+)+12-HETrE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4702 \"12-hydroxyeicosatetraenoate+NADH+H('+)=NAD('+)+12-HETrE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4704 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+8-hydroxyeicosatetraenoate\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4704 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+8-hydroxyeicosatetraenoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4706 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+11-hydroxyeicosatetraenoate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4706 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+11-hydroxyeicosatetraenoate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4708 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+9-HETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4708 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+9-HETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4710 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+13-HETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4710 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+13-HETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4712 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+7-HETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4712 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+7-HETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4714 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+10-HETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4714 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+10-HETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4716 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+18-HETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4716 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+18-HETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4718 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+17-HETE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4718 \"arachidonate+NADPH+O(,2)+H('+)=H(,2)O+NADP('+)+17-HETE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4738 \"L-thyroxine=3,5,3',5'-tetraiodothyroacetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4738 \"L-thyroxine=3,5,3',5'-tetraiodothyroacetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4739 \"L-epinephrine=adrenochrome\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4739 \"L-epinephrine=adrenochrome\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4744 \"3',5'-diiodo-L-thyronine=3',5'-diiodothyroacetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4744 \"3',5'-diiodo-L-thyronine=3',5'-diiodothyroacetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4748 \"3,5,3' - triiodothyronine=3,5,3'-triiodothyroacetate\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4748 \"3,5,3' - triiodothyronine=3,5,3'-triiodothyroacetate\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4751 \"Benzo[a]pyrene-2,3-oxide+H2O=Benzo[a]pyrene-2,3-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4751 \"Benzo[a]pyrene-2,3-oxide+H2O=Benzo[a]pyrene-2,3-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4752 \"Benzo[a]pyrene-2,3-oxide+H2O=Benzo[a]pyrene-2,3-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4752 \"Benzo[a]pyrene-2,3-oxide+H2O=Benzo[a]pyrene-2,3-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4753 \"Benzo[a]pyrene-2,3-oxide+H2O=Benzo[a]pyrene-2,3-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4753 \"Benzo[a]pyrene-2,3-oxide+H2O=Benzo[a]pyrene-2,3-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4754 \"Benzo[a]pyrene-4,5-oxide+H2O=Benzo[a]pyrene-4,5-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4754 \"Benzo[a]pyrene-4,5-oxide+H2O=Benzo[a]pyrene-4,5-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4755 \"Benzo[a]pyrene-4,5-oxide+H2O=Benzo[a]pyrene-4,5-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4755 \"Benzo[a]pyrene-4,5-oxide+H2O=Benzo[a]pyrene-4,5-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4756 \"Benzo[a]pyrene-4,5-oxide+H2O=Benzo[a]pyrene-4,5-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4756 \"Benzo[a]pyrene-4,5-oxide+H2O=Benzo[a]pyrene-4,5-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4757 \"Benzo[a]pyrene-9,10-oxide+H2O=Benzo[a]pyrene-9,10-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4757 \"Benzo[a]pyrene-9,10-oxide+H2O=Benzo[a]pyrene-9,10-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4758 \"Benzo[a]pyrene-9,10-oxide+H2O=Benzo[a]pyrene-9,10-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4758 \"Benzo[a]pyrene-9,10-oxide+H2O=Benzo[a]pyrene-9,10-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4759 \"Benzo[a]pyrene-9,10-oxide+H2O=Benzo[a]pyrene-9,10-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4759 \"Benzo[a]pyrene-9,10-oxide+H2O=Benzo[a]pyrene-9,10-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4768 \"7,12-Dimethylbenz[a]anthracene-3,4-oxide+H2O=7,12-Dimethylbenz[a]anthracene-3,4-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4768 \"7,12-Dimethylbenz[a]anthracene-3,4-oxide+H2O=7,12-Dimethylbenz[a]anthracene-3,4-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4770 \"7,12-Dimethylbenz[a]anthracene-3,4-oxide+H2O=7,12-Dimethylbenz[a]anthracene-3,4-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4770 \"7,12-Dimethylbenz[a]anthracene-3,4-oxide+H2O=7,12-Dimethylbenz[a]anthracene-3,4-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4775 \"Dibenzo[a,l]pyrene-11,12-epoxide+H2O=Dibenzo[a,l]pyrene-11,12-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4775 \"Dibenzo[a,l]pyrene-11,12-epoxide+H2O=Dibenzo[a,l]pyrene-11,12-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4777 \"Dibenzo[a,l]pyrene-11,12-epoxide+H2O=Dibenzo[a,l]pyrene-11,12-diol\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4777 \"Dibenzo[a,l]pyrene-11,12-epoxide+H2O=Dibenzo[a,l]pyrene-11,12-diol\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4786 \"NAD+[C]=NAD+[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4786 \"NAD+[C]=NAD+[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4787 \"NADP+[C]+H+[C]=NADP+[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4787 \"NADP+[C]+H+[C]=NADP+[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4790 \"ADP[C]=ADP[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4790 \"ADP[C]=ADP[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4792 \"CoA[C]=CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4792 \"CoA[C]=CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4799 \"S-Adenosyl-L-methionine[C]=S-Adenosyl-L-methionine[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4799 \"S-Adenosyl-L-methionine[C]=S-Adenosyl-L-methionine[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4801 \"AMP[C]+H+[C]=AMP[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4801 \"AMP[C]+H+[C]=AMP[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4802 \"H+[C]+Pyruvate[C]=H+[X]+Pyruvate[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4802 \"H+[C]+Pyruvate[C]=H+[X]+Pyruvate[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4806 \"L-Glutamate[C]=L-Glutamate[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4806 \"L-Glutamate[C]=L-Glutamate[G]\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4809 \"UDPglucose[C]=UDPglucose[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4809 \"UDPglucose[C]=UDPglucose[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4816 \"GDP[C]=GDP[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4816 \"GDP[C]=GDP[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4817 \"GDP[C]+H+[C]=GDP[G]+H+[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4817 \"GDP[C]+H+[C]=GDP[G]+H+[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4821 \"Glycine[C]=Glycine[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4821 \"Glycine[C]=Glycine[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4822 \"Glycine[C]=Glycine[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4822 \"Glycine[C]=Glycine[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4829 \"UDP-N-acetyl-D-glucosamine[C]+H+[C]=UDP-N-acetyl-D-glucosamine[R]+H+[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4829 \"UDP-N-acetyl-D-glucosamine[C]+H+[C]=UDP-N-acetyl-D-glucosamine[R]+H+[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4830 \"UDP-N-acetyl-D-glucosamine[C]+UMP[G]=UDP-N-acetyl-D-glucosamine[G]+UMP[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4830 \"UDP-N-acetyl-D-glucosamine[C]+UMP[G]=UDP-N-acetyl-D-glucosamine[G]+UMP[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4836 \"L-Aspartate[C]=L-Aspartate[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4836 \"L-Aspartate[C]=L-Aspartate[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4841 \"UDP-D-galactose[C]+UMP[G]=UDP-D-galactose[G]+UMP[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4841 \"UDP-D-galactose[C]+UMP[G]=UDP-D-galactose[G]+UMP[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4842 \"CMP[C]=CMP[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4842 \"CMP[C]=CMP[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4843 \"CMP[C]+H+[C]=CMP[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4843 \"CMP[C]+H+[C]=CMP[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4845 \"Formate[C]+H+[C]=Formate[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4845 \"Formate[C]+H+[C]=Formate[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4850 \"Carboxylate[L]=Carboxylate[C] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4850 \"Carboxylate[L]=Carboxylate[C] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4857 \"CTP[C]=CTP[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4857 \"CTP[C]=CTP[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4858 \"CTP[C]=CTP[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4858 \"CTP[C]=CTP[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4861 \"L-Serine[C]=L-Serine[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4861 \"L-Serine[C]=L-Serine[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4863 \"L-Serine[C]=L-Serine[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4863 \"L-Serine[C]=L-Serine[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4865 \"Thiamin diphosphate[M]+H+[M]=Thiamin diphosphate[C]+H+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4865 \"Thiamin diphosphate[M]+H+[M]=Thiamin diphosphate[C]+H+[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4870 \"UTP[C]=UTP[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4870 \"UTP[C]=UTP[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4871 \"UTP[C]=UTP[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4871 \"UTP[C]=UTP[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4872 \"L-Tryptophan[C]+H+[C]=L-Tryptophan[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4872 \"L-Tryptophan[C]+H+[C]=L-Tryptophan[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4880 \"L-Tyrosine[C]+H+[C]=L-Tyrosine[X]+H+[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4880 \"L-Tyrosine[C]+H+[C]=L-Tyrosine[X]+H+[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4884 \"Malonyl-CoA[C]=Malonyl-CoA[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4884 \"Malonyl-CoA[C]=Malonyl-CoA[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4885 \"Malonyl-CoA[C]=Malonyl-CoA[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4885 \"Malonyl-CoA[C]=Malonyl-CoA[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4886 \"Malonyl-CoA[C]=Malonyl-CoA[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4886 \"Malonyl-CoA[C]=Malonyl-CoA[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4890 \"L-Cysteine[C]+H+[C]=L-Cysteine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4890 \"L-Cysteine[C]+H+[C]=L-Cysteine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4893 \"beta-Alanine[C]+H+[C]=beta-Alanine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4893 \"beta-Alanine[C]+H+[C]=beta-Alanine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4898 \"UMP[C]=UMP[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4898 \"UMP[C]=UMP[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4900 \"Dolichyl phosphate[C]=Dolichyl phosphate[R] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4900 \"Dolichyl phosphate[C]=Dolichyl phosphate[R] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4903 \"Glycerone phosphate[C]=Glycerone phosphate[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4903 \"Glycerone phosphate[C]=Glycerone phosphate[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4920 \"myo-Inositol[C]=myo-Inositol[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4920 \"myo-Inositol[C]=myo-Inositol[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4922 \"N-Acetyl-D-glucosamine[C]=N-Acetyl-D-glucosamine[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4922 \"N-Acetyl-D-glucosamine[C]=N-Acetyl-D-glucosamine[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4929 \"Palmitoyl-CoA[C]=Palmitoyl-CoA[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4929 \"Palmitoyl-CoA[C]=Palmitoyl-CoA[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4930 \"Palmitoyl-CoA[C]=Palmitoyl-CoA[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4930 \"Palmitoyl-CoA[C]=Palmitoyl-CoA[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4932 \"ATP[C]+H2O[C]+Phosphatidylcholine[C]=ADP[C]+H+[C]+Phosphatidylcholine[E]+Orthophosphate[C] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4932 \"ATP[C]+H2O[C]+Phosphatidylcholine[C]=ADP[C]+H+[C]+Phosphatidylcholine[E]+Orthophosphate[C] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4933 \"Phosphatidylcholine[C]=Phosphatidylcholine[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4933 \"Phosphatidylcholine[C]=Phosphatidylcholine[R]\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4934 \"ADP[C]+H+[C]+Phosphatidylcholine[C]+Orthophosphate[C]=ATP[C]+H2O[C]+Phosphatidylcholine[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4934 \"ADP[C]+H+[C]+Phosphatidylcholine[C]+Orthophosphate[C]=ATP[C]+H2O[C]+Phosphatidylcholine[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4935 \"D-Mannose[E]=D-Mannose[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4935 \"D-Mannose[E]=D-Mannose[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4939 \"Propanoate[C]=Propanoate[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4939 \"Propanoate[C]=Propanoate[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4941 \"Acetoacetate[M]+H+[M]=Acetoacetate[C]+H+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4941 \"Acetoacetate[M]+H+[M]=Acetoacetate[C]+H+[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4942 \"Phenylpyruvate[C]=Phenylpyruvate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4942 \"Phenylpyruvate[C]=Phenylpyruvate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4948 \"Cholesterol[C]=Cholesterol[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4948 \"Cholesterol[C]=Cholesterol[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4949 \"Cholesterol[C]=Cholesterol[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4949 \"Cholesterol[C]=Cholesterol[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4951 \"Cholesterol[C]=Cholesterol[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4951 \"Cholesterol[C]=Cholesterol[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4954 \"N-Acylsphingosine[C]+H+[C]=N-Acylsphingosine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4954 \"N-Acylsphingosine[C]+H+[C]=N-Acylsphingosine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4956 \"Sarcosine[C]=Sarcosine[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4956 \"Sarcosine[C]=Sarcosine[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4960 \"Arachidonate[C]=Arachidonate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4960 \"Arachidonate[C]=Arachidonate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4963 \"3-Oxopropanoate[C]=3-Oxopropanoate[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4963 \"3-Oxopropanoate[C]=3-Oxopropanoate[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4964 \"3-Oxopropanoate[C]=3-Oxopropanoate[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4964 \"3-Oxopropanoate[C]=3-Oxopropanoate[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4965 \"3-Oxopropanoate[C]+H+[C]=3-Oxopropanoate[M]+H+[M] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4965 \"3-Oxopropanoate[C]+H+[C]=3-Oxopropanoate[M]+H+[M] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4966 \"10-Formyltetrahydrofolate[C]=10-Formyltetrahydrofolate[M] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4966 \"10-Formyltetrahydrofolate[C]=10-Formyltetrahydrofolate[M] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4968 \"Dimethylallyl diphosphate[C]=Dimethylallyl diphosphate[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4968 \"Dimethylallyl diphosphate[C]=Dimethylallyl diphosphate[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4969 \"dCMP[C]=dCMP[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4969 \"dCMP[C]=dCMP[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4971 \"Lipoamide[C]+H+[C]=Lipoamide[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4971 \"Lipoamide[C]+H+[C]=Lipoamide[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4973 \"Riboflavin[E]=Riboflavin[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4973 \"Riboflavin[E]=Riboflavin[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4976 \"D-Glycerate[C]+H+[C]=D-Glycerate[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4976 \"D-Glycerate[C]+H+[C]=D-Glycerate[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4978 \"alpha-D-Glucose[C]=alpha-D-Glucose[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4978 \"alpha-D-Glucose[C]=alpha-D-Glucose[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4991 \"Isocitrate[C]=Isocitrate[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4991 \"Isocitrate[C]=Isocitrate[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_4993 \"Sphingosine[M]+H+[M]=Sphingosine[C]+H+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_4993 \"Sphingosine[M]+H+[M]=Sphingosine[C]+H+[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5001 \"(S)-Dihydroorotate[C]+H+[C]=(S)-Dihydroorotate[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5001 \"(S)-Dihydroorotate[C]+H+[C]=(S)-Dihydroorotate[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5008 \"6-Phospho-D-gluconate[C]=6-Phospho-D-gluconate[R] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5008 \"6-Phospho-D-gluconate[C]=6-Phospho-D-gluconate[R] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5010 \"Phosphatidylethanolamine[C]=Phosphatidylethanolamine[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5010 \"Phosphatidylethanolamine[C]=Phosphatidylethanolamine[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5011 \"Phosphatidylethanolamine[C]=Phosphatidylethanolamine[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5011 \"Phosphatidylethanolamine[C]=Phosphatidylethanolamine[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5016 \"(S)-3-Hydroxy-3-methylglutaryl-CoA[C]=(S)-3-Hydroxy-3-methylglutaryl-CoA[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5016 \"(S)-3-Hydroxy-3-methylglutaryl-CoA[C]=(S)-3-Hydroxy-3-methylglutaryl-CoA[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5018 \"dAMP[C]=dAMP[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5018 \"dAMP[C]=dAMP[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5020 \"dGMP[C]=dGMP[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5020 \"dGMP[C]=dGMP[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5028 \"L-Pipecolate[C]=L-Pipecolate[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5028 \"L-Pipecolate[C]=L-Pipecolate[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5030 \"Stearoyl-CoA[C]=Stearoyl-CoA[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5030 \"Stearoyl-CoA[C]=Stearoyl-CoA[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5031 \"Stearoyl-CoA[C]=Stearoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5031 \"Stearoyl-CoA[C]=Stearoyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5032 \"Stearoyl-CoA[C]+H+[C]=Stearoyl-CoA[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5032 \"Stearoyl-CoA[C]+H+[C]=Stearoyl-CoA[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5039 \"Triacylglycerol[C]=Triacylglycerol[R]\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5039 \"Triacylglycerol[C]=Triacylglycerol[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5040 \"Prostaglandin H2[C]+H+[C]=Prostaglandin H2[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5040 \"Prostaglandin H2[C]+H+[C]=Prostaglandin H2[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5041 \"5-Aminolevulinate[M]=5-Aminolevulinate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5041 \"5-Aminolevulinate[M]=5-Aminolevulinate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5043 \"Sedoheptulose 1,7-bisphosphate[C]=Sedoheptulose 1,7-bisphosphate[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5043 \"Sedoheptulose 1,7-bisphosphate[C]=Sedoheptulose 1,7-bisphosphate[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5044 \"trans,trans-Farnesyl diphosphate[C]=trans,trans-Farnesyl diphosphate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5044 \"trans,trans-Farnesyl diphosphate[C]=trans,trans-Farnesyl diphosphate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5045 \"N6-(L-1,3-Dicarboxypropyl)-L-lysine[C]+H+[C]=N6-(L-1,3-Dicarboxypropyl)-L-lysine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5045 \"N6-(L-1,3-Dicarboxypropyl)-L-lysine[C]+H+[C]=N6-(L-1,3-Dicarboxypropyl)-L-lysine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5047 \"Nicotinamide D-ribonucleotide[E]=Nicotinamide D-ribonucleotide[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5047 \"Nicotinamide D-ribonucleotide[E]=Nicotinamide D-ribonucleotide[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5049 \"dTTP[C]=dTTP[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5049 \"dTTP[C]=dTTP[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5050 \"dTTP[C]=dTTP[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5050 \"dTTP[C]=dTTP[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5054 \"Tyramine[C]=Tyramine[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5054 \"Tyramine[C]=Tyramine[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5055 \"Tyramine[C]+H+[C]=Tyramine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5055 \"Tyramine[C]+H+[C]=Tyramine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5066 \"Methylglyoxal[X]=Methylglyoxal[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5066 \"Methylglyoxal[X]=Methylglyoxal[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5067 \"L-Noradrenaline[C]+H+[C]=L-Noradrenaline[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5067 \"L-Noradrenaline[C]+H+[C]=L-Noradrenaline[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5069 \"Sphingomyelin[C]=Sphingomyelin[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5069 \"Sphingomyelin[C]=Sphingomyelin[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5074 \"D-Glyceraldehyde[C]=D-Glyceraldehyde[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5074 \"D-Glyceraldehyde[C]=D-Glyceraldehyde[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5082 \"3-Sulfino-L-alanine[C]=3-Sulfino-L-alanine[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5082 \"3-Sulfino-L-alanine[C]=3-Sulfino-L-alanine[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5083 \"3-Sulfino-L-alanine[C]+L-Aspartate[M]=3-Sulfino-L-alanine[M]+L-Aspartate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5083 \"3-Sulfino-L-alanine[C]+L-Aspartate[M]=3-Sulfino-L-alanine[M]+L-Aspartate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5088 \"1,2-Diacyl-sn-glycerol[C]=1,2-Diacyl-sn-glycerol[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5088 \"1,2-Diacyl-sn-glycerol[C]=1,2-Diacyl-sn-glycerol[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5090 \"5-Formiminotetrahydrofolate[C]=5-Formiminotetrahydrofolate[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5090 \"5-Formiminotetrahydrofolate[C]=5-Formiminotetrahydrofolate[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5094 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[C]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5094 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[C]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5095 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[C]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5095 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[C]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5097 \"(5Z,13E)-(15S)-9alpha,15-Dihydroxy-11-oxoprosta-5,13-dienoate[C]=(5Z,13E)-(15S)-9alpha,15-Dihydroxy-11-oxoprosta-5,13-dienoate[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5097 \"(5Z,13E)-(15S)-9alpha,15-Dihydroxy-11-oxoprosta-5,13-dienoate[C]=(5Z,13E)-(15S)-9alpha,15-Dihydroxy-11-oxoprosta-5,13-dienoate[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5098 \"Betaine[C]=Betaine[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5098 \"Betaine[C]=Betaine[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5100 \"3-(2-Aminoethyl)-1H-indol-5-ol[C]+H+[C]=3-(2-Aminoethyl)-1H-indol-5-ol[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5100 \"3-(2-Aminoethyl)-1H-indol-5-ol[C]+H+[C]=3-(2-Aminoethyl)-1H-indol-5-ol[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5101 \"L-Adrenaline[C]+H+[C]=L-Adrenaline[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5101 \"L-Adrenaline[C]+H+[C]=L-Adrenaline[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5102 \"D-Sorbitol[E]=D-Sorbitol[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5102 \"D-Sorbitol[E]=D-Sorbitol[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5108 \"Naphthalene[C]=Naphthalene[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5108 \"Naphthalene[C]=Naphthalene[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5111 \"Sphinganine[C]=Sphinganine[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5111 \"Sphinganine[C]=Sphinganine[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5114 \"Pantothenate[C]+H+[C]=Pantothenate[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5114 \"Pantothenate[C]+H+[C]=Pantothenate[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5120 \"Estradiol-17beta[R]=Estradiol-17beta[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5120 \"Estradiol-17beta[R]=Estradiol-17beta[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5124 \"N,N-Dimethylglycine[C]=N,N-Dimethylglycine[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5124 \"N,N-Dimethylglycine[C]=N,N-Dimethylglycine[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5125 \"4-Guanidinobutanoate[C]+H+[C]=4-Guanidinobutanoate[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5125 \"4-Guanidinobutanoate[C]+H+[C]=4-Guanidinobutanoate[M]+H+[M]\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5126 \"(S)-2,3-Epoxysqualene[C]=(S)-2,3-Epoxysqualene[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5126 \"(S)-2,3-Epoxysqualene[C]=(S)-2,3-Epoxysqualene[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5140 \"3-(4-Hydroxyphenyl)pyruvate[C]=3-(4-Hydroxyphenyl)pyruvate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5140 \"3-(4-Hydroxyphenyl)pyruvate[C]=3-(4-Hydroxyphenyl)pyruvate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5143 \"Nicotinate D-ribonucleotide[E]=Nicotinate D-ribonucleotide[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5143 \"Nicotinate D-ribonucleotide[E]=Nicotinate D-ribonucleotide[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5144 \"Nicotinate D-ribonucleotide[C]=Nicotinate D-ribonucleotide[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5144 \"Nicotinate D-ribonucleotide[C]=Nicotinate D-ribonucleotide[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5147 \"1-Phosphatidyl-D-myo-inositol[C]+H+[C]=1-Phosphatidyl-D-myo-inositol[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5147 \"1-Phosphatidyl-D-myo-inositol[C]+H+[C]=1-Phosphatidyl-D-myo-inositol[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5149 \"3beta-Hydroxyandrost-5-en-17-one[C]=3beta-Hydroxyandrost-5-en-17-one[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5149 \"3beta-Hydroxyandrost-5-en-17-one[C]=3beta-Hydroxyandrost-5-en-17-one[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5150 \"3beta-Hydroxyandrost-5-en-17-one[C]=3beta-Hydroxyandrost-5-en-17-one[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5150 \"3beta-Hydroxyandrost-5-en-17-one[C]=3beta-Hydroxyandrost-5-en-17-one[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5151 \"3beta-Hydroxyandrost-5-en-17-one[R]=3beta-Hydroxyandrost-5-en-17-one[C] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5151 \"3beta-Hydroxyandrost-5-en-17-one[R]=3beta-Hydroxyandrost-5-en-17-one[C] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5155 \"D-Glucono-1,5-lactone 6-phosphate[C]=D-Glucono-1,5-lactone 6-phosphate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5155 \"D-Glucono-1,5-lactone 6-phosphate[C]=D-Glucono-1,5-lactone 6-phosphate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5157 \"D-myo-Inositol 1,4,5-trisphosphate[C]=D-myo-Inositol 1,4,5-trisphosphate[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5157 \"D-myo-Inositol 1,4,5-trisphosphate[C]=D-myo-Inositol 1,4,5-trisphosphate[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5160 \"1D-myo-Inositol 1,3,4,5-tetrakisphosphate[C]=1D-myo-Inositol 1,3,4,5-tetrakisphosphate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5160 \"1D-myo-Inositol 1,3,4,5-tetrakisphosphate[C]=1D-myo-Inositol 1,3,4,5-tetrakisphosphate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5161 \"1-Phosphatidyl-1D-myo-inositol 4-phosphate[M]+H+[M]=1-Phosphatidyl-1D-myo-inositol 4-phosphate[C]+H+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5161 \"1-Phosphatidyl-1D-myo-inositol 4-phosphate[M]+H+[M]=1-Phosphatidyl-1D-myo-inositol 4-phosphate[C]+H+[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5163 \"beta-D-Galactosyl-1,4-beta-D-glucosylceramide[C]=beta-D-Galactosyl-1,4-beta-D-glucosylceramide[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5163 \"beta-D-Galactosyl-1,4-beta-D-glucosylceramide[C]=beta-D-Galactosyl-1,4-beta-D-glucosylceramide[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5168 \"Prostaglandin I2[C]=Prostaglandin I2[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5168 \"Prostaglandin I2[C]=Prostaglandin I2[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5171 \"Linoleate[C]=Linoleate[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5171 \"Linoleate[C]=Linoleate[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5176 \"Psychosine[C]=Psychosine[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5176 \"Psychosine[C]=Psychosine[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5177 \"Psychosine[C]=Psychosine[L] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5177 \"Psychosine[C]=Psychosine[L] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5187 \"Aminoacetone[C]=Aminoacetone[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5187 \"Aminoacetone[C]=Aminoacetone[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5194 \"Acetylcholine[C]=Acetylcholine[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5194 \"Acetylcholine[C]=Acetylcholine[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5196 \"Eicosanoyl-CoA[C]=Eicosanoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5196 \"Eicosanoyl-CoA[C]=Eicosanoyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5198 \"Linoleoyl-CoA[C]=Linoleoyl-CoA[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5198 \"Linoleoyl-CoA[C]=Linoleoyl-CoA[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5200 \"Linoleoyl-CoA[C]+H+[C]=Linoleoyl-CoA[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5200 \"Linoleoyl-CoA[C]+H+[C]=Linoleoyl-CoA[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5202 \"Phytanoyl-CoA[C]=Phytanoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5202 \"Phytanoyl-CoA[C]=Phytanoyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5205 \"Methylmalonate[C]+H+[C]=Methylmalonate[M]+H+[M] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5205 \"Methylmalonate[C]+H+[C]=Methylmalonate[M]+H+[M] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5207 \"Protein serine[C]=Protein serine[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5207 \"Protein serine[C]=Protein serine[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5208 \"Protein serine[C]=Protein serine[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5208 \"Protein serine[C]=Protein serine[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5209 \"Arachidonyl-CoA[C]=Arachidonyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5209 \"Arachidonyl-CoA[C]=Arachidonyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5210 \"Arachidonyl-CoA[C]+H+[C]=Arachidonyl-CoA[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5210 \"Arachidonyl-CoA[C]+H+[C]=Arachidonyl-CoA[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5216 \"Xanthurenic acid[C]+H+[C]=Xanthurenic acid[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5216 \"Xanthurenic acid[C]+H+[C]=Xanthurenic acid[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5217 \"alpha-Tocopherol[C]=alpha-Tocopherol[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5217 \"alpha-Tocopherol[C]=alpha-Tocopherol[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5219 \"Chenodeoxycholate[C]=Chenodeoxycholate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5219 \"Chenodeoxycholate[C]=Chenodeoxycholate[R]\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5220 \"Chenodeoxycholate[C]=Chenodeoxycholate[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5220 \"Chenodeoxycholate[C]=Chenodeoxycholate[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5221 \"Chenodeoxycholate[C]+H+[C]=Chenodeoxycholate[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5221 \"Chenodeoxycholate[C]+H+[C]=Chenodeoxycholate[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5223 \"Estrone 3-sulfate[C]=Estrone 3-sulfate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5223 \"Estrone 3-sulfate[C]=Estrone 3-sulfate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5224 \"Estrone 3-sulfate[C]=Estrone 3-sulfate[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5224 \"Estrone 3-sulfate[C]=Estrone 3-sulfate[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5225 \"Estrone 3-sulfate[C]=Estrone 3-sulfate[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5225 \"Estrone 3-sulfate[C]=Estrone 3-sulfate[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5231 \"3-cis-Dodecenoyl-CoA[C]+H+[C]=3-cis-Dodecenoyl-CoA[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5231 \"3-cis-Dodecenoyl-CoA[C]+H+[C]=3-cis-Dodecenoyl-CoA[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5232 \"3-cis-Dodecenoyl-CoA[C]=3-cis-Dodecenoyl-CoA[X] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5232 \"3-cis-Dodecenoyl-CoA[C]=3-cis-Dodecenoyl-CoA[X] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5233 \"Ceramide 1-phosphate[C]=Ceramide 1-phosphate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5233 \"Ceramide 1-phosphate[C]=Ceramide 1-phosphate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5234 \"Ceramide 1-phosphate[C]=Ceramide 1-phosphate[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5234 \"Ceramide 1-phosphate[C]=Ceramide 1-phosphate[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5237 \"Coproporphyrinogen III[C]+H+[C]=Coproporphyrinogen III[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5237 \"Coproporphyrinogen III[C]+H+[C]=Coproporphyrinogen III[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5240 \"O-beta-D-Xylosylprotein[G]=O-beta-D-Xylosylprotein[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5240 \"O-beta-D-Xylosylprotein[G]=O-beta-D-Xylosylprotein[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5242 \"5-Formyltetrahydrofolate[C]+H+[C]=5-Formyltetrahydrofolate[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5242 \"5-Formyltetrahydrofolate[C]+H+[C]=5-Formyltetrahydrofolate[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5244 \"L-2-Amino-3-oxobutanoic acid[C]+H+[C]=L-2-Amino-3-oxobutanoic acid[M]+H+[M] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5244 \"L-2-Amino-3-oxobutanoic acid[C]+H+[C]=L-2-Amino-3-oxobutanoic acid[M]+H+[M] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5249 \"4-(2-Aminoethyl)-1,2-benzenediol[C]+H+[C]=4-(2-Aminoethyl)-1,2-benzenediol[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5249 \"4-(2-Aminoethyl)-1,2-benzenediol[C]+H+[C]=4-(2-Aminoethyl)-1,2-benzenediol[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5254 \"Dolichyl phosphate D-mannose[C]=Dolichyl phosphate D-mannose[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5254 \"Dolichyl phosphate D-mannose[C]=Dolichyl phosphate D-mannose[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5255 \"17beta-Hydroxyandrostan-3-one[C]=17beta-Hydroxyandrostan-3-one[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5255 \"17beta-Hydroxyandrostan-3-one[C]=17beta-Hydroxyandrostan-3-one[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5262 \"1-Acyl-sn-glycero-3-phosphocholine[C]=1-Acyl-sn-glycero-3-phosphocholine[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5262 \"1-Acyl-sn-glycero-3-phosphocholine[C]=1-Acyl-sn-glycero-3-phosphocholine[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5265 \"D-Glucosaminylphosphatidylinositol[C]=D-Glucosaminylphosphatidylinositol[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5265 \"D-Glucosaminylphosphatidylinositol[C]=D-Glucosaminylphosphatidylinositol[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5266 \"D-Glucosaminylphosphatidylinositol[C]=D-Glucosaminylphosphatidylinositol[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5266 \"D-Glucosaminylphosphatidylinositol[C]=D-Glucosaminylphosphatidylinositol[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5270 \"1-Organyl-2-lyso-sn-glycero-3-phosphocholine[C]+H+[C]=1-Organyl-2-lyso-sn-glycero-3-phosphocholine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5270 \"1-Organyl-2-lyso-sn-glycero-3-phosphocholine[C]+H+[C]=1-Organyl-2-lyso-sn-glycero-3-phosphocholine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5272 \"3alpha-Hydroxy-5beta-androstan-17-one[C]=3alpha-Hydroxy-5beta-androstan-17-one[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5272 \"3alpha-Hydroxy-5beta-androstan-17-one[C]=3alpha-Hydroxy-5beta-androstan-17-one[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5273 \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA[C]+H+[C]=(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA[M]+H+[M] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5273 \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA[C]+H+[C]=(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA[M]+H+[M] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5275 \"1-Acyl-sn-glycero-3-phosphoethanolamine[C]=1-Acyl-sn-glycero-3-phosphoethanolamine[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5275 \"1-Acyl-sn-glycero-3-phosphoethanolamine[C]=1-Acyl-sn-glycero-3-phosphoethanolamine[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5282 \"1-Alkyl-2-acetyl-sn-glycero-3-phosphocholine[C]=1-Alkyl-2-acetyl-sn-glycero-3-phosphocholine[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5282 \"1-Alkyl-2-acetyl-sn-glycero-3-phosphocholine[C]=1-Alkyl-2-acetyl-sn-glycero-3-phosphocholine[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5289 \"Trihexosylceramide[C]=Trihexosylceramide[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5289 \"Trihexosylceramide[C]=Trihexosylceramide[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5291 \"Trihexosylceramide[C]=Trihexosylceramide[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5291 \"Trihexosylceramide[C]=Trihexosylceramide[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5292 \"Alprostadil[C]=Alprostadil[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5292 \"Alprostadil[C]=Alprostadil[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5293 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA[C]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5293 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA[C]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5294 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA[R]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA[C]\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5294 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA[R]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5296 \"5(S)-HETE[C]=5(S)-HETE[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5296 \"5(S)-HETE[C]=5(S)-HETE[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5297 \"(5Z,9E,14Z)-(8xi,11R,12S)-11,12-Epoxy-8-hydroxyicosa-5,9,14-trienoic[C]=(5Z,9E,14Z)-(8xi,11R,12S)-11,12-Epoxy-8-hydroxyicosa-5,9,14-trienoic[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5297 \"(5Z,9E,14Z)-(8xi,11R,12S)-11,12-Epoxy-8-hydroxyicosa-5,9,14-trienoic[C]=(5Z,9E,14Z)-(8xi,11R,12S)-11,12-Epoxy-8-hydroxyicosa-5,9,14-trienoic[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5301 \"N-Acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-[C]=N-Acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5301 \"N-Acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-[C]=N-Acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-galactosyl-D-[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5302 \"2-Amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl)dihydropteridine[C]+H+[C]=2-Amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl)dihydropteridine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5302 \"2-Amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl)dihydropteridine[C]+H+[C]=2-Amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl)dihydropteridine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5303 \"alpha-N-Acetylneuraminyl-2,3-beta-D-galactosyl-1,3-N-acetyl-alpha-D-[C]=alpha-N-Acetylneuraminyl-2,3-beta-D-galactosyl-1,3-N-acetyl-alpha-D-[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5303 \"alpha-N-Acetylneuraminyl-2,3-beta-D-galactosyl-1,3-N-acetyl-alpha-D-[C]=alpha-N-Acetylneuraminyl-2,3-beta-D-galactosyl-1,3-N-acetyl-alpha-D-[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5305 \"D-Galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-[C]=D-Galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5305 \"D-Galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-[C]=D-Galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D-[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5308 \"2,3,2',3'-Tetrakis(3-hydroxytetradecanoyl)-D-glucosaminyl-1,6-beta-[C]=2,3,2',3'-Tetrakis(3-hydroxytetradecanoyl)-D-glucosaminyl-1,6-beta-[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5308 \"2,3,2',3'-Tetrakis(3-hydroxytetradecanoyl)-D-glucosaminyl-1,6-beta-[C]=2,3,2',3'-Tetrakis(3-hydroxytetradecanoyl)-D-glucosaminyl-1,6-beta-[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5309 \"N-Acetyl-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-N-acetyl-beta-D-[C]=N-Acetyl-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-N-acetyl-beta-D-[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5309 \"N-Acetyl-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-N-acetyl-beta-D-[C]=N-Acetyl-D-glucosaminyl-1,3-beta-D-galactosyl-1,4-N-acetyl-beta-D-[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5310 \"2-(alpha-Hydroxyethyl)thiamine diphosphate[C]+H+[C]=2-(alpha-Hydroxyethyl)thiamine diphosphate[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5310 \"2-(alpha-Hydroxyethyl)thiamine diphosphate[C]+H+[C]=2-(alpha-Hydroxyethyl)thiamine diphosphate[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5311 \"N-Methylhistamine[C]+H+[C]=N-Methylhistamine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5311 \"N-Methylhistamine[C]+H+[C]=N-Methylhistamine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5312 \"Imidazole-4-acetaldehyde[E]=Imidazole-4-acetaldehyde[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5312 \"Imidazole-4-acetaldehyde[E]=Imidazole-4-acetaldehyde[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5313 \"Imidazole-4-acetaldehyde[C]+H+[C]=Imidazole-4-acetaldehyde[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5313 \"Imidazole-4-acetaldehyde[C]+H+[C]=Imidazole-4-acetaldehyde[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5316 \"trans-Dec-2-enoyl-CoA[C]+H+[C]=trans-Dec-2-enoyl-CoA[M]+H+[M] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5316 \"trans-Dec-2-enoyl-CoA[C]+H+[C]=trans-Dec-2-enoyl-CoA[M]+H+[M] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5317 \"trans-Dec-2-enoyl-CoA[C]=trans-Dec-2-enoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5317 \"trans-Dec-2-enoyl-CoA[C]=trans-Dec-2-enoyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5322 \"2-Hydroxyestradiol-17beta[C]=2-Hydroxyestradiol-17beta[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5322 \"2-Hydroxyestradiol-17beta[C]=2-Hydroxyestradiol-17beta[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5324 \"Phenethylamine[C]=Phenethylamine[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5324 \"Phenethylamine[C]=Phenethylamine[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5325 \"Phenethylamine[C]+H+[C]=Phenethylamine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5325 \"Phenethylamine[C]+H+[C]=Phenethylamine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5331 \"5(S)-HPETE[C]=5(S)-HPETE[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5331 \"5(S)-HPETE[C]=5(S)-HPETE[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5333 \"3-beta-D-Galactosyl-sn-glycerol[C]=3-beta-D-Galactosyl-sn-glycerol[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5333 \"3-beta-D-Galactosyl-sn-glycerol[C]=3-beta-D-Galactosyl-sn-glycerol[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5334 \"3-beta-D-Galactosyl-sn-glycerol[C]=3-beta-D-Galactosyl-sn-glycerol[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5334 \"3-beta-D-Galactosyl-sn-glycerol[C]=3-beta-D-Galactosyl-sn-glycerol[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5335 \"3-beta-D-Galactosyl-sn-glycerol[C]=3-beta-D-Galactosyl-sn-glycerol[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5335 \"3-beta-D-Galactosyl-sn-glycerol[C]=3-beta-D-Galactosyl-sn-glycerol[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5336 \"Vitamin D3[C]+H+[C]=Vitamin D3[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5336 \"Vitamin D3[C]+H+[C]=Vitamin D3[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5338 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestane[C]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestane[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5338 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestane[C]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestane[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5343 \"21-Deoxycortisol[R]=21-Deoxycortisol[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5343 \"21-Deoxycortisol[R]=21-Deoxycortisol[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5351 \"Deoxyinosine[C]=Deoxyinosine[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5351 \"Deoxyinosine[C]=Deoxyinosine[E] (reversible)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5357 \"3-Methoxytyramine[C]+H+[C]=3-Methoxytyramine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5357 \"3-Methoxytyramine[C]+H+[C]=3-Methoxytyramine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5358 \"L-Metanephrine[C]+H+[C]=L-Metanephrine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5358 \"L-Metanephrine[C]+H+[C]=L-Metanephrine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5359 \"L-Normetanephrine[C]+H+[C]=L-Normetanephrine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5359 \"L-Normetanephrine[C]+H+[C]=L-Normetanephrine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5365 \"Uroporphyrinogen I[C]+H+[C]=Uroporphyrinogen I[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5365 \"Uroporphyrinogen I[C]+H+[C]=Uroporphyrinogen I[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5371 \"(alpha-D-Mannosyl)4-beta-D-mannosyl-[C]=(alpha-D-Mannosyl)4-beta-D-mannosyl-[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5371 \"(alpha-D-Mannosyl)4-beta-D-mannosyl-[C]=(alpha-D-Mannosyl)4-beta-D-mannosyl-[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5374 \"Leukotriene E4[C]=Leukotriene E4[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5374 \"Leukotriene E4[C]=Leukotriene E4[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5377 \"Phosphatidylinositol-3,4,5-trisphosphate[C]=Phosphatidylinositol-3,4,5-trisphosphate[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5377 \"Phosphatidylinositol-3,4,5-trisphosphate[C]=Phosphatidylinositol-3,4,5-trisphosphate[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5382 \"Digalactosylceramide[C]=Digalactosylceramide[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5382 \"Digalactosylceramide[C]=Digalactosylceramide[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5385 \"S-Glutaryldihydrolipoamide[C]+H+[C]=S-Glutaryldihydrolipoamide[M]+H+[M] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5385 \"S-Glutaryldihydrolipoamide[C]+H+[C]=S-Glutaryldihydrolipoamide[M]+H+[M] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5389 \"Lipoxin A4[C]=Lipoxin A4[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5389 \"Lipoxin A4[C]=Lipoxin A4[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5390 \"Lipoxin B4[C]=Lipoxin B4[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5390 \"Lipoxin B4[C]=Lipoxin B4[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5398 \"S-(PGA1)-glutathione[C]=S-(PGA1)-glutathione[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5398 \"S-(PGA1)-glutathione[C]=S-(PGA1)-glutathione[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5402 \"Anandamide[C]=Anandamide[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5402 \"Anandamide[C]=Anandamide[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5405 \"Dihydroceramide[C]=Dihydroceramide[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5405 \"Dihydroceramide[C]=Dihydroceramide[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5408 \"2-Arachidonoylglycerol[C]=2-Arachidonoylglycerol[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5408 \"2-Arachidonoylglycerol[C]=2-Arachidonoylglycerol[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5409 \"2-Arachidonoylglycerol[C]=2-Arachidonoylglycerol[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5409 \"2-Arachidonoylglycerol[C]=2-Arachidonoylglycerol[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5412 \"alpha-Tocotrienol[C]=alpha-Tocotrienol[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5412 \"alpha-Tocotrienol[C]=alpha-Tocotrienol[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5415 \"5,6-EET[C]=5,6-EET[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5415 \"5,6-EET[C]=5,6-EET[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5416 \"8,9-EET[C]=8,9-EET[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5416 \"8,9-EET[C]=8,9-EET[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5417 \"11,12-EET[C]=11,12-EET[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5417 \"11,12-EET[C]=11,12-EET[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5418 \"14,15-EET[C]=14,15-EET[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5418 \"14,15-EET[C]=14,15-EET[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5419 \"(1R,2S)-Naphthalene 1,2-oxide[C]=(1R,2S)-Naphthalene 1,2-oxide[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5419 \"(1R,2S)-Naphthalene 1,2-oxide[C]=(1R,2S)-Naphthalene 1,2-oxide[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5420 \"(1R,2S)-Naphthalene 1,2-oxide[C]+H+[C]=(1R,2S)-Naphthalene 1,2-oxide[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5420 \"(1R,2S)-Naphthalene 1,2-oxide[C]+H+[C]=(1R,2S)-Naphthalene 1,2-oxide[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5429 \"Fe2+[C]+H+[C]=Fe2+[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5429 \"Fe2+[C]+H+[C]=Fe2+[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5434 \"Bromobenzene-3,4-dihydrodiol[C]=Bromobenzene-3,4-dihydrodiol[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5434 \"Bromobenzene-3,4-dihydrodiol[C]=Bromobenzene-3,4-dihydrodiol[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5436 \"4-Bromophenol-2,3-epoxide[R]=4-Bromophenol-2,3-epoxide[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5436 \"4-Bromophenol-2,3-epoxide[R]=4-Bromophenol-2,3-epoxide[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5449 \"S-(Formylmethyl)glutathione[C]=S-(Formylmethyl)glutathione[R] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5449 \"S-(Formylmethyl)glutathione[C]=S-(Formylmethyl)glutathione[R] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5451 \"Glutathione episulfonium ion[C]=Glutathione episulfonium ion[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5451 \"Glutathione episulfonium ion[C]=Glutathione episulfonium ion[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5454 \"12-hydroxyeicosatetraenoate[C]=12-hydroxyeicosatetraenoate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5454 \"12-hydroxyeicosatetraenoate[C]=12-hydroxyeicosatetraenoate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5455 \"3-oxolaur-cis-5-enoyl-CoA[C]=3-oxolaur-cis-5-enoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5455 \"3-oxolaur-cis-5-enoyl-CoA[C]=3-oxolaur-cis-5-enoyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5456 \"3-oxolaur-cis-5-enoyl-CoA[C]+H+[C]=3-oxolaur-cis-5-enoyl-CoA[M]+H+[M] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5456 \"3-oxolaur-cis-5-enoyl-CoA[C]+H+[C]=3-oxolaur-cis-5-enoyl-CoA[M]+H+[M] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5467 \"palmitoleoyl-CoA[C]+H+[C]=palmitoleoyl-CoA[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5467 \"palmitoleoyl-CoA[C]+H+[C]=palmitoleoyl-CoA[M]+H+[M]\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5488 \"spermine dialdehyde[E]=spermine dialdehyde[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5488 \"spermine dialdehyde[E]=spermine dialdehyde[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5489 \"spermine dialdehyde[C]=spermine dialdehyde[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5489 \"spermine dialdehyde[C]=spermine dialdehyde[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5490 \"spermine dialdehyde[C]+H+[C]=spermine dialdehyde[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5490 \"spermine dialdehyde[C]+H+[C]=spermine dialdehyde[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5492 \"spermidine dialdehyde[E]=spermidine dialdehyde[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5492 \"spermidine dialdehyde[E]=spermidine dialdehyde[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5493 \"spermidine dialdehyde[C]=spermidine dialdehyde[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5493 \"spermidine dialdehyde[C]=spermidine dialdehyde[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5494 \"spermidine dialdehyde[C]+H+[C]=spermidine dialdehyde[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5494 \"spermidine dialdehyde[C]+H+[C]=spermidine dialdehyde[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5497 \"20-oxo-leukotriene B4[C]+H+[C]=20-oxo-leukotriene B4[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5497 \"20-oxo-leukotriene B4[C]+H+[C]=20-oxo-leukotriene B4[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5499 \"4-hydroxy-17beta-estradiol[C]=4-hydroxy-17beta-estradiol[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5499 \"4-hydroxy-17beta-estradiol[C]=4-hydroxy-17beta-estradiol[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5502 \"trans-docos-2-enoyl-CoA[C]=trans-docos-2-enoyl-CoA[N]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5502 \"trans-docos-2-enoyl-CoA[C]=trans-docos-2-enoyl-CoA[N]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5506 \"3-oxodocosanoyl-CoA[E]=3-oxodocosanoyl-CoA[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5506 \"3-oxodocosanoyl-CoA[E]=3-oxodocosanoyl-CoA[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5509 \"3-oxotetracosanoyl-CoA[N]=3-oxotetracosanoyl-CoA[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5509 \"3-oxotetracosanoyl-CoA[N]=3-oxotetracosanoyl-CoA[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5512 \"tetracosanoyl-CoA[C]=tetracosanoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5512 \"tetracosanoyl-CoA[C]=tetracosanoyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5530 \"trans,cis,cis-2,7,10-hexadecatrienoyl-CoA[C]+H+[C]=trans,cis,cis-2,7,10-hexadecatrienoyl-CoA[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5530 \"trans,cis,cis-2,7,10-hexadecatrienoyl-CoA[C]+H+[C]=trans,cis,cis-2,7,10-hexadecatrienoyl-CoA[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5531 \"trans,cis,cis-2,7,10-hexadecatrienoyl-CoA[C]=trans,cis,cis-2,7,10-hexadecatrienoyl-CoA[X] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5531 \"trans,cis,cis-2,7,10-hexadecatrienoyl-CoA[C]=trans,cis,cis-2,7,10-hexadecatrienoyl-CoA[X] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5536 \"9-hydroxyoctadecadienoate[C]=9-hydroxyoctadecadienoate[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5536 \"9-hydroxyoctadecadienoate[C]=9-hydroxyoctadecadienoate[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5537 \"9-hydroxyoctadecadienoate[C]=9-hydroxyoctadecadienoate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5537 \"9-hydroxyoctadecadienoate[C]=9-hydroxyoctadecadienoate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5540 \"timnodonate[C]=timnodonate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5540 \"timnodonate[C]=timnodonate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5541 \"timnodonate[C]=timnodonate[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5541 \"timnodonate[C]=timnodonate[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5547 \"3(S)-3-hydroxydodecen-(5Z)-oyl-CoA[C]=3(S)-3-hydroxydodecen-(5Z)-oyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5547 \"3(S)-3-hydroxydodecen-(5Z)-oyl-CoA[C]=3(S)-3-hydroxydodecen-(5Z)-oyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5548 \"3(S)-3-hydroxydodecen-(5Z)-oyl-CoA[C]+H+[C]=3(S)-3-hydroxydodecen-(5Z)-oyl-CoA[M]+H+[M] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5548 \"3(S)-3-hydroxydodecen-(5Z)-oyl-CoA[C]+H+[C]=3(S)-3-hydroxydodecen-(5Z)-oyl-CoA[M]+H+[M] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5549 \"3,3'-diiodo-L-thyronine[C]=3,3'-diiodo-L-thyronine[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5549 \"3,3'-diiodo-L-thyronine[C]=3,3'-diiodo-L-thyronine[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5550 \"3,3'-diiodo-L-thyronine 4'-O-sulfate[C]=3,3'-diiodo-L-thyronine 4'-O-sulfate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5550 \"3,3'-diiodo-L-thyronine 4'-O-sulfate[C]=3,3'-diiodo-L-thyronine 4'-O-sulfate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5559 \"cis-6-dodecenoyl-CoA[C]=cis-6-dodecenoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5559 \"cis-6-dodecenoyl-CoA[C]=cis-6-dodecenoyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5560 \"cis-6-dodecenoyl-CoA[M]+H+[M]=cis-6-dodecenoyl-CoA[C]+H+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5560 \"cis-6-dodecenoyl-CoA[M]+H+[M]=cis-6-dodecenoyl-CoA[C]+H+[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5564 \"tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA[C]=tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5564 \"tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA[C]=tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5565 \"docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA[C]=docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5565 \"docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA[C]=docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5566 \"docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA[C]=docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA[X] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5566 \"docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA[C]=docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA[X] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5567 \"adrenoyl-CoA[C]=adrenoyl-CoA[X] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5567 \"adrenoyl-CoA[C]=adrenoyl-CoA[X] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5568 \"tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA[C]=tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5568 \"tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA[C]=tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA[X]\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5569 \"3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-27-al[C]=3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-27-al[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5569 \"3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-27-al[C]=3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-27-al[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5572 \"dopaminochrome[C]=dopaminochrome[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5572 \"dopaminochrome[C]=dopaminochrome[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5574 \"10,11-dihydro-12-epi-leukotriene B4[C]=10,11-dihydro-12-epi-leukotriene B4[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5574 \"10,11-dihydro-12-epi-leukotriene B4[C]=10,11-dihydro-12-epi-leukotriene B4[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5575 \"10,11-dihydro-12-epi-leukotriene B4[C]+H+[C]=10,11-dihydro-12-epi-leukotriene B4[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5575 \"10,11-dihydro-12-epi-leukotriene B4[C]+H+[C]=10,11-dihydro-12-epi-leukotriene B4[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5576 \"10,11-dihydro-12R-hydroxy-leukotriene C4[C]=10,11-dihydro-12R-hydroxy-leukotriene C4[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5576 \"10,11-dihydro-12R-hydroxy-leukotriene C4[C]=10,11-dihydro-12R-hydroxy-leukotriene C4[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5577 \"10,11-dihydro-12R-hydroxy-leukotriene C4[C]+H+[C]=10,11-dihydro-12R-hydroxy-leukotriene C4[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5577 \"10,11-dihydro-12R-hydroxy-leukotriene C4[C]+H+[C]=10,11-dihydro-12R-hydroxy-leukotriene C4[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5579 \"trans-3-decenoyl-CoA[C]=trans-3-decenoyl-CoA[X] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5579 \"trans-3-decenoyl-CoA[C]=trans-3-decenoyl-CoA[X] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5584 \"24(R),25(R)-varanoyl-CoA[C]=24(R),25(R)-varanoyl-CoA[X] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5584 \"24(R),25(R)-varanoyl-CoA[C]=24(R),25(R)-varanoyl-CoA[X] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5585 \"24(R),25(R)-varanoyl-CoA[C]+H+[C]=24(R),25(R)-varanoyl-CoA[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5585 \"24(R),25(R)-varanoyl-CoA[C]+H+[C]=24(R),25(R)-varanoyl-CoA[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5588 \"5-oxo-6-trans-leukotriene B4[C]=5-oxo-6-trans-leukotriene B4[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5588 \"5-oxo-6-trans-leukotriene B4[C]=5-oxo-6-trans-leukotriene B4[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5589 \"5-oxo-6-trans-leukotriene B4[C]+H+[C]=5-oxo-6-trans-leukotriene B4[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5589 \"5-oxo-6-trans-leukotriene B4[C]+H+[C]=5-oxo-6-trans-leukotriene B4[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5590 \"17beta-estradiol-3,4-quinone[C]=17beta-estradiol-3,4-quinone[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5590 \"17beta-estradiol-3,4-quinone[C]=17beta-estradiol-3,4-quinone[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5591 \"17beta-estradiol-3,4-quinone[C]=17beta-estradiol-3,4-quinone[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5591 \"17beta-estradiol-3,4-quinone[C]=17beta-estradiol-3,4-quinone[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5592 \"17beta-estradiol-3,4-quinone[C]+H+[C]=17beta-estradiol-3,4-quinone[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5592 \"17beta-estradiol-3,4-quinone[C]+H+[C]=17beta-estradiol-3,4-quinone[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5593 \"estrone-3,4-quinone[C]=estrone-3,4-quinone[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5593 \"estrone-3,4-quinone[C]=estrone-3,4-quinone[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5594 \"estrone-3,4-quinone[C]=estrone-3,4-quinone[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5594 \"estrone-3,4-quinone[C]=estrone-3,4-quinone[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5595 \"estrone-3,4-quinone[C]+H+[C]=estrone-3,4-quinone[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5595 \"estrone-3,4-quinone[C]+H+[C]=estrone-3,4-quinone[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5598 \"estrone-3,4-semiquinone[C]=estrone-3,4-semiquinone[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5598 \"estrone-3,4-semiquinone[C]=estrone-3,4-semiquinone[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5599 \"estrone-2,3-semiquinone[C]=estrone-2,3-semiquinone[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5599 \"estrone-2,3-semiquinone[C]=estrone-2,3-semiquinone[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5622 \"6,7-dihydro-leukotriene B4[C]=6,7-dihydro-leukotriene B4[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5622 \"6,7-dihydro-leukotriene B4[C]=6,7-dihydro-leukotriene B4[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5623 \"6,7-dihydro-leukotriene B4[C]=6,7-dihydro-leukotriene B4[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5623 \"6,7-dihydro-leukotriene B4[C]=6,7-dihydro-leukotriene B4[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5625 \"12-oxo-20-carboxy-leukotriene B4[C]=12-oxo-20-carboxy-leukotriene B4[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5625 \"12-oxo-20-carboxy-leukotriene B4[C]=12-oxo-20-carboxy-leukotriene B4[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5626 \"12-oxo-20-carboxy-leukotriene B4[C]+H+[C]=12-oxo-20-carboxy-leukotriene B4[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5626 \"12-oxo-20-carboxy-leukotriene B4[C]+H+[C]=12-oxo-20-carboxy-leukotriene B4[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5630 \"12-oxo-c-LTB3[C]=12-oxo-c-LTB3[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5630 \"12-oxo-c-LTB3[C]=12-oxo-c-LTB3[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5631 \"adrenochrome[C]=adrenochrome[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5631 \"adrenochrome[C]=adrenochrome[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5641 \"13'-carboxy-alpha-tocopherol[R]=13'-carboxy-alpha-tocopherol[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5641 \"13'-carboxy-alpha-tocopherol[R]=13'-carboxy-alpha-tocopherol[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5643 \"11'-carboxy-alpha-chromanol[C]+H+[C]=11'-carboxy-alpha-chromanol[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5643 \"11'-carboxy-alpha-chromanol[C]+H+[C]=11'-carboxy-alpha-chromanol[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5645 \"20-COOH-10,11-dihydro-LTB4[C]=20-COOH-10,11-dihydro-LTB4[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5645 \"20-COOH-10,11-dihydro-LTB4[C]=20-COOH-10,11-dihydro-LTB4[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5649 \"3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoyl-CoA[C]=3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoyl-CoA[X] (reversible)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5649 \"3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoyl-CoA[C]=3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoyl-CoA[X] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5651 \"10,11-dihydro-LTB4-CoA[C]=10,11-dihydro-LTB4-CoA[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5651 \"10,11-dihydro-LTB4-CoA[C]=10,11-dihydro-LTB4-CoA[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5654 \"androsterone sulfate[C]=androsterone sulfate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5654 \"androsterone sulfate[C]=androsterone sulfate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5655 \"androsterone sulfate[C]=androsterone sulfate[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5655 \"androsterone sulfate[C]=androsterone sulfate[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5659 \"18-CoA-18-oxo-dinorleukotriene B4[C]=18-CoA-18-oxo-dinorleukotriene B4[X] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5659 \"18-CoA-18-oxo-dinorleukotriene B4[C]=18-CoA-18-oxo-dinorleukotriene B4[X] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5664 \"prostaglandin H1[C]=prostaglandin H1[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5664 \"prostaglandin H1[C]=prostaglandin H1[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5665 \"S-(PGA2)-glutathione[C]=S-(PGA2)-glutathione[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5665 \"S-(PGA2)-glutathione[C]=S-(PGA2)-glutathione[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5666 \"S-(PGA2)-glutathione[C]+H+[C]=S-(PGA2)-glutathione[M]+H+[M] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5666 \"S-(PGA2)-glutathione[C]+H+[C]=S-(PGA2)-glutathione[M]+H+[M] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5668 \"S-(9-deoxy-delta12-PGD2)-glutathione[C]=S-(9-deoxy-delta12-PGD2)-glutathione[R] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5668 \"S-(9-deoxy-delta12-PGD2)-glutathione[C]=S-(9-deoxy-delta12-PGD2)-glutathione[R] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5669 \"S-(9-deoxy-delta12-PGD2)-glutathione[C]+H+[C]=S-(9-deoxy-delta12-PGD2)-glutathione[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5669 \"S-(9-deoxy-delta12-PGD2)-glutathione[C]+H+[C]=S-(9-deoxy-delta12-PGD2)-glutathione[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5670 \"S-(9-deoxy-delta9,12-PGD2)-glutathione[C]=S-(9-deoxy-delta9,12-PGD2)-glutathione[R] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5670 \"S-(9-deoxy-delta9,12-PGD2)-glutathione[C]=S-(9-deoxy-delta9,12-PGD2)-glutathione[R] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5671 \"S-(9-deoxy-delta9,12-PGD2)-glutathione[C]+H+[C]=S-(9-deoxy-delta9,12-PGD2)-glutathione[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5671 \"S-(9-deoxy-delta9,12-PGD2)-glutathione[C]+H+[C]=S-(9-deoxy-delta9,12-PGD2)-glutathione[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5673 \"5,12-DiHETE[C]=5,12-DiHETE[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5673 \"5,12-DiHETE[C]=5,12-DiHETE[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5674 \"5,12-DiHETE[C]=5,12-DiHETE[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5674 \"5,12-DiHETE[C]=5,12-DiHETE[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5675 \"5,12-DiHETE[C]+H+[C]=5,12-DiHETE[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5675 \"5,12-DiHETE[C]+H+[C]=5,12-DiHETE[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5680 \"1-O-1'-alkenyl-2-acyl-phosphatidylethanolamine[C]=1-O-1'-alkenyl-2-acyl-phosphatidylethanolamine[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5680 \"1-O-1'-alkenyl-2-acyl-phosphatidylethanolamine[C]=1-O-1'-alkenyl-2-acyl-phosphatidylethanolamine[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5686 \"alk-1-enyl-2-lyso-glycerophosphate[C]=alk-1-enyl-2-lyso-glycerophosphate[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5686 \"alk-1-enyl-2-lyso-glycerophosphate[C]=alk-1-enyl-2-lyso-glycerophosphate[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5691 \"1-acyl-2-lyso-phosphatidylserine[C]=1-acyl-2-lyso-phosphatidylserine[L]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5691 \"1-acyl-2-lyso-phosphatidylserine[C]=1-acyl-2-lyso-phosphatidylserine[L]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5692 \"1-acyl-2-lyso-phosphatidylserine[C]+H+[C]=1-acyl-2-lyso-phosphatidylserine[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5692 \"1-acyl-2-lyso-phosphatidylserine[C]+H+[C]=1-acyl-2-lyso-phosphatidylserine[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5695 \"leukotriene B5[C]=leukotriene B5[R]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5695 \"leukotriene B5[C]=leukotriene B5[R]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5696 \"5-HEPE[C]=5-HEPE[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5696 \"5-HEPE[C]=5-HEPE[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5700 \"1-alkyl-2-acyl-sn-glycero-3-phosphocholine[C]=1-alkyl-2-acyl-sn-glycero-3-phosphocholine[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5700 \"1-alkyl-2-acyl-sn-glycero-3-phosphocholine[C]=1-alkyl-2-acyl-sn-glycero-3-phosphocholine[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5740 \"(Gal)2 (GalNAc)1 (Glc)1 (Neu5Ac)1 (Cer)1[C]=(Gal)2 (GalNAc)1 (Glc)1 (Neu5Ac)1 (Cer)1[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5740 \"(Gal)2 (GalNAc)1 (Glc)1 (Neu5Ac)1 (Cer)1[C]=(Gal)2 (GalNAc)1 (Glc)1 (Neu5Ac)1 (Cer)1[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5741 \"(Gal)2 (GalNAc)1 (Glc)1 (Neu5Ac)1 (Cer)1[C]=(Gal)2 (GalNAc)1 (Glc)1 (Neu5Ac)1 (Cer)1[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5741 \"(Gal)2 (GalNAc)1 (Glc)1 (Neu5Ac)1 (Cer)1[C]=(Gal)2 (GalNAc)1 (Glc)1 (Neu5Ac)1 (Cer)1[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5761 \"dCDP[M]+dUDP[C]=dCDP[C]+dUDP[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5761 \"dCDP[M]+dUDP[C]=dCDP[C]+dUDP[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5780 \"dCDP[M]+dGDP[C]=dCDP[C]+dGDP[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5780 \"dCDP[M]+dGDP[C]=dCDP[C]+dGDP[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5809 \"gamma-Linolenoyl-CoA[C]=gamma-Linolenoyl-CoA[X]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5809 \"gamma-Linolenoyl-CoA[C]=gamma-Linolenoyl-CoA[X]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5811 \"ADPribose[E]=ADPribose[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5811 \"ADPribose[E]=ADPribose[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5814 \"Guanine[E]=Guanine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5814 \"Guanine[E]=Guanine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5831 \"L-Asparagine[E]+L-Cysteine[C]+Na+[E]=L-Asparagine[C]+L-Cysteine[E]+Na+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5831 \"L-Asparagine[E]+L-Cysteine[C]+Na+[E]=L-Asparagine[C]+L-Cysteine[E]+Na+[C]\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5846 \"Cl-[E]+Glycine[E]+(2)Na+[E]=Cl-[C]+Glycine[C]+(2)Na+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5846 \"Cl-[E]+Glycine[E]+(2)Na+[E]=Cl-[C]+Glycine[C]+(2)Na+[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5847 \"Betaine[E]+(2)Na+[E]=Betaine[C]+(2)Na+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5847 \"Betaine[E]+(2)Na+[E]=Betaine[C]+(2)Na+[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5864 \"L-Carnitine[E]=L-Carnitine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5864 \"L-Carnitine[E]=L-Carnitine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5869 \"ATP[C]+H2O[C]+Lipoate[E]+(2)Na+[E]=ADP[C]+H+[C]+Lipoate[C]+(2)Na+[C]+Orthophosphate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5869 \"ATP[C]+H2O[C]+Lipoate[E]+(2)Na+[E]=ADP[C]+H+[C]+Lipoate[C]+(2)Na+[C]+Orthophosphate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5870 \"ATP[C]+H2O[C]+(2)Na+[E]+Pantothenate[E]=ADP[C]+H+[C]+(2)Na+[C]+Orthophosphate[C]+Pantothenate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5870 \"ATP[C]+H2O[C]+(2)Na+[E]+Pantothenate[E]=ADP[C]+H+[C]+(2)Na+[C]+Orthophosphate[C]+Pantothenate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5872 \"Citrate[E]+(2)Na+[E]=Citrate[C]+(2)Na+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5872 \"Citrate[E]+(2)Na+[E]=Citrate[C]+(2)Na+[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5877 \"Bilirubin[E]+HCO3-[C]=Bilirubin[C]+HCO3-[E]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5877 \"Bilirubin[E]+HCO3-[C]=Bilirubin[C]+HCO3-[E]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5893 \"Heme[E]=Heme[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5893 \"Heme[E]=Heme[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5894 \"H+[E]+myo-Inositol[E]=H+[C]+myo-Inositol[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5894 \"H+[E]+myo-Inositol[E]=H+[C]+myo-Inositol[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5896 \"L-Tryptophan[E]=L-Tryptophan[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5896 \"L-Tryptophan[E]=L-Tryptophan[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5908 \"Na+[C]+Octadecanoic acid[C]=Na+[E]+Octadecanoic acid[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5908 \"Na+[C]+Octadecanoic acid[C]=Na+[E]+Octadecanoic acid[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5910 \"ATP[C]+H2O[C]+Phosphatidylserine[E]=ADP[C]+H+[C]+Orthophosphate[C]+Phosphatidylserine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5910 \"ATP[C]+H2O[C]+Phosphatidylserine[E]=ADP[C]+H+[C]+Orthophosphate[C]+Phosphatidylserine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5911 \"H+[C]+Thiamin[E]=H+[E]+Thiamin[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5911 \"H+[C]+Thiamin[E]=H+[E]+Thiamin[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5912 \"HCO3-[C]+Leukotriene C4[E]=HCO3-[E]+Leukotriene C4[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5912 \"HCO3-[C]+Leukotriene C4[E]=HCO3-[E]+Leukotriene C4[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5924 \"L-Phenylalanine[E]=L-Phenylalanine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5924 \"L-Phenylalanine[E]=L-Phenylalanine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5931 \"5-Methyltetrahydrofolate[E]=5-Methyltetrahydrofolate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5931 \"5-Methyltetrahydrofolate[E]=5-Methyltetrahydrofolate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5932 \"Na+[E]+L-Proline[E]=Na+[C]+L-Proline[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5932 \"Na+[E]+L-Proline[E]=Na+[C]+L-Proline[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5947 \"Na+[E]+L-Serine[C]+L-Threonine[E]=Na+[C]+L-Serine[E]+L-Threonine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5947 \"Na+[E]+L-Serine[C]+L-Threonine[E]=Na+[C]+L-Serine[E]+L-Threonine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5952 \"D-Glucosamine[E]=D-Glucosamine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5952 \"D-Glucosamine[E]=D-Glucosamine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5959 \"L-Leucine[C]+L-Methionine[E]=L-Leucine[E]+L-Methionine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5959 \"L-Leucine[C]+L-Methionine[E]=L-Leucine[E]+L-Methionine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5967 \"L-Lysine[E]=L-Lysine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5967 \"L-Lysine[E]=L-Lysine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5968 \"L-Arginine[E]=L-Arginine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5968 \"L-Arginine[E]=L-Arginine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5971 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[E]+(2)Na+[E]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[C]+(2)Na+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5971 \"3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[E]+(2)Na+[E]=3alpha,7alpha,12alpha-Trihydroxy-5beta-cholanate[C]+(2)Na+[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5989 \"HCO3-[C]+Prostaglandin E2[E]=HCO3-[E]+Prostaglandin E2[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5989 \"HCO3-[C]+Prostaglandin E2[E]=HCO3-[E]+Prostaglandin E2[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5991 \"Estrone 3-sulfate[E]+HCO3-[C]=Estrone 3-sulfate[C]+HCO3-[E]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5991 \"Estrone 3-sulfate[E]+HCO3-[C]=Estrone 3-sulfate[C]+HCO3-[E]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5992 \"3beta-Hydroxyandrost-5-en-17-one 3-sulfate[E]+HCO3-[C]=3beta-Hydroxyandrost-5-en-17-one 3-sulfate[C]+HCO3-[E]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5992 \"3beta-Hydroxyandrost-5-en-17-one 3-sulfate[E]+HCO3-[C]=3beta-Hydroxyandrost-5-en-17-one 3-sulfate[C]+HCO3-[E]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_5994 \"HCO3-[C]+Triiodothyronine[E]=HCO3-[E]+Triiodothyronine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_5994 \"HCO3-[C]+Triiodothyronine[E]=HCO3-[E]+Triiodothyronine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6006 \"L-Leucine[E]=L-Leucine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6006 \"L-Leucine[E]=L-Leucine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6007 \"L-Valine[E]=L-Valine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6007 \"L-Valine[E]=L-Valine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6008 \"L-Isoleucine[E]=L-Isoleucine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6008 \"L-Isoleucine[E]=L-Isoleucine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6020 \"C00076[E]+(4)Na+[C]+NH4+[E]=C00076[C]+(4)Na+[E]+NH4+[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6020 \"C00076[E]+(4)Na+[C]+NH4+[E]=C00076[C]+(4)Na+[E]+NH4+[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6026 \"ATP[C]+Bilirubin-glucuronoside[C]+H2O[C]=ADP[C]+Bilirubin-glucuronoside[E]+H+[C]+Orthophosphate[C] (reversible)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6026 \"ATP[C]+Bilirubin-glucuronoside[C]+H2O[C]=ADP[C]+Bilirubin-glucuronoside[E]+H+[C]+Orthophosphate[C] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6028 \"ATP[C]+Bilirubin beta-diglucuronide[C]+H2O[C]=ADP[C]+Bilirubin beta-diglucuronide[E]+H+[C]+Orthophosphate[C] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6028 \"ATP[C]+Bilirubin beta-diglucuronide[C]+H2O[C]=ADP[C]+Bilirubin beta-diglucuronide[E]+H+[C]+Orthophosphate[C] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6070 \"Ascorbate[E]=Ascorbate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6070 \"Ascorbate[E]=Ascorbate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6074 \"Methanol[E]=Methanol[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6074 \"Methanol[E]=Methanol[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6079 \"Arachidonate[E]=Arachidonate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6079 \"Arachidonate[E]=Arachidonate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6084 \"L-Arabinose[E]=L-Arabinose[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6084 \"L-Arabinose[E]=L-Arabinose[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6085 \"Pyridoxine[E]=Pyridoxine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6085 \"Pyridoxine[E]=Pyridoxine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6090 \"Pyridoxamine[E]=Pyridoxamine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6090 \"Pyridoxamine[E]=Pyridoxamine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6094 \"O2.-[C]=O2.-[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6094 \"O2.-[C]=O2.-[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6096 \"Naphthalene[E]=Naphthalene[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6096 \"Naphthalene[E]=Naphthalene[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6098 \"4-Nitrophenol[E]=4-Nitrophenol[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6098 \"4-Nitrophenol[E]=4-Nitrophenol[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6101 \"Sphinganine 1-phosphate[C]=Sphinganine 1-phosphate[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6101 \"Sphinganine 1-phosphate[C]=Sphinganine 1-phosphate[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6105 \"Linoleate[E]=Linoleate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6105 \"Linoleate[E]=Linoleate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6106 \"Phytanate[E]=Phytanate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6106 \"Phytanate[E]=Phytanate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6112 \"Thromboxane A2[E]=Thromboxane A2[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6112 \"Thromboxane A2[E]=Thromboxane A2[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6113 \"alpha-Tocopherol[E]=alpha-Tocopherol[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6113 \"alpha-Tocopherol[E]=alpha-Tocopherol[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6120 \"Vitamin D3[E]=Vitamin D3[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6120 \"Vitamin D3[E]=Vitamin D3[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6121 \"Taurochenodeoxycholate[E]=Taurochenodeoxycholate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6121 \"Taurochenodeoxycholate[E]=Taurochenodeoxycholate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6122 \"3,4-Dihydroxyphenylethyleneglycol[C]=3,4-Dihydroxyphenylethyleneglycol[E] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6122 \"3,4-Dihydroxyphenylethyleneglycol[C]=3,4-Dihydroxyphenylethyleneglycol[E] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6128 \"(9Z,12Z,15Z)-Octadecatrienoic acid[E]=(9Z,12Z,15Z)-Octadecatrienoic acid[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6128 \"(9Z,12Z,15Z)-Octadecatrienoic acid[E]=(9Z,12Z,15Z)-Octadecatrienoic acid[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6130 \"alpha-Tocotrienol[E]=alpha-Tocotrienol[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6130 \"alpha-Tocotrienol[E]=alpha-Tocotrienol[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6132 \"palmitoleate[E]=palmitoleate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6132 \"palmitoleate[E]=palmitoleate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6133 \"13-cis-retinol[E]=13-cis-retinol[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6133 \"13-cis-retinol[E]=13-cis-retinol[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6134 \"(Gal)3 (Glc)1 (GlcNAc)1 (LFuc)2 (Cer)1[E]=(Gal)3 (Glc)1 (GlcNAc)1 (LFuc)2 (Cer)1[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6134 \"(Gal)3 (Glc)1 (GlcNAc)1 (LFuc)2 (Cer)1[E]=(Gal)3 (Glc)1 (GlcNAc)1 (LFuc)2 (Cer)1[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6137 \"(Gal)2 (Glc)1 (GlcNAc)1 (LFuc)1 (Cer)1[E]=(Gal)2 (Glc)1 (GlcNAc)1 (LFuc)1 (Cer)1[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6137 \"(Gal)2 (Glc)1 (GlcNAc)1 (LFuc)1 (Cer)1[E]=(Gal)2 (Glc)1 (GlcNAc)1 (LFuc)1 (Cer)1[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6153 \"L-Leucine[C]=L-Leucine[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6153 \"L-Leucine[C]=L-Leucine[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6157 \"L-Isoleucine[C]=L-Isoleucine[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6157 \"L-Isoleucine[C]=L-Isoleucine[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6164 \"L-Histidine[E]=L-Histidine[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6164 \"L-Histidine[E]=L-Histidine[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6175 \"2-Oxoadipate[C]+2-Oxoglutarate[M]=2-Oxoadipate[M]+2-Oxoglutarate[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6175 \"2-Oxoadipate[C]+2-Oxoglutarate[M]=2-Oxoadipate[M]+2-Oxoglutarate[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6176 \"3-Methyl-2-oxobutanoic acid[C]+H+[C]=3-Methyl-2-oxobutanoic acid[M]+H+[M]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6176 \"3-Methyl-2-oxobutanoic acid[C]+H+[C]=3-Methyl-2-oxobutanoic acid[M]+H+[M]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6179 \"Acetyl-CoA[C]=Acetyl-CoA[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6179 \"Acetyl-CoA[C]=Acetyl-CoA[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6193 \"(S)-Malate[C]+Sulfite[M]=(S)-Malate[M]+Sulfite[C] (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6193 \"(S)-Malate[C]+Sulfite[M]=(S)-Malate[M]+Sulfite[C] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6195 \"3'-Phosphoadenylyl sulfate[C]=3'-Phosphoadenylyl sulfate[G]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6195 \"3'-Phosphoadenylyl sulfate[C]=3'-Phosphoadenylyl sulfate[G]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6197 \"UDPglucuronate[R]+UMP[C]=UDPglucuronate[C]+UMP[R] (reversible)\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6197 \"UDPglucuronate[R]+UMP[C]=UDPglucuronate[C]+UMP[R] (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6198 \"UDPglucose[C]+UMP[G]=UDPglucose[G]+UMP[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6198 \"UDPglucose[C]+UMP[G]=UDPglucose[G]+UMP[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6199 \"UDP-D-xylose[C]+UMP[G]=UDP-D-xylose[G]+UMP[C]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6199 \"UDP-D-xylose[C]+UMP[G]=UDP-D-xylose[G]+UMP[C]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6202 \"2-methylacetoacetyl-CoA thiolase (reversible)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6202 \"2-methylacetoacetyl-CoA thiolase (reversible)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6203 \"2-oxoisovalerate dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6203 \"2-oxoisovalerate dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6204 \"2-oxobutanoate dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6204 \"2-oxobutanoate dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6206 \"acetol monooxygenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6206 \"acetol monooxygenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6208 \"malate dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6208 \"malate dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6209 \"long chain fatty acid elongase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6209 \"long chain fatty acid elongase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6210 \"stearoyl-CoA hydrolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6210 \"stearoyl-CoA hydrolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6214 \"arachidoyl-CoA hydrolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6214 \"arachidoyl-CoA hydrolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6220 \"succinyl-CoA hydrolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6220 \"succinyl-CoA hydrolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6221 \"long-chain-fatty-acid--CoA ligase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6221 \"long-chain-fatty-acid--CoA ligase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6222 \"long-chain-fatty-acid--CoA ligase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6222 \"long-chain-fatty-acid--CoA ligase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6223 \"long-chain-fatty-acid--CoA ligase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6223 \"long-chain-fatty-acid--CoA ligase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6229 \"long-chain-fatty-acid--CoA ligase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6229 \"long-chain-fatty-acid--CoA ligase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6234 \"2-oxo-3-methyl-isovalerate dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6234 \"2-oxo-3-methyl-isovalerate dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6235 \"oxalosuccinate decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6235 \"oxalosuccinate decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6236 \"oxalosuccinate reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6236 \"oxalosuccinate reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6240 \"serotonin oxidase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6240 \"serotonin oxidase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6241 \"serotonin sulfotransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6241 \"serotonin sulfotransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6242 \"5-hydroxyindole acetaldehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6242 \"5-hydroxyindole acetaldehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6244 \"phenylacetaldehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6244 \"phenylacetaldehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6245 \"noradrenaline _ST_i_LT_O_ST_/i_LT_-methyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6245 \"noradrenaline _ST_i_LT_O_ST_/i_LT_-methyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6246 \"adrenaline _ST_i_LT_O_ST_/i_LT_-methyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6246 \"adrenaline _ST_i_LT_O_ST_/i_LT_-methyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6248 \"3,4-dihydroxyphenylglycolaldehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6248 \"3,4-dihydroxyphenylglycolaldehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6249 \"phosphatidylinositol 3,5-bisphosphate 5-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6249 \"phosphatidylinositol 3,5-bisphosphate 5-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6250 \"phosphatidylinositol-4,5-bisphosphate 3-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6250 \"phosphatidylinositol-4,5-bisphosphate 3-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6256 \"inositol-1,3,4,5-tetrakisphosphate 5-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6256 \"inositol-1,3,4,5-tetrakisphosphate 5-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6257 \"phosphatidylinositol-3,4,5-trisphosphate 5-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6257 \"phosphatidylinositol-3,4,5-trisphosphate 5-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6264 \"phosphatidylinositol-4,5-bisphosphate 5-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6264 \"phosphatidylinositol-4,5-bisphosphate 5-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6273 \"inositol 1,3,4-trisphosphate 1-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6273 \"inositol 1,3,4-trisphosphate 1-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6274 \"inositol polyphosphate phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6274 \"inositol polyphosphate phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6275 \"inositol polyphosphate phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6275 \"inositol polyphosphate phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6276 \"inositol polyphosphate phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6276 \"inositol polyphosphate phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6277 \"inositol polyphosphate phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6277 \"inositol polyphosphate phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6278 \"inositol polyphosphate phosphatase\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6278 \"inositol polyphosphate phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6279 \"inositol-1,3,4,5-tetrakisphosphate 3-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6279 \"inositol-1,3,4,5-tetrakisphosphate 3-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6285 \"inositol-1,3,4,5,6-pentakisphosphate 3-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6285 \"inositol-1,3,4,5,6-pentakisphosphate 3-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6286 \"inositol-1,4,5,6-tetrakisphosphate 6-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6286 \"inositol-1,4,5,6-tetrakisphosphate 6-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6291 \"protein-tyrosine-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6291 \"protein-tyrosine-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6293 \"inositol 1,3,4,6-tetrakisphosphate 5-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6293 \"inositol 1,3,4,6-tetrakisphosphate 5-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6294 \"inositol 1,4,5-trisphosphate 3-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6294 \"inositol 1,4,5-trisphosphate 3-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6295 \"inositol-1,3,4,5,6-pentakisphosphate 2-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6295 \"inositol-1,3,4,5,6-pentakisphosphate 2-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6329 \"myo-inositol (1,4,5,6)-tetrakisphosphate 3-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6329 \"myo-inositol (1,4,5,6)-tetrakisphosphate 3-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6331 \"phosphatidylinositol-3,5-bisphosphate 3-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6331 \"phosphatidylinositol-3,5-bisphosphate 3-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6337 \"phosphatidylinositol 4,5-bisphosphate 4-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6337 \"phosphatidylinositol 4,5-bisphosphate 4-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6339 \"inositol-1,3,4-trisphosphate 4-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6339 \"inositol-1,3,4-trisphosphate 4-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6343 \"phosphatidylinositol 4-phosphate 4-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6343 \"phosphatidylinositol 4-phosphate 4-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6354 \"inositol-1,3,4,5,6-pentakisphosphate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6354 \"inositol-1,3,4,5,6-pentakisphosphate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6357 \"inositol hexakisphosphate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6357 \"inositol hexakisphosphate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6360 \"inositol hexakisphosphate 3-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6360 \"inositol hexakisphosphate 3-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6362 \"5-diphosphoinositol-pentakisphosphate 1-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6362 \"5-diphosphoinositol-pentakisphosphate 1-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6364 \"diphosphoinositol pentakisphosphate phosphohydrolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6364 \"diphosphoinositol pentakisphosphate phosphohydrolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6370 \"bis(diphospho)inositol tetrakisphosphate phosphohydrolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6370 \"bis(diphospho)inositol tetrakisphosphate phosphohydrolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6371 \"5-phosphoribosyl 1-pyrophosphate pyrophosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6371 \"5-phosphoribosyl 1-pyrophosphate pyrophosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6373 \"diphosphoinositol tetrakisphosphate phosphohydrolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6373 \"diphosphoinositol tetrakisphosphate phosphohydrolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6393 \"1-diphosphoinositol pentakisphosphate 5-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6393 \"1-diphosphoinositol pentakisphosphate 5-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6394 \"3-diphosphoinositol pentakisphosphate 5-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6394 \"3-diphosphoinositol pentakisphosphate 5-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6403 \"5-diphosphoinositol-pentakisphosphate 3-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6403 \"5-diphosphoinositol-pentakisphosphate 3-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6405 \"2,3-bisphosphoglycerate 3-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6405 \"2,3-bisphosphoglycerate 3-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6430 \"4-acetamidobutanal dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6430 \"4-acetamidobutanal dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6431 \"Fructose-2,6-bisphosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6431 \"Fructose-2,6-bisphosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6435 \"alpha-methylacyl-CoA racemase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6435 \"alpha-methylacyl-CoA racemase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6438 \"hydroxymethylglutaryl-CoA synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6438 \"hydroxymethylglutaryl-CoA synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6440 \"peptidyl-prolyl cis-trans isomerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6440 \"peptidyl-prolyl cis-trans isomerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6441 \"Adenylyl-sulfate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6441 \"Adenylyl-sulfate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6442 \"argininosuccinate synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6442 \"argininosuccinate synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6443 \"Lipid phosphate phosphohydrolase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6443 \"Lipid phosphate phosphohydrolase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6447 \"Gamma-glutamyl phosphate reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6447 \"Gamma-glutamyl phosphate reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6451 \"3-hydroxy-3-methylglutaryl-coenzyme A reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6451 \"3-hydroxy-3-methylglutaryl-coenzyme A reductase\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6453 \"_ST_i_LT_N_ST_/i_LT_-acetylgalactosamine-4-sulfatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6453 \"_ST_i_LT_N_ST_/i_LT_-acetylgalactosamine-4-sulfatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6455 \"Gamma-glutamyl phosphate reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6455 \"Gamma-glutamyl phosphate reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6456 \"dihydrofolate reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6456 \"dihydrofolate reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6458 \"Acyl-coenzyme A thioesterase 12\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6458 \"Acyl-coenzyme A thioesterase 12\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6468 \"Long-chain-fatty-acid--CoA ligase 6\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6468 \"Long-chain-fatty-acid--CoA ligase 6\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6474 \"Processed beta-1,4-galactosyltransferase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6474 \"Processed beta-1,4-galactosyltransferase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6476 \"Rod cGMP-specific 3',5'-cyclic phosphodiesterase subunit alpha\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6476 \"Rod cGMP-specific 3',5'-cyclic phosphodiesterase subunit alpha\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6477 \"Beta-type platelet-derived growth factor receptor\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6477 \"Beta-type platelet-derived growth factor receptor\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6478 \"Glutathione peroxidase 3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6478 \"Glutathione peroxidase 3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6480 \"Pantothenate kinase 3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6480 \"Pantothenate kinase 3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6482 \"glyceraldehyde 3 phosphate dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6482 \"glyceraldehyde 3 phosphate dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6483 \"Hexokinase-3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6483 \"Hexokinase-3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6491 \"GDP-mannose 4,6 dehydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6491 \"GDP-mannose 4,6 dehydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6495 \"Ribosyldihydronicotinamide dehydrogenase [quinone]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6495 \"Ribosyldihydronicotinamide dehydrogenase [quinone]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6500 \"Ribosyldihydronicotinamide dehydrogenase [quinone]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6500 \"Ribosyldihydronicotinamide dehydrogenase [quinone]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6504 \"GMP reductase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6504 \"GMP reductase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6506 \"inosine 5' monophosphate dehydrogenase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6506 \"inosine 5' monophosphate dehydrogenase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6508 \"Succinate-semialdehyde dehydrogenase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6508 \"Succinate-semialdehyde dehydrogenase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6518 \"1-acyl-sn-glycerol-3-phosphate acyltransferase alpha\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6518 \"1-acyl-sn-glycerol-3-phosphate acyltransferase alpha\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6521 \"Lactoylglutathione lyase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6521 \"Lactoylglutathione lyase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6523 \"Platelet-activating factor acetylhydrolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6523 \"Platelet-activating factor acetylhydrolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6525 \"Phosphoglycerate kinase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6525 \"Phosphoglycerate kinase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6535 \"galactosylgalactosylxylosylprotein 3-_AND_beta;-glucuronosyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6535 \"galactosylgalactosylxylosylprotein 3-_AND_beta;-glucuronosyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6537 \"Phosphoacetylglucosamine mutase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6537 \"Phosphoacetylglucosamine mutase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6540 \"5'-nucleotidase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6540 \"5'-nucleotidase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6550 \"Sphingomyelin phosphodiesterase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6550 \"Sphingomyelin phosphodiesterase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6560 \"Arginase-1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6560 \"Arginase-1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6565 \"L-lactate dehydrogenase chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6565 \"L-lactate dehydrogenase chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6566 \"inositol-1,4,5-trisphosphate 5-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6566 \"inositol-1,4,5-trisphosphate 5-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6567 \"Superoxide dismutase [Mn], mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6567 \"Superoxide dismutase [Mn], mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6570 \"Diacylglycerol kinase beta\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6570 \"Diacylglycerol kinase beta\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6577 \"Formimidoyltetrahydrofolate cyclodeaminase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6577 \"Formimidoyltetrahydrofolate cyclodeaminase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6583 \"Biliverdin reductase A\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6583 \"Biliverdin reductase A\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6584 \"Biliverdin reductase A\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6584 \"Biliverdin reductase A\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6585 \"Phosphoglycerate mutase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6585 \"Phosphoglycerate mutase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6586 \"Phosphoglycerate mutase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6586 \"Phosphoglycerate mutase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6599 \"Adenylate cyclase type 1\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6599 \"Adenylate cyclase type 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6600 \"L-tryptophan decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6600 \"L-tryptophan decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6603 \"L-3-phosphoserine phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6603 \"L-3-phosphoserine phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6608 \"argininosuccinate lyase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6608 \"argininosuccinate lyase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6617 \"Serum paraoxonase/arylesterase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6617 \"Serum paraoxonase/arylesterase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6618 \"Serum paraoxonase/arylesterase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6618 \"Serum paraoxonase/arylesterase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6619 \"Serum paraoxonase/arylesterase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6619 \"Serum paraoxonase/arylesterase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6620 \"Serum paraoxonase/arylesterase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6620 \"Serum paraoxonase/arylesterase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6621 \"Serum paraoxonase/arylesterase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6621 \"Serum paraoxonase/arylesterase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6630 \"Acetylcholinesterase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6630 \"Acetylcholinesterase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6634 \"Dihydrolipoyl dehydrogenase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6634 \"Dihydrolipoyl dehydrogenase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6639 \"Saccharopine dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6639 \"Saccharopine dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6640 \"Saccharopine dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6640 \"Saccharopine dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6644 \"Putative adenosylhomocysteinase 3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6644 \"Putative adenosylhomocysteinase 3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6648 \"Formimidoyltetrahydrofolate cyclodeaminase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6648 \"Formimidoyltetrahydrofolate cyclodeaminase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6656 \"Nitric oxide synthase, endothelial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6656 \"Nitric oxide synthase, endothelial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6657 \"Diacylglycerol O-acyltransferase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6657 \"Diacylglycerol O-acyltransferase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6661 \"farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6661 \"farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6670 \"Glutathione reductase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6670 \"Glutathione reductase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6690 \"2,4-dienoyl-CoA reductase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6690 \"2,4-dienoyl-CoA reductase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6691 \"Dihydropyrimidinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6691 \"Dihydropyrimidinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6698 \"Alanine aminotransferase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6698 \"Alanine aminotransferase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6709 \"Processed beta-1,4-galactosyltransferase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6709 \"Processed beta-1,4-galactosyltransferase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6712 \"Galactose-1-phosphate uridylyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6712 \"Galactose-1-phosphate uridylyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6713 \"glutamine synthetase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6713 \"glutamine synthetase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6715 \"Atrial natriuretic peptide receptor B\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6715 \"Atrial natriuretic peptide receptor B\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6722 \"formyltetrahydrofolate synthetase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6722 \"formyltetrahydrofolate synthetase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6731 \"Guanine deaminase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6731 \"Guanine deaminase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6732 \"Beta-1,3-galactosyl-O-glycosyl-glycoprotein beta-1,6-N-acetylglucosaminyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6732 \"Beta-1,3-galactosyl-O-glycosyl-glycoprotein beta-1,6-N-acetylglucosaminyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6734 \"Phosphoserine aminotransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6734 \"Phosphoserine aminotransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6737 \"Serine palmitoyltransferase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6737 \"Serine palmitoyltransferase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6739 \"Fructose-1,6-bisphosphatase isozyme 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6739 \"Fructose-1,6-bisphosphatase isozyme 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6741 \"Sialic acid synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6741 \"Sialic acid synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6742 \"Sialic acid synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6742 \"Sialic acid synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6744 \"acyl-coenzyme A thioester hydrolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6744 \"acyl-coenzyme A thioester hydrolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6745 \"Bile acid-CoA:amino acid N-acyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6745 \"Bile acid-CoA:amino acid N-acyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6746 \"fructose-bisphosphate aldolase B\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6746 \"fructose-bisphosphate aldolase B\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6749 \"Ceramide glucosyltransferase\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6749 \"Ceramide glucosyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6750 \"Delta-aminolevulinic acid dehydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6750 \"Delta-aminolevulinic acid dehydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6753 \"Adenylate kinase isoenzyme 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6753 \"Adenylate kinase isoenzyme 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6758 \"Uridine-cytidine kinase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6758 \"Uridine-cytidine kinase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6759 \"Uridine-cytidine kinase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6759 \"Uridine-cytidine kinase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6760 \"Bile salt-activated lipase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6760 \"Bile salt-activated lipase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6765 \"dopamine _AND_beta;-monooxygenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6765 \"dopamine _AND_beta;-monooxygenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6768 \"UDP-N-acetylglucosamine pyrophosphorylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6768 \"UDP-N-acetylglucosamine pyrophosphorylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6771 \"isopentenyl-diphosphate delta-isomerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6771 \"isopentenyl-diphosphate delta-isomerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6773 \"6-phosphofructokinase type C\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6773 \"6-phosphofructokinase type C\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6775 \"Aldo-keto reductase family 1 member C2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6775 \"Aldo-keto reductase family 1 member C2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6776 \"Aldo-keto reductase family 1 member C2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6776 \"Aldo-keto reductase family 1 member C2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6781 \"Aldo-keto reductase family 1 member C3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6781 \"Aldo-keto reductase family 1 member C3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6782 \"Aldo-keto reductase family 1 member C4\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6782 \"Aldo-keto reductase family 1 member C4\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6783 \"Fructose-2,6-bisphosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6783 \"Fructose-2,6-bisphosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6789 \"phosphatidylinositol-5-phosphate 4-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6789 \"phosphatidylinositol-5-phosphate 4-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6790 \"Glutamate decarboxylase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6790 \"Glutamate decarboxylase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6801 \"Phosphoribosylaminoimidazole carboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6801 \"Phosphoribosylaminoimidazole carboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6806 \"Phosphoribosylaminoimidazole carboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6806 \"Phosphoribosylaminoimidazole carboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6807 \"Inorganic pyrophosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6807 \"Inorganic pyrophosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6809 \"Adenosine kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6809 \"Adenosine kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6814 \"S-adenosylmethionine synthetase isoform type-1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6814 \"S-adenosylmethionine synthetase isoform type-1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6825 \"Ectonucleoside triphosphate diphosphohydrolase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6825 \"Ectonucleoside triphosphate diphosphohydrolase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6850 \"Glycerol-3-phosphate acyltransferase 1, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6850 \"Glycerol-3-phosphate acyltransferase 1, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6863 \"Ornithine aminotransferase, renal form\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6863 \"Ornithine aminotransferase, renal form\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6890 \"Transaldolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6890 \"Transaldolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6899 \"AMP deaminase 3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6899 \"AMP deaminase 3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6903 \"phosphatidylinositol-4-phosphate 3-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6903 \"phosphatidylinositol-4-phosphate 3-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6904 \"Tryptophan 5-hydroxylase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6904 \"Tryptophan 5-hydroxylase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6910 \"catalase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6910 \"catalase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6919 \"1-phosphatidylinositol-4,5-bisphosphate phosphodiesterase beta-3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6919 \"1-phosphatidylinositol-4,5-bisphosphate phosphodiesterase beta-3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6920 \"Glycogen phosphorylase, muscle form\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6920 \"Glycogen phosphorylase, muscle form\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6928 \"Choline kinase alpha\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6928 \"Choline kinase alpha\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6929 \"Carnitine O-palmitoyltransferase 1, liver isoform\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6929 \"Carnitine O-palmitoyltransferase 1, liver isoform\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6938 \"dynein heavy chain, isotype 1B (fragment)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6938 \"dynein heavy chain, isotype 1B (fragment)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6941 \"acetyl-CoA acetyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6941 \"acetyl-CoA acetyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6946 \"Porphobilinogen deaminase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6946 \"Porphobilinogen deaminase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6947 \"UDP-N-acetylglucosamine--dolichyl-phosphate N-acetylglucosaminephosphotransferase\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6947 \"UDP-N-acetylglucosamine--dolichyl-phosphate N-acetylglucosaminephosphotransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6954 \"Serine hydroxymethyltransferase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6954 \"Serine hydroxymethyltransferase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6960 \"Gamma-enolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6960 \"Gamma-enolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6961 \"Triosephosphate isomerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6961 \"Triosephosphate isomerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6968 \"enoy CoA hydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6968 \"enoy CoA hydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6969 \"enoy CoA hydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6969 \"enoy CoA hydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6983 \"Glycogen [starch] synthase, liver\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6983 \"Glycogen [starch] synthase, liver\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_6987 \"Ethanolamine kinase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_6987 \"Ethanolamine kinase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7001 \"Cysteine sulfinic acid decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7001 \"Cysteine sulfinic acid decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7003 \"11-cis retinol dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7003 \"11-cis retinol dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7007 \"Sulfite oxidase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7007 \"Sulfite oxidase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7009 \"Glutaminase liver isoform, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7009 \"Glutaminase liver isoform, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7018 \"Acyl-CoA synthetase short-chain family member 3, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7018 \"Acyl-CoA synthetase short-chain family member 3, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7026 \"Histidine ammonia-lyase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7026 \"Histidine ammonia-lyase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7028 \"Orotidine 5'-phosphate decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7028 \"Orotidine 5'-phosphate decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7030 \"Phenylalanine-4-hydroxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7030 \"Phenylalanine-4-hydroxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7035 \"Biotin carboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7035 \"Biotin carboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7036 \"mevalonate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7036 \"mevalonate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7037 \"Orotidine 5'-phosphate decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7037 \"Orotidine 5'-phosphate decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7039 \"L-serine dehydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7039 \"L-serine dehydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7041 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7041 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7042 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7042 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7043 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7043 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7044 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7044 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7046 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7046 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7047 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7047 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7048 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7048 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7049 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7049 \"Putative nucleoside diphosphate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7052 \"Phospholipase A2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7052 \"Phospholipase A2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7053 \"4-hydroxyphenylpyruvate dioxygenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7053 \"4-hydroxyphenylpyruvate dioxygenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7070 \"succinyl-CoA ligase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7070 \"succinyl-CoA ligase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7093 \"GTP cyclohydrolase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7093 \"GTP cyclohydrolase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7094 \"5 aminolevulinic acid synthase erythroid specific mitochondrial precursor\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7094 \"5 aminolevulinic acid synthase erythroid specific mitochondrial precursor\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7106 \"Maleylacetoacetate isomerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7106 \"Maleylacetoacetate isomerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7115 \"isocitrate dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7115 \"isocitrate dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7129 \"Glycine amidinotransferase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7129 \"Glycine amidinotransferase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7131 \"Histidine decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7131 \"Histidine decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7139 \"lactase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7139 \"lactase\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7140 \"Pyruvate kinase isozymes M1/M2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7140 \"Pyruvate kinase isozymes M1/M2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7147 \"5-formyltetrahydrofolate cyclo-ligase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7147 \"5-formyltetrahydrofolate cyclo-ligase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7148 \"dihydroorotase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7148 \"dihydroorotase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7164 \"Xanthine oxidase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7164 \"Xanthine oxidase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7171 \"Xanthine oxidase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7171 \"Xanthine oxidase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7178 \"Hydroxyacylglutathione hydrolase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7178 \"Hydroxyacylglutathione hydrolase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7182 \"Phosphomannomutase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7182 \"Phosphomannomutase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7190 \"Sulfotransferase 1A1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7190 \"Sulfotransferase 1A1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7246 \"Tyrosine aminotransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7246 \"Tyrosine aminotransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7248 \"IMP cyclohydrolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7248 \"IMP cyclohydrolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7253 \"Malonyl-CoA decarboxylase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7253 \"Malonyl-CoA decarboxylase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7254 \"IMP cyclohydrolase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7254 \"IMP cyclohydrolase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7256 \"diphosphomevalonate decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7256 \"diphosphomevalonate decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7257 \"chondroitin _ST_i_LT_N_ST_/i_LT_-acetylgalactosamine-6-sulfatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7257 \"chondroitin _ST_i_LT_N_ST_/i_LT_-acetylgalactosamine-6-sulfatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7258 \"Adenine phosphoribosyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7258 \"Adenine phosphoribosyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7265 \"Arachidonate 12-lipoxygenase, 12S-type\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7265 \"Arachidonate 12-lipoxygenase, 12S-type\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7271 \"Phosphoribosylformylglycinamidine synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7271 \"Phosphoribosylformylglycinamidine synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7285 \"Peroxisomal sarcosine oxidase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7285 \"Peroxisomal sarcosine oxidase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7289 \"adenylosuccinate synthetase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7289 \"adenylosuccinate synthetase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7292 \"Phosphatidylinositol-5-phosphate 4-kinase type-2 beta\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7292 \"Phosphatidylinositol-5-phosphate 4-kinase type-2 beta\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7296 \"ATP-citrate synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7296 \"ATP-citrate synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7302 \"6-phosphogluconolactonase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7302 \"6-phosphogluconolactonase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7337 \"Galactokinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7337 \"Galactokinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7349 \"Thymidylate synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7349 \"Thymidylate synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7381 \"Leukotriene-B(4) omega-hydroxylase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7381 \"Leukotriene-B(4) omega-hydroxylase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7387 \"Glucose-6-phosphate isomerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7387 \"Glucose-6-phosphate isomerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7442 \"Adenosine deaminase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7442 \"Adenosine deaminase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7459 \"Carbonyl reductase [NADPH] 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7459 \"Carbonyl reductase [NADPH] 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7466 \"lanosterol synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7466 \"lanosterol synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7482 \"Beta-ureidopropionase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7482 \"Beta-ureidopropionase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7489 \"Phosphatidylserine decarboxylase beta chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7489 \"Phosphatidylserine decarboxylase beta chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7490 \"Thiosulfate sulfurtransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7490 \"Thiosulfate sulfurtransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7491 \"3-mercaptopyruvate sulfurtransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7491 \"3-mercaptopyruvate sulfurtransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7492 \"2-amino-3-ketobutyrate coenzyme A ligase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7492 \"2-amino-3-ketobutyrate coenzyme A ligase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7494 \"adenylosuccinate lyase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7494 \"adenylosuccinate lyase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7495 \"adenylosuccinate lyase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7495 \"adenylosuccinate lyase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7497 \"citrate dehydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7497 \"citrate dehydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7501 \"Thymidine phosphorylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7501 \"Thymidine phosphorylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7508 \"Ethanolamine-phosphate cytidylyltransferase\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7508 \"Ethanolamine-phosphate cytidylyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7510 \"acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7510 \"acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7511 \"acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7511 \"acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7512 \"acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7512 \"acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7513 \"acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7513 \"acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7514 \"acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7514 \"acyl-CoA:malonyl-CoA C-acyltransferase (decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7527 \"Steryl-sulfatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7527 \"Steryl-sulfatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7530 \"CTP synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7530 \"CTP synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7531 \"Pyruvate dehydrogenase E1 component subunit alpha, somatic form, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7531 \"Pyruvate dehydrogenase E1 component subunit alpha, somatic form, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7534 \"Diamine acetyltransferase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7534 \"Diamine acetyltransferase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7535 \"Diamine acetyltransferase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7535 \"Diamine acetyltransferase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7536 \"Choline-phosphate cytidylyltransferase A\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7536 \"Choline-phosphate cytidylyltransferase A\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7544 \"Glycerol kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7544 \"Glycerol kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7545 \"Ornithine carbamoyltransferase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7545 \"Ornithine carbamoyltransferase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7576 \"Hypoxanthine-guanine phosphoribosyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7576 \"Hypoxanthine-guanine phosphoribosyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7583 \"Transketolase-like protein 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7583 \"Transketolase-like protein 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7584 \"Transketolase-like protein 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7584 \"Transketolase-like protein 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7585 \"Glucose-6-phosphate 1-dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7585 \"Glucose-6-phosphate 1-dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7590 \"Kynurenine 3-monooxygenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7590 \"Kynurenine 3-monooxygenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7598 \"Methylmalonyl-CoA epimerase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7598 \"Methylmalonyl-CoA epimerase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7600 \"Probable allantoicase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7600 \"Probable allantoicase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7610 \"Thiamin pyrophosphokinase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7610 \"Thiamin pyrophosphokinase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7614 \"Uridine phosphorylase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7614 \"Uridine phosphorylase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7619 \"Prostaglandin E synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7619 \"Prostaglandin E synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7623 \"[chondroitin]-_ST_i_LT_N_ST_/i_LT_-acetylgalactosamine 6-_ST_i_LT_O_ST_/i_LT_-sulfotransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7623 \"[chondroitin]-_ST_i_LT_N_ST_/i_LT_-acetylgalactosamine 6-_ST_i_LT_O_ST_/i_LT_-sulfotransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7628 \"[chondroitin]-_ST_i_LT_N_ST_/i_LT_-acetylgalactosamine 4-_ST_i_LT_O_ST_/i_LT_-sulfotransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7628 \"[chondroitin]-_ST_i_LT_N_ST_/i_LT_-acetylgalactosamine 4-_ST_i_LT_O_ST_/i_LT_-sulfotransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7633 \"phosphoglycerate dehydrogenase like 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7633 \"phosphoglycerate dehydrogenase like 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7635 \"Thiamine-triphosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7635 \"Thiamine-triphosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7637 \"glucosamine-phosphate N-acetyltransferase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7637 \"glucosamine-phosphate N-acetyltransferase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7644 \"N-acetylglucosamine-1-phosphodiester alpha-N-acetylglucosaminidase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7644 \"N-acetylglucosamine-1-phosphodiester alpha-N-acetylglucosaminidase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7651 \"Protoheme IX farnesyltransferase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7651 \"Protoheme IX farnesyltransferase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7701 \"Nicotinamide mononucleotide adenylyltransferase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7701 \"Nicotinamide mononucleotide adenylyltransferase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7702 \"6-phosphogluconate dehydrogenase, decarboxylating\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7702 \"6-phosphogluconate dehydrogenase, decarboxylating\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7703 \"6-phosphogluconate dehydrogenase, decarboxylating\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7703 \"6-phosphogluconate dehydrogenase, decarboxylating\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7705 \"Spermidine synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7705 \"Spermidine synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7721 \"UDP-glucuronic acid decarboxylase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7721 \"UDP-glucuronic acid decarboxylase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7735 \"thymidylate kinase homologue\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7735 \"thymidylate kinase homologue\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7759 \"Kynurenine/alpha-aminoadipate aminotransferase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7759 \"Kynurenine/alpha-aminoadipate aminotransferase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7761 \"Kynurenine/alpha-aminoadipate aminotransferase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7761 \"Kynurenine/alpha-aminoadipate aminotransferase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7792 \"Ribulose-phosphate 3-epimerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7792 \"Ribulose-phosphate 3-epimerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7807 \"Phosphatidylinositol N-acetylglucosaminyltransferase subunit C\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7807 \"Phosphatidylinositol N-acetylglucosaminyltransferase subunit C\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7808 \"Phosphatidylinositol N-acetylglucosaminyltransferase subunit C\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7808 \"Phosphatidylinositol N-acetylglucosaminyltransferase subunit C\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7832 \"pyridoxine-5-prime-phosphate oxidase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7832 \"pyridoxine-5-prime-phosphate oxidase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7844 \"UDP-glucose 4-epimerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7844 \"UDP-glucose 4-epimerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7849 \"Succinate-CoA ligase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7849 \"Succinate-CoA ligase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7853 \"dynamin\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7853 \"dynamin\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7855 \"6-pyruvoyl tetrahydrobiopterin synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7855 \"6-pyruvoyl tetrahydrobiopterin synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7863 \"Arginine decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7863 \"Arginine decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7875 \"Flavin reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7875 \"Flavin reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7892 \"aconitate hydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7892 \"aconitate hydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7894 \"Uroporphyrinogen decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7894 \"Uroporphyrinogen decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7896 \"Dephospho-CoA kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7896 \"Dephospho-CoA kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7899 \"N-acetylmannosamine kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7899 \"N-acetylmannosamine kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7900 \"glycerol-3-phosphate dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7900 \"glycerol-3-phosphate dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7903 \"Aldose 1-epimerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7903 \"Aldose 1-epimerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7905 \"Glucokinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7905 \"Glucokinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7906 \"Enoyl-CoA hydratase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7906 \"Enoyl-CoA hydratase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7907 \"Probable imidazolonepropionase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7907 \"Probable imidazolonepropionase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7909 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7909 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7910 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7910 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7911 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7911 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7912 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7912 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7913 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7913 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7914 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7914 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7915 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7915 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7916 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7916 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7917 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7917 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7918 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7918 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7919 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7919 \"Decaprenyl-diphosphate synthase subunit 1\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7920 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7920 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7921 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7921 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7922 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7922 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7923 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7923 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7924 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7924 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7925 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7925 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7926 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7926 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7927 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7927 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7928 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7928 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7929 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7929 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7930 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7930 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7931 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7931 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7932 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7932 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7933 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7933 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7934 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7934 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7935 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7935 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7936 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7936 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7937 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7937 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7938 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7938 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7939 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7939 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7940 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7940 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7941 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7941 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7942 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7942 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7943 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7943 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7944 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7944 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7945 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7945 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7946 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7946 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7947 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7947 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7948 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7948 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7949 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7949 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7950 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7950 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7951 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7951 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7952 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7952 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7954 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7954 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7955 \"Decaprenyl-diphosphate synthase subunit 1\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7955 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7956 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7956 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7957 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7957 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7958 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7958 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7959 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7959 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7960 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7960 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7961 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7961 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7962 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7962 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7963 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7963 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7964 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7964 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7965 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7965 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7966 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7966 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7967 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7967 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7968 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7968 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7969 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7969 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7970 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7970 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7971 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7971 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7972 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7972 \"Decaprenyl-diphosphate synthase subunit 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7974 \"Uridine-cytidine kinase-like 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7974 \"Uridine-cytidine kinase-like 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7977 \"N-acetylneuraminate lyase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7977 \"N-acetylneuraminate lyase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7980 \"acetaldehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7980 \"acetaldehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7982 \"cytochrome P450 2E1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7982 \"cytochrome P450 2E1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7983 \"catalase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7983 \"catalase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7998 \"dopa decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7998 \"dopa decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_7999 \"noradrenaline _ST_i_LT_N_ST_/i_LT_-methyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_7999 \"noradrenaline _ST_i_LT_N_ST_/i_LT_-methyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8002 \"presqualene diphosphate:farnesyl-diphosphate farnesyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8002 \"presqualene diphosphate:farnesyl-diphosphate farnesyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8003 \"presqualene diphosphate:farnesyl-diphosphate farnesyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8003 \"presqualene diphosphate:farnesyl-diphosphate farnesyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8005 \"Cystathionine gamma-lyase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8005 \"Cystathionine gamma-lyase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8014 \"Quinone oxidoreductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8014 \"Quinone oxidoreductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8019 \"Medium-chain specific acyl-CoA dehydrogenase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8019 \"Medium-chain specific acyl-CoA dehydrogenase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8022 \"4,4-dimethylzymosterol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8022 \"4,4-dimethylzymosterol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8024 \"4_AND_alpha;-hydroxymethyl-4_AND_beta;-methyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8024 \"4_AND_alpha;-hydroxymethyl-4_AND_beta;-methyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8025 \"4_AND_alpha;-formyl-4_AND_beta;-methyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8025 \"4_AND_alpha;-formyl-4_AND_beta;-methyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8028 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8028 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8029 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8029 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8031 \"4_AND_alpha;-methylzymosterol:NADP+ 3-oxidoreductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8031 \"4_AND_alpha;-methylzymosterol:NADP+ 3-oxidoreductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8033 \"4-_AND_alpha;-methylzymosterol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8033 \"4-_AND_alpha;-methylzymosterol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8035 \"4_AND_alpha;-hydroxymethyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8035 \"4_AND_alpha;-hydroxymethyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8037 \"4_AND_alpha;-formyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8037 \"4_AND_alpha;-formyl-5_AND_alpha;-cholesta-8,24-dien-3_AND_beta;-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8038 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8038 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8039 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8039 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8041 \"_AND_Delta;_ST_sup_LT_8_ST_/sup_LT__AND_Delta;_ST_sup_LT_24_ST_/sup_LT_-cholestadien-3_AND_beta;-ol _AND_Delta;_ST_sup_LT_7_ST_/sup_LT_-_AND_Delta;_ST_sup_LT_8_ST_/sup_LT_-isomerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8041 \"_AND_Delta;_ST_sup_LT_8_ST_/sup_LT__AND_Delta;_ST_sup_LT_24_ST_/sup_LT_-cholestadien-3_AND_beta;-ol _AND_Delta;_ST_sup_LT_7_ST_/sup_LT_-_AND_Delta;_ST_sup_LT_8_ST_/sup_LT_-isomerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8046 \"cholesterol:NADP+ _AND_Delta;_ST_sup_LT_7_ST_/sup_LT_ -oxidoreductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8046 \"cholesterol:NADP+ _AND_Delta;_ST_sup_LT_7_ST_/sup_LT_ -oxidoreductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8047 \"GDP-L-fucose synthetase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8047 \"GDP-L-fucose synthetase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8063 \"4_AND_alpha;-methyl-cholesta-8-enol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8063 \"4_AND_alpha;-methyl-cholesta-8-enol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8066 \"4_AND_alpha;-hydroxymethyl-5_AND_alpha;-cholesta-8-en-3_AND_beta;-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8066 \"4_AND_alpha;-hydroxymethyl-5_AND_alpha;-cholesta-8-en-3_AND_beta;-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8067 \"4_AND_alpha;-formyl-5_AND_alpha;-cholesta-8-en-3_AND_beta;-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8067 \"4_AND_alpha;-formyl-5_AND_alpha;-cholesta-8-en-3_AND_beta;-ol,NAD(P)H:oxygen oxidoreductase (hydroxylating)\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8069 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8069 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8070 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8070 \"3_AND_beta;-hydroxysteroid dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8071 \"5_AND_alpha;-cholesta-8-en-3-one reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8071 \"5_AND_alpha;-cholesta-8-en-3-one reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8074 \"5_AND_alpha;-cholest-7,24-dien-3_AND_beta;-ol,NAD(P)H:oxygen 5-oxidoreductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8074 \"5_AND_alpha;-cholest-7,24-dien-3_AND_beta;-ol,NAD(P)H:oxygen 5-oxidoreductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8075 \"7-dehydrodesmosterol-reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8075 \"7-dehydrodesmosterol-reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8077 \"oxidized glutaredoxin\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8077 \"oxidized glutaredoxin\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8088 \"Cholesterol 24-hydroxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8088 \"Cholesterol 24-hydroxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8092 \"Glycogen phosphorylase, muscle form\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8092 \"Glycogen phosphorylase, muscle form\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8108 \"Aldose reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8108 \"Aldose reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8112 \"Flavin reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8112 \"Flavin reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8114 \"Xylulose kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8114 \"Xylulose kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8116 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8116 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8117 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8117 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8118 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8118 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8119 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8119 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8120 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8120 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8121 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8121 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8122 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8122 \"Gamma-glutamyltransferase 5 light chain\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8123 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8123 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8124 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8124 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8125 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8125 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8126 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8126 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8127 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8127 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8128 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8128 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8129 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8129 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8130 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8130 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8131 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8131 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8132 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8132 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8133 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8133 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8134 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8134 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8135 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8135 \"Gamma-glutamyltransferase 5 light chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8160 \"Purine nucleoside phosphorylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8160 \"Purine nucleoside phosphorylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8161 \"Adenosine deaminase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8161 \"Adenosine deaminase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8173 \"Glucose-6-phosphate isomerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8173 \"Glucose-6-phosphate isomerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8177 \"7,8-dihydro-8-oxoguanine triphosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8177 \"7,8-dihydro-8-oxoguanine triphosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8201 \"Glycerol-3-phosphate dehydrogenase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8201 \"Glycerol-3-phosphate dehydrogenase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8205 \"Sepiapterin reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8205 \"Sepiapterin reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8206 \"Sepiapterin reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8206 \"Sepiapterin reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8208 \"Cystathionine gamma-lyase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8208 \"Cystathionine gamma-lyase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8285 \"Indoleamine 2,3-dioxygenase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8285 \"Indoleamine 2,3-dioxygenase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8315 \"L-serine dehydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8315 \"L-serine dehydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8327 \"Sorbitol dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8327 \"Sorbitol dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8328 \"Sorbitol dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8328 \"Sorbitol dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8329 \"Tyrosine aminotransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8329 \"Tyrosine aminotransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8333 \"Beta-galactoside alpha-2,6-sialyltransferase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8333 \"Beta-galactoside alpha-2,6-sialyltransferase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8340 \"phosphomevalonate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8340 \"phosphomevalonate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8347 \"glucosylceramidase precursor\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8347 \"glucosylceramidase precursor\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8354 \"Inositol polyphosphate multikinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8354 \"Inositol polyphosphate multikinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8357 \"2-amino-3-carboxymuconate-6-semialdehyde decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8357 \"2-amino-3-carboxymuconate-6-semialdehyde decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8361 \"Putative hexokinase HKDC1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8361 \"Putative hexokinase HKDC1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8376 \"Pyridoxal kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8376 \"Pyridoxal kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8400 \"Hypoxanthine-guanine phosphoribosyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8400 \"Hypoxanthine-guanine phosphoribosyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8401 \"Protoporphyrinogen oxidase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8401 \"Protoporphyrinogen oxidase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8402 \"Pterin-4-alpha-carbinolamine dehydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8402 \"Pterin-4-alpha-carbinolamine dehydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8475 \"4-trimethylaminobutyraldehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8475 \"4-trimethylaminobutyraldehyde dehydrogenase\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8494 \"SIMILAR TO SEPIAPTERIN REDUCTASE\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8494 \"SIMILAR TO SEPIAPTERIN REDUCTASE\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8499 \"4-trimethylaminobutyraldehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8499 \"4-trimethylaminobutyraldehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8525 \"Gamma-glutamylcyclotransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8525 \"Gamma-glutamylcyclotransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8530 \"NADPH oxidase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8530 \"NADPH oxidase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8531 \"NADPH oxidase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8531 \"NADPH oxidase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8534 \"glucuronyl-galactosyl-proteoglycan 4-alpha-N-acetylglucosaminyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8534 \"glucuronyl-galactosyl-proteoglycan 4-alpha-N-acetylglucosaminyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8535 \"1,25-dihydroxyvitamin D(3) 24-hydroxylase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8535 \"1,25-dihydroxyvitamin D(3) 24-hydroxylase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8541 \"Methylthioribose-1-phosphate isomerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8541 \"Methylthioribose-1-phosphate isomerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8544 \"Large subunit GTPase 1 homolog\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8544 \"Large subunit GTPase 1 homolog\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8551 \"Sphingosine kinase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8551 \"Sphingosine kinase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8554 \"Fatty acyl-CoA reductase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8554 \"Fatty acyl-CoA reductase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8560 \"ATP-dependent RNA helicase DHX8\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8560 \"ATP-dependent RNA helicase DHX8\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8568 \"Cytochrome b reductase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8568 \"Cytochrome b reductase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8569 \"Vesicle-fusing ATPase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8569 \"Vesicle-fusing ATPase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8570 \"Phosphoribosylglycinamide formyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8570 \"Phosphoribosylglycinamide formyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8583 \"Bile acyl-CoA synthetase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8583 \"Bile acyl-CoA synthetase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8584 \"Bile acyl-CoA synthetase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8584 \"Bile acyl-CoA synthetase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8589 \"Lysophosphatidylcholine acyltransferase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8589 \"Lysophosphatidylcholine acyltransferase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8623 \"Long-chain-fatty-acid--CoA ligase ACSBG1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8623 \"Long-chain-fatty-acid--CoA ligase ACSBG1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8625 \"Long-chain-fatty-acid--CoA ligase ACSBG1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8625 \"Long-chain-fatty-acid--CoA ligase ACSBG1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8663 \"Nicotinamide phosphoribosyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8663 \"Nicotinamide phosphoribosyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8669 \"Prostaglandin reductase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8669 \"Prostaglandin reductase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8670 \"Prostaglandin reductase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8670 \"Prostaglandin reductase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8684 \"N-acetylglucosaminyl-phosphatidylinositol de-N-acetylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8684 \"N-acetylglucosaminyl-phosphatidylinositol de-N-acetylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8698 \"Aldehyde dehydrogenase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8698 \"Aldehyde dehydrogenase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8703 \"Aldehyde dehydrogenase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8703 \"Aldehyde dehydrogenase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8708 \"Aldehyde dehydrogenase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8708 \"Aldehyde dehydrogenase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8733 \"Non-specific lipid-transfer protein\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8733 \"Non-specific lipid-transfer protein\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8739 \"Trans-2-enoyl-CoA reductase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8739 \"Trans-2-enoyl-CoA reductase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8748 \"Dehydrodolichyl diphosphate synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8748 \"Dehydrodolichyl diphosphate synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8751 \"3-ketodihydrosphingosine reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8751 \"3-ketodihydrosphingosine reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8769 \"Phosphopantothenate--cysteine ligase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8769 \"Phosphopantothenate--cysteine ligase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8821 \"Kynurenine--oxoglutarate transaminase 3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8821 \"Kynurenine--oxoglutarate transaminase 3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8827 \"Phosphopantothenoylcysteine decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8827 \"Phosphopantothenoylcysteine decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8858 \"Diamine acetyltransferase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8858 \"Diamine acetyltransferase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8869 \"Cytosolic 5'-nucleotidase III-like protein\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8869 \"Cytosolic 5'-nucleotidase III-like protein\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8871 \"Transient receptor potential cation channel subfamily M member 2\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8871 \"Transient receptor potential cation channel subfamily M member 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8878 \"Pyridoxal phosphate phosphatase PHOSPHO2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8878 \"Pyridoxal phosphate phosphatase PHOSPHO2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8889 \"Guanylate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8889 \"Guanylate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8893 \"Nicotinate phosphoribosyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8893 \"Nicotinate phosphoribosyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8894 \"Probable gluconokinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8894 \"Probable gluconokinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8910 \"Exostosin-2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8910 \"Exostosin-2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8911 \"Exostosin-2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8911 \"Exostosin-2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8935 \"Acid phosphatase-like protein 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8935 \"Acid phosphatase-like protein 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8947 \"fumarate hydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8947 \"fumarate hydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8969 \"Phosphatidylcholine:ceramide cholinephosphotransferase 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8969 \"Phosphatidylcholine:ceramide cholinephosphotransferase 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8983 \"Decaprenyl-diphosphate synthase subunit 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8983 \"Decaprenyl-diphosphate synthase subunit 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8984 \"Decaprenyl-diphosphate synthase subunit 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8984 \"Decaprenyl-diphosphate synthase subunit 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8985 \"Decaprenyl-diphosphate synthase subunit 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8985 \"Decaprenyl-diphosphate synthase subunit 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8986 \"Decaprenyl-diphosphate synthase subunit 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8986 \"Decaprenyl-diphosphate synthase subunit 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8987 \"Decaprenyl-diphosphate synthase subunit 2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8987 \"Decaprenyl-diphosphate synthase subunit 2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_8995 \"Lambda-crystallin homolog\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_8995 \"Lambda-crystallin homolog\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9024 \"Nucleoredoxin\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9024 \"Nucleoredoxin\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9026 \"Serine racemase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9026 \"Serine racemase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9049 \"Ferritin heavy chain\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9049 \"Ferritin heavy chain\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9050 \"Glycerate kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9050 \"Glycerate kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9063 \"Carnitine O-palmitoyltransferase 1, brain isoform\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9063 \"Carnitine O-palmitoyltransferase 1, brain isoform\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9071 \"N-acylneuraminate-9-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9071 \"N-acylneuraminate-9-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9072 \"Manganese-dependent ADP-ribose/CDP-alcohol diphosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9072 \"Manganese-dependent ADP-ribose/CDP-alcohol diphosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9141 \"dCTP pyrophosphatase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9141 \"dCTP pyrophosphatase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9143 \"7-alpha-hydroxycholest-4-en-3-one 12-alpha-hydroxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9143 \"7-alpha-hydroxycholest-4-en-3-one 12-alpha-hydroxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9180 \"aspartate carbamoyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9180 \"aspartate carbamoyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9185 \"glutamine-dependent carbamoyl-phosphate synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9185 \"glutamine-dependent carbamoyl-phosphate synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9193 \"1,2-dihydroxy-3-keto-5-methylthiopentene dioxygenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9193 \"1,2-dihydroxy-3-keto-5-methylthiopentene dioxygenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9197 \"Ribokinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9197 \"Ribokinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9221 \"Amine oxidase [flavin-containing] B\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9221 \"Amine oxidase [flavin-containing] B\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9223 \"Amine oxidase [flavin-containing] B\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9223 \"Amine oxidase [flavin-containing] B\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9226 \"Amine oxidase [flavin-containing] B\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9226 \"Amine oxidase [flavin-containing] B\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9229 \"Amine oxidase [flavin-containing] B\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9229 \"Amine oxidase [flavin-containing] B\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9240 \"3-hydroxyanthranilate 3,4-dioxygenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9240 \"3-hydroxyanthranilate 3,4-dioxygenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9243 \"Hydroxysteroid 17-beta dehydrogenase 6\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9243 \"Hydroxysteroid 17-beta dehydrogenase 6\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9272 \"3-hydroxyacyl-CoA dehydrogenase type-2\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9272 \"3-hydroxyacyl-CoA dehydrogenase type-2\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9278 \"Aldose reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9278 \"Aldose reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9279 \"Amine oxidase [flavin-containing] B\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9279 \"Amine oxidase [flavin-containing] B\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9284 \"Catechol O-methyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9284 \"Catechol O-methyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9285 \"Catechol O-methyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9285 \"Catechol O-methyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9286 \"Catechol O-methyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9286 \"Catechol O-methyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9287 \"Catechol O-methyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9287 \"Catechol O-methyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9289 \"Sialic acid synthase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9289 \"Sialic acid synthase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9307 \"Dihydroorotate dehydrogenase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9307 \"Dihydroorotate dehydrogenase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9310 \"Glycogen [starch] synthase, muscle\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9310 \"Glycogen [starch] synthase, muscle\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9311 \"UTP--glucose-1-phosphate uridylyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9311 \"UTP--glucose-1-phosphate uridylyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9336 \"Methylmalonate-semialdehyde dehydrogenase [acylating], mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9336 \"Methylmalonate-semialdehyde dehydrogenase [acylating], mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9337 \"Methylmalonate-semialdehyde dehydrogenase [acylating], mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9337 \"Methylmalonate-semialdehyde dehydrogenase [acylating], mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9338 \"Phosphoribosylglycinamide formyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9338 \"Phosphoribosylglycinamide formyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9340 \"3-oxo-5-beta-steroid 4-dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9340 \"3-oxo-5-beta-steroid 4-dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9348 \"Long-chain-fatty-acid--CoA ligase 3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9348 \"Long-chain-fatty-acid--CoA ligase 3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9361 \"Aromatic-L-amino-acid decarboxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9361 \"Aromatic-L-amino-acid decarboxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9368 \"Bile acid-CoA:amino acid N-acyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9368 \"Bile acid-CoA:amino acid N-acyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9369 \"Bile acid-CoA:amino acid N-acyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9369 \"Bile acid-CoA:amino acid N-acyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9370 \"Bile acid-CoA:amino acid N-acyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9370 \"Bile acid-CoA:amino acid N-acyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9371 \"Ectonucleoside triphosphate diphosphohydrolase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9371 \"Ectonucleoside triphosphate diphosphohydrolase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9373 \"Deoxyguanosine kinase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9373 \"Deoxyguanosine kinase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9380 \"Aldose 1-epimerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9380 \"Aldose 1-epimerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9381 \"Mannose-1-phosphate guanyltransferase alpha\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9381 \"Mannose-1-phosphate guanyltransferase alpha\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9385 \"Sterol-4-alpha-carboxylate 3-dehydrogenase, decarboxylating\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9385 \"Sterol-4-alpha-carboxylate 3-dehydrogenase, decarboxylating\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9388 \"Long chain 3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9388 \"Long chain 3-hydroxyacyl-CoA dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9411 \"Alcohol dehydrogenase [NADP+]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9411 \"Alcohol dehydrogenase [NADP+]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9420 \"Enolase-phosphatase E1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9420 \"Enolase-phosphatase E1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9430 \"Ribose-5-phosphate isomerase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9430 \"Ribose-5-phosphate isomerase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9435 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9435 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9436 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9436 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9437 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9437 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9439 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9439 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9441 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9441 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9443 \"Retinal dehydrogenase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9443 \"Retinal dehydrogenase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9454 \"L-xylulose reductase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9454 \"L-xylulose reductase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9456 \"Glutaredoxin-1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9456 \"Glutaredoxin-1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9481 \"Inositol polyphosphate multikinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9481 \"Inositol polyphosphate multikinase\">'\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9488 \"4-hydroxyphenylpyruvate dioxygenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9488 \"4-hydroxyphenylpyruvate dioxygenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9491 \"Cytochrome P450 3A4\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9491 \"Cytochrome P450 3A4\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9499 \"phosphatidylinositol-3,4-bisphosphate 4-phosphatase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9499 \"phosphatidylinositol-3,4-bisphosphate 4-phosphatase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9533 \"Leukotriene-B(4) omega-hydroxylase 1\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9533 \"Leukotriene-B(4) omega-hydroxylase 1\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9534 \"Aldo-keto reductase family 1 member C3\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9534 \"Aldo-keto reductase family 1 member C3\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9546 \"Histamine N-methyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9546 \"Histamine N-methyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9562 \"lactone hydrolysis\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9562 \"lactone hydrolysis\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9577 \"LTC4 gamma-glutamyl transferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9577 \"LTC4 gamma-glutamyl transferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9590 \"LTE4 dipeptidase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9590 \"LTE4 dipeptidase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9592 \"xanthine dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9592 \"xanthine dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9593 \"cholesterol 22-_AND_beta;-hydroxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9593 \"cholesterol 22-_AND_beta;-hydroxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9594 \"22-_AND_beta;-hydroxycholesterol 20-_AND_alpha;-hydroxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9594 \"22-_AND_beta;-hydroxycholesterol 20-_AND_alpha;-hydroxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9595 \"20-_AND_alpha;-22-_AND_beta;-dihydroxycholesterol side-chain lyase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9595 \"20-_AND_alpha;-22-_AND_beta;-dihydroxycholesterol side-chain lyase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9616 \"phosphatidylinositol-3-phosphate 5-kinase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9616 \"phosphatidylinositol-3-phosphate 5-kinase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9620 \"progesterone 17-hydroxylase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9620 \"progesterone 17-hydroxylase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9627 \"Carbamoyl-phosphate synthase [ammonia], mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9627 \"Carbamoyl-phosphate synthase [ammonia], mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9631 \"Phosphoribosylglycinamide formyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9631 \"Phosphoribosylglycinamide formyltransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9632 \"Mannose-1-phosphate guanyltransferase alpha\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9632 \"Mannose-1-phosphate guanyltransferase alpha\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9663 \"Putative L-aspartate dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9663 \"Putative L-aspartate dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9664 \"Putative L-aspartate dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9664 \"Putative L-aspartate dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9666 \"Probable methylthioribulose-1-phosphate dehydratase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9666 \"Probable methylthioribulose-1-phosphate dehydratase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9672 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9672 \"Alpha-aminoadipic semialdehyde dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9681 \"Probable arylformamidase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9681 \"Probable arylformamidase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9722 \"Carbonic anhydrase 13\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9722 \"Carbonic anhydrase 13\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9727 \"UDP-N-acetylglucosamine transferase subunit ALG13 homolog\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9727 \"UDP-N-acetylglucosamine transferase subunit ALG13 homolog\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9803 \"1,4-alpha-glucan-branching enzyme\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9803 \"1,4-alpha-glucan-branching enzyme\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9804 \"amidophosphoribosyltransferase precursor\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9804 \"amidophosphoribosyltransferase precursor\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9809 \"Coproporphyrinogen-III oxidase, mitochondrial\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9809 \"Coproporphyrinogen-III oxidase, mitochondrial\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9821 \"10-formyltetrahydrofolate dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9821 \"10-formyltetrahydrofolate dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9857 \"UDP-glucose 6-dehydrogenase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9857 \"UDP-glucose 6-dehydrogenase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9865 \"Estrogen sulfotransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9865 \"Estrogen sulfotransferase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9909 \"15-hydroxyprostaglandin dehydrogenase [NAD+]\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9909 \"15-hydroxyprostaglandin dehydrogenase [NAD+]\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9911 \"Deoxycytidylate deaminase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9911 \"Deoxycytidylate deaminase\">'\n",
      "Missing lower flux bound set to '-1000.0' for reaction: '<Reaction R_9925 \"catechol _ST_i_LT_O_ST_/i_LT_-methyltransferase\">'\n",
      "Missing upper flux bound set to '1000.0' for reaction: '<Reaction R_9925 \"catechol _ST_i_LT_O_ST_/i_LT_-methyltransferase\">'\n",
      "No objective coefficients in model. Unclear what should be optimized\n",
      "Missing flux bounds on reactions set to default bounds.As best practise and to avoid confusion flux bounds should be set explicitly on all reactions.\n"
     ]
    }
   ],
   "source": [
    "init_2012 = cobra.io.read_sbml_model('./2012INIT_liver/iLiverCancer1788.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "f9e89ccf",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>iLiverCancer1788</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>7fac77670070</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>4601</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>2794</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>cytosol, endoplasmic reticulum, mitochondria, extracellular, nucleus, peroxisomes, Golgi apparatus, lysosomes</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model iLiverCancer1788 at 0x7fac77670070>"
      ]
     },
     "execution_count": 62,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "init_2012"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e7c6c26a",
   "metadata": {},
   "source": [
    "## iHepatocytes model : Modèle de cellule du foie non cancéreuse, utilisé dans l'étude des perturbations métaboliques causées par NAFLD\n",
    "\n",
    "[Article](./iHepatocytes2322/iHepatocytes.pdf)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "bf884d47",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[60], line 3\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[38;5;66;03m# Model used to study NAFLD, not cancer\u001b[39;00m\n\u001b[0;32m----> 3\u001b[0m iHep, errors \u001b[38;5;241m=\u001b[39m \u001b[43mvalidate_sbml_model\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43m./iHepatocytes2322/iHepatocytes2322_Cobra.xml\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/io/sbml.py:2017\u001b[0m, in \u001b[0;36mvalidate_sbml_model\u001b[0;34m(filename, check_model, internal_consistency, check_units_consistency, check_modeling_practice, **kwargs)\u001b[0m\n\u001b[1;32m   2013\u001b[0m LOGGER\u001b[38;5;241m.\u001b[39mpropagate \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mFalse\u001b[39;00m\n\u001b[1;32m   2015\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m   2016\u001b[0m     \u001b[38;5;66;03m# read model and allow additional parser arguments\u001b[39;00m\n\u001b[0;32m-> 2017\u001b[0m     model \u001b[38;5;241m=\u001b[39m \u001b[43m_sbml_to_model\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdoc\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   2018\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m CobraSBMLError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m   2019\u001b[0m     errors[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mCOBRA_ERROR\u001b[39m\u001b[38;5;124m\"\u001b[39m]\u001b[38;5;241m.\u001b[39mappend(\u001b[38;5;28mstr\u001b[39m(e))\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/io/sbml.py:956\u001b[0m, in \u001b[0;36m_sbml_to_model\u001b[0;34m(doc, number, f_replace, set_missing_bounds, **kwargs)\u001b[0m\n\u001b[1;32m    954\u001b[0m             \u001b[38;5;28;01mif\u001b[39;00m f_replace \u001b[38;5;129;01mand\u001b[39;00m F_GENE \u001b[38;5;129;01min\u001b[39;00m f_replace:\n\u001b[1;32m    955\u001b[0m                 gpr \u001b[38;5;241m=\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m \u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;241m.\u001b[39mjoin(f_replace[F_GENE](t) \u001b[38;5;28;01mfor\u001b[39;00m t \u001b[38;5;129;01min\u001b[39;00m gpr\u001b[38;5;241m.\u001b[39msplit(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m \u001b[39m\u001b[38;5;124m\"\u001b[39m))\n\u001b[0;32m--> 956\u001b[0m         cobra_reaction\u001b[38;5;241m.\u001b[39mgpr \u001b[38;5;241m=\u001b[39m \u001b[43mGPR\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mfrom_string\u001b[49m\u001b[43m(\u001b[49m\u001b[43mgpr\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    958\u001b[0m cobra_model\u001b[38;5;241m.\u001b[39madd_reactions(reactions)\n\u001b[1;32m    960\u001b[0m \u001b[38;5;66;03m# Objective\u001b[39;00m\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/core/gene.py:378\u001b[0m, in \u001b[0;36mGPR.from_string\u001b[0;34m(cls, string_gpr)\u001b[0m\n\u001b[1;32m    373\u001b[0m         warn(\n\u001b[1;32m    374\u001b[0m             \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mMalformed gene_reaction_rule \u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;132;01m{}\u001b[39;00m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;241m.\u001b[39mformat(escaped_str),\n\u001b[1;32m    375\u001b[0m             \u001b[38;5;167;01mSyntaxWarning\u001b[39;00m,\n\u001b[1;32m    376\u001b[0m         )\n\u001b[1;32m    377\u001b[0m         \u001b[38;5;28;01mreturn\u001b[39;00m gpr\n\u001b[0;32m--> 378\u001b[0m gpr \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mcls\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43mtree\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    379\u001b[0m gpr\u001b[38;5;241m.\u001b[39mupdate_genes()\n\u001b[1;32m    380\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m gpr\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/cobra/core/gene.py:306\u001b[0m, in \u001b[0;36mGPR.__init__\u001b[0;34m(self, gpr_from, **kwargs)\u001b[0m\n\u001b[1;32m    304\u001b[0m     cleaner\u001b[38;5;241m.\u001b[39mvisit(gpr_from)\n\u001b[1;32m    305\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_genes \u001b[38;5;241m=\u001b[39m deepcopy(cleaner\u001b[38;5;241m.\u001b[39mgene_set)\n\u001b[0;32m--> 306\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mbody \u001b[38;5;241m=\u001b[39m \u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43mgpr_from\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    307\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39meval()\n\u001b[1;32m    308\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    170\u001b[0m                 y \u001b[38;5;241m=\u001b[39m x\n\u001b[1;32m    171\u001b[0m             \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[0;32m--> 172\u001b[0m                 y \u001b[38;5;241m=\u001b[39m \u001b[43m_reconstruct\u001b[49m\u001b[43m(\u001b[49m\u001b[43mx\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mrv\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    174\u001b[0m \u001b[38;5;66;03m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[1;32m    175\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m y \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m x:\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[0;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[1;32m    268\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m state \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[1;32m    269\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m deep:\n\u001b[0;32m--> 270\u001b[0m         state \u001b[38;5;241m=\u001b[39m \u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43mstate\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    271\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mhasattr\u001b[39m(y, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124m__setstate__\u001b[39m\u001b[38;5;124m'\u001b[39m):\n\u001b[1;32m    272\u001b[0m         y\u001b[38;5;241m.\u001b[39m__setstate__(state)\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    144\u001b[0m copier \u001b[38;5;241m=\u001b[39m _deepcopy_dispatch\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;28mcls\u001b[39m)\n\u001b[1;32m    145\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m copier \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[0;32m--> 146\u001b[0m     y \u001b[38;5;241m=\u001b[39m \u001b[43mcopier\u001b[49m\u001b[43m(\u001b[49m\u001b[43mx\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    147\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    148\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28missubclass\u001b[39m(\u001b[38;5;28mcls\u001b[39m, \u001b[38;5;28mtype\u001b[39m):\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    228\u001b[0m memo[\u001b[38;5;28mid\u001b[39m(x)] \u001b[38;5;241m=\u001b[39m y\n\u001b[1;32m    229\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m key, value \u001b[38;5;129;01min\u001b[39;00m x\u001b[38;5;241m.\u001b[39mitems():\n\u001b[0;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[38;5;241m=\u001b[39m \u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43mvalue\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    231\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m y\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    170\u001b[0m                 y \u001b[38;5;241m=\u001b[39m x\n\u001b[1;32m    171\u001b[0m             \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[0;32m--> 172\u001b[0m                 y \u001b[38;5;241m=\u001b[39m \u001b[43m_reconstruct\u001b[49m\u001b[43m(\u001b[49m\u001b[43mx\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mrv\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    174\u001b[0m \u001b[38;5;66;03m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[1;32m    175\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m y \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m x:\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[0;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[1;32m    268\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m state \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[1;32m    269\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m deep:\n\u001b[0;32m--> 270\u001b[0m         state \u001b[38;5;241m=\u001b[39m \u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43mstate\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    271\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mhasattr\u001b[39m(y, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124m__setstate__\u001b[39m\u001b[38;5;124m'\u001b[39m):\n\u001b[1;32m    272\u001b[0m         y\u001b[38;5;241m.\u001b[39m__setstate__(state)\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    144\u001b[0m copier \u001b[38;5;241m=\u001b[39m _deepcopy_dispatch\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;28mcls\u001b[39m)\n\u001b[1;32m    145\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m copier \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[0;32m--> 146\u001b[0m     y \u001b[38;5;241m=\u001b[39m \u001b[43mcopier\u001b[49m\u001b[43m(\u001b[49m\u001b[43mx\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    147\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    148\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28missubclass\u001b[39m(\u001b[38;5;28mcls\u001b[39m, \u001b[38;5;28mtype\u001b[39m):\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    228\u001b[0m memo[\u001b[38;5;28mid\u001b[39m(x)] \u001b[38;5;241m=\u001b[39m y\n\u001b[1;32m    229\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m key, value \u001b[38;5;129;01min\u001b[39;00m x\u001b[38;5;241m.\u001b[39mitems():\n\u001b[0;32m--> 230\u001b[0m     y[deepcopy(key, memo)] \u001b[38;5;241m=\u001b[39m \u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43mvalue\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    231\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m y\n",
      "    \u001b[0;31m[... skipping similar frames: deepcopy at line 146 (7 times), _deepcopy_dict at line 230 (6 times), _reconstruct at line 270 (6 times), deepcopy at line 172 (6 times)]\u001b[0m\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:205\u001b[0m, in \u001b[0;36m_deepcopy_list\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    203\u001b[0m append \u001b[38;5;241m=\u001b[39m y\u001b[38;5;241m.\u001b[39mappend\n\u001b[1;32m    204\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m a \u001b[38;5;129;01min\u001b[39;00m x:\n\u001b[0;32m--> 205\u001b[0m     append(\u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43ma\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m)\n\u001b[1;32m    206\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m y\n",
      "    \u001b[0;31m[... skipping similar frames: deepcopy at line 146 (2 times), _deepcopy_dict at line 230 (1 times), _reconstruct at line 270 (1 times), deepcopy at line 172 (1 times)]\u001b[0m\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:205\u001b[0m, in \u001b[0;36m_deepcopy_list\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    203\u001b[0m append \u001b[38;5;241m=\u001b[39m y\u001b[38;5;241m.\u001b[39mappend\n\u001b[1;32m    204\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m a \u001b[38;5;129;01min\u001b[39;00m x:\n\u001b[0;32m--> 205\u001b[0m     append(\u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43ma\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m)\n\u001b[1;32m    206\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m y\n",
      "    \u001b[0;31m[... skipping similar frames: deepcopy at line 146 (20 times), _deepcopy_dict at line 230 (16 times), _reconstruct at line 270 (16 times), deepcopy at line 172 (16 times), _deepcopy_list at line 205 (3 times)]\u001b[0m\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:205\u001b[0m, in \u001b[0;36m_deepcopy_list\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    203\u001b[0m append \u001b[38;5;241m=\u001b[39m y\u001b[38;5;241m.\u001b[39mappend\n\u001b[1;32m    204\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m a \u001b[38;5;129;01min\u001b[39;00m x:\n\u001b[0;32m--> 205\u001b[0m     append(\u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43ma\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m)\n\u001b[1;32m    206\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m y\n",
      "    \u001b[0;31m[... skipping similar frames: _deepcopy_dict at line 230 (1 times), _reconstruct at line 270 (1 times), deepcopy at line 172 (1 times), deepcopy at line 146 (1 times)]\u001b[0m\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:172\u001b[0m, in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    170\u001b[0m                 y \u001b[38;5;241m=\u001b[39m x\n\u001b[1;32m    171\u001b[0m             \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[0;32m--> 172\u001b[0m                 y \u001b[38;5;241m=\u001b[39m \u001b[43m_reconstruct\u001b[49m\u001b[43m(\u001b[49m\u001b[43mx\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mrv\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    174\u001b[0m \u001b[38;5;66;03m# If is its own copy, don't memoize.\u001b[39;00m\n\u001b[1;32m    175\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m y \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m x:\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:270\u001b[0m, in \u001b[0;36m_reconstruct\u001b[0;34m(x, memo, func, args, state, listiter, dictiter, deepcopy)\u001b[0m\n\u001b[1;32m    268\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m state \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[1;32m    269\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m deep:\n\u001b[0;32m--> 270\u001b[0m         state \u001b[38;5;241m=\u001b[39m \u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43mstate\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    271\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mhasattr\u001b[39m(y, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124m__setstate__\u001b[39m\u001b[38;5;124m'\u001b[39m):\n\u001b[1;32m    272\u001b[0m         y\u001b[38;5;241m.\u001b[39m__setstate__(state)\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:146\u001b[0m, in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    144\u001b[0m copier \u001b[38;5;241m=\u001b[39m _deepcopy_dispatch\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;28mcls\u001b[39m)\n\u001b[1;32m    145\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m copier \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[0;32m--> 146\u001b[0m     y \u001b[38;5;241m=\u001b[39m \u001b[43mcopier\u001b[49m\u001b[43m(\u001b[49m\u001b[43mx\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    147\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m    148\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28missubclass\u001b[39m(\u001b[38;5;28mcls\u001b[39m, \u001b[38;5;28mtype\u001b[39m):\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:230\u001b[0m, in \u001b[0;36m_deepcopy_dict\u001b[0;34m(x, memo, deepcopy)\u001b[0m\n\u001b[1;32m    228\u001b[0m memo[\u001b[38;5;28mid\u001b[39m(x)] \u001b[38;5;241m=\u001b[39m y\n\u001b[1;32m    229\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m key, value \u001b[38;5;129;01min\u001b[39;00m x\u001b[38;5;241m.\u001b[39mitems():\n\u001b[0;32m--> 230\u001b[0m     y[\u001b[43mdeepcopy\u001b[49m\u001b[43m(\u001b[49m\u001b[43mkey\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mmemo\u001b[49m\u001b[43m)\u001b[49m] \u001b[38;5;241m=\u001b[39m deepcopy(value, memo)\n\u001b[1;32m    231\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m y\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/copy.py:138\u001b[0m, in \u001b[0;36mdeepcopy\u001b[0;34m(x, memo, _nil)\u001b[0m\n\u001b[1;32m    135\u001b[0m     memo \u001b[38;5;241m=\u001b[39m {}\n\u001b[1;32m    137\u001b[0m d \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mid\u001b[39m(x)\n\u001b[0;32m--> 138\u001b[0m y \u001b[38;5;241m=\u001b[39m \u001b[43mmemo\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[43md\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_nil\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    139\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m y \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m _nil:\n\u001b[1;32m    140\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m y\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "# Model used to study NAFLD, not cancer\n",
    "\n",
    "iHep, errors = validate_sbml_model('./iHepatocytes2322/iHepatocytes2322_Cobra.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "id": "8e4dff7b",
   "metadata": {},
   "outputs": [
    {
     "ename": "HTTPError",
     "evalue": "400 Client Error: Bad Request for url: https://rest.uniprot.org/uniprotkb/ENSG00000180011.json",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mHTTPError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[89], line 4\u001b[0m\n\u001b[1;32m      2\u001b[0m genes \u001b[38;5;241m=\u001b[39m [gene\u001b[38;5;241m.\u001b[39mid \u001b[38;5;28;01mfor\u001b[39;00m gene \u001b[38;5;129;01min\u001b[39;00m reaction\u001b[38;5;241m.\u001b[39mgenes]\n\u001b[1;32m      3\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m g \u001b[38;5;129;01min\u001b[39;00m genes :\n\u001b[0;32m----> 4\u001b[0m     \u001b[43mUniprotkbClient\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mfetch_one\u001b[49m\u001b[43m(\u001b[49m\u001b[43mg\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/unipressed/dataset/core.py:88\u001b[0m, in \u001b[0;36mDatasetClient.fetch_one\u001b[0;34m(cls, id, format, parse)\u001b[0m\n\u001b[1;32m     74\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m     75\u001b[0m \u001b[38;5;124;03mFetches a single record from this dataset using its ID.\u001b[39;00m\n\u001b[1;32m     76\u001b[0m \n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     83\u001b[0m \u001b[38;5;124;03m    : If parse is True, a dictionary. Otherwise, a file object containing the results in the specified format.\u001b[39;00m\n\u001b[1;32m     84\u001b[0m \u001b[38;5;124;03m\"\"\"\u001b[39;00m\n\u001b[1;32m     85\u001b[0m res \u001b[38;5;241m=\u001b[39m requests\u001b[38;5;241m.\u001b[39mget(\n\u001b[1;32m     86\u001b[0m     \u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mhttps://rest.uniprot.org/\u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mcls\u001b[39m\u001b[38;5;241m.\u001b[39mname()\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m/\u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mid\u001b[39m\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m.\u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mformat\u001b[39m\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m, stream\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mTrue\u001b[39;00m\n\u001b[1;32m     87\u001b[0m )\n\u001b[0;32m---> 88\u001b[0m \u001b[43mres\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mraise_for_status\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m     89\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m parse \u001b[38;5;129;01mand\u001b[39;00m \u001b[38;5;28mformat\u001b[39m \u001b[38;5;241m==\u001b[39m \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mjson\u001b[39m\u001b[38;5;124m\"\u001b[39m:\n\u001b[1;32m     90\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m res\u001b[38;5;241m.\u001b[39mjson()\n",
      "File \u001b[0;32m~/miniconda3/envs/modmetab/lib/python3.9/site-packages/requests/models.py:1021\u001b[0m, in \u001b[0;36mResponse.raise_for_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1016\u001b[0m     http_error_msg \u001b[38;5;241m=\u001b[39m (\n\u001b[1;32m   1017\u001b[0m         \u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mstatus_code\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m Server Error: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00mreason\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m for url: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39murl\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m   1018\u001b[0m     )\n\u001b[1;32m   1020\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m http_error_msg:\n\u001b[0;32m-> 1021\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m HTTPError(http_error_msg, response\u001b[38;5;241m=\u001b[39m\u001b[38;5;28mself\u001b[39m)\n",
      "\u001b[0;31mHTTPError\u001b[0m: 400 Client Error: Bad Request for url: https://rest.uniprot.org/uniprotkb/ENSG00000180011.json"
     ]
    }
   ],
   "source": [
    "for reaction in iHep.reactions :\n",
    "    genes = [gene.id for gene in reaction.genes]\n",
    "    for g in genes :\n",
    "        UniprotkbClient.fetch_one(g)\n",
    "        \n",
    "    #print(genes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "40344bd8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Name</strong></td>\n",
       "                <td>iHepatocytes2322</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>7efbcb106c70</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of metabolites</strong></td>\n",
       "                <td>5242</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of reactions</strong></td>\n",
       "                <td>7930</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of genes</strong></td>\n",
       "                <td>2322</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Number of groups</strong></td>\n",
       "                <td>135</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Objective expression</strong></td>\n",
       "                <td>0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Compartments</strong></td>\n",
       "                <td>c, s, l, r, m, p, g, n</td>\n",
       "            </tr>\n",
       "          </table>"
      ],
      "text/plain": [
       "<Model iHepatocytes2322 at 0x7efbcb106c70>"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "iHep"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f114b755",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "iHep.objective = 'biomass_components'\n",
    "iHep.summary()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "62602c89",
   "metadata": {},
   "source": [
    "HMR2 = cobra.io.read_sbml_model('hmr/HMRdatabase2_00_Cobra.xml-b36c4d8bbafd1a3b8bf3b49e8b7cad35/HMRdatabase2_00_Cobra.xml')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9061bfe2",
   "metadata": {},
   "source": [
    "HMR2.objective = \"biomass_components\"\n",
    "HMR2.summary()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "82944003",
   "metadata": {},
   "source": [
    "#### Reaction bounds correction for Hep_G2 model :\n",
    "For the case where bounds are 'inf' or '-inf'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7f2a8f2e",
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in Hep_G2.reactions:\n",
    "\tif i.upper_bound == float('inf'):\n",
    "\t\ti.upper_bound = float(1000)\n",
    "\n",
    "\tif i.lower_bound == float('-inf'):\n",
    "\t\ti.lower_bound = float(-1000)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "94368d60",
   "metadata": {},
   "source": [
    "#### Checking that the bounds of the shared reactions between Hep_G2 and iHep are the same\n",
    "\n",
    "If they are different, the cell will print each reaction that has different bounds and its bounds.\n",
    "The cell will also copy-paste the bounds from the iHep model's reactions to Hep_G2's"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4a5b9495",
   "metadata": {},
   "outputs": [],
   "source": [
    "i = 0\n",
    "for reaction_Hep_G2 in Hep_G2.reactions :\n",
    "    if reaction_Hep_G2 in iHep.reactions :\n",
    "        #Comparaison des bounds pour une même réaction entre les deux modèles :\n",
    "        iHep_reaction_correspondance = iHep.reactions.get_by_id(reaction_Hep_G2.id)\n",
    "        if reaction_Hep_G2.bounds != iHep_reaction_correspondance.bounds :\n",
    "            print(f\"bounds in Hep_G2 : {reaction_Hep_G2.bounds}\\n---\\nbounds in iHep : {iHep_reaction_correspondance.bounds}\\n#########\\n\")\n",
    "            Hep_G2.reactions.get_by_id(reaction_Hep_G2.id).bounds = iHep_reaction_correspondance.bounds\n",
    "        else :\n",
    "            #print(f\"Reaction {reaction_Hep_G2} (b : {reaction_Hep_G2.bounds}) and {iHep_reaction_correspondance} (b: {iHep_reaction_correspondance.bounds}) have the same bounds\\n\\n\")\n",
    "            i+=1"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e4f32ee8",
   "metadata": {},
   "source": [
    "### Comparaison des réactions en fonction de leur ID et vérification des métabolites composant cette réaction."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "804ca63e",
   "metadata": {},
   "source": [
    "#### Comparaison des réactions du model1 avec celles du model2. Les Réactions seront cherchées dans le model2 à partir de leurs id du model1."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a5422c89",
   "metadata": {},
   "outputs": [],
   "source": [
    "def compare_models_reactions(model1, model2, v = False, fig=True) :\n",
    "    in_both = 0\n",
    "    only_in_model1 = 0\n",
    "    only_in_model2 = 0\n",
    "    model2_reactions = [id_model2.id for id_model2 in model2.reactions]\n",
    "    model1_reactions = [id_model1.id for id_model1 in model1.reactions]\n",
    "    \n",
    "    only_in_model1_list = []\n",
    "    only_in_model2_list = []\n",
    "    in_both_list = []\n",
    "    for rea in model1.reactions :\n",
    "        \n",
    "        if rea.id in model2_reactions :\n",
    "            model2_id = model2.reactions.get_by_id(rea.id)\n",
    "            in_both +=1\n",
    "            in_both_list.append(rea)\n",
    "        else :\n",
    "            model2_id = \"False\"\n",
    "            only_in_model1+=1\n",
    "            only_in_model1_list.append(rea)\n",
    "        if v :\n",
    "            print(f\"reaction_iHep : {rea}\\n {rea.name}\\n---\\nreaction_init: {model2.id} \\n\\n=====\\n\\n\")\n",
    "        \n",
    "    for rea in model2.reactions :\n",
    "        if rea.id in model1_reactions :\n",
    "            pass\n",
    "        else :\n",
    "            only_in_model2 +=1\n",
    "            only_in_model2_list.append(rea)\n",
    "    if fig :\n",
    "        fig, ax = plt.subplots()\n",
    "        both_fig = ax.bar(\"In model1 and model2\", in_both)\n",
    "        only_1_fig = ax.bar(\"only in model1\", only_in_model1)\n",
    "        only_2_fig = ax.bar(\"only in model2\", only_in_model2)\n",
    "\n",
    "        fig.legend([both_fig, only_1_fig, only_2_fig], [f\"{model1.name} vs {model2.name}\", model1.name, model2.name])\n",
    "        return fig, ax\n",
    "    else :\n",
    "        return only_in_model1_list, only_in_model2_list, in_both_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "cfbafbfa",
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "fig, ax = compare_models_reactions(Hep_G2, iHep)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d4ae8e23",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "l1, l2, _ = compare_models_reactions(Hep_G2, iHep, fig=False)\n",
    "print(len(l1))\n",
    "for reaction_to_print in l1 : \n",
    "    print(reaction_to_print.id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "de972c7f",
   "metadata": {},
   "outputs": [],
   "source": [
    "biomass_Hep_G2 = Hep_G2.reactions.biomass_components\n",
    "\n",
    "\n",
    "print_reactions(biomass_Hep_G2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8c5bfbf1",
   "metadata": {},
   "outputs": [],
   "source": [
    "Hep_G2.add_reactions([biomass_iHep])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "03c13830",
   "metadata": {},
   "outputs": [],
   "source": [
    "biomass_Hep_G2 = Hep_G2.reactions.biomass_components.metabolites"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1c6536d2",
   "metadata": {},
   "outputs": [],
   "source": [
    "temp001x= cobra.Metabolite(\n",
    "    'temp001x',\n",
    "    formula='',\n",
    "    name='biomass_',\n",
    "    compartment='C_c')\n",
    "output_biom = cobra.Reaction('EX_temp001x')\n",
    "output_biom.lower_bound = 0.0\n",
    "output_biom.upper_bound = 1000.0\n",
    "output_biom.add_metabolites({temp001x :-1.0})\n",
    "Hep_G2.add_reactions([output_biom])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b1891df6",
   "metadata": {},
   "outputs": [],
   "source": [
    "Hep_G2.reactions.EX_temp001x.lower_bound = 0.0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f5a23c79",
   "metadata": {},
   "outputs": [],
   "source": [
    "Hep_G2.reactions.EX_temp001x"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "bea7f02d",
   "metadata": {},
   "outputs": [],
   "source": [
    "biomass_iHep = iHep.reactions.biomass_components\n",
    "biomass_iHep.metabolites"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "36483fe4",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "biomass_components_ids = []\n",
    "biomass_components_formulas = []\n",
    "for metabolite in model_G2.reactions.get_by_id('biomass_components').metabolites :\n",
    "    print(metabolite, metabolite.name)\n",
    "    biomass_components_ids.append(metabolite.id)\n",
    "    #biomass_components_formulas.append(metabolite.name.split('_')[1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "97e096e1",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "# Checking which reactions are exchange reactions :\n",
    "\n",
    "for reaction in model_iH.reactions :\n",
    "    if len(reaction.compartments) >1 :\n",
    "        print_reactions(reaction)\n",
    "        \n",
    "        #print(reaction, reaction.compartments, [m.name for m in reaction.metabolites])\n",
    "        #print(\"\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a89d887f",
   "metadata": {},
   "outputs": [],
   "source": [
    "biomass_components_"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e3467e7a",
   "metadata": {},
   "outputs": [],
   "source": [
    "for metab in model_init.metabolites :\n",
    "    print(metab.name)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "dded4060",
   "metadata": {},
   "outputs": [],
   "source": [
    "for m_id in biomass_components_ids :\n",
    "    print(model_init.reactions.get_by_id(m_id).name)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "29786e2e",
   "metadata": {},
   "outputs": [],
   "source": [
    "def print_reactions(reaction, flux = 0.0, v=True):\n",
    "    '''\n",
    "    Affiche les reactions d'un modele de façon plus lisible\n",
    "    '''\n",
    "    liste_reactif = [i for i in reaction.reactants]\n",
    "    liste_produit = [i for i in reaction.products]\n",
    "    string = \"\"\n",
    "    string2 = \"\"\n",
    "    if flux >= float('0'):\n",
    "            fleche = \"-->\"\n",
    "    elif flux <= float('0') :\n",
    "            fleche = \"<--\"\n",
    "    else:\n",
    "            fleche = \"<=>\"\n",
    "    \n",
    "    for i in liste_reactif:\n",
    "        if i != liste_reactif[-1]:\n",
    "            string += str(float(abs(reaction.metabolites[i])))\n",
    "            string += \" \"\n",
    "            string += str(i.name)\n",
    "            string += \" + \"\n",
    "            \n",
    "            string2 += str(float(abs(reaction.metabolites[i])))\n",
    "            string2 += \" \"\n",
    "            string2 += str(i.id)\n",
    "            string2 += \" + \"\n",
    "            \n",
    "        else:\n",
    "            string += str(float(abs(reaction.metabolites[i])))\n",
    "            string += \" \"\n",
    "            string += str(i.name)\n",
    "            string += \" \"\n",
    "            \n",
    "            string2 += str(float(abs(reaction.metabolites[i])))\n",
    "            string2 += \" \"\n",
    "            string2 += str(i.id)\n",
    "            string2 += \" \"\n",
    "            \n",
    "    string += fleche\n",
    "    string += \" \"\n",
    "    \n",
    "    string2 += fleche\n",
    "    string2 += \" \"\n",
    "    \n",
    "    \n",
    "    for i in liste_produit:\n",
    "        if i != liste_produit[-1]:\n",
    "            string += str(float(abs(reaction.metabolites[i])))\n",
    "            string += \" \"\n",
    "            string += str(i.name)\n",
    "            string += \" + \"\n",
    "            \n",
    "            string2 += str(float(abs(reaction.metabolites[i])))\n",
    "            string2 += \" \"\n",
    "            string2 += str(i.id)\n",
    "            string2 += \" + \"\n",
    "            \n",
    "        else:\n",
    "            string += str(float(abs(reaction.metabolites[i])))\n",
    "            string += \" \"\n",
    "            string += str(i.name)\n",
    "            \n",
    "            string2 += str(float(abs(reaction.metabolites[i])))\n",
    "            string2 += \" \"\n",
    "            string2 += str(i.id)\n",
    "    if v :    \n",
    "        print(string)\n",
    "        print(\"\")\n",
    "        print(string2)\n",
    "    else :\n",
    "        return(string, string2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c16c5e14",
   "metadata": {},
   "outputs": [],
   "source": [
    "model_iH.objective = \"biomass_components\"\n",
    "model_iH.summary()\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "modmetab",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  },
  "vscode": {
   "interpreter": {
    "hash": "c5aefada315dfae8845832e07b2baab4de16ceb0c7c9b8bc0c07e77ac8bb77ca"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
